{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "gpuType": "T4"
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "MUk9mUkhI8Je"
      },
      "source": [
        "Устанавливаем HMMER"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "1E2ttARVSKuE",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "outputId": "36d6e288-3529-4fd2-c0e1-7da53a8c6cc0"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "--2024-06-15 23:52:06--  http://eddylab.org/software/hmmer/hmmer-3.3.2.tar.gz\n",
            "Resolving eddylab.org (eddylab.org)... 96.126.110.11, 2600:3c03::f03c:91ff:fec8:383c\n",
            "Connecting to eddylab.org (eddylab.org)|96.126.110.11|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 18213049 (17M) [application/x-gzip]\n",
            "Saving to: ‘hmmer-3.3.2.tar.gz’\n",
            "\n",
            "     0K .......... .......... .......... .......... ..........  0%  116K 2m34s\n",
            "    50K .......... .......... .......... .......... ..........  0%  231K 1m55s\n",
            "   100K .......... .......... .......... .......... ..........  0% 34.3M 77s\n",
            "   150K .......... .......... .......... .......... ..........  1%  232K 76s\n",
            "   200K .......... .......... .......... .......... ..........  1% 27.9M 61s\n",
            "   250K .......... .......... .......... .......... ..........  1% 65.3M 51s\n",
            "   300K .......... .......... .......... .......... ..........  1% 60.6M 43s\n",
            "   350K .......... .......... .......... .......... ..........  2% 63.3M 38s\n",
            "   400K .......... .......... .......... .......... ..........  2%  233K 42s\n",
            "   450K .......... .......... .......... .......... ..........  2% 32.8M 38s\n",
            "   500K .......... .......... .......... .......... ..........  3% 68.6M 34s\n",
            "   550K .......... .......... .......... .......... ..........  3% 64.0M 31s\n",
            "   600K .......... .......... .......... .......... ..........  3% 60.8M 29s\n",
            "   650K .......... .......... .......... .......... ..........  3% 49.3M 27s\n",
            "   700K .......... .......... .......... .......... ..........  4% 59.6M 25s\n",
            "   750K .......... .......... .......... .......... ..........  4%  238K 28s\n",
            "   800K .......... .......... .......... .......... ..........  4%  120M 26s\n",
            "   850K .......... .......... .......... .......... ..........  5% 70.5M 24s\n",
            "   900K .......... .......... .......... .......... ..........  5% 59.4M 23s\n",
            "   950K .......... .......... .......... .......... ..........  5% 61.6M 22s\n",
            "  1000K .......... .......... .......... .......... ..........  5%  105M 21s\n",
            "  1050K .......... .......... .......... .......... ..........  6%  260M 20s\n",
            "  1100K .......... .......... .......... .......... ..........  6%  258M 19s\n",
            "  1150K .......... .......... .......... .......... ..........  6%  315M 18s\n",
            "  1200K .......... .......... .......... .......... ..........  7%  274M 17s\n",
            "  1250K .......... .......... .......... .......... ..........  7%  275M 17s\n",
            "  1300K .......... .......... .......... .......... ..........  7%  230M 16s\n",
            "  1350K .......... .......... .......... .......... ..........  7%  286M 15s\n",
            "  1400K .......... .......... .......... .......... ..........  8%  285M 15s\n",
            "  1450K .......... .......... .......... .......... ..........  8%  299M 14s\n",
            "  1500K .......... .......... .......... .......... ..........  8%  236K 16s\n",
            "  1550K .......... .......... .......... .......... ..........  8%  107M 15s\n",
            "  1600K .......... .......... .......... .......... ..........  9%  165M 15s\n",
            "  1650K .......... .......... .......... .......... ..........  9%  298M 14s\n",
            "  1700K .......... .......... .......... .......... ..........  9%  224M 14s\n",
            "  1750K .......... .......... .......... .......... .......... 10%  127M 13s\n",
            "  1800K .......... .......... .......... .......... .......... 10%  132M 13s\n",
            "  1850K .......... .......... .......... .......... .......... 10% 81.6M 13s\n",
            "  1900K .......... .......... .......... .......... .......... 10% 78.2M 12s\n",
            "  1950K .......... .......... .......... .......... .......... 11%  120M 12s\n",
            "  2000K .......... .......... .......... .......... .......... 11%  374M 12s\n",
            "  2050K .......... .......... .......... .......... .......... 11%  471M 11s\n",
            "  2100K .......... .......... .......... .......... .......... 12%  444M 11s\n",
            "  2150K .......... .......... .......... .......... .......... 12%  504M 11s\n",
            "  2200K .......... .......... .......... .......... .......... 12%  540M 10s\n",
            "  2250K .......... .......... .......... .......... .......... 12%  522M 10s\n",
            "  2300K .......... .......... .......... .......... .......... 13%  412M 10s\n",
            "  2350K .......... .......... .......... .......... .......... 13%  548M 10s\n",
            "  2400K .......... .......... .......... .......... .......... 13%  552M 10s\n",
            "  2450K .......... .......... .......... .......... .......... 14%  322M 9s\n",
            "  2500K .......... .......... .......... .......... .......... 14%  244M 9s\n",
            "  2550K .......... .......... .......... .......... .......... 14%  311M 9s\n",
            "  2600K .......... .......... .......... .......... .......... 14%  312M 9s\n",
            "  2650K .......... .......... .......... .......... .......... 15%  301M 8s\n",
            "  2700K .......... .......... .......... .......... .......... 15%  271M 8s\n",
            "  2750K .......... .......... .......... .......... .......... 15%  283M 8s\n",
            "  2800K .......... .......... .......... .......... .......... 16%  303M 8s\n",
            "  2850K .......... .......... .......... .......... .......... 16%  237K 9s\n",
            "  2900K .......... .......... .......... .......... .......... 16% 70.1M 9s\n",
            "  2950K .......... .......... .......... .......... .......... 16%  100M 9s\n",
            "  3000K .......... .......... .......... .......... .......... 17%  124M 8s\n",
            "  3050K .......... .......... .......... .......... .......... 17%  137M 8s\n",
            "  3100K .......... .......... .......... .......... .......... 17%  133M 8s\n",
            "  3150K .......... .......... .......... .......... .......... 17%  303M 8s\n",
            "  3200K .......... .......... .......... .......... .......... 18%  273M 8s\n",
            "  3250K .......... .......... .......... .......... .......... 18%  266M 8s\n",
            "  3300K .......... .......... .......... .......... .......... 18%  231M 7s\n",
            "  3350K .......... .......... .......... .......... .......... 19%  220M 7s\n",
            "  3400K .......... .......... .......... .......... .......... 19%  118M 7s\n",
            "  3450K .......... .......... .......... .......... .......... 19%  114M 7s\n",
            "  3500K .......... .......... .......... .......... .......... 19%  135M 7s\n",
            "  3550K .......... .......... .......... .......... .......... 20%  139M 7s\n",
            "  3600K .......... .......... .......... .......... .......... 20%  107M 7s\n",
            "  3650K .......... .......... .......... .......... .......... 20%  306M 7s\n",
            "  3700K .......... .......... .......... .......... .......... 21%  307M 7s\n",
            "  3750K .......... .......... .......... .......... .......... 21%  308M 6s\n",
            "  3800K .......... .......... .......... .......... .......... 21%  239M 6s\n",
            "  3850K .......... .......... .......... .......... .......... 21%  270M 6s\n",
            "  3900K .......... .......... .......... .......... .......... 22%  281M 6s\n",
            "  3950K .......... .......... .......... .......... .......... 22%  293M 6s\n",
            "  4000K .......... .......... .......... .......... .......... 22%  254M 6s\n",
            "  4050K .......... .......... .......... .......... .......... 23%  285M 6s\n",
            "  4100K .......... .......... .......... .......... .......... 23%  265M 6s\n",
            "  4150K .......... .......... .......... .......... .......... 23%  288M 6s\n",
            "  4200K .......... .......... .......... .......... .......... 23%  262M 6s\n",
            "  4250K .......... .......... .......... .......... .......... 24%  316M 5s\n",
            "  4300K .......... .......... .......... .......... .......... 24%  316M 5s\n",
            "  4350K .......... .......... .......... .......... .......... 24%  318M 5s\n",
            "  4400K .......... .......... .......... .......... .......... 25%  243M 5s\n",
            "  4450K .......... .......... .......... .......... .......... 25%  311M 5s\n",
            "  4500K .......... .......... .......... .......... .......... 25%  309M 5s\n",
            "  4550K .......... .......... .......... .......... .......... 25%  310M 5s\n",
            "  4600K .......... .......... .......... .......... .......... 26%  269M 5s\n",
            "  4650K .......... .......... .......... .......... .......... 26%  311M 5s\n",
            "  4700K .......... .......... .......... .......... .......... 26%  291M 5s\n",
            "  4750K .......... .......... .......... .......... .......... 26%  306M 5s\n",
            "  4800K .......... .......... .......... .......... .......... 27%  271M 5s\n",
            "  4850K .......... .......... .......... .......... .......... 27%  312M 5s\n",
            "  4900K .......... .......... .......... .......... .......... 27%  315M 5s\n",
            "  4950K .......... .......... .......... .......... .......... 28%  317M 4s\n",
            "  5000K .......... .......... .......... .......... .......... 28%  244M 4s\n",
            "  5050K .......... .......... .......... .......... .......... 28%  315M 4s\n",
            "  5100K .......... .......... .......... .......... .......... 28%  307M 4s\n",
            "  5150K .......... .......... .......... .......... .......... 29%  269M 4s\n",
            "  5200K .......... .......... .......... .......... .......... 29%  254M 4s\n",
            "  5250K .......... .......... .......... .......... .......... 29%  283M 4s\n",
            "  5300K .......... .......... .......... .......... .......... 30%  278M 4s\n",
            "  5350K .......... .......... .......... .......... .......... 30%  293M 4s\n",
            "  5400K .......... .......... .......... .......... .......... 30%  256M 4s\n",
            "  5450K .......... .......... .......... .......... .......... 30%  275M 4s\n",
            "  5500K .......... .......... .......... .......... .......... 31%  317M 4s\n",
            "  5550K .......... .......... .......... .......... .......... 31%  278M 4s\n",
            "  5600K .......... .......... .......... .......... .......... 31%  266M 4s\n",
            "  5650K .......... .......... .......... .......... .......... 32%  186M 4s\n",
            "  5700K .......... .......... .......... .......... .......... 32%  299M 4s\n",
            "  5750K .......... .......... .......... .......... .......... 32%  244K 4s\n",
            "  5800K .......... .......... .......... .......... .......... 32%  107M 4s\n",
            "  5850K .......... .......... .......... .......... .......... 33%  127M 4s\n",
            "  5900K .......... .......... .......... .......... .......... 33%  143M 4s\n",
            "  5950K .......... .......... .......... .......... .......... 33%  210M 4s\n",
            "  6000K .......... .......... .......... .......... .......... 34%  268M 4s\n",
            "  6050K .......... .......... .......... .......... .......... 34%  289M 4s\n",
            "  6100K .......... .......... .......... .......... .......... 34%  294M 4s\n",
            "  6150K .......... .......... .......... .......... .......... 34%  311M 4s\n",
            "  6200K .......... .......... .......... .......... .......... 35%  278M 4s\n",
            "  6250K .......... .......... .......... .......... .......... 35%  254M 4s\n",
            "  6300K .......... .......... .......... .......... .......... 35%  270M 4s\n",
            "  6350K .......... .......... .......... .......... .......... 35%  290M 3s\n",
            "  6400K .......... .......... .......... .......... .......... 36%  192M 3s\n",
            "  6450K .......... .......... .......... .......... .......... 36%  262M 3s\n",
            "  6500K .......... .......... .......... .......... .......... 36%  176M 3s\n",
            "  6550K .......... .......... .......... .......... .......... 37%  147M 3s\n",
            "  6600K .......... .......... .......... .......... .......... 37%  113M 3s\n",
            "  6650K .......... .......... .......... .......... .......... 37%  146M 3s\n",
            "  6700K .......... .......... .......... .......... .......... 37%  127M 3s\n",
            "  6750K .......... .......... .......... .......... .......... 38%  209M 3s\n",
            "  6800K .......... .......... .......... .......... .......... 38%  275M 3s\n",
            "  6850K .......... .......... .......... .......... .......... 38%  307M 3s\n",
            "  6900K .......... .......... .......... .......... .......... 39%  289M 3s\n",
            "  6950K .......... .......... .......... .......... .......... 39%  310M 3s\n",
            "  7000K .......... .......... .......... .......... .......... 39%  265M 3s\n",
            "  7050K .......... .......... .......... .......... .......... 39%  311M 3s\n",
            "  7100K .......... .......... .......... .......... .......... 40%  310M 3s\n",
            "  7150K .......... .......... .......... .......... .......... 40%  309M 3s\n",
            "  7200K .......... .......... .......... .......... .......... 40%  254M 3s\n",
            "  7250K .......... .......... .......... .......... .......... 41%  313M 3s\n",
            "  7300K .......... .......... .......... .......... .......... 41%  300M 3s\n",
            "  7350K .......... .......... .......... .......... .......... 41%  319M 3s\n",
            "  7400K .......... .......... .......... .......... .......... 41%  276M 3s\n",
            "  7450K .......... .......... .......... .......... .......... 42%  273M 3s\n",
            "  7500K .......... .......... .......... .......... .......... 42%  278M 3s\n",
            "  7550K .......... .......... .......... .......... .......... 42%  317M 3s\n",
            "  7600K .......... .......... .......... .......... .......... 43%  260M 3s\n",
            "  7650K .......... .......... .......... .......... .......... 43%  315M 3s\n",
            "  7700K .......... .......... .......... .......... .......... 43%  313M 3s\n",
            "  7750K .......... .......... .......... .......... .......... 43%  319M 3s\n",
            "  7800K .......... .......... .......... .......... .......... 44%  256M 2s\n",
            "  7850K .......... .......... .......... .......... .......... 44%  313M 2s\n",
            "  7900K .......... .......... .......... .......... .......... 44%  299M 2s\n",
            "  7950K .......... .......... .......... .......... .......... 44%  310M 2s\n",
            "  8000K .......... .......... .......... .......... .......... 45%  243M 2s\n",
            "  8050K .......... .......... .......... .......... .......... 45%  302M 2s\n",
            "  8100K .......... .......... .......... .......... .......... 45%  266M 2s\n",
            "  8150K .......... .......... .......... .......... .......... 46%  241K 3s\n",
            "  8200K .......... .......... .......... .......... .......... 46%  129M 3s\n",
            "  8250K .......... .......... .......... .......... .......... 46%  118M 2s\n",
            "  8300K .......... .......... .......... .......... .......... 46%  144M 2s\n",
            "  8350K .......... .......... .......... .......... .......... 47%  244M 2s\n",
            "  8400K .......... .......... .......... .......... .......... 47%  162M 2s\n",
            "  8450K .......... .......... .......... .......... .......... 47%  111M 2s\n",
            "  8500K .......... .......... .......... .......... .......... 48%  114M 2s\n",
            "  8550K .......... .......... .......... .......... .......... 48%  105M 2s\n",
            "  8600K .......... .......... .......... .......... .......... 48%  114M 2s\n",
            "  8650K .......... .......... .......... .......... .......... 48%  157M 2s\n",
            "  8700K .......... .......... .......... .......... .......... 49%  264M 2s\n",
            "  8750K .......... .......... .......... .......... .......... 49%  290M 2s\n",
            "  8800K .......... .......... .......... .......... .......... 49%  280M 2s\n",
            "  8850K .......... .......... .......... .......... .......... 50%  275M 2s\n",
            "  8900K .......... .......... .......... .......... .......... 50%  313M 2s\n",
            "  8950K .......... .......... .......... .......... .......... 50%  317M 2s\n",
            "  9000K .......... .......... .......... .......... .......... 50%  265M 2s\n",
            "  9050K .......... .......... .......... .......... .......... 51%  266M 2s\n",
            "  9100K .......... .......... .......... .......... .......... 51%  319M 2s\n",
            "  9150K .......... .......... .......... .......... .......... 51%  315M 2s\n",
            "  9200K .......... .......... .......... .......... .......... 52%  308M 2s\n",
            "  9250K .......... .......... .......... .......... .......... 52%  277M 2s\n",
            "  9300K .......... .......... .......... .......... .......... 52%  283M 2s\n",
            "  9350K .......... .......... .......... .......... .......... 52%  311M 2s\n",
            "  9400K .......... .......... .......... .......... .......... 53%  310M 2s\n",
            "  9450K .......... .......... .......... .......... .......... 53%  267M 2s\n",
            "  9500K .......... .......... .......... .......... .......... 53%  311M 2s\n",
            "  9550K .......... .......... .......... .......... .......... 53%  326M 2s\n",
            "  9600K .......... .......... .......... .......... .......... 54%  284M 2s\n",
            "  9650K .......... .......... .......... .......... .......... 54%  262M 2s\n",
            "  9700K .......... .......... .......... .......... .......... 54%  318M 2s\n",
            "  9750K .......... .......... .......... .......... .......... 55%  316M 2s\n",
            "  9800K .......... .......... .......... .......... .......... 55%  299M 2s\n",
            "  9850K .......... .......... .......... .......... .......... 55%  282M 2s\n",
            "  9900K .......... .......... .......... .......... .......... 55%  285M 2s\n",
            "  9950K .......... .......... .......... .......... .......... 56%  312M 2s\n",
            " 10000K .......... .......... .......... .......... .......... 56%  299M 2s\n",
            " 10050K .......... .......... .......... .......... .......... 56%  275M 2s\n",
            " 10100K .......... .......... .......... .......... .......... 57%  320M 2s\n",
            " 10150K .......... .......... .......... .......... .......... 57%  318M 2s\n",
            " 10200K .......... .......... .......... .......... .......... 57%  301M 2s\n",
            " 10250K .......... .......... .......... .......... .......... 57%  260M 2s\n",
            " 10300K .......... .......... .......... .......... .......... 58%  314M 2s\n",
            " 10350K .......... .......... .......... .......... .......... 58%  320M 2s\n",
            " 10400K .......... .......... .......... .......... .......... 58%  311M 2s\n",
            " 10450K .......... .......... .......... .......... .......... 59%  279M 2s\n",
            " 10500K .......... .......... .......... .......... .......... 59%  307M 1s\n",
            " 10550K .......... .......... .......... .......... .......... 59%  287M 1s\n",
            " 10600K .......... .......... .......... .......... .......... 59%  310M 1s\n",
            " 10650K .......... .......... .......... .......... .......... 60%  275M 1s\n",
            " 10700K .......... .......... .......... .......... .......... 60%  316M 1s\n",
            " 10750K .......... .......... .......... .......... .......... 60%  306M 1s\n",
            " 10800K .......... .......... .......... .......... .......... 61%  311M 1s\n",
            " 10850K .......... .......... .......... .......... .......... 61%  243K 2s\n",
            " 10900K .......... .......... .......... .......... .......... 61%  145M 1s\n",
            " 10950K .......... .......... .......... .......... .......... 61%  168M 1s\n",
            " 11000K .......... .......... .......... .......... .......... 62%  259M 1s\n",
            " 11050K .......... .......... .......... .......... .......... 62%  171M 1s\n",
            " 11100K .......... .......... .......... .......... .......... 62% 99.5M 1s\n",
            " 11150K .......... .......... .......... .......... .......... 62%  133M 1s\n",
            " 11200K .......... .......... .......... .......... .......... 63%  135M 1s\n",
            " 11250K .......... .......... .......... .......... .......... 63%  132M 1s\n",
            " 11300K .......... .......... .......... .......... .......... 63%  129M 1s\n",
            " 11350K .......... .......... .......... .......... .......... 64%  139M 1s\n",
            " 11400K .......... .......... .......... .......... .......... 64%  167M 1s\n",
            " 11450K .......... .......... .......... .......... .......... 64%  290M 1s\n",
            " 11500K .......... .......... .......... .......... .......... 64%  258M 1s\n",
            " 11550K .......... .......... .......... .......... .......... 65%  291M 1s\n",
            " 11600K .......... .......... .......... .......... .......... 65%  276M 1s\n",
            " 11650K .......... .......... .......... .......... .......... 65%  286M 1s\n",
            " 11700K .......... .......... .......... .......... .......... 66%  251M 1s\n",
            " 11750K .......... .......... .......... .......... .......... 66%  290M 1s\n",
            " 11800K .......... .......... .......... .......... .......... 66%  288M 1s\n",
            " 11850K .......... .......... .......... .......... .......... 66%  274M 1s\n",
            " 11900K .......... .......... .......... .......... .......... 67%  248M 1s\n",
            " 11950K .......... .......... .......... .......... .......... 67%  298M 1s\n",
            " 12000K .......... .......... .......... .......... .......... 67%  289M 1s\n",
            " 12050K .......... .......... .......... .......... .......... 68%  353M 1s\n",
            " 12100K .......... .......... .......... .......... .......... 68%  472M 1s\n",
            " 12150K .......... .......... .......... .......... .......... 68%  547M 1s\n",
            " 12200K .......... .......... .......... .......... .......... 68%  551M 1s\n",
            " 12250K .......... .......... .......... .......... .......... 69%  215M 1s\n",
            " 12300K .......... .......... .......... .......... .......... 69%  235M 1s\n",
            " 12350K .......... .......... .......... .......... .......... 69%  293M 1s\n",
            " 12400K .......... .......... .......... .......... .......... 69%  300M 1s\n",
            " 12450K .......... .......... .......... .......... .......... 70%  296M 1s\n",
            " 12500K .......... .......... .......... .......... .......... 70%  226M 1s\n",
            " 12550K .......... .......... .......... .......... .......... 70%  283M 1s\n",
            " 12600K .......... .......... .......... .......... .......... 71%  298M 1s\n",
            " 12650K .......... .......... .......... .......... .......... 71%  293M 1s\n",
            " 12700K .......... .......... .......... .......... .......... 71%  257M 1s\n",
            " 12750K .......... .......... .......... .......... .......... 71%  276M 1s\n",
            " 12800K .......... .......... .......... .......... .......... 72%  286M 1s\n",
            " 12850K .......... .......... .......... .......... .......... 72%  293M 1s\n",
            " 12900K .......... .......... .......... .......... .......... 72%  254M 1s\n",
            " 12950K .......... .......... .......... .......... .......... 73%  290M 1s\n",
            " 13000K .......... .......... .......... .......... .......... 73%  298M 1s\n",
            " 13050K .......... .......... .......... .......... .......... 73%  267M 1s\n",
            " 13100K .......... .......... .......... .......... .......... 73%  254M 1s\n",
            " 13150K .......... .......... .......... .......... .......... 74%  298M 1s\n",
            " 13200K .......... .......... .......... .......... .......... 74%  293M 1s\n",
            " 13250K .......... .......... .......... .......... .......... 74%  295M 1s\n",
            " 13300K .......... .......... .......... .......... .......... 75%  242K 1s\n",
            " 13350K .......... .......... .......... .......... .......... 75%  172M 1s\n",
            " 13400K .......... .......... .......... .......... .......... 75%  181M 1s\n",
            " 13450K .......... .......... .......... .......... .......... 75%  149M 1s\n",
            " 13500K .......... .......... .......... .......... .......... 76%  153M 1s\n",
            " 13550K .......... .......... .......... .......... .......... 76%  152M 1s\n",
            " 13600K .......... .......... .......... .......... .......... 76%  142M 1s\n",
            " 13650K .......... .......... .......... .......... .......... 77%  132M 1s\n",
            " 13700K .......... .......... .......... .......... .......... 77%  148M 1s\n",
            " 13750K .......... .......... .......... .......... .......... 77%  120M 1s\n",
            " 13800K .......... .......... .......... .......... .......... 77%  170M 1s\n",
            " 13850K .......... .......... .......... .......... .......... 78%  168M 1s\n",
            " 13900K .......... .......... .......... .......... .......... 78%  135M 1s\n",
            " 13950K .......... .......... .......... .......... .......... 78%  263M 1s\n",
            " 14000K .......... .......... .......... .......... .......... 78%  270M 1s\n",
            " 14050K .......... .......... .......... .......... .......... 79%  251M 1s\n",
            " 14100K .......... .......... .......... .......... .......... 79%  288M 1s\n",
            " 14150K .......... .......... .......... .......... .......... 79%  274M 1s\n",
            " 14200K .......... .......... .......... .......... .......... 80%  281M 1s\n",
            " 14250K .......... .......... .......... .......... .......... 80%  246M 1s\n",
            " 14300K .......... .......... .......... .......... .......... 80%  293M 1s\n",
            " 14350K .......... .......... .......... .......... .......... 80%  298M 1s\n",
            " 14400K .......... .......... .......... .......... .......... 81%  271M 1s\n",
            " 14450K .......... .......... .......... .......... .......... 81%  242M 1s\n",
            " 14500K .......... .......... .......... .......... .......... 81%  296M 1s\n",
            " 14550K .......... .......... .......... .......... .......... 82%  293M 1s\n",
            " 14600K .......... .......... .......... .......... .......... 82%  293M 1s\n",
            " 14650K .......... .......... .......... .......... .......... 82%  254M 1s\n",
            " 14700K .......... .......... .......... .......... .......... 82%  274M 1s\n",
            " 14750K .......... .......... .......... .......... .......... 83%  290M 1s\n",
            " 14800K .......... .......... .......... .......... .......... 83%  298M 1s\n",
            " 14850K .......... .......... .......... .......... .......... 83%  255M 1s\n",
            " 14900K .......... .......... .......... .......... .......... 84%  297M 0s\n",
            " 14950K .......... .......... .......... .......... .......... 84%  296M 0s\n",
            " 15000K .......... .......... .......... .......... .......... 84%  272M 0s\n",
            " 15050K .......... .......... .......... .......... .......... 84%  253M 0s\n",
            " 15100K .......... .......... .......... .......... .......... 85%  288M 0s\n",
            " 15150K .......... .......... .......... .......... .......... 85%  293M 0s\n",
            " 15200K .......... .......... .......... .......... .......... 85%  314M 0s\n",
            " 15250K .......... .......... .......... .......... .......... 86%  272M 0s\n",
            " 15300K .......... .......... .......... .......... .......... 86%  259M 0s\n",
            " 15350K .......... .......... .......... .......... .......... 86%  293M 0s\n",
            " 15400K .......... .......... .......... .......... .......... 86%  288M 0s\n",
            " 15450K .......... .......... .......... .......... .......... 87%  260M 0s\n",
            " 15500K .......... .......... .......... .......... .......... 87%  295M 0s\n",
            " 15550K .......... .......... .......... .......... .......... 87%  268M 0s\n",
            " 15600K .......... .......... .......... .......... .......... 87%  303M 0s\n",
            " 15650K .......... .......... .......... .......... .......... 88%  233M 0s\n",
            " 15700K .......... .......... .......... .......... .......... 88%  298M 0s\n",
            " 15750K .......... .......... .......... .......... .......... 88%  293M 0s\n",
            " 15800K .......... .......... .......... .......... .......... 89%  308M 0s\n",
            " 15850K .......... .......... .......... .......... .......... 89%  234M 0s\n",
            " 15900K .......... .......... .......... .......... .......... 89%  315M 0s\n",
            " 15950K .......... .......... .......... .......... .......... 89%  243K 0s\n",
            " 16000K .......... .......... .......... .......... .......... 90%  126M 0s\n",
            " 16050K .......... .......... .......... .......... .......... 90%  154M 0s\n",
            " 16100K .......... .......... .......... .......... .......... 90%  126M 0s\n",
            " 16150K .......... .......... .......... .......... .......... 91%  154M 0s\n",
            " 16200K .......... .......... .......... .......... .......... 91%  251M 0s\n",
            " 16250K .......... .......... .......... .......... .......... 91%  271M 0s\n",
            " 16300K .......... .......... .......... .......... .......... 91%  182M 0s\n",
            " 16350K .......... .......... .......... .......... .......... 92%  132M 0s\n",
            " 16400K .......... .......... .......... .......... .......... 92%  238M 0s\n",
            " 16450K .......... .......... .......... .......... .......... 92%  285M 0s\n",
            " 16500K .......... .......... .......... .......... .......... 93%  253M 0s\n",
            " 16550K .......... .......... .......... .......... .......... 93%  268M 0s\n",
            " 16600K .......... .......... .......... .......... .......... 93%  282M 0s\n",
            " 16650K .......... .......... .......... .......... .......... 93%  290M 0s\n",
            " 16700K .......... .......... .......... .......... .......... 94%  247M 0s\n",
            " 16750K .......... .......... .......... .......... .......... 94%  316M 0s\n",
            " 16800K .......... .......... .......... .......... .......... 94%  281M 0s\n",
            " 16850K .......... .......... .......... .......... .......... 95%  316M 0s\n",
            " 16900K .......... .......... .......... .......... .......... 95%  258M 0s\n",
            " 16950K .......... .......... .......... .......... .......... 95%  314M 0s\n",
            " 17000K .......... .......... .......... .......... .......... 95%  317M 0s\n",
            " 17050K .......... .......... .......... .......... .......... 96%  311M 0s\n",
            " 17100K .......... .......... .......... .......... .......... 96%  254M 0s\n",
            " 17150K .......... .......... .......... .......... .......... 96%  314M 0s\n",
            " 17200K .......... .......... .......... .......... .......... 96%  306M 0s\n",
            " 17250K .......... .......... .......... .......... .......... 97%  315M 0s\n",
            " 17300K .......... .......... .......... .......... .......... 97%  270M 0s\n",
            " 17350K .......... .......... .......... .......... .......... 97%  318M 0s\n",
            " 17400K .......... .......... .......... .......... .......... 98%  298M 0s\n",
            " 17450K .......... .......... .......... .......... .......... 98%  312M 0s\n",
            " 17500K .......... .......... .......... .......... .......... 98%  269M 0s\n",
            " 17550K .......... .......... .......... .......... .......... 98%  305M 0s\n",
            " 17600K .......... .......... .......... .......... .......... 99%  309M 0s\n",
            " 17650K .......... .......... .......... .......... .......... 99%  324M 0s\n",
            " 17700K .......... .......... .......... .......... .......... 99%  274M 0s\n",
            " 17750K .......... .......... .......... ......               100%  297M=2.8s\n",
            "\n",
            "2024-06-15 23:52:09 (6.15 MB/s) - ‘hmmer-3.3.2.tar.gz’ saved [18213049/18213049]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "%%bash\n",
        "wget http://eddylab.org/software/hmmer/hmmer-3.3.2.tar.gz\n",
        "tar xf hmmer-3.3.2.tar.gz\n",
        "cd hmmer-3.3.2"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "FZjhG2leSWW9",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "outputId": "5caa3ed8-232f-4cea-e0db-62bee1d091cc"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "configure: Configuring HMMER3 for your system.\n",
            "checking build system type... x86_64-pc-linux-gnu\n",
            "checking host system type... x86_64-pc-linux-gnu\n",
            "checking whether to compile using MPI... no\n",
            "checking for gcc... gcc\n",
            "checking whether the C compiler works... yes\n",
            "checking for C compiler default output file name... a.out\n",
            "checking for suffix of executables... \n",
            "checking whether we are cross compiling... no\n",
            "checking for suffix of object files... o\n",
            "checking whether we are using the GNU C compiler... yes\n",
            "checking whether gcc accepts -g... yes\n",
            "checking for gcc option to accept ISO C89... none needed\n",
            "checking for gcc option to accept ISO C99... none needed\n",
            "checking for gcc option to accept ISO Standard C... (cached) none needed\n",
            "checking how to run the C preprocessor... gcc -E\n",
            "checking for a BSD-compatible install... /usr/bin/install -c\n",
            "checking for strip... strip\n",
            "checking for ranlib... ranlib\n",
            "checking for ar... /usr/bin/ar\n",
            "checking whether ln -s works... yes\n",
            "checking for a sed that does not truncate output... /usr/bin/sed\n",
            "checking for grep that handles long lines and -e... /usr/bin/grep\n",
            "checking for egrep... /usr/bin/grep -E\n",
            "checking whether gcc is Clang... no\n",
            "checking whether pthreads work with -pthread... yes\n",
            "checking for joinable pthread attribute... PTHREAD_CREATE_JOINABLE\n",
            "checking whether more special flags are required for pthreads... no\n",
            "checking for PTHREAD_PRIO_INHERIT... yes\n",
            "checking whether gcc can compile our Altivec/VMX code... no\n",
            "checking whether gcc can compile our SSE code... yes\n",
            "configure: Activating Intel/AMD SSE vector DP implementation\n",
            "checking whether _MM_SET_FLUSH_ZERO_MODE is supported... yes\n",
            "checking compiler support --start-group... yes\n",
            "checking for gzip... /usr/bin/gzip\n",
            "checking for python3... yes\n",
            "checking for library containing clock_gettime... none required\n",
            "checking for ANSI C header files... yes\n",
            "checking for sys/types.h... yes\n",
            "checking for sys/stat.h... yes\n",
            "checking for stdlib.h... yes\n",
            "checking for string.h... yes\n",
            "checking for memory.h... yes\n",
            "checking for strings.h... yes\n",
            "checking for inttypes.h... yes\n",
            "checking for stdint.h... yes\n",
            "checking for unistd.h... yes\n",
            "checking endian.h usability... yes\n",
            "checking endian.h presence... yes\n",
            "checking for endian.h... yes\n",
            "checking for inttypes.h... (cached) yes\n",
            "checking for stdint.h... (cached) yes\n",
            "checking for unistd.h... (cached) yes\n",
            "checking for sys/types.h... (cached) yes\n",
            "checking netinet/in.h usability... yes\n",
            "checking netinet/in.h presence... yes\n",
            "checking for netinet/in.h... yes\n",
            "checking sys/param.h usability... yes\n",
            "checking sys/param.h presence... yes\n",
            "checking for sys/param.h... yes\n",
            "checking for sys/sysctl.h... no\n",
            "checking for uint8_t... yes\n",
            "checking for uint16_t... yes\n",
            "checking for uint32_t... yes\n",
            "checking for uint64_t... yes\n",
            "checking for off_t... yes\n",
            "checking whether byte ordering is bigendian... no\n",
            "checking for __attribute__((noreturn))... yes\n",
            "checking for __attribute__((format))... yes\n",
            "checking for mkstemp... yes\n",
            "checking for popen... yes\n",
            "checking for putenv... yes\n",
            "checking for strcasecmp... yes\n",
            "checking for strsep... yes\n",
            "checking for times... yes\n",
            "checking for getpid... yes\n",
            "checking for sysctl... no\n",
            "checking for sysconf... yes\n",
            "checking for getcwd... yes\n",
            "checking for chmod... yes\n",
            "checking for stat... yes\n",
            "checking for fstat... yes\n",
            "checking for erfc... no\n",
            "checking for library containing ntohs... none required\n",
            "checking for library containing ntohl... none required\n",
            "checking for library containing htons... none required\n",
            "checking for library containing htonl... none required\n",
            "checking for library containing socket... none required\n",
            "checking for library containing inet_pton... none required\n",
            "checking for _LARGEFILE_SOURCE value needed for large files... no\n",
            "checking for special C compiler options needed for large files... no\n",
            "checking for _FILE_OFFSET_BITS value needed for large files... no\n",
            "configure: creating ./config.status\n",
            "config.status: creating Makefile\n",
            "config.status: creating src/Makefile\n",
            "config.status: creating testsuite/Makefile\n",
            "config.status: creating profmark/Makefile\n",
            "config.status: creating src/impl_sse/Makefile\n",
            "config.status: creating documentation/Makefile\n",
            "config.status: creating documentation/man/Makefile\n",
            "config.status: creating documentation/userguide/Makefile\n",
            "config.status: creating documentation/userguide/inclusions/Makefile\n",
            "config.status: creating libdivsufsort/Makefile\n",
            "config.status: creating easel/Makefile\n",
            "config.status: creating easel/miniapps/Makefile\n",
            "config.status: creating easel/testsuite/Makefile\n",
            "config.status: creating easel/documentation/Makefile\n",
            "config.status: creating documentation/userguide/titlepage.tex\n",
            "config.status: creating documentation/userguide/titlepage_daemon.tex\n",
            "config.status: creating documentation/userguide/copyright.tex\n",
            "config.status: creating documentation/man/alimask.man\n",
            "config.status: creating documentation/man/hmmalign.man\n",
            "config.status: creating documentation/man/hmmbuild.man\n",
            "config.status: creating documentation/man/hmmc2.man\n",
            "config.status: creating documentation/man/hmmconvert.man\n",
            "config.status: creating documentation/man/hmmemit.man\n",
            "config.status: creating documentation/man/hmmer.man\n",
            "config.status: creating documentation/man/hmmfetch.man\n",
            "config.status: creating documentation/man/hmmlogo.man\n",
            "config.status: creating documentation/man/hmmpgmd.man\n",
            "config.status: creating documentation/man/hmmpgmd_shard.man\n",
            "config.status: creating documentation/man/hmmpress.man\n",
            "config.status: creating documentation/man/hmmscan.man\n",
            "config.status: creating documentation/man/hmmsearch.man\n",
            "config.status: creating documentation/man/hmmsim.man\n",
            "config.status: creating documentation/man/hmmstat.man\n",
            "config.status: creating documentation/man/jackhmmer.man\n",
            "config.status: creating documentation/man/makehmmerdb.man\n",
            "config.status: creating documentation/man/nhmmer.man\n",
            "config.status: creating documentation/man/nhmmscan.man\n",
            "config.status: creating documentation/man/phmmer.man\n",
            "config.status: creating easel/miniapps/esl-afetch.man\n",
            "config.status: creating easel/miniapps/esl-alimanip.man\n",
            "config.status: creating easel/miniapps/esl-alimap.man\n",
            "config.status: creating easel/miniapps/esl-alimask.man\n",
            "config.status: creating easel/miniapps/esl-alimerge.man\n",
            "config.status: creating easel/miniapps/esl-alipid.man\n",
            "config.status: creating easel/miniapps/esl-alirev.man\n",
            "config.status: creating easel/miniapps/esl-alistat.man\n",
            "config.status: creating easel/miniapps/esl-compalign.man\n",
            "config.status: creating easel/miniapps/esl-compstruct.man\n",
            "config.status: creating easel/miniapps/esl-construct.man\n",
            "config.status: creating easel/miniapps/esl-histplot.man\n",
            "config.status: creating easel/miniapps/esl-mask.man\n",
            "config.status: creating easel/miniapps/esl-mixdchlet.man\n",
            "config.status: creating easel/miniapps/esl-reformat.man\n",
            "config.status: creating easel/miniapps/esl-selectn.man\n",
            "config.status: creating easel/miniapps/esl-seqrange.man\n",
            "config.status: creating easel/miniapps/esl-seqstat.man\n",
            "config.status: creating easel/miniapps/esl-sfetch.man\n",
            "config.status: creating easel/miniapps/esl-shuffle.man\n",
            "config.status: creating easel/miniapps/esl-ssdraw.man\n",
            "config.status: creating easel/miniapps/esl-translate.man\n",
            "config.status: creating easel/miniapps/esl-weight.man\n",
            "config.status: creating easel/decoy_config.h\n",
            "config.status: creating src/p7_config.h\n",
            "config.status: creating easel/esl_config.h\n",
            "config.status: creating libdivsufsort/divsufsort.h\n",
            "config.status: linking src/impl_sse to src/impl\n",
            "\n",
            "\n",
            "HMMER configuration:\n",
            "   compiler:             gcc -O3   -pthread \n",
            "   host:                 x86_64-pc-linux-gnu\n",
            "   linker:               \n",
            "   libraries:              \n",
            "   DP implementation:    sse\n",
            "\n",
            "Now do 'make'  to build HMMER, and optionally:\n",
            "       'make check'  to run self tests,\n",
            "       'make install'  to install programs and man pages,\n",
            "       '(cd easel; make install)'  to install Easel tools.\n",
            "\n",
            "     SUBDIR easel\n",
            "     CC easel.o\n",
            "     CC esl_alloc.o\n",
            "     CC esl_alphabet.o\n",
            "     CC esl_arr2.o\n",
            "     CC esl_arr3.o\n",
            "     CC esl_bitfield.o\n",
            "     CC esl_buffer.o\n",
            "     CC esl_cluster.o\n",
            "     CC esl_composition.o\n",
            "     CC esl_cpu.o\n",
            "     CC esl_dirichlet.o\n",
            "     CC esl_distance.o\n",
            "     CC esl_dmatrix.o\n",
            "     CC esl_dsqdata.o\n",
            "     CC esl_exponential.o\n",
            "     CC esl_fileparser.o\n",
            "     CC esl_gamma.o\n",
            "     CC esl_gencode.o\n",
            "     CC esl_getopts.o\n",
            "     CC esl_gev.o\n",
            "     CC esl_graph.o\n",
            "     CC esl_gumbel.o\n",
            "     CC esl_heap.o\n",
            "     CC esl_histogram.o\n",
            "     CC esl_hmm.o\n",
            "     CC esl_huffman.o\n",
            "     CC esl_hyperexp.o\n",
            "     CC esl_json.o\n",
            "     CC esl_keyhash.o\n",
            "     CC esl_matrixops.o\n",
            "     CC esl_mem.o\n",
            "     CC esl_minimizer.o\n",
            "     CC esl_mixdchlet.o\n",
            "     CC esl_mixgev.o\n",
            "     CC esl_mpi.o\n",
            "     CC esl_msa.o\n",
            "     CC esl_msacluster.o\n",
            "     CC esl_msafile.o\n",
            "     CC esl_msafile2.o\n",
            "     CC esl_msafile_a2m.o\n",
            "     CC esl_msafile_afa.o\n",
            "     CC esl_msafile_clustal.o\n",
            "     CC esl_msafile_phylip.o\n",
            "     CC esl_msafile_psiblast.o\n",
            "     CC esl_msafile_selex.o\n",
            "     CC esl_msafile_stockholm.o\n",
            "     CC esl_msashuffle.o\n",
            "     CC esl_msaweight.o\n",
            "     CC esl_normal.o\n",
            "     CC esl_paml.o\n",
            "     CC esl_quicksort.o\n",
            "     CC esl_random.o\n",
            "     CC esl_rand64.o\n",
            "     CC esl_randomseq.o\n",
            "     CC esl_ratematrix.o\n",
            "     CC esl_recorder.o\n",
            "     CC esl_red_black.o\n",
            "     CC esl_regexp.o\n",
            "     CC esl_rootfinder.o\n",
            "     CC esl_scorematrix.o\n",
            "     CC esl_sq.o\n",
            "     CC esl_sqio.o\n",
            "     CC esl_sqio_ascii.o\n",
            "     CC esl_sqio_ncbi.o\n",
            "     CC esl_ssi.o\n",
            "     CC esl_stack.o\n",
            "     CC esl_stats.o\n",
            "     CC esl_stopwatch.o\n",
            "     CC esl_stretchexp.o\n",
            "     CC esl_subcmd.o\n",
            "     CC esl_threads.o\n",
            "     CC esl_tree.o\n",
            "     CC esl_varint.o\n",
            "     CC esl_vectorops.o\n",
            "     CC esl_weibull.o\n",
            "     CC esl_workqueue.o\n",
            "     CC esl_wuss.o\n",
            "     CC esl_sse.o\n",
            "     CC esl_avx.o\n",
            "     CC esl_avx512.o\n",
            "     CC esl_neon.o\n",
            "     CC esl_vmx.o\n",
            "     AR libeasel.a\n",
            "     SUBDIR miniapps\n",
            "     GEN esl-afetch\n",
            "     GEN esl-alimanip\n",
            "     GEN esl-alimap\n",
            "     GEN esl-alimask\n",
            "     GEN esl-alimerge\n",
            "     GEN esl-alipid\n",
            "     GEN esl-alirev\n",
            "     GEN esl-alistat\n",
            "     GEN esl-compalign\n",
            "     GEN esl-compstruct\n",
            "     GEN esl-construct\n",
            "     GEN esl-histplot\n",
            "     GEN esl-mask\n",
            "     GEN esl-mixdchlet\n",
            "     GEN esl-reformat\n",
            "     GEN esl-selectn\n",
            "     GEN esl-seqrange\n",
            "     GEN esl-seqstat\n",
            "     GEN esl-sfetch\n",
            "     GEN esl-shuffle\n",
            "     GEN esl-ssdraw\n",
            "     GEN esl-translate\n",
            "     GEN esl-weight\n",
            "     CC cmd_alistat.o\n",
            "     CC cmd_downsample.o\n",
            "     CC cmd_filter.o\n",
            "     CC cmd_index.o\n",
            "     GEN easel\n",
            "     SUBDIR libdivsufsort\n",
            "     CC divsufsort.o\n",
            "     AR libdivsufsort.a\n",
            "     SUBDIR src\n",
            "     CC alimask.o\n",
            "     CC build.o\n",
            "     CC cachedb.o\n",
            "     CC cachedb_shard.o\n",
            "     CC emit.o\n",
            "     CC errors.o\n",
            "     CC evalues.o\n",
            "     CC eweight.o\n",
            "     CC generic_decoding.o\n",
            "     CC generic_fwdback.o\n",
            "     CC generic_fwdback_chk.o\n",
            "     CC generic_fwdback_banded.o\n",
            "     CC generic_null2.o\n",
            "     CC generic_msv.o\n",
            "     CC generic_optacc.o\n",
            "     CC generic_stotrace.o\n",
            "     CC generic_viterbi.o\n",
            "     CC generic_vtrace.o\n",
            "     CC h2_io.o\n",
            "     CC heatmap.o\n",
            "     CC hmmlogo.o\n",
            "     CC hmmdmstr.o\n",
            "     CC hmmdmstr_shard.o\n",
            "     CC hmmd_search_status.o\n",
            "     CC hmmdwrkr.o\n",
            "     CC hmmdwrkr_shard.o\n",
            "     CC hmmdutils.o\n",
            "     CC hmmer.o\n",
            "     CC logsum.o\n",
            "     CC modelconfig.o\n",
            "     CC modelstats.o\n",
            "     CC mpisupport.o\n",
            "     CC seqmodel.o\n",
            "     CC tracealign.o\n",
            "     CC p7_alidisplay.o\n",
            "     CC p7_bg.o\n",
            "     CC p7_builder.o\n",
            "     CC p7_domain.o\n",
            "     CC p7_domaindef.o\n",
            "     CC p7_gbands.o\n",
            "     CC p7_gmx.o\n",
            "     CC p7_gmxb.o\n",
            "     CC p7_gmxchk.o\n",
            "     CC p7_hit.o\n",
            "     CC p7_hmm.o\n",
            "     CC p7_hmmcache.o\n",
            "     CC p7_hmmd_search_stats.o\n",
            "     CC p7_hmmfile.o\n",
            "     CC p7_hmmwindow.o\n",
            "     CC p7_pipeline.o\n",
            "     CC p7_prior.o\n",
            "     CC p7_profile.o\n",
            "     CC p7_spensemble.o\n",
            "     CC p7_tophits.o\n",
            "     CC p7_trace.o\n",
            "     CC p7_scoredata.o\n",
            "     CC hmmpgmd2msa.o\n",
            "     CC fm_alphabet.o\n",
            "     CC fm_general.o\n",
            "     CC fm_sse.o\n",
            "     CC fm_ssv.o\n",
            "     AR libhmmer-src.stamp\n",
            "     SUBDIR impl_sse\n",
            "     CC decoding.o\n",
            "     CC fwdback.o\n",
            "     CC io.o\n",
            "     CC ssvfilter.o\n",
            "     CC msvfilter.o\n",
            "     CC null2.o\n",
            "     CC optacc.o\n",
            "     CC stotrace.o\n",
            "     CC vitfilter.o\n",
            "     CC p7_omx.o\n",
            "     CC p7_oprofile.o\n",
            "     CC mpi.o\n",
            "     AR libhmmer-impl.stamp\n",
            "     GEN alimask\n",
            "     CC hmmalign.o\n",
            "     GEN hmmalign\n",
            "     CC hmmbuild.o\n",
            "     GEN hmmbuild\n",
            "     CC hmmconvert.o\n",
            "     GEN hmmconvert\n",
            "     CC hmmemit.o\n",
            "     GEN hmmemit\n",
            "     CC hmmfetch.o\n",
            "     GEN hmmfetch\n",
            "     GEN hmmlogo\n",
            "     CC hmmpgmd.o\n",
            "     GEN hmmpgmd\n",
            "     CC hmmpgmd_shard.o\n",
            "     GEN hmmpgmd_shard\n",
            "     CC hmmpress.o\n",
            "     GEN hmmpress\n",
            "     CC hmmscan.o\n",
            "     GEN hmmscan\n",
            "     CC hmmsearch.o\n",
            "     GEN hmmsearch\n",
            "     CC hmmsim.o\n",
            "     GEN hmmsim\n",
            "     CC hmmstat.o\n",
            "     GEN hmmstat\n",
            "     CC jackhmmer.o\n",
            "     GEN jackhmmer\n",
            "     CC phmmer.o\n",
            "     GEN phmmer\n",
            "     CC nhmmer.o\n",
            "     GEN nhmmer\n",
            "     CC nhmmscan.o\n",
            "     GEN nhmmscan\n",
            "     CC makehmmerdb.o\n",
            "     GEN makehmmerdb\n",
            "     CC hmmc2.o\n",
            "     GEN hmmc2\n",
            "     CC hmmerfm-exactmatch.o\n",
            "     GEN hmmerfm-exactmatch\n",
            "     SUBDIR profmark\n",
            "     CC create-profmark.o\n",
            "     GEN create-profmark\n",
            "     CC rocplot.o\n",
            "     GEN rocplot\n",
            "     SUBDIR easel\n",
            "     SUBDIR miniapps\n",
            "     SUBDIR libdivsufsort\n",
            "     SUBDIR src\n",
            "     SUBDIR impl_sse\n",
            "     SUBDIR profmark\n",
            "/usr/bin/install -c -d /usr/local/bin\n",
            "/usr/bin/install -c -d /usr/local/share/man/man1\n",
            "make -C src install\n",
            "make[1]: Entering directory '/content/hmmer-3.3.2/src'\n",
            "for file in alimask hmmalign hmmbuild hmmconvert hmmemit hmmfetch hmmlogo hmmpgmd hmmpgmd_shard hmmpress hmmscan hmmsearch hmmsim hmmstat jackhmmer phmmer nhmmer nhmmscan makehmmerdb; do \\\n",
            "   /usr/bin/install -c -m 0755 $file /usr/local/bin/ ;\\\n",
            "done\n",
            "make[1]: Leaving directory '/content/hmmer-3.3.2/src'\n",
            "make -C documentation install\n",
            "make[1]: Entering directory '/content/hmmer-3.3.2/documentation'\n",
            "make -C man install\n",
            "make[2]: Entering directory '/content/hmmer-3.3.2/documentation/man'\n",
            "for file in hmmer hmmalign hmmbuild hmmc2 hmmconvert hmmemit hmmfetch hmmlogo hmmpgmd hmmpgmd_shard hmmpress hmmscan hmmsearch hmmsim hmmstat jackhmmer makehmmerdb phmmer nhmmer nhmmscan alimask; do \\\n",
            "   /usr/bin/install -c -m 0664 $file.man /usr/local/share/man/man1/${file}.1 ;\\\n",
            "done\n",
            "make[2]: Leaving directory '/content/hmmer-3.3.2/documentation/man'\n",
            "make[1]: Leaving directory '/content/hmmer-3.3.2/documentation'\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "In file included from hmmbuild.c:13:\n",
            "In function ‘usual_master’,\n",
            "    inlined from ‘main’ at hmmbuild.c:463:7:\n",
            "../easel/easel.h:164:17: warning: argument 1 range [18446744022169944064, 18446744073709551592] exceeds maximum object size 9223372036854775807 [-Walloc-size-larger-than=]\n",
            "  164 |     if ( ((p) = malloc(size)) == NULL)  { \\\n",
            "      |                 ^~~~~~~~~~~~\n",
            "hmmbuild.c:561:3: note: in expansion of macro ‘ESL_ALLOC’\n",
            "  561 |   ESL_ALLOC(info, sizeof(*info) * infocnt);\n",
            "      |   ^~~~~~~~~\n",
            "In file included from hmmbuild.c:6:\n",
            "hmmbuild.c: In function ‘main’:\n",
            "/usr/include/stdlib.h:540:14: note: in a call to allocation function ‘malloc’ declared here\n",
            "  540 | extern void *malloc (size_t __size) __THROW __attribute_malloc__\n",
            "      |              ^~~~~~\n",
            "In file included from hmmscan.c:14:\n",
            "In function ‘serial_master’,\n",
            "    inlined from ‘main’ at hmmscan.c:302:16:\n",
            "../easel/easel.h:164:17: warning: argument 1 range [18446743987810205696, 18446744073709551576] exceeds maximum object size 9223372036854775807 [-Walloc-size-larger-than=]\n",
            "  164 |     if ( ((p) = malloc(size)) == NULL)  { \\\n",
            "      |                 ^~~~~~~~~~~~\n",
            "hmmscan.c:402:3: note: in expansion of macro ‘ESL_ALLOC’\n",
            "  402 |   ESL_ALLOC(info, sizeof(*info) * infocnt);\n",
            "      |   ^~~~~~~~~\n",
            "In file included from hmmscan.c:11:\n",
            "hmmscan.c: In function ‘main’:\n",
            "/usr/include/stdlib.h:540:14: note: in a call to allocation function ‘malloc’ declared here\n",
            "  540 | extern void *malloc (size_t __size) __THROW __attribute_malloc__\n",
            "      |              ^~~~~~\n"
          ]
        }
      ],
      "source": [
        "%%bash\n",
        "cd hmmer-3.3.2\n",
        "./configure\n",
        "make\n",
        "make install"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Qr4NxI_9JFwA"
      },
      "source": [
        "Скачиваем Pfam\n",
        "По ftp доступны разны версии базы. Нас интересует последняя в формате hmm"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "XfY1wB3oUvL6",
        "outputId": "721fb281-7bc1-4fcb-8b61-ed610cca6074"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2024-06-15 23:53:19--  http://ftp.ebi.ac.uk/pub/databases/Pfam/releases/Pfam35.0/Pfam-A.hmm.gz\n",
            "Resolving ftp.ebi.ac.uk (ftp.ebi.ac.uk)... 193.62.193.165\n",
            "Connecting to ftp.ebi.ac.uk (ftp.ebi.ac.uk)|193.62.193.165|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 293000230 (279M) [application/x-gzip]\n",
            "Saving to: ‘Pfam-A.hmm.gz’\n",
            "\n",
            "Pfam-A.hmm.gz       100%[===================>] 279.43M   288KB/s    in 16m 35s \n",
            "\n",
            "2024-06-16 00:09:54 (288 KB/s) - ‘Pfam-A.hmm.gz’ saved [293000230/293000230]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "!wget http://ftp.ebi.ac.uk/pub/databases/Pfam/releases/Pfam35.0/Pfam-A.hmm.gz"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "nWnlyTK9Jo0Q"
      },
      "source": [
        "Извлекаем из архива"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "AXN716JmU-O0"
      },
      "outputs": [],
      "source": [
        "!gunzip Pfam-A.hmm.gz"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "CHrwjpTlJtbd"
      },
      "source": [
        "Создаем нужные для hmmer'a файлы"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "abD_KzEBWVqB",
        "outputId": "ee93150a-2568-48ec-e909-445d64479496"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Working...    done.\n",
            "Pressed and indexed 19632 HMMs (19632 names and 19632 accessions).\n",
            "Models pressed into binary file:   Pfam-A.hmm.h3m\n",
            "SSI index for binary model file:   Pfam-A.hmm.h3i\n",
            "Profiles (MSV part) pressed into:  Pfam-A.hmm.h3f\n",
            "Profiles (remainder) pressed into: Pfam-A.hmm.h3p\n"
          ]
        }
      ],
      "source": [
        "!hmmpress Pfam-A.hmm"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "h5ajvyOxoaS5",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "outputId": "d857a744-168d-44ff-d299-e1bed183143b"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "HMMER3/f [3.1b2 | February 2015]\n",
            "NAME  1-cysPrx_C\n",
            "ACC   PF10417.12\n",
            "DESC  C-terminal domain of 1-Cys peroxiredoxin\n",
            "LENG  40\n",
            "ALPH  amino\n",
            "RF    no\n",
            "MM    no\n",
            "CONS  yes\n",
            "CS    yes\n",
            "MAP   yes\n",
            "DATE  Thu Nov  4 19:20:02 2021\n",
            "NSEQ  40\n",
            "EFFN  17.426758\n",
            "CKSUM 4086680297\n",
            "GA    21.1 21.1;\n",
            "TC    21.1 21.1;\n",
            "NC    21 21;\n",
            "BM    hmmbuild HMM.ann SEED.ann\n",
            "SM    hmmsearch -Z 61295632 -E 1000 --cpu 4 HMM pfamseq\n",
            "STATS LOCAL MSV       -7.5463  0.71948\n",
            "STATS LOCAL VITERBI   -7.8624  0.71948\n",
            "STATS LOCAL FORWARD   -4.3303  0.71948\n",
            "HMM          A        C        D        E        F        G        H        I        K        L        M        N        P        Q        R        S        T        V        W        Y   \n",
            "            m->m     m->i     m->d     i->m     i->i     d->m     d->d\n",
            "  COMPO   2.28046  4.31208  2.83393  2.63913  3.90855  2.69988  3.89812  3.33401  2.56310  2.85023  3.99954  3.22924  2.52123  2.90328  3.31238  2.94055  2.70512  2.59551  3.49266  3.82715\n",
            "          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503\n",
            "          0.00226  6.48754  7.20989  0.61958  0.77255  0.00000        *\n",
            "      1   0.29666  6.14436  6.78514  6.79783  7.06332  2.55785  7.22049  6.57837  6.66651  6.27638  3.28757  5.91223  5.83978  6.69238  6.58162  2.20136  4.83343  5.59959  8.41086  7.43107      1 A - - H\n",
            "          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503\n",
            "          0.00226  6.48754  7.20989  0.61958  0.77255  0.48576  0.95510\n",
            "      2   4.59591  5.92009  6.57211  5.96147  1.92899  5.81035  6.10135  2.33093  5.75927  0.69439  2.86149  5.97820  6.07717  5.78793  5.72916  5.13924  4.81708  2.59612  3.18569  3.35842      2 l - - H\n",
            "          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503\n",
            "          0.00226  6.48754  7.20989  0.61958  0.77255  0.48576  0.95510\n",
            "      3   4.81290  7.05274  3.71696  4.47757  6.60126  5.41623  3.72993  5.92180  2.06538  3.59487  6.10993  4.89014  5.75663  0.42291  2.54802  4.76779  4.95656  5.56452  7.24472  6.08615      3 Q - - H\n",
            "          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503\n",
            "          0.00226  6.48754  7.20989  0.61958  0.77255  0.48576  0.95510\n",
            "      4   2.64100  5.28125  5.84007  3.33362  1.80025  5.06896  2.72827  3.71332  5.01717  1.75203  2.65498  5.22280  5.43290  5.15329  5.03455  4.37913  1.80041  2.31249  5.90246  2.63298      4 l - - H\n",
            "          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503\n",
            "          0.00226  6.48754  7.20989  0.61958  0.77255  0.48576  0.95510\n"
          ]
        }
      ],
      "source": [
        "!head -n 40 Pfam-A.hmm"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "zL3sy2rvJ5Wl"
      },
      "source": [
        "Смотрим статистику"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "_5JS0hV7Z-Zq",
        "outputId": "ee7a0a06-5cef-43af-96be-e653babb70cc",
        "collapsed": true
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "# hmmstat :: display summary statistics for a profile file\n",
            "# HMMER 3.3.2 (Nov 2020); http://hmmer.org/\n",
            "# Copyright (C) 2020 Howard Hughes Medical Institute.\n",
            "# Freely distributed under the BSD open source license.\n",
            "# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -\n",
            "#\n",
            "# idx  name                 accession        nseq eff_nseq      M relent   info p relE compKL\n",
            "# ---- -------------------- ------------ -------- -------- ------ ------ ------ ------ ------\n",
            "1      1-cysPrx_C           PF10417.12         40    17.43     40   1.37   1.32   1.29   0.09\n",
            "2      120_Rick_ant         PF12574.11          2     0.45    238   0.59   0.60   0.52   0.02\n",
            "3      12TM_1               PF09847.12          7     2.28    449   0.59   0.62   0.51   0.09\n",
            "4      14-3-3               PF00244.23        149     2.50    222   0.59   0.59   0.52   0.05\n",
            "5      17kDa_Anti_2         PF16998.8           6     0.99    116   0.59   0.57   0.52   0.03\n",
            "6      2-Hacid_dh           PF00389.33         78     5.69    134   0.59   0.62   0.44   0.03\n",
            "7      2-Hacid_dh_C         PF02826.22        135     3.55    178   0.59   0.60   0.53   0.01\n",
            "8      2-oxoacid_dh         PF00198.26         70     3.33    233   0.59   0.61   0.53   0.02\n",
            "9      2-oxogl_dehyd_N      PF16078.8         208    11.82     41   1.34   1.23   1.33   0.13\n",
            "10     2-ph_phosp           PF04029.17        125     3.02    226   0.59   0.62   0.51   0.02\n",
            "11     2-thiour_desulf      PF04463.15        338     2.11    137   0.59   0.57   0.46   0.02\n",
            "12     23ISL                PF16620.8          16     1.48    162   0.59   0.61   0.54   0.07\n"
          ]
        }
      ],
      "source": [
        "!hmmstat Pfam-A.hmm.h3m | head -n 20"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "xBqiKE1AdaCY"
      },
      "source": [
        "The columns are:\n",
        "\n",
        "**idx**\n",
        "The index of this profile, numbering each on in the file starting from 1.\n",
        "\n",
        "**name**\n",
        "The name of the profile.\n",
        "\n",
        "**accession**\n",
        "The optional accession of the profile, or \"-\" if there is none.\n",
        "\n",
        "**nseq**\n",
        "The number of sequences that the profile was estimated from.\n",
        "\n",
        "**eff_nseq**\n",
        "The effective number of sequences that the profile was estimated from, after HMMER applied an effective sequence number calculation such as the default entropy weighting.\n",
        "\n",
        "**M**\n",
        "The length of the model in consensus residues (match states).\n",
        "\n",
        "**relent**\n",
        "Mean relative entropy per match state, in bits. This is the expected (mean) score per consensus position. This is what the default entropy-weighting method for effective sequence number estimation focuses on, so for default HMMER3 models, you expect this value to reflect the default target for entropy-weighting.\n",
        "\n",
        "**info**\n",
        "Mean information content per match state, in bits. Probably not useful. Information content is a slightly different calculation than relative entropy.\n",
        "\n",
        "**p relE**\n",
        "Mean positional relative entropy, in bits. This is a fancier version of the per-match-state relative entropy, taking into account the transition (insertion/deletion) probabilities; it may be a more accurate estimation of the average score contributed per model consensus position.\n",
        "\n",
        "**compKL**\n",
        "Kullback-Leibler distance between the model's overall average residue composition and the default background frequency distribution. The higher this number, the more biased the residue composition of the profile is. Highly biased profiles can slow the HMMER3 acceleration pipeline, by causing too many nonhomologous sequences to pass the filters."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "2N22F0eaKD4E"
      },
      "source": [
        "Индексируем файл"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "2CZAZX4yU-Tw",
        "outputId": "d093595f-189d-46fa-b005-f99c3d323a92"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Working...    done.\n",
            "Indexed 19632 HMMs (19632 names and 19632 accessions).\n",
            "SSI index written to file Pfam-A.hmm.h3m.ssi\n"
          ]
        }
      ],
      "source": [
        "!hmmfetch --index Pfam-A.hmm.h3m"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Vqhqu2GEKHE6"
      },
      "source": [
        "протеом"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!head protein.faa\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "id": "8ab3LRLtJ97P",
        "outputId": "17e8648a-cb66-4cc1-e8ca-0824191d1e4a"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            ">XP_004349659.1 uncharacterized protein DFA_00001, partial [Cavenderia fasciculata]\n",
            "MAEIVEEAVKQKVSLGESTLDSDSTICCLKLSESLFPTKREKYNKIYVRDCYIPVYDIISNHQALHGQGSIVLVKGNPGI\n",
            "GKSVFLFYYLYRKRSDPLVKSIIYDVVDAPVYHFTYTEDGSPIVHQGDRSSFKDILLRPDSSSLYLVDGQRPLPSKRPTL\n",
            "IVSSPNKSNYEEYMKDGKTSRIIMPLWSSLEILASNCLIFDHPIQDVQSRYFRWGGIPRYVLDKISRPDQELLVNAISST\n",
            "NLEACISSINMPDSPESGSHKVIHLDAEPGTDFVRQRVSFASNYVKEKLVQRFYWNWREEVIRKVSLVGTSQIEGLLFEQ\n",
            "LSHQTLINGGEFHIRSLDGKGNLGEYLHFAPAKCDSIQGTTELASRLASKNRFPPDTYVRPTDLNFPAVDSFMGQYLFQI\n",
            "TKTNNHPVSFKELNFITFHLDPSTKNPAATTTDTTTTTTATTTTDASTTAGTQFHLFFVVPPHLFDSFPRQEYVEAKTQP\n",
            "DANNPGKKTTTKNKVKAPQNVHQYALKMAFSSVPPSVLYPVISGEDAIINSDPKTILIDHLQSTQKISTTDSIRMDQDIK\n",
            "KDKRSITSALNLKKARDARKNQFIIYDSDSDSDNDSDNTKKRTKNKAPVKKKRIVIDNNKKDFEQMNKKMEFLIEMFTCH\n",
            "QQKNNFIVNDEVVEKEIKQQPINSIKESDIIKNKMIMKSKKNDKKLIRKLIREALDDERKQQQHLSKESSPVLAWSNPNE\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!hmmstat Pfam-A.hmm"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "FLIsZm4N4PLP",
        "outputId": "ab861315-88e5-46de-a18c-6908f9b6b00d",
        "collapsed": true
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mВыходные данные были обрезаны до нескольких последних строк (5000).\u001b[0m\n",
            "14633  PKI                  PF02827.19         21     1.92     69   0.82   0.86   0.78   0.06\n",
            "14634  pKID                 PF02173.20          9     3.37     41   1.34   1.39   1.28   0.15\n",
            "14635  Pkinase              PF00069.28         38     2.64    264   0.59   0.57   0.50   0.02\n",
            "14636  Pkinase_C            PF00433.27        421    22.24     46   1.20   1.16   1.05   0.15\n",
            "14637  Pkinase_fungal       PF17667.4          24     2.15    405   0.59   0.57   0.47   0.02\n",
            "14638  Pkip-1               PF06878.14         19     2.24    163   0.59   0.59   0.52   0.06\n",
            "14639  PKK                  PF12474.11         56     4.04    139   0.59   0.58   0.55   0.18\n",
            "14640  PknG_rubred          PF16919.8          12     0.96    139   0.59   0.58   0.51   0.03\n",
            "14641  PknG_TPR             PF16918.8          12     1.12    340   0.59   0.62   0.52   0.02\n",
            "14642  PknH_C               PF14032.9          49     7.23    188   0.59   0.59   0.49   0.03\n",
            "14643  Pkr1                 PF08636.13         75     2.42     73   0.77   0.76   0.72   0.12\n",
            "14644  PKS_DE               PF18369.4         127     5.97     45   1.22   1.24   1.17   0.15\n",
            "14645  PK_C                 PF02887.19        599     5.49    116   0.59   0.62   0.47   0.04\n",
            "14646  PK_Tyr_Ser-Thr       PF07714.20        118     1.91    259   0.59   0.57   0.52   0.02\n",
            "14647  PL48                 PF15903.8          14     0.70    348   0.59   0.58   0.53   0.02\n",
            "14648  PLA1                 PF02253.18         64     1.77    257   0.59   0.55   0.52   0.03\n",
            "14649  PLA2G12              PF06951.14          6     0.67    186   0.59   0.53   0.51   0.04\n",
            "14650  PLA2_B               PF01735.21         14     1.33    491   0.59   0.57   0.52   0.01\n",
            "14651  PLA2_inh             PF02988.18          5     0.80     83   0.69   0.58   0.63   0.12\n",
            "14652  PLAC                 PF08686.14         80     5.51     31   1.74   1.36   1.66   0.54\n",
            "14653  PLAC8                PF04749.20        284     3.74     95   0.60   0.47   0.31   0.09\n",
            "14654  PLAC9                PF15205.9           6     0.71     74   0.76   0.79   0.70   0.02\n",
            "14655  Planc_extracel       PF07595.15         22    22.00     24   1.59   1.66   1.60   0.40\n",
            "14656  Plant_all_beta       PF10532.12         10     2.05    114   0.59   0.59   0.52   0.06\n",
            "14657  Plant_NMP1           PF06694.14          4     0.50    318   0.59   0.60   0.52   0.01\n",
            "14658  Plant_tran           PF04827.17          5     0.59    205   0.59   0.55   0.52   0.01\n",
            "14659  Plant_vir_prot       PF01307.20          9     1.39    100   0.59   0.57   0.52   0.01\n",
            "14660  Plant_zn_clust       PF10533.12         34     2.59     48   1.15   1.06   1.05   0.25\n",
            "14661  Plasmid_parti        PF01672.19         40     1.57     85   0.67   0.66   0.62   0.06\n",
            "14662  Plasmid_RAQPRD       PF09686.13         21     2.49     76   0.74   0.76   0.68   0.05\n",
            "14663  Plasmid_stab_B       PF10784.12         11     3.14     72   0.78   0.83   0.76   0.03\n",
            "14664  Plasmodium_HRP       PF05403.14          3     0.57    218   0.59   0.50   0.49   0.25\n",
            "14665  Plasmodium_Vir       PF05795.14         43     7.60    381   0.59   0.52   0.47   0.07\n",
            "14666  Plasmod_dom_1        PF09715.13          4     1.37     66   0.85   0.85   0.80   0.08\n",
            "14667  Plasmod_MYXSPDY      PF07981.14         14    14.00     17   2.84   2.81   2.84   0.50\n",
            "14668  Plasmod_Pvs28        PF06247.14         36    12.17     41   1.34   1.14   1.16   0.33\n",
            "14669  Plasmo_rep           PF12135.11          3     3.00     25   1.81   1.68   1.78   0.36\n",
            "14670  PLAT                 PF01477.26         84    10.17    117   0.59   0.55   0.45   0.03\n",
            "14671  PLATZ                PF04640.17         57     1.49     82   0.69   0.60   0.60   0.08\n",
            "14672  Plavaka              PF18759.4         339     2.81    325   0.59   0.53   0.51   0.02\n",
            "14673  PLC-beta_C           PF08703.13         13     1.58    176   0.59   0.60   0.55   0.11\n",
            "14674  PLCC                 PF14466.9           2     0.50    127   0.59   0.59   0.51   0.03\n",
            "14675  PLDc                 PF00614.25         30    30.00     28   1.51   1.44   1.45   0.14\n",
            "14676  PLDc_2               PF13091.9          75     6.08    133   0.59   0.57   0.44   0.03\n",
            "14677  PLDc_3               PF13918.9           2     0.63    180   0.59   0.58   0.52   0.02\n",
            "14678  PLDc_N               PF13396.9         639    70.11     42   1.31   1.31   1.19   0.28\n",
            "14679  PLD_C                PF12357.11         94     1.56     72   0.78   0.74   0.73   0.01\n",
            "14680  Plectin              PF00681.23        306    48.00     39   1.40   1.49   1.38   0.05\n",
            "14681  Plexin_cytopl        PF08337.15         44     1.36    441   0.59   0.57   0.50   0.03\n",
            "14682  Plexin_RBD           PF20170.1          45     1.95    113   0.59   0.59   0.52   0.04\n",
            "14683  PliI                 PF16743.8           6     0.91    123   0.59   0.63   0.52   0.02\n",
            "14684  PLipase_C_C          PF05506.15        166     5.74     82   0.69   0.66   0.53   0.03\n",
            "14685  Plk4_PB1             PF18190.4          10     1.17    107   0.59   0.58   0.52   0.03\n",
            "14686  Plk4_PB2             PF18409.4          51     2.08    109   0.59   0.56   0.50   0.03\n",
            "14687  PLL                  PF18587.4          25     1.00    134   0.59   0.54   0.53   0.03\n",
            "14688  PLN_propep           PF16485.8          12     5.23     43   1.28   1.24   1.24   0.08\n",
            "14689  Ploopntkinase1       PF18748.4           3     0.51    196   0.59   0.59   0.53   0.01\n",
            "14690  Ploopntkinase2       PF18747.4           3     0.59    298   0.59   0.57   0.52   0.01\n",
            "14691  Ploopntkinase3       PF18751.4           3     0.57    184   0.59   0.61   0.52   0.01\n",
            "14692  PLRV_ORF5            PF01690.20          2     0.61    485   0.59   0.57   0.51   0.01\n",
            "14693  PLU-1                PF08429.14         94     5.45    339   0.59   0.59   0.49   0.05\n",
            "14694  Plug                 PF07715.18        617     9.65    108   0.59   0.64   0.49   0.04\n",
            "14695  Plug_translocon      PF10559.12        115    21.34     35   1.55   1.58   1.48   0.13\n",
            "14696  Plus-3               PF03126.21        171     4.21    109   0.59   0.56   0.46   0.03\n",
            "14697  PlyB_C               PF18684.4           3     0.58    173   0.59   0.57   0.52   0.01\n",
            "14698  PM0188               PF11477.11          3     0.70    385   0.59   0.58   0.51   0.03\n",
            "14699  PMAIP1               PF15150.9           7     1.13     54   1.03   1.04   0.96   0.04\n",
            "14700  PmbA_TldD            PF01523.19        479    76.97     66   0.85   0.92   0.65   0.07\n",
            "14701  PmbA_TldD_C          PF19289.2         288     6.39    220   0.59   0.61   0.47   0.01\n",
            "14702  PmbA_TldD_M          PF19290.2         491    61.78    113   0.59   0.63   0.30   0.03\n",
            "14703  PMBR                 PF09373.13         49    49.00     33   1.22   1.21   1.21   0.09\n",
            "14704  PMC2NT               PF08066.15        130     5.38     92   0.62   0.65   0.54   0.05\n",
            "14705  PMD                  PF10536.12         30     2.03    355   0.59   0.54   0.48   0.02\n",
            "14706  PMEI                 PF04043.18        629    12.06    154   0.59   0.58   0.34   0.06\n",
            "14707  PMG                  PF05287.15         11     2.06    180   0.59   0.50   0.51   0.11\n",
            "14708  PMI_typeI_C          PF01238.24          8     6.30     48   1.15   1.19   1.08   0.04\n",
            "14709  PMI_typeI_cat        PF20511.1           8     1.04    144   0.59   0.56   0.49   0.02\n",
            "14710  PMI_typeI_hel        PF20512.1           6     1.40     94   0.61   0.63   0.52   0.03\n",
            "14711  PMM                  PF03332.16          9     0.66    221   0.59   0.57   0.52   0.02\n",
            "14712  PMP1_2               PF08114.14          2     0.91     43   1.27   1.27   1.22   0.13\n",
            "14713  PMP22_Claudin        PF00822.23         13     1.87    166   0.59   0.55   0.52   0.06\n",
            "14714  Pmp3                 PF01679.20        320     7.32     49   1.13   1.11   1.05   0.22\n",
            "14715  PMR5N                PF14416.9         339     2.36     54   1.03   0.79   0.94   0.17\n",
            "14716  PmrD                 PF11183.11          4     1.01     81   0.70   0.66   0.63   0.04\n",
            "14717  PMSI1                PF15322.9           9     0.65    315   0.59   0.57   0.51   0.02\n",
            "14718  PMSR                 PF01625.24        590     1.72    153   0.59   0.53   0.53   0.04\n",
            "14719  PMT                  PF02366.21         19     4.08    245   0.59   0.57   0.54   0.06\n",
            "14720  PMT2_N               PF17987.4           9     0.81    125   0.59   0.59   0.52   0.03\n",
            "14721  PMT_2                PF13231.9          32     4.59    160   0.59   0.59   0.53   0.10\n",
            "14722  PMT_4TMC             PF16192.8         388     3.07    200   0.59   0.50   0.50   0.07\n",
            "14723  PND                  PF17949.4           5     0.79    125   0.59   0.60   0.53   0.04\n",
            "14724  Pneumovirus_M2       PF06436.14          2     0.59    157   0.59   0.60   0.52   0.03\n",
            "14725  Pneumo_att_G         PF05539.14          3     0.82    408   0.59   0.55   0.53   0.05\n",
            "14726  Pneumo_M2            PF07380.14          1     0.42     89   0.64   0.60   0.57   0.04\n",
            "14727  Pneumo_matrix        PF03393.19          2     0.57    252   0.59   0.60   0.53   0.02\n",
            "14728  Pneumo_ncap          PF03246.16          2     0.56    392   0.59   0.61   0.52   0.02\n",
            "14729  Pneumo_NS1           PF03438.16          1     0.39    136   0.59   0.59   0.52   0.03\n",
            "14730  Pneumo_phosprot      PF02478.19          2     0.50    286   0.59   0.63   0.52   0.04\n",
            "14731  PNGaseA              PF12222.11         21     1.52    434   0.59   0.57   0.52   0.02\n",
            "14732  PNISR                PF15996.8          21     1.47    182   0.59   0.60   0.49   0.07\n",
            "14733  PNK3P                PF08645.14         24     1.34    162   0.59   0.57   0.48   0.02\n",
            "14734  PNKP-ligase_C        PF16536.8          29     1.56     60   0.93   0.99   0.90   0.05\n",
            "14735  PNKP_ligase          PF16542.8          61     0.90    320   0.59   0.57   0.53   0.01\n",
            "14736  PNMA                 PF14893.9          22     1.78    328   0.59   0.59   0.52   0.02\n",
            "14737  PNPase               PF03726.17         17     2.44     83   0.68   0.72   0.63   0.07\n",
            "14738  PNPase_C             PF12111.11         15     3.54     37   1.47   1.39   1.46   0.19\n",
            "14739  PnpCD_PnpD_N         PF18191.4           1     0.37    151   0.59   0.57   0.52   0.01\n",
            "14740  PNP_phzG_C           PF10590.12        478     5.43     43   1.28   1.13   1.12   0.13\n",
            "14741  PNP_UDP_1            PF01048.23        107    12.68    234   0.59   0.61   0.47   0.02\n",
            "14742  PNRC                 PF15365.9          97    97.00     20   2.39   2.39   2.29   0.53\n",
            "14743  Pns11_12             PF17464.5           2     0.66    203   0.59   0.58   0.51   0.04\n",
            "14744  PNTB                 PF02233.19        266     1.42    460   0.59   0.62   0.52   0.03\n",
            "14745  PNTB_4TM             PF12769.10        183     1.42     86   0.66   0.69   0.59   0.08\n",
            "14746  pNTSase1             PF20301.1          17     2.30    165   0.59   0.58   0.51   0.02\n",
            "14747  POB3_N               PF17292.5          85     3.26     93   0.61   0.59   0.55   0.03\n",
            "14748  POC1                 PF10450.12          3     0.67    273   0.59   0.58   0.51   0.02\n",
            "14749  POC3_POC4            PF10448.12         17     4.56    144   0.59   0.59   0.47   0.03\n",
            "14750  PocR                 PF10114.12        195     6.06    162   0.59   0.56   0.52   0.02\n",
            "14751  Podoplanin           PF05808.14          9     0.92    135   0.59   0.63   0.51   0.05\n",
            "14752  Podovirus_Gp16       PF05894.15          2     0.46    331   0.59   0.57   0.52   0.02\n",
            "14753  Polbeta              PF18765.4          77     4.09     93   0.61   0.63   0.54   0.04\n",
            "14754  PolC_DP2             PF03833.16         61     1.04    865   0.59   0.59   0.52   0.01\n",
            "14755  Pollen_allerg_2      PF01620.19          1     0.43    155   0.59   0.63   0.52   0.04\n",
            "14756  Pollen_Ole_e_1       PF01190.20         87     2.25     99   0.59   0.52   0.49   0.07\n",
            "14757  POLO_box             PF00659.21        198    10.59     61   0.92   0.87   0.80   0.05\n",
            "14758  Polo_box_2           PF18531.4           3     0.71    112   0.59   0.58   0.52   0.03\n",
            "14759  Polo_box_3           PF18544.4           9     0.91     98   0.59   0.58   0.52   0.03\n",
            "14760  PolyA_pol            PF01743.23         17     1.61    126   0.59   0.60   0.50   0.03\n",
            "14761  PolyA_pol_arg_C      PF12626.10        119     2.47    120   0.59   0.55   0.52   0.06\n",
            "14762  PolyA_pol_RNAbd      PF12627.10        183    12.71     64   0.88   0.91   0.85   0.04\n",
            "14763  Polycystin_dom       PF20519.1          11     1.71    199   0.59   0.55   0.52   0.02\n",
            "14764  PolyG_pol            PF01518.19          4     0.97    366   0.59   0.60   0.52   0.02\n",
            "14765  Polyhedrin           PF00738.21          3     0.54    232   0.59   0.56   0.52   0.02\n",
            "14766  Polyketide_cyc       PF03364.23         95    12.45    130   0.59   0.56   0.45   0.02\n",
            "14767  Polyketide_cyc2      PF10604.12        103    13.24    142   0.59   0.58   0.42   0.03\n",
            "14768  Polyoma_agno         PF01736.19          1     0.63     62   0.90   0.91   0.84   0.05\n",
            "14769  Polyoma_coat         PF00718.23          1     0.39    293   0.59   0.58   0.52   0.01\n",
            "14770  Polyoma_coat2        PF00761.23          8     1.07    322   0.59   0.59   0.50   0.01\n",
            "14771  Polyoma_lg_T_C       PF06431.14          3     0.45    417   0.59   0.57   0.52   0.01\n",
            "14772  polyprenyl_synt      PF00348.20        415     3.71    256   0.59   0.59   0.49   0.02\n",
            "14773  Polysacc_deac_1      PF01522.24         72     5.34    124   0.59   0.56   0.51   0.02\n",
            "14774  Polysacc_deac_2      PF04748.16        202     4.58    213   0.59   0.61   0.54   0.02\n",
            "14775  Polysacc_deac_3      PF15421.9           3     0.54    423   0.59   0.56   0.52   0.02\n",
            "14776  Polysacc_lyase       PF14099.9          41     4.71    213   0.59   0.53   0.40   0.03\n",
            "14777  Polysacc_synt        PF01943.20         17     5.72    275   0.59   0.61   0.53   0.08\n",
            "14778  Polysacc_synt_2      PF02719.18         62     1.30    294   0.59   0.61   0.53   0.02\n",
            "14779  Polysacc_synt_3      PF13440.9          23     2.54    293   0.59   0.60   0.56   0.05\n",
            "14780  Polysacc_synt_4      PF04669.16         27     1.97    190   0.59   0.57   0.50   0.02\n",
            "14781  Polysacc_synt_C      PF14667.9         317    12.41    143   0.59   0.64   0.52   0.15\n",
            "14782  Polysacc_syn_2C      PF08485.13         74     3.35     48   1.15   1.14   1.12   0.10\n",
            "14783  Poly_A_pol_cat       PF19244.2           9     1.06    131   0.59   0.56   0.52   0.03\n",
            "14784  Poly_export          PF02563.19        541     6.44     82   0.69   0.73   0.54   0.03\n",
            "14785  Pol_alpha_B_N        PF08418.13         77     3.37    235   0.59   0.58   0.41   0.04\n",
            "14786  Pom                  PF17251.5           6     1.28    285   0.59   0.53   0.50   0.03\n",
            "14787  POM121               PF15229.9          25     2.07    226   0.59   0.61   0.52   0.06\n",
            "14788  Pombe_5TM            PF09437.13          2     0.59    256   0.59   0.55   0.50   0.04\n",
            "14789  Ponericin            PF07442.14          3     1.85     29   1.85   1.79   1.82   0.39\n",
            "14790  POP1                 PF06978.14         71     1.89    206   0.59   0.55   0.50   0.07\n",
            "14791  Popeye               PF04831.16         29     2.01    140   0.59   0.55   0.51   0.05\n",
            "14792  POPLD                PF08170.15        137     3.61     93   0.61   0.53   0.53   0.03\n",
            "14793  POR                  PF01558.21         80     8.21    176   0.59   0.62   0.48   0.02\n",
            "14794  PorA                 PF11271.11         72     4.94    300   0.59   0.61   0.47   0.02\n",
            "14795  PorB                 PF11565.11          3     0.80     99   0.59   0.60   0.51   0.04\n",
            "14796  Porin_1              PF00267.24         10     1.22    340   0.59   0.58   0.50   0.04\n",
            "14797  Porin_10             PF14121.9          77     3.27    600   0.59   0.54   0.49   0.04\n",
            "14798  Porin_2              PF02530.17         16     2.03    379   0.59   0.57   0.48   0.03\n",
            "14799  Porin_3              PF01459.25         68     5.34    273   0.59   0.59   0.51   0.02\n",
            "14800  Porin_4              PF13609.9         103    12.21    312   0.59   0.61   0.43   0.07\n",
            "14801  Porin_5              PF16930.8          19     1.17    532   0.59   0.58   0.50   0.02\n",
            "14802  Porin_6              PF16939.8           3     0.72    282   0.59   0.56   0.52   0.03\n",
            "14803  Porin_7              PF16956.8           2     0.68    267   0.59   0.58   0.50   0.04\n",
            "14804  Porin_8              PF16966.8           5     0.69    366   0.59   0.57   0.52   0.02\n",
            "14805  Porin_OmpG           PF09381.13          7     0.74    285   0.59   0.54   0.52   0.03\n",
            "14806  Porin_OmpG_1_2       PF16946.8           5     0.89    294   0.59   0.55   0.52   0.03\n",
            "14807  Porin_OmpL1          PF11389.11          6     0.88    272   0.59   0.60   0.50   0.02\n",
            "14808  Porin_O_P            PF07396.14         19     5.25    370   0.59   0.56   0.49   0.03\n",
            "14809  Porphobil_deam       PF01379.23        666     2.36    208   0.59   0.63   0.48   0.02\n",
            "14810  Porphobil_deamC      PF03900.18         16     3.46     75   0.75   0.79   0.66   0.04\n",
            "14811  Porphyrn_cat_1       PF18206.4          38     2.85    105   0.59   0.60   0.56   0.05\n",
            "14812  Porph_ging           PF09697.13         22     1.20     83   0.68   0.64   0.60   0.03\n",
            "14813  PorP_SprF            PF11751.11         52     5.23    273   0.59   0.57   0.49   0.03\n",
            "14814  PORR                 PF11955.11         96     2.70    326   0.59   0.59   0.52   0.03\n",
            "14815  Portal_Gp20          PF07230.14         65     3.27    432   0.59   0.57   0.51   0.03\n",
            "14816  PorV                 PF19572.2           4     0.50    243   0.59   0.60   0.53   0.01\n",
            "14817  POR_N                PF01855.22         31     3.05    231   0.59   0.59   0.53   0.02\n",
            "14818  Por_Secre_tail       PF18962.3         209     4.16     75   0.75   0.75   0.61   0.06\n",
            "14819  Post_transc_reg      PF13797.9          23     2.92     83   0.68   0.63   0.65   0.06\n",
            "14820  POT1                 PF02765.20         47     8.31    143   0.59   0.57   0.50   0.03\n",
            "14821  POT1PC               PF16686.8          88     4.69    154   0.59   0.55   0.48   0.02\n",
            "14822  Potassium_chann      PF11404.11          8     3.43     29   1.85   1.77   1.84   0.28\n",
            "14823  Potass_KdpF          PF09604.13         78    78.00     24   2.07   2.24   2.05   0.42\n",
            "14824  potato_inhibit       PF00280.21        123     4.41     64   0.88   0.86   0.82   0.05\n",
            "14825  Potex_coat           PF06184.14          2     0.56    153   0.59   0.59   0.53   0.02\n",
            "14826  POTRA                PF07244.18        233    16.59     80   0.71   0.74   0.56   0.05\n",
            "14827  POTRA_1              PF08478.13         78    10.73     69   0.82   0.84   0.78   0.06\n",
            "14828  POTRA_2              PF08479.14         48     5.97     76   0.74   0.78   0.71   0.03\n",
            "14829  POTRA_3              PF17287.5          45     7.15     56   0.99   1.03   0.93   0.04\n",
            "14830  POTRA_TamA_1         PF17243.5         156     7.29     77   0.73   0.75   0.65   0.03\n",
            "14831  Potyvirid-P3         PF13608.9           3     0.92    452   0.59   0.58   0.50   0.03\n",
            "14832  Poty_coat            PF00767.21          4     0.93    244   0.59   0.56   0.53   0.02\n",
            "14833  Poty_PP              PF08440.13          5     1.40    277   0.59   0.58   0.54   0.03\n",
            "14834  Pou                  PF00157.20         26     2.35     72   0.78   0.80   0.74   0.04\n",
            "14835  POU2F1_C             PF19536.2          12     0.88    284   0.59   0.66   0.51   0.08\n",
            "14836  POX                  PF07526.14        105     2.39    136   0.59   0.59   0.49   0.04\n",
            "14837  Poxvirus_B22R        PF04395.15         12     1.76    210   0.59   0.55   0.50   0.05\n",
            "14838  Poxvirus_B22R_C      PF13168.9          14     0.93    195   0.59   0.59   0.53   0.02\n",
            "14839  Poxvirus_B22R_N      PF13169.9          12     2.04     89   0.64   0.56   0.58   0.07\n",
            "14840  Poxv_Bcl-2-like      PF06227.15         37     5.67    173   0.59   0.54   0.46   0.06\n",
            "14841  Pox_A11              PF05061.16          7     0.62    315   0.59   0.61   0.51   0.03\n",
            "14842  Pox_A12              PF04651.16          3     0.61    183   0.59   0.60   0.51   0.04\n",
            "14843  Pox_A14              PF05767.15          7     0.90     93   0.62   0.61   0.54   0.05\n",
            "14844  Pox_A21              PF05323.15         10     0.92    111   0.59   0.54   0.52   0.05\n",
            "14845  Pox_A22              PF04848.16          9     0.99    145   0.59   0.58   0.51   0.03\n",
            "14846  Pox_A28              PF04584.17         10     0.77    140   0.59   0.53   0.52   0.04\n",
            "14847  Pox_A30L_A26L        PF06086.15          2     0.38    220   0.59   0.55   0.52   0.03\n",
            "14848  Pox_A31              PF05771.14          4     0.73    113   0.59   0.58   0.53   0.04\n",
            "14849  Pox_A32              PF04665.15          8     0.86    242   0.59   0.57   0.52   0.03\n",
            "14850  Pox_A3L              PF05288.14          9     1.33     70   0.80   0.79   0.76   0.06\n",
            "14851  Pox_A51              PF04948.16          4     0.71    337   0.59   0.57   0.52   0.04\n",
            "14852  Pox_A6               PF04924.15          9     0.77    370   0.59   0.59   0.53   0.05\n",
            "14853  Pox_A8               PF04745.15          8     0.66    289   0.59   0.58   0.52   0.03\n",
            "14854  Pox_A9               PF04835.15          8     1.35     53   1.05   1.01   1.01   0.12\n",
            "14855  Pox_Ag35             PF03286.17          8     1.17    196   0.59   0.60   0.50   0.06\n",
            "14856  Pox_ATPase-GT        PF10640.12         10     1.01    312   0.59   0.59   0.51   0.04\n",
            "14857  Pox_A_type_inc       PF04508.15         11    11.00     23   1.79   1.82   1.76   0.37\n",
            "14858  Pox_B6               PF07389.15          3     0.68    181   0.59   0.56   0.52   0.05\n",
            "14859  Pox_C4_C10           PF03336.16          5     1.16    322   0.59   0.56   0.51   0.04\n",
            "14860  Pox_C7_F8A           PF03287.17         10     1.00    148   0.59   0.57   0.53   0.04\n",
            "14861  Pox_D2               PF04701.16          8     1.17    139   0.59   0.57   0.52   0.03\n",
            "14862  Pox_D3               PF04580.16          7     1.32    248   0.59   0.56   0.52   0.04\n",
            "14863  Pox_D5               PF03288.19         36     6.49     86   0.66   0.60   0.55   0.05\n",
            "14864  Pox_E10              PF04805.15          9     1.02     70   0.80   0.75   0.75   0.06\n",
            "14865  Pox_E2-like          PF04497.15         14     3.74    741   0.59   0.59   0.47   0.02\n",
            "14866  Pox_E6               PF04656.15          9     0.65    566   0.59   0.57   0.53   0.04\n",
            "14867  Pox_E8               PF03394.16          8     0.62    239   0.59   0.57   0.52   0.03\n",
            "14868  Pox_EPC_I2-L1        PF12575.11          7     1.15     72   0.78   0.78   0.72   0.07\n",
            "14869  Pox_F11              PF04943.15          3     0.95    409   0.59   0.59   0.50   0.03\n",
            "14870  Pox_F12L             PF03337.16          8     1.32    648   0.59   0.57   0.52   0.02\n",
            "14871  Pox_F15              PF04596.15          8     0.70    137   0.59   0.54   0.53   0.03\n",
            "14872  Pox_F16              PF04708.15          6     0.83    215   0.59   0.56   0.52   0.04\n",
            "14873  Pox_F17              PF04767.15          9     0.90     94   0.61   0.58   0.50   0.04\n",
            "14874  Pox_G5               PF04599.15          8     0.98    425   0.59   0.58   0.53   0.03\n",
            "14875  Pox_G7               PF05503.15          8     0.90    367   0.59   0.60   0.52   0.03\n",
            "14876  Pox_G9-A16           PF03003.19         30     2.13    128   0.59   0.50   0.49   0.07\n",
            "14877  Pox_H7               PF04787.15          9     1.30    143   0.59   0.58   0.52   0.04\n",
            "14878  Pox_I1               PF03289.16          9     0.66    307   0.59   0.60   0.53   0.04\n",
            "14879  Pox_I3               PF04661.15          8     0.83    262   0.59   0.59   0.52   0.03\n",
            "14880  Pox_I5               PF04713.15          8     1.39     75   0.75   0.81   0.71   0.11\n",
            "14881  Pox_I6               PF04595.16         13     1.55    321   0.59   0.57   0.52   0.03\n",
            "14882  Pox_int_trans        PF05718.15          9     0.69    383   0.59   0.58   0.53   0.04\n",
            "14883  Pox_J1               PF03338.18          9     0.95    146   0.59   0.60   0.53   0.03\n",
            "14884  Pox_L3_FP4           PF03339.17          6     0.61    316   0.59   0.57   0.52   0.03\n",
            "14885  Pox_L5               PF04872.16          8     1.57     79   0.72   0.72   0.65   0.04\n",
            "14886  Pox_LP_H2            PF03356.18         10     0.69    188   0.59   0.55   0.53   0.02\n",
            "14887  Pox_M2               PF04887.15          3     0.63    196   0.59   0.56   0.50   0.03\n",
            "14888  Pox_MCEL             PF03291.19         10     1.31    333   0.59   0.57   0.51   0.03\n",
            "14889  Pox_mRNA-cap         PF03341.16          7     0.58    286   0.59   0.58   0.53   0.03\n",
            "14890  Pox_P21              PF05313.15          8     0.81    189   0.59   0.60   0.50   0.04\n",
            "14891  Pox_P35              PF03213.17          7     0.95    323   0.59   0.57   0.52   0.03\n",
            "14892  Pox_P4A              PF03395.17          3     0.59    882   0.59   0.59   0.52   0.02\n",
            "14893  Pox_P4B              PF03292.17          7     0.61    657   0.59   0.57   0.51   0.02\n",
            "14894  Pox_polyA_pol        PF03296.16          7     0.73    148   0.59   0.58   0.52   0.04\n",
            "14895  Pox_polyA_pol_C      PF12629.10         10     1.07    201   0.59   0.58   0.50   0.04\n",
            "14896  Pox_polyA_pol_N      PF12630.10          9     0.76    108   0.59   0.59   0.53   0.05\n",
            "14897  Pox_Rap94            PF03294.17          9     0.87    800   0.59   0.56   0.51   0.05\n",
            "14898  Pox_Rif              PF03340.16          8     0.80    541   0.59   0.57   0.52   0.03\n",
            "14899  Pox_RNA_pol          PF03293.17          9     0.59    160   0.59   0.58   0.53   0.03\n",
            "14900  Pox_RNA_Pol_19       PF05320.15          9     0.69    166   0.59   0.59   0.51   0.03\n",
            "14901  Pox_RNA_Pol_22       PF05273.16          8     0.60    184   0.59   0.58   0.52   0.03\n",
            "14902  Pox_RNA_pol_35       PF03396.18          9     0.78    293   0.59   0.57   0.53   0.03\n",
            "14903  Pox_ser-thr_kin      PF05445.14          8     0.80    434   0.59   0.55   0.51   0.04\n",
            "14904  Pox_T4_C             PF04490.15          3     0.63    146   0.59   0.54   0.51   0.04\n",
            "14905  Pox_T4_N             PF04491.15          3     1.42     46   1.20   1.11   1.14   0.07\n",
            "14906  Pox_TAA1             PF03295.17          8     1.14     63   0.89   0.91   0.84   0.03\n",
            "14907  Pox_TAP              PF03355.17          5     0.50    260   0.59   0.58   0.52   0.03\n",
            "14908  Pox_VERT_large       PF04441.16          9     0.73    697   0.59   0.57   0.51   0.04\n",
            "14909  Pox_vIL-18BP         PF05566.15          2     0.39    126   0.59   0.55   0.51   0.02\n",
            "14910  Pox_VLTF3            PF04947.17         14     2.78    168   0.59   0.55   0.52   0.06\n",
            "14911  Pox_VP8_L4R          PF04498.15          7     0.76    217   0.59   0.60   0.52   0.03\n",
            "14912  PP-binding           PF00550.28        152    33.39     67   0.84   0.92   0.79   0.05\n",
            "14913  PP-binding_2         PF14573.9           2     0.47     96   0.59   0.60   0.53   0.03\n",
            "14914  PP1                  PF07430.14          6     0.83     78   0.72   0.70   0.67   0.05\n",
            "14915  PP1c_bdg             PF10488.12          6     0.73    287   0.59   0.57   0.51   0.03\n",
            "14916  PP1_bind             PF15276.9           8     1.30     62   0.90   0.92   0.85   0.05\n",
            "14917  PP1_inhibitor        PF05361.15          5     0.63    141   0.59   0.59   0.52   0.04\n",
            "14918  PP2                  PF14299.9         192     3.00    146   0.59   0.55   0.41   0.03\n",
            "14919  PP28                 PF10252.12         87     2.23     85   0.67   0.69   0.58   0.17\n",
            "14920  PP2C                 PF00481.24         28     4.75    258   0.59   0.59   0.49   0.02\n",
            "14921  PP2C_2               PF13672.9          38     5.31    210   0.59   0.61   0.47   0.02\n",
            "14922  PP2C_C               PF07830.16         39     2.32     80   0.71   0.74   0.66   0.03\n",
            "14923  PP4R3                PF04802.18        110     1.96    191   0.59   0.58   0.53   0.05\n",
            "14924  PPAK                 PF02818.18         19     9.38     27   1.98   2.07   1.97   0.70\n",
            "14925  PPARgamma_N          PF12577.11          8     0.86     84   0.67   0.63   0.57   0.04\n",
            "14926  PPC                  PF04151.18         86    17.51     71   0.79   0.82   0.61   0.07\n",
            "14927  PPDFL                PF15060.9          17     0.92    110   0.59   0.56   0.53   0.03\n",
            "14928  PPDK_N               PF01326.22        103     3.04    328   0.59   0.59   0.49   0.02\n",
            "14929  PPE                  PF00823.22        115     2.64    158   0.59   0.59   0.56   0.06\n",
            "14930  PPE-PPW              PF18878.3          26     3.38     48   1.15   1.17   1.12   0.12\n",
            "14931  PPE-SVP              PF12484.11         89     5.72     80   0.71   0.75   0.61   0.11\n",
            "14932  PPIP5K2_N            PF18086.4          46     0.92     90   0.63   0.60   0.58   0.03\n",
            "14933  pPIWI_RE_REase       PF18154.4          34     3.92    117   0.59   0.58   0.50   0.02\n",
            "14934  pPIWI_RE_X           PF13111.9          37     7.01    400   0.59   0.58   0.46   0.02\n",
            "14935  pPIWI_RE_Y           PF18156.4          22     2.78    144   0.59   0.58   0.50   0.03\n",
            "14936  pPIWI_RE_Z           PF18155.4          26     4.68    167   0.59   0.58   0.54   0.02\n",
            "14937  PPI_Ypi1             PF07491.14         92     2.46     60   0.93   0.85   0.83   0.11\n",
            "14938  PPK2                 PF03976.17          8     0.94    229   0.59   0.56   0.54   0.03\n",
            "14939  PPL4                 PF18188.4           6     1.57    159   0.59   0.55   0.50   0.03\n",
            "14940  PPL5                 PF18168.4           2     0.58    337   0.59   0.60   0.51   0.01\n",
            "14941  Ppnp                 PF06865.14        130     1.50     92   0.62   0.58   0.57   0.03\n",
            "14942  PPO1_DWL             PF12142.11         29     3.01     52   1.07   0.98   1.01   0.10\n",
            "14943  PPO1_KFDV            PF12143.11         32     2.36    130   0.59   0.62   0.51   0.03\n",
            "14944  PPP1R26_N            PF15740.8          12     1.32    864   0.59   0.60   0.52   0.06\n",
            "14945  PPP1R32              PF15691.8          14     1.30    422   0.59   0.56   0.53   0.03\n",
            "14946  PPP1R35_C            PF15503.9          12     2.35    140   0.59   0.60   0.51   0.04\n",
            "14947  PPP4R2               PF09184.14         24     2.67    258   0.59   0.58   0.48   0.05\n",
            "14948  PPP5                 PF08321.15        117     2.42     94   0.61   0.60   0.50   0.04\n",
            "14949  PPPI_inhib           PF14895.9          20     0.97    346   0.59   0.57   0.51   0.04\n",
            "14950  PPR                  PF01535.23        458   458.00     31   1.12   1.06   1.12   0.09\n",
            "14951  PPR_1                PF12854.10        161    39.67     34   1.59   1.53   1.60   0.07\n",
            "14952  PPR_2                PF13041.9         182    16.17     50   1.11   1.03   1.08   0.05\n",
            "14953  PPR_3                PF13812.9           7     2.90     63   0.89   0.88   0.86   0.05\n",
            "14954  PPR_long             PF17177.7           9     1.14    212   0.59   0.59   0.53   0.03\n",
            "14955  PPS_PS               PF02006.19         76     1.03    176   0.59   0.63   0.53   0.03\n",
            "14956  PPTA                 PF01239.25        612   612.00     32   1.16   1.01   1.15   0.14\n",
            "14957  Pput2613-deam        PF14427.9           4     0.85    118   0.59   0.55   0.51   0.02\n",
            "14958  PPV_E1_C             PF00519.20         11     0.69    289   0.59   0.55   0.53   0.02\n",
            "14959  PPV_E1_DBD           PF20450.1          11     0.97    140   0.59   0.55   0.54   0.03\n",
            "14960  PPV_E1_N             PF00524.21          7     1.07    121   0.59   0.60   0.47   0.05\n",
            "14961  PPV_E2_C             PF00511.20         15     1.89     80   0.71   0.64   0.65   0.03\n",
            "14962  PPV_E2_N             PF00508.20         13     1.06    200   0.59   0.55   0.54   0.03\n",
            "14963  Ppx-GppA             PF02541.19         10     2.68    285   0.59   0.62   0.53   0.02\n",
            "14964  PP_kinase            PF02503.20        564     4.07    202   0.59   0.59   0.50   0.03\n",
            "14965  PP_kinase_C          PF13090.9          22     1.26    172   0.59   0.59   0.55   0.03\n",
            "14966  PP_kinase_C_1        PF17941.4         317     1.19    167   0.59   0.59   0.54   0.03\n",
            "14967  PP_kinase_N          PF13089.9         539     2.04    107   0.59   0.59   0.53   0.04\n",
            "14968  PP_M1                PF03012.17          5     0.60    296   0.59   0.60   0.53   0.02\n",
            "14969  pP_pnuc_1            PF18165.4          25     2.91    133   0.59   0.59   0.52   0.03\n",
            "14970  pP_pnuc_2            PF18166.4          15     2.16    122   0.59   0.57   0.54   0.03\n",
            "14971  PQ-loop              PF04193.17        207    23.83     61   0.92   0.90   0.91   0.10\n",
            "14972  PqiA                 PF04403.16        232     3.92    156   0.59   0.59   0.55   0.08\n",
            "14973  PQQ                  PF01011.24         65    65.00     38   1.07   1.04   1.04   0.06\n",
            "14974  PqqA                 PF08042.14         32    32.00     19   2.25   2.10   2.25   0.25\n",
            "14975  PqqD                 PF05402.15        386    21.02     67   0.84   0.88   0.72   0.04\n",
            "14976  PQQ_2                PF13360.9          45     7.32    237   0.59   0.60   0.43   0.03\n",
            "14977  PQQ_3                PF13570.9         622   622.00     40   1.17   1.18   1.02   0.10\n",
            "14978  PRA-CH               PF01502.21        562     1.64     74   0.76   0.66   0.70   0.05\n",
            "14979  PRA-PH               PF01503.20         37     4.68     83   0.68   0.71   0.61   0.04\n",
            "14980  PRA1                 PF03208.22        166     3.72    143   0.59   0.58   0.51   0.06\n",
            "14981  PRAI                 PF00697.25         17     2.25    193   0.59   0.62   0.51   0.02\n",
            "14982  PRANC                PF09372.13         34     4.09     97   0.59   0.56   0.53   0.07\n",
            "14983  PRAP                 PF15314.9          13     1.79     45   1.22   1.15   1.18   0.15\n",
            "14984  PRAS                 PF15798.8           8     0.75    123   0.59   0.58   0.51   0.03\n",
            "14985  PRC                  PF05239.19        114    22.07     79   0.72   0.76   0.63   0.05\n",
            "14986  PRC2_HTH_1           PF18118.4          13     1.27    101   0.59   0.59   0.50   0.05\n",
            "14987  PrcB_C               PF14343.9         147    11.63     58   0.96   0.96   0.89   0.07\n",
            "14988  PRCC                 PF10253.12         64     5.99    209   0.59   0.59   0.42   0.08\n",
            "14989  PRCH                 PF03967.16         25     1.06    132   0.59   0.58   0.53   0.01\n",
            "14990  PRD                  PF00874.23        164    10.44     90   0.63   0.62   0.57   0.07\n",
            "14991  Prd1-P2              PF09214.14          1     0.38    555   0.59   0.59   0.52   0.01\n",
            "14992  PRD1_DD              PF11087.11          2     0.99     54   1.03   1.12   0.98   0.13\n",
            "14993  PRD1_P5_C            PF08949.13          1     0.40    219   0.59   0.64   0.52   0.04\n",
            "14994  PRD1_P5_spike_N      PF08948.13          2     0.57    126   0.59   0.59   0.52   0.02\n",
            "14995  PRD_Mga              PF08270.14          3     1.05    220   0.59   0.56   0.52   0.05\n",
            "14996  Pre-PUA              PF17832.4          57     3.87     87   0.65   0.64   0.57   0.03\n",
            "14997  Pre-SET              PF05033.19        298     2.37    110   0.59   0.46   0.37   0.12\n",
            "14998  PreAtp-grasp         PF18604.4          83     4.69     94   0.61   0.65   0.52   0.03\n",
            "14999  preATP-grasp_3       PF18301.4          32     4.68     76   0.74   0.76   0.69   0.03\n",
            "15000  Prefoldin            PF02996.20         35     7.25    120   0.59   0.62   0.56   0.06\n",
            "15001  Prefoldin_2          PF01920.23         53     7.26    106   0.59   0.62   0.58   0.11\n",
            "15002  Prefoldin_3          PF13758.9          21     1.39     99   0.59   0.59   0.53   0.06\n",
            "15003  PRELI                PF04707.17         15     2.29    157   0.59   0.57   0.53   0.03\n",
            "15004  Prenylcys_lyase      PF07156.17         25     1.49    363   0.59   0.58   0.52   0.02\n",
            "15005  Prenyltrans          PF00432.24         63    63.00     44   1.23   1.17   1.18   0.07\n",
            "15006  Prenyltransf         PF01255.22       1161     1.97    221   0.59   0.57   0.46   0.02\n",
            "15007  PRESAN               PF09687.13        140     8.69    125   0.59   0.51   0.53   0.11\n",
            "15008  Presenilin           PF01080.20         53     1.80    417   0.59   0.59   0.51   0.03\n",
            "15009  Preseq_ALAS          PF09029.13         12     1.17    114   0.59   0.57   0.51   0.04\n",
            "15010  preSET_CXC           PF18264.4           5     2.03     32   1.69   1.29   1.60   0.66\n",
            "15011  PRE_C2HC             PF07530.14         10     3.29     68   0.83   0.83   0.76   0.05\n",
            "15012  PRF                  PF06875.14          2     0.42    214   0.59   0.60   0.51   0.02\n",
            "15013  PrgH                 PF09480.13          7     1.39    372   0.59   0.59   0.52   0.02\n",
            "15014  PrgI                 PF12666.10         80     5.71     91   0.63   0.62   0.50   0.05\n",
            "15015  PrgU                 PF09627.13          1     0.39    106   0.59   0.60   0.52   0.03\n",
            "15016  PRiA4_ORF3           PF07929.14         94     2.64    174   0.59   0.55   0.49   0.02\n",
            "15017  PriA_3primeBD        PF17764.4         329     4.24     96   0.60   0.60   0.52   0.02\n",
            "15018  PriA_C               PF18074.4         321     5.90     94   0.61   0.61   0.50   0.02\n",
            "15019  PriA_CRR             PF18319.4         113    19.66     27   1.98   1.58   1.98   0.50\n",
            "15020  Pribosyltran         PF00156.30         26     8.16    158   0.59   0.62   0.50   0.02\n",
            "15021  Pribosyltran_N       PF13793.9         303     2.72    116   0.59   0.61   0.52   0.03\n",
            "15022  Pribosyl_synth       PF14572.9           6     0.58    184   0.59   0.61   0.52   0.02\n",
            "15023  PriC                 PF07445.15         30     2.60    174   0.59   0.58   0.52   0.08\n",
            "15024  PriCT_1              PF08708.14         17     8.21     67   0.84   0.84   0.77   0.03\n",
            "15025  PriCT_2              PF08707.14         53     5.11     77   0.73   0.65   0.60   0.06\n",
            "15026  Prim-Pol             PF09250.14         99     5.76    158   0.59   0.57   0.37   0.02\n",
            "15027  PRIMA1               PF16101.8           6     0.58    122   0.59   0.57   0.52   0.04\n",
            "15028  Prim_Zn_Ribbon       PF08273.15         12     3.96     38   1.43   1.23   1.31   0.28\n",
            "15029  Prion                PF00377.21         12     1.12    117   0.59   0.54   0.52   0.04\n",
            "15030  Prion_bPrPp          PF11587.11          6     3.37     30   1.80   1.76   1.77   0.21\n",
            "15031  Prion_octapep        PF03991.15          5     5.00      8   3.40   3.03   3.39   1.64\n",
            "15032  Prismane             PF03063.23        155     3.16    511   0.59   0.57   0.48   0.02\n",
            "15033  PriX                 PF18689.4          10     0.67     95   0.60   0.59   0.54   0.04\n",
            "15034  PRK                  PF00485.21          9     1.51    196   0.59   0.59   0.53   0.02\n",
            "15035  PrkA                 PF06798.15         62     3.28    257   0.59   0.59   0.49   0.04\n",
            "15036  PRKCSH               PF07915.16        100     6.79     81   0.70   0.58   0.55   0.08\n",
            "15037  PRKCSH-like          PF12999.10          5     0.65    176   0.59   0.53   0.51   0.04\n",
            "15038  PRKCSH_1             PF13015.9           5     0.83    154   0.59   0.56   0.51   0.03\n",
            "15039  PRKG1_interact       PF15898.8          29     1.71    102   0.59   0.62   0.52   0.11\n",
            "15040  PrlF_antitoxin       PF15937.8           7     0.86     99   0.59   0.61   0.51   0.03\n",
            "15041  PrmA                 PF06325.16         27     1.53    295   0.59   0.59   0.51   0.02\n",
            "15042  PrmC_N               PF17827.4         470    10.98     71   0.79   0.84   0.75   0.04\n",
            "15043  PRMT5                PF05185.19         30     1.01    173   0.59   0.59   0.51   0.02\n",
            "15044  PRMT5_C              PF17286.5          30     1.29    177   0.59   0.52   0.48   0.03\n",
            "15045  PRMT5_TIM            PF17285.5          29     3.49    256   0.59   0.56   0.49   0.02\n",
            "15046  pRN1_helical         PF13010.9           2     0.46    136   0.59   0.57   0.51   0.03\n",
            "15047  PRNT                 PF15174.9           2     0.98     51   1.09   0.98   1.03   0.08\n",
            "15048  Pro-kuma_activ       PF09286.14         87     4.72    142   0.59   0.57   0.52   0.02\n",
            "15049  Pro-MCH              PF05824.15          4     0.61     86   0.66   0.63   0.59   0.02\n",
            "15050  Pro-NT_NN            PF07421.14          7     0.84    161   0.59   0.57   0.51   0.03\n",
            "15051  Pro-rich             PF15240.9          13     1.30    180   0.59   0.58   0.42   0.16\n",
            "15052  Pro-rich_19          PF15455.9          10     0.72    363   0.59   0.59   0.52   0.03\n",
            "15053  PRO8NT               PF08082.14         39     0.97    152   0.59   0.55   0.53   0.03\n",
            "15054  PROCN                PF08083.14         15     0.64    407   0.59   0.56   0.52   0.02\n",
            "15055  PROCT                PF08084.14         79     1.34    123   0.59   0.53   0.52   0.02\n",
            "15056  PRODH                PF18327.4         146    10.40     48   1.15   1.23   1.14   0.15\n",
            "15057  Profilin             PF00235.22        233     5.36    127   0.59   0.57   0.48   0.03\n",
            "15058  Prog_receptor        PF02161.18          9     0.62    564   0.59   0.62   0.52   0.03\n",
            "15059  Proho_convert        PF12177.11          8     1.77     39   1.40   1.33   1.37   0.16\n",
            "15060  Prok-E2_A            PF14457.9           6     2.83    163   0.59   0.57   0.54   0.02\n",
            "15061  Prok-E2_B            PF14461.9           5     1.91    135   0.59   0.56   0.50   0.01\n",
            "15062  Prok-E2_C            PF14459.9           1     0.40    129   0.59   0.61   0.52   0.02\n",
            "15063  Prok-E2_D            PF14460.9          16     1.69    171   0.59   0.54   0.50   0.02\n",
            "15064  Prok-E2_E            PF14462.9          10     1.12    123   0.59   0.54   0.52   0.02\n",
            "15065  Prok-JAB             PF14464.9          59     7.52    117   0.59   0.55   0.44   0.03\n",
            "15066  Prok-RING_1          PF14446.9          22     1.80     54   1.03   0.81   0.98   0.20\n",
            "15067  Prok-RING_2          PF14445.9           3     0.69     56   0.99   0.84   0.94   0.12\n",
            "15068  Prok-RING_4          PF14447.9          10     2.04     46   1.20   0.96   1.11   0.27\n",
            "15069  Prok-TraM            PF09228.13         14     1.60    102   0.59   0.60   0.54   0.04\n",
            "15070  Prokineticin         PF06607.14          7     0.57     97   0.59   0.53   0.51   0.05\n",
            "15071  prok_STING           PF20300.1          38     5.86    139   0.59   0.60   0.46   0.02\n",
            "15072  Prok_Ub              PF14454.9          24     3.34     64   0.88   0.88   0.82   0.02\n",
            "15073  PROL5-SMR            PF15621.9           9     1.23    104   0.59   0.56   0.48   0.06\n",
            "15074  Prolactin_RP         PF15172.9           6     1.56     45   1.22   1.16   1.19   0.10\n",
            "15075  Prolamin_like        PF05617.14         81     4.41     67   0.84   0.67   0.70   0.16\n",
            "15076  Promethin            PF16015.8          68     3.91    106   0.59   0.61   0.54   0.13\n",
            "15077  Prominin             PF05478.14         27     2.10    799   0.59   0.57   0.53   0.03\n",
            "15078  PRONE                PF03759.16         53     0.86    363   0.59   0.59   0.52   0.02\n",
            "15079  Propeptide_C1        PF08127.16         72    11.25     40   1.37   1.31   1.26   0.06\n",
            "15080  Propeptide_C25       PF08126.14          4     1.01    205   0.59   0.59   0.52   0.02\n",
            "15081  Propep_M14           PF02244.19        324    13.38     73   0.77   0.77   0.70   0.05\n",
            "15082  Prophage_tail        PF06605.14         22     4.28    267   0.59   0.57   0.48   0.03\n",
            "15083  Prophage_tailD1      PF18994.3          53     6.27     85   0.67   0.65   0.55   0.05\n",
            "15084  ProQ                 PF04352.16         51     2.56    110   0.59   0.59   0.53   0.03\n",
            "15085  ProQ_C               PF17516.5          51     4.70     51   1.09   1.17   1.07   0.10\n",
            "15086  PRORP                PF16953.8          16     1.07    242   0.59   0.53   0.52   0.03\n",
            "15087  ProRS-C_1            PF09180.14        370     5.26     70   0.80   0.73   0.66   0.05\n",
            "15088  ProRS-C_2            PF09181.13          4     0.89     66   0.85   0.88   0.78   0.06\n",
            "15089  ProSAAS              PF07259.15         11     1.63    269   0.59   0.61   0.43   0.04\n",
            "15090  Prosystemin          PF07376.14          2     0.43    202   0.59   0.59   0.51   0.06\n",
            "15091  Protamine_3          PF08188.14          2     0.80     48   1.15   1.11   1.10   0.24\n",
            "15092  Protamine_like       PF06382.14          2     0.63    146   0.59   0.54   0.50   0.05\n",
            "15093  Protamine_P1         PF00260.23          6     0.99     49   1.13   0.96   1.06   0.55\n",
            "15094  Protamine_P2         PF00841.22          4     0.63     89   0.64   0.57   0.55   0.17\n",
            "15095  Proteasome           PF00227.29        153     6.76    190   0.59   0.60   0.54   0.02\n",
            "15096  Proteasome_A_N       PF10584.12         99    36.72     23   2.31   2.27   2.28   0.14\n",
            "15097  Proteasom_PSMB       PF10508.12          6     1.40    499   0.59   0.60   0.52   0.02\n",
            "15098  Proteasom_Rpn13      PF04683.16        128     2.57     87   0.65   0.59   0.53   0.03\n",
            "15099  Protein_K            PF12283.11          1     0.72     56   0.99   1.06   0.93   0.07\n",
            "15100  protein_MS5          PF04776.15         51     1.86    120   0.59   0.57   0.48   0.03\n",
            "15101  Prothymosin          PF03247.17         13     1.43    112   0.59   0.62   0.47   0.24\n",
            "15102  Protocadherin        PF08374.14         16     1.07    217   0.59   0.59   0.49   0.03\n",
            "15103  Protoglobin          PF11563.11         72     6.35    158   0.59   0.58   0.54   0.03\n",
            "15104  Proton_antipo_C      PF01010.22         77     1.61    243   0.59   0.57   0.51   0.03\n",
            "15105  Proton_antipo_M      PF00361.23         14     2.47    293   0.59   0.60   0.53   0.07\n",
            "15106  Proton_antipo_N      PF00662.23        411     6.05     51   1.09   1.00   1.01   0.12\n",
            "15107  Prot_ATP_ID_OB       PF16450.8          88    12.76     56   0.99   1.09   0.91   0.06\n",
            "15108  Prot_ATP_OB_N        PF17758.4          66     7.58     62   0.90   0.97   0.85   0.05\n",
            "15109  Prot_inhib_II        PF02428.18         23     2.02     51   1.08   0.86   1.01   0.26\n",
            "15110  Pro_3_hydrox_C       PF05373.14          3     0.71    101   0.59   0.59   0.52   0.02\n",
            "15111  Pro_Al_protease      PF02983.17         85    10.81     59   0.95   1.03   0.87   0.12\n",
            "15112  Pro_CA               PF00484.22        252     2.49    156   0.59   0.58   0.49   0.03\n",
            "15113  Pro_CA_2             PF20393.1          38     1.33    125   0.59   0.55   0.54   0.04\n",
            "15114  Pro_dh               PF01619.21        197     3.12    282   0.59   0.58   0.48   0.03\n",
            "15115  Pro_dh-DNA_bdg       PF14850.9          86     1.26    112   0.59   0.62   0.53   0.02\n",
            "15116  Pro_isomerase        PF00160.24         62     3.79    158   0.59   0.58   0.43   0.02\n",
            "15117  Pro_racemase         PF05544.14         14     1.21    325   0.59   0.60   0.54   0.01\n",
            "15118  Pro_sub2             PF18513.4           2     0.50     88   0.65   0.63   0.58   0.05\n",
            "15119  Prp18                PF02840.18         83     1.36    143   0.59   0.57   0.53   0.03\n",
            "15120  Prp19                PF08606.14         76     2.17     68   0.83   0.85   0.79   0.06\n",
            "15121  PRP1_N               PF06424.15        105     1.59    144   0.59   0.58   0.50   0.06\n",
            "15122  PRP21_like_P         PF12230.11         13     2.35    217   0.59   0.56   0.51   0.06\n",
            "15123  PRP3                 PF08572.13        115     2.38    222   0.59   0.56   0.46   0.07\n",
            "15124  Prp31_C              PF09785.12         85     1.96    124   0.59   0.60   0.48   0.06\n",
            "15125  PRP38                PF03371.18        132     2.81    174   0.59   0.56   0.50   0.03\n",
            "15126  PRP38_assoc          PF12871.10         30     7.80     99   0.59   0.61   0.48   0.24\n",
            "15127  PRP4                 PF08799.14        150    26.08     29   1.85   1.94   1.84   0.24\n",
            "15128  PRP8_domainIV        PF12134.11         16     0.74    230   0.59   0.58   0.53   0.02\n",
            "15129  PRP9_N               PF16958.8          15     1.60    150   0.59   0.58   0.49   0.07\n",
            "15130  PrpF                 PF04303.16          5     0.77    384   0.59   0.61   0.52   0.02\n",
            "15131  PrpR_N               PF06506.14         80     4.83    167   0.59   0.63   0.55   0.03\n",
            "15132  PRR18                PF15671.8           3     0.54    266   0.59   0.62   0.52   0.03\n",
            "15133  PRR20                PF15708.8           3     0.68    221   0.59   0.59   0.51   0.03\n",
            "15134  PRR22                PF15776.8           8     0.59    366   0.59   0.60   0.51   0.02\n",
            "15135  PRRSV_2b             PF07069.14          2     0.52     73   0.77   0.78   0.71   0.07\n",
            "15136  PRRSV_Env            PF02340.18          4     0.64    234   0.59   0.56   0.52   0.03\n",
            "15137  PrsW-protease        PF13367.9          64     3.89    196   0.59   0.60   0.48   0.06\n",
            "15138  PRT6_C               PF18995.3         149     3.96    456   0.59   0.55   0.48   0.02\n",
            "15139  PRTase_1             PF11202.11         55     1.03    246   0.59   0.60   0.52   0.02\n",
            "15140  PRTase_2             PF15609.9          57     1.40    190   0.59   0.60   0.50   0.02\n",
            "15141  PRTase_3             PF15610.9          11     1.24    264   0.59   0.57   0.53   0.03\n",
            "15142  PRTP                 PF01366.21         12     1.59    659   0.59   0.57   0.52   0.02\n",
            "15143  PRTRC_E              PF19556.2          27     1.48    175   0.59   0.59   0.53   0.05\n",
            "15144  PRT_C                PF08372.13         22     0.94    156   0.59   0.58   0.53   0.02\n",
            "15145  PRY                  PF13765.9         114    10.28     49   1.13   1.11   1.10   0.04\n",
            "15146  Pr_beta_C            PF12465.11         98    22.49     36   1.51   1.54   1.47   0.12\n",
            "15147  PS-DH                PF14765.9          75     6.78    297   0.59   0.59   0.49   0.01\n",
            "15148  PsaA_PsaB            PF00223.22         15     0.63    717   0.59   0.55   0.51   0.02\n",
            "15149  PsaD                 PF02531.19         25     0.70    136   0.59   0.59   0.53   0.01\n",
            "15150  PsaF                 PF17550.5           1     0.38    162   0.59   0.58   0.52   0.03\n",
            "15151  PsaL                 PF02605.18         50     1.08    150   0.59   0.60   0.52   0.01\n",
            "15152  PsaM                 PF07465.16         19     7.11     29   1.85   2.04   1.84   0.27\n",
            "15153  PsaN                 PF05479.14          7     0.76    132   0.59   0.59   0.52   0.04\n",
            "15154  PsaX                 PF08078.15         10     2.43     35   1.55   1.55   1.52   0.20\n",
            "15155  PSA_CBD              PF18341.4           9     3.35     61   0.92   0.82   0.86   0.07\n",
            "15156  Psb28                PF03912.17         66     1.20    106   0.59   0.57   0.52   0.03\n",
            "15157  PsbH                 PF00737.23         12     1.56     52   1.07   1.09   1.02   0.08\n",
            "15158  PsbI                 PF02532.17         18     2.96     36   1.51   1.54   1.48   0.12\n",
            "15159  PsbJ                 PF01788.20         17     1.92     39   1.40   1.44   1.37   0.20\n",
            "15160  PsbK                 PF02533.18         14     1.56     41   1.34   1.35   1.30   0.18\n",
            "15161  PsbL                 PF02419.20          7     1.73     37   1.47   1.50   1.44   0.15\n",
            "15162  PsbM                 PF05151.15         14     4.08     31   1.74   1.81   1.70   0.26\n",
            "15163  PsbN                 PF02468.18         40     2.38     43   1.28   1.28   1.25   0.07\n",
            "15164  PsbP                 PF01789.19         75     2.95    156   0.59   0.58   0.48   0.03\n",
            "15165  PsbP_2               PF18933.3           6     2.77    193   0.59   0.58   0.48   0.04\n",
            "15166  PsbQ                 PF05757.14         20     1.97    200   0.59   0.63   0.50   0.03\n",
            "15167  PsbR                 PF04725.15          7     0.57     98   0.59   0.60   0.52   0.03\n",
            "15168  PsbT                 PF01405.20         14     4.12     29   1.85   1.87   1.83   0.27\n",
            "15169  PsbU                 PF06514.14         10     0.83     93   0.62   0.61   0.55   0.02\n",
            "15170  PsbW                 PF07123.15         15     0.99    132   0.59   0.63   0.50   0.02\n",
            "15171  PsbX                 PF06596.14         34     4.79     38   1.43   1.54   1.39   0.14\n",
            "15172  PsbY                 PF06298.14         42     5.07     35   1.55   1.59   1.52   0.22\n",
            "15173  PSCyt1               PF07635.14         62     5.68     59   0.95   0.88   0.88   0.06\n",
            "15174  PSCyt2               PF07583.14        220     2.15    209   0.59   0.55   0.52   0.02\n",
            "15175  PSCyt3               PF07627.14        111     1.60     98   0.59   0.56   0.51   0.02\n",
            "15176  PSD1                 PF07587.14        191     2.86    244   0.59   0.58   0.50   0.03\n",
            "15177  PSD2                 PF07624.14         84     5.77     74   0.76   0.80   0.74   0.03\n",
            "15178  PSD3                 PF07626.14         75     5.47     65   0.86   0.90   0.81   0.04\n",
            "15179  PSD4                 PF07631.14        110     2.66    128   0.59   0.59   0.53   0.03\n",
            "15180  PSD5                 PF07637.14         94     6.77     62   0.90   0.93   0.86   0.05\n",
            "15181  PSDC                 PF12588.11         85     1.94    140   0.59   0.56   0.52   0.02\n",
            "15182  PseudoU_synth_1      PF01416.23        262     5.33    108   0.59   0.57   0.49   0.02\n",
            "15183  PseudoU_synth_2      PF00849.25         53     7.02    160   0.59   0.57   0.48   0.03\n",
            "15184  Psg1                 PF14610.9          93     4.38    194   0.59   0.54   0.46   0.02\n",
            "15185  PSGP                 PF07276.14          3     3.00     13   2.55   2.73   2.53   0.81\n",
            "15186  PSI                  PF01437.28         95     6.08     51   1.09   0.78   0.90   0.35\n",
            "15187  PsiA                 PF06952.14          3     0.47    237   0.59   0.58   0.52   0.02\n",
            "15188  PsiB                 PF06290.14          7     0.99    138   0.59   0.58   0.54   0.01\n",
            "15189  PsiE                 PF06146.15        223     5.28    120   0.59   0.65   0.52   0.12\n",
            "15190  PsiF_repeat          PF07769.17         54     5.16     34   1.59   1.47   1.58   0.29\n",
            "15191  PSII                 PF00421.22         16     0.89    511   0.59   0.56   0.51   0.01\n",
            "15192  PSII_BNR             PF14870.9          15     0.79    304   0.59   0.57   0.52   0.02\n",
            "15193  PSII_Pbs27           PF13326.9          17     2.88    141   0.59   0.61   0.51   0.04\n",
            "15194  PSII_Pbs31           PF18240.4           4     0.92     93   0.61   0.66   0.55   0.06\n",
            "15195  PSII_Ycf12           PF05969.14         11     5.16     33   1.64   1.76   1.60   0.24\n",
            "15196  PSI_8                PF00796.21         25     8.52     25   2.13   2.20   2.12   0.58\n",
            "15197  PSI_integrin         PF17205.6          94     5.03     46   1.20   0.92   1.08   0.35\n",
            "15198  PSI_PsaE             PF02427.20         25     1.24     60   0.93   0.89   0.86   0.06\n",
            "15199  PSI_PsaF             PF02507.18          9     0.74    172   0.59   0.59   0.53   0.01\n",
            "15200  PSI_PsaH             PF03244.17          8     0.71    139   0.59   0.61   0.52   0.01\n",
            "15201  PSI_PsaJ             PF01701.21         30     6.22     37   1.47   1.50   1.44   0.18\n",
            "15202  PSI_PSAK             PF01241.21         24     2.43     70   0.80   0.85   0.71   0.06\n",
            "15203  PSK                  PF06404.15         38     1.78     90   0.63   0.59   0.50   0.07\n",
            "15204  PSK_trans_fac        PF07704.14         52     4.83     82   0.69   0.74   0.61   0.07\n",
            "15205  PSMalpha             PF17063.8           5     5.00     21   1.90   1.98   1.88   0.40\n",
            "15206  PSMbeta              PF05480.14         18     4.65     43   1.28   1.38   1.25   0.14\n",
            "15207  PSMdelta             PF19693.2           3     2.92     22   2.41   2.46   2.39   0.60\n",
            "15208  PSP                  PF04046.19         89     4.75     46   1.20   1.16   1.12   0.09\n",
            "15209  PSP1                 PF04468.15        401     2.92     86   0.66   0.64   0.58   0.06\n",
            "15210  PSP94                PF05825.14          4     0.57     94   0.61   0.49   0.54   0.08\n",
            "15211  PspA_IM30            PF04012.15         11     3.53    221   0.59   0.64   0.54   0.10\n",
            "15212  PspB                 PF06667.15         36     1.80     73   0.77   0.76   0.73   0.05\n",
            "15213  PspC                 PF04024.15        612     6.39     53   1.05   1.07   0.91   0.09\n",
            "15214  PSRP-3_Ycf65         PF04839.16         16     1.30     48   1.15   1.00   1.11   0.14\n",
            "15215  PSRT                 PF07636.14          4     2.56     32   1.69   1.66   1.67   0.35\n",
            "15216  PSS                  PF03034.18         58     1.29    273   0.59   0.52   0.52   0.05\n",
            "15217  Psu                  PF07455.14          1     0.40    182   0.59   0.59   0.52   0.02\n",
            "15218  PSY3                 PF16836.8          12     1.63    216   0.59   0.58   0.48   0.03\n",
            "15219  PS_Dcarbxylase       PF02666.18        107     4.82    200   0.59   0.58   0.45   0.01\n",
            "15220  PS_pyruv_trans       PF04230.16        154    54.31    286   0.59   0.58   0.47   0.01\n",
            "15221  PT                   PF04886.15         14     6.48     36   1.51   1.49   1.49   0.61\n",
            "15222  PT-HINT              PF07591.14          3     0.64    137   0.59   0.59   0.52   0.02\n",
            "15223  PT-TG                PF14449.9          31     8.23     69   0.82   0.89   0.71   0.05\n",
            "15224  PT-VENN              PF04829.16         25     4.80     52   1.07   1.19   1.03   0.13\n",
            "15225  PTAC                 PF06130.15        274     4.39     67   0.84   0.84   0.75   0.07\n",
            "15226  ptaRNA1_toxin        PF12703.10          2     0.70     73   0.77   0.79   0.72   0.06\n",
            "15227  PTase_Orf2           PF11468.11          5     1.11    287   0.59   0.57   0.51   0.01\n",
            "15228  PTA_PTB              PF01515.22         17     1.95    319   0.59   0.62   0.54   0.02\n",
            "15229  PTB                  PF08416.16         11     1.67    131   0.59   0.58   0.52   0.03\n",
            "15230  PTCB-BRCT            PF12738.10         35     5.25     63   0.89   0.85   0.85   0.04\n",
            "15231  PTCRA                PF15028.9           6     0.57    127   0.59   0.59   0.52   0.02\n",
            "15232  PTE                  PF02126.21          3     0.84    298   0.59   0.59   0.52   0.02\n",
            "15233  PTEN_C2              PF10409.12         48     3.93    135   0.59   0.55   0.48   0.04\n",
            "15234  Pterin_4a            PF01329.22        481     4.07     92   0.62   0.59   0.51   0.04\n",
            "15235  Pterin_bind          PF00809.25         29     2.96    244   0.59   0.61   0.51   0.02\n",
            "15236  PTH2                 PF01981.19        274     2.98    117   0.59   0.59   0.50   0.03\n",
            "15237  PTN13_u3             PF16599.8          17     1.85    191   0.59   0.59   0.53   0.08\n",
            "15238  PTN_MK_C             PF01091.21         20     1.85     65   0.86   0.75   0.80   0.17\n",
            "15239  PTN_MK_N             PF05196.16         12     1.02     57   0.98   0.82   0.91   0.18\n",
            "15240  PTP2                 PF17022.8          11     2.36    217   0.59   0.54   0.51   0.08\n",
            "15241  PTPA                 PF03095.18        247     1.86    303   0.59   0.56   0.49   0.02\n",
            "15242  PTPLA                PF04387.17        213     2.33    161   0.59   0.55   0.51   0.07\n",
            "15243  PTPlike_phytase      PF14566.9          41     3.67    156   0.59   0.57   0.50   0.02\n",
            "15244  PTPRCAP              PF15713.8          15     1.25    156   0.59   0.59   0.49   0.06\n",
            "15245  PTPS                 PF01242.22         87     3.71    123   0.59   0.55   0.48   0.03\n",
            "15246  PTPS_related         PF10131.12          9     2.47    621   0.59   0.58   0.51   0.02\n",
            "15247  PTP_N                PF12453.11          4     3.39     26   2.06   2.16   2.04   0.28\n",
            "15248  PTP_tm               PF18861.4          39     2.17    126   0.59   0.57   0.51   0.03\n",
            "15249  PTR                  PF12789.10         16     1.35     60   0.93   0.89   0.87   0.16\n",
            "15250  PTR2                 PF00854.24         21     3.13    397   0.59   0.58   0.51   0.04\n",
            "15251  PTRF_SDPR            PF15237.9          22     1.08    252   0.59   0.61   0.50   0.07\n",
            "15252  PTS-HPr              PF00381.22        939     6.35     82   0.69   0.75   0.60   0.04\n",
            "15253  PTSIIA_gutA          PF03829.16         79     2.16    113   0.59   0.60   0.53   0.02\n",
            "15254  PTSIIB_sorb          PF03830.18        250     2.75    148   0.59   0.59   0.54   0.03\n",
            "15255  PTS_2-RNA            PF01885.19        220     2.48    187   0.59   0.59   0.43   0.03\n",
            "15256  PTS_EIIA_1           PF00358.23        606     2.69    127   0.59   0.61   0.54   0.03\n",
            "15257  PTS_EIIA_2           PF00359.25         84     6.47    144   0.59   0.61   0.50   0.03\n",
            "15258  PTS_EIIB             PF00367.23        712    22.36     34   1.59   1.62   1.56   0.11\n",
            "15259  PTS_EIIC             PF02378.21         49     7.04    324   0.59   0.61   0.49   0.07\n",
            "15260  PTS_EIIC_2           PF13303.9          32     2.68    331   0.59   0.63   0.53   0.08\n",
            "15261  PTS_IIA              PF02255.19        188     2.68     94   0.61   0.62   0.58   0.07\n",
            "15262  PTS_IIB              PF02302.20        227    10.31     90   0.63   0.66   0.51   0.05\n",
            "15263  PUA                  PF01472.23         89     9.44     74   0.76   0.82   0.72   0.05\n",
            "15264  PUA_2                PF14306.9          61     2.13    158   0.59   0.60   0.52   0.01\n",
            "15265  PUA_3                PF17785.4         143    10.09     64   0.88   0.89   0.82   0.02\n",
            "15266  PUA_4                PF20260.1          84    68.69     48   1.15   1.18   1.12   0.07\n",
            "15267  PUB                  PF09409.13         82     7.14     82   0.69   0.70   0.61   0.04\n",
            "15268  PUB_1                PF18486.4           4     0.92     64   0.87   0.94   0.82   0.04\n",
            "15269  PUCC                 PF03209.18          9     1.29    401   0.59   0.63   0.53   0.05\n",
            "15270  PucR                 PF07905.14        165     4.81    117   0.59   0.63   0.50   0.02\n",
            "15271  PUD                  PF03714.17        119     4.90     94   0.61   0.54   0.38   0.05\n",
            "15272  PUD1_2               PF18457.4          25     1.93    176   0.59   0.56   0.52   0.04\n",
            "15273  PUF                  PF00806.22       1109  1109.00     35   1.47   1.38   1.46   0.12\n",
            "15274  PUFD                 PF16553.8          12     0.76    114   0.59   0.59   0.53   0.02\n",
            "15275  PufQ                 PF05398.14          6     0.95     74   0.76   0.74   0.70   0.04\n",
            "15276  PUL                  PF08324.14        174     8.48    276   0.59   0.63   0.42   0.03\n",
            "15277  PulA_N1              PF17999.4          40     2.82     85   0.67   0.66   0.64   0.03\n",
            "15278  PulG                 PF11773.11          6     1.13     82   0.69   0.76   0.65   0.08\n",
            "15279  Pullulanase_Ins      PF18494.4          30     1.70     74   0.76   0.75   0.70   0.06\n",
            "15280  Pullulanase_N2       PF17967.4          38     2.60    112   0.59   0.61   0.51   0.04\n",
            "15281  Pullul_strch_C       PF11852.11         68     2.05    167   0.59   0.59   0.53   0.02\n",
            "15282  PUMA                 PF15826.8           4     0.58    189   0.59   0.60   0.50   0.02\n",
            "15283  PUNUT                PF16913.8          16     1.57    322   0.59   0.60   0.53   0.06\n",
            "15284  Pup                  PF05639.14         23     1.45     69   0.82   0.85   0.74   0.10\n",
            "15285  Pup_ligase           PF03136.18        108     1.84    440   0.59   0.58   0.49   0.02\n",
            "15286  PurA                 PF04845.16          5     0.72    219   0.59   0.59   0.51   0.02\n",
            "15287  PurK_C               PF17769.4         107     4.91     56   0.99   0.97   0.91   0.05\n",
            "15288  PurL_C               PF16904.8           2     0.57     92   0.62   0.62   0.54   0.02\n",
            "15289  PurS                 PF02700.17        297     3.65     77   0.73   0.77   0.69   0.04\n",
            "15290  Pur_ac_phosph_N      PF16656.8         221     7.98     94   0.61   0.56   0.48   0.06\n",
            "15291  Pur_DNA_glyco        PF02245.19        331     2.00    183   0.59   0.59   0.49   0.01\n",
            "15292  PuR_N                PF09182.13         77     1.90     70   0.80   0.84   0.77   0.02\n",
            "15293  putAbiC              PF16872.8          17     3.21     80   0.71   0.67   0.64   0.08\n",
            "15294  Putative_G5P         PF17426.5           2     0.56    108   0.59   0.59   0.51   0.03\n",
            "15295  Putative_PNPOx       PF01243.23        117    11.39     88   0.65   0.66   0.57   0.01\n",
            "15296  PutA_N               PF18083.4          11     1.66    113   0.59   0.59   0.54   0.02\n",
            "15297  Put_DNA-bind_N       PF06971.16        201     6.38     49   1.13   1.19   1.10   0.06\n",
            "15298  Put_Phosphatase      PF06888.15          9     1.02    234   0.59   0.56   0.52   0.02\n",
            "15299  PV-1                 PF06637.14          6     0.55    439   0.59   0.60   0.53   0.03\n",
            "15300  Pvc16_N              PF14065.9         118     7.90    177   0.59   0.61   0.46   0.02\n",
            "15301  PvlArgDC             PF01862.19         88     2.61    162   0.59   0.61   0.51   0.02\n",
            "15302  PVL_ORF50            PF07768.14          3     0.63    116   0.59   0.54   0.51   0.05\n",
            "15303  PV_NSP1              PF12433.11          2     0.74     82   0.69   0.66   0.63   0.04\n",
            "15304  PWI                  PF01480.20         98     3.60     72   0.78   0.77   0.69   0.05\n",
            "15305  PWWP                 PF00855.20        198     5.95     93   0.61   0.55   0.48   0.03\n",
            "15306  PX                   PF00787.27         65     5.83     84   0.68   0.64   0.56   0.06\n",
            "15307  PXA                  PF02194.18        280     6.17    182   0.59   0.59   0.47   0.03\n",
            "15308  PXB                  PF12828.10         75     2.38    130   0.59   0.59   0.48   0.03\n",
            "15309  pXO2-34              PF17362.5           2     0.77     79   0.72   0.72   0.66   0.03\n",
            "15310  pXO2-72              PF17443.5           2     0.75     62   0.90   0.88   0.85   0.05\n",
            "15311  PXPV                 PF12778.10          9     7.77     22   2.41   2.45   2.34   1.00\n",
            "15312  PXT1                 PF15214.9           6     1.19     51   1.09   1.04   1.04   0.10\n",
            "15313  PYC_OADA             PF02436.21        373     3.32    201   0.59   0.59   0.50   0.02\n",
            "15314  pYEATS               PF20305.1          58    10.84     73   0.77   0.73   0.70   0.03\n",
            "15315  PYNP_C               PF07831.16         44     3.76     75   0.75   0.79   0.72   0.04\n",
            "15316  PyocinActivator      PF11112.11         16     2.52     74   0.76   0.79   0.71   0.03\n",
            "15317  Pyocin_S             PF06958.15         36     4.85    139   0.59   0.59   0.52   0.03\n",
            "15318  PyrBI_leader         PF08052.14          2     0.86     44   1.25   1.20   1.20   0.10\n",
            "15319  PyrI                 PF01948.21        207     1.84     94   0.61   0.63   0.55   0.06\n",
            "15320  Pyridoxal_deC        PF00282.22          9     1.52    375   0.59   0.57   0.52   0.01\n",
            "15321  Pyridox_oxase_2      PF12766.10         47     1.71    100   0.59   0.55   0.47   0.03\n",
            "15322  Pyridox_ox_2         PF12900.10         79     6.79    142   0.59   0.60   0.47   0.02\n",
            "15323  Pyrid_oxidase_2      PF13883.9          30     2.47    170   0.59   0.56   0.50   0.01\n",
            "15324  Pyrid_ox_like        PF16242.8          18     1.62    149   0.59   0.55   0.54   0.03\n",
            "15325  PYRIN                PF02758.19        180    10.17     75   0.75   0.77   0.67   0.06\n",
            "15326  PyrI_C               PF02748.18        129     3.72     48   1.15   0.99   1.07   0.19\n",
            "15327  Pyrophosphatase      PF00719.22        498     2.80    160   0.59   0.57   0.49   0.02\n",
            "15328  Pyr_excise           PF03013.17         81     6.47     82   0.69   0.63   0.56   0.04\n",
            "15329  Pyr_redox            PF00070.30         90     8.01     81   0.70   0.76   0.60   0.04\n",
            "15330  Pyr_redox_2          PF07992.17         64     5.86    295   0.59   0.62   0.45   0.02\n",
            "15331  Pyr_redox_3          PF13738.9          60     2.70    305   0.59   0.59   0.52   0.02\n",
            "15332  Pyr_redox_dim        PF02852.25         82     4.13    110   0.59   0.60   0.53   0.02\n",
            "15333  PYST-C1              PF09690.13         10     2.64     55   1.01   0.99   0.91   0.23\n",
            "15334  PY_rept_46           PF09689.13          6     1.60     51   1.09   1.09   1.01   0.22\n",
            "15335  P_C                  PF06640.14          2     0.52    235   0.59   0.60   0.50   0.04\n",
            "15336  P_C10                PF14974.9          18     1.40    103   0.59   0.59   0.54   0.03\n",
            "15337  P_gingi_FimA         PF06321.14         32     8.20    156   0.59   0.61   0.40   0.05\n",
            "15338  P_proprotein         PF01483.23         42     2.99     86   0.66   0.65   0.60   0.03\n",
            "15339  QCR10                PF09796.12         92     4.20     63   0.89   0.87   0.85   0.04\n",
            "15340  QcrA_N               PF19297.2          85     1.42    221   0.59   0.60   0.52   0.01\n",
            "15341  QH-AmDH_gamma        PF08992.14          9     0.68     75   0.76   0.65   0.70   0.06\n",
            "15342  QIL1                 PF15884.8          63     6.12     77   0.73   0.68   0.67   0.03\n",
            "15343  QLQ                  PF08880.14        115    18.89     35   1.55   1.52   1.53   0.20\n",
            "15344  Qn_am_d_aII          PF14930.9          11     1.31    107   0.59   0.58   0.53   0.03\n",
            "15345  Qn_am_d_aIII         PF09099.13          5     1.45     82   0.69   0.77   0.63   0.04\n",
            "15346  Qn_am_d_aIV          PF09100.13         12     1.00    133   0.59   0.59   0.52   0.02\n",
            "15347  QPE                  PF18874.3          36     6.36     44   1.25   1.28   1.21   0.32\n",
            "15348  QRPTase_C            PF01729.22         15     1.76    169   0.59   0.62   0.52   0.03\n",
            "15349  QRPTase_N            PF02749.19        110     4.81     89   0.64   0.68   0.59   0.02\n",
            "15350  QslA_E               PF18226.4           4     0.85     71   0.79   0.75   0.74   0.03\n",
            "15351  QSOX_Trx1            PF18108.4           3     0.63    108   0.59   0.60   0.53   0.01\n",
            "15352  QSregVF              PF13652.9           7     0.56    112   0.59   0.59   0.53   0.02\n",
            "15353  QSregVF_b            PF12843.10         79     1.32     66   0.85   0.83   0.80   0.03\n",
            "15354  Qua1                 PF16274.8          12     2.60     52   1.07   1.09   1.03   0.06\n",
            "15355  Quaking_NLS          PF16551.8           3     1.63     30   1.79   1.74   1.76   0.22\n",
            "15356  QueC                 PF06508.16         28     1.36    209   0.59   0.57   0.52   0.02\n",
            "15357  QueF                 PF14489.9          20     1.43     81   0.70   0.67   0.66   0.02\n",
            "15358  QueF_N               PF14819.9          64     0.89    111   0.59   0.58   0.53   0.02\n",
            "15359  QueG_DUF1730         PF08331.13        353     3.64     76   0.74   0.73   0.60   0.04\n",
            "15360  QueH                 PF02677.17        258     1.33    175   0.59   0.51   0.52   0.05\n",
            "15361  Questin_oxidase      PF14027.9         202     4.86    348   0.59   0.57   0.41   0.02\n",
            "15362  QueT                 PF06177.14        189     3.23    139   0.59   0.63   0.44   0.10\n",
            "15363  Queuosine_synth      PF02547.18        594     1.81    325   0.59   0.59   0.47   0.02\n",
            "15364  QVR                  PF17064.8          48     1.61     85   0.67   0.48   0.51   0.21\n",
            "15365  QWRF                 PF04484.15         33     2.68    301   0.59   0.61   0.47   0.04\n",
            "15366  Q_salvage            PF10343.12        111     1.60    288   0.59   0.56   0.51   0.02\n",
            "15367  R-HINP1I             PF11463.11          3     0.47    205   0.59   0.58   0.51   0.03\n",
            "15368  R2K_2                PF18299.4          42     2.90    146   0.59   0.57   0.51   0.02\n",
            "15369  R2K_3                PF14243.9          47     5.63    201   0.59   0.58   0.43   0.02\n",
            "15370  R3H                  PF01424.25        635    17.44     60   0.93   0.91   0.86   0.05\n",
            "15371  R3H-assoc            PF13902.9          48     5.42    128   0.59   0.56   0.47   0.08\n",
            "15372  RA                   PF00788.26         39     8.29     93   0.61   0.60   0.53   0.03\n",
            "15373  Rab15_effector       PF15208.9           9     0.65    240   0.59   0.58   0.52   0.01\n",
            "15374  Rab3-GAP_cat_C       PF19533.2           9     0.70    204   0.59   0.61   0.52   0.02\n",
            "15375  Rab3-GTPase_cat      PF13890.9          43     1.62    164   0.59   0.55   0.51   0.04\n",
            "15376  RAB3GAP2_C           PF14656.9          22     1.85    594   0.59   0.58   0.52   0.03\n",
            "15377  RAB3GAP2_N           PF14655.9          35     2.59    417   0.59   0.58   0.49   0.02\n",
            "15378  Rab5-bind            PF09311.14          4     0.79    307   0.59   0.61   0.54   0.08\n",
            "15379  Rabaptin             PF03528.18          3     0.52    486   0.59   0.60   0.52   0.04\n",
            "15380  RabGAP-TBC           PF00566.21         69     6.38    216   0.59   0.54   0.49   0.04\n",
            "15381  RabGGT_insert        PF07711.14          9     0.92    101   0.59   0.54   0.53   0.03\n",
            "15382  Rab_bind             PF16704.8           3     0.78     66   0.85   0.88   0.80   0.08\n",
            "15383  Rab_eff_C            PF04698.15          7     0.67    717   0.59   0.60   0.52   0.03\n",
            "15384  Rac1                 PF09632.13          2     0.62    297   0.59   0.61   0.53   0.03\n",
            "15385  RACo_C_ter           PF14574.9         131     1.83    260   0.59   0.61   0.53   0.02\n",
            "15386  RACo_linker          PF17650.4         191     5.63     85   0.67   0.69   0.58   0.03\n",
            "15387  Raco_middle          PF17651.4         101     1.77    163   0.59   0.60   0.55   0.02\n",
            "15388  RAC_head             PF16717.8          71     2.42     98   0.59   0.62   0.53   0.10\n",
            "15389  Rad1                 PF02144.19         23     2.51    275   0.59   0.59   0.49   0.03\n",
            "15390  Rad10                PF03834.17        110     2.19    116   0.59   0.57   0.52   0.04\n",
            "15391  Rad17                PF03215.18          3     0.83    186   0.59   0.58   0.51   0.02\n",
            "15392  Rad21_Rec8           PF04824.19         14     5.19     55   1.01   1.05   0.99   0.04\n",
            "15393  Rad21_Rec8_N         PF04825.16        105     2.18    111   0.59   0.58   0.49   0.03\n",
            "15394  Rad33                PF08730.13         12     0.88    165   0.59   0.58   0.52   0.04\n",
            "15395  Rad4                 PF03835.18         25     3.79    147   0.59   0.55   0.48   0.05\n",
            "15396  Rad50_zn_hook        PF04423.17         52    18.65     52   1.07   1.04   1.03   0.13\n",
            "15397  Rad51                PF08423.14         21     0.89    255   0.59   0.60   0.52   0.02\n",
            "15398  RAD51_interact       PF15696.8          18     3.73     39   1.40   1.47   1.38   0.16\n",
            "15399  Rad52_Rad22          PF04098.18         34     2.62    153   0.59   0.58   0.51   0.03\n",
            "15400  Rad54_N              PF08658.13         81     1.96    183   0.59   0.59   0.46   0.05\n",
            "15401  Rad60-SLD            PF11976.11         32     5.58     72   0.78   0.80   0.76   0.06\n",
            "15402  Rad60-SLD_2          PF13881.9          20     1.56    111   0.59   0.57   0.50   0.03\n",
            "15403  Rad9                 PF04139.16          8     1.39    256   0.59   0.58   0.51   0.03\n",
            "15404  Rad9_Rad53_bind      PF08605.13          7     1.13    129   0.59   0.56   0.49   0.02\n",
            "15405  RadC                 PF04002.18        790     3.81    122   0.59   0.60   0.50   0.03\n",
            "15406  Radial_spoke         PF04712.15         37     1.85    495   0.59   0.57   0.49   0.02\n",
            "15407  Radial_spoke_3       PF06098.14         43     2.17    286   0.59   0.59   0.54   0.05\n",
            "15408  Radical_SAM          PF04055.24        491    22.44    167   0.59   0.56   0.45   0.01\n",
            "15409  Radical_SAM_2        PF19238.2          97     0.99    151   0.59   0.57   0.54   0.03\n",
            "15410  Radical_SAM_C        PF16199.8         501     5.74     82   0.69   0.68   0.61   0.03\n",
            "15411  Radical_SAM_N        PF08497.13        272     1.08    298   0.59   0.58   0.51   0.01\n",
            "15412  Radical_SAM_N2       PF19864.2         711     1.75    151   0.59   0.59   0.53   0.01\n",
            "15413  Raf1_HTH             PF18579.4          67     3.41     61   0.92   0.90   0.90   0.05\n",
            "15414  Raf1_N               PF18578.4          70     1.37    106   0.59   0.61   0.54   0.04\n",
            "15415  Raffinose_syn        PF05691.15          4     0.64    750   0.59   0.57   0.51   0.01\n",
            "15416  Raftlin              PF15250.9          14     0.88    459   0.59   0.58   0.52   0.01\n",
            "15417  RAG1                 PF12940.10          2     0.41    651   0.59   0.58   0.51   0.01\n",
            "15418  RAG1_imp_bd          PF12560.11         17     0.98    286   0.59   0.53   0.52   0.04\n",
            "15419  RAG2                 PF03089.17         11     0.59    339   0.59   0.57   0.52   0.01\n",
            "15420  RAG2_PHD             PF13341.9           8     0.57     78   0.72   0.61   0.66   0.04\n",
            "15421  Ragweed_pollen       PF03913.16          3     1.09     44   1.25   1.03   1.18   0.25\n",
            "15422  RAI1                 PF08652.14        117     3.07     68   0.83   0.78   0.71   0.03\n",
            "15423  RAI16-like           PF10257.12         20     2.31    360   0.59   0.61   0.50   0.03\n",
            "15424  Ral                  PF11058.11          2     0.64     66   0.85   0.78   0.79   0.06\n",
            "15425  RALF                 PF05498.14         52     3.34     65   0.86   0.76   0.76   0.10\n",
            "15426  RalF_SCD             PF18248.4           3     0.55    147   0.59   0.60   0.52   0.02\n",
            "15427  RALGAPB_N            PF20412.1          42     3.06    126   0.59   0.54   0.49   0.06\n",
            "15428  RAM                  PF15320.9          26     2.69     86   0.66   0.58   0.53   0.11\n",
            "15429  RAMA                 PF18755.4          84     5.47    108   0.59   0.60   0.50   0.03\n",
            "15430  RAMP                 PF04901.16         48     1.16    110   0.59   0.49   0.54   0.04\n",
            "15431  RAMP4                PF06624.15         56     4.21     60   0.93   0.96   0.87   0.05\n",
            "15432  RAMPs                PF03787.18        147    25.43    213   0.59   0.60   0.39   0.02\n",
            "15433  RamS                 PF19402.2          70     5.05     44   1.25   1.24   1.10   0.18\n",
            "15434  Ran-binding          PF05508.14         77     1.46    310   0.59   0.61   0.50   0.03\n",
            "15435  RanGAP1_C            PF07834.14         13     0.97    178   0.59   0.61   0.53   0.03\n",
            "15436  RANK_CRD_2           PF18278.4           5     1.15     41   1.33   1.13   1.29   0.18\n",
            "15437  Ran_BP1              PF00638.21          8     1.06    122   0.59   0.59   0.54   0.04\n",
            "15438  RAP                  PF08373.13         63    11.53     58   0.96   0.86   0.86   0.08\n",
            "15439  RAP-1                PF03085.18          6     1.35    241   0.59   0.55   0.53   0.05\n",
            "15440  RAP1                 PF07218.14          2     0.55    782   0.59   0.58   0.52   0.03\n",
            "15441  Rap1-DNA-bind        PF09197.13         14     1.83    108   0.59   0.54   0.52   0.05\n",
            "15442  Rap1a                PF18602.4         130     7.41     92   0.62   0.57   0.49   0.04\n",
            "15443  Rap1_C               PF11626.11         74     4.74     84   0.68   0.65   0.56   0.03\n",
            "15444  RAP80_UIM            PF18282.4           2     0.91     57   0.98   1.02   0.93   0.13\n",
            "15445  RapA_C               PF12137.11         62     2.33    360   0.59   0.60   0.55   0.03\n",
            "15446  RapH_N               PF18801.4          47     3.49     62   0.90   0.86   0.88   0.08\n",
            "15447  Rapsyn_N             PF10579.12          2     0.71     80   0.71   0.68   0.65   0.02\n",
            "15448  Raptor_N             PF14538.9          54     3.24    151   0.59   0.54   0.50   0.02\n",
            "15449  Rap_GAP              PF02145.18        212     3.13    186   0.59   0.56   0.46   0.03\n",
            "15450  Ras                  PF00071.25         60     1.46    162   0.59   0.59   0.52   0.03\n",
            "15451  RasGAP               PF00616.22         50     5.17    208   0.59   0.58   0.50   0.04\n",
            "15452  RasGAP_C             PF03836.18        110     4.41    136   0.59   0.59   0.52   0.07\n",
            "15453  RasGEF               PF00617.22        619     6.17    179   0.59   0.56   0.40   0.04\n",
            "15454  RasGEF_N             PF00618.23         52     5.43    105   0.59   0.57   0.44   0.04\n",
            "15455  RasGEF_N_2           PF14663.9          72     3.56    108   0.59   0.56   0.52   0.04\n",
            "15456  Ras_bdg_2            PF14847.9          42     1.98    100   0.59   0.58   0.50   0.04\n",
            "15457  Rav1p_C              PF12234.11         61     1.54    640   0.59   0.57   0.52   0.03\n",
            "15458  RAWUL                PF16207.8          56     2.87     73   0.77   0.73   0.64   0.05\n",
            "15459  Rax2                 PF12768.10         69     3.73    148   0.59   0.58   0.52   0.03\n",
            "15460  RBB1NT               PF08169.14         28     1.32     94   0.61   0.60   0.54   0.03\n",
            "15461  RbcS                 PF12338.11         10     1.67     45   1.22   1.28   1.15   0.12\n",
            "15462  RbcX                 PF02341.18         29     1.01    111   0.59   0.59   0.53   0.04\n",
            "15463  RBD                  PF02196.18         54     6.70     69   0.81   0.82   0.78   0.04\n",
            "15464  RBD-FIP              PF09457.13         10     5.09     41   1.34   1.39   1.28   0.13\n",
            "15465  RBDV_coat            PF06593.14          1     0.39    274   0.58   0.59   0.51   0.02\n",
            "15466  RBFA                 PF02033.21        709     4.73    105   0.59   0.61   0.47   0.04\n",
            "15467  RBM1CTR              PF08081.14          9     1.92     45   1.22   1.17   1.17   0.23\n",
            "15468  RBM39linker          PF15519.9          86     3.54     86   0.66   0.67   0.57   0.04\n",
            "15469  RbpA                 PF13397.9          49     1.46    104   0.59   0.58   0.53   0.03\n",
            "15470  RBP_receptor         PF14752.9          31     1.54    608   0.59   0.57   0.52   0.03\n",
            "15471  RBR                  PF17208.6          40     8.09     57   0.98   1.01   0.87   0.22\n",
            "15472  RbsD_FucU            PF05025.16        246     2.91    134   0.59   0.61   0.50   0.02\n",
            "15473  Rbsn                 PF11464.11         91    12.06     42   1.31   1.36   1.30   0.15\n",
            "15474  Rbx_binding          PF18113.4          41     4.62     71   0.79   0.79   0.76   0.02\n",
            "15475  RB_A                 PF01858.20         62     2.67    197   0.59   0.57   0.53   0.04\n",
            "15476  RB_B                 PF01857.23         74     1.65    132   0.59   0.55   0.50   0.05\n",
            "15477  Rb_C                 PF08934.13          5     0.56    152   0.59   0.59   0.52   0.02\n",
            "15478  RC-P840_PscD         PF10657.12          4     0.46    144   0.59   0.57   0.52   0.01\n",
            "15479  RCC1                 PF00415.21        101    14.75     50   1.11   1.07   0.96   0.05\n",
            "15480  RCC1_2               PF13540.9         175   175.00     30   1.64   1.57   1.57   0.20\n",
            "15481  RCC_reductase        PF06405.14         11     0.86    253   0.59   0.60   0.53   0.02\n",
            "15482  Rcd1                 PF04078.16         52     2.12    259   0.59   0.60   0.54   0.04\n",
            "15483  RCDG1                PF15725.8           3     0.65     83   0.68   0.74   0.60   0.05\n",
            "15484  Rce1-like            PF02517.19        683     6.70     92   0.62   0.61   0.45   0.09\n",
            "15485  RcnB                 PF11776.11         66     8.83     51   1.09   1.01   1.06   0.10\n",
            "15486  RcpB                 PF16971.8           2     0.64    167   0.59   0.58   0.51   0.03\n",
            "15487  RcpC                 PF16976.8         129     4.27    116   0.59   0.66   0.52   0.04\n",
            "15488  RCR                  PF12273.11         49     1.98    119   0.59   0.53   0.38   0.04\n",
            "15489  RCS1                 PF07326.14         14     1.21    232   0.59   0.59   0.53   0.04\n",
            "15490  RcsC                 PF09456.13         13     1.63     91   0.63   0.62   0.58   0.02\n",
            "15491  RCSD                 PF05177.15          8     1.27     91   0.63   0.65   0.51   0.20\n",
            "15492  RcsD_ABL             PF16359.8          11     1.08    103   0.59   0.62   0.54   0.05\n",
            "15493  RcsF                 PF16358.8           8     0.79    110   0.59   0.58   0.52   0.03\n",
            "15494  RD3                  PF14473.9          29     2.21    129   0.59   0.56   0.53   0.05\n",
            "15495  RDD                  PF06271.15         49     7.32    138   0.59   0.60   0.50   0.07\n",
            "15496  RdDM_RDM1            PF09187.13         11     0.63    119   0.59   0.54   0.53   0.03\n",
            "15497  RdgC                 PF04381.15         96     2.16    298   0.59   0.60   0.55   0.03\n",
            "15498  RDM                  PF11002.11         16     2.63     42   1.30   1.33   1.28   0.11\n",
            "15499  RdRP                 PF05183.15        256     4.31    586   0.59   0.57   0.41   0.01\n",
            "15500  RdRP_1               PF00680.23         19     5.54    462   0.59   0.56   0.49   0.01\n",
            "15501  RdRP_2               PF00978.24         36     3.05    440   0.59   0.57   0.52   0.02\n",
            "15502  RdRP_3               PF00998.26         32     2.81    486   0.59   0.56   0.53   0.02\n",
            "15503  RdRP_4               PF02123.19         28     5.98    479   0.59   0.56   0.45   0.02\n",
            "15504  RdRP_5               PF07925.14          3     0.56   1271   0.59   0.58   0.52   0.01\n",
            "15505  RDV-p3               PF09231.13          2     0.53    965   0.59   0.59   0.53   0.02\n",
            "15506  Rdx                  PF10262.12        227     4.30     74   0.76   0.71   0.64   0.03\n",
            "15507  Read-through         PF16814.8           1     0.36    182   0.59   0.57   0.52   0.01\n",
            "15508  RebB                 PF11747.11         30     2.84     68   0.83   0.86   0.80   0.12\n",
            "15509  REC1                 PF18501.4          10     1.62    248   0.59   0.58   0.51   0.05\n",
            "15510  REC104               PF17378.5           1     0.41    182   0.59   0.59   0.52   0.02\n",
            "15511  REC114-like          PF15165.9           7     0.75    240   0.59   0.58   0.52   0.01\n",
            "15512  RecA                 PF00154.24         14     0.58    262   0.59   0.62   0.52   0.02\n",
            "15513  RecA_dep_nuc         PF16786.8           3     0.90    103   0.59   0.55   0.50   0.02\n",
            "15514  RecC_C               PF17946.4         130     6.53    202   0.59   0.61   0.46   0.03\n",
            "15515  Receptor_2B4         PF11465.11          3     0.62    108   0.59   0.57   0.52   0.02\n",
            "15516  Receptor_IA-2        PF11548.11         24     1.79     89   0.64   0.67   0.60   0.03\n",
            "15517  Recep_L_domain       PF01030.27         50     3.32    110   0.59   0.58   0.49   0.04\n",
            "15518  RecG_dom3_C          PF19833.2           8     0.59     87   0.65   0.68   0.59   0.02\n",
            "15519  RecG_N               PF17190.7           7     1.45     89   0.64   0.66   0.59   0.07\n",
            "15520  RecG_wedge           PF17191.7           6     0.73    162   0.59   0.59   0.53   0.03\n",
            "15521  RecJ_OB              PF17768.4         523     4.86    108   0.59   0.57   0.52   0.01\n",
            "15522  Recombinase          PF07508.16         66     6.69    103   0.59   0.56   0.50   0.04\n",
            "15523  RecO_C               PF02565.18         84    11.79    157   0.59   0.60   0.53   0.02\n",
            "15524  RecO_N               PF11967.11         87     6.43     80   0.71   0.73   0.64   0.02\n",
            "15525  RecO_N_2             PF13114.9          11     1.79     71   0.79   0.78   0.75   0.03\n",
            "15526  RecQ5                PF06959.14         12     0.86    203   0.59   0.59   0.52   0.05\n",
            "15527  RecQ_Zn_bind         PF16124.8         689     4.67     66   0.85   0.72   0.66   0.13\n",
            "15528  RecR                 PF02132.18        748    10.77     40   1.37   1.22   1.34   0.24\n",
            "15529  RecT                 PF03837.17         72     8.25    199   0.59   0.56   0.47   0.03\n",
            "15530  RecU                 PF03838.17        144     1.61    162   0.59   0.56   0.53   0.04\n",
            "15531  RecX                 PF02631.19         88     4.43    122   0.59   0.60   0.50   0.05\n",
            "15532  Red1                 PF07964.14         10     3.33    754   0.59   0.58   0.52   0.05\n",
            "15533  Redoxin              PF08534.13         52     7.16    147   0.59   0.59   0.46   0.01\n",
            "15534  Reductase_C          PF14759.9          65     2.91     85   0.67   0.64   0.62   0.03\n",
            "15535  RED_C                PF07807.14         13     0.71    112   0.59   0.55   0.52   0.06\n",
            "15536  RED_N                PF07808.16         25     1.25    235   0.59   0.59   0.51   0.06\n",
            "15537  Reeler               PF02014.19        129     3.08    129   0.59   0.52   0.40   0.04\n",
            "15538  REF                  PF05755.15         29     1.19    206   0.59   0.60   0.54   0.03\n",
            "15539  REGB_T4              PF10715.12          1     0.38    150   0.59   0.58   0.51   0.01\n",
            "15540  Regnase_1_C          PF18561.4           8     1.61     46   1.20   1.17   1.16   0.06\n",
            "15541  Regulator_TrmB       PF11495.11         22     4.11    233   0.59   0.60   0.53   0.02\n",
            "15542  Reg_prop             PF07494.14        659   659.00     24   1.05   0.99   0.98   0.09\n",
            "15543  REJ                  PF02010.18         23     4.30    451   0.59   0.57   0.49   0.02\n",
            "15544  Relaxase             PF03432.17         17     4.51    242   0.59   0.56   0.49   0.05\n",
            "15545  RelA_AH_RIS          PF19296.2          82     1.25    185   0.59   0.61   0.51   0.02\n",
            "15546  RelA_SpoT            PF04607.20         89     2.15    117   0.59   0.56   0.49   0.06\n",
            "15547  RelB                 PF04221.15          5     1.67     83   0.68   0.73   0.63   0.03\n",
            "15548  RelB_leu_zip         PF16180.8           5     0.93     84   0.68   0.74   0.59   0.03\n",
            "15549  RelB_N               PF18506.4          25     5.35     47   1.17   1.24   1.16   0.13\n",
            "15550  RelB_transactiv      PF16181.8           2     0.50    178   0.59   0.61   0.51   0.02\n",
            "15551  RelE                 PF06296.15         19     1.28    120   0.59   0.61   0.54   0.04\n",
            "15552  RELT                 PF12606.11         13     1.93     42   1.30   1.28   1.27   0.12\n",
            "15553  Remorin_C            PF03763.16         94     3.16    107   0.59   0.57   0.56   0.15\n",
            "15554  Remorin_N            PF03766.16         21     3.18     54   1.03   1.12   0.92   0.13\n",
            "15555  Renin_r              PF07850.17         28     1.24    100   0.59   0.57   0.50   0.04\n",
            "15556  Reoviridae_Vp9       PF08978.13          2     0.58    280   0.59   0.60   0.52   0.02\n",
            "15557  Reovirus_cap         PF00979.20          5     0.95    367   0.59   0.55   0.52   0.01\n",
            "15558  Reovirus_L2          PF06016.14          3     0.69   1297   0.59   0.59   0.53   0.01\n",
            "15559  Reovirus_M2          PF05993.15          3     0.71    647   0.59   0.61   0.52   0.02\n",
            "15560  Reovirus_Mu2         PF07781.14          3     0.73    727   0.59   0.59   0.51   0.01\n",
            "15561  Reo_P9               PF06043.14          1     0.39    334   0.59   0.59   0.52   0.02\n",
            "15562  Reo_sigma1           PF01664.19          3     0.69    219   0.59   0.58   0.53   0.01\n",
            "15563  Reo_sigmaC           PF04582.15          4     0.50    130   0.59   0.58   0.52   0.02\n",
            "15564  Reo_sigmaC_M         PF17750.4           4     1.57     40   1.37   1.38   1.33   0.11\n",
            "15565  Rep-A_N              PF04057.15         11     1.23    101   0.59   0.62   0.50   0.05\n",
            "15566  RepA1_leader         PF08048.15          5     3.20     25   2.13   2.04   2.07   0.26\n",
            "15567  RepA_C               PF04796.15          3     0.76    161   0.59   0.58   0.52   0.01\n",
            "15568  RepA_N               PF06970.14         14     1.84     76   0.74   0.72   0.71   0.05\n",
            "15569  REPA_OB_2            PF16900.8         102     1.50     98   0.59   0.60   0.53   0.04\n",
            "15570  RepB                 PF07506.14         15     4.64    185   0.59   0.62   0.51   0.03\n",
            "15571  RepB-RCR_reg         PF10723.12          3     0.75     81   0.70   0.70   0.63   0.05\n",
            "15572  RepB_primase         PF16793.8          12     3.76    240   0.59   0.59   0.47   0.02\n",
            "15573  RepC                 PF06504.14          4     0.57    273   0.59   0.60   0.52   0.01\n",
            "15574  RepL                 PF05732.14          3     0.74    164   0.59   0.60   0.52   0.03\n",
            "15575  Replicase            PF03090.20         11     1.47    131   0.59   0.56   0.52   0.02\n",
            "15576  Replic_Relax         PF13814.9          27     4.29    193   0.59   0.58   0.44   0.03\n",
            "15577  Repressor_Mnt        PF11423.11          2     2.00     28   1.84   1.85   1.81   0.16\n",
            "15578  Reprolysin           PF01421.22         17     1.30    200   0.59   0.54   0.53   0.03\n",
            "15579  Reprolysin_2         PF13574.9           6     1.60    194   0.59   0.57   0.45   0.02\n",
            "15580  Reprolysin_3         PF13582.9          23     3.38    124   0.59   0.56   0.46   0.04\n",
            "15581  Reprolysin_4         PF13583.9           4     1.49    203   0.59   0.57   0.49   0.03\n",
            "15582  Reprolysin_5         PF13688.9          47     5.13    196   0.59   0.54   0.42   0.04\n",
            "15583  RepSA                PF20199.1          98     2.65    437   0.59   0.55   0.48   0.03\n",
            "15584  Rep_1                PF01446.20         16     1.68    251   0.59   0.54   0.50   0.04\n",
            "15585  Rep_1B               PF17873.4           1     0.70     53   1.05   1.09   0.99   0.06\n",
            "15586  Rep_2                PF01719.20          7     1.53    188   0.59   0.54   0.48   0.05\n",
            "15587  Rep_3                PF01051.24        159    10.39    222   0.59   0.58   0.44   0.04\n",
            "15588  Rep_4                PF05797.14          2     0.66    344   0.59   0.58   0.50   0.02\n",
            "15589  Rep_fac-A_3          PF08661.14         18     3.08    105   0.59   0.59   0.51   0.03\n",
            "15590  Rep_fac-A_C          PF08646.13         29     1.50    146   0.59   0.52   0.52   0.05\n",
            "15591  Rep_fac_C            PF08542.14        316     8.21     88   0.65   0.67   0.55   0.03\n",
            "15592  Rep_N                PF08724.13          9     1.26    186   0.59   0.55   0.51   0.02\n",
            "15593  Rep_Org_C            PF06926.14          3     0.64     95   0.60   0.58   0.54   0.05\n",
            "15594  Rep_trans            PF02486.22         34     3.47    203   0.59   0.56   0.51   0.03\n",
            "15595  Rer1                 PF03248.16        117     1.12    171   0.59   0.53   0.52   0.05\n",
            "15596  RES                  PF08808.14        180    14.48    162   0.59   0.57   0.38   0.02\n",
            "15597  ResB                 PF05140.17        184     4.39    496   0.59   0.59   0.44   0.01\n",
            "15598  ResIII               PF04851.18         52     4.26    171   0.59   0.58   0.44   0.03\n",
            "15599  Resistin             PF06954.14         28     0.88     88   0.65   0.52   0.58   0.13\n",
            "15600  Resolvase            PF00239.24        112     6.33    146   0.59   0.62   0.50   0.04\n",
            "15601  RESP18               PF14948.9          10     0.92    103   0.59   0.57   0.52   0.03\n",
            "15602  Response_reg         PF00072.27         52     3.85    112   0.59   0.61   0.54   0.03\n",
            "15603  Response_reg_2       PF19192.3          42     5.44    181   0.59   0.60   0.47   0.04\n",
            "15604  RestrictionMunI      PF11407.11          2     0.49    202   0.59   0.55   0.52   0.01\n",
            "15605  RestrictionSfiI      PF11487.11          3     0.79    215   0.59   0.58   0.50   0.02\n",
            "15606  Ret2_MD              PF18528.4           2     0.42     93   0.61   0.59   0.54   0.02\n",
            "15607  RETICULATA-like      PF11891.11         40     1.60    177   0.59   0.59   0.53   0.02\n",
            "15608  Reticulon            PF02453.20        212     5.00    157   0.59   0.59   0.54   0.07\n",
            "15609  Retinal              PF15449.9          17     1.15   1290   0.59   0.59   0.53   0.04\n",
            "15610  Retinin_C            PF04527.15         14     1.45     63   0.89   0.91   0.79   0.15\n",
            "15611  Retrotrans_gag       PF03732.20         55     5.60     96   0.60   0.56   0.51   0.04\n",
            "15612  Retrotran_gag_2      PF14223.9          41     4.12    138   0.59   0.59   0.56   0.04\n",
            "15613  Retrotran_gag_3      PF14244.9          14     5.62     48   1.15   1.13   1.12   0.03\n",
            "15614  Retrotran_gag_4      PF17241.5          20     3.93    154   0.59   0.55   0.53   0.04\n",
            "15615  Retro_M              PF02813.17          1     0.45     86   0.67   0.70   0.60   0.02\n",
            "15616  RET_CLD1             PF17756.4          18     1.02    124   0.59   0.56   0.52   0.02\n",
            "15617  RET_CLD3             PF17812.4          13     0.83    116   0.59   0.58   0.52   0.03\n",
            "15618  RET_CLD4             PF17813.4          23     1.01    105   0.59   0.54   0.52   0.05\n",
            "15619  REV                  PF00424.21          2     0.65     94   0.61   0.62   0.53   0.04\n",
            "15620  REV1_C               PF16727.8         184     4.41     92   0.62   0.58   0.55   0.05\n",
            "15621  RexA                 PF15969.8           4     1.06    236   0.59   0.60   0.53   0.03\n",
            "15622  RexB                 PF15968.8           1     0.38    139   0.59   0.61   0.52   0.06\n",
            "15623  RE_AccI              PF09545.13          3     0.48    366   0.59   0.59   0.52   0.02\n",
            "15624  RE_Alw26IDE          PF09665.13          3     0.70    507   0.59   0.56   0.52   0.02\n",
            "15625  RE_AlwI              PF09491.13          5     1.54    451   0.59   0.59   0.50   0.03\n",
            "15626  RE_ApaLI             PF09499.13          3     0.50    189   0.59   0.55   0.52   0.03\n",
            "15627  RE_AspBHI_N          PF18062.4          12     1.47    185   0.59   0.55   0.51   0.02\n",
            "15628  RE_Bpu10I            PF09549.13          7     0.91    220   0.59   0.58   0.52   0.02\n",
            "15629  RE_BsaWI             PF18643.4          15     2.79    105   0.59   0.57   0.49   0.04\n",
            "15630  RE_Bsp6I             PF09504.13          1     0.36    179   0.60   0.56   0.52   0.02\n",
            "15631  RE_BstXI             PF09552.13          2     0.49    288   0.59   0.58   0.52   0.01\n",
            "15632  RE_CfrBI             PF09516.13          2     0.64    250   0.59   0.61   0.51   0.04\n",
            "15633  RE_Eco29kI           PF09517.13         14     1.02    164   0.59   0.56   0.52   0.01\n",
            "15634  RE_Eco47II           PF09553.13         14     1.04    202   0.59   0.56   0.53   0.04\n",
            "15635  RE_EcoO109I          PF14511.9          13     1.05    198   0.59   0.58   0.52   0.02\n",
            "15636  RE_endonuc           PF19778.2          27     1.49    109   0.59   0.57   0.50   0.03\n",
            "15637  RE_HaeII             PF09554.13          3     0.57    338   0.59   0.59   0.52   0.02\n",
            "15638  RE_HaeIII            PF09556.13          7     0.64    298   0.59   0.57   0.53   0.02\n",
            "15639  RE_HindIII           PF09518.13          4     0.96    285   0.59   0.59   0.52   0.03\n",
            "15640  RE_HindVP            PF09519.13         11     0.97    332   0.59   0.56   0.53   0.02\n",
            "15641  RE_HpaII             PF09561.13         20     1.61    357   0.59   0.58   0.53   0.03\n",
            "15642  RE_LlaJI             PF09563.13          3     0.77    365   0.59   0.56   0.52   0.03\n",
            "15643  RE_LlaMI             PF09562.13          3     0.58    242   0.59   0.53   0.51   0.03\n",
            "15644  RE_MamI              PF09567.13          3     0.81    183   0.59   0.57   0.50   0.02\n",
            "15645  RE_MjaI              PF09568.13         11     1.15    170   0.59   0.57   0.52   0.04\n",
            "15646  RE_NgoBV             PF09564.13          2     0.41    238   0.59   0.56   0.52   0.02\n",
            "15647  RE_NgoFVII           PF09565.13         13     1.95    287   0.59   0.57   0.49   0.02\n",
            "15648  RE_NgoPII            PF09521.13          8     0.72    262   0.59   0.58   0.52   0.02\n",
            "15649  RE_R_Pab1            PF09522.13          2     0.57    119   0.59   0.59   0.52   0.04\n",
            "15650  RE_SacI              PF09566.13          6     1.40    273   0.59   0.60   0.48   0.02\n",
            "15651  RE_ScaI              PF09569.13          3     0.61    192   0.59   0.57   0.52   0.02\n",
            "15652  RE_SinI              PF09570.13          2     0.54    217   0.59   0.56   0.52   0.01\n",
            "15653  RE_TaqI              PF09573.13          3     0.56    229   0.59   0.55   0.51   0.03\n",
            "15654  RE_TdeIII            PF09520.13         17     1.93    243   0.59   0.57   0.50   0.03\n",
            "15655  RE_XamI              PF09572.13          6     0.92    254   0.59   0.60   0.50   0.01\n",
            "15656  RE_XcyI              PF09571.13          3     0.69    305   0.59   0.62   0.52   0.02\n",
            "15657  RF-1                 PF00472.23        797     2.98    116   0.59   0.59   0.46   0.07\n",
            "15658  RF3_C                PF16658.8         267     1.63    131   0.59   0.59   0.52   0.02\n",
            "15659  RFamide_26RFa        PF11109.11         14     1.09    131   0.59   0.60   0.54   0.02\n",
            "15660  RFC1                 PF08519.15        138     3.27    157   0.59   0.57   0.49   0.03\n",
            "15661  Rft-1                PF04506.16         31     1.99    514   0.59   0.59   0.52   0.05\n",
            "15662  RFX1_trans_act       PF04589.16         15     0.99    161   0.59   0.59   0.49   0.07\n",
            "15663  RFX5_DNA_bdg         PF14621.9           9     0.82    219   0.59   0.62   0.53   0.06\n",
            "15664  RFX5_N               PF18326.4           3     0.99     59   0.95   0.98   0.90   0.05\n",
            "15665  RFXA_RFXANK_bdg      PF15289.9           4     0.67    122   0.59   0.55   0.51   0.05\n",
            "15666  RFX_DNA_binding      PF02257.18         65     2.26     79   0.72   0.66   0.67   0.04\n",
            "15667  RGCC                 PF15151.9           8     0.69    128   0.59   0.61   0.51   0.03\n",
            "15668  RGI1                 PF10843.11         10     0.55    194   0.59   0.55   0.52   0.03\n",
            "15669  RGI_lyase            PF18370.4          73     1.61     86   0.66   0.64   0.60   0.03\n",
            "15670  RGM_C                PF06534.16         32     1.22    173   0.59   0.56   0.51   0.03\n",
            "15671  RGM_N                PF06535.15         26     1.22    166   0.59   0.51   0.49   0.05\n",
            "15672  RGP                  PF03214.16         11     0.63    342   0.59   0.57   0.52   0.01\n",
            "15673  Rgp1                 PF08737.13         70     3.11    414   0.59   0.60   0.46   0.02\n",
            "15674  RgpF                 PF05045.15         13     1.24    503   0.59   0.55   0.52   0.02\n",
            "15675  RGS                  PF00615.22         54     3.48    118   0.59   0.55   0.51   0.05\n",
            "15676  RGS-like             PF09128.14          6     1.04    191   0.59   0.59   0.53   0.03\n",
            "15677  RGS12_us1            PF16613.8           6     0.64    118   0.59   0.60   0.52   0.06\n",
            "15678  RGS12_us2            PF16611.8           6     1.37     72   0.78   0.82   0.72   0.13\n",
            "15679  RGS12_usC            PF16612.8          11     1.14    146   0.59   0.60   0.52   0.06\n",
            "15680  RGS_DHEX             PF18148.4           4     0.63    100   0.59   0.53   0.52   0.04\n",
            "15681  Rh5                  PF18515.4           8     2.10    255   0.59   0.55   0.56   0.08\n",
            "15682  RhaA                 PF06134.14          5     0.51    417   0.59   0.58   0.53   0.01\n",
            "15683  Rhabdo_glycop        PF00974.21          4     1.00    502   0.59   0.55   0.51   0.01\n",
            "15684  Rhabdo_M1            PF03342.16          1     0.41    227   0.60   0.61   0.52   0.02\n",
            "15685  Rhabdo_M2            PF04785.15          2     0.39    202   0.59   0.57   0.52   0.01\n",
            "15686  Rhabdo_matrix        PF03397.17          1     0.40    168   0.59   0.59   0.52   0.01\n",
            "15687  Rhabdo_ncap          PF00945.21          3     0.94    410   0.59   0.57   0.53   0.01\n",
            "15688  Rhabdo_ncap_2        PF03216.16         13     4.47    295   0.59   0.58   0.50   0.02\n",
            "15689  Rhabdo_NV            PF02484.18          1     0.37    111   0.59   0.57   0.51   0.02\n",
            "15690  rhaM                 PF05336.16        245     2.79    108   0.59   0.51   0.49   0.05\n",
            "15691  Rhamnogal_lyase      PF06045.14          4     0.61    213   0.59   0.57   0.51   0.02\n",
            "15692  Rhamno_transf        PF11316.11         10     1.72    235   0.59   0.57   0.54   0.02\n",
            "15693  RhaT                 PF06379.15          2     0.38    344   0.59   0.58   0.51   0.04\n",
            "15694  RHD3_GTPase          PF05879.15          8     0.66    245   0.59   0.59   0.52   0.02\n",
            "15695  RHD_dimer            PF16179.8          48     1.71    103   0.59   0.55   0.53   0.02\n",
            "15696  RHD_DNA_bind         PF00554.25         21     1.56    169   0.59   0.57   0.51   0.03\n",
            "15697  RhgB_N               PF09284.13         52     1.34    251   0.59   0.57   0.51   0.02\n",
            "15698  RHH_1                PF01402.24         30    30.00     39   1.15   1.26   1.14   0.14\n",
            "15699  RHH_3                PF12651.10         15     9.20     44   1.25   1.32   1.24   0.11\n",
            "15700  RHH_4                PF13467.9         117     3.57     65   0.86   0.88   0.79   0.05\n",
            "15701  RHH_5                PF07878.14         13     7.65     43   1.28   1.35   1.27   0.06\n",
            "15702  RHH_6                PF16762.8           9     1.08     77   0.73   0.77   0.67   0.04\n",
            "15703  RHH_7                PF16777.8           2     0.86     73   0.77   0.79   0.71   0.05\n",
            "15704  RHH_8                PF17723.4           7     0.78    119   0.59   0.64   0.54   0.04\n",
            "15705  RHH_9                PF19839.2           3     0.75     65   0.86   0.86   0.80   0.04\n",
            "15706  RHIM                 PF12721.10         28    28.00     22   1.95   1.82   1.95   0.36\n",
            "15707  RHINO                PF15319.9          19     1.29    248   0.59   0.56   0.50   0.03\n",
            "15708  RhlB                 PF12300.11          2     0.52    175   0.59   0.64   0.51   0.06\n",
            "15709  Rhodanese            PF00581.23        141    11.73    107   0.59   0.57   0.41   0.01\n",
            "15710  Rhodanese_C          PF12368.11        145     2.64     67   0.84   0.66   0.70   0.18\n",
            "15711  RhodobacterPufX      PF11511.11          5     1.50     67   0.84   0.84   0.79   0.03\n",
            "15712  Rhodopsin_N          PF10413.12          5     1.23     36   1.51   1.40   1.47   0.14\n",
            "15713  RhoGAP               PF00620.30         83     3.87    149   0.59   0.59   0.52   0.03\n",
            "15714  RhoGAP-FF1           PF16512.8          23     1.69     80   0.71   0.67   0.67   0.07\n",
            "15715  RhoGAP_pG1_pG2       PF19518.2          22     1.04    700   0.59   0.59   0.50   0.02\n",
            "15716  RhoGEF               PF00621.23        157     5.09    183   0.59   0.58   0.45   0.05\n",
            "15717  RhoGEF67_u1          PF16615.8          14     4.62     47   1.17   1.18   1.12   0.20\n",
            "15718  RhoGEF67_u2          PF16614.8           4     0.47     99   0.59   0.59   0.52   0.03\n",
            "15719  Rhomboid             PF01694.25         60     4.33    150   0.59   0.59   0.52   0.08\n",
            "15720  Rhomboid_2           PF20401.1          16     0.60    194   0.59   0.57   0.53   0.04\n",
            "15721  Rhomboid_N           PF12122.11         15     2.08     85   0.67   0.62   0.60   0.04\n",
            "15722  Rhomboid_SP          PF12595.11          9     0.65    189   0.59   0.58   0.51   0.02\n",
            "15723  Rho_Binding          PF08912.14         35     2.76     69   0.82   0.85   0.76   0.15\n",
            "15724  Rho_GDI              PF02115.20         34     1.09    193   0.59   0.58   0.52   0.02\n",
            "15725  Rho_N                PF07498.15        104    19.53     43   1.28   1.37   1.25   0.14\n",
            "15726  Rho_RNA_bind         PF07497.15        246     2.25     73   0.77   0.78   0.71   0.02\n",
            "15727  RHS                  PF03527.17         22     6.78     38   1.44   1.31   1.41   0.10\n",
            "15728  RHSP                 PF07999.14          3     0.75    312   0.59   0.57   0.52   0.01\n",
            "15729  RHS_N                PF20445.1           3     0.66    127   0.59   0.54   0.51   0.02\n",
            "15730  RHS_repeat           PF05593.17        202   202.00     38   1.35   1.36   1.25   0.20\n",
            "15731  Rhv                  PF00073.23         49     4.93    171   0.59   0.53   0.50   0.03\n",
            "15732  Rib                  PF08428.13         78     3.77     76   0.74   0.75   0.65   0.10\n",
            "15733  RIB43A               PF05914.15         59     2.87    372   0.59   0.57   0.56   0.11\n",
            "15734  RibD_C               PF01872.20         27     4.65    200   0.59   0.61   0.45   0.01\n",
            "15735  RIBIOP_C             PF04950.15        185     2.65    292   0.59   0.57   0.50   0.02\n",
            "15736  RibLong              PF18957.3          40     2.22     96   0.60   0.60   0.50   0.06\n",
            "15737  Ribonuclease         PF00545.23        129     2.51     86   0.66   0.62   0.50   0.03\n",
            "15738  Ribonuclease_3       PF00636.29        229     7.61    105   0.59   0.60   0.45   0.01\n",
            "15739  Ribonuclease_P       PF00825.21         87     4.08    110   0.59   0.60   0.51   0.05\n",
            "15740  Ribonuclease_T2      PF00445.21        195     2.87    188   0.59   0.49   0.39   0.05\n",
            "15741  Ribonucleas_3_2      PF11469.11          3     0.66    119   0.59   0.64   0.52   0.02\n",
            "15742  Ribonucleas_3_3      PF14622.9          42     2.39    128   0.59   0.61   0.50   0.03\n",
            "15743  Ribonuc_2-5A         PF06479.15        153     2.05    127   0.59   0.55   0.51   0.04\n",
            "15744  Ribonuc_L-PSP        PF01042.24         25     2.82    121   0.59   0.61   0.51   0.03\n",
            "15745  Ribonuc_P_40         PF08584.14         48     3.78    288   0.59   0.57   0.47   0.01\n",
            "15746  Ribonuc_red_2_N      PF08471.13         58     1.12     99   0.59   0.56   0.52   0.03\n",
            "15747  Ribonuc_red_lgC      PF02867.18         67     2.69    527   0.59   0.57   0.50   0.02\n",
            "15748  Ribonuc_red_lgN      PF00317.24        132     7.96     83   0.68   0.67   0.58   0.03\n",
            "15749  Ribonuc_red_sm       PF00268.24        268     4.49    280   0.59   0.55   0.51   0.04\n",
            "15750  Ribophorin_I         PF04597.17        168     3.62    446   0.59   0.57   0.49   0.02\n",
            "15751  Ribophorin_II        PF05817.17         36     2.77    633   0.59   0.61   0.51   0.03\n",
            "15752  Ribosomal_60s        PF00428.22         67     5.00     88   0.65   0.73   0.55   0.09\n",
            "15753  Ribosomal_L1         PF00687.24        541     6.10    204   0.59   0.61   0.46   0.03\n",
            "15754  Ribosomal_L10        PF00466.23        111     6.21    100   0.59   0.62   0.52   0.05\n",
            "15755  Ribosomal_L11        PF00298.22        635     5.17     70   0.80   0.85   0.70   0.05\n",
            "15756  Ribosomal_L11_N      PF03946.17        250     6.42     58   0.96   0.99   0.91   0.04\n",
            "15757  Ribosomal_L12        PF00542.22        328     2.57     67   0.84   0.93   0.79   0.09\n",
            "15758  Ribosomal_L12_N      PF16320.8         479    11.23     48   1.15   1.29   1.03   0.14\n",
            "15759  Ribosomal_L13        PF00572.21         53     2.73    119   0.59   0.57   0.54   0.03\n",
            "15760  Ribosomal_L13e       PF01294.21         99     1.50    180   0.59   0.58   0.52   0.05\n",
            "15761  Ribosomal_L14        PF00238.22        280     1.55    122   0.59   0.61   0.51   0.05\n",
            "15762  Ribosomal_L14e       PF01929.20        133     3.53     75   0.75   0.74   0.71   0.08\n",
            "15763  Ribosomal_L15e       PF00827.20         86     1.26    189   0.59   0.57   0.54   0.04\n",
            "15764  Ribosomal_L16        PF00252.21        441     3.56    132   0.59   0.59   0.47   0.03\n",
            "15765  Ribosomal_L17        PF01196.22        560     2.38     97   0.59   0.61   0.45   0.05\n",
            "15766  Ribosomal_L18        PF17135.7          66     1.15    187   0.59   0.61   0.52   0.03\n",
            "15767  Ribosomal_L18A       PF01775.20         60     2.39    121   0.59   0.56   0.53   0.05\n",
            "15768  Ribosomal_L18p       PF00861.25         15     1.09    116   0.59   0.62   0.53   0.04\n",
            "15769  Ribosomal_L18_c      PF14204.9         135     8.34     56   0.99   1.00   0.91   0.18\n",
            "15770  Ribosomal_L19        PF01245.23        381     1.53    111   0.59   0.60   0.53   0.04\n",
            "15771  Ribosomal_L19e       PF01280.23        123     1.60    144   0.59   0.59   0.55   0.06\n",
            "15772  Ribosomal_L2         PF00181.26         26     3.22     77   0.73   0.73   0.66   0.03\n",
            "15773  Ribosomal_L20        PF00453.21        339     1.76    107   0.59   0.59   0.55   0.06\n",
            "15774  Ribosomal_L21e       PF01157.21        153     2.89    101   0.59   0.57   0.55   0.04\n",
            "15775  Ribosomal_L21p       PF00829.24        601     3.24    100   0.59   0.60   0.51   0.06\n",
            "15776  Ribosomal_L22        PF00237.22        702     4.90    103   0.59   0.60   0.51   0.04\n",
            "15777  Ribosomal_L22e       PF01776.20         90     1.57    109   0.59   0.57   0.53   0.05\n",
            "15778  Ribosomal_L23        PF00276.23        592     3.45     86   0.66   0.67   0.55   0.07\n",
            "15779  Ribosomal_L23eN      PF03939.16        114     6.05     51   1.09   1.09   1.02   0.19\n",
            "15780  ribosomal_L24        PF17136.7         431     6.90     65   0.86   0.85   0.72   0.07\n",
            "15781  Ribosomal_L24e       PF01246.23         93     3.18     66   0.85   0.76   0.82   0.07\n",
            "15782  Ribosomal_L25p       PF01386.22        622     5.50     81   0.70   0.72   0.50   0.03\n",
            "15783  Ribosomal_L26        PF16906.8          98     2.53     82   0.69   0.70   0.58   0.06\n",
            "15784  Ribosomal_L27        PF01016.22        345     1.70     79   0.72   0.73   0.65   0.05\n",
            "15785  Ribosomal_L27A       PF00828.22        465     3.23    118   0.59   0.62   0.37   0.04\n",
            "15786  Ribosomal_L27e       PF01777.21         80     2.32     85   0.67   0.63   0.60   0.09\n",
            "15787  Ribosomal_L27_C      PF18471.4          16     1.74    240   0.59   0.60   0.54   0.06\n",
            "15788  Ribosomal_L28        PF00830.22        415     9.48     60   0.93   0.89   0.82   0.10\n",
            "15789  Ribosomal_L28e       PF01778.20        209     4.34    115   0.59   0.57   0.49   0.05\n",
            "15790  Ribosomal_L29        PF00831.26        712    12.69     57   0.98   1.04   0.93   0.14\n",
            "15791  Ribosomal_L29e       PF01779.20         61     5.47     40   1.37   1.26   1.32   0.28\n",
            "15792  Ribosomal_L2_C       PF03947.21        373     2.02    126   0.59   0.60   0.51   0.04\n",
            "15793  Ribosomal_L3         PF00297.25          7     1.02    369   0.59   0.58   0.51   0.02\n",
            "15794  Ribosomal_L30        PF00327.23        759    20.13     51   1.09   1.13   1.04   0.09\n",
            "15795  Ribosomal_L30_N      PF08079.15        138     6.18     72   0.78   0.80   0.77   0.17\n",
            "15796  Ribosomal_L31        PF01197.21        482     3.14     67   0.84   0.76   0.75   0.05\n",
            "15797  Ribosomal_L31e       PF01198.22         82     2.87     82   0.69   0.67   0.66   0.06\n",
            "15798  Ribosomal_L32e       PF01655.21        148     2.49    108   0.59   0.57   0.54   0.04\n",
            "15799  Ribosomal_L32p       PF01783.26        116     8.56     56   0.99   0.92   0.93   0.13\n",
            "15800  Ribosomal_L33        PF00471.23        333     6.70     47   1.17   1.07   1.05   0.17\n",
            "15801  Ribosomal_L34        PF00468.20        265     5.51     44   1.25   1.24   1.23   0.21\n",
            "15802  Ribosomal_L34e       PF01199.21         89     2.05     94   0.61   0.57   0.56   0.06\n",
            "15803  Ribosomal_L35Ae      PF01247.21         89     1.45     95   0.60   0.59   0.53   0.03\n",
            "15804  Ribosomal_L35p       PF01632.22        538     7.54     61   0.92   0.91   0.87   0.15\n",
            "15805  Ribosomal_L36        PF00444.21        168     7.19     38   1.44   1.30   1.36   0.26\n",
            "15806  Ribosomal_L36e       PF01158.21         77     1.87     96   0.60   0.61   0.54   0.07\n",
            "15807  Ribosomal_L37        PF08561.13         76     2.44    114   0.59   0.58   0.47   0.06\n",
            "15808  Ribosomal_L37ae      PF01780.22         98     2.11     85   0.67   0.60   0.62   0.07\n",
            "15809  Ribosomal_L37e       PF01907.22         79     2.13     54   1.03   0.86   0.97   0.17\n",
            "15810  Ribosomal_L38e       PF01781.21         81     2.50     69   0.81   0.83   0.74   0.07\n",
            "15811  Ribosomal_L39        PF00832.23         77     6.30     42   1.31   1.20   1.29   0.25\n",
            "15812  Ribosomal_L4         PF00573.25        115     4.12    192   0.59   0.61   0.49   0.03\n",
            "15813  Ribosomal_L40e       PF01020.20         24     1.94     50   1.11   0.99   1.07   0.18\n",
            "15814  Ribosomal_L41        PF05162.16          6     3.07     25   2.13   1.99   2.10   0.84\n",
            "15815  Ribosomal_L44        PF00935.22         74     2.78     76   0.74   0.69   0.69   0.11\n",
            "15816  Ribosomal_L5         PF00281.22        490     7.20     57   0.98   1.00   0.92   0.06\n",
            "15817  Ribosomal_L50        PF10501.12         45     4.24    110   0.59   0.59   0.52   0.04\n",
            "15818  Ribosomal_L5e        PF17144.7          94     1.24    163   0.59   0.59   0.53   0.03\n",
            "15819  Ribosomal_L5_C       PF00673.24        386     3.07     94   0.61   0.60   0.51   0.02\n",
            "15820  Ribosomal_L6         PF00347.26         91    10.97     76   0.74   0.77   0.66   0.06\n",
            "15821  Ribosomal_L6e        PF01159.22        136     1.67    112   0.59   0.58   0.50   0.05\n",
            "15822  Ribosomal_L6e_N      PF03868.18         42     3.45     60   0.93   0.91   0.87   0.13\n",
            "15823  Ribosomal_L7Ae       PF01248.29         51     4.88     95   0.60   0.63   0.55   0.04\n",
            "15824  Ribosomal_L9_C       PF03948.17        819     4.33     87   0.65   0.71   0.58   0.06\n",
            "15825  Ribosomal_L9_N       PF01281.22        527    12.62     47   1.17   1.23   1.16   0.06\n",
            "15826  Ribosomal_S10        PF00338.25        343     5.61     98   0.59   0.57   0.53   0.04\n",
            "15827  Ribosomal_S11        PF00411.22          7     0.73    110   0.59   0.61   0.52   0.03\n",
            "15828  Ribosomal_S12        PF17487.5           5     0.59     87   0.65   0.60   0.58   0.11\n",
            "15829  Ribosomal_S13        PF00416.25         11     1.11    128   0.59   0.59   0.53   0.04\n",
            "15830  Ribosomal_S13_N      PF08069.15        113     2.94     60   0.93   0.92   0.90   0.05\n",
            "15831  Ribosomal_S14        PF00253.24        336     6.04     54   1.03   0.98   0.99   0.08\n",
            "15832  Ribosomal_S15        PF00312.25        605     5.03     81   0.70   0.70   0.66   0.07\n",
            "15833  Ribosomal_S16        PF00886.22        577     3.79     62   0.90   0.87   0.80   0.03\n",
            "15834  Ribosomal_S17        PF00366.23        531     5.22     68   0.83   0.79   0.79   0.08\n",
            "15835  Ribosomal_S17e       PF00833.21         40     1.24    122   0.59   0.59   0.52   0.05\n",
            "15836  Ribosomal_S17_N      PF16205.8          62     1.62     67   0.84   0.80   0.76   0.04\n",
            "15837  Ribosomal_S18        PF01084.23        340     8.37     52   1.07   1.07   1.05   0.06\n",
            "15838  Ribosomal_S19        PF00203.24        299     2.67     81   0.70   0.67   0.63   0.03\n",
            "15839  Ribosomal_S19e       PF01090.22        150     1.88    137   0.59   0.59   0.54   0.02\n",
            "15840  Ribosomal_S2         PF00318.23        367     2.41    215   0.59   0.59   0.53   0.03\n",
            "15841  Ribosomal_S20p       PF01649.21        626     4.30     82   0.69   0.72   0.62   0.15\n",
            "15842  Ribosomal_S21        PF01165.23        320    16.09     55   1.01   1.01   0.98   0.16\n",
            "15843  Ribosomal_S21e       PF01249.21         61     1.68     79   0.72   0.71   0.67   0.03\n",
            "15844  Ribosomal_S22        PF08136.14          2     1.13     46   1.20   1.20   1.14   0.08\n",
            "15845  Ribosomal_S24e       PF01282.22        141     5.86     79   0.72   0.72   0.69   0.04\n",
            "15846  Ribosomal_S25        PF03297.18         66     2.68    101   0.59   0.61   0.54   0.07\n",
            "15847  Ribosomal_S26e       PF01283.22         32     0.99    108   0.59   0.55   0.53   0.07\n",
            "15848  Ribosomal_S27        PF01599.22         59     3.57     44   1.25   1.08   1.18   0.17\n",
            "15849  Ribosomal_S27e       PF01667.20          7     1.56     55   1.01   0.92   0.98   0.10\n",
            "15850  Ribosomal_S28e       PF01200.21         40     2.35     64   0.87   0.92   0.84   0.06\n",
            "15851  Ribosomal_S30        PF04758.17         31     2.55     58   0.96   0.95   0.89   0.14\n",
            "15852  Ribosomal_S30AE      PF02482.22        756     9.64     86   0.66   0.68   0.46   0.07\n",
            "15853  Ribosomal_S3Ae       PF01015.21        129     2.85    191   0.59   0.58   0.52   0.04\n",
            "15854  Ribosomal_S3_C       PF00189.23        231     2.65     83   0.68   0.71   0.64   0.05\n",
            "15855  Ribosomal_S4         PF00163.22        392     6.27     93   0.61   0.61   0.45   0.06\n",
            "15856  Ribosomal_S4e        PF00900.23        161     4.58     75   0.75   0.75   0.70   0.04\n",
            "15857  Ribosomal_S4Pg       PF11993.11         13     1.75    163   0.59   0.60   0.52   0.04\n",
            "15858  Ribosomal_S5         PF00333.23        345     6.18     65   0.86   0.90   0.84   0.07\n",
            "15859  Ribosomal_S5_C       PF03719.18        454     6.65     71   0.79   0.83   0.76   0.03\n",
            "15860  Ribosomal_S6         PF01250.20        719     4.85     90   0.63   0.63   0.58   0.03\n",
            "15861  Ribosomal_S6e        PF01092.22        101     2.06    124   0.59   0.61   0.53   0.03\n",
            "15862  Ribosomal_S7         PF00177.24        407     2.50    149   0.59   0.61   0.51   0.04\n",
            "15863  Ribosomal_S7e        PF01251.21         62     1.27    182   0.59   0.61   0.53   0.04\n",
            "15864  Ribosomal_S8         PF00410.22        402     1.54    126   0.59   0.60   0.49   0.03\n",
            "15865  Ribosomal_S8e        PF01201.25        230     2.59    133   0.59   0.61   0.48   0.04\n",
            "15866  Ribosomal_S9         PF00380.22        621     2.14    121   0.59   0.61   0.48   0.04\n",
            "15867  Ribosomal_TL5_C      PF14693.9         622     5.92     85   0.67   0.72   0.60   0.05\n",
            "15868  Ribosom_S12_S23      PF00164.28         12     0.93    114   0.59   0.61   0.51   0.03\n",
            "15869  Ribosom_S30AE_C      PF16321.8         314     6.37     57   0.98   0.96   0.95   0.05\n",
            "15870  Ribos_L4_asso_C      PF14374.9          88     2.41     77   0.73   0.74   0.68   0.07\n",
            "15871  Ribo_biogen_C        PF04034.16        173     1.29    127   0.59   0.58   0.53   0.02\n",
            "15872  Ribul_P_3_epim       PF00834.22         12     1.12    198   0.59   0.59   0.53   0.02\n",
            "15873  Rib_5-P_isom_A       PF06026.17        468     2.71    173   0.59   0.62   0.50   0.02\n",
            "15874  Rib_hydrolayse       PF02267.20         27     1.34    241   0.59   0.52   0.53   0.04\n",
            "15875  Rib_recp_KP_reg      PF05104.15         46     2.75    142   0.59   0.62   0.46   0.15\n",
            "15876  RIC1                 PF07064.16         68     2.59    251   0.59   0.58   0.48   0.03\n",
            "15877  RIC3                 PF15361.9          26     2.10    144   0.59   0.59   0.45   0.03\n",
            "15878  Ric8                 PF10165.12         97     3.63    459   0.59   0.61   0.45   0.02\n",
            "15879  RICH                 PF05062.15         25     2.68     83   0.68   0.71   0.61   0.12\n",
            "15880  RicinB_lectin_2      PF14200.9         236     9.17     89   0.64   0.59   0.47   0.06\n",
            "15881  Ricin_B_lectin       PF00652.25         85     5.57    127   0.59   0.50   0.45   0.07\n",
            "15882  Rick_17kDa_Anti      PF05433.18         77    23.33     42   1.31   1.51   1.26   0.51\n",
            "15883  RICTOR_M             PF14666.9          97     3.41    104   0.59   0.59   0.53   0.04\n",
            "15884  RICTOR_N             PF14664.9          95     3.44    373   0.59   0.61   0.51   0.03\n",
            "15885  RICTOR_phospho       PF14665.9           8     0.73    112   0.59   0.61   0.51   0.04\n",
            "15886  RICTOR_V             PF14668.9          93     3.78     71   0.79   0.79   0.76   0.05\n",
            "15887  Rieske               PF00355.29         51     3.74     89   0.64   0.55   0.49   0.04\n",
            "15888  Rieske_2             PF13806.9          39     1.70    104   0.59   0.58   0.52   0.02\n",
            "15889  Rieske_3             PF18465.4          19     0.81     96   0.60   0.54   0.53   0.03\n",
            "15890  Rif1_N               PF12231.11         65     3.96    374   0.59   0.57   0.54   0.04\n",
            "15891  RIF5_SNase_1         PF18187.4           2     0.52    161   0.59   0.59   0.52   0.02\n",
            "15892  RIF5_SNase_2         PF18189.4           2     0.60    178   0.59   0.60   0.52   0.02\n",
            "15893  RIFIN                PF02009.19        112     1.53    327   0.59   0.58   0.51   0.05\n",
            "15894  RIG-I_C              PF18119.4          17     2.92    139   0.59   0.57   0.51   0.05\n",
            "15895  RIG-I_C-RD           PF11648.11         63     2.23    117   0.59   0.49   0.53   0.06\n",
            "15896  RIH_assoc            PF08454.14         63     2.34     99   0.59   0.56   0.52   0.07\n",
            "15897  RIIa                 PF02197.20         12     9.30     38   1.43   1.44   1.42   0.09\n",
            "15898  RII_binding_1        PF10522.12          7     7.00     19   1.46   1.57   1.45   0.23\n",
            "15899  RILP                 PF11461.11         29     2.79     69   0.81   0.85   0.77   0.09\n",
            "15900  RimK                 PF08443.14          4     0.97    188   0.59   0.62   0.52   0.03\n",
            "15901  Rimk_N               PF18030.4         197     2.31     94   0.61   0.61   0.57   0.03\n",
            "15902  RimM                 PF01782.21        106     7.02     84   0.68   0.71   0.60   0.02\n",
            "15903  RimP_N               PF02576.20         30     3.31     73   0.77   0.81   0.71   0.03\n",
            "15904  RinB                 PF06116.15          2     1.27     51   1.08   1.14   1.03   0.09\n",
            "15905  RINGv                PF12906.10         21     2.00     48   1.15   0.86   1.06   0.31\n",
            "15906  RING_CBP-p300        PF06001.16         14     1.90     40   1.37   1.15   1.33   0.20\n",
            "15907  Ring_hydroxyl_A      PF00848.22         42     6.64    214   0.59   0.51   0.49   0.03\n",
            "15908  Ring_hydroxyl_B      PF00866.21         21     2.68    144   0.59   0.56   0.53   0.04\n",
            "15909  RINT1_TIP1           PF04437.16         58     3.20    513   0.59   0.58   0.48   0.04\n",
            "15910  RIO1                 PF01163.25         27     1.90    188   0.59   0.56   0.51   0.03\n",
            "15911  Rio2_N               PF09202.14         26     1.60     82   0.69   0.70   0.64   0.03\n",
            "15912  RIP                  PF00161.22         36     3.96    196   0.59   0.58   0.49   0.02\n",
            "15913  Ripply               PF14998.9          26     1.19     84   0.68   0.61   0.59   0.04\n",
            "15914  RisS_PPD             PF16524.8          28     1.33    107   0.59   0.59   0.54   0.03\n",
            "15915  Rit1_C               PF17184.7         107     1.15    267   0.59   0.55   0.48   0.02\n",
            "15916  RITA                 PF17066.8           9     0.95    268   0.59   0.58   0.51   0.04\n",
            "15917  RIX1                 PF08167.15         88     6.45    193   0.59   0.61   0.51   0.04\n",
            "15918  RKP_N                PF19322.2          30     0.99    137   0.59   0.62   0.53   0.02\n",
            "15919  RL                   PF17797.4          14     1.00     71   0.79   0.77   0.75   0.05\n",
            "15920  RL10P_insert         PF17777.4          21     2.81     71   0.79   0.85   0.73   0.03\n",
            "15921  RL11D                PF11088.11          2     0.57     99   0.59   0.55   0.52   0.04\n",
            "15922  RLAN                 PF14401.9          62     1.97    150   0.59   0.58   0.52   0.02\n",
            "15923  RlaP                 PF10127.12         24     2.87    246   0.59   0.58   0.52   0.02\n",
            "15924  RLI                  PF04068.18         68    17.89     35   1.55   1.40   1.52   0.20\n",
            "15925  RLL                  PF10036.12         42     3.17    257   0.59   0.60   0.48   0.04\n",
            "15926  RlmM_FDX             PF18125.4          11     1.39     71   0.79   0.76   0.75   0.04\n",
            "15927  RloB                 PF13707.9         110     5.76    196   0.59   0.56   0.33   0.04\n",
            "15928  RME-8_N              PF19432.2          34     1.17    815   0.59   0.58   0.52   0.02\n",
            "15929  RMF                  PF04957.15         22     1.43     54   1.03   0.94   0.98   0.13\n",
            "15930  RMI1_C               PF16099.8          22     2.10    143   0.59   0.59   0.52   0.03\n",
            "15931  RMI1_N               PF08585.15         99     4.43    217   0.59   0.60   0.43   0.02\n",
            "15932  RMI2                 PF16100.8          25     1.76    135   0.59   0.58   0.48   0.03\n",
            "15933  RmlD_sub_bind        PF04321.20         22     1.82    287   0.59   0.59   0.53   0.02\n",
            "15934  RMMBL                PF07521.15         84    16.61     63   0.89   0.89   0.81   0.03\n",
            "15935  RMP                  PF14996.9          19     1.37    154   0.59   0.50   0.53   0.06\n",
            "15936  RmuC                 PF02646.19        286     3.74    292   0.59   0.61   0.51   0.04\n",
            "15937  RNA12                PF10443.12         17     1.14    444   0.59   0.60   0.52   0.02\n",
            "15938  RnaseA               PF00074.23        140     2.28    121   0.59   0.47   0.50   0.08\n",
            "15939  RNaseH_C             PF09293.13          2     0.54    124   0.59   0.59   0.52   0.02\n",
            "15940  RNaseH_like          PF04308.15         52     1.66    145   0.59   0.59   0.53   0.03\n",
            "15941  RNaseH_pPIWI_RE      PF13032.9          42     6.13    292   0.59   0.56   0.46   0.03\n",
            "15942  RNase_3_N            PF18497.4          40     3.03     95   0.60   0.63   0.50   0.05\n",
            "15943  RNAse_A_bac          PF18431.4          30     2.54    113   0.59   0.58   0.49   0.03\n",
            "15944  RNase_E_G            PF10150.12        501     2.34    270   0.59   0.61   0.51   0.04\n",
            "15945  RNase_H              PF00075.27         23     2.26    143   0.59   0.57   0.45   0.03\n",
            "15946  RNase_H2-Ydr279      PF09468.13         85     5.71    159   0.59   0.59   0.44   0.03\n",
            "15947  RNase_H2_suC         PF08615.14         19     2.87    141   0.59   0.56   0.45   0.02\n",
            "15948  RNase_HII            PF01351.21         11     2.12    198   0.59   0.61   0.51   0.02\n",
            "15949  RNase_H_2            PF13482.9          68     3.91    166   0.59   0.56   0.49   0.03\n",
            "15950  RNase_II_C_S1        PF18614.4         101     8.92     59   0.95   1.04   0.88   0.10\n",
            "15951  RNase_J_C            PF17770.4          77     2.34    102   0.59   0.62   0.55   0.03\n",
            "15952  RNase_P-MRP_p29      PF01868.19        201     2.92     91   0.63   0.65   0.47   0.03\n",
            "15953  RNase_PH             PF01138.24        145     7.46    132   0.59   0.63   0.44   0.02\n",
            "15954  RNase_PH_C           PF03725.18         79    12.96     67   0.84   0.88   0.74   0.04\n",
            "15955  RNase_P_p30          PF01876.19         40     4.92    215   0.59   0.61   0.51   0.02\n",
            "15956  RNase_P_pop3         PF08228.14          5     1.18    158   0.59   0.59   0.50   0.03\n",
            "15957  RNase_P_Rpp14        PF01900.22        196     3.45    103   0.59   0.59   0.46   0.03\n",
            "15958  RNase_T              PF00929.27         40     7.09    165   0.59   0.57   0.50   0.02\n",
            "15959  RNase_Y_N            PF12072.11         44     1.93    201   0.59   0.63   0.55   0.10\n",
            "15960  RNase_Zc3h12a        PF11977.11         20     2.57    156   0.59   0.57   0.51   0.02\n",
            "15961  RNase_Zc3h12a_2      PF14626.9           3     0.48    122   0.59   0.57   0.51   0.02\n",
            "15962  RNA_bind             PF08675.14         10     0.97     79   0.72   0.65   0.66   0.03\n",
            "15963  RNA_binding          PF01877.20         93     4.08    119   0.59   0.61   0.49   0.04\n",
            "15964  RNA_capsid           PF03035.17         34     1.23    226   0.59   0.58   0.49   0.03\n",
            "15965  RNA_helicase         PF00910.25         99     4.57    107   0.59   0.55   0.51   0.02\n",
            "15966  RNA_ligase           PF09414.13        149     8.13    165   0.59   0.57   0.22   0.02\n",
            "15967  RNA_lig_T4_1         PF09511.13        124     2.96    223   0.59   0.57   0.44   0.03\n",
            "15968  RNA_Me_trans         PF04252.16         90     1.54    200   0.59   0.59   0.50   0.02\n",
            "15969  RNA_pol              PF00940.22        190     1.70    426   0.59   0.56   0.49   0.02\n",
            "15970  RNA_polI_A14         PF08203.14         24     2.13    117   0.59   0.62   0.50   0.05\n",
            "15971  RNA_polI_A34         PF08208.14         73     6.15    207   0.59   0.60   0.47   0.08\n",
            "15972  RNA_pol_3_Rpc31      PF11705.11         89     4.59    214   0.59   0.58   0.47   0.08\n",
            "15973  RNA_pol_A_bac        PF01000.29         59     3.40    112   0.59   0.59   0.45   0.03\n",
            "15974  RNA_pol_A_CTD        PF03118.18         59     4.29     67   0.84   0.90   0.81   0.07\n",
            "15975  RNA_pol_inhib        PF16857.8          11     2.33     47   1.17   1.17   1.11   0.06\n",
            "15976  RNA_pol_I_A49        PF06870.15        104     4.62    375   0.59   0.60   0.49   0.03\n",
            "15977  RNA_pol_I_TF         PF04090.15         10     1.19    199   0.59   0.55   0.51   0.03\n",
            "15978  RNA_pol_L            PF01193.27        380     8.45     74   0.76   0.79   0.56   0.02\n",
            "15979  RNA_pol_L_2          PF13656.9         239     5.34     75   0.75   0.74   0.68   0.04\n",
            "15980  RNA_POL_M_15KD       PF02150.19          9     5.27     36   1.51   1.29   1.45   0.25\n",
            "15981  RNA_pol_N            PF01194.20         98     3.03     58   0.96   0.86   0.89   0.08\n",
            "15982  RNA_pol_Rbc25        PF08292.15         23     1.70    125   0.59   0.55   0.51   0.03\n",
            "15983  RNA_pol_Rpa2_4       PF06883.15        116     4.38     58   0.96   0.96   0.88   0.02\n",
            "15984  RNA_pol_Rpb1_1       PF04997.15         25     1.93    312   0.59   0.57   0.50   0.03\n",
            "15985  RNA_pol_Rpb1_2       PF00623.23         19     1.16    166   0.59   0.58   0.53   0.02\n",
            "15986  RNA_pol_Rpb1_3       PF04983.21        112     5.42    157   0.59   0.59   0.44   0.02\n",
            "15987  RNA_pol_Rpb1_4       PF05000.20        106     5.23    108   0.59   0.58   0.49   0.03\n",
            "15988  RNA_pol_Rpb1_5       PF04998.20         24     1.68    267   0.59   0.59   0.46   0.03\n",
            "15989  RNA_pol_Rpb1_6       PF04992.17        117     2.72    189   0.59   0.59   0.51   0.04\n",
            "15990  RNA_pol_Rpb1_7       PF04990.15         70     1.69    135   0.59   0.59   0.50   0.05\n",
            "15991  RNA_pol_Rpb1_R       PF05001.16         81    81.00     14   2.75   2.71   2.55   1.07\n",
            "15992  RNA_pol_Rpb2_1       PF04563.18         13     1.70    203   0.59   0.59   0.48   0.02\n",
            "15993  RNA_pol_Rpb2_2       PF04561.17         22     4.20    190   0.59   0.59   0.50   0.03\n",
            "15994  RNA_pol_Rpb2_3       PF04565.19         99     3.65     68   0.83   0.80   0.78   0.02\n",
            "15995  RNA_pol_Rpb2_4       PF04566.16        244     3.56     62   0.90   0.91   0.85   0.05\n",
            "15996  RNA_pol_Rpb2_45      PF10385.12        621     3.95     65   0.86   0.87   0.73   0.07\n",
            "15997  RNA_pol_Rpb2_5       PF04567.20        140     4.24     67   0.84   0.79   0.72   0.06\n",
            "15998  RNA_pol_Rpb2_6       PF00562.31         68     1.99    389   0.59   0.59   0.50   0.02\n",
            "15999  RNA_pol_Rpb2_7       PF04560.23         45     1.49     87   0.65   0.65   0.58   0.03\n",
            "16000  RNA_pol_Rpb4         PF03874.19         48     4.70    125   0.59   0.60   0.45   0.06\n",
            "16001  RNA_pol_Rpb5_C       PF01191.22        142     3.23     73   0.77   0.77   0.73   0.04\n",
            "16002  RNA_pol_Rpb5_N       PF03871.17        130     2.58     89   0.64   0.59   0.55   0.05\n",
            "16003  RNA_pol_Rpb6         PF01192.25        342    25.10     52   1.07   1.14   0.88   0.06\n",
            "16004  RNA_pol_Rpb8         PF03870.18        112     1.35    140   0.59   0.58   0.50   0.03\n",
            "16005  RNA_pol_RpbG         PF16992.8           8     1.14    119   0.59   0.59   0.52   0.03\n",
            "16006  RNA_pol_Rpc34        PF05158.15         14     2.05    336   0.59   0.58   0.51   0.02\n",
            "16007  RNA_pol_Rpc4         PF05132.17        163     4.32    150   0.59   0.60   0.49   0.03\n",
            "16008  RNA_pol_Rpc82        PF05645.16         29     5.78    258   0.59   0.59   0.45   0.03\n",
            "16009  RNA_pol_Rpo13        PF12136.11          7     4.84     43   1.28   1.34   1.23   0.19\n",
            "16010  RNA_replicase_B      PF03431.16          5     1.01    540   0.59   0.58   0.51   0.01\n",
            "16011  RNB                  PF00773.22         54     4.82    323   0.59   0.59   0.47   0.02\n",
            "16012  Rnf-Nqr              PF02508.17        622     2.77    174   0.59   0.62   0.52   0.09\n",
            "16013  RNF111_N             PF15303.9           8     0.68    279   0.59   0.60   0.50   0.05\n",
            "16014  RNF152_C             PF19325.2           6     0.68    136   0.59   0.59   0.50   0.02\n",
            "16015  RNF180_C             PF19332.2           9     0.71    101   0.59   0.51   0.52   0.04\n",
            "16016  RNF220               PF15926.8           7     0.97    248   0.59   0.57   0.50   0.04\n",
            "16017  RnfC_N               PF13375.9          47     1.69    101   0.59   0.61   0.54   0.03\n",
            "16018  RNHCP                PF12647.10         51     1.10     90   0.63   0.52   0.57   0.09\n",
            "16019  Rnk_N                PF14760.9          81    15.97     41   1.34   1.47   1.28   0.15\n",
            "16020  RnlA-toxin_DBD       PF19034.3           7     1.47    125   0.59   0.55   0.53   0.04\n",
            "16021  RnlA_toxin           PF15935.8           7     2.38     87   0.65   0.64   0.60   0.05\n",
            "16022  RnlA_toxin_N         PF19417.2          19     4.64     82   0.69   0.68   0.63   0.06\n",
            "16023  RnlB_antitoxin       PF15933.8           9     2.24     94   0.61   0.61   0.55   0.04\n",
            "16024  RNPP_C               PF18768.4           4     0.92    213   0.59   0.57   0.53   0.05\n",
            "16025  RNR_Alpha            PF17975.4          33     0.99    101   0.59   0.54   0.53   0.03\n",
            "16026  RNR_inhib            PF08591.13         40     2.71    107   0.59   0.59   0.45   0.04\n",
            "16027  RNR_N                PF08343.13        113     2.21     82   0.69   0.65   0.65   0.08\n",
            "16028  Robl_LC7             PF03259.20         37     8.85     91   0.63   0.69   0.59   0.03\n",
            "16029  Roc                  PF08477.16        164     2.63    119   0.59   0.56   0.50   0.03\n",
            "16030  Rod-binding          PF10135.12        249    18.39     51   1.09   1.08   0.94   0.12\n",
            "16031  Rod_C                PF10493.12         15     1.38    551   0.59   0.58   0.53   0.03\n",
            "16032  Rod_cone_degen       PF15201.9           3     0.85     54   1.03   1.02   0.97   0.03\n",
            "16033  ROF                  PF07073.15         30     1.92     80   0.71   0.69   0.66   0.05\n",
            "16034  Rogdi_lz             PF10259.12         85     4.08    300   0.59   0.61   0.46   0.03\n",
            "16035  ROH1-like            PF05633.14          6     0.71    384   0.59   0.59   0.50   0.02\n",
            "16036  ROK                  PF00480.23         11     2.22    294   0.59   0.60   0.52   0.02\n",
            "16037  ROKNT                PF08067.14          7     1.61     41   1.33   1.31   1.25   0.22\n",
            "16038  RolB_RolC            PF02027.20         13     2.95    184   0.59   0.55   0.47   0.02\n",
            "16039  Rol_Rep_N            PF18106.4          16     1.59     91   0.63   0.58   0.56   0.03\n",
            "16040  Romo1                PF10247.12         56     3.24     66   0.85   0.85   0.82   0.13\n",
            "16041  rOmpB                PF12334.11          2     0.49    223   0.59   0.62   0.52   0.04\n",
            "16042  Rootletin            PF15035.9          30     2.50    191   0.59   0.58   0.50   0.10\n",
            "16043  Root_cap             PF06830.14         36     1.70     57   0.98   0.94   0.94   0.04\n",
            "16044  Rop                  PF01815.19          4     1.05     57   0.98   0.99   0.93   0.09\n",
            "16045  Rop-like             PF05082.16         53     2.75     64   0.88   0.89   0.84   0.08\n",
            "16046  ROQ_II               PF18386.4           4     1.13     56   0.99   0.97   0.95   0.05\n",
            "16047  Rossmann-like        PF10727.12          5     0.69    127   0.59   0.65   0.52   0.03\n",
            "16048  ROS_MUCR             PF05443.14        193     1.93    125   0.59   0.59   0.51   0.03\n",
            "16049  Rot1                 PF10681.12         62     0.93    209   0.59   0.54   0.51   0.01\n",
            "16050  Rotamase             PF00639.24         38     3.64     97   0.59   0.60   0.51   0.03\n",
            "16051  Rotamase_2           PF13145.9         101    16.99    121   0.59   0.63   0.50   0.05\n",
            "16052  Rotamase_3           PF13616.9          16     2.32    116   0.59   0.60   0.50   0.03\n",
            "16053  Rotavirus_VP1        PF12289.11          2     0.53    313   0.59   0.59   0.52   0.03\n",
            "16054  Rotavirus_VP3        PF06929.14          2     0.54    692   0.59   0.56   0.51   0.03\n",
            "16055  Rotavirus_VP7        PF05868.14          3     0.52    249   0.59   0.55   0.52   0.03\n",
            "16056  Rota_Capsid_VP6      PF00980.20          2     0.54    396   0.59   0.58   0.53   0.02\n",
            "16057  Rota_NS26            PF01525.19          3     0.86    198   0.59   0.58   0.48   0.06\n",
            "16058  Rota_NS35            PF02509.17          2     0.57    316   0.59   0.57   0.52   0.02\n",
            "16059  Rota_NS53            PF00981.20         14     0.71    488   0.59   0.53   0.53   0.03\n",
            "16060  Rota_NS6             PF04866.15          1     0.40     92   0.62   0.57   0.55   0.05\n",
            "16061  Rota_NSP3            PF01665.19          2     0.65    299   0.59   0.58   0.51   0.03\n",
            "16062  Rota_NSP4            PF01452.19          1     0.40    173   0.59   0.59   0.52   0.03\n",
            "16063  Rota_VP2             PF05087.15          2     0.52    884   0.59   0.59   0.52   0.02\n",
            "16064  Rota_VP4_MID         PF17477.5           2     0.51    227   0.59   0.57   0.51   0.02\n",
            "16065  Roughex              PF06020.14          2     0.55    365   0.59   0.60   0.51   0.02\n",
            "16066  Rox3                 PF08633.13         58     1.90    204   0.59   0.58   0.45   0.04\n",
            "16067  ROXA-like_wH         PF20514.1          24     6.72    119   0.59   0.61   0.46   0.02\n",
            "16068  RP-C                 PF03428.16         36     2.31    175   0.59   0.62   0.54   0.03\n",
            "16069  RP-C_C               PF11800.11         34     1.54    102   0.59   0.60   0.54   0.02\n",
            "16070  RP1-2                PF12042.11         17     2.24    168   0.59   0.68   0.53   0.15\n",
            "16071  RP853                PF17542.5           2     0.44    317   0.59   0.60   0.52   0.04\n",
            "16072  RP854                PF17460.5           1     0.39    212   0.59   0.58   0.51   0.05\n",
            "16073  RPA                  PF10134.12         20     1.57    229   0.59   0.59   0.51   0.02\n",
            "16074  RPA43_OB             PF17875.4          64     4.04    127   0.59   0.58   0.48   0.03\n",
            "16075  RPAP1_C              PF08620.13         98     2.72     69   0.82   0.82   0.73   0.02\n",
            "16076  RPAP1_N              PF08621.13        118    11.03     46   1.20   1.26   1.19   0.19\n",
            "16077  RPAP2_Rtr1           PF04181.16        117     3.11     76   0.74   0.64   0.65   0.08\n",
            "16078  RPAP3_C              PF13877.9         123     8.39     92   0.62   0.63   0.52   0.03\n",
            "16079  RPA_C                PF08784.14         15     2.66    104   0.59   0.59   0.51   0.04\n",
            "16080  RPA_interact_C       PF14768.9          66     2.01     81   0.70   0.56   0.61   0.12\n",
            "16081  RPA_interact_M       PF14767.9          22     2.72     94   0.61   0.59   0.50   0.08\n",
            "16082  RPA_interact_N       PF14766.9          33    14.32     39   1.40   1.33   1.37   0.26\n",
            "16083  RPC5                 PF04801.16         47     3.82    429   0.59   0.57   0.48   0.03\n",
            "16084  RPC5_C               PF19725.2          27     1.04    254   0.59   0.58   0.51   0.03\n",
            "16085  RPE65                PF03055.18        494     5.57    476   0.59   0.55   0.37   0.01\n",
            "16086  RPEL                 PF02755.18        123   123.00     24   1.91   2.01   1.91   0.28\n",
            "16087  Rpf1_C               PF11574.11         12     1.69     88   0.65   0.67   0.57   0.05\n",
            "16088  RPGR1_C              PF18111.4           7     1.08    166   0.59   0.60   0.52   0.02\n",
            "16089  RPM2                 PF08579.14          5     0.77    119   0.59   0.59   0.52   0.05\n",
            "16090  RPN13_C              PF16550.8          49     2.59    112   0.59   0.63   0.50   0.04\n",
            "16091  RPN1_C               PF18051.4          40     1.95     54   1.03   1.05   0.99   0.02\n",
            "16092  RPN1_RPN2_N          PF17781.4          19     1.22    304   0.59   0.59   0.53   0.03\n",
            "16093  RPN2_C               PF18004.4          84     1.80    161   0.59   0.60   0.51   0.06\n",
            "16094  Rpn3_C               PF08375.14         62     2.63     66   0.85   0.81   0.77   0.08\n",
            "16095  RPN5_C               PF18098.4          37     6.26     33   1.64   1.59   1.63   0.23\n",
            "16096  RPN6_C_helix         PF18503.4          75    44.22     27   1.98   2.06   1.99   0.16\n",
            "16097  RPN6_N               PF18055.4         115     1.67    117   0.59   0.61   0.54   0.05\n",
            "16098  RPN7                 PF10602.12         55     4.52    174   0.59   0.59   0.54   0.04\n",
            "16099  Rpn9_C               PF18261.4          36     5.87     33   1.64   1.58   1.63   0.20\n",
            "16100  rpo132               PF12415.11          8     2.69     32   1.69   1.69   1.67   0.10\n",
            "16101  rpo30_N              PF12410.11          9     0.63    135   0.59   0.56   0.52   0.05\n",
            "16102  RPOL_N               PF14700.9         171     4.49    335   0.59   0.58   0.49   0.03\n",
            "16103  RpoY                 PF07288.14         40     1.54     68   0.83   0.80   0.77   0.09\n",
            "16104  Rpp20                PF12328.11         39     2.67    137   0.59   0.61   0.49   0.06\n",
            "16105  Rpr2                 PF04032.19        271     4.10     90   0.63   0.55   0.49   0.09\n",
            "16106  RPS31                PF17067.8           1     0.40     99   0.59   0.60   0.52   0.03\n",
            "16107  RPT                  PF13446.9         159    13.80     59   0.95   0.99   0.86   0.06\n",
            "16108  RPW8                 PF05659.14         13     1.93    139   0.59   0.61   0.54   0.04\n",
            "16109  RQC                  PF09382.13         99     5.60    114   0.59   0.60   0.53   0.02\n",
            "16110  RraA-like            PF03737.18        353     3.90    150   0.59   0.61   0.43   0.03\n",
            "16111  RraB                 PF06877.14         74     6.37    102   0.59   0.57   0.54   0.04\n",
            "16112  RRF                  PF01765.22        780     4.13    162   0.59   0.62   0.51   0.06\n",
            "16113  Rrf2                 PF02082.23         99     4.40    132   0.59   0.63   0.55   0.02\n",
            "16114  RRF_GI               PF12614.11          8     0.68    126   0.59   0.59   0.52   0.03\n",
            "16115  RRG8                 PF17068.8           5     1.07    279   0.59   0.59   0.52   0.03\n",
            "16116  RRM                  PF10378.12         13     2.62     52   1.07   1.04   1.01   0.12\n",
            "16117  RRM_1                PF00076.25         68    11.56     70   0.80   0.82   0.76   0.03\n",
            "16118  RRM_2                PF04059.15          4     0.51     97   0.59   0.56   0.52   0.02\n",
            "16119  RRM_3                PF08777.14         11     1.58    102   0.59   0.60   0.52   0.04\n",
            "16120  RRM_4                PF10598.12         34     0.67     92   0.62   0.62   0.56   0.03\n",
            "16121  RRM_5                PF13893.9           6     1.08    125   0.59   0.58   0.52   0.02\n",
            "16122  RRM_7                PF16367.8          11     0.78     97   0.59   0.57   0.50   0.01\n",
            "16123  RRM_8                PF11835.11         10     0.81     89   0.64   0.63   0.58   0.04\n",
            "16124  RRM_9                PF18444.4          33     2.82     70   0.80   0.82   0.77   0.04\n",
            "16125  RRM_DME              PF15628.9          13     0.66    102   0.59   0.56   0.53   0.01\n",
            "16126  RRM_occluded         PF16842.8          44     3.01     79   0.72   0.76   0.66   0.03\n",
            "16127  RRM_Rrp7             PF17799.4           8     1.20    161   0.59   0.57   0.49   0.03\n",
            "16128  RRN3                 PF05327.14         99     2.60    556   0.59   0.57   0.49   0.03\n",
            "16129  Rrn6                 PF10214.12         47     7.03    853   0.59   0.58   0.45   0.03\n",
            "16130  RRN9                 PF10680.12         46     2.63     69   0.81   0.75   0.71   0.07\n",
            "16131  RrnaAD               PF00398.23         11     2.89    265   0.59   0.59   0.50   0.02\n",
            "16132  rRNA_methylase       PF06962.15        168     1.65    137   0.59   0.59   0.53   0.02\n",
            "16133  rRNA_proc-arch       PF13234.9         249     6.34    270   0.59   0.59   0.42   0.04\n",
            "16134  rRNA_processing      PF08524.14         28     2.22    146   0.59   0.58   0.51   0.15\n",
            "16135  RRP14                PF15459.9          50     4.92     62   0.90   0.90   0.80   0.11\n",
            "16136  Rrp15p               PF07890.15        138     3.34    134   0.59   0.62   0.45   0.09\n",
            "16137  RRP36                PF06102.15        159     3.39    166   0.59   0.59   0.48   0.12\n",
            "16138  Rrp40_N              PF18311.4          24     4.85     48   1.15   1.18   1.01   0.08\n",
            "16139  Rrp44_CSD1           PF17216.6           1     0.39    148   0.59   0.59   0.52   0.01\n",
            "16140  Rrp44_S1             PF17215.6          60     1.34     89   0.64   0.65   0.56   0.04\n",
            "16141  RRP7                 PF12923.10         39     2.46    119   0.59   0.56   0.49   0.11\n",
            "16142  RRS1                 PF04939.15        143     2.13    166   0.59   0.56   0.50   0.06\n",
            "16143  RRT14                PF17075.8          12     1.54    197   0.59   0.61   0.50   0.08\n",
            "16144  RRXRR                PF14239.9          53     1.76    174   0.59   0.59   0.51   0.04\n",
            "16145  RR_TM4-6             PF06459.15         42     1.94    281   0.59   0.58   0.49   0.03\n",
            "16146  RS4NT                PF08071.15         45     6.75     37   1.47   1.38   1.45   0.18\n",
            "16147  Rsa3                 PF14615.9          63     6.56     46   1.20   1.22   1.17   0.08\n",
            "16148  RsaA_NTD             PF19198.3           7     0.77    180   0.59   0.60   0.51   0.03\n",
            "16149  RsbRD_N              PF14361.9         111     6.65    104   0.59   0.63   0.49   0.05\n",
            "16150  Rsbr_N               PF08678.13         14     1.49    130   0.59   0.56   0.54   0.04\n",
            "16151  RsbU_N               PF08673.13         25     2.66     75   0.75   0.77   0.71   0.06\n",
            "16152  RSB_motif            PF16294.8          22     1.72    102   0.59   0.58   0.51   0.10\n",
            "16153  Rsc14                PF08586.13         13     0.87     99   0.59   0.59   0.51   0.03\n",
            "16154  RSD-2                PF07547.16         17     2.12     85   0.67   0.62   0.59   0.05\n",
            "16155  RsdA_SigD_bd         PF16751.8          16     3.60     46   1.20   1.25   1.17   0.13\n",
            "16156  Rsd_AlgQ             PF04353.16         44     1.45    150   0.59   0.58   0.53   0.04\n",
            "16157  RseA_C               PF03873.16         33     5.36     55   1.01   1.01   0.91   0.08\n",
            "16158  RseA_N               PF03872.16         28     2.23     87   0.65   0.65   0.57   0.05\n",
            "16159  RseC_MucC            PF04246.15        161     4.00    130   0.59   0.58   0.50   0.04\n",
            "16160  RSF                  PF14876.9           7     1.49    217   0.59   0.54   0.52   0.05\n",
            "16161  RsfS                 PF02410.18        786     3.11    100   0.59   0.56   0.45   0.04\n",
            "16162  RsgA_GTPase          PF03193.19         60     1.72    167   0.59   0.62   0.52   0.02\n",
            "16163  RsgA_N               PF16745.8          57     3.00     54   1.03   1.02   0.97   0.05\n",
            "16164  RsgI_N               PF12791.10         45    19.17     53   1.05   1.07   0.98   0.04\n",
            "16165  RskA                 PF10099.12        193     8.98    180   0.59   0.64   0.43   0.05\n",
            "16166  Rsm1                 PF08600.13         10     1.21     92   0.62   0.55   0.53   0.03\n",
            "16167  Rsm22                PF09243.13          5     0.98    287   0.59   0.56   0.51   0.02\n",
            "16168  RsmF_methylt_CI      PF17126.8         106     4.30     61   0.92   0.94   0.84   0.03\n",
            "16169  RsmJ                 PF04378.16          6     0.87    245   0.59   0.59   0.53   0.01\n",
            "16170  RSN1_7TM             PF02714.18         75     3.96    274   0.59   0.57   0.55   0.07\n",
            "16171  RSN1_TM              PF13967.9         129     4.18    156   0.59   0.56   0.51   0.04\n",
            "16172  RSRP                 PF17069.8           3     0.65    299   0.59   0.58   0.48   0.08\n",
            "16173  RSS_P20              PF11757.11         11     3.85     94   0.61   0.60   0.54   0.07\n",
            "16174  RST                  PF12174.11         48     3.96     67   0.84   0.84   0.81   0.05\n",
            "16175  RSV_NS2              PF03113.17          1     0.38    124   0.59   0.56   0.51   0.03\n",
            "16176  RTA1                 PF04479.16        250     2.85    206   0.59   0.58   0.51   0.06\n",
            "16177  RTBV_P12             PF06361.14          1     0.41    110   0.59   0.60   0.52   0.02\n",
            "16178  RTBV_P46             PF06216.14          1     0.37    389   0.59   0.56   0.51   0.01\n",
            "16179  RTC                  PF01137.24        157     2.78    222   0.59   0.61   0.47   0.01\n",
            "16180  RTC4                 PF14474.9          98     2.87    121   0.59   0.59   0.51   0.02\n",
            "16181  RtcB                 PF01139.20        247     1.83    413   0.59   0.58   0.49   0.02\n",
            "16182  RtcR                 PF06956.14         32     0.82    183   0.59   0.57   0.52   0.02\n",
            "16183  RTC_insert           PF05189.16        237     5.19    103   0.59   0.63   0.49   0.02\n",
            "16184  RteC                 PF09357.13         49     2.33    219   0.59   0.55   0.55   0.05\n",
            "16185  Rtf2                 PF04641.15         18     1.81    262   0.59   0.57   0.50   0.05\n",
            "16186  RTP                  PF02334.19          9     0.68    113   0.59   0.58   0.52   0.05\n",
            "16187  RTP1_C1              PF10363.12        121     2.97    112   0.59   0.62   0.52   0.03\n",
            "16188  RTP1_C2              PF10304.12         62    62.00     34   1.55   1.61   1.54   0.17\n",
            "16189  RTP801_C             PF07809.14         37     1.77    119   0.59   0.59   0.52   0.02\n",
            "16190  Rtt102p              PF09510.13          9     1.16    132   0.59   0.55   0.50   0.05\n",
            "16191  Rtt106               PF08512.15        193     4.26     88   0.65   0.63   0.53   0.03\n",
            "16192  Rtt106_N             PF18215.4          18     4.28     46   1.20   1.23   1.15   0.07\n",
            "16193  RTT107_BRCT_5        PF16770.8          63     2.53    100   0.59   0.59   0.50   0.03\n",
            "16194  RTT107_BRCT_6        PF16771.8          21     1.32    114   0.59   0.57   0.50   0.05\n",
            "16195  RTTN_N               PF14726.9          29     2.06     97   0.59   0.62   0.54   0.05\n",
            "16196  RTX                  PF02382.18          7     1.27    324   0.59   0.63   0.52   0.03\n",
            "16197  RtxA                 PF07634.14         71    71.00     18   1.76   1.82   1.70   0.35\n",
            "16198  RTX_C                PF08339.13          5     0.90    131   0.59   0.60   0.49   0.05\n",
            "16199  RT_RNaseH            PF17917.4         118     3.68    105   0.59   0.56   0.52   0.03\n",
            "16200  RT_RNaseH_2          PF17919.4         174     4.80    100   0.59   0.57   0.56   0.03\n",
            "16201  Rua1_C               PF14616.9          32     1.93    122   0.59   0.48   0.43   0.06\n",
            "16202  Rubella_Capsid       PF05750.14          1     0.38    300   0.58   0.58   0.51   0.02\n",
            "16203  Rubella_E1           PF05748.14          1     0.36    496   0.59   0.55   0.51   0.02\n",
            "16204  Rubella_E2           PF05749.14          1     0.36    267   0.59   0.54   0.52   0.03\n",
            "16205  Rubis-subs-bind      PF09273.14        117     8.04    130   0.59   0.63   0.43   0.04\n",
            "16206  RuBisCo_chap_C       PF18087.4          85     2.12    138   0.59   0.60   0.49   0.03\n",
            "16207  RuBisCO_large        PF00016.23         16     1.16    299   0.59   0.58   0.52   0.02\n",
            "16208  RuBisCO_large_N      PF02788.19         37     1.36    120   0.59   0.58   0.53   0.02\n",
            "16209  RuBisCO_small        PF00101.23         82     1.33     97   0.59   0.52   0.53   0.03\n",
            "16210  Rubi_NSP_C           PF12601.11          1     0.55     66   0.85   0.87   0.79   0.04\n",
            "16211  Rubredoxin           PF00301.23         24     1.72     47   1.17   0.96   1.13   0.15\n",
            "16212  Rubredoxin_2         PF18073.4          32    10.25     28   1.92   1.50   1.91   0.45\n",
            "16213  Rubredoxin_C         PF18267.4          32     3.52     70   0.80   0.84   0.76   0.03\n",
            "16214  Rubrerythrin         PF02915.20         38     4.52    137   0.59   0.59   0.54   0.07\n",
            "16215  RUN                  PF02759.22         35     3.90    141   0.59   0.59   0.49   0.03\n",
            "16216  Runt                 PF00853.22         21     0.76    129   0.59   0.60   0.53   0.02\n",
            "16217  RunxI                PF08504.14          5     0.72     99   0.59   0.56   0.51   0.04\n",
            "16218  RusA                 PF05866.14         69     4.20    126   0.59   0.57   0.48   0.04\n",
            "16219  RuvA_C               PF07499.16         79    25.03     47   1.17   1.32   1.12   0.11\n",
            "16220  RuvA_N               PF01330.24        730     8.21     60   0.93   0.93   0.77   0.05\n",
            "16221  RuvB_C               PF05491.16         28     2.60     72   0.78   0.82   0.75   0.03\n",
            "16222  RuvB_N               PF05496.15         91     0.66    159   0.59   0.62   0.53   0.02\n",
            "16223  RuvC                 PF02075.20          5     0.83    148   0.59   0.62   0.53   0.03\n",
            "16224  RuvC_1               PF18516.4          10     0.89    244   0.59   0.58   0.50   0.03\n",
            "16225  RuvC_III             PF18541.4          55     2.72    180   0.59   0.56   0.48   0.05\n",
            "16226  RuvX                 PF03652.18        111     2.60    134   0.59   0.62   0.53   0.02\n",
            "16227  Rv0078B              PF18993.3           8     2.21     63   0.89   0.88   0.85   0.05\n",
            "16228  Rv2175c_C            PF18367.4          36     2.11     56   0.99   1.03   0.94   0.02\n",
            "16229  Rv2179c-like         PF16473.8          27     2.75    166   0.59   0.54   0.39   0.03\n",
            "16230  Rv2632c-like         PF08962.14         33     1.75     82   0.69   0.74   0.64   0.05\n",
            "16231  Rv2949c-like         PF01947.19         20     1.37    149   0.59   0.57   0.53   0.02\n",
            "16232  rve                  PF00665.29         69     7.80    102   0.59   0.54   0.49   0.02\n",
            "16233  rve_2                PF13333.9          13     2.85     52   1.07   0.98   0.97   0.09\n",
            "16234  rve_3                PF13683.9          51     4.90     67   0.84   0.77   0.81   0.04\n",
            "16235  RVP                  PF00077.23          8     1.83    101   0.59   0.59   0.50   0.01\n",
            "16236  RVP_2                PF08284.14          7     0.97    136   0.59   0.59   0.53   0.03\n",
            "16237  RVT_1                PF00078.30         69     6.27    222   0.59   0.57   0.47   0.02\n",
            "16238  RVT_2                PF07727.17          3     0.69    243   0.59   0.58   0.52   0.02\n",
            "16239  RVT_3                PF13456.9          24     4.35    124   0.59   0.59   0.55   0.02\n",
            "16240  RVT_connect          PF06815.16          2     0.43    102   0.59   0.51   0.51   0.05\n",
            "16241  RVT_N                PF13655.9          95     1.62     83   0.68   0.67   0.63   0.07\n",
            "16242  RVT_thumb            PF06817.17         15     2.14     66   0.85   0.86   0.81   0.05\n",
            "16243  RWD                  PF05773.25        114     6.73    116   0.59   0.58   0.44   0.04\n",
            "16244  RWP-RK               PF02042.18         80     4.35     49   1.13   1.09   1.08   0.06\n",
            "16245  RXLR                 PF16810.8          18     3.25    141   0.59   0.58   0.48   0.05\n",
            "16246  RXLR_WY              PF18634.4          46     6.03     51   1.09   1.10   1.05   0.07\n",
            "16247  RXT2_N               PF08595.14         58     2.41    153   0.59   0.60   0.49   0.04\n",
            "16248  Rxt3                 PF08642.13         73     1.54    114   0.59   0.56   0.48   0.02\n",
            "16249  Rx_N                 PF18052.4         457     9.88     93   0.61   0.63   0.52   0.06\n",
            "16250  RYDR_ITPR            PF01365.24         52     2.55    201   0.59   0.59   0.49   0.04\n",
            "16251  RyR                  PF02026.19        103     2.13     91   0.63   0.58   0.59   0.03\n",
            "16252  Rz1                  PF06085.14          6     2.25     41   1.34   1.21   1.25   0.18\n",
            "16253  R_equi_Vir           PF05526.14          8     1.17    177   0.59   0.60   0.52   0.02\n",
            "16254  S-AdoMet_synt_C      PF02773.19        302     1.03    138   0.59   0.60   0.52   0.02\n",
            "16255  S-AdoMet_synt_M      PF02772.19        685     1.64    120   0.59   0.58   0.46   0.02\n",
            "16256  S-AdoMet_synt_N      PF00438.23        404     1.22     99   0.59   0.59   0.51   0.03\n",
            "16257  S-antigen            PF05756.14          1     0.41     92   0.63   0.62   0.55   0.05\n",
            "16258  S-layer              PF07752.14         44     1.70    259   0.59   0.58   0.51   0.04\n",
            "16259  S-l_SbsC_C           PF18316.4          21     4.41     86   0.66   0.68   0.56   0.06\n",
            "16260  S-methyl_trans       PF02574.19        664     4.96    274   0.59   0.58   0.35   0.02\n",
            "16261  S1                   PF00575.26         31    10.18     75   0.75   0.79   0.69   0.06\n",
            "16262  S1-like              PF14444.9          13     1.26     58   0.96   0.94   0.91   0.03\n",
            "16263  S1-P1_nuclease       PF02265.19         45     2.35    255   0.59   0.55   0.48   0.03\n",
            "16264  S100PBPR             PF15427.9          12     0.76    390   0.59   0.58   0.52   0.03\n",
            "16265  S10_plectin          PF03501.18         69     1.50     93   0.61   0.58   0.56   0.03\n",
            "16266  S1FA                 PF04689.16          6     0.73     66   0.85   0.89   0.80   0.05\n",
            "16267  S1_2                 PF13509.9          51     6.29     61   0.92   0.95   0.88   0.04\n",
            "16268  S36_mt               PF10937.11         56     2.01    115   0.59   0.58   0.48   0.03\n",
            "16269  S4                   PF01479.28        116    48.99     48   1.15   1.20   1.12   0.07\n",
            "16270  s48_45               PF07422.16         74     3.34    108   0.59   0.49   0.40   0.08\n",
            "16271  S4_2                 PF13275.9         105     4.08     65   0.86   0.92   0.83   0.06\n",
            "16272  S6OS1                PF15676.8           8     0.93    558   0.59   0.57   0.52   0.04\n",
            "16273  S6PP                 PF05116.16         18     1.70    247   0.59   0.59   0.53   0.02\n",
            "16274  S6PP_C               PF08472.13         26     1.22    133   0.59   0.55   0.53   0.02\n",
            "16275  S8_pro-domain        PF16470.8         121     4.88     79   0.72   0.69   0.60   0.06\n",
            "16276  SAA                  PF00277.21         22     0.92    102   0.59   0.56   0.52   0.05\n",
            "16277  SAB                  PF04382.16         15     3.41     50   1.11   1.13   1.05   0.12\n",
            "16278  SabA_adhesion        PF18304.4           4     0.96    299   0.59   0.58   0.51   0.04\n",
            "16279  SAC3                 PF12209.11         26     3.97     77   0.73   0.74   0.70   0.07\n",
            "16280  SAC3_GANP            PF03399.19        118     2.93    289   0.59   0.56   0.51   0.04\n",
            "16281  Saccharop_dh_N       PF04455.15         77     1.48    100   0.59   0.61   0.53   0.03\n",
            "16282  Sacchrp_dh_C         PF16653.8         333     7.11    267   0.59   0.58   0.32   0.01\n",
            "16283  Sacchrp_dh_NADP      PF03435.21         81     6.16    130   0.59   0.61   0.50   0.02\n",
            "16284  Sad1_UNC             PF07738.16         23     2.80    131   0.59   0.54   0.49   0.02\n",
            "16285  SAD_SRA              PF02182.20        161     2.40    153   0.59   0.56   0.45   0.02\n",
            "16286  SAE2                 PF08573.13         65     2.30    109   0.59   0.51   0.47   0.06\n",
            "16287  SAF                  PF08666.15        125    35.22     63   0.89   0.95   0.79   0.04\n",
            "16288  Saf-Nte_pilin        PF09460.13          1     0.40    144   0.59   0.61   0.52   0.02\n",
            "16289  Saf4_Yju2            PF04502.16         78     2.72    331   0.59   0.56   0.44   0.06\n",
            "16290  SafA                 PF17073.8           2     0.64     66   0.85   0.88   0.78   0.07\n",
            "16291  Saf_2TM              PF18303.4           9     2.53    152   0.59   0.61   0.53   0.03\n",
            "16292  SAG                  PF04092.16         89     3.71    127   0.59   0.54   0.34   0.08\n",
            "16293  SAGA-Tad1            PF12767.10        120     4.30    235   0.59   0.60   0.44   0.03\n",
            "16294  SAICAR_synt          PF01259.21        584     2.29    252   0.59   0.59   0.49   0.02\n",
            "16295  Salp15               PF12115.11         44     3.50    118   0.59   0.50   0.42   0.07\n",
            "16296  Salt_tol_Pase        PF09506.13         14     0.76    391   0.59   0.60   0.53   0.01\n",
            "16297  Salyut               PF19227.3           7     5.29     51   1.09   1.02   1.05   0.11\n",
            "16298  SAM35                PF10806.11         21     1.36    127   0.59   0.58   0.50   0.03\n",
            "16299  Sam68-YY             PF16568.8          26     1.84     55   1.01   0.90   0.94   0.34\n",
            "16300  SAMP                 PF05924.14         15    15.00     22   2.09   2.10   2.08   0.27\n",
            "16301  SAM_1                PF00536.33         55    11.07     64   0.88   0.86   0.83   0.04\n",
            "16302  SAM_2                PF07647.20         46    16.60     66   0.85   0.83   0.81   0.03\n",
            "16303  SAM_3                PF18016.4          17     3.76     65   0.86   0.87   0.83   0.03\n",
            "16304  SAM_4                PF18017.4           2     0.51     84   0.68   0.67   0.61   0.03\n",
            "16305  SAM_decarbox         PF01536.19         92     1.55    354   0.59   0.56   0.49   0.03\n",
            "16306  SAM_DrpA             PF18255.4           8     1.39     62   0.90   0.91   0.86   0.06\n",
            "16307  SAM_Exu              PF18609.4          18     1.77     73   0.77   0.78   0.73   0.04\n",
            "16308  SAM_HAT_C            PF20257.1         291     5.89     85   0.67   0.68   0.55   0.02\n",
            "16309  SAM_HAT_N            PF01887.19        291     2.46    147   0.59   0.60   0.50   0.02\n",
            "16310  SAM_KSR1             PF13543.9           8     1.93     70   0.80   0.83   0.74   0.06\n",
            "16311  SAM_KSR1_N           PF20406.1           8     1.71     54   1.03   1.05   0.99   0.10\n",
            "16312  SAM_LFY              PF01698.19          7     0.71     80   0.71   0.73   0.65   0.03\n",
            "16313  SAM_MT               PF04445.16         33     1.08    230   0.59   0.60   0.53   0.02\n",
            "16314  SAM_PNT              PF02198.19         14     2.20     83   0.68   0.62   0.63   0.04\n",
            "16315  SAM_Ste50p           PF09235.13          3     0.88     75   0.75   0.75   0.69   0.06\n",
            "16316  SANBR_BTB            PF11822.11          7     0.76     97   0.59   0.55   0.50   0.04\n",
            "16317  SAND                 PF01342.24         58     2.61     77   0.73   0.66   0.67   0.05\n",
            "16318  SANTA                PF09133.13         30     1.95     93   0.61   0.57   0.53   0.03\n",
            "16319  SANT_DAMP1_like      PF16282.8         102     1.57     80   0.71   0.62   0.63   0.08\n",
            "16320  SAP                  PF02037.30        145   145.00     35   1.44   1.54   1.43   0.12\n",
            "16321  SAP130_C             PF16014.8           7     0.57    406   0.59   0.59   0.51   0.02\n",
            "16322  SAP18                PF06487.15        111     2.70    141   0.59   0.58   0.46   0.02\n",
            "16323  SAP25                PF15476.9           9     0.65    202   0.59   0.57   0.52   0.02\n",
            "16324  SAP30_Sin3_bdg       PF13867.9          48     5.14     54   1.03   1.01   0.99   0.08\n",
            "16325  SapA                 PF02199.18         87     7.34     33   1.64   1.31   1.63   0.35\n",
            "16326  SapB_1               PF05184.18        417   417.00     38   1.41   1.33   1.40   0.20\n",
            "16327  SapB_2               PF03489.20        461   461.00     34   1.56   1.41   1.54   0.20\n",
            "16328  SapC                 PF07277.14        103     3.78    221   0.59   0.60   0.53   0.02\n",
            "16329  SAPI                 PF16560.8           1     0.39    213   0.59   0.58   0.52   0.02\n",
            "16330  SAPIS-gp6            PF16722.8           3     0.98     72   0.78   0.79   0.72   0.10\n",
            "16331  SAPS                 PF04499.18         49     3.17    490   0.59   0.59   0.49   0.03\n",
            "16332  SAP_new25            PF18953.3         124     7.57     51   1.09   1.05   1.07   0.07\n",
            "16333  Sar8_2               PF03058.17          4     0.64     93   0.61   0.58   0.52   0.07\n",
            "16334  SARA                 PF11409.11          7     1.69     37   1.47   1.39   1.40   0.27\n",
            "16335  SARAF                PF06682.15         45     1.71    340   0.59   0.55   0.48   0.03\n",
            "16336  Sarcoglycan_1        PF04790.16         50     3.32    260   0.59   0.61   0.49   0.03\n",
            "16337  Sarcoglycan_2        PF05510.16          5     0.75    410   0.59   0.56   0.50   0.01\n",
            "16338  Sarcolipin           PF05366.14          3     1.61     31   1.74   1.70   1.71   0.26\n",
            "16339  SARG                 PF15385.9          21     1.44    568   0.59   0.61   0.50   0.05\n",
            "16340  SARS_3b              PF12383.11          1     0.42    153   0.59   0.58   0.52   0.06\n",
            "16341  SART-1               PF03343.16        102     3.18    585   0.59   0.60   0.41   0.10\n",
            "16342  SAS-6_N              PF16531.8          79     3.23     97   0.59   0.57   0.48   0.06\n",
            "16343  Sas10                PF09368.13         95     2.99     74   0.76   0.77   0.71   0.12\n",
            "16344  Sas10_Utp3           PF04000.18        310    11.58     86   0.66   0.71   0.47   0.06\n",
            "16345  SAS4                 PF15460.9          59     2.45     99   0.59   0.57   0.54   0.08\n",
            "16346  Sas6_CC              PF18594.4          18     5.60     30   1.80   1.84   1.78   0.29\n",
            "16347  SASA                 PF03629.21         74     1.88    226   0.59   0.57   0.47   0.02\n",
            "16348  SasG_E               PF17041.8           4     1.00     48   1.15   1.14   1.10   0.13\n",
            "16349  SASP                 PF00269.23        157     3.68     60   0.93   0.96   0.83   0.06\n",
            "16350  SASP_gamma           PF04259.17          3     0.84     84   0.68   0.69   0.60   0.13\n",
            "16351  SASRP1               PF15160.9           6     0.52    238   0.59   0.57   0.52   0.01\n",
            "16352  SAT                  PF16073.8         105     4.51    240   0.59   0.60   0.52   0.01\n",
            "16353  SATase_N             PF06426.17        130     2.11    105   0.59   0.60   0.55   0.03\n",
            "16354  SatD                 PF16264.8           4     0.60    211   0.59   0.59   0.53   0.02\n",
            "16355  SatRNA_48            PF17485.5           4     0.45    299   0.59   0.57   0.52   0.02\n",
            "16356  SAUGI                PF06106.14          1     0.37    112   0.59   0.56   0.51   0.03\n",
            "16357  SAVED                PF18145.4          90     8.67    189   0.59   0.58   0.49   0.01\n",
            "16358  Saw1                 PF11561.11         17     1.74    244   0.59   0.60   0.48   0.04\n",
            "16359  SAWADEE              PF16719.8          33     1.56    141   0.59   0.52   0.51   0.05\n",
            "16360  Say1_Mug180          PF10340.12          5     1.05    374   0.59   0.56   0.51   0.02\n",
            "16361  SAYSvFN              PF10260.12         41     2.70     70   0.80   0.78   0.76   0.05\n",
            "16362  Sa_NUDIX             PF18167.4          15     1.53    199   0.59   0.58   0.53   0.03\n",
            "16363  SBBP                 PF06739.14         52    46.79     38   1.44   1.41   1.38   0.17\n",
            "16364  SbcC_Walker_B        PF13558.9          91     4.81     90   0.63   0.66   0.56   0.03\n",
            "16365  SbcD_C               PF12320.11        216    11.99     92   0.62   0.65   0.41   0.05\n",
            "16366  SBD                  PF17882.4          57     3.75     77   0.73   0.68   0.63   0.06\n",
            "16367  SBDS                 PF01172.21        237     3.25     88   0.65   0.65   0.55   0.04\n",
            "16368  SBDS_C               PF20268.1         139    11.96     71   0.79   0.82   0.67   0.04\n",
            "16369  SBDS_domain_II       PF09377.13        185     2.66     82   0.69   0.69   0.52   0.05\n",
            "16370  SBD_N                PF07005.14        206     4.17    227   0.59   0.62   0.45   0.02\n",
            "16371  SBE2                 PF17076.8           3     0.59    820   0.59   0.58   0.51   0.02\n",
            "16372  SBF                  PF01758.19         17     2.80    195   0.59   0.61   0.53   0.07\n",
            "16373  SBF2                 PF12335.11         24     1.02    227   0.59   0.56   0.53   0.04\n",
            "16374  SBF_like             PF13593.9          39     2.64    313   0.59   0.60   0.52   0.06\n",
            "16375  Sbi-IV               PF11621.11          1     0.61     69   0.81   0.85   0.75   0.09\n",
            "16376  SbmA_BacA            PF05992.15          8     0.67    315   0.59   0.57   0.53   0.04\n",
            "16377  SBP                  PF03110.17         75     1.81     75   0.75   0.62   0.69   0.15\n",
            "16378  SBP56                PF05694.14         20     0.80    453   0.59   0.55   0.51   0.01\n",
            "16379  SBP_bac_1            PF01547.28        129    39.74    315   0.59   0.59   0.41   0.04\n",
            "16380  SBP_bac_10           PF07596.14         33     6.01    274   0.59   0.55   0.49   0.02\n",
            "16381  SBP_bac_11           PF13531.9         115    10.64    230   0.59   0.62   0.46   0.02\n",
            "16382  SBP_bac_3            PF00497.23        241     9.01    225   0.59   0.61   0.50   0.02\n",
            "16383  SBP_bac_5            PF00496.25        151    12.01    377   0.59   0.57   0.48   0.02\n",
            "16384  SBP_bac_6            PF13343.9          50     7.86    244   0.59   0.57   0.49   0.02\n",
            "16385  SBP_bac_8            PF13416.9          57    12.03    284   0.59   0.57   0.40   0.04\n",
            "16386  SbsC_C               PF18058.4          12     2.05    132   0.59   0.60   0.54   0.07\n",
            "16387  Sbt_1                PF05982.15         91     1.51    308   0.59   0.64   0.53   0.05\n",
            "16388  SCA7                 PF08313.15         38     2.02     68   0.83   0.76   0.79   0.09\n",
            "16389  SCAB-ABD             PF16711.8          10     2.00     43   1.28   1.39   1.23   0.14\n",
            "16390  SCAB-Ig              PF16709.8          15     0.74     98   0.59   0.60   0.53   0.01\n",
            "16391  SCAB-PH              PF17684.4          15     0.60    108   0.59   0.57   0.52   0.02\n",
            "16392  SCAB_CC              PF16712.8           8     0.77    168   0.59   0.63   0.53   0.07\n",
            "16393  sCache_2             PF17200.7          22     4.29    156   0.59   0.56   0.53   0.03\n",
            "16394  sCache_3_2           PF17203.7          16     4.82    137   0.59   0.62   0.49   0.02\n",
            "16395  sCache_3_3           PF17202.7         269     4.54    107   0.59   0.60   0.48   0.03\n",
            "16396  sCache_4             PF09984.12         27     2.00    148   0.59   0.61   0.53   0.03\n",
            "16397  sCache_like          PF16736.8           9     2.44    117   0.59   0.60   0.53   0.04\n",
            "16398  Scaffold             PF20142.2         323     9.00    140   0.59   0.60   0.49   0.04\n",
            "16399  Scaffolding_pro      PF11418.11          2     0.62     99   0.59   0.61   0.52   0.04\n",
            "16400  SCAI                 PF12070.11         32     1.14    538   0.59   0.57   0.50   0.03\n",
            "16401  SCAMP                PF04144.16        122     3.62    173   0.59   0.53   0.50   0.08\n",
            "16402  SCAN                 PF02023.20        173     2.80     90   0.63   0.60   0.61   0.06\n",
            "16403  SCAPER_N             PF16501.8          12     0.76     98   0.59   0.55   0.53   0.04\n",
            "16404  ScdA_N               PF04405.17         47     4.05     56   0.99   0.98   0.97   0.05\n",
            "16405  SCF                  PF02404.18          6     0.59    275   0.59   0.57   0.52   0.02\n",
            "16406  SCFA_trans           PF02667.17          4     0.64    453   0.59   0.60   0.52   0.04\n",
            "16407  ScfRs                PF09856.12        100     1.34    157   0.59   0.55   0.54   0.02\n",
            "16408  SCGB3A               PF20490.1          31     1.91     68   0.83   0.90   0.72   0.04\n",
            "16409  SchA_CurD            PF04486.15         12     0.96    119   0.59   0.60   0.53   0.02\n",
            "16410  SCHIP-1              PF10148.12          6     0.78    248   0.59   0.60   0.51   0.04\n",
            "16411  SCIFF                PF13165.9          36     1.58     44   1.25   1.09   1.20   0.29\n",
            "16412  SCIMP                PF15050.9          10     0.67    132   0.59   0.55   0.52   0.02\n",
            "16413  Sclerostin           PF05463.14          4     0.63    198   0.59   0.55   0.51   0.03\n",
            "16414  Scm3                 PF10384.12         73     9.86     54   1.03   1.03   0.96   0.08\n",
            "16415  SCNM1_acidic         PF15805.8          12     2.25     47   1.17   1.10   1.15   0.11\n",
            "16416  SCO1-SenC            PF02630.17         55     2.77    134   0.59   0.56   0.54   0.02\n",
            "16417  SCP-1                PF05483.15          2     0.47    786   0.59   0.60   0.52   0.06\n",
            "16418  SCP2                 PF02036.20        106     8.17    101   0.59   0.62   0.49   0.02\n",
            "16419  SCP2_2               PF13530.9         166     7.84    104   0.59   0.62   0.44   0.01\n",
            "16420  SCPU                 PF05229.18         54     3.83    139   0.59   0.57   0.48   0.05\n",
            "16421  SCP_3                PF17844.4          92     0.99     93   0.61   0.64   0.56   0.02\n",
            "16422  Scramblase           PF03803.18         10     1.40    221   0.59   0.56   0.51   0.02\n",
            "16423  SCRE                 PF15162.9          11     0.71    162   0.59   0.59   0.53   0.03\n",
            "16424  SCRG1                PF15224.9           6     0.53     76   0.75   0.65   0.68   0.05\n",
            "16425  SCRL                 PF06876.15         43     3.29     67   0.84   0.65   0.73   0.20\n",
            "16426  Scs3p                PF10261.12         74     1.61    246   0.59   0.53   0.46   0.04\n",
            "16427  ScsC_N               PF18312.4          51    23.66     34   1.59   1.67   1.59   0.21\n",
            "16428  SCVP                 PF17619.5          47     6.69     97   0.59   0.55   0.52   0.05\n",
            "16429  Scytalone_dh         PF02982.17          3     0.45    160   0.59   0.54   0.52   0.02\n",
            "16430  Sda                  PF08970.13         18     4.12     45   1.22   1.25   1.19   0.11\n",
            "16431  SDA1                 PF05285.15        121     2.95    343   0.59   0.61   0.46   0.10\n",
            "16432  SDE2_2C              PF13297.9          47     2.33     82   0.69   0.74   0.64   0.05\n",
            "16433  Sde2_N_Ubi           PF13019.9          16     1.37    164   0.59   0.63   0.51   0.03\n",
            "16434  SDF                  PF00375.21         81     4.12    391   0.59   0.63   0.54   0.07\n",
            "16435  SDG2_C               PF19633.2          17     0.70    420   0.59   0.56   0.52   0.01\n",
            "16436  Sdh5                 PF03937.19        244     5.16     75   0.75   0.74   0.70   0.04\n",
            "16437  SDH5_plant           PF14290.9           8     0.65    237   0.59   0.60   0.52   0.01\n",
            "16438  SDH_alpha            PF03313.18        521     3.12    260   0.59   0.61   0.45   0.04\n",
            "16439  SDH_beta             PF03315.18        292     1.44    155   0.59   0.60   0.50   0.01\n",
            "16440  SDH_C                PF18317.4         181    83.57     31   1.74   1.72   1.72   0.15\n",
            "16441  Sdh_cyt              PF01127.25         59     6.67    121   0.59   0.58   0.47   0.07\n",
            "16442  SDH_sah              PF01972.19          4     0.54    286   0.59   0.60   0.52   0.02\n",
            "16443  SdiA-regulated       PF06977.14         18     1.51    249   0.59   0.61   0.53   0.02\n",
            "16444  SdpA                 PF17418.5           7     1.08    142   0.59   0.54   0.53   0.02\n",
            "16445  SdpI                 PF13630.9         211    10.18     68   0.83   0.80   0.70   0.09\n",
            "16446  SDP_N                PF12278.11          9     1.00    140   0.59   0.58   0.52   0.07\n",
            "16447  SdrD_B               PF17210.6          43     1.49    112   0.59   0.59   0.51   0.04\n",
            "16448  SdrG_C_C             PF10425.12         10     2.70    156   0.59   0.56   0.50   0.07\n",
            "16449  Sds3                 PF08598.14         59     4.57    218   0.59   0.59   0.43   0.07\n",
            "16450  SE                   PF08491.13         12     0.92    276   0.59   0.59   0.53   0.01\n",
            "16451  Se-cys_synth_N       PF12390.11        164    25.36     40   1.37   1.47   1.32   0.15\n",
            "16452  SEA                  PF01390.23        183    10.13    106   0.59   0.59   0.52   0.04\n",
            "16453  Seadorna_Vp10        PF07322.14          1     0.39    241   0.59   0.59   0.52   0.02\n",
            "16454  Seadorna_VP6         PF07407.14          1     0.38    420   0.59   0.58   0.51   0.01\n",
            "16455  Seadorna_VP7         PF07387.14          5     0.85    308   0.59   0.58   0.51   0.03\n",
            "16456  Sec-ASP3             PF15432.9          14     1.47    126   0.59   0.55   0.55   0.03\n",
            "16457  SEC-C                PF02810.18        165   121.48     19   2.77   2.35   2.74   0.77\n",
            "16458  Sec1                 PF00995.26        585     9.99    577   0.59   0.59   0.38   0.03\n",
            "16459  Sec10                PF07393.14         26     3.70    711   0.59   0.59   0.51   0.03\n",
            "16460  Sec15                PF04091.15        103     3.35    316   0.59   0.58   0.51   0.04\n",
            "16461  Sec16                PF12932.10         93     2.82    118   0.59   0.60   0.44   0.01\n",
            "16462  Sec16_C              PF12931.10         93     2.83    302   0.59   0.59   0.49   0.02\n",
            "16463  Sec16_N              PF12935.10         15     2.46    238   0.59   0.59   0.48   0.09\n",
            "16464  Sec20                PF03908.16          6     1.39     92   0.62   0.64   0.58   0.05\n",
            "16465  Sec23_BS             PF08033.15        138     4.33     96   0.60   0.57   0.50   0.03\n",
            "16466  Sec23_helical        PF04815.18        332     6.36    102   0.59   0.60   0.50   0.03\n",
            "16467  Sec23_trunk          PF04811.18         21     1.98    243   0.59   0.59   0.51   0.02\n",
            "16468  Sec2p                PF06428.14         42     1.96     92   0.62   0.65   0.58   0.12\n",
            "16469  Sec3-PIP2_bind       PF15277.9         105     3.93     90   0.63   0.60   0.54   0.04\n",
            "16470  Sec31                PF11549.11         12     1.77     50   1.11   1.09   1.06   0.09\n",
            "16471  Sec34                PF04136.18         97     2.82    149   0.59   0.58   0.55   0.06\n",
            "16472  Sec39                PF08314.14         66     3.70    750   0.59   0.59   0.47   0.03\n",
            "16473  Sec3_C               PF09763.12         20     3.81    707   0.59   0.59   0.52   0.04\n",
            "16474  Sec3_C_2             PF15278.9           2     0.74     86   0.66   0.67   0.60   0.04\n",
            "16475  Sec5                 PF15469.9          45     5.57    825   0.59   0.59   0.47   0.04\n",
            "16476  Sec6                 PF06046.16         32     4.71    576   0.59   0.57   0.51   0.03\n",
            "16477  Sec61_beta           PF03911.19        109    18.10     41   1.34   1.42   1.32   0.12\n",
            "16478  Sec62                PF03839.19         70     1.33    192   0.59   0.54   0.52   0.03\n",
            "16479  Sec63                PF02889.19         84     3.62    259   0.59   0.58   0.47   0.03\n",
            "16480  Sec66                PF09802.12         60     1.65    180   0.59   0.58   0.54   0.03\n",
            "16481  Sec7                 PF01369.23        370     3.35    183   0.59   0.57   0.47   0.04\n",
            "16482  Sec7_N               PF12783.10        215     4.47    158   0.59   0.59   0.49   0.05\n",
            "16483  Sec8_exocyst         PF04048.17         16     1.68    142   0.59   0.60   0.55   0.05\n",
            "16484  Secapin              PF17521.5           1     0.78     45   1.22   1.19   1.17   0.11\n",
            "16485  SecA_DEAD            PF07517.17         84     1.45    290   0.59   0.60   0.49   0.03\n",
            "16486  SecA_PP_bind         PF01043.23        541     1.57    110   0.59   0.58   0.49   0.04\n",
            "16487  SecA_SW              PF07516.16        736     2.39    199   0.59   0.57   0.46   0.05\n",
            "16488  SecB                 PF02556.17        157     1.46    143   0.59   0.58   0.53   0.02\n",
            "16489  SecD-TM1             PF13721.9         124     2.54    103   0.59   0.60   0.54   0.03\n",
            "16490  SecD_SecF            PF02355.19         17     1.73    189   0.59   0.63   0.54   0.06\n",
            "16491  SecE                 PF00584.23        695    19.27     55   1.01   1.03   1.00   0.13\n",
            "16492  SecG                 PF03840.17        633     9.10     71   0.79   0.88   0.67   0.14\n",
            "16493  SecIII_SopE_N        PF05364.17          1     0.54     74   0.76   0.75   0.70   0.04\n",
            "16494  SecM                 PF06558.15         12     1.08    147   0.59   0.59   0.50   0.03\n",
            "16495  SecM_small           PF10818.11          7     1.27     96   0.60   0.58   0.50   0.04\n",
            "16496  Secretin             PF00263.24         87     4.01    168   0.59   0.63   0.52   0.04\n",
            "16497  Secretin_N           PF03958.20        227    21.08     82   0.69   0.76   0.59   0.04\n",
            "16498  Secretin_N_2         PF07655.16         55     3.90     95   0.60   0.61   0.50   0.08\n",
            "16499  Secretogranin_V      PF05281.14          7     0.59    227   0.59   0.55   0.49   0.02\n",
            "16500  Securin              PF04856.16          7     2.29    232   0.59   0.61   0.42   0.05\n",
            "16501  SecY                 PF00344.23        746     3.12    308   0.59   0.61   0.43   0.06\n",
            "16502  Sec_GG               PF07549.17        358   358.00     28   1.63   1.71   1.64   0.14\n",
            "16503  Sedlin_N             PF04628.16         23     1.78    131   0.59   0.57   0.49   0.03\n",
            "16504  SEEEED               PF14797.9          16     1.13    111   0.59   0.64   0.51   0.29\n",
            "16505  SEEK1                PF15357.9           2     0.44    149   0.59   0.58   0.52   0.02\n",
            "16506  SEF14_adhesin        PF06443.14          2     0.57    165   0.59   0.60   0.52   0.03\n",
            "16507  SEFIR                PF08357.14         15     2.76    150   0.59   0.56   0.50   0.03\n",
            "16508  SegE_GIY-YIG         PF19835.2          30     1.32    119   0.59   0.53   0.50   0.05\n",
            "16509  Seipin               PF06775.17        117     3.95    198   0.59   0.57   0.46   0.03\n",
            "16510  Sel1                 PF08238.15        167   167.00     38   1.21   1.18   1.10   0.11\n",
            "16511  SelA                 PF03841.16         10     0.79    367   0.59   0.63   0.53   0.02\n",
            "16512  SelB-wing_1          PF09105.13          1     0.68     61   0.92   0.97   0.86   0.04\n",
            "16513  SelB-wing_2          PF09106.14        131    14.32     57   0.98   1.03   0.92   0.05\n",
            "16514  SelB-wing_3          PF09107.14        164    11.44     47   1.17   1.22   1.14   0.07\n",
            "16515  SeleniumBinding      PF11524.11          3     0.72     82   0.69   0.71   0.62   0.04\n",
            "16516  Selenoprotein_S      PF06936.14          8     0.83    192   0.59   0.57   0.52   0.03\n",
            "16517  Self-incomp_S1       PF05938.14        184     3.27    104   0.59   0.47   0.47   0.06\n",
            "16518  SelK_SelG            PF10961.11         36     2.73     86   0.66   0.65   0.52   0.04\n",
            "16519  SelO                 PF02696.17         52     1.14    468   0.59   0.57   0.51   0.01\n",
            "16520  SelP_C               PF04593.17          3     0.45    133   0.59   0.52   0.52   0.08\n",
            "16521  SelP_N               PF04592.17          8     0.84    233   0.59   0.54   0.51   0.03\n",
            "16522  SelR                 PF01641.21        805     1.35    121   0.59   0.53   0.53   0.03\n",
            "16523  Sel_put              PF04328.16         74     3.96     62   0.90   0.77   0.84   0.08\n",
            "16524  Sema                 PF01403.22        191     4.77    180   0.59   0.57   0.48   0.02\n",
            "16525  Sema4F_C             PF19428.2           5     0.65    197   0.59   0.59   0.52   0.01\n",
            "16526  Semenogelin          PF05474.14          1     0.41    582   0.59   0.57   0.52   0.04\n",
            "16527  Semialdhyde_dh       PF01118.27        149     4.22    121   0.59   0.62   0.51   0.01\n",
            "16528  Semialdhyde_dhC      PF02774.21         32     3.92    184   0.59   0.60   0.47   0.01\n",
            "16529  Sen15                PF09631.13         41     2.46    101   0.59   0.58   0.47   0.02\n",
            "16530  SEN1_N               PF12726.10         92     6.07    749   0.59   0.57   0.49   0.03\n",
            "16531  Senescence           PF06911.15        106     5.72    187   0.59   0.63   0.51   0.06\n",
            "16532  Senescence_reg       PF04520.16         82     2.24    167   0.59   0.59   0.40   0.04\n",
            "16533  SENP3_5_N            PF19722.2          10     0.96    575   0.59   0.57   0.52   0.03\n",
            "16534  Sensor               PF13796.9          85     4.78    162   0.59   0.65   0.48   0.11\n",
            "16535  Sensor_TM1           PF13755.9          22     2.01     68   0.83   0.91   0.79   0.08\n",
            "16536  SEO_C                PF14577.9          27     1.13    232   0.59   0.53   0.53   0.03\n",
            "16537  SEO_N                PF14576.9          26     1.60    287   0.59   0.57   0.53   0.04\n",
            "16538  SEP                  PF08059.16        151     3.12     74   0.76   0.74   0.67   0.02\n",
            "16539  Sep15_SelM           PF08806.14         32     3.82     75   0.75   0.74   0.71   0.03\n",
            "16540  SepA                 PF17080.8           1     0.36    144   0.59   0.56   0.51   0.08\n",
            "16541  SepF                 PF04472.15        299     8.21     72   0.78   0.82   0.73   0.04\n",
            "16542  SepQ                 PF06622.14          2     0.40    305   0.59   0.57   0.52   0.02\n",
            "16543  SepRS_C              PF18006.4          23     6.85     31   1.74   1.74   1.66   0.15\n",
            "16544  SepSecS              PF05889.16          5     1.09    389   0.59   0.60   0.51   0.01\n",
            "16545  Septin               PF00735.21         13     1.00    281   0.59   0.58   0.53   0.02\n",
            "16546  Septum_form          PF13845.9          56     2.33    226   0.59   0.52   0.52   0.05\n",
            "16547  SepZ                 PF06066.14          4     0.54     99   0.59   0.66   0.52   0.05\n",
            "16548  SeqA                 PF03925.16         30     1.09    111   0.59   0.57   0.54   0.03\n",
            "16549  SeqA_N               PF17206.6           8     2.04     36   1.51   1.56   1.48   0.10\n",
            "16550  Serendipity_A        PF05482.15          7     0.65    541   0.59   0.60   0.52   0.03\n",
            "16551  Serglycin            PF04360.15         15     0.87    149   0.59   0.58   0.52   0.01\n",
            "16552  SerH                 PF06873.14          9     0.94    417   0.59   0.52   0.50   0.11\n",
            "16553  Serinc               PF03348.18        151     2.38    435   0.59   0.54   0.50   0.05\n",
            "16554  Serine_rich          PF08824.13         50     1.98    157   0.59   0.60   0.54   0.04\n",
            "16555  Serpentine_r_xa      PF03383.18          7     0.89    153   0.59   0.55   0.52   0.07\n",
            "16556  Serpin               PF00079.23        113     3.90    370   0.59   0.58   0.49   0.03\n",
            "16557  Serpulina_VSP        PF05540.14          2     0.55    387   0.59   0.59   0.51   0.02\n",
            "16558  SERRATE_Ars2_N       PF12066.11         50     2.95    110   0.59   0.53   0.56   0.08\n",
            "16559  SERTA                PF06031.16         29     6.57     36   1.51   1.58   1.50   0.16\n",
            "16560  Serum_albumin        PF00273.23        155     3.20    176   0.59   0.50   0.54   0.09\n",
            "16561  Seryl_tRNA_N         PF02403.25         78     4.84    108   0.59   0.63   0.55   0.08\n",
            "16562  Ser_hydrolase        PF06821.16         19     1.60    171   0.59   0.56   0.53   0.03\n",
            "16563  SesA                 PF17107.8          68     6.33    122   0.59   0.63   0.54   0.04\n",
            "16564  Ses_B                PF17046.8          21    21.00     28   1.84   1.80   1.75   0.29\n",
            "16565  SET                  PF00856.31        257    16.98    169   0.59   0.56   0.28   0.01\n",
            "16566  SET_assoc            PF11767.11         59     4.40     66   0.85   0.76   0.81   0.05\n",
            "16567  Sex_peptide          PF08138.14          2     0.57     55   1.00   0.89   0.94   0.09\n",
            "16568  Sey1_3HB             PF20428.1           8     1.39    419   0.59   0.59   0.52   0.03\n",
            "16569  Se_S_carrier         PF11823.11        196     8.98     69   0.82   0.82   0.76   0.03\n",
            "16570  SF-assemblin         PF06705.14          3     1.03    247   0.59   0.62   0.54   0.08\n",
            "16571  SF1-HH               PF16275.8          82     2.34    114   0.59   0.56   0.52   0.05\n",
            "16572  SF3A2                PF16835.8          85     1.68     96   0.60   0.56   0.46   0.03\n",
            "16573  SF3A3                PF16837.8         139     7.22     46   1.20   1.14   1.00   0.12\n",
            "16574  SF3a60_bindingd      PF12108.11         53    19.75     27   1.98   1.86   1.97   0.23\n",
            "16575  SF3a60_Prp9_C        PF11931.11         69     0.86    121   0.59   0.54   0.53   0.02\n",
            "16576  SF3b1                PF08920.13         43     1.43    131   0.59   0.56   0.42   0.04\n",
            "16577  SF3b10               PF07189.14         67     2.39     78   0.73   0.67   0.68   0.04\n",
            "16578  Sfi1                 PF08457.13         31     3.24    570   0.59   0.54   0.57   0.10\n",
            "16579  Sfi1_C               PF10638.12          8     0.96    103   0.59   0.59   0.52   0.05\n",
            "16580  SfLAP                PF11139.11         30     4.49    214   0.59   0.65   0.54   0.08\n",
            "16581  SfsA                 PF03749.16        161     3.13    138   0.59   0.59   0.50   0.02\n",
            "16582  SfsA_N               PF17746.4          94     5.27     63   0.89   0.88   0.75   0.03\n",
            "16583  SFTA2                PF15210.9           5     1.02     59   0.95   0.96   0.90   0.05\n",
            "16584  SFXNs                PF03820.20        134     2.07    319   0.59   0.58   0.53   0.02\n",
            "16585  SGBP_B_XBD           PF18329.4          42     3.59    189   0.59   0.52   0.41   0.04\n",
            "16586  Sgf11                PF08209.14         31     8.14     33   1.64   1.45   1.63   0.23\n",
            "16587  Sgf11_N              PF18519.4          17     3.27     39   1.40   1.41   1.38   0.11\n",
            "16588  SGIII                PF15467.9          10     0.66    449   0.59   0.60   0.52   0.05\n",
            "16589  SGL                  PF08450.15         39     3.68    246   0.59   0.57   0.49   0.01\n",
            "16590  SGNH                 PF19040.3         219    15.87    254   0.59   0.56   0.34   0.01\n",
            "16591  Sgo0707_N1           PF18873.3           4     0.82    269   0.59   0.57   0.52   0.03\n",
            "16592  Sgo0707_N2           PF20623.1          10     0.86    141   0.59   0.59   0.53   0.04\n",
            "16593  SGP                  PF17228.5          27     1.08     97   0.59   0.51   0.39   0.10\n",
            "16594  SgrR_N               PF12793.10         20     1.85    115   0.59   0.57   0.55   0.04\n",
            "16595  SgrT                 PF15894.8           1     0.66     53   1.05   0.95   0.99   0.07\n",
            "16596  SGS                  PF05002.18        125     1.28     83   0.68   0.63   0.60   0.06\n",
            "16597  SGT1                 PF07093.14         28     1.25    584   0.59   0.57   0.52   0.02\n",
            "16598  SGTA_dimer           PF16546.8          77     4.06     67   0.84   0.87   0.81   0.06\n",
            "16599  SH                   PF01445.20          1     0.65     57   0.98   0.96   0.92   0.10\n",
            "16600  SH2                  PF00017.27         52     3.69     77   0.73   0.70   0.65   0.03\n",
            "16601  SH2_2                PF14633.9          84     1.91    215   0.59   0.56   0.53   0.03\n",
            "16602  SH3-RhoG_link        PF16609.8           3     0.50    261   0.59   0.60   0.52   0.03\n",
            "16603  SH3-WW_linker        PF16618.8          13     1.18    198   0.59   0.60   0.49   0.06\n",
            "16604  SH3BGR               PF04908.18          6     0.83     99   0.59   0.58   0.52   0.02\n",
            "16605  SH3BP5               PF05276.17         20     1.67    231   0.59   0.59   0.55   0.07\n",
            "16606  SH3DP                PF20287.1          33     3.31    122   0.59   0.57   0.54   0.03\n",
            "16607  SH3_1                PF00018.31         55    11.40     48   1.15   1.10   1.10   0.04\n",
            "16608  SH3_10               PF17902.4          20     2.34     65   0.86   0.83   0.81   0.03\n",
            "16609  SH3_11               PF18103.4           1     0.60     63   0.89   0.88   0.83   0.05\n",
            "16610  SH3_12               PF18129.4         109     6.11     67   0.84   0.86   0.68   0.04\n",
            "16611  SH3_13               PF18335.4         201     3.42     65   0.86   0.85   0.77   0.06\n",
            "16612  SH3_14               PF18343.4           2     0.61    130   0.59   0.57   0.51   0.02\n",
            "16613  SH3_15               PF18346.4          81     4.82     68   0.83   0.78   0.68   0.05\n",
            "16614  SH3_16               PF18348.4         136    13.71     48   1.15   1.14   1.07   0.05\n",
            "16615  SH3_17               PF18350.4           4     1.08     53   1.04   1.09   1.00   0.09\n",
            "16616  SH3_18               PF18354.4           4     0.59     85   0.67   0.70   0.61   0.03\n",
            "16617  SH3_19               PF18597.4          15     2.17     52   1.07   1.00   1.03   0.04\n",
            "16618  SH3_2                PF07653.20         28    11.94     57   0.98   0.95   0.81   0.03\n",
            "16619  SH3_3                PF08239.14        131    40.55     55   1.01   1.01   0.88   0.06\n",
            "16620  SH3_4                PF06347.16         18     6.12     56   0.99   0.94   0.96   0.07\n",
            "16621  SH3_5                PF08460.13         10     2.50     68   0.83   0.76   0.78   0.05\n",
            "16622  SH3_6                PF12913.10         38     4.11     53   1.05   0.96   1.02   0.05\n",
            "16623  SH3_7                PF12914.10         18    13.99     47   1.17   1.18   1.10   0.03\n",
            "16624  SH3_9                PF14604.9          23    10.22     49   1.13   1.10   1.03   0.03\n",
            "16625  Shadoo               PF14999.9           5     0.49    133   0.59   0.63   0.51   0.07\n",
            "16626  Shal-type            PF11601.11         11     3.02     28   1.92   1.87   1.87   0.38\n",
            "16627  Sharpin_PH           PF16764.8          11     0.96    121   0.59   0.60   0.52   0.01\n",
            "16628  SHD1                 PF03983.15         48     1.67     67   0.84   0.82   0.80   0.03\n",
            "16629  She2p                PF11435.11         17     0.91    201   0.59   0.58   0.53   0.05\n",
            "16630  SHE3                 PF17078.8           8     1.21    228   0.59   0.59   0.54   0.08\n",
            "16631  She9_MDM33           PF05546.14         53     1.03    198   0.59   0.57   0.54   0.05\n",
            "16632  Shigella_OspC        PF06128.14          2     0.44    292   0.59   0.57   0.51   0.03\n",
            "16633  Shikimate_DH         PF01488.23         35     1.97    138   0.59   0.62   0.53   0.03\n",
            "16634  Shikimate_dh_N       PF08501.14         37     2.43     83   0.68   0.69   0.64   0.02\n",
            "16635  SHIPPO-rpt           PF07004.15        323   323.00     31   1.31   1.26   1.02   0.22\n",
            "16636  SHIRT                PF18655.4          22     2.35     82   0.69   0.62   0.61   0.06\n",
            "16637  Shisa                PF13908.9          24     1.67    194   0.59   0.50   0.46   0.07\n",
            "16638  ShK                  PF01549.27        136    13.96     37   1.47   1.12   1.36   0.46\n",
            "16639  ShlB                 PF03865.16         83     4.10    313   0.59   0.58   0.52   0.03\n",
            "16640  SHMT                 PF00464.22         10     0.69    399   0.59   0.59   0.52   0.01\n",
            "16641  SHNi-TPR             PF10516.12         22    12.66     38   1.44   1.48   1.43   0.09\n",
            "16642  SHOCT                PF09851.12        557   557.00     28   1.66   1.81   1.66   0.29\n",
            "16643  SHOCT_2              PF20612.1          60     7.28     53   1.05   1.02   1.04   0.10\n",
            "16644  SHP                  PF03579.16          1     0.60     64   0.88   0.85   0.82   0.09\n",
            "16645  SHQ1                 PF04925.18         61     1.55    186   0.59   0.57   0.53   0.04\n",
            "16646  SHR-BD               PF06650.15         24     2.96    271   0.59   0.56   0.49   0.03\n",
            "16647  SHR3_chaperone       PF08229.14         52     1.79    188   0.59   0.59   0.53   0.03\n",
            "16648  SHS2_FTSA            PF02491.23        284     5.52     79   0.72   0.73   0.66   0.04\n",
            "16649  SHS2_Rpb7-N          PF03876.20        120    30.42     57   0.98   1.02   0.86   0.04\n",
            "16650  Shufflon_N           PF04917.15          3     0.57    324   0.59   0.59   0.52   0.03\n",
            "16651  Shugoshin_C          PF07557.14         67    67.00     24   2.21   2.23   2.18   0.44\n",
            "16652  Shugoshin_N          PF07558.14         47    17.74     45   1.22   1.28   1.21   0.21\n",
            "16653  Sh_2                 PF17512.5           3     0.62    174   0.59   0.55   0.53   0.07\n",
            "16654  SiaC                 PF09345.13        135     2.56    121   0.59   0.56   0.54   0.03\n",
            "16655  Siah-Interact_N      PF09032.14         20     2.82     78   0.73   0.77   0.67   0.09\n",
            "16656  Sial-lect-inser      PF09264.13          3     0.84    198   0.59   0.59   0.52   0.03\n",
            "16657  Sialidase            PF02973.19          3     1.03    189   0.59   0.59   0.51   0.04\n",
            "16658  SIC                  PF03482.16          2     1.65     30   1.79   1.75   1.74   0.27\n",
            "16659  SICA_alpha           PF12887.10         18     2.32    183   0.59   0.54   0.47   0.05\n",
            "16660  SICA_beta            PF12878.10         66     1.66    172   0.59   0.50   0.45   0.09\n",
            "16661  SICA_C               PF12879.10          2     0.45    144   0.59   0.59   0.51   0.02\n",
            "16662  SicP-binding         PF09119.13          2     0.76     84   0.67   0.69   0.61   0.03\n",
            "16663  SID                  PF11778.11         20     1.75    135   0.59   0.59   0.55   0.05\n",
            "16664  SID-1_RNA_chan       PF13965.9          31     1.74    601   0.59   0.55   0.52   0.04\n",
            "16665  Sid-5                PF17204.6           7     1.06     76   0.74   0.74   0.68   0.03\n",
            "16666  SidC_N               PF18219.4           6     0.81    465   0.59   0.58   0.51   0.02\n",
            "16667  SidE_DUB             PF19049.3           3     0.45    173   0.59   0.59   0.52   0.01\n",
            "16668  SidE_mART            PF19048.3           3     0.46    337   0.59   0.59   0.52   0.01\n",
            "16669  SidE_PDE             PF12252.11         29     3.73    233   0.59   0.56   0.47   0.04\n",
            "16670  SieB                 PF14163.9          25     4.81    147   0.59   0.60   0.54   0.04\n",
            "16671  Sif                  PF06767.14          4     0.53    337   0.59   0.58   0.52   0.01\n",
            "16672  Sigma54_activat      PF00158.29        221     1.37    168   0.59   0.62   0.53   0.02\n",
            "16673  Sigma54_activ_2      PF14532.9           9     2.15    138   0.59   0.63   0.54   0.02\n",
            "16674  Sigma54_AID          PF00309.23        286    16.24     46   1.20   1.25   1.19   0.24\n",
            "16675  Sigma54_CBD          PF04963.16        412     3.02    180   0.59   0.61   0.48   0.02\n",
            "16676  Sigma54_DBD          PF04552.16         21     1.19    160   0.59   0.61   0.54   0.04\n",
            "16677  Sigma70_ECF          PF07638.14          7     1.40    185   0.59   0.61   0.53   0.03\n",
            "16678  Sigma70_ner          PF04546.16        170     4.74    205   0.59   0.60   0.49   0.05\n",
            "16679  Sigma70_r1_1         PF03979.17        125     5.05     82   0.69   0.74   0.62   0.09\n",
            "16680  Sigma70_r1_2         PF00140.23        402    62.31     34   1.59   1.69   1.58   0.12\n",
            "16681  Sigma70_r2           PF04542.17        200    17.47     71   0.79   0.77   0.74   0.05\n",
            "16682  Sigma70_r3           PF04539.19         67     8.74     78   0.73   0.78   0.70   0.06\n",
            "16683  Sigma70_r4           PF04545.19        106    15.41     50   1.11   1.18   1.07   0.09\n",
            "16684  Sigma70_r4_2         PF08281.15        141    43.26     54   1.03   1.09   1.02   0.07\n",
            "16685  Sigma_1s             PF02454.19          5     1.11    116   0.59   0.58   0.53   0.05\n",
            "16686  Sigma_1_2            PF03084.17          2     0.63    418   0.59   0.57   0.50   0.01\n",
            "16687  Sigma_M_inh          PF17453.5           2     0.39     96   0.60   0.62   0.52   0.06\n",
            "16688  Sigma_reg_C          PF13791.9          26     3.73    154   0.59   0.58   0.49   0.03\n",
            "16689  Sigma_reg_N          PF13800.9          21     4.82     91   0.63   0.62   0.57   0.05\n",
            "16690  SIKE                 PF05769.14         32     2.21    180   0.59   0.60   0.53   0.08\n",
            "16691  SIL1                 PF16782.8          24     1.98    292   0.59   0.58   0.50   0.04\n",
            "16692  Silic_transp         PF03842.16          4     0.59    513   0.59   0.60   0.52   0.02\n",
            "16693  SIMPL                PF04402.17        576    14.15    197   0.59   0.64   0.32   0.05\n",
            "16694  SIM_C                PF06621.15         14     1.58    305   0.59   0.55   0.50   0.04\n",
            "16695  SIN1                 PF05422.15         18     2.40    141   0.59   0.57   0.50   0.06\n",
            "16696  SIN1_PH              PF16979.8          54     2.73    108   0.59   0.56   0.48   0.05\n",
            "16697  Sin3a_C              PF16879.8         118     3.49    288   0.59   0.55   0.49   0.05\n",
            "16698  Sin3_corepress       PF08295.15         90     2.35     98   0.59   0.55   0.53   0.05\n",
            "16699  Sina                 PF03145.19         40     1.89    200   0.59   0.52   0.52   0.04\n",
            "16700  SinI                 PF08671.13         30    18.86     28   1.92   1.93   1.90   0.18\n",
            "16701  SIP                  PF04954.16         85     3.25    119   0.59   0.58   0.52   0.03\n",
            "16702  SIP1                 PF04938.15        152     8.66    218   0.59   0.56   0.43   0.03\n",
            "16703  SipA                 PF09052.13          3     0.81    214   0.59   0.61   0.54   0.02\n",
            "16704  SipA_VBS             PF17985.4           3     3.00     22   1.63   1.74   1.61   0.21\n",
            "16705  Sipho_Gp157          PF05565.14         37     3.48    163   0.59   0.62   0.53   0.06\n",
            "16706  Sipho_Gp37           PF14594.9          32     4.22    352   0.59   0.59   0.49   0.02\n",
            "16707  Sipho_tail           PF05709.14         39     6.57    255   0.59   0.58   0.46   0.02\n",
            "16708  Sir1                 PF11603.11         35     4.26    120   0.59   0.56   0.53   0.04\n",
            "16709  SIR2                 PF02146.20         18     1.22    179   0.59   0.56   0.51   0.01\n",
            "16710  SIR2_2               PF13289.9          63     7.09    144   0.59   0.57   0.44   0.02\n",
            "16711  SIR4_SID             PF16991.8          10     2.24    163   0.59   0.56   0.51   0.06\n",
            "16712  SirA                 PF10747.12         14     1.42    140   0.59   0.56   0.54   0.04\n",
            "16713  SirB                 PF04247.15         81     2.86    121   0.59   0.59   0.53   0.10\n",
            "16714  Sirohm_synth_C       PF14823.9          68     2.51     68   0.83   0.77   0.75   0.05\n",
            "16715  Sirohm_synth_M       PF14824.9          81    35.05     28   1.92   2.02   1.92   0.13\n",
            "16716  SIS                  PF01380.25         33     8.11    131   0.59   0.62   0.54   0.02\n",
            "16717  SIS_2                PF13580.9          34     3.45    138   0.59   0.63   0.55   0.02\n",
            "16718  SIT                  PF15330.9          31     2.23    114   0.59   0.58   0.50   0.03\n",
            "16719  Siva                 PF05458.15          4     0.63    173   0.59   0.54   0.52   0.05\n",
            "16720  SIX1_SD              PF16878.8          52     1.27    115   0.59   0.55   0.51   0.04\n",
            "16721  SKA1                 PF07160.15         34     2.64    237   0.59   0.59   0.52   0.05\n",
            "16722  SKA2                 PF16740.8          33     3.48    110   0.59   0.61   0.54   0.07\n",
            "16723  SKG6                 PF08693.13         10     1.30    666   0.59   0.57   0.51   0.05\n",
            "16724  SKI                  PF01202.25         78     3.75    158   0.59   0.60   0.52   0.02\n",
            "16725  Ski2_N               PF17911.4          34     3.68    135   0.59   0.60   0.50   0.03\n",
            "16726  SKICH                PF17751.4          71     2.34    104   0.59   0.52   0.50   0.04\n",
            "16727  SKIP_SNW             PF02731.18         60     1.61    159   0.59   0.58   0.52   0.06\n",
            "16728  Ski_Sno              PF02437.20         22     1.24    108   0.59   0.58   0.53   0.04\n",
            "16729  SKN1_KRE6_Sbg1       PF03935.18         24     0.84    501   0.59   0.54   0.52   0.01\n",
            "16730  Skp1                 PF01466.22        188    12.65     48   1.15   1.15   1.11   0.09\n",
            "16731  Skp1_POZ             PF03931.18         13     2.78     63   0.89   0.88   0.85   0.06\n",
            "16732  SK_channel           PF03530.17         25     1.59    118   0.59   0.59   0.53   0.05\n",
            "16733  SL4P                 PF17618.5          48     4.36     88   0.65   0.63   0.61   0.04\n",
            "16734  SLAC1                PF03595.20        431     8.18    326   0.59   0.57   0.40   0.11\n",
            "16735  SLAIN                PF15301.9          29     1.30    436   0.59   0.59   0.49   0.05\n",
            "16736  SLAM                 PF06214.14         11     0.81    125   0.59   0.61   0.53   0.03\n",
            "16737  SLATT_1              PF18181.4          52     2.85    122   0.59   0.56   0.55   0.06\n",
            "16738  SLATT_2              PF18183.4          13     0.58    192   0.59   0.59   0.53   0.02\n",
            "16739  SLATT_3              PF18184.4          28     1.97    156   0.59   0.58   0.53   0.03\n",
            "16740  SLATT_4              PF18186.4          27     1.73    165   0.59   0.60   0.55   0.04\n",
            "16741  SLATT_5              PF18160.4          26     5.16    193   0.59   0.58   0.52   0.04\n",
            "16742  SLATT_6              PF18169.4          11     1.11    176   0.59   0.57   0.53   0.04\n",
            "16743  SLATT_fungal         PF18142.4         125     3.93    120   0.59   0.63   0.49   0.03\n",
            "16744  SLBB                 PF10531.12         59    16.11     58   0.96   1.01   0.88   0.05\n",
            "16745  SLBP_RNA_bind        PF15247.9          56     2.31     70   0.80   0.71   0.75   0.09\n",
            "16746  SLC12                PF03522.18         64     1.60    418   0.59   0.57   0.51   0.02\n",
            "16747  SLC25_like           PF18405.4           9     1.15    285   0.59   0.60   0.52   0.02\n",
            "16748  SLC35F               PF06027.15          5     0.68    299   0.59   0.59   0.53   0.06\n",
            "16749  SLC3A2_N             PF16028.8          21     1.41     77   0.73   0.69   0.68   0.04\n",
            "16750  SLD3                 PF08639.13         46     2.60    539   0.59   0.59   0.46   0.05\n",
            "16751  Sld3_N               PF18523.4          15     2.67    116   0.59   0.55   0.54   0.04\n",
            "16752  Sld5                 PF05916.14        166    11.42    109   0.59   0.60   0.45   0.04\n",
            "16753  SLD5_C               PF16922.8          74     7.85     56   0.99   0.99   0.90   0.05\n",
            "16754  Sld7_C               PF18596.4          54     2.68     78   0.73   0.73   0.65   0.05\n",
            "16755  Sld7_N               PF18636.4          13     1.06    125   0.59   0.56   0.49   0.03\n",
            "16756  SLED                 PF12140.11         36     2.33    112   0.59   0.56   0.53   0.03\n",
            "16757  SLH                  PF00395.23         99    86.33     44   1.25   1.24   1.17   0.05\n",
            "16758  SLIDE                PF09111.13        100     2.76    116   0.59   0.56   0.53   0.05\n",
            "16759  SlipAM               PF04575.16         11     2.14    288   0.59   0.53   0.53   0.04\n",
            "16760  SLM4                 PF16818.8          16     1.46    161   0.59   0.60   0.49   0.04\n",
            "16761  Slp                  PF03843.16         61     2.25    153   0.59   0.56   0.51   0.01\n",
            "16762  SlpA                 PF03217.17         52     6.80     54   1.03   1.00   0.94   0.16\n",
            "16763  SLR1-BP              PF07333.15         27     2.73     58   0.96   0.72   0.85   0.31\n",
            "16764  Slr4                 PF19526.2          13     3.67    366   0.59   0.62   0.44   0.06\n",
            "16765  SLS                  PF14611.9          86     7.62    221   0.59   0.58   0.44   0.04\n",
            "16766  SLT                  PF01464.23         26     3.71    117   0.59   0.59   0.52   0.03\n",
            "16767  SLT_2                PF13406.9         163     2.91    292   0.59   0.58   0.50   0.02\n",
            "16768  SLT_3                PF18896.3          16     1.41     89   0.64   0.62   0.56   0.05\n",
            "16769  SLT_4                PF19489.2          56     1.53    184   0.59   0.53   0.53   0.03\n",
            "16770  SLT_beta             PF02258.19          1     0.54     70   0.81   0.78   0.74   0.03\n",
            "16771  SLT_L                PF14718.9          57     5.67     68   0.83   0.79   0.81   0.10\n",
            "16772  Slu7                 PF11708.11        101     1.96    266   0.59   0.56   0.49   0.07\n",
            "16773  Slx4                 PF09494.13         74     6.05     61   0.92   0.86   0.82   0.05\n",
            "16774  SLX9                 PF15341.9         132     5.45    131   0.59   0.61   0.46   0.12\n",
            "16775  SLY                  PF12485.11         26     1.74    156   0.59   0.59   0.51   0.08\n",
            "16776  SlyX                 PF04102.15        179     5.89     67   0.84   0.84   0.79   0.09\n",
            "16777  SM-ATX               PF14438.9         103     6.42     81   0.70   0.72   0.62   0.03\n",
            "16778  Smac_DIABLO          PF09057.13         11     0.84    231   0.59   0.61   0.51   0.03\n",
            "16779  SmaI                 PF17411.5           3     0.57    241   0.59   0.58   0.52   0.01\n",
            "16780  SmAKAP               PF15127.9          13     0.97     93   0.61   0.58   0.53   0.04\n",
            "16781  SMAP                 PF15477.9          74     3.40     79   0.72   0.70   0.54   0.08\n",
            "16782  SMBP                 PF16785.8          23     2.59    111   0.59   0.61   0.53   0.10\n",
            "16783  SMC_hinge            PF06470.16         97     8.02    117   0.59   0.63   0.51   0.02\n",
            "16784  SMC_N                PF02463.22         28     1.78    220   0.59   0.61   0.49   0.03\n",
            "16785  SMC_Nse1             PF07574.16         80     2.83    194   0.59   0.59   0.45   0.03\n",
            "16786  SMC_ScpA             PF02616.17          5     1.46    241   0.59   0.61   0.49   0.04\n",
            "16787  SMC_ScpB             PF04079.19         94     3.03    159   0.59   0.63   0.53   0.03\n",
            "16788  SMG1                 PF15785.8           7     1.16    607   0.59   0.57   0.52   0.03\n",
            "16789  SMG1_N               PF17229.5           9     0.81    106   0.59   0.60   0.52   0.05\n",
            "16790  Smg4_UPF3            PF03467.18         84     3.40    176   0.59   0.58   0.48   0.04\n",
            "16791  Smg8_Smg9            PF10220.12         22     1.45    882   0.59   0.55   0.51   0.02\n",
            "16792  SMI1_KNR4            PF09346.13        154    12.41    133   0.59   0.56   0.41   0.02\n",
            "16793  Smim3                PF17307.5           6     0.84     60   0.93   0.93   0.88   0.11\n",
            "16794  SMN                  PF06003.15          3     0.53    264   0.59   0.56   0.51   0.02\n",
            "16795  Smoa_sbd             PF17885.4          63     2.08    108   0.59   0.58   0.54   0.01\n",
            "16796  SMODS                PF18144.4          32     2.63    164   0.59   0.56   0.53   0.03\n",
            "16797  Smoothelin           PF12510.11         15     4.20     50   1.11   1.18   1.09   0.16\n",
            "16798  SMP                  PF04927.15         90     6.12     59   0.95   1.04   0.87   0.15\n",
            "16799  SmpA_OmlA            PF04355.16         86     6.57     71   0.79   0.76   0.73   0.03\n",
            "16800  SmpB                 PF01668.21        460     1.34    143   0.59   0.60   0.53   0.04\n",
            "16801  SMP_2                PF10144.12         17     1.40    159   0.59   0.62   0.54   0.04\n",
            "16802  SMP_C2CD2L           PF18696.4          34     1.52    152   0.59   0.57   0.54   0.04\n",
            "16803  SMP_LBD              PF17047.8           3     0.38    180   0.59   0.59   0.52   0.02\n",
            "16804  Smr                  PF01713.24        565     5.14     80   0.71   0.74   0.56   0.04\n",
            "16805  SMRP1                PF15181.9           3     0.45    262   0.59   0.54   0.51   0.02\n",
            "16806  SMYLE_N              PF18615.4          14     0.89    388   0.59   0.57   0.51   0.03\n",
            "16807  Sm_like              PF16243.8          11     0.70     85   0.67   0.67   0.61   0.03\n",
            "16808  Sm_multidrug_ex      PF06695.14        151     3.26    121   0.59   0.62   0.49   0.05\n",
            "16809  SnAC                 PF14619.9          36     3.57     73   0.77   0.76   0.62   0.11\n",
            "16810  SNAD1                PF18744.4           6     1.20    214   0.59   0.53   0.50   0.04\n",
            "16811  SNAD2                PF18745.4           1     0.38    200   0.59   0.56   0.52   0.02\n",
            "16812  SNAD3                PF18749.4           3     0.77    379   0.59   0.55   0.51   0.03\n",
            "16813  SNAD4                PF18750.4          65     1.91    116   0.59   0.51   0.49   0.04\n",
            "16814  SNAP                 PF14938.9          49     2.05    282   0.59   0.59   0.54   0.05\n",
            "16815  SNAP-25              PF00835.22         33     3.50     56   0.99   0.92   0.89   0.08\n",
            "16816  SNAPC1               PF09808.12         64     3.57    192   0.59   0.55   0.52   0.05\n",
            "16817  SNAPC2               PF11035.11         25     1.87    371   0.59   0.60   0.45   0.03\n",
            "16818  SNAPC3               PF12251.11         63     2.20    200   0.59   0.50   0.51   0.05\n",
            "16819  SNAPC5               PF15497.9          58     4.96    103   0.59   0.63   0.47   0.13\n",
            "16820  Snapin_Pallidin      PF14712.9          69     5.46     90   0.63   0.67   0.59   0.09\n",
            "16821  SNARE                PF05739.22         64    25.17     53   1.05   1.07   1.05   0.11\n",
            "16822  SNARE_assoc          PF09335.14         54     7.16    123   0.59   0.60   0.51   0.07\n",
            "16823  SNase                PF00565.20         18     2.09    108   0.59   0.58   0.49   0.03\n",
            "16824  SNCAIP_SNCA_bd       PF16700.8           2     1.18     46   1.20   1.21   1.15   0.08\n",
            "16825  SNF                  PF00209.21         15     0.88    523   0.59   0.56   0.52   0.04\n",
            "16826  SNF2-rel_dom         PF00176.26         19     2.40    318   0.59   0.56   0.44   0.02\n",
            "16827  SNF2_assoc           PF08455.13        102     6.89    375   0.59   0.60   0.48   0.04\n",
            "16828  SNF5                 PF04855.15         92     1.76    236   0.59   0.57   0.43   0.03\n",
            "16829  Snf7                 PF03357.24         32     4.23    173   0.59   0.61   0.54   0.10\n",
            "16830  SNN_cytoplasm        PF09051.13          2     1.93     26   2.05   2.07   2.02   0.12\n",
            "16831  SNN_linker           PF09050.13          2     1.74     26   2.05   2.05   2.02   0.20\n",
            "16832  SNN_transmemb        PF09049.13          2     1.32     32   1.69   1.71   1.65   0.26\n",
            "16833  SNO                  PF01174.22          4     0.71    188   0.59   0.60   0.50   0.02\n",
            "16834  SnoaL                PF07366.15         29     4.54    126   0.59   0.57   0.54   0.02\n",
            "16835  SnoaL_2              PF12680.10        272    14.29    102   0.59   0.57   0.46   0.04\n",
            "16836  SnoaL_3              PF13474.9          37     5.89    121   0.59   0.55   0.52   0.03\n",
            "16837  SnoaL_4              PF13577.9          89     8.34    126   0.59   0.55   0.49   0.03\n",
            "16838  SnoaL_5              PF20409.1           6     0.71    118   0.59   0.57   0.53   0.02\n",
            "16839  SNRNP27              PF08648.15         78     2.61     56   0.99   0.94   0.95   0.08\n",
            "16840  Snu56_snRNP          PF19097.3          21     1.94    438   0.59   0.57   0.52   0.04\n",
            "16841  SNURF                PF07192.14          1     0.56     69   0.81   0.80   0.75   0.05\n",
            "16842  Snurportin1          PF11538.11         17     4.84     40   1.37   1.30   1.31   0.26\n",
            "16843  SNX17_FERM_C         PF18116.4          10     1.21    109   0.59   0.55   0.51   0.02\n",
            "16844  Snx8_BAR_dom         PF19566.2          41     1.07    258   0.59   0.59   0.54   0.03\n",
            "16845  SOAR                 PF16533.8          21     1.63    101   0.59   0.58   0.55   0.06\n",
            "16846  SOBP                 PF15279.9          12     1.37    323   0.59   0.54   0.50   0.04\n",
            "16847  Soc                  PF16855.8           1     0.50     74   0.76   0.73   0.69   0.03\n",
            "16848  SOCS                 PF12610.11         10     1.93     58   0.96   0.92   0.89   0.10\n",
            "16849  SOCS_box             PF07525.19        365    45.69     38   1.44   1.44   1.29   0.19\n",
            "16850  SoDot-IcmSS          PF16848.8           4     1.22    178   0.59   0.60   0.50   0.04\n",
            "16851  Sod_Cu               PF00080.23        398     4.10    133   0.59   0.58   0.38   0.04\n",
            "16852  Sod_Fe_C             PF02777.21        829     1.65    101   0.59   0.53   0.49   0.04\n",
            "16853  Sod_Fe_N             PF00081.25         20     1.31     82   0.69   0.62   0.64   0.06\n",
            "16854  Sod_Ni               PF09055.14         29     1.38    127   0.59   0.54   0.51   0.05\n",
            "16855  Sof1                 PF04158.17        121     2.74     87   0.65   0.65   0.61   0.12\n",
            "16856  SOG2                 PF10428.12         43     4.26    504   0.59   0.60   0.47   0.03\n",
            "16857  SOGA                 PF11365.11         25     1.60     95   0.60   0.64   0.55   0.08\n",
            "16858  SOK                  PF06136.16         41     1.71    383   0.59   0.58   0.45   0.04\n",
            "16859  Solute_trans_a       PF03619.19        263     4.15    243   0.59   0.56   0.50   0.08\n",
            "16860  Somatomedin_B        PF01033.20        148     3.59     40   1.37   0.99   1.27   0.56\n",
            "16861  Somatostatin         PF03002.18          5     4.92     18   2.91   2.53   2.90   0.64\n",
            "16862  SOP4                 PF17081.8          13     0.89    209   0.59   0.58   0.53   0.02\n",
            "16863  SopA                 PF13981.9           3     0.93    127   0.59   0.57   0.51   0.03\n",
            "16864  SopA_C               PF13979.9           2     0.60    168   0.59   0.57   0.52   0.01\n",
            "16865  SoPB_HTH             PF18090.4           2     0.86     75   0.76   0.79   0.70   0.03\n",
            "16866  SopD                 PF11047.11          2     0.53    319   0.59   0.57   0.52   0.02\n",
            "16867  SopE_GEF             PF07487.16          4     1.07    136   0.59   0.58   0.53   0.02\n",
            "16868  SOR                  PF07682.14          9     0.86    302   0.59   0.55   0.52   0.01\n",
            "16869  Sorb                 PF02208.19          4     1.55     45   1.22   1.13   1.16   0.08\n",
            "16870  Sororin              PF09666.13          9     1.97    157   0.59   0.59   0.48   0.06\n",
            "16871  Sortase              PF04203.16         71     3.64    125   0.59   0.57   0.42   0.02\n",
            "16872  Sortilin-Vps10       PF15902.8         129     3.10    445   0.59   0.56   0.50   0.03\n",
            "16873  Sortilin_C           PF15901.8         301     2.56    171   0.59   0.48   0.45   0.09\n",
            "16874  Sorting_nexin        PF03700.16          8     1.46     76   0.74   0.77   0.63   0.06\n",
            "16875  SOR_SNZ              PF01680.20        114     0.80    207   0.59   0.62   0.53   0.03\n",
            "16876  SOSSC                PF15925.8           8     1.22    101   0.59   0.59   0.47   0.02\n",
            "16877  SOTI                 PF17079.8           6     0.68    101   0.59   0.55   0.52   0.02\n",
            "16878  SOUL                 PF04832.15        229     4.53    173   0.59   0.56   0.43   0.03\n",
            "16879  Sox17_18_mid         PF12067.11         30     2.47     49   1.13   1.08   0.99   0.10\n",
            "16880  SoxD                 PF04267.15         83     1.66     82   0.69   0.59   0.64   0.04\n",
            "16881  SoxE                 PF06525.14          6     0.67    149   0.59   0.58   0.52   0.03\n",
            "16882  SoxG                 PF04268.15          1     0.39    152   0.59   0.60   0.51   0.01\n",
            "16883  SOXp                 PF12336.11         14     1.07     91   0.63   0.58   0.49   0.05\n",
            "16884  SoxY                 PF13501.9         149     3.53    112   0.59   0.61   0.52   0.03\n",
            "16885  SoxZ                 PF08770.14        124     3.08     98   0.59   0.58   0.52   0.03\n",
            "16886  Sox_N                PF12444.11         19     1.57     77   0.73   0.73   0.65   0.05\n",
            "16887  Soyouz_module        PF14313.9          16     4.13     58   0.96   0.99   0.94   0.06\n",
            "16888  SO_alpha_A3          PF17806.4          10     1.34     87   0.65   0.67   0.60   0.02\n",
            "16889  Sp-CxC               PF20304.1          11     6.38     63   0.89   0.90   0.84   0.10\n",
            "16890  Sp-DndD              PF20306.1          43    43.00     51   1.07   1.11   1.03   0.12\n",
            "16891  SP2                  PF03014.17          2     0.68    628   0.59   0.60   0.50   0.01\n",
            "16892  SP24                 PF16504.8          44     6.70    147   0.59   0.58   0.53   0.10\n",
            "16893  Sp38                 PF07354.15          5     0.64    177   0.59   0.52   0.52   0.04\n",
            "16894  SPA                  PF08616.13         23     2.74    113   0.59   0.57   0.51   0.02\n",
            "16895  Spa1_C               PF18218.4          17     2.16     99   0.59   0.58   0.54   0.04\n",
            "16896  SpaA                 PF17802.4         412    12.60     69   0.82   0.85   0.72   0.07\n",
            "16897  SpaA_2               PF19403.2          14     1.14     97   0.59   0.59   0.50   0.04\n",
            "16898  SPACA7               PF15307.9           5     0.83    107   0.59   0.58   0.51   0.04\n",
            "16899  SPACA9               PF15120.9           6     1.05    164   0.59   0.58   0.51   0.03\n",
            "16900  Spaetzle             PF16077.8          42     1.53     98   0.59   0.47   0.52   0.09\n",
            "16901  SPAM                 PF02090.18          3     0.75    140   0.59   0.58   0.53   0.08\n",
            "16902  SPAN                 PF02510.17          1     0.41    336   0.59   0.61   0.52   0.02\n",
            "16903  SPAN-X               PF07458.15          7     0.86     94   0.61   0.60   0.54   0.07\n",
            "16904  SPARC_Ca_bdg         PF10591.12         19     1.37    113   0.59   0.50   0.52   0.06\n",
            "16905  SPARK                PF19160.3         396     5.29    163   0.59   0.50   0.26   0.11\n",
            "16906  SPAR_C               PF11881.11         22     1.68    243   0.59   0.60   0.49   0.04\n",
            "16907  SPASM                PF13186.9         162     7.40     68   0.83   0.68   0.59   0.09\n",
            "16908  SPATA19              PF15212.9           6     0.52    130   0.59   0.57   0.52   0.05\n",
            "16909  SPATA1_C             PF15743.8          12     1.18    152   0.59   0.59   0.53   0.10\n",
            "16910  SPATA24              PF15175.9           5     1.01    182   0.59   0.59   0.53   0.07\n",
            "16911  SPATA25              PF15218.9           6     0.55    228   0.59   0.59   0.52   0.02\n",
            "16912  SPATA3               PF15662.8           6     0.65    192   0.59   0.55   0.52   0.06\n",
            "16913  SPATA48              PF15073.9           8     0.93    128   0.59   0.55   0.54   0.03\n",
            "16914  SPATA6               PF14909.9          25     1.42    139   0.59   0.59   0.53   0.02\n",
            "16915  SPATA9               PF15824.8           4     0.57    253   0.59   0.60   0.52   0.02\n",
            "16916  Spatacsin_C          PF14649.9          33     2.04    299   0.59   0.58   0.51   0.04\n",
            "16917  SPATIAL              PF15256.9          25     1.49    201   0.59   0.57   0.50   0.03\n",
            "16918  Spb1_C               PF07780.15         96     1.77    219   0.59   0.59   0.52   0.09\n",
            "16919  Spc110_C             PF18520.4          14     2.37     52   1.07   1.03   1.04   0.10\n",
            "16920  SPC12                PF06645.16        132     3.79     71   0.79   0.74   0.77   0.07\n",
            "16921  SPC22                PF04573.15         93     2.62    172   0.59   0.55   0.52   0.03\n",
            "16922  Spc24                PF08286.14         81     4.14    107   0.59   0.59   0.46   0.06\n",
            "16923  SPC25                PF06703.14        154     4.90    161   0.59   0.58   0.49   0.03\n",
            "16924  Spc29                PF17082.8           4     1.18    244   0.59   0.58   0.49   0.05\n",
            "16925  Spc42p               PF11544.11          6     1.13     75   0.75   0.78   0.69   0.08\n",
            "16926  Spc7                 PF08317.14         51     2.89    196   0.59   0.57   0.54   0.05\n",
            "16927  SpdB                 PF05122.16          3     1.11     50   1.10   1.04   1.05   0.10\n",
            "16928  SPDY                 PF03771.19         61     8.56     62   0.90   0.85   0.82   0.09\n",
            "16929  Spec3                PF15795.8          25     1.08     85   0.67   0.64   0.60   0.09\n",
            "16930  SpecificRecomb       PF10136.12         53     1.92    641   0.59   0.61   0.55   0.02\n",
            "16931  SPECT1               PF18680.4           1     0.40    180   0.59   0.62   0.52   0.03\n",
            "16932  Spectrin             PF00435.24         80     9.06    105   0.59   0.59   0.55   0.09\n",
            "16933  Spectrin_like        PF18373.4          35     1.62     78   0.73   0.68   0.69   0.06\n",
            "16934  SpeFL                PF10940.11          2     1.08     40   1.37   1.26   1.32   0.16\n",
            "16935  SPEG_u2              PF16650.8           2     0.77     57   0.98   0.97   0.93   0.12\n",
            "16936  Spem1                PF15670.8          12     0.72    253   0.59   0.56   0.52   0.02\n",
            "16937  Speriolin_C          PF15059.9          11     0.86    148   0.59   0.57   0.52   0.02\n",
            "16938  Speriolin_N          PF15058.9           5     1.42    232   0.59   0.63   0.50   0.03\n",
            "16939  Spermine_synth       PF01564.20         15     2.53    188   0.59   0.58   0.50   0.02\n",
            "16940  Spermine_synt_N      PF17284.5          13     3.65     53   1.05   1.01   1.00   0.05\n",
            "16941  Sperm_act_pep        PF08250.14         11    11.00     10   2.62   2.81   2.62   1.14\n",
            "16942  Sperm_Ag_HE2         PF05324.16          9     1.33     70   0.80   0.83   0.73   0.04\n",
            "16943  SPESP1               PF15754.8           9     0.81    322   0.59   0.59   0.52   0.03\n",
            "16944  Spexin               PF15171.9           6     0.64     90   0.63   0.64   0.57   0.03\n",
            "16945  SPG4                 PF17325.5          12     1.10    109   0.59   0.56   0.52   0.06\n",
            "16946  SPG48                PF14764.9          30     1.81    118   0.59   0.59   0.54   0.04\n",
            "16947  Spheroidin           PF05541.15          4     0.85    944   0.59   0.57   0.52   0.03\n",
            "16948  Spherulin4           PF12138.11         89     2.64    240   0.59   0.55   0.43   0.02\n",
            "16949  SPICE                PF15678.8          17     2.57    412   0.59   0.59   0.50   0.06\n",
            "16950  Spidroin_MaSp        PF11260.11         26     1.89     85   0.67   0.76   0.62   0.10\n",
            "16951  Spidroin_N           PF16763.8          22     1.65    125   0.59   0.60   0.54   0.05\n",
            "16952  Spike_torovirin      PF17072.8          12     3.59   1291   0.59   0.55   0.50   0.03\n",
            "16953  Spin-Ssty            PF02513.20         32     3.94     50   1.11   1.04   1.08   0.06\n",
            "16954  SPIN90_LRD           PF09431.13         83     1.64    139   0.59   0.60   0.50   0.06\n",
            "16955  Spindle_Spc25        PF08234.15        132     7.55     73   0.77   0.77   0.73   0.05\n",
            "16956  Spiralin             PF05215.16        129    14.64     82   0.69   0.75   0.51   0.12\n",
            "16957  SPK                  PF04435.21        202     5.71    104   0.59   0.57   0.52   0.05\n",
            "16958  SplA                 PF11132.11          9     1.09     74   0.76   0.73   0.71   0.05\n",
            "16959  SpmSyn_N             PF17950.4          11     0.68     96   0.60   0.61   0.53   0.02\n",
            "16960  Spo0A_C              PF08769.14        132     3.44    104   0.59   0.58   0.55   0.03\n",
            "16961  Spo0M                PF07070.14         77     1.88    210   0.59   0.62   0.54   0.02\n",
            "16962  SPO11_like           PF03533.17          9     1.79     43   1.28   1.34   1.24   0.10\n",
            "16963  Spo12                PF05032.15         30     4.98     35   1.55   1.54   1.50   0.18\n",
            "16964  Spo16                PF19225.3          13     2.51    165   0.59   0.58   0.51   0.06\n",
            "16965  SPO22                PF08631.13         74     7.35    275   0.59   0.59   0.50   0.04\n",
            "16966  Spo7                 PF03907.16         59     1.65    221   0.59   0.57   0.50   0.03\n",
            "16967  Spo7_2_N             PF15407.9          38     4.26     64   0.88   0.85   0.85   0.04\n",
            "16968  SPOB_a               PF14689.9          31     5.63     62   0.90   0.87   0.88   0.09\n",
            "16969  SPOB_ab              PF14682.9          14     1.91    113   0.59   0.57   0.53   0.04\n",
            "16970  SPOC                 PF07744.16         52     5.44    151   0.59   0.58   0.42   0.01\n",
            "16971  SpoIIAA-like         PF11964.11        172    10.35     95   0.60   0.59   0.32   0.04\n",
            "16972  SpoIID               PF08486.13        465     3.41     94   0.61   0.60   0.43   0.05\n",
            "16973  SpoIIE               PF07228.15        152     6.61    193   0.59   0.62   0.50   0.01\n",
            "16974  SpoIIE_N             PF19732.2         198     1.21    605   0.59   0.60   0.54   0.03\n",
            "16975  SpoIIIAC             PF06686.14        408    17.77     56   0.99   1.09   0.97   0.17\n",
            "16976  SpoIIIAH             PF12685.10        141     5.34    212   0.59   0.62   0.51   0.08\n",
            "16977  SpoIIID              PF12116.11         14     0.85     82   0.69   0.70   0.64   0.06\n",
            "16978  SpoIIM               PF01944.20        356     8.41    156   0.59   0.63   0.41   0.10\n",
            "16979  SpoIIP               PF07454.14         58     2.83    264   0.59   0.58   0.48   0.04\n",
            "16980  SpoIISA_toxin        PF14171.9           9     1.63    243   0.59   0.57   0.51   0.05\n",
            "16981  SpoIISB_antitox      PF14185.9           3     1.08     56   0.99   0.96   0.94   0.09\n",
            "16982  SpoIVA_ATPase        PF09547.13        159     0.74    237   0.59   0.59   0.53   0.02\n",
            "16983  SpoIVA_C             PF20439.1         159     1.53     76   0.74   0.75   0.70   0.05\n",
            "16984  SpoIVA_middle        PF20438.1         159     1.42    179   0.59   0.59   0.55   0.02\n",
            "16985  Spond_N              PF06468.16         48     1.33    192   0.59   0.53   0.52   0.02\n",
            "16986  SpoOE-like           PF09388.13        155    29.82     42   1.31   1.35   1.26   0.18\n",
            "16987  SPOR                 PF05036.16        133    31.04     76   0.74   0.77   0.66   0.04\n",
            "16988  Spore-coat_CotD      PF11122.11          6     1.25     99   0.59   0.53   0.49   0.06\n",
            "16989  Spore-coat_CotZ      PF10612.12          9     1.12    159   0.59   0.53   0.49   0.07\n",
            "16990  Spore_coat_CotO      PF14153.9           7     1.40    192   0.59   0.58   0.50   0.06\n",
            "16991  Spore_Cse60          PF10957.11          5     0.92     60   0.93   0.90   0.88   0.08\n",
            "16992  Spore_GerAC          PF05504.14        324     9.94    164   0.59   0.58   0.42   0.04\n",
            "16993  Spore_GerQ           PF09671.13         23     0.91     76   0.74   0.72   0.69   0.04\n",
            "16994  Spore_III_AA         PF19568.2           4     0.54    306   0.59   0.60   0.52   0.02\n",
            "16995  Spore_III_AB         PF09548.13        180     7.24    170   0.59   0.62   0.54   0.04\n",
            "16996  Spore_III_AE         PF09546.13        145     3.70    321   0.59   0.63   0.56   0.09\n",
            "16997  Spore_III_AF         PF09581.13         94     5.47    190   0.59   0.63   0.50   0.05\n",
            "16998  Spore_II_R           PF09551.13        191     1.19    128   0.59   0.56   0.53   0.02\n",
            "16999  Spore_permease       PF03845.16         15     2.58    321   0.59   0.59   0.55   0.09\n",
            "17000  Spore_SspJ           PF09575.13          3     1.30     46   1.20   1.22   1.14   0.14\n",
            "17001  Spore_YabQ           PF09578.13        181     6.12     78   0.73   0.71   0.68   0.15\n",
            "17002  Spore_YhaL           PF14147.9          14     2.05     52   1.07   1.01   1.04   0.08\n",
            "17003  Spore_YhcN_YlaJ      PF09580.13         40     5.20    159   0.59   0.58   0.52   0.08\n",
            "17004  Spore_YpjB           PF09577.13         15     1.53    230   0.59   0.59   0.54   0.04\n",
            "17005  Spore_YtfJ           PF09579.13        195     3.45     81   0.70   0.75   0.58   0.04\n",
            "17006  Spore_YtrH           PF14034.9          36     1.20    101   0.59   0.64   0.54   0.06\n",
            "17007  Spore_YunB           PF09560.13        126     2.86     91   0.63   0.65   0.60   0.04\n",
            "17008  Sporozoite_P67       PF05642.14          3     0.59    727   0.59   0.62   0.50   0.02\n",
            "17009  SporV_AA             PF12164.11         86     3.23     89   0.64   0.65   0.58   0.06\n",
            "17010  Spot_14              PF07084.15         24     1.07    153   0.59   0.58   0.48   0.03\n",
            "17011  SPOUT_MTase          PF02590.20        559     2.28    154   0.59   0.58   0.51   0.02\n",
            "17012  SPOUT_MTase_2        PF14419.9          15     0.85    173   0.59   0.61   0.53   0.02\n",
            "17013  SpoU_methylase       PF00588.22        116     4.25    142   0.59   0.61   0.51   0.02\n",
            "17014  SpoU_methylas_C      PF12105.11         51     3.73     54   1.03   0.99   0.99   0.06\n",
            "17015  SpoU_sub_bind        PF08032.15         93    13.46     76   0.74   0.76   0.68   0.03\n",
            "17016  SpoV                 PF08183.14          6     4.41     25   2.13   2.14   2.12   0.29\n",
            "17017  SpoVAB               PF13782.9          90     1.68    109   0.59   0.62   0.53   0.07\n",
            "17018  SpoVAC_SpoVAEB       PF03862.16        328     1.71    114   0.59   0.62   0.53   0.07\n",
            "17019  SpoVAD               PF07451.14         33     0.89    329   0.59   0.60   0.54   0.02\n",
            "17020  SpoVAE               PF14097.9          30     0.92    180   0.59   0.63   0.53   0.04\n",
            "17021  SpoVG                PF04026.15        130     2.17     82   0.69   0.68   0.60   0.04\n",
            "17022  SpoVIF               PF14069.9          30     3.09     72   0.78   0.83   0.74   0.09\n",
            "17023  SpoVR                PF04293.16         98     1.22    418   0.59   0.55   0.52   0.03\n",
            "17024  SpoVS                PF04232.15         50     1.63     83   0.68   0.74   0.64   0.05\n",
            "17025  SpoVT_C              PF15714.8         188     2.28    128   0.59   0.62   0.52   0.04\n",
            "17026  SPP                  PF06550.14          3     0.70    283   0.59   0.62   0.52   0.06\n",
            "17027  Spp-24               PF07448.14         13     0.97     64   0.88   0.79   0.82   0.06\n",
            "17028  SPR1                 PF15356.9           3     0.41    114   0.59   0.54   0.52   0.13\n",
            "17029  SprA-related         PF12118.11          3     0.82    335   0.59   0.62   0.51   0.05\n",
            "17030  SprA_N               PF14349.9          25     1.85    508   0.59   0.57   0.49   0.03\n",
            "17031  SprB                 PF13573.9          52    22.80     37   1.47   1.43   1.45   0.23\n",
            "17032  SPRING1              PF10218.12         40     0.86    131   0.59   0.46   0.50   0.12\n",
            "17033  Sprouty              PF05210.16         40     1.22    116   0.59   0.45   0.47   0.10\n",
            "17034  SPRR2                PF14820.9           3     0.57     68   0.83   0.67   0.76   0.31\n",
            "17035  SprT-like            PF10263.12         52     3.35    116   0.59   0.54   0.47   0.05\n",
            "17036  SPRY                 PF00622.31         47     4.80    120   0.59   0.54   0.47   0.02\n",
            "17037  SPT16                PF08644.14         74     1.94    151   0.59   0.59   0.50   0.03\n",
            "17038  SPT2                 PF08243.14         39     3.26    111   0.59   0.59   0.50   0.18\n",
            "17039  Spt20                PF12090.11         24     3.45    183   0.59   0.57   0.45   0.04\n",
            "17040  Spt4                 PF06093.16         36     1.84     77   0.73   0.65   0.67   0.06\n",
            "17041  Spt46                PF17734.4           8     0.56    228   0.59   0.56   0.53   0.02\n",
            "17042  Spt5-NGN             PF03439.16         48     4.23     84   0.68   0.68   0.61   0.04\n",
            "17043  Spt5_N               PF11942.11         85     3.48     99   0.59   0.60   0.45   0.11\n",
            "17044  SPT6_acidic          PF14632.9         128     2.29     89   0.64   0.67   0.53   0.20\n",
            "17045  SPT_ssu-like         PF11779.11         61     7.45     54   1.03   0.95   0.98   0.15\n",
            "17046  SpuA_C               PF18033.4           7     0.93     93   0.61   0.64   0.54   0.03\n",
            "17047  Spuma_A9PTase        PF03539.17          4     0.53    163   0.59   0.56   0.52   0.02\n",
            "17048  SpvB                 PF03534.16         43     1.24    290   0.59   0.54   0.50   0.02\n",
            "17049  SpvD                 PF05563.15          1     0.39    212   0.59   0.57   0.52   0.03\n",
            "17050  SPW                  PF03779.17         45     7.12     48   1.15   1.14   1.12   0.24\n",
            "17051  SPX                  PF03105.22        199    10.55    378   0.59   0.59   0.45   0.06\n",
            "17052  Spy1                 PF11357.11         29     1.04    131   0.59   0.54   0.53   0.04\n",
            "17053  SP_C-Propep          PF08999.13          2     0.43     96   0.59   0.61   0.52   0.04\n",
            "17054  SQAPI                PF16845.8          13     1.14     83   0.68   0.69   0.63   0.05\n",
            "17055  SQHop_cyclase_C      PF13243.9          14     0.69    319   0.59   0.55   0.53   0.02\n",
            "17056  SQHop_cyclase_N      PF13249.9         121     2.32    291   0.59   0.57   0.54   0.01\n",
            "17057  SQS_PSY              PF00494.22         88     7.08    263   0.59   0.59   0.48   0.02\n",
            "17058  Squash               PF00299.21         12     2.06     29   1.85   1.52   1.81   0.47\n",
            "17059  SR-25                PF10500.12         10     0.75    225   0.59   0.60   0.51   0.10\n",
            "17060  SR1P                 PF13790.9          11     2.75     37   1.47   1.27   1.44   0.24\n",
            "17061  SRA                  PF18491.4          17     2.49    145   0.59   0.53   0.51   0.04\n",
            "17062  SRA1                 PF07304.14          7     0.68    147   0.59   0.58   0.52   0.03\n",
            "17063  SRAP                 PF02586.17        404     2.03    220   0.59   0.53   0.44   0.02\n",
            "17064  SRC-1                PF08832.13         32     1.33     92   0.62   0.65   0.51   0.06\n",
            "17065  SRCR                 PF00530.21         66     2.53    100   0.59   0.51   0.47   0.07\n",
            "17066  SRCR_2               PF15494.9          32     1.65     99   0.59   0.51   0.52   0.05\n",
            "17067  Sre                  PF03125.21         19     2.88    365   0.59   0.54   0.53   0.07\n",
            "17068  SRF-TF               PF00319.21        231    10.30     48   1.15   1.16   1.13   0.06\n",
            "17069  SrfB                 PF07520.14         19     1.19    988   0.59   0.58   0.51   0.01\n",
            "17070  Srg                  PF02118.24         47     3.92    275   0.59   0.55   0.55   0.10\n",
            "17071  SRI                  PF08236.14         98     4.21     85   0.67   0.64   0.56   0.09\n",
            "17072  SRP-alpha_N          PF04086.16        103     4.17    301   0.59   0.59   0.43   0.06\n",
            "17073  SRP14                PF02290.18        123     3.06    109   0.59   0.60   0.48   0.05\n",
            "17074  SRP19                PF01922.20        199     2.41     95   0.60   0.59   0.51   0.03\n",
            "17075  SRP1_TIP1            PF00660.20         57     2.21     99   0.59   0.57   0.53   0.04\n",
            "17076  SRP40_C              PF05022.15         85     2.45     76   0.74   0.71   0.65   0.09\n",
            "17077  SRP54                PF00448.25         49     1.74    196   0.59   0.62   0.54   0.03\n",
            "17078  SRP54_N              PF02881.22       1220    11.25     75   0.75   0.84   0.66   0.08\n",
            "17079  SRP68                PF16969.8          68     3.24    563   0.59   0.60   0.50   0.04\n",
            "17080  SRP72                PF08492.15         23     2.97     58   0.96   0.92   0.90   0.17\n",
            "17081  SRP9-21              PF05486.15         95     2.88     97   0.59   0.59   0.51   0.05\n",
            "17082  SRPRB                PF09439.13          7     1.09    181   0.59   0.62   0.52   0.03\n",
            "17083  SRP_SPB              PF02978.22        742     2.20    101   0.59   0.58   0.47   0.05\n",
            "17084  SRP_TPR_like         PF17004.8           9     0.99    118   0.59   0.58   0.51   0.06\n",
            "17085  SRR                  PF07709.14         14    14.00     14   1.43   1.38   1.43   0.26\n",
            "17086  SRR1                 PF07985.15        132     5.85    177   0.59   0.58   0.41   0.02\n",
            "17087  SRRM_C               PF15230.9          11     1.33     67   0.84   0.81   0.77   0.20\n",
            "17088  SRTM1                PF15872.8           3     0.55    103   0.59   0.63   0.51   0.05\n",
            "17089  SRX                  PF09201.13         12     0.85    148   0.59   0.57   0.52   0.04\n",
            "17090  SSB                  PF00436.28         48     4.19    104   0.59   0.59   0.52   0.03\n",
            "17091  ssDBP                PF17878.4           1     0.50     72   0.78   0.76   0.72   0.02\n",
            "17092  ssDBP_DBD            PF18361.4           1     0.37    136   0.60   0.59   0.52   0.01\n",
            "17093  ssDNA-exonuc_C       PF10141.12         39     2.79    202   0.59   0.57   0.53   0.04\n",
            "17094  ssDNA_DBD            PF18333.4           6     1.21    106   0.59   0.61   0.54   0.07\n",
            "17095  ssDNA_TraI_N         PF18272.4           4     1.45     53   1.05   1.12   1.01   0.07\n",
            "17096  SSDP                 PF04503.16         15     1.04    291   0.59   0.58   0.40   0.07\n",
            "17097  SseB                 PF07179.15        128     9.19    126   0.59   0.62   0.44   0.03\n",
            "17098  SseB_C               PF14581.9          47     4.06    106   0.59   0.59   0.52   0.02\n",
            "17099  SseC                 PF04888.15         17     4.82    315   0.59   0.64   0.49   0.08\n",
            "17100  SSF                  PF00474.20         10     1.52    406   0.59   0.62   0.53   0.06\n",
            "17101  SSFA2_C              PF14723.9           6     0.69    171   0.59   0.59   0.51   0.03\n",
            "17102  SsgA                 PF04686.15         93     2.12     98   0.59   0.61   0.52   0.01\n",
            "17103  SSI                  PF00720.20         22     1.18     92   0.62   0.58   0.55   0.03\n",
            "17104  Ssl1                 PF04056.17          6     0.82    193   0.59   0.60   0.53   0.02\n",
            "17105  SSL_N                PF20067.2          74     8.87     46   1.20   1.24   1.18   0.04\n",
            "17106  SSL_OB               PF09199.13         11     1.52     84   0.68   0.66   0.63   0.05\n",
            "17107  SSP160               PF06933.14          2     0.46    756   0.59   0.60   0.52   0.04\n",
            "17108  SspB                 PF04386.16        113     2.82    160   0.59   0.59   0.47   0.03\n",
            "17109  SspH                 PF08141.15         60     4.95     58   0.96   0.95   0.93   0.08\n",
            "17110  SSPI                 PF14098.9          28     1.76     65   0.86   0.89   0.82   0.04\n",
            "17111  SspK                 PF08176.15         12     1.95     47   1.17   1.12   1.12   0.13\n",
            "17112  SspN                 PF08177.14         10     2.03     46   1.20   1.13   1.16   0.15\n",
            "17113  SspO                 PF08175.15          8     2.15     50   1.11   1.06   1.05   0.18\n",
            "17114  SspP                 PF08179.15          7     2.18     44   1.25   1.19   1.20   0.24\n",
            "17115  SSrecog              PF03531.17         90     3.26     69   0.82   0.80   0.74   0.03\n",
            "17116  SSSPR-51             PF18877.3          11     3.23     48   1.15   1.12   1.10   0.11\n",
            "17117  SSTK-IP              PF15836.8           6     0.60    125   0.59   0.60   0.52   0.03\n",
            "17118  Ssu72                PF04722.16        103     1.22    191   0.59   0.56   0.52   0.02\n",
            "17119  SSURE                PF11966.11          5     0.53    149   0.59   0.60   0.52   0.02\n",
            "17120  SSV1_ORF_D-335       PF07935.14         10     1.11     67   0.84   0.82   0.74   0.05\n",
            "17121  SSXRD                PF09514.13         13     5.44     31   1.74   1.71   1.71   0.25\n",
            "17122  SSXT                 PF05030.15         22     2.38     62   0.90   0.90   0.86   0.08\n",
            "17123  ST7                  PF04184.15          7     0.62    532   0.59   0.58   0.52   0.02\n",
            "17124  STAC2_u1             PF16664.8          15     1.12    126   0.59   0.60   0.50   0.04\n",
            "17125  STAG                 PF08514.14        101     3.12    114   0.59   0.58   0.53   0.05\n",
            "17126  STALD                PF18185.4          20     1.41    207   0.59   0.61   0.53   0.02\n",
            "17127  Stanniocalcin        PF03298.16          8     0.71    200   0.59   0.56   0.52   0.04\n",
            "17128  Staphopain_pro       PF14731.9           3     0.64    169   0.59   0.59   0.52   0.04\n",
            "17129  Staphostatin_A       PF09022.13          2     0.57    105   0.59   0.57   0.53   0.03\n",
            "17130  Staphostatin_B       PF09023.13          1     0.39    105   0.59   0.56   0.52   0.04\n",
            "17131  Staphylcoagulse      PF04022.15          2     1.86     27   1.98   1.91   1.95   0.28\n",
            "17132  Staphylokinase       PF02821.19          3     1.06    112   0.59   0.59   0.46   0.04\n",
            "17133  Staph_opine_DH       PF10100.12          9     0.90    424   0.59   0.59   0.53   0.02\n",
            "17134  Stap_Strp_toxin      PF01123.23          9     2.27     87   0.65   0.62   0.58   0.05\n",
            "17135  Stap_Strp_tox_C      PF02876.20          2     0.60    106   0.59   0.56   0.52   0.04\n",
            "17136  START                PF01852.22         16     5.32    209   0.59   0.55   0.51   0.02\n",
            "17137  START_2              PF19569.2           8     0.49    126   0.59   0.56   0.52   0.04\n",
            "17138  STAR_dimer           PF16544.8          12     2.34     53   1.05   1.05   1.00   0.07\n",
            "17139  STAS                 PF01740.24         68     8.26    117   0.59   0.61   0.51   0.02\n",
            "17140  STAS_2               PF13466.9          54    10.82     80   0.71   0.80   0.67   0.06\n",
            "17141  STAT1_TAZ2bind       PF12162.11          6     6.00     23   2.20   2.17   2.13   0.26\n",
            "17142  STAT2_C              PF12188.11          7     1.62     56   0.99   0.99   0.95   0.07\n",
            "17143  STAT6_C              PF14596.9           6     0.59    193   0.59   0.57   0.51   0.02\n",
            "17144  STATa_Ig             PF18214.4           6     1.20    122   0.59   0.60   0.52   0.04\n",
            "17145  Statherin            PF03875.16          2     1.00     41   1.33   1.23   1.26   0.23\n",
            "17146  Stathmin             PF00836.22         23     1.88    136   0.59   0.61   0.54   0.10\n",
            "17147  STAT_alpha           PF01017.23         55     3.38    169   0.59   0.57   0.51   0.07\n",
            "17148  STAT_bind            PF02864.18         47     2.27    135   0.59   0.59   0.50   0.04\n",
            "17149  STAT_int             PF02865.20         51     2.13    119   0.59   0.56   0.52   0.05\n",
            "17150  Staufen_C            PF16482.8           6     0.82    111   0.59   0.61   0.51   0.02\n",
            "17151  Staygreen            PF12638.10         32     1.38    148   0.59   0.57   0.52   0.02\n",
            "17152  Stb3                 PF10330.12         37     0.91     93   0.61   0.61   0.53   0.02\n",
            "17153  StbA                 PF06406.14          6     0.81    317   0.59   0.60   0.53   0.02\n",
            "17154  STb_secrete          PF09075.13          2     0.78     48   1.15   1.11   1.09   0.10\n",
            "17155  Stc1                 PF12898.10         58     2.28     85   0.67   0.52   0.57   0.16\n",
            "17156  STD1                 PF17235.5          40     2.91    204   0.59   0.58   0.41   0.04\n",
            "17157  STE                  PF02200.19         18     0.61    110   0.59   0.55   0.52   0.03\n",
            "17158  STE2                 PF02116.18         85     3.07    280   0.59   0.59   0.54   0.08\n",
            "17159  STE3                 PF02076.18        173     3.66    289   0.59   0.53   0.53   0.06\n",
            "17160  Ste5                 PF11610.11          1     1.00     30   1.38   1.39   1.34   0.15\n",
            "17161  Ste5_C               PF12194.11         14     1.03    189   0.59   0.59   0.53   0.03\n",
            "17162  Stealth_CR1          PF17101.8         115    57.84     29   1.85   1.74   1.86   0.15\n",
            "17163  Stealth_CR2          PF11380.11         96     1.17    108   0.59   0.54   0.53   0.03\n",
            "17164  Stealth_CR3          PF17102.8         101    13.89     49   1.13   1.02   1.09   0.11\n",
            "17165  Stealth_CR4          PF17103.8          44     5.90     55   1.01   0.95   0.92   0.07\n",
            "17166  STELLO               PF03385.20         13     0.56    387   0.59   0.54   0.52   0.02\n",
            "17167  Sterile              PF03015.22        311     4.60     93   0.61   0.55   0.57   0.04\n",
            "17168  Steroid_dh           PF02544.19         10     1.30    150   0.59   0.52   0.52   0.05\n",
            "17169  Sterol-sensing       PF12349.11         22     2.10    153   0.59   0.60   0.55   0.06\n",
            "17170  Sterol_MT_C          PF08498.13        120     3.18     66   0.85   0.86   0.82   0.04\n",
            "17171  Stevor               PF17410.5          19     0.90    275   0.59   0.57   0.52   0.03\n",
            "17172  STG                  PF15809.8           7     0.76    238   0.59   0.56   0.52   0.03\n",
            "17173  STI1                 PF17830.4          46     8.40     55   1.01   0.98   0.98   0.09\n",
            "17174  Stig1                PF04885.16         41     1.09    140   0.59   0.45   0.49   0.15\n",
            "17175  STIL_N               PF15253.9          11     0.80    404   0.59   0.59   0.53   0.01\n",
            "17176  STIMATE              PF12400.11        107     1.61    125   0.59   0.55   0.50   0.07\n",
            "17177  Stimulus_sens_1      PF13756.9          30     3.15    111   0.59   0.62   0.50   0.03\n",
            "17178  Stirrup              PF09061.9           2     0.53     79   0.71   0.71   0.65   0.03\n",
            "17179  STIV_B116-like       PF08960.13          9     1.37    104   0.59   0.60   0.51   0.03\n",
            "17180  Stk19                PF10494.12         47     2.13    245   0.59   0.60   0.47   0.02\n",
            "17181  Stm1_N               PF09598.13         67     5.97     65   0.86   0.90   0.74   0.12\n",
            "17182  STN                  PF07660.17        151   141.12     52   1.07   1.14   1.01   0.07\n",
            "17183  Stn1                 PF10451.12          5     1.57    252   0.59   0.56   0.51   0.02\n",
            "17184  STN1_2               PF09170.13         22     1.11    167   0.59   0.58   0.53   0.04\n",
            "17185  Stn1_C               PF12659.10         13     2.38    165   0.59   0.58   0.53   0.05\n",
            "17186  stn_TNFRSF12A        PF12191.11          3     0.41    129   0.59   0.59   0.52   0.02\n",
            "17187  Stomagen             PF16851.8           7     0.95     50   1.10   0.94   1.05   0.15\n",
            "17188  Stomoxyn             PF11585.11          1     1.00     40   1.30   1.37   1.25   0.17\n",
            "17189  Stonin2_N            PF12016.11          4     0.59    338   0.59   0.58   0.52   0.04\n",
            "17190  STOP                 PF05217.15         23     5.41     36   1.51   1.16   1.47   0.47\n",
            "17191  Stork_head           PF10264.12         15     1.39     79   0.72   0.70   0.67   0.03\n",
            "17192  STPPase_N            PF16891.8         206     8.85     48   1.15   1.22   1.08   0.09\n",
            "17193  Strabismus           PF06638.14         17     0.81    504   0.59   0.58   0.51   0.02\n",
            "17194  Streptin-Immun       PF11083.11          3     0.65     93   0.62   0.63   0.55   0.07\n",
            "17195  Strep_his_triad      PF04270.16         14     3.72     51   1.09   1.06   1.02   0.05\n",
            "17196  Strep_pep            PF14404.9           7     1.06     63   0.89   0.89   0.84   0.03\n",
            "17197  Strep_SA_rep         PF06696.14          9     9.00     25   1.92   1.95   1.89   0.45\n",
            "17198  Stress-antifung      PF01657.20        425     3.87     94   0.61   0.51   0.23   0.10\n",
            "17199  Striatin             PF08232.15         92     2.84    155   0.59   0.59   0.48   0.08\n",
            "17200  Strumpellin          PF10266.12         33     1.58   1085   0.59   0.58   0.53   0.03\n",
            "17201  Str_synth            PF03088.19          6     1.25     89   0.64   0.63   0.59   0.02\n",
            "17202  STT3                 PF02516.17         10     2.84    488   0.59   0.58   0.50   0.05\n",
            "17203  STT3_PglB_C          PF18527.4           7     1.69     80   0.71   0.71   0.66   0.05\n",
            "17204  Styelin              PF17562.5           2     0.80     59   0.94   0.90   0.88   0.06\n",
            "17205  SUa-2TM              PF18179.4           3     0.96    279   0.59   0.57   0.51   0.04\n",
            "17206  Sua5_C               PF03481.16        557     4.52    135   0.59   0.61   0.35   0.03\n",
            "17207  Sua5_yciO_yrdC       PF01300.21        107     4.07    179   0.59   0.61   0.51   0.01\n",
            "17208  SUB1_ProdP9          PF18213.4           5     0.61     80   0.71   0.76   0.65   0.04\n",
            "17209  Sublancin            PF19151.3           2     0.77     57   0.98   0.92   0.92   0.11\n",
            "17210  Subtilosin_A         PF11420.11          4     1.32     35   1.55   1.52   1.51   0.20\n",
            "17211  Succ_CoA_lig         PF13607.9         136     4.07    138   0.59   0.62   0.56   0.03\n",
            "17212  Succ_DH_flav_C       PF02910.23        625     2.86    129   0.59   0.57   0.51   0.04\n",
            "17213  Sucrose_synth        PF00862.22         22     0.76    548   0.59   0.58   0.53   0.01\n",
            "17214  Suc_Fer-like         PF06999.15        248     2.58    195   0.59   0.56   0.39   0.02\n",
            "17215  Suf                  PF05843.17         19     2.12    296   0.59   0.57   0.50   0.03\n",
            "17216  SUFBD                PF01458.20        805     5.81    218   0.59   0.60   0.51   0.03\n",
            "17217  SufBD_N              PF19295.2          87     1.57    172   0.59   0.57   0.54   0.03\n",
            "17218  SufE                 PF02657.18        319     3.20    122   0.59   0.59   0.53   0.02\n",
            "17219  SUFU                 PF05076.16        110     6.36    172   0.59   0.58   0.42   0.01\n",
            "17220  SUFU_C               PF12470.11         14     1.33    215   0.59   0.59   0.51   0.02\n",
            "17221  Sugar-bind           PF04198.16         22     3.26    257   0.59   0.62   0.55   0.02\n",
            "17222  Sugarporin_N         PF11471.11         10    10.00     31   1.70   1.86   1.69   0.45\n",
            "17223  Sugar_tr             PF00083.27         33     4.36    452   0.59   0.57   0.52   0.05\n",
            "17224  Sugar_transport      PF06800.15         13     1.01    280   0.59   0.61   0.53   0.06\n",
            "17225  SUI1                 PF01253.25        337     6.10     79   0.72   0.74   0.63   0.05\n",
            "17226  SUIM_assoc           PF16619.8          13     1.56     64   0.88   0.86   0.78   0.13\n",
            "17227  SUKH-3               PF14433.9          38     4.01    143   0.59   0.58   0.47   0.01\n",
            "17228  SUKH-4               PF14435.9          24     5.04    176   0.59   0.60   0.48   0.02\n",
            "17229  SUKH_5               PF14567.9          27     3.85    142   0.59   0.56   0.50   0.02\n",
            "17230  SUKH_6               PF14568.9          51     8.04    122   0.59   0.56   0.51   0.03\n",
            "17231  SulA                 PF03846.17          9     0.67    115   0.59   0.57   0.53   0.03\n",
            "17232  Sulfakinin           PF08257.14          5     5.00      9   3.34   3.04   3.32   1.00\n",
            "17233  Sulfatase            PF00884.26         57     8.12    309   0.59   0.56   0.46   0.01\n",
            "17234  Sulfatase_C          PF14707.9          79     3.01    124   0.59   0.51   0.45   0.04\n",
            "17235  Sulfate_transp       PF00916.23         22     2.87    380   0.59   0.62   0.52   0.05\n",
            "17236  Sulfolobus_pRN       PF05584.14          2     0.66     72   0.78   0.78   0.72   0.06\n",
            "17237  Sulfotransfer_1      PF00685.30         45     5.38    267   0.59   0.55   0.46   0.02\n",
            "17238  Sulfotransfer_2      PF03567.17         65     6.30    246   0.59   0.53   0.42   0.03\n",
            "17239  Sulfotransfer_3      PF13469.9         155     7.91    217   0.59   0.56   0.47   0.01\n",
            "17240  Sulfotransfer_4      PF17784.4         218     4.00    216   0.59   0.55   0.50   0.01\n",
            "17241  Sulfotransfer_5      PF19798.2           3     0.42    226   0.59   0.57   0.52   0.01\n",
            "17242  Sulf_coat_C          PF12193.11          8     0.71    134   0.59   0.58   0.51   0.03\n",
            "17243  Sulf_transp          PF04143.17         76     4.55    310   0.59   0.60   0.51   0.07\n",
            "17244  Sulphotransf         PF09037.13          6     0.84    244   0.59   0.57   0.52   0.02\n",
            "17245  SUN                  PF03856.16        167     1.10    245   0.59   0.51   0.49   0.04\n",
            "17246  Suppressor_APC       PF11414.11         24     2.48     82   0.69   0.68   0.66   0.07\n",
            "17247  Suppressor_P21       PF11479.11          6     1.98     80   0.71   0.71   0.65   0.06\n",
            "17248  SUR7                 PF06687.15         93     5.78    212   0.59   0.57   0.43   0.07\n",
            "17249  SurA_N               PF09312.14         13     1.04    118   0.59   0.63   0.54   0.05\n",
            "17250  SurA_N_2             PF13623.9         155     4.89    145   0.59   0.60   0.51   0.04\n",
            "17251  SurA_N_3             PF13624.9         142     5.14    162   0.59   0.60   0.51   0.04\n",
            "17252  SurE                 PF01975.20        646     2.89    195   0.59   0.61   0.44   0.02\n",
            "17253  SURF1                PF02104.18        457     5.14    195   0.59   0.56   0.27   0.02\n",
            "17254  SURF2                PF05477.14         21     1.61    242   0.59   0.56   0.51   0.07\n",
            "17255  SURF4                PF02077.18          6     0.79    267   0.59   0.59   0.53   0.04\n",
            "17256  SURF6                PF04935.15        123     3.34    202   0.59   0.60   0.50   0.17\n",
            "17257  Surface_Ag_2         PF01617.20         10     1.17    250   0.59   0.58   0.50   0.02\n",
            "17258  Surface_antigen      PF11054.11        188     2.10    207   0.59   0.54   0.44   0.05\n",
            "17259  Surfac_D-trimer      PF09006.14          5     1.40     46   1.20   1.23   1.16   0.05\n",
            "17260  SURNod19             PF07712.15         23     0.79    377   0.59   0.54   0.52   0.02\n",
            "17261  Surp                 PF01805.23        312    14.52     53   1.05   0.99   0.98   0.07\n",
            "17262  SusD-like            PF12741.10         25     1.43    496   0.59   0.55   0.52   0.03\n",
            "17263  SusD-like_2          PF12771.10         17     1.80    415   0.59   0.56   0.46   0.03\n",
            "17264  SusD-like_3          PF14322.9          15     3.28    190   0.59   0.58   0.46   0.03\n",
            "17265  SusD_RagB            PF07980.14         37     6.38    299   0.59   0.54   0.46   0.04\n",
            "17266  SusE                 PF14292.9          84     9.57    106   0.59   0.61   0.48   0.05\n",
            "17267  SusF_SusE            PF16411.8          43     3.99    130   0.59   0.54   0.31   0.04\n",
            "17268  Sushi                PF00084.23         32     8.27     56   0.99   0.80   0.87   0.13\n",
            "17269  Sushi_2              PF09014.13         24     0.96     88   0.65   0.55   0.59   0.06\n",
            "17270  SUV3_C               PF12513.11        157    16.74     48   1.15   1.12   1.12   0.09\n",
            "17271  Suv3_C_1             PF18147.4          33    13.63     43   1.28   1.29   1.25   0.08\n",
            "17272  Suv3_N               PF18114.4           8     0.89    116   0.59   0.58   0.53   0.03\n",
            "17273  SUZ                  PF12752.10        138    11.27     59   0.95   0.97   0.84   0.14\n",
            "17274  SUZ-C                PF12901.10         19    14.36     33   1.64   1.62   1.61   0.19\n",
            "17275  SVA                  PF05326.14         10     1.11    124   0.59   0.56   0.52   0.05\n",
            "17276  Svf1                 PF08622.13        108     2.02    163   0.59   0.55   0.46   0.03\n",
            "17277  Svf1_C               PF17187.7         108     1.60    164   0.59   0.55   0.49   0.03\n",
            "17278  SVIP                 PF15811.8          27     2.32     77   0.74   0.73   0.66   0.12\n",
            "17279  SVM_signal           PF12113.11         18     5.03     33   1.64   1.63   1.60   0.38\n",
            "17280  Svs_4_5_6            PF17381.5           6     1.40     91   0.63   0.64   0.54   0.11\n",
            "17281  SVS_QK               PF10578.12         11    11.00     12   2.35   2.28   2.34   0.75\n",
            "17282  SVWC                 PF15430.9          80     3.46     66   0.85   0.65   0.73   0.19\n",
            "17283  SWC7                 PF17330.5          13     0.92     98   0.59   0.60   0.53   0.04\n",
            "17284  SWI-SNF_Ssr4_C       PF20497.1          28     1.60    460   0.59   0.59   0.49   0.04\n",
            "17285  SWI-SNF_Ssr4_N       PF08549.13         28     0.83    217   0.59   0.54   0.52   0.02\n",
            "17286  SWI2_SNF2            PF18766.4         292     1.60    225   0.59   0.58   0.48   0.03\n",
            "17287  Swi3                 PF07962.15        115     3.72     84   0.68   0.65   0.61   0.03\n",
            "17288  Swi5                 PF07061.14         57     3.57     80   0.71   0.73   0.66   0.06\n",
            "17289  Swi6_N               PF18530.4           8     1.06    108   0.59   0.60   0.52   0.03\n",
            "17290  SWIB                 PF02201.21        333     5.82     74   0.76   0.75   0.71   0.04\n",
            "17291  SWIM                 PF04434.20         85    85.00     40   1.31   1.15   1.22   0.19\n",
            "17292  SWIRM                PF04433.20        225     4.82     90   0.63   0.58   0.50   0.04\n",
            "17293  SWIRM-assoc_1        PF16495.8          54     3.73     84   0.68   0.74   0.66   0.11\n",
            "17294  SWIRM-assoc_2        PF16496.8          23     0.99    412   0.59   0.56   0.51   0.03\n",
            "17295  SWIRM-assoc_3        PF16498.8          18     1.72     67   0.84   0.89   0.78   0.06\n",
            "17296  Swm2                 PF17083.8          10     1.33    130   0.59   0.58   0.52   0.05\n",
            "17297  SWM_repeat           PF13753.9          61     2.75     86   0.66   0.70   0.57   0.06\n",
            "17298  SwrA                 PF17423.5           1     0.39    116   0.59   0.59   0.52   0.04\n",
            "17299  SxtJ                 PF19588.2           2     0.40     99   0.59   0.58   0.52   0.05\n",
            "17300  Sybindin             PF04099.15         11     1.21    143   0.59   0.58   0.51   0.02\n",
            "17301  SYCE1                PF15233.9          14     1.43    152   0.59   0.61   0.54   0.10\n",
            "17302  SYCP2_ARLD           PF18581.4          18     1.23    175   0.59   0.60   0.54   0.05\n",
            "17303  SYCP2_SLD            PF18584.4          21     0.97    112   0.59   0.56   0.53   0.03\n",
            "17304  Syd                  PF07348.15         33     1.75    175   0.59   0.57   0.54   0.02\n",
            "17305  SYF2                 PF08231.15         91     2.50    163   0.59   0.55   0.49   0.11\n",
            "17306  Syja_N               PF02383.21        450     3.45    323   0.59   0.56   0.39   0.02\n",
            "17307  SymE_toxin           PF08845.13         22     5.03     54   1.03   1.02   0.91   0.03\n",
            "17308  SYMPK_PTA1_N         PF11935.11         92     4.26    228   0.59   0.59   0.47   0.04\n",
            "17309  Symplekin_C          PF12295.11         50     3.26    186   0.59   0.59   0.50   0.04\n",
            "17310  Synaphin             PF05835.15         22     1.76    142   0.59   0.60   0.51   0.09\n",
            "17311  Synapsin             PF02078.19         29     0.96    102   0.59   0.57   0.51   0.02\n",
            "17312  Synapsin_C           PF02750.17          2     0.41    203   0.60   0.57   0.52   0.02\n",
            "17313  Synapsin_N           PF10581.12          1     1.00     32   1.50   1.42   1.45   0.19\n",
            "17314  Synaptobrevin        PF00957.24         93     4.32     89   0.64   0.63   0.62   0.06\n",
            "17315  Synaptonemal_3       PF15191.9           2     0.68     90   0.63   0.62   0.56   0.05\n",
            "17316  Syncollin            PF15138.9           9     0.59    112   0.59   0.53   0.53   0.03\n",
            "17317  Syndecan             PF01034.23         14     1.58     63   0.89   0.91   0.81   0.04\n",
            "17318  Syntaphilin          PF15290.9          18     0.87    305   0.59   0.59   0.52   0.04\n",
            "17319  Syntaxin             PF00804.28         67     2.17    199   0.59   0.60   0.54   0.07\n",
            "17320  Syntaxin-18_N        PF10496.12         71     5.92     87   0.65   0.65   0.54   0.07\n",
            "17321  Syntaxin-5_N         PF11416.11         29    29.00     23   2.11   2.08   2.11   0.23\n",
            "17322  Syntaxin-6_N         PF09177.14        138     4.32    100   0.59   0.60   0.50   0.06\n",
            "17323  Syntaxin_2           PF14523.9         124     7.53    101   0.59   0.60   0.53   0.10\n",
            "17324  Synthase_beta        PF11421.11         10     2.18     49   1.13   1.21   1.07   0.22\n",
            "17325  Synuclein            PF01387.20         12     0.95    134   0.59   0.63   0.52   0.07\n",
            "17326  SyrA                 PF11089.11          4     2.29     38   1.44   1.47   1.41   0.29\n",
            "17327  SYS1                 PF09801.12         25     1.50    144   0.59   0.55   0.54   0.07\n",
            "17328  S_100                PF01023.22        161    14.76     45   1.22   1.24   1.21   0.08\n",
            "17329  S_2TMBeta            PF18153.4          16     1.99    180   0.59   0.55   0.51   0.03\n",
            "17330  S_4TM                PF18159.4          26     3.36    290   0.59   0.57   0.55   0.03\n",
            "17331  S_layer_C            PF05124.15          8     1.57    220   0.59   0.62   0.50   0.04\n",
            "17332  S_layer_N            PF05123.15          8     1.26    284   0.59   0.62   0.48   0.04\n",
            "17333  S_locus_glycop       PF00954.23         27     1.17    111   0.59   0.47   0.51   0.05\n",
            "17334  S_tail_recep_bd      PF14928.9           1     0.39     93   0.62   0.58   0.54   0.02\n",
            "17335  T-box                PF00907.25        101     1.58    185   0.59   0.54   0.50   0.03\n",
            "17336  T-box_assoc          PF16176.8          13     0.76    226   0.59   0.57   0.47   0.03\n",
            "17337  T2SS-T3SS_pil_N      PF13629.9          87     8.76     72   0.78   0.85   0.76   0.05\n",
            "17338  T2SSB                PF16537.8          42     5.40     60   0.93   0.94   0.90   0.03\n",
            "17339  T2SSC                PF11356.11         26     3.30    145   0.59   0.63   0.50   0.02\n",
            "17340  T2SSE                PF00437.23         17     1.63    277   0.59   0.62   0.51   0.03\n",
            "17341  T2SSE_N              PF05157.18        100    11.48    111   0.59   0.63   0.51   0.03\n",
            "17342  T2SSF                PF00482.26        247    22.07    125   0.59   0.65   0.54   0.04\n",
            "17343  T2SSG                PF08334.14         78     1.60    108   0.59   0.57   0.51   0.02\n",
            "17344  T2SSI                PF02501.20         97     6.93     80   0.71   0.71   0.65   0.05\n",
            "17345  T2SSJ                PF11612.11         73     4.36    140   0.59   0.58   0.47   0.03\n",
            "17346  T2SSK                PF03934.16        103     5.60    280   0.59   0.61   0.49   0.03\n",
            "17347  T2SSL                PF05134.16          9     2.19    233   0.59   0.60   0.52   0.02\n",
            "17348  T2SSM                PF04612.15         29     3.89    160   0.59   0.60   0.54   0.04\n",
            "17349  T2SSM_b              PF10741.12         18     3.51    111   0.59   0.64   0.54   0.03\n",
            "17350  T2SSN                PF01203.22         49     6.96    211   0.59   0.62   0.47   0.04\n",
            "17351  T2SSppdC             PF12528.11         15     3.07     82   0.69   0.58   0.65   0.08\n",
            "17352  T2SSS_2              PF16549.8          17     1.02    104   0.59   0.58   0.54   0.03\n",
            "17353  T2SS_PulS_OutS       PF09691.13          4     0.84    106   0.59   0.57   0.52   0.04\n",
            "17354  T3RM_EcoP15I_C       PF18273.4          25     1.58     97   0.59   0.58   0.54   0.04\n",
            "17355  T3SchapCesA          PF11439.11          2     0.46     95   0.60   0.63   0.53   0.04\n",
            "17356  T3SSipB              PF16535.8           4     1.17    155   0.59   0.63   0.52   0.07\n",
            "17357  T3SS_ATPase_C        PF18269.4         234     5.75     72   0.78   0.79   0.77   0.04\n",
            "17358  T3SS_basalb_I        PF17001.8          12     2.52    100   0.59   0.61   0.52   0.07\n",
            "17359  T3SS_ExsE            PF18286.4           2     1.49     46   1.20   1.26   1.14   0.07\n",
            "17360  T3SS_HrpK1           PF16937.8           5     0.86    256   0.59   0.62   0.52   0.04\n",
            "17361  T3SS_LEE_assoc       PF13327.9           6     1.06    162   0.59   0.58   0.53   0.03\n",
            "17362  T3SS_needle_E        PF08988.13          7     3.17     66   0.85   0.89   0.81   0.08\n",
            "17363  T3SS_needle_F        PF09392.13         19     3.42     71   0.79   0.80   0.75   0.07\n",
            "17364  T3SS_needle_reg      PF09025.13          3     0.82    145   0.59   0.61   0.51   0.02\n",
            "17365  T3SS_NleG            PF06416.15         12     0.89    113   0.59   0.55   0.53   0.03\n",
            "17366  T3SS_SCTL            PF06635.15          1     0.39    206   0.59   0.61   0.51   0.01\n",
            "17367  T3SS_TC              PF06511.14          3     0.71    355   0.59   0.60   0.51   0.02\n",
            "17368  T4-Gluco-transf      PF09198.13          1     1.00     38   1.35   1.35   1.30   0.13\n",
            "17369  T4-gp15_tss          PF16724.8           2     0.54    234   0.59   0.57   0.52   0.02\n",
            "17370  T4BSS_DotH_IcmK      PF12293.11          7     1.13    238   0.59   0.60   0.53   0.01\n",
            "17371  T4BSS_DotI_IcmL      PF11393.11          8     1.43    180   0.59   0.57   0.54   0.02\n",
            "17372  T4bSS_IcmS           PF12608.11          3     1.09     93   0.61   0.65   0.53   0.03\n",
            "17373  T4SS                 PF07996.14         36     4.25    197   0.59   0.61   0.50   0.07\n",
            "17374  T4SS-DNA_transf      PF02534.17          9     2.25    469   0.59   0.58   0.51   0.02\n",
            "17375  T4SS_CagC            PF16943.8           2     0.66    107   0.59   0.63   0.49   0.05\n",
            "17376  T4SS_pilin           PF18895.3          28    11.43     71   0.79   0.83   0.70   0.12\n",
            "17377  T4SS_TraI            PF16932.8           8     1.14    213   0.59   0.57   0.53   0.02\n",
            "17378  T4_baseplate         PF12322.11         18     2.76    205   0.59   0.57   0.51   0.04\n",
            "17379  T4_deiodinase        PF00837.20          3     0.68    237   0.59   0.57   0.51   0.01\n",
            "17380  T4_Gp59_C            PF08994.13          2     0.55    104   0.59   0.58   0.50   0.05\n",
            "17381  T4_Gp59_N            PF08993.13          2     0.55     93   0.61   0.59   0.55   0.04\n",
            "17382  T4_gp9_10            PF07880.14          3     0.89    285   0.59   0.59   0.50   0.02\n",
            "17383  T4_neck-protein      PF11649.11          2     0.54    278   0.59   0.57   0.52   0.02\n",
            "17384  T4_Rnl2_C            PF18043.4          15     3.46     82   0.69   0.70   0.64   0.06\n",
            "17385  T4_tail_cap          PF11091.11          6     1.16    342   0.59   0.59   0.51   0.02\n",
            "17386  T5orf172             PF10544.12         98     7.10    104   0.59   0.54   0.41   0.03\n",
            "17387  T6PP_N               PF18572.4          31     2.38    100   0.59   0.60   0.52   0.05\n",
            "17388  T6SS-SciN            PF12790.10         75     4.19    120   0.59   0.61   0.54   0.02\n",
            "17389  T6SS_FHA_C           PF20232.1          57     3.23    180   0.59   0.58   0.55   0.03\n",
            "17390  T6SS_HCP             PF05638.15        152     4.18    129   0.59   0.57   0.47   0.03\n",
            "17391  T6SS_TssF            PF05947.15         94     3.74    605   0.59   0.59   0.52   0.01\n",
            "17392  T6SS_TssG            PF06996.14        125     4.67    304   0.59   0.60   0.49   0.01\n",
            "17393  T6SS_VasE            PF05936.15        123     4.37    429   0.59   0.60   0.54   0.02\n",
            "17394  T6SS_VasJ            PF16989.8          33     3.79    257   0.59   0.59   0.54   0.04\n",
            "17395  T6SS_Vgr             PF13296.9          73     2.25    108   0.59   0.63   0.54   0.05\n",
            "17396  T6SS_VipA            PF05591.15         97     1.68    155   0.59   0.62   0.54   0.04\n",
            "17397  T6_Ig_like           PF18002.4           3     0.59    145   0.59   0.60   0.52   0.04\n",
            "17398  T7-like_gp12         PF10922.11          1     0.41     85   0.67   0.61   0.60   0.04\n",
            "17399  T7-like_gp67         PF17570.5          16     1.93     62   0.90   0.90   0.82   0.07\n",
            "17400  T7-like_Y65          PF10911.11          4     0.84     77   0.74   0.74   0.67   0.02\n",
            "17401  T7SS_ESX1_EccB       PF05108.16         48     2.18    474   0.59   0.61   0.51   0.01\n",
            "17402  T7SS_ESX_EspC        PF10824.11         21     4.46    100   0.59   0.62   0.56   0.07\n",
            "17403  Ta0938               PF11494.11         11     0.63     92   0.62   0.54   0.56   0.04\n",
            "17404  TA0956               PF11513.11          5     0.64    110   0.59   0.57   0.52   0.03\n",
            "17405  Ta1207               PF19020.3          36     8.78    277   0.59   0.58   0.56   0.04\n",
            "17406  TAA-Trp-ring         PF15401.9          20     5.18     65   0.86   0.92   0.74   0.15\n",
            "17407  Tab2-like_C          PF20429.1          79     1.77    211   0.59   0.58   0.52   0.02\n",
            "17408  Tab2-like_N          PF06485.14         82     3.70     55   1.01   0.91   0.96   0.09\n",
            "17409  TACC_C               PF05010.17         23     1.99    201   0.59   0.59   0.54   0.10\n",
            "17410  Tachykinin           PF02202.19         14    14.00     11   2.82   2.62   2.82   0.60\n",
            "17411  Tachylectin          PF14517.9           7     1.40    234   0.59   0.54   0.49   0.03\n",
            "17412  Tachystatin_A        PF11406.11          2     0.71     44   1.24   1.09   1.19   0.19\n",
            "17413  Tachystatin_B        PF11478.11          2     0.73     42   1.30   1.13   1.25   0.25\n",
            "17414  TACI-CRD2            PF09305.13         13     1.55     39   1.40   1.15   1.36   0.27\n",
            "17415  TackOD1              PF18551.4          24     1.54    188   0.59   0.49   0.54   0.07\n",
            "17416  Tad                  PF13400.9         155    43.98     48   1.15   1.30   1.06   0.26\n",
            "17417  TAD2                 PF18521.4           1     1.00     25   1.52   1.46   1.47   0.19\n",
            "17418  TadB_TadC_N          PF19360.2          88     0.67    143   0.59   0.61   0.53   0.04\n",
            "17419  TadE                 PF07811.15        349    58.47     43   1.28   1.39   1.24   0.20\n",
            "17420  TadF                 PF16964.8           9     1.48    182   0.59   0.60   0.51   0.03\n",
            "17421  TadZ_N               PF16968.8           1     0.41    133   0.59   0.64   0.51   0.02\n",
            "17422  Tad_C                PF09977.12         33     7.06     93   0.61   0.65   0.46   0.07\n",
            "17423  Tae4                 PF14113.9          26     2.25    121   0.59   0.54   0.39   0.03\n",
            "17424  Taeniidae_ag         PF05596.14          6     1.50     64   0.88   0.89   0.84   0.06\n",
            "17425  TAF                  PF02969.20          8     1.90     66   0.85   0.89   0.81   0.05\n",
            "17426  TAF1D                PF15333.9          12     0.88    220   0.59   0.58   0.51   0.04\n",
            "17427  TAF1_subA            PF14929.9          34     2.88    371   0.59   0.55   0.46   0.04\n",
            "17428  TAF4                 PF05236.17         67     3.47    273   0.59   0.62   0.48   0.06\n",
            "17429  TAF6_C               PF07571.16        111     3.13     90   0.63   0.66   0.54   0.03\n",
            "17430  TAF8_C               PF10406.12        113    11.28     49   1.13   1.08   1.11   0.10\n",
            "17431  TAFA                 PF12020.11          8     0.61     89   0.64   0.56   0.57   0.07\n",
            "17432  TAFH                 PF07531.17         28     2.09     90   0.63   0.66   0.59   0.05\n",
            "17433  Tafi-CsgC            PF10610.12          4     0.71     98   0.59   0.59   0.53   0.03\n",
            "17434  TAFII28              PF04719.17         77     2.33     86   0.66   0.66   0.59   0.04\n",
            "17435  TAFII55_N            PF04658.16        121     1.85    161   0.59   0.59   0.49   0.04\n",
            "17436  TagA                 PF12561.11         27     1.40    101   0.59   0.57   0.50   0.02\n",
            "17437  TagF_N               PF09867.12         47     2.33    135   0.59   0.57   0.54   0.01\n",
            "17438  Tai4                 PF16695.8          13     1.79     92   0.62   0.59   0.57   0.04\n",
            "17439  Tail_P2_I            PF09684.13         62     3.59    136   0.59   0.59   0.47   0.03\n",
            "17440  Tail_spike_N         PF18668.4           6     1.47     70   0.80   0.78   0.76   0.04\n",
            "17441  Tail_tube            PF10618.12         25     3.43    117   0.59   0.59   0.55   0.03\n",
            "17442  Takusan              PF04822.16         32     3.65     85   0.67   0.66   0.63   0.09\n",
            "17443  Talin_middle         PF09141.13         21     1.72    161   0.59   0.63   0.52   0.05\n",
            "17444  TALPID3              PF15324.9           8     0.99   1287   0.59   0.60   0.51   0.03\n",
            "17445  TAL_effector         PF03377.16         41    15.12     33   1.64   1.73   1.60   0.10\n",
            "17446  TAL_FSA              PF00923.22        122     4.11    287   0.59   0.61   0.47   0.03\n",
            "17447  Tam41_Mmp37          PF09139.14        114     1.49    323   0.59   0.57   0.51   0.02\n",
            "17448  TamB                 PF04357.16         37     8.39    383   0.59   0.64   0.43   0.02\n",
            "17449  TAN                  PF11640.11         91     5.47    151   0.59   0.61   0.50   0.04\n",
            "17450  TANGO2               PF05742.15        103     2.78    254   0.59   0.57   0.43   0.02\n",
            "17451  Tankyrase_bdg_C      PF15327.9          15     1.32    171   0.59   0.60   0.48   0.06\n",
            "17452  Tannase              PF07519.14          7     1.49    469   0.59   0.55   0.50   0.01\n",
            "17453  Tantalus             PF15386.9          10     2.83     57   0.98   0.96   0.83   0.10\n",
            "17454  Tap-RNA_bind         PF09162.13         53     2.47     80   0.71   0.65   0.67   0.04\n",
            "17455  TAP35_44             PF07265.14          7     0.71    119   0.59   0.58   0.49   0.02\n",
            "17456  TAP42                PF04177.15        147     4.11    349   0.59   0.59   0.47   0.05\n",
            "17457  Tape_meas_lam_C      PF09718.13         34     5.57     76   0.74   0.75   0.71   0.07\n",
            "17458  TAP_C                PF03943.16        136     8.21     49   1.13   1.03   1.11   0.08\n",
            "17459  Taq-exonuc           PF09281.13          8     1.07    132   0.59   0.61   0.51   0.03\n",
            "17460  TaqI_C               PF12950.10         20     2.51    120   0.59   0.56   0.49   0.03\n",
            "17461  TarH                 PF02203.18         67     6.42    177   0.59   0.62   0.55   0.05\n",
            "17462  TarS_C1              PF18674.4          13     2.96    148   0.59   0.57   0.54   0.04\n",
            "17463  TAS2R                PF05296.16         20     1.86    303   0.59   0.57   0.54   0.09\n",
            "17464  Tash_PEST            PF07708.14         22    22.00     18   2.53   2.64   2.52   0.73\n",
            "17465  TASL                 PF15133.9          15     1.40    304   0.59   0.58   0.51   0.03\n",
            "17466  Tat                  PF00539.21          2     0.69     63   0.88   0.76   0.81   0.11\n",
            "17467  TatA_B_E             PF02416.19         11     2.83     53   1.05   1.14   0.98   0.05\n",
            "17468  TatC                 PF00902.21        633     3.89    207   0.59   0.60   0.45   0.08\n",
            "17469  TatD_DNase           PF01026.24         79     4.45    255   0.59   0.58   0.46   0.02\n",
            "17470  TATR                 PF03430.16          1     0.39    575   0.59   0.56   0.51   0.03\n",
            "17471  TAtT                 PF16811.8          16     1.92    263   0.59   0.60   0.55   0.02\n",
            "17472  TAT_signal           PF10518.12        403   403.00     26   1.62   1.79   1.61   0.34\n",
            "17473  TAT_ubiq             PF07706.15         22     4.49     40   1.37   1.35   1.27   0.11\n",
            "17474  Tau95                PF09734.12         62     2.77    140   0.59   0.54   0.47   0.03\n",
            "17475  Tau95_N              PF17682.4          71     3.63    127   0.59   0.59   0.47   0.02\n",
            "17476  TauD                 PF02668.19        119    12.08    268   0.59   0.56   0.46   0.02\n",
            "17477  TauE                 PF01925.22        536    11.49    237   0.59   0.66   0.47   0.13\n",
            "17478  Tautomerase          PF01361.24         20     7.77     60   0.93   0.96   0.90   0.04\n",
            "17479  Tautomerase_2        PF14552.9          68     3.61     82   0.69   0.67   0.65   0.03\n",
            "17480  Tautomerase_3        PF14832.9          17     1.35    137   0.59   0.55   0.52   0.02\n",
            "17481  Tax                  PF02959.19          2     0.39    222   0.59   0.56   0.52   0.02\n",
            "17482  Taxilin              PF09728.12         52     3.88    302   0.59   0.59   0.55   0.13\n",
            "17483  TAXi_C               PF14541.9          57     3.67    161   0.59   0.57   0.52   0.01\n",
            "17484  TAXi_N               PF14543.9         199     3.79    178   0.59   0.56   0.48   0.03\n",
            "17485  TA_inhibitor         PF17574.5           1     0.46     89   0.65   0.64   0.58   0.05\n",
            "17486  TB                   PF00683.20        208     5.84     44   1.25   0.98   1.18   0.31\n",
            "17487  TB2_DP1_HVA22        PF03134.22        313     4.83     78   0.73   0.65   0.62   0.09\n",
            "17488  TBC1D23_C            PF19430.2           3     0.43    226   0.59   0.58   0.51   0.02\n",
            "17489  TBCA                 PF02970.19        145     3.46     99   0.59   0.62   0.53   0.09\n",
            "17490  TBCC                 PF07986.15         40     3.12    120   0.59   0.54   0.54   0.05\n",
            "17491  TBCC_N               PF16752.8          58     3.82    115   0.59   0.61   0.55   0.08\n",
            "17492  TBD                  PF12845.10         10     3.13     56   0.99   0.96   0.97   0.07\n",
            "17493  TBK1_CCD1            PF18394.4          10     1.57    256   0.59   0.58   0.54   0.05\n",
            "17494  TBK1_ULD             PF18396.4          23     1.64     88   0.65   0.62   0.60   0.05\n",
            "17495  TBP                  PF00352.24        274     4.58     85   0.67   0.67   0.60   0.03\n",
            "17496  TBP-binding          PF09247.14         18     1.77     61   0.92   0.98   0.85   0.09\n",
            "17497  TbpB_A               PF17484.5           7     1.69    135   0.59   0.57   0.49   0.04\n",
            "17498  TbpB_B_D             PF01298.21         54     2.51    135   0.59   0.59   0.44   0.06\n",
            "17499  TbpB_C               PF17483.5           9     1.32    109   0.59   0.57   0.48   0.04\n",
            "17500  TBPIP                PF07106.16         11     2.45     62   0.90   0.90   0.86   0.05\n",
            "17501  TBPIP_N              PF15517.9           2     0.50    100   0.59   0.58   0.52   0.03\n",
            "17502  TBSV_P22             PF03558.16          3     0.71    187   0.59   0.59   0.51   0.01\n",
            "17503  TBX                  PF12598.11         13     1.72     81   0.70   0.68   0.60   0.12\n",
            "17504  Tc-38                PF20054.2          19    11.27     69   0.81   0.78   0.70   0.02\n",
            "17505  TC1                  PF15063.9           8     1.39     75   0.75   0.79   0.70   0.05\n",
            "17506  TCAD2                PF19971.2           2     0.67     90   0.63   0.67   0.57   0.04\n",
            "17507  TCAD3                PF19972.2           2     0.96     48   1.15   1.14   1.09   0.05\n",
            "17508  TCAD7                PF19973.2          10     2.58    107   0.59   0.62   0.49   0.03\n",
            "17509  TCAD9                PF19974.2          16     3.54    435   0.59   0.60   0.50   0.02\n",
            "17510  TcA_RBD              PF18518.4          11     1.33    131   0.59   0.56   0.50   0.03\n",
            "17511  TcA_TcB_BD           PF18276.4          41     1.46    287   0.59   0.57   0.53   0.02\n",
            "17512  TcdA_TcdB            PF12919.10          5     1.49    407   0.59   0.59   0.50   0.03\n",
            "17513  TcdA_TcdB_pore       PF12920.10          5     0.95    655   0.59   0.60   0.50   0.02\n",
            "17514  TcdB_N               PF12918.10         19     8.37     66   0.85   0.88   0.80   0.09\n",
            "17515  TcdB_toxin_midC      PF12255.11         19     1.93    148   0.59   0.56   0.52   0.03\n",
            "17516  TcdB_toxin_midN      PF12256.11          5     1.63    181   0.59   0.58   0.51   0.02\n",
            "17517  Tcell_CD4_C          PF12104.11          1     1.00     28   1.45   1.32   1.41   0.35\n",
            "17518  Tcf25                PF04910.17         80     2.82    351   0.59   0.56   0.48   0.03\n",
            "17519  TcfC                 PF16967.8          14     2.74     68   0.83   0.85   0.80   0.03\n",
            "17520  TCL1_MTCP1           PF01840.20         17     1.19    116   0.59   0.53   0.51   0.04\n",
            "17521  TCO89                PF10452.12          4     1.31    596   0.59   0.59   0.50   0.06\n",
            "17522  TCP                  PF03634.16         45     1.83    154   0.59   0.59   0.48   0.08\n",
            "17523  Tcp10_C              PF07202.16        148   148.00     35   1.54   1.52   1.53   0.11\n",
            "17524  Tcp11                PF05794.16        206     6.11    447   0.59   0.59   0.46   0.03\n",
            "17525  TcpA                 PF05946.15          2     0.58    130   0.59   0.62   0.52   0.03\n",
            "17526  TcpE                 PF12648.10         20     2.97    100   0.59   0.56   0.49   0.06\n",
            "17527  TcpF                 PF06340.14          1     0.39    317   0.59   0.57   0.52   0.01\n",
            "17528  TcpQ                 PF10671.12         30     4.53     80   0.71   0.66   0.64   0.05\n",
            "17529  TcpS                 PF17456.5           3     0.76    152   0.59   0.61   0.52   0.04\n",
            "17530  TCR                  PF03638.18        181     2.27     38   1.43   1.02   1.38   0.66\n",
            "17531  TCRP1                PF14944.9           8     0.61    243   0.59   0.54   0.51   0.05\n",
            "17532  TCR_zetazeta         PF11628.11         10     3.13     31   1.74   1.72   1.72   0.21\n",
            "17533  TctA                 PF01970.19        362     4.46    409   0.59   0.64   0.50   0.07\n",
            "17534  TctB                 PF07331.14        420    10.22    134   0.59   0.64   0.29   0.12\n",
            "17535  TctC                 PF03401.17          5     1.04    274   0.59   0.61   0.53   0.01\n",
            "17536  Tctex-1              PF03645.16        286     3.99     94   0.61   0.55   0.49   0.05\n",
            "17537  TCTN_DUF1619         PF07773.14         58     3.35    317   0.59   0.52   0.46   0.06\n",
            "17538  TCTP                 PF00838.20        133     2.42    166   0.59   0.57   0.50   0.04\n",
            "17539  TDA11                PF17084.8           1     0.42    465   0.59   0.60   0.52   0.03\n",
            "17540  TDBD                 PF16135.8          56     8.65     52   1.07   0.97   0.98   0.08\n",
            "17541  TDH                  PF03347.16          2     0.39    165   0.59   0.56   0.52   0.02\n",
            "17542  TDP43_N              PF18694.4          66     3.40     74   0.76   0.75   0.70   0.01\n",
            "17543  TDRP                 PF15683.8          13     0.89    147   0.59   0.57   0.50   0.05\n",
            "17544  TEA                  PF01285.21         33     2.49     68   0.83   0.79   0.74   0.06\n",
            "17545  TEBP_beta            PF07404.14          5     0.79    140   0.59   0.57   0.53   0.04\n",
            "17546  Tecti-min-caps       PF09300.13          1     0.44     84   0.67   0.66   0.60   0.03\n",
            "17547  Tectonin             PF19193.3          40     2.32    214   0.59   0.54   0.49   0.04\n",
            "17548  TED                  PF08341.14        127     9.14    104   0.59   0.54   0.39   0.05\n",
            "17549  TEDC1                PF14970.9          33     2.28    165   0.59   0.56   0.47   0.05\n",
            "17550  TED_2                PF20610.1          42     1.91    131   0.59   0.53   0.49   0.05\n",
            "17551  TED_complement       PF07678.17         21     1.65    313   0.59   0.57   0.51   0.03\n",
            "17552  Tegument_dsDNA       PF12818.10          5     1.20    277   0.59   0.59   0.53   0.01\n",
            "17553  TehB                 PF03848.17          4     0.53    193   0.59   0.57   0.52   0.02\n",
            "17554  Tektin               PF03148.17        135     4.42    383   0.59   0.59   0.57   0.08\n",
            "17555  TelA                 PF05816.14        136     3.41    331   0.59   0.62   0.54   0.06\n",
            "17556  Telethonin           PF09470.13          8     0.73    163   0.59   0.57   0.52   0.02\n",
            "17557  Telomerase_RBD       PF12009.11        116     2.75    134   0.59   0.53   0.51   0.05\n",
            "17558  Telomere_reg-2       PF10193.12        147     3.83    109   0.59   0.61   0.52   0.04\n",
            "17559  Telomere_res         PF16684.8           5     1.21    274   0.59   0.58   0.47   0.02\n",
            "17560  Ten1                 PF12658.10         22     2.52    121   0.59   0.60   0.46   0.02\n",
            "17561  Ten1_2               PF15490.9          23     2.25    118   0.59   0.60   0.52   0.02\n",
            "17562  TENA_THI-4           PF03070.19         18     4.64    210   0.59   0.55   0.54   0.03\n",
            "17563  Tenui_N              PF05733.14         12     1.57    246   0.59   0.60   0.52   0.03\n",
            "17564  Tenui_NCP            PF04876.15          1     0.38    173   0.59   0.58   0.52   0.01\n",
            "17565  Tenui_NS3            PF05310.15          9     1.30    186   0.59   0.52   0.52   0.04\n",
            "17566  Tenui_NS4            PF03300.16          1     0.39    282   0.59   0.59   0.52   0.01\n",
            "17567  Tenui_PVC2           PF06656.14          1     0.38    784   0.59   0.58   0.52   0.02\n",
            "17568  Ten_N                PF06484.15          7     0.73    367   0.59   0.57   0.51   0.03\n",
            "17569  TEP1_N               PF05386.14         22     5.20     29   1.85   1.86   1.84   0.13\n",
            "17570  Ter                  PF05472.14         16     1.39    295   0.59   0.58   0.53   0.03\n",
            "17571  TerB                 PF05099.16         64     4.23    118   0.59   0.62   0.55   0.05\n",
            "17572  TERB2                PF15101.9           8     1.28    207   0.59   0.55   0.51   0.03\n",
            "17573  TerB_C               PF15615.9          20     4.66    151   0.59   0.63   0.51   0.05\n",
            "17574  TerB_N               PF13208.9          30     5.06    208   0.59   0.56   0.50   0.03\n",
            "17575  TerC                 PF03741.19         63     2.54    179   0.59   0.63   0.52   0.08\n",
            "17576  TerD                 PF02342.21        211     2.44    187   0.59   0.59   0.52   0.02\n",
            "17577  TERF2_RBM            PF16772.8           7     2.64     41   1.34   1.36   1.31   0.07\n",
            "17578  TerL_ATPase          PF03354.18          3     1.10    178   0.59   0.58   0.51   0.02\n",
            "17579  TerL_nuclease        PF20441.1           3     1.03    286   0.59   0.57   0.52   0.02\n",
            "17580  Terminase_2          PF03592.19         40     3.45    140   0.59   0.61   0.46   0.06\n",
            "17581  Terminase_3          PF04466.16          7     1.53    201   0.59   0.57   0.51   0.03\n",
            "17582  Terminase_3C         PF17288.5           7     2.63    155   0.59   0.57   0.48   0.03\n",
            "17583  Terminase_4          PF05119.15         96     5.35     98   0.59   0.59   0.50   0.03\n",
            "17584  Terminase_5          PF06056.15          9     3.15     58   0.96   0.90   0.93   0.06\n",
            "17585  Terminase_6C         PF17289.5          22     7.18    155   0.59   0.57   0.47   0.02\n",
            "17586  Terminase_6N         PF03237.18         22     7.03    215   0.59   0.57   0.47   0.02\n",
            "17587  Terpene_synth        PF01397.24        167     3.28    192   0.59   0.58   0.46   0.04\n",
            "17588  Terpene_synth_C      PF03936.19        203     3.60    267   0.59   0.55   0.52   0.04\n",
            "17589  Terpene_syn_C_2      PF19086.3        2073    10.82    199   0.59   0.55   0.52   0.03\n",
            "17590  TERT_thumb           PF17984.4           1     0.37    191   0.59   0.55   0.51   0.03\n",
            "17591  TerY_C               PF15616.9          13     0.78    129   0.59   0.51   0.52   0.04\n",
            "17592  TES                  PF08034.14          5     1.87     66   0.85   0.82   0.80   0.11\n",
            "17593  TetM_leader          PF08076.14          2     1.24     28   1.92   1.70   1.88   0.22\n",
            "17594  TetR                 PF13972.9          74     5.40    144   0.59   0.56   0.54   0.04\n",
            "17595  Tetrabrachion        PF11401.11          1     0.84     49   1.13   1.18   1.07   0.13\n",
            "17596  Tetradecapep         PF08187.14          2     2.00     14   2.38   2.36   2.35   0.54\n",
            "17597  Tetraspanin          PF00335.23        114     6.39    232   0.59   0.55   0.44   0.05\n",
            "17598  TetR_C_1             PF02909.20         11     2.78    145   0.59   0.60   0.53   0.01\n",
            "17599  TetR_C_10            PF16295.8           4     0.83    132   0.59   0.57   0.54   0.03\n",
            "17600  TetR_C_11            PF16859.8         294     8.94    113   0.59   0.65   0.56   0.05\n",
            "17601  TetR_C_12            PF16914.8           5     1.14    105   0.59   0.61   0.54   0.03\n",
            "17602  TetR_C_13            PF16925.8          17     1.05    115   0.59   0.57   0.54   0.02\n",
            "17603  TetR_C_14            PF17754.4          43     3.58    112   0.59   0.63   0.55   0.04\n",
            "17604  TetR_C_15            PF17918.4          74    10.74    108   0.59   0.63   0.56   0.05\n",
            "17605  TetR_C_16            PF17920.4          74     5.13    107   0.59   0.65   0.56   0.04\n",
            "17606  TetR_C_17            PF17922.4          11     1.43    100   0.59   0.59   0.54   0.03\n",
            "17607  TetR_C_18            PF17923.4           1     0.41    113   0.59   0.62   0.52   0.03\n",
            "17608  TetR_C_19            PF17924.4          57     5.69    117   0.59   0.56   0.55   0.06\n",
            "17609  TetR_C_2             PF08361.14          8     1.06    121   0.59   0.58   0.54   0.03\n",
            "17610  TetR_C_20            PF17925.4          30     2.66    107   0.59   0.62   0.56   0.03\n",
            "17611  TetR_C_21            PF17926.4          33     2.62    111   0.59   0.59   0.54   0.04\n",
            "17612  TetR_C_22            PF17928.4          13     3.71    113   0.59   0.61   0.57   0.03\n",
            "17613  TetR_C_23            PF17931.4          80     2.79    127   0.59   0.58   0.56   0.04\n",
            "17614  TetR_C_24            PF17932.4         162     5.92    114   0.59   0.59   0.57   0.02\n",
            "17615  TetR_C_25            PF17933.4          43     1.76    106   0.59   0.61   0.55   0.02\n",
            "17616  TetR_C_26            PF17934.4          26     0.97    109   0.59   0.57   0.54   0.04\n",
            "17617  TetR_C_27            PF17935.4          63     3.36    106   0.59   0.58   0.57   0.03\n",
            "17618  TetR_C_28            PF17937.4         204     9.42     96   0.60   0.63   0.52   0.05\n",
            "17619  TetR_C_29            PF17938.4         184     4.32    119   0.59   0.59   0.57   0.02\n",
            "17620  TetR_C_3             PF08362.14         14     1.10    143   0.59   0.58   0.54   0.02\n",
            "17621  TetR_C_30            PF17939.4         169     8.25    115   0.59   0.60   0.56   0.02\n",
            "17622  TetR_C_31            PF17940.4         249     8.54    107   0.59   0.67   0.50   0.06\n",
            "17623  TetR_C_32            PF19344.2          20     0.65    139   0.59   0.59   0.52   0.01\n",
            "17624  TetR_C_33            PF13305.9         384     8.27    104   0.59   0.60   0.45   0.04\n",
            "17625  TetR_C_34            PF17929.4          42     5.03    120   0.59   0.62   0.57   0.03\n",
            "17626  TetR_C_35            PF18556.4          62     2.30    105   0.59   0.65   0.55   0.04\n",
            "17627  TetR_C_36            PF18598.4          46     2.88    111   0.59   0.64   0.56   0.03\n",
            "17628  TetR_C_37            PF18665.4          15     1.39    117   0.59   0.57   0.54   0.05\n",
            "17629  TetR_C_38            PF19352.2          20     0.76    135   0.59   0.62   0.53   0.02\n",
            "17630  TetR_C_4             PF08359.14          8     0.72    133   0.59   0.60   0.53   0.04\n",
            "17631  TetR_C_5             PF08360.14          3     0.90    131   0.59   0.57   0.54   0.03\n",
            "17632  TetR_C_6             PF13977.9         105     9.69    115   0.59   0.63   0.54   0.05\n",
            "17633  TetR_C_7             PF14246.9          71     4.33    119   0.59   0.61   0.56   0.03\n",
            "17634  TetR_C_8             PF14278.9         311     9.78    103   0.59   0.54   0.56   0.04\n",
            "17635  TetR_C_9             PF14514.9           2     0.54    130   0.59   0.59   0.52   0.01\n",
            "17636  TetR_N               PF00440.26         73    17.32     47   1.17   1.25   1.17   0.07\n",
            "17637  Tet_JBP              PF12851.10         13     2.05    335   0.59   0.56   0.42   0.02\n",
            "17638  Tet_res_leader       PF08050.15          4     3.46     20   2.64   2.54   2.62   0.42\n",
            "17639  TEX12                PF15219.9           2     0.57     85   0.67   0.70   0.60   0.04\n",
            "17640  TEX13                PF15186.9          30     1.21    150   0.59   0.58   0.54   0.04\n",
            "17641  TEX15                PF15326.9           7     0.57    236   0.59   0.57   0.52   0.02\n",
            "17642  TEX19                PF15553.9          11     0.67    159   0.59   0.53   0.51   0.02\n",
            "17643  TEX29                PF15839.8           7     0.82     76   0.74   0.67   0.68   0.08\n",
            "17644  TEX33                PF15400.9           9     1.10    138   0.59   0.56   0.51   0.04\n",
            "17645  Tex_N                PF09371.13        557     2.23    184   0.59   0.62   0.53   0.05\n",
            "17646  Tex_YqgF             PF16921.8         209     1.02    126   0.59   0.62   0.53   0.02\n",
            "17647  TFA2_Winged_2        PF18121.4         100     8.47     62   0.90   0.92   0.79   0.04\n",
            "17648  Tfb2                 PF03849.17        113     2.18    356   0.59   0.59   0.50   0.03\n",
            "17649  Tfb2_C               PF18307.4          40     3.51     68   0.83   0.76   0.80   0.06\n",
            "17650  Tfb4                 PF03850.17         22     1.17    276   0.59   0.56   0.50   0.03\n",
            "17651  Tfb5                 PF06331.15         82     4.23     68   0.83   0.83   0.78   0.06\n",
            "17652  TFB6                 PF17110.8          47     1.43    172   0.59   0.59   0.49   0.02\n",
            "17653  TFCD_C               PF12612.11         99     5.11    188   0.59   0.61   0.50   0.02\n",
            "17654  TFIIA                PF03153.16         56     2.99    415   0.59   0.57   0.42   0.06\n",
            "17655  TFIIA_gamma_C        PF02751.17         89     4.85     44   1.25   1.16   1.18   0.10\n",
            "17656  TFIIA_gamma_N        PF02268.19         63     3.86     47   1.17   1.19   1.15   0.06\n",
            "17657  TFIIB                PF00382.22         13     2.34     92   0.62   0.63   0.59   0.04\n",
            "17658  TFIIB_C_1            PF18542.4           2     0.57     99   0.59   0.62   0.53   0.03\n",
            "17659  TFIID-18kDa          PF02269.19          6     1.04     93   0.61   0.62   0.56   0.04\n",
            "17660  TFIID-31kDa          PF02291.18         90     1.88    123   0.59   0.60   0.52   0.02\n",
            "17661  TFIID_20kDa          PF03847.16          8     1.43     68   0.83   0.85   0.78   0.07\n",
            "17662  TFIID_30kDa          PF03540.16         59     3.24     50   1.11   1.12   1.04   0.04\n",
            "17663  TFIID_NTD2           PF04494.18        109     2.09    133   0.59   0.56   0.54   0.04\n",
            "17664  TFIIE-A_C            PF11521.11         26     1.86     86   0.66   0.64   0.58   0.09\n",
            "17665  TFIIE_alpha          PF02002.20          6     1.61    105   0.59   0.61   0.54   0.05\n",
            "17666  TFIIE_beta           PF02186.18         66     5.48     68   0.83   0.81   0.72   0.06\n",
            "17667  TFIIF_alpha          PF05793.15          8     1.25    528   0.59   0.58   0.50   0.08\n",
            "17668  TFIIF_beta           PF02270.18         23     2.20     65   0.86   0.82   0.82   0.05\n",
            "17669  TFIIF_beta_N         PF17683.4          43     3.91    132   0.59   0.57   0.42   0.04\n",
            "17670  TFIIIC_delta         PF12657.10         42     4.61    184   0.59   0.55   0.46   0.03\n",
            "17671  TFIIIC_sub6          PF10419.12         85     3.90    103   0.59   0.59   0.46   0.02\n",
            "17672  TFIIS_C              PF01096.21         92     6.11     39   1.40   1.13   1.37   0.26\n",
            "17673  TFIIS_M              PF07500.17        273     6.45    116   0.59   0.60   0.48   0.07\n",
            "17674  TfoX_C               PF04994.16         31     2.86     81   0.70   0.73   0.66   0.04\n",
            "17675  TfoX_N               PF04993.16        111     5.75     91   0.63   0.62   0.54   0.02\n",
            "17676  TFR_dimer            PF04253.18        279     6.42    125   0.59   0.57   0.49   0.03\n",
            "17677  TfuA                 PF07812.15         66     1.19    120   0.59   0.61   0.54   0.02\n",
            "17678  TFX_C                PF14601.9          40     3.11     83   0.68   0.72   0.66   0.04\n",
            "17679  TF_AP-2              PF03299.17         29     1.97    197   0.59   0.60   0.54   0.03\n",
            "17680  TF_Otx               PF03529.16         16     0.92     92   0.62   0.58   0.51   0.08\n",
            "17681  TF_Zn_Ribbon         PF08271.15         58    18.57     43   1.28   1.11   1.22   0.19\n",
            "17682  TGase_elicitor       PF16683.8          15     0.70    361   0.59   0.54   0.53   0.02\n",
            "17683  TGBp3                PF02495.20         60     9.29     59   0.95   0.95   0.87   0.10\n",
            "17684  TGFb_propeptide      PF00688.21         60     3.12    242   0.59   0.57   0.45   0.03\n",
            "17685  TGF_beta             PF00019.23        263     2.60    102   0.59   0.48   0.46   0.08\n",
            "17686  TGF_beta_GS          PF08515.15         15     6.65     28   1.92   1.99   1.87   0.13\n",
            "17687  Tgi2PP               PF18250.4           1     0.89     43   1.27   1.31   1.22   0.06\n",
            "17688  TGL                  PF20085.2         209     2.38    248   0.59   0.55   0.46   0.02\n",
            "17689  TgMIC1               PF11476.11          1     0.42    137   0.59   0.64   0.52   0.03\n",
            "17690  TgpA_N               PF11992.11        144    14.29    339   0.59   0.63   0.45   0.05\n",
            "17691  TGS                  PF02824.24         53     7.90     60   0.93   0.94   0.84   0.03\n",
            "17692  TGT                  PF01702.21        513     4.24    355   0.59   0.57   0.47   0.01\n",
            "17693  TGT_C1               PF14809.9           2     0.59     70   0.81   0.80   0.75   0.04\n",
            "17694  TGT_C2               PF14810.9          85     5.11     71   0.79   0.82   0.72   0.02\n",
            "17695  TH1                  PF04858.16         11     0.91    580   0.59   0.59   0.52   0.02\n",
            "17696  ThaI                 PF15514.9           3     0.47    202   0.59   0.58   0.52   0.02\n",
            "17697  THAP                 PF05485.15        457     6.38     86   0.66   0.55   0.43   0.07\n",
            "17698  THAP4_heme-bd        PF08768.14        146     3.73    153   0.59   0.57   0.45   0.02\n",
            "17699  Thaumatin            PF00314.20        261     1.64    214   0.59   0.49   0.45   0.07\n",
            "17700  THB                  PF18362.4          10     2.69     34   1.59   1.59   1.57   0.08\n",
            "17701  THDPS_M              PF14789.9          68     5.62     41   1.34   1.31   1.28   0.05\n",
            "17702  THDPS_N              PF14790.9          21     1.02    168   0.59   0.59   0.51   0.03\n",
            "17703  THDPS_N_2            PF14805.9         242     3.35     67   0.84   0.85   0.73   0.06\n",
            "17704  THEG                 PF14912.9          81     8.93     58   0.96   0.97   0.83   0.10\n",
            "17705  THEG4                PF15834.8           9     0.62    201   0.59   0.56   0.52   0.01\n",
            "17706  Thermopsin           PF05317.14         31     2.03    260   0.59   0.56   0.50   0.04\n",
            "17707  THF_DHG_CYH          PF00763.26       1140     3.45    114   0.59   0.61   0.48   0.03\n",
            "17708  THF_DHG_CYH_C        PF02882.22         81     1.57    161   0.59   0.61   0.53   0.03\n",
            "17709  Thg1                 PF04446.15        166     1.82    132   0.59   0.56   0.50   0.03\n",
            "17710  Thg1C                PF14413.9          94     1.11    124   0.59   0.54   0.50   0.05\n",
            "17711  THH1_TOM1-3_dom      PF06454.14         17     0.72    273   0.59   0.57   0.53   0.05\n",
            "17712  Thi4                 PF01946.20          7     0.82    235   0.59   0.61   0.51   0.02\n",
            "17713  Thiamine_BP          PF01910.20         82     3.20     92   0.62   0.64   0.57   0.04\n",
            "17714  Thia_YuaJ            PF09515.13         26     1.66    177   0.59   0.61   0.54   0.07\n",
            "17715  ThiC-associated      PF13667.9         200     1.79     81   0.70   0.69   0.62   0.03\n",
            "17716  ThiC_Rad_SAM         PF01964.21        217     1.14    418   0.59   0.58   0.53   0.02\n",
            "17717  ThiD2                PF17792.4          23     1.34    124   0.59   0.60   0.52   0.06\n",
            "17718  ThiF                 PF00899.24         50     4.56    219   0.59   0.60   0.47   0.01\n",
            "17719  ThiG                 PF05690.17         22     0.95    247   0.59   0.63   0.54   0.02\n",
            "17720  ThiI                 PF02568.17          6     1.05    197   0.59   0.61   0.53   0.02\n",
            "17721  ThiJ_like            PF17124.8          10     1.35    200   0.59   0.57   0.50   0.01\n",
            "17722  Thioesterase         PF00975.23         18     4.94    230   0.59   0.58   0.49   0.01\n",
            "17723  Thiolase_C           PF02803.21         21     1.04    123   0.59   0.62   0.53   0.02\n",
            "17724  Thiolase_N           PF00108.26         30     2.11    260   0.59   0.61   0.53   0.03\n",
            "17725  Thiol_cytolysin      PF01289.22          8     0.98    354   0.59   0.59   0.53   0.02\n",
            "17726  Thiol_cytolys_C      PF17440.5           9     0.70    102   0.59   0.54   0.52   0.03\n",
            "17727  Thionin              PF00321.20         16     3.04     46   1.20   0.93   1.16   0.32\n",
            "17728  Thiopep_pre          PF19409.2          36    10.10     36   1.51   1.50   1.46   0.33\n",
            "17729  Thioredoxin          PF00085.23         34     1.83    103   0.59   0.55   0.53   0.02\n",
            "17730  Thioredoxin_10       PF17991.4         158     2.42    142   0.59   0.59   0.51   0.02\n",
            "17731  Thioredoxin_11       PF18078.4          21     1.89    126   0.59   0.55   0.53   0.04\n",
            "17732  Thioredoxin_12       PF18400.4          96     3.53    193   0.59   0.57   0.49   0.03\n",
            "17733  Thioredoxin_13       PF18401.4         110     3.78    136   0.59   0.62   0.49   0.03\n",
            "17734  Thioredoxin_14       PF18402.4         137     5.79    253   0.59   0.58   0.43   0.02\n",
            "17735  Thioredoxin_15       PF18403.4         136     5.96    213   0.59   0.63   0.49   0.03\n",
            "17736  Thioredoxin_16       PF18569.4          23     1.82     93   0.61   0.59   0.54   0.04\n",
            "17737  Thioredoxin_2        PF13098.9          77     8.29    109   0.59   0.56   0.44   0.01\n",
            "17738  Thioredoxin_3        PF13192.9          67     4.68     76   0.74   0.73   0.66   0.04\n",
            "17739  Thioredoxin_4        PF13462.9          25     3.92    165   0.59   0.56   0.49   0.02\n",
            "17740  Thioredoxin_5        PF13743.9          24     1.84    186   0.59   0.57   0.53   0.03\n",
            "17741  Thioredoxin_6        PF13848.9          42     8.71    185   0.59   0.57   0.50   0.04\n",
            "17742  Thioredoxin_7        PF13899.9          26     6.81     83   0.68   0.65   0.57   0.02\n",
            "17743  Thioredoxin_8        PF13905.9          52     7.37     94   0.61   0.57   0.49   0.03\n",
            "17744  Thioredoxin_9        PF14595.9          39     3.39    129   0.59   0.55   0.53   0.02\n",
            "17745  Thioredox_DsbH       PF03190.18         73     1.00    163   0.59   0.54   0.53   0.02\n",
            "17746  ThiP_synth           PF10120.12        121     4.32    165   0.59   0.62   0.52   0.02\n",
            "17747  ThiS                 PF02597.23        145    11.89     77   0.73   0.79   0.62   0.03\n",
            "17748  ThiS-like            PF14453.9          25     4.07     57   0.98   0.97   0.94   0.06\n",
            "17749  ThiW                 PF09512.13         20     1.28    150   0.59   0.64   0.53   0.08\n",
            "17750  Tho1_MOS11_C         PF18592.4         149    14.73     37   1.47   1.57   1.41   0.25\n",
            "17751  Tho2                 PF11262.11        117     3.02    301   0.59   0.55   0.50   0.04\n",
            "17752  Thoc2                PF11732.11         87     2.75     76   0.74   0.72   0.70   0.04\n",
            "17753  THOC2_N              PF16134.8          77     2.91    624   0.59   0.59   0.49   0.02\n",
            "17754  THOC7                PF05615.16        128     6.86    132   0.59   0.61   0.49   0.10\n",
            "17755  THP2                 PF09432.13         14     1.02    129   0.59   0.59   0.54   0.05\n",
            "17756  THRAP3_BCLAF1        PF15440.9          25     2.33    662   0.59   0.57   0.47   0.10\n",
            "17757  ThrE                 PF06738.15        114     5.48    241   0.59   0.62   0.53   0.04\n",
            "17758  ThrE_2               PF12821.10         48     3.10    129   0.59   0.61   0.55   0.09\n",
            "17759  Thrombin_light       PF09396.13         21     1.62     48   1.15   1.13   1.12   0.09\n",
            "17760  Thr_dehydrat_C       PF00585.21         12     2.39     91   0.63   0.63   0.58   0.02\n",
            "17761  Thr_synth_N          PF14821.9         168     2.00     79   0.72   0.72   0.66   0.01\n",
            "17762  Tht1                 PF04163.15          2     0.64    544   0.59   0.57   0.50   0.03\n",
            "17763  ThuA                 PF06283.14        184     3.64    210   0.59   0.56   0.41   0.02\n",
            "17764  THUMP                PF02926.20         48    11.38    144   0.59   0.62   0.50   0.04\n",
            "17765  Thump_like           PF18096.4          75     8.18     78   0.73   0.76   0.66   0.04\n",
            "17766  Thx                  PF17070.8           1     1.00     27   1.59   1.51   1.54   0.40\n",
            "17767  Thy1                 PF02511.18        282     5.49    191   0.59   0.56   0.41   0.03\n",
            "17768  ThylakoidFormat      PF11264.11         49     1.71    216   0.59   0.59   0.54   0.03\n",
            "17769  Thymidylate_kin      PF02223.20         22     3.86    185   0.59   0.60   0.52   0.02\n",
            "17770  Thymidylat_synt      PF00303.22        432     1.72    270   0.59   0.55   0.48   0.02\n",
            "17771  Thymopoietin         PF08198.14         15     2.27     48   1.15   1.18   1.12   0.07\n",
            "17772  Thymosin             PF01290.23         28     6.69     39   1.40   1.43   1.35   0.19\n",
            "17773  Thyroglobulin_1      PF00086.21        165     3.23     67   0.84   0.68   0.73   0.13\n",
            "17774  Thyroglob_assoc      PF16597.8          10     2.14     61   0.92   0.90   0.83   0.08\n",
            "17775  Tiam_CC_Ex           PF18385.4           1     0.40     98   0.59   0.59   0.52   0.02\n",
            "17776  Tic110               PF16940.8          13     0.94    574   0.59   0.62   0.53   0.02\n",
            "17777  TIC20                PF16166.8          13     1.16    181   0.59   0.55   0.52   0.04\n",
            "17778  Tic22                PF04278.15          7     0.93    244   0.59   0.61   0.50   0.02\n",
            "17779  tify                 PF06200.17         96    16.63     34   1.59   1.62   1.59   0.06\n",
            "17780  TIG                  PF01833.27         83    15.08     85   0.67   0.66   0.54   0.05\n",
            "17781  TIG_2                PF18020.4          23     2.73     94   0.61   0.57   0.54   0.03\n",
            "17782  TIG_plexin           PF17960.4          29     2.69     89   0.64   0.60   0.56   0.04\n",
            "17783  TIG_SUH              PF20144.2          42     2.16     94   0.61   0.59   0.52   0.02\n",
            "17784  TIL                  PF01826.20         98     2.74     55   1.01   0.74   0.90   0.38\n",
            "17785  TILa                 PF12714.10         33     2.86     55   1.01   0.76   0.96   0.27\n",
            "17786  TilS                 PF09179.14        217    17.43     67   0.84   0.85   0.71   0.09\n",
            "17787  TilS_C               PF11734.11         59     5.69     74   0.76   0.75   0.70   0.03\n",
            "17788  TIM                  PF00121.21        207     2.35    245   0.59   0.60   0.53   0.02\n",
            "17789  Tim17                PF02466.22        385     5.74    111   0.59   0.60   0.46   0.06\n",
            "17790  TIM21                PF08294.14         13     1.49    145   0.59   0.58   0.52   0.02\n",
            "17791  Tim29                PF10171.12         31     1.71    169   0.59   0.55   0.55   0.02\n",
            "17792  Tim44                PF04280.18         66     4.89    147   0.59   0.57   0.54   0.03\n",
            "17793  Tim54                PF11711.11         81     1.29    374   0.59   0.57   0.50   0.02\n",
            "17794  TIMELESS             PF04821.17        117     3.92    277   0.59   0.60   0.50   0.04\n",
            "17795  TIMELESS_C           PF05029.16         40     5.37     88   0.65   0.64   0.55   0.03\n",
            "17796  TIMP                 PF00965.20         14     1.18    183   0.59   0.49   0.51   0.06\n",
            "17797  TINF2_N              PF14973.9          21     1.54    148   0.59   0.59   0.53   0.03\n",
            "17798  TIP120               PF08623.13         95     2.28    168   0.59   0.60   0.52   0.03\n",
            "17799  TIP39                PF14980.9           4     1.11     51   1.08   1.09   1.04   0.08\n",
            "17800  TIP41                PF04176.16        132     1.37    171   0.59   0.57   0.48   0.03\n",
            "17801  TIP49                PF06068.16         15     0.81    351   0.59   0.61   0.52   0.02\n",
            "17802  TIP49_C              PF17856.4          15     2.29     66   0.85   0.90   0.82   0.05\n",
            "17803  Tipalpha             PF16753.8           2     0.56    151   0.59   0.59   0.52   0.03\n",
            "17804  TipAS                PF07739.16        227     6.92    117   0.59   0.54   0.50   0.06\n",
            "17805  TipE                 PF16972.8           3     0.55    486   0.59   0.57   0.51   0.01\n",
            "17806  TIP_N                PF12457.11         32     2.63    105   0.59   0.57   0.43   0.12\n",
            "17807  TIR                  PF01582.23         23     3.11    176   0.59   0.57   0.50   0.03\n",
            "17808  TIR-like             PF10137.12         43     2.46    119   0.59   0.60   0.50   0.02\n",
            "17809  TIR_2                PF13676.9         429     9.11    121   0.59   0.58   0.41   0.02\n",
            "17810  TIR_3                PF18567.4          24     1.17    131   0.59   0.58   0.53   0.04\n",
            "17811  Tir_receptor_C       PF07489.14          6     0.72    221   0.59   0.62   0.51   0.04\n",
            "17812  Tir_receptor_M       PF03549.17          3     0.79     66   0.85   0.89   0.80   0.09\n",
            "17813  Tir_receptor_N       PF07490.14          4     0.52    266   0.59   0.62   0.51   0.02\n",
            "17814  Tis11B_N             PF04553.15         16     0.98    122   0.59   0.59   0.46   0.03\n",
            "17815  TisB_toxin           PF13939.9           2     2.00     28   1.90   2.02   1.87   0.43\n",
            "17816  Tissue_fac           PF01108.20         30     3.44    107   0.59   0.52   0.51   0.04\n",
            "17817  Titin_Z              PF09042.14         13    11.11     43   1.28   1.39   1.10   0.22\n",
            "17818  TK                   PF00265.21         11     1.22    177   0.59   0.56   0.52   0.03\n",
            "17819  Tk-SP_N-pro          PF18237.4           4     1.15     68   0.83   0.83   0.77   0.06\n",
            "17820  TLC                  PF03219.17         11     0.94    491   0.59   0.58   0.53   0.05\n",
            "17821  TLD                  PF07534.19         97     5.31    139   0.59   0.55   0.45   0.02\n",
            "17822  TLE_N                PF03920.18         27     1.05    117   0.59   0.57   0.52   0.05\n",
            "17823  Tli4_C               PF18426.4          53     4.07    161   0.59   0.58   0.51   0.01\n",
            "17824  Tli4_N               PF18443.4          27     3.26    138   0.59   0.58   0.50   0.02\n",
            "17825  Tlp                  PF19824.2          14     1.38     70   0.80   0.80   0.74   0.14\n",
            "17826  TLP-20               PF06088.14         21     1.98    169   0.59   0.59   0.49   0.05\n",
            "17827  TLP1_add_C           PF18313.4          24     1.92     83   0.68   0.71   0.60   0.03\n",
            "17828  Tlr3_TMD             PF17968.4          17     3.12     33   1.64   1.60   1.62   0.19\n",
            "17829  TLV_coat             PF00429.22         16     1.76    561   0.59   0.56   0.49   0.02\n",
            "17830  TM140                PF14985.9          14     0.69    180   0.59   0.60   0.53   0.04\n",
            "17831  TM1506               PF08973.13         40     1.92    130   0.59   0.60   0.54   0.03\n",
            "17832  TM1586_NiRdase       PF14512.9          25     2.58    215   0.59   0.57   0.52   0.02\n",
            "17833  TM2                  PF05154.19        113    10.69     51   1.09   1.08   1.00   0.20\n",
            "17834  TM231                PF10149.12         30     2.44    301   0.59   0.55   0.53   0.04\n",
            "17835  Tma16                PF11176.11        130     3.04    146   0.59   0.57   0.53   0.06\n",
            "17836  TMA7                 PF09072.13         55     2.70     62   0.90   0.96   0.84   0.20\n",
            "17837  TMC                  PF07810.16         85     1.54    111   0.59   0.55   0.53   0.05\n",
            "17838  TmcA_N               PF08351.14         31     2.98    180   0.59   0.59   0.50   0.02\n",
            "17839  TMCCDC2              PF15844.8           8     0.58    171   0.59   0.58   0.52   0.03\n",
            "17840  TMCO5                PF14992.9          18     1.33    275   0.59   0.59   0.52   0.05\n",
            "17841  Tme5_EGF_like        PF09064.13          3     1.74     34   1.59   1.33   1.52   0.33\n",
            "17842  TMEM100              PF16311.8           4     0.67    133   0.59   0.60   0.51   0.03\n",
            "17843  TMEM101              PF15111.9           6     0.83    249   0.59   0.58   0.52   0.04\n",
            "17844  TMEM107              PF14995.9          28     3.31    123   0.59   0.56   0.53   0.09\n",
            "17845  TMEM108              PF15759.8           6     0.71    514   0.59   0.59   0.51   0.03\n",
            "17846  TMEM117              PF15113.9           8     0.53    410   0.59   0.54   0.52   0.04\n",
            "17847  TMEM119              PF15724.8           8     0.60    252   0.59   0.61   0.52   0.02\n",
            "17848  TMEM125              PF15109.9           5     0.52    111   0.59   0.61   0.52   0.05\n",
            "17849  TMEM126              PF07114.14         33     1.48    181   0.59   0.59   0.54   0.03\n",
            "17850  TMEM127              PF20517.1          15     1.54    113   0.59   0.59   0.55   0.09\n",
            "17851  TMEM128              PF20479.1          78     4.63    124   0.59   0.55   0.51   0.11\n",
            "17852  TMEM131_like         PF19532.2          11     0.92   1430   0.59   0.56   0.52   0.02\n",
            "17853  TMEM131_like_N       PF12371.11         10     1.88     84   0.68   0.67   0.64   0.03\n",
            "17854  TMEM132              PF16070.8          47     2.32    344   0.59   0.60   0.51   0.03\n",
            "17855  TMEM132D_C           PF15706.8          36     1.64     87   0.65   0.61   0.56   0.03\n",
            "17856  TMEM132D_N           PF15705.8          44     1.47    129   0.59   0.57   0.52   0.02\n",
            "17857  TMEM135_C_rich       PF15982.8           3     0.54    134   0.59   0.57   0.52   0.03\n",
            "17858  TMEM138              PF14935.9          21     1.23    119   0.59   0.56   0.54   0.09\n",
            "17859  TMEM141              PF15110.9          16     1.07     91   0.63   0.59   0.57   0.03\n",
            "17860  TMEM144              PF07857.15          6     0.84    336   0.59   0.58   0.52   0.04\n",
            "17861  TMEM151              PF14857.9          15     0.79    429   0.59   0.54   0.51   0.02\n",
            "17862  TMEM154              PF15102.9           5     1.28    153   0.59   0.60   0.51   0.04\n",
            "17863  TMEM156              PF15106.9          14     1.02    226   0.59   0.54   0.53   0.05\n",
            "17864  TMEM164              PF14808.9          24     1.09    254   0.59   0.56   0.52   0.04\n",
            "17865  TMEM169              PF15052.9          14     0.63    131   0.59   0.54   0.53   0.05\n",
            "17866  TMEM171              PF15471.9          16     0.77    320   0.59   0.58   0.53   0.01\n",
            "17867  TMEM173              PF15009.9          26     2.05    293   0.59   0.58   0.52   0.02\n",
            "17868  TMEM174              PF15029.9           8     0.64    235   0.59   0.57   0.52   0.01\n",
            "17869  TMEM175              PF06736.14        195     3.37     88   0.65   0.63   0.55   0.07\n",
            "17870  TMEM18               PF14770.9          48     1.87    118   0.59   0.56   0.55   0.06\n",
            "17871  TMEM187              PF15100.9          15     0.96    244   0.59   0.55   0.53   0.04\n",
            "17872  TMEM190              PF15431.9           1     0.35    128   0.60   0.54   0.52   0.02\n",
            "17873  TMEM191C             PF15194.9           5     0.61    121   0.59   0.65   0.53   0.03\n",
            "17874  TMEM192              PF14802.9          27     2.47    235   0.59   0.57   0.53   0.04\n",
            "17875  TMEM206              PF15122.9           8     0.52    296   0.59   0.56   0.52   0.02\n",
            "17876  TMEM208_SND2         PF05620.14        149     3.49    168   0.59   0.57   0.51   0.03\n",
            "17877  TMEM210              PF15195.9           8     0.55    114   0.59   0.61   0.52   0.03\n",
            "17878  TMEM213              PF15192.9           3     0.52     79   0.72   0.68   0.65   0.03\n",
            "17879  TMEM214              PF10151.12         14     1.31    662   0.59   0.57   0.52   0.02\n",
            "17880  TMEM215              PF15746.8           6     0.61    225   0.59   0.58   0.51   0.01\n",
            "17881  TMEM219              PF14940.9          10     1.62    236   0.59   0.56   0.51   0.02\n",
            "17882  TMEM220              PF15071.9          41     2.68    101   0.59   0.56   0.48   0.05\n",
            "17883  TMEM232              PF15877.8          14     0.82    452   0.59   0.57   0.53   0.03\n",
            "17884  TMEM234              PF10639.12         15     1.29    116   0.59   0.60   0.53   0.05\n",
            "17885  TMEM237              PF15383.9          16     1.50    249   0.59   0.58   0.50   0.02\n",
            "17886  TMEM238              PF15125.9          16     1.37     64   0.87   0.85   0.81   0.13\n",
            "17887  TMEM239              PF15841.8           2     0.45    151   0.59   0.57   0.51   0.05\n",
            "17888  TMEM240              PF15207.9           3     0.45    174   0.59   0.54   0.51   0.02\n",
            "17889  TMEM247              PF15444.9           7     0.54    206   0.59   0.57   0.51   0.01\n",
            "17890  TMEM251              PF15190.9           4     0.55    128   0.59   0.55   0.51   0.04\n",
            "17891  TMEM252              PF15664.8          11     0.70    139   0.59   0.58   0.52   0.01\n",
            "17892  TMEM254              PF14934.9          37     3.30     91   0.63   0.60   0.58   0.06\n",
            "17893  Tmem26               PF09772.12          7     0.98    285   0.59   0.59   0.53   0.05\n",
            "17894  TMEM33_Pom33         PF03661.16         20     1.78    240   0.59   0.58   0.52   0.05\n",
            "17895  TMEM37               PF15108.9           9     0.58    182   0.59   0.58   0.52   0.05\n",
            "17896  TMEM40               PF15817.8           3     0.42    120   0.59   0.58   0.51   0.02\n",
            "17897  TMEM43               PF07787.15         58     3.66    252   0.59   0.55   0.51   0.03\n",
            "17898  TMEM51               PF15345.9          14     0.95    238   0.59   0.60   0.50   0.02\n",
            "17899  TMEM52               PF14979.9          10     0.92    149   0.59   0.55   0.51   0.03\n",
            "17900  TMEM61               PF15105.9           6     0.61    189   0.59   0.59   0.53   0.01\n",
            "17901  TMEM65               PF10507.12         46     1.22    109   0.59   0.62   0.53   0.04\n",
            "17902  TMEM70               PF06979.15         50     4.33    165   0.59   0.57   0.50   0.03\n",
            "17903  TMEM71               PF15121.9          12     0.75    150   0.59   0.56   0.52   0.02\n",
            "17904  TMEM72               PF16054.8           8     1.36    153   0.59   0.57   0.52   0.05\n",
            "17905  TMEM82               PF15816.8          13     0.78    305   0.59   0.60   0.52   0.04\n",
            "17906  TMEM89               PF15098.9           8     0.52    134   0.59   0.56   0.52   0.02\n",
            "17907  TMEM95               PF15203.9           9     0.65    151   0.59   0.54   0.52   0.04\n",
            "17908  Tmemb_14             PF03647.16        207     3.84     92   0.62   0.68   0.53   0.11\n",
            "17909  Tmemb_161AB          PF10268.12         24     1.43    481   0.59   0.58   0.52   0.04\n",
            "17910  Tmemb_170            PF10190.12         27     1.91    106   0.59   0.57   0.55   0.10\n",
            "17911  Tmemb_185A           PF10269.12         24     2.17    250   0.59   0.56   0.48   0.07\n",
            "17912  Tmemb_18A            PF09771.12         20     1.01    118   0.59   0.59   0.53   0.03\n",
            "17913  Tmemb_40             PF10160.12         15     1.22    258   0.59   0.57   0.52   0.06\n",
            "17914  Tmemb_55A            PF09788.12         17     0.83    251   0.59   0.53   0.50   0.04\n",
            "17915  Tmemb_9              PF05434.14         17     1.03    145   0.59   0.54   0.52   0.05\n",
            "17916  Tmemb_cc2            PF10267.12         28     1.61    397   0.59   0.60   0.50   0.06\n",
            "17917  TMEMspv1-c74-12      PF11044.11          1     0.79     49   1.13   1.13   1.08   0.07\n",
            "17918  TMEM_230_134         PF05915.15          9     1.02    119   0.59   0.58   0.51   0.02\n",
            "17919  TMF_DNA_bd           PF12329.11         47     8.97     74   0.76   0.85   0.74   0.20\n",
            "17920  TMF_TATA_bd          PF12325.11         77     2.78    115   0.59   0.63   0.57   0.10\n",
            "17921  TMIE                 PF16038.8          14     1.17     85   0.67   0.63   0.62   0.06\n",
            "17922  TmoB                 PF06234.15         11     1.97     79   0.72   0.74   0.65   0.04\n",
            "17923  TMP                  PF05017.17         95    95.00     11   1.56   1.42   1.56   0.40\n",
            "17924  TMP-TENI             PF02581.20         77     3.83    181   0.59   0.62   0.53   0.02\n",
            "17925  Tmp39                PF10271.12         13     1.16    427   0.59   0.55   0.51   0.04\n",
            "17926  TMPIT                PF07851.16         39     1.44    324   0.59   0.55   0.53   0.04\n",
            "17927  Tmpp129              PF10272.12         34     1.47    349   0.59   0.51   0.51   0.04\n",
            "17928  TMP_2                PF06791.16         28     2.68    207   0.59   0.63   0.54   0.05\n",
            "17929  TMP_3                PF20155.2         132     8.00    192   0.59   0.64   0.55   0.07\n",
            "17930  TMTC_DUF1736         PF08409.14         90     2.65     75   0.75   0.70   0.71   0.05\n",
            "17931  TMV_coat             PF00721.24          7     2.10    163   0.59   0.58   0.48   0.03\n",
            "17932  TM_helix             PF05552.15        266    30.75     50   1.11   1.21   1.10   0.15\n",
            "17933  TM_PBP2_N            PF16296.8          19     1.62     84   0.68   0.69   0.62   0.03\n",
            "17934  Tn7_Tnp_TnsA_C       PF08721.14         44     6.64     83   0.68   0.68   0.63   0.04\n",
            "17935  Tn7_TnsA-like_N      PF08722.14         95     6.98     85   0.67   0.62   0.48   0.05\n",
            "17936  Tn7_TnsC_Int         PF11426.11          5     2.29     47   1.17   1.12   1.14   0.09\n",
            "17937  Tn916-Xis            PF09035.13          2     0.91     69   0.81   0.81   0.75   0.03\n",
            "17938  Tna_leader           PF08053.14          1     1.00     24   1.56   1.38   1.51   0.19\n",
            "17939  TNF                  PF00229.21         27     2.77    127   0.59   0.57   0.50   0.03\n",
            "17940  TNFR_16_TM           PF18422.4           5     1.75     38   1.44   1.41   1.40   0.10\n",
            "17941  TNFR_c6              PF00020.21        489    15.79     39   1.40   1.09   1.29   0.41\n",
            "17942  TniB                 PF05621.14         31     2.10    189   0.59   0.59   0.54   0.02\n",
            "17943  TniQ                 PF06527.14         78     6.67    137   0.59   0.53   0.45   0.04\n",
            "17944  TnpB_IS66            PF05717.16        157     2.09    101   0.59   0.57   0.54   0.02\n",
            "17945  TnpV                 PF14198.9          91     2.16    112   0.59   0.56   0.55   0.04\n",
            "17946  TnpW                 PF14202.9          18    11.59     35   1.55   1.57   1.54   0.15\n",
            "17947  Tnp_22_dsRBD         PF17490.5           1     0.58     65   0.86   0.86   0.80   0.02\n",
            "17948  Tnp_22_trimer        PF17489.5           1     0.88     43   1.27   1.21   1.22   0.15\n",
            "17949  Tnp_DNA_bind         PF14706.9          30     3.80     58   0.96   0.94   0.93   0.05\n",
            "17950  Tnp_P_element        PF12017.11          5     0.73    219   0.59   0.58   0.51   0.03\n",
            "17951  Tnp_P_element_C      PF12596.11         11     2.08    106   0.59   0.58   0.44   0.05\n",
            "17952  Tnp_zf-ribbon_2      PF13842.9           9     5.57     31   1.74   1.34   1.57   0.58\n",
            "17953  TNRC6-PABC_bdg       PF16608.8           9     0.84    289   0.59   0.56   0.49   0.04\n",
            "17954  TnsD                 PF15978.8          16     3.08    360   0.59   0.56   0.52   0.03\n",
            "17955  TnsE_C               PF18623.4          15     1.86    145   0.59   0.57   0.54   0.03\n",
            "17956  TNT                  PF14021.9         100     3.02     88   0.65   0.66   0.55   0.01\n",
            "17957  TNV_CP               PF03898.16          4     0.80    198   0.59   0.60   0.53   0.03\n",
            "17958  Toast_rack_N         PF17115.8           6     1.50     92   0.62   0.63   0.57   0.03\n",
            "17959  TOBE                 PF03459.20         39    13.55     64   0.88   0.95   0.82   0.04\n",
            "17960  TOBE_2               PF08402.13         91     9.28     76   0.74   0.77   0.63   0.03\n",
            "17961  TOBE_3               PF12857.10         53    16.78     59   0.95   1.03   0.88   0.06\n",
            "17962  Tobravirus_2B        PF05271.14          4     0.55    117   0.59   0.54   0.52   0.03\n",
            "17963  TOC159_MAD           PF11886.11         26     1.31    267   0.59   0.60   0.54   0.02\n",
            "17964  Tocopherol_cycl      PF14249.9          36     0.93    336   0.59   0.53   0.51   0.02\n",
            "17965  TOH_N                PF12549.11          8     8.00     25   1.86   1.92   1.85   0.29\n",
            "17966  TolA                 PF06519.14          9     1.14     96   0.60   0.60   0.54   0.03\n",
            "17967  TolA_bind_tri        PF16331.8          57     5.43     76   0.74   0.77   0.72   0.13\n",
            "17968  TolB_like            PF15869.8           6     1.51    295   0.59   0.56   0.50   0.02\n",
            "17969  TolB_N               PF04052.16         90     3.50    106   0.59   0.61   0.53   0.02\n",
            "17970  Toluene_X            PF03349.19         10     2.01    431   0.59   0.58   0.51   0.02\n",
            "17971  TOM13                PF08219.14         43     0.99     84   0.68   0.66   0.60   0.01\n",
            "17972  TOM20_plant          PF06552.15          5     0.68    187   0.59   0.58   0.53   0.02\n",
            "17973  Tom22                PF04281.16         82     1.82    139   0.59   0.61   0.51   0.04\n",
            "17974  Tom37                PF10568.12        136     5.28    125   0.59   0.56   0.48   0.03\n",
            "17975  Tom37_C              PF11801.11         19     2.08    147   0.59   0.59   0.52   0.04\n",
            "17976  Tom5                 PF10642.12         71    11.49     46   1.20   1.20   1.14   0.07\n",
            "17977  Tom6                 PF17112.8          17     2.02     45   1.22   1.24   1.19   0.08\n",
            "17978  TOM6p                PF15184.9           2     0.65     74   0.77   0.77   0.69   0.02\n",
            "17979  Tom7                 PF08038.15         54     4.97     41   1.34   1.32   1.24   0.08\n",
            "17980  Tombus_movement      PF05318.15          6     1.63     68   0.83   0.83   0.72   0.07\n",
            "17981  Tombus_P19           PF03220.16          2     0.49    171   0.59   0.58   0.52   0.01\n",
            "17982  Tombus_P33           PF08500.13         10     1.34    142   0.59   0.59   0.50   0.03\n",
            "17983  TonB_2               PF13103.9          50     6.64     85   0.67   0.66   0.57   0.02\n",
            "17984  TonB_C               PF03544.17         55     7.01     79   0.72   0.72   0.67   0.02\n",
            "17985  TonB_dep_Rec         PF00593.27        657    58.82    470   0.59   0.56   0.31   0.07\n",
            "17986  TonB_N               PF16031.8           7     0.90    134   0.59   0.60   0.47   0.14\n",
            "17987  Top6b_C              PF18000.4          16     1.13    112   0.59   0.59   0.51   0.04\n",
            "17988  Topo-VIb_trans       PF09239.14         67     1.34    159   0.59   0.59   0.52   0.02\n",
            "17989  Topoisom_bac         PF01131.23         61     3.24    412   0.59   0.59   0.49   0.03\n",
            "17990  Topoisom_I           PF01028.23         94     1.68    230   0.59   0.58   0.51   0.04\n",
            "17991  Topoisom_I_N         PF02919.18        119     1.14    213   0.59   0.54   0.53   0.03\n",
            "17992  Topo_C_assoc         PF14370.9          68     1.78     70   0.80   0.74   0.74   0.06\n",
            "17993  Topo_Zn_Ribbon       PF08272.14         44     6.17     41   1.34   1.27   1.27   0.09\n",
            "17994  Toprim               PF01751.25        133     8.63    104   0.59   0.61   0.42   0.02\n",
            "17995  Toprim_2             PF13155.9          87     4.10     89   0.64   0.65   0.51   0.03\n",
            "17996  Toprim_3             PF13362.9          17     3.47     95   0.60   0.64   0.48   0.03\n",
            "17997  Toprim_4             PF13662.9          79     7.90     89   0.64   0.69   0.53   0.03\n",
            "17998  TOPRIM_C             PF16898.8         144     1.47    128   0.59   0.53   0.49   0.06\n",
            "17999  Toprim_Crpt          PF13342.9         177     6.11     61   0.92   0.92   0.85   0.07\n",
            "18000  Toprim_C_rpt         PF13368.9         303     8.19     56   0.99   1.02   0.91   0.03\n",
            "18001  Toprim_N             PF08275.14         73     1.81    128   0.59   0.57   0.52   0.03\n",
            "18002  TOP_N                PF19310.2          50     0.86    123   0.59   0.60   0.53   0.03\n",
            "18003  TORC_C               PF12886.10         16     1.59     75   0.75   0.79   0.66   0.04\n",
            "18004  TORC_M               PF12885.10         21     1.20    158   0.59   0.59   0.50   0.03\n",
            "18005  TORC_N               PF12884.10         12     1.69     71   0.79   0.79   0.71   0.07\n",
            "18006  Torsin               PF06309.14          8     0.77    127   0.59   0.55   0.53   0.02\n",
            "18007  Torus                PF16131.8          64     1.11    109   0.59   0.51   0.50   0.05\n",
            "18008  Tospo_nucleocap      PF01533.19          1     0.40    246   0.59   0.60   0.52   0.02\n",
            "18009  Totivirus_coat       PF05518.14          7     1.70    759   0.59   0.59   0.53   0.01\n",
            "18010  Tower                PF09121.13         15     3.22     42   1.30   1.31   1.28   0.16\n",
            "18011  Tox-ART-HYD1         PF15633.9           6     1.63     98   0.59   0.58   0.51   0.03\n",
            "18012  Tox-ART-HYE1         PF15634.9           1     0.39    282   0.59   0.58   0.52   0.02\n",
            "18013  Tox-GHH              PF15636.9          30     3.06     78   0.73   0.70   0.67   0.06\n",
            "18014  Tox-GHH2             PF15635.9           3     0.91    122   0.59   0.57   0.52   0.05\n",
            "18015  Tox-HDC              PF15656.9           3     0.79    130   0.59   0.58   0.50   0.02\n",
            "18016  Tox-HNH-EHHH         PF15657.9          29     4.13     72   0.78   0.68   0.67   0.11\n",
            "18017  Tox-HNH-HHH          PF15637.9           6     0.88    103   0.59   0.52   0.53   0.03\n",
            "18018  Tox-MPTase2          PF15638.9           6     1.56    194   0.59   0.56   0.51   0.04\n",
            "18019  Tox-MPTase3          PF15639.9           4     1.01    137   0.59   0.56   0.50   0.02\n",
            "18020  Tox-MPTase4          PF15640.9           6     0.81    132   0.59   0.57   0.53   0.04\n",
            "18021  Tox-MPTase5          PF15641.9           1     0.38    110   0.59   0.57   0.51   0.02\n",
            "18022  Tox-ODYAM1           PF15642.9           3     0.58    385   0.59   0.60   0.52   0.02\n",
            "18023  Tox-PAAR-like        PF13665.9          66     3.36    108   0.59   0.56   0.54   0.04\n",
            "18024  Tox-PL-2             PF15643.9          10     1.10    102   0.59   0.56   0.52   0.03\n",
            "18025  Tox-PLDMTX           PF15645.9           4     1.03    141   0.59   0.54   0.51   0.03\n",
            "18026  Tox-REase-2          PF15646.9           5     1.20    129   0.59   0.59   0.49   0.03\n",
            "18027  Tox-REase-3          PF15647.9           4     0.80    102   0.59   0.59   0.48   0.04\n",
            "18028  Tox-REase-5          PF15648.9          16     1.88     95   0.60   0.54   0.52   0.04\n",
            "18029  Tox-REase-7          PF15649.9           9     1.68     86   0.66   0.67   0.58   0.04\n",
            "18030  Tox-REase-9          PF15650.9           7     1.06     88   0.65   0.62   0.57   0.03\n",
            "18031  Tox-SGS              PF15651.9           5     0.78     97   0.59   0.52   0.50   0.02\n",
            "18032  Tox-SHH              PF15652.9           6     0.98     99   0.59   0.55   0.49   0.04\n",
            "18033  Tox-URI2             PF15653.9           5     1.59     86   0.66   0.64   0.60   0.05\n",
            "18034  Tox-WTIP             PF15654.9           3     0.82     67   0.84   0.80   0.75   0.07\n",
            "18035  ToxB_N               PF18224.4           7     1.96     61   0.92   0.87   0.86   0.08\n",
            "18036  Toxin-deaminase      PF14424.9           6     1.46    146   0.59   0.58   0.50   0.03\n",
            "18037  Toxin-JAB1           PF15659.9          12     2.49     87   0.65   0.61   0.56   0.06\n",
            "18038  Toxin_10             PF05431.14          6     1.33    169   0.59   0.54   0.51   0.04\n",
            "18039  Toxin_11             PF07473.14          2     2.00     28   1.74   1.40   1.69   0.51\n",
            "18040  Toxin_12             PF07740.15         31     8.16     30   1.80   1.35   1.70   0.60\n",
            "18041  Toxin_13             PF07822.14          2     0.65     55   1.01   0.85   0.95   0.18\n",
            "18042  Toxin_14             PF07829.14          2     1.33     26   2.06   1.70   2.01   0.62\n",
            "18043  Toxin_15             PF07906.16          5     1.26    278   0.59   0.57   0.51   0.03\n",
            "18044  Toxin_16             PF07945.14          2     0.91     36   1.51   1.30   1.45   0.39\n",
            "18045  Toxin_17             PF08086.14          4     0.91     41   1.33   1.09   1.29   0.34\n",
            "18046  Toxin_18             PF08087.14          7     1.83     31   1.74   1.54   1.70   0.37\n",
            "18047  Toxin_19             PF08088.15          8     1.64     40   1.37   1.04   1.28   0.42\n",
            "18048  Toxin_2              PF00451.22         47     7.20     32   1.69   1.33   1.66   0.58\n",
            "18049  Toxin_20             PF08089.14          3     0.96     39   1.40   1.22   1.31   0.28\n",
            "18050  Toxin_21             PF08091.14          2     1.14     39   1.40   1.14   1.33   0.37\n",
            "18051  Toxin_22             PF08092.14          3     1.23     38   1.44   1.12   1.36   0.30\n",
            "18052  Toxin_23             PF08093.14          5     2.54     30   1.79   1.49   1.73   0.48\n",
            "18053  Toxin_24             PF08094.14          2     1.18     33   1.64   1.42   1.59   0.36\n",
            "18054  Toxin_25             PF08095.14          1     1.00     22   1.79   1.49   1.75   0.45\n",
            "18055  Toxin_26             PF08097.14          2     2.00     11   3.38   2.53   3.35   1.61\n",
            "18056  Toxin_27             PF08099.14          2     1.08     33   1.64   1.44   1.59   0.40\n",
            "18057  Toxin_28             PF08115.14          2     1.43     35   1.55   1.32   1.51   0.18\n",
            "18058  Toxin_29             PF08116.14          4     1.49     31   1.74   1.45   1.70   0.41\n",
            "18059  Toxin_3              PF00537.21         33     2.45     55   1.01   0.79   0.93   0.24\n",
            "18060  Toxin_30             PF08117.14          2     1.33     35   1.55   1.35   1.49   0.30\n",
            "18061  Toxin_31             PF08119.14          2     0.89     37   1.47   1.31   1.42   0.27\n",
            "18062  Toxin_32             PF08120.14          2     0.93     35   1.55   1.37   1.51   0.27\n",
            "18063  Toxin_33             PF08121.14          2     1.46     22   2.40   2.18   2.37   0.70\n",
            "18064  Toxin_34             PF08396.13          7     0.89     75   0.75   0.57   0.65   0.20\n",
            "18065  Toxin_35             PF10530.12          9     1.73     61   0.92   0.72   0.86   0.25\n",
            "18066  Toxin_36             PF10550.12         13    13.00     17   2.00   2.08   1.94   0.44\n",
            "18067  Toxin_37             PF11415.11          8     1.61     35   1.55   1.21   1.52   0.36\n",
            "18068  Toxin_38             PF14866.9          13     1.73     57   0.98   0.81   0.90   0.18\n",
            "18069  Toxin_4              PF00706.20          7     1.55     43   1.28   1.05   1.21   0.25\n",
            "18070  Toxin_5              PF05294.16          6     1.54     32   1.69   1.32   1.63   0.57\n",
            "18071  Toxin_6              PF05453.15          5     3.76     28   1.92   1.58   1.87   0.64\n",
            "18072  Toxin_7              PF05980.15         30     3.53     34   1.60   1.28   1.57   0.43\n",
            "18073  Toxin_8              PF07365.15         22     3.55     50   1.11   1.00   1.03   0.15\n",
            "18074  Toxin_9              PF02819.18          6     1.64     44   1.25   0.96   1.16   0.36\n",
            "18075  Toxin_GhoT_OrtT      PF10753.12         10     1.43     55   1.01   0.99   0.96   0.16\n",
            "18076  Toxin_R_bind_C       PF07951.15          2     0.61    210   0.59   0.55   0.51   0.03\n",
            "18077  Toxin_R_bind_N       PF07953.15         13     1.11    195   0.59   0.56   0.52   0.06\n",
            "18078  Toxin_TOLIP          PF00087.24          4     1.10     70   0.80   0.64   0.72   0.18\n",
            "18079  Toxin_ToxA           PF11584.11          2     0.50    117   0.59   0.57   0.51   0.03\n",
            "18080  Toxin_trans          PF07952.15          2     0.54    322   0.59   0.58   0.52   0.03\n",
            "18081  Toxin_YhaV           PF11663.11         28     1.11    138   0.59   0.56   0.53   0.03\n",
            "18082  ToxN_toxin           PF13958.9          26     1.88    159   0.59   0.56   0.50   0.05\n",
            "18083  ToxS                 PF17323.5           8     0.71    147   0.59   0.57   0.53   0.03\n",
            "18084  TP1                  PF02079.19          1     0.72     51   1.08   1.06   1.02   0.19\n",
            "18085  TP2                  PF01254.21          7     0.70    133   0.59   0.52   0.52   0.12\n",
            "18086  TP53IP5              PF15331.9          14     0.81    217   0.59   0.57   0.53   0.04\n",
            "18087  TP6A_N               PF04406.17        170     7.46     62   0.90   0.89   0.85   0.06\n",
            "18088  TPALS                PF18178.4           5     1.23    232   0.59   0.63   0.52   0.01\n",
            "18089  TpcC                 PF12642.10         25     5.66    230   0.59   0.58   0.49   0.04\n",
            "18090  TPD                  PF14811.9          30     1.20    138   0.59   0.55   0.53   0.02\n",
            "18091  TPD52                PF04201.18         18     0.84    108   0.59   0.62   0.53   0.05\n",
            "18092  TPH                  PF13868.9         119     9.53    352   0.59   0.61   0.57   0.25\n",
            "18093  tPilZ                PF16823.8           3     0.69    136   0.59   0.62   0.50   0.03\n",
            "18094  TPIP1                PF15338.9           2     0.43    123   0.59   0.61   0.52   0.02\n",
            "18095  TPK_B1_binding       PF04265.17        222     8.08     70   0.80   0.79   0.71   0.03\n",
            "18096  TPK_catalytic        PF04263.19        115     3.11    121   0.59   0.58   0.48   0.03\n",
            "18097  TPMT                 PF05724.14         11     1.51    218   0.59   0.56   0.52   0.01\n",
            "18098  TPM_phosphatase      PF04536.17        539     9.53    125   0.59   0.61   0.49   0.03\n",
            "18099  TPP1                 PF10341.12         40     5.20    110   0.59   0.57   0.48   0.03\n",
            "18100  TPPII                PF12580.11         50     2.56    187   0.59   0.59   0.54   0.03\n",
            "18101  TPPII_N              PF12583.11          7     0.92    139   0.59   0.61   0.52   0.05\n",
            "18102  TPPK_C               PF12555.11         82    11.84     52   1.07   1.14   1.03   0.14\n",
            "18103  TPP_enzyme_C         PF02775.24        323     6.85    153   0.59   0.59   0.47   0.02\n",
            "18104  TPP_enzyme_M         PF00205.25        220     6.72    137   0.59   0.63   0.52   0.01\n",
            "18105  TPP_enzyme_M_2       PF16582.8          18     1.24    208   0.59   0.55   0.53   0.04\n",
            "18106  TPP_enzyme_N         PF02776.21         75     6.78    172   0.59   0.61   0.53   0.02\n",
            "18107  TPR-S                PF20308.1         117     6.95    103   0.59   0.60   0.38   0.05\n",
            "18108  TPR_1                PF00515.31        550   550.00     34   1.23   1.21   1.24   0.12\n",
            "18109  TPR_10               PF13374.9         171   171.00     42   1.11   1.13   1.08   0.07\n",
            "18110  TPR_11               PF13414.9          43    25.69     42   1.31   1.32   1.30   0.12\n",
            "18111  TPR_12               PF13424.9         120    74.19     77   0.73   0.77   0.70   0.08\n",
            "18112  TPR_14               PF13428.9         389   389.00     44   0.69   0.75   0.65   0.10\n",
            "18113  TPR_15               PF13429.9          12     2.33    280   0.59   0.59   0.53   0.06\n",
            "18114  TPR_16               PF13432.9          64    64.00     68   0.81   0.88   0.76   0.12\n",
            "18115  TPR_17               PF13431.9         203   203.00     34   1.02   1.00   0.96   0.08\n",
            "18116  TPR_18               PF13512.9           4     0.70    145   0.59   0.56   0.53   0.03\n",
            "18117  TPR_19               PF14559.9         146    39.69     68   0.83   0.90   0.77   0.10\n",
            "18118  TPR_2                PF07719.20        361   361.00     34   0.96   0.96   0.96   0.12\n",
            "18119  TPR_20               PF14561.9         108     4.06     90   0.63   0.67   0.61   0.05\n",
            "18120  TPR_21               PF09976.12         53     2.30    193   0.59   0.60   0.54   0.05\n",
            "18121  TPR_22               PF18833.4         107     3.30     92   0.62   0.61   0.56   0.05\n",
            "18122  TPR_3                PF07720.15         17    17.00     36   1.44   1.38   1.36   0.20\n",
            "18123  TPR_4                PF07721.17        142   142.00     26   1.11   1.21   1.11   0.28\n",
            "18124  TPR_5                PF12688.10          5     0.84    119   0.59   0.64   0.53   0.03\n",
            "18125  TPR_6                PF13174.9         241   241.00     33   1.08   1.10   1.01   0.08\n",
            "18126  TPR_7                PF13176.9         118   118.00     36   0.94   0.98   0.87   0.07\n",
            "18127  TPR_8                PF13181.9          92    92.00     34   0.94   0.95   0.94   0.09\n",
            "18128  TPR_9                PF13371.9          49     9.35     73   0.77   0.81   0.74   0.06\n",
            "18129  TPR_MalT             PF17874.4          16     4.00    336   0.59   0.61   0.54   0.05\n",
            "18130  TPR_MLP1_2           PF07926.15        110     5.56    129   0.59   0.62   0.58   0.15\n",
            "18131  TPS                  PF05860.16         41     4.91    255   0.59   0.63   0.49   0.05\n",
            "18132  TPT                  PF03151.19         15     1.26    290   0.59   0.59   0.52   0.06\n",
            "18133  TPX2                 PF06886.14         81     5.02     77   0.73   0.73   0.67   0.15\n",
            "18134  TPX2_importin        PF12214.11         31     2.19    167   0.59   0.59   0.51   0.07\n",
            "18135  TP_methylase         PF00590.23         79     9.63    213   0.59   0.63   0.47   0.02\n",
            "18136  TQ                   PF18202.4          55     4.64    131   0.59   0.60   0.46   0.07\n",
            "18137  Tr-sialidase_C       PF11052.11         43    13.74     24   2.22   2.40   2.07   0.49\n",
            "18138  TRA-1_regulated      PF02343.19         66     2.36    130   0.59   0.52   0.49   0.04\n",
            "18139  Tra1_central         PF20175.1          28     1.73    593   0.59   0.60   0.52   0.03\n",
            "18140  Tra1_ring            PF20206.1          29     3.51   1691   0.59   0.59   0.50   0.03\n",
            "18141  TraA                 PF05513.14          2     0.50    120   0.59   0.62   0.52   0.05\n",
            "18142  TraB_PrgY_gumN       PF01963.20        193     9.64    263   0.59   0.59   0.46   0.02\n",
            "18143  TraC                 PF07820.15          6     1.03     88   0.65   0.70   0.59   0.07\n",
            "18144  TraC_F_IV            PF11130.11         48     6.04    232   0.59   0.58   0.49   0.01\n",
            "18145  TraD                 PF06412.14         41     6.18     61   0.92   0.99   0.85   0.07\n",
            "18146  TRADD_N              PF09034.13         14     1.01    112   0.59   0.59   0.54   0.04\n",
            "18147  TraD_N               PF12615.11          5     1.60     92   0.62   0.59   0.54   0.08\n",
            "18148  TraE                 PF05309.14         27     4.23    182   0.59   0.60   0.56   0.03\n",
            "18149  TraF                 PF13728.9          37     3.92    224   0.59   0.57   0.54   0.02\n",
            "18150  TRAF6_Z2             PF18048.4          15     3.65     27   1.98   1.67   1.97   0.38\n",
            "18151  TraF_2               PF13729.9          28     4.35    278   0.59   0.61   0.46   0.02\n",
            "18152  TRAF_BIRC3_bd        PF16673.8           8     1.58     64   0.87   0.90   0.84   0.09\n",
            "18153  TraG-D_C             PF12696.10         81     5.48    126   0.59   0.59   0.51   0.02\n",
            "18154  TraG_N               PF07916.14         44     7.69    468   0.59   0.58   0.45   0.03\n",
            "18155  TraH                 PF06122.14         28     3.14    358   0.59   0.58   0.47   0.02\n",
            "18156  TraH_2               PF06871.14          2     0.49    206   0.59   0.61   0.52   0.01\n",
            "18157  TraI                 PF07057.14          2     0.46    124   0.59   0.60   0.52   0.01\n",
            "18158  TraI_2               PF07514.14          7     1.33    327   0.59   0.59   0.53   0.02\n",
            "18159  TraI_2B              PF18340.4          17     2.35     79   0.72   0.71   0.66   0.04\n",
            "18160  TraI_2_C             PF07515.14         22     1.85    123   0.59   0.56   0.51   0.03\n",
            "18161  TraK                 PF06586.14         30     5.61    233   0.59   0.62   0.50   0.02\n",
            "18162  TraL                 PF07178.14         20     2.73     90   0.63   0.60   0.58   0.04\n",
            "18163  TraL_transposon      PF13150.9           6     1.61     86   0.66   0.66   0.61   0.03\n",
            "18164  TRAM                 PF01938.23         27    10.71     62   0.90   0.96   0.79   0.04\n",
            "18165  TRAM1                PF08390.14        220     7.24     63   0.89   0.84   0.82   0.05\n",
            "18166  TRAM_2               PF18693.4          97     4.44     62   0.90   0.94   0.80   0.06\n",
            "18167  TRAM_LAG1_CLN8       PF03798.19        133     7.05    198   0.59   0.51   0.48   0.10\n",
            "18168  TraN                 PF06986.14         15     1.18    240   0.59   0.50   0.51   0.07\n",
            "18169  Transcript_VP30      PF11507.11          2     0.57    131   0.59   0.59   0.52   0.02\n",
            "18170  Transcrip_act        PF04949.16         20     0.74    154   0.59   0.62   0.53   0.07\n",
            "18171  Transcrip_reg        PF01709.23        695     3.96    239   0.59   0.60   0.49   0.03\n",
            "18172  Transferase          PF02458.18          7     1.98    434   0.59   0.58   0.51   0.01\n",
            "18173  Transferrin          PF00405.20          8     0.65    329   0.59   0.56   0.52   0.02\n",
            "18174  Transformer          PF06495.14          4     0.62    182   0.59   0.55   0.48   0.08\n",
            "18175  Transglut_C          PF00927.25         25     4.34    106   0.59   0.60   0.51   0.02\n",
            "18176  Transglut_core       PF01841.22         83     6.59    112   0.59   0.55   0.46   0.02\n",
            "18177  Transglut_core2      PF13369.9          96     3.95    155   0.59   0.61   0.51   0.01\n",
            "18178  Transglut_core3      PF13471.9          51     5.64    117   0.59   0.57   0.53   0.03\n",
            "18179  Transglut_i_TM       PF14400.9          47     2.31    161   0.59   0.58   0.53   0.03\n",
            "18180  Transglut_N          PF00868.23        194     3.24    114   0.59   0.56   0.50   0.03\n",
            "18181  Transglut_prok       PF09017.13          4     0.45    414   0.59   0.56   0.51   0.02\n",
            "18182  Transgly             PF00912.25        108     2.65    178   0.59   0.60   0.49   0.02\n",
            "18183  Transglycosylas      PF06737.17         15     1.33     77   0.73   0.68   0.68   0.08\n",
            "18184  Transgly_assoc       PF04226.16         92     8.34     49   1.13   1.24   1.08   0.28\n",
            "18185  Transketolase_C      PF02780.23         80     5.72    124   0.59   0.62   0.54   0.02\n",
            "18186  Transketolase_N      PF00456.24         11     0.84    334   0.59   0.58   0.53   0.01\n",
            "18187  Transket_pyr         PF02779.27         80     4.88    178   0.59   0.58   0.51   0.01\n",
            "18188  Translat_reg         PF01818.20          2     0.46    121   0.59   0.58   0.52   0.03\n",
            "18189  Translin             PF01997.19        213     3.48    206   0.59   0.59   0.48   0.04\n",
            "18190  Transmemb_17         PF09799.12         89     5.33    107   0.59   0.58   0.55   0.12\n",
            "18191  Transpeptidase       PF00905.25         40     5.73    306   0.59   0.59   0.47   0.03\n",
            "18192  Transport_MerF       PF11431.11         10     1.59     45   1.22   1.30   1.19   0.16\n",
            "18193  Transposase_1        PF01359.21         26     3.63     80   0.71   0.64   0.66   0.05\n",
            "18194  Transposase_20       PF02371.19         74     6.04     87   0.65   0.67   0.60   0.02\n",
            "18195  Transposase_21       PF02992.17          8     0.71    213   0.59   0.55   0.53   0.01\n",
            "18196  Transposase_22       PF02994.17          1     0.41     98   0.59   0.61   0.52   0.03\n",
            "18197  Transposase_23       PF03017.17          5     1.61     66   0.85   0.89   0.80   0.03\n",
            "18198  Transposase_24       PF03004.17         40     5.22    142   0.59   0.56   0.49   0.05\n",
            "18199  Transposase_28       PF04195.15         15     2.25     69   0.82   0.76   0.76   0.09\n",
            "18200  Transposase_31       PF04754.15         27     1.64    205   0.59   0.57   0.53   0.03\n",
            "18201  Transposase_32       PF20167.1         124    10.68    178   0.59   0.54   0.45   0.02\n",
            "18202  Transposase_mut      PF00872.21          9     1.26    381   0.59   0.59   0.54   0.02\n",
            "18203  Transposon_TraM      PF12508.11         26     2.26    194   0.59   0.61   0.54   0.03\n",
            "18204  Transpos_assoc       PF13963.9          18     1.47     74   0.76   0.62   0.69   0.07\n",
            "18205  Transp_cyt_pur       PF02133.18         14     3.13    440   0.59   0.58   0.52   0.06\n",
            "18206  Transp_inhibit       PF18791.4          71     2.65     47   1.17   1.13   1.15   0.06\n",
            "18207  Transp_Tc5_C         PF04236.18          8     1.06     63   0.89   0.70   0.82   0.12\n",
            "18208  Transthyretin        PF00576.24        326     1.41    110   0.59   0.56   0.48   0.02\n",
            "18209  Trans_coact          PF16805.8          33     3.94     69   0.82   0.84   0.77   0.07\n",
            "18210  Trans_reg_C          PF00486.31        267     6.25     77   0.73   0.76   0.69   0.03\n",
            "18211  TraO                 PF10626.12         19     1.41    168   0.59   0.58   0.53   0.02\n",
            "18212  TraP                 PF07296.14          3     0.80    202   0.59   0.58   0.52   0.02\n",
            "18213  TRAP-delta           PF05404.15         35     1.88    162   0.59   0.57   0.54   0.03\n",
            "18214  TRAP-gamma           PF07074.15         15     0.84    170   0.59   0.60   0.53   0.05\n",
            "18215  TRAPP                PF04051.19        300     3.81    155   0.59   0.58   0.44   0.02\n",
            "18216  TRAPPC-Trs85         PF12739.10         71     4.25    422   0.59   0.56   0.46   0.03\n",
            "18217  TRAPPC10             PF12584.11         13     2.24    152   0.59   0.57   0.48   0.02\n",
            "18218  TRAPPC9-Trs120       PF08626.14         57     2.70   1231   0.59   0.59   0.48   0.02\n",
            "18219  TRAP_alpha           PF03896.19         19     0.93    282   0.59   0.59   0.50   0.02\n",
            "18220  TRAP_beta            PF05753.17         17     2.65    178   0.59   0.59   0.53   0.02\n",
            "18221  TraQ                 PF09679.13          4     0.78     87   0.65   0.66   0.58   0.02\n",
            "18222  TraQ_transposon      PF12988.10         21     1.00    133   0.59   0.56   0.54   0.03\n",
            "18223  TraS                 PF10624.12          1     0.36    162   0.59   0.56   0.51   0.02\n",
            "18224  TraT                 PF05818.15         19     1.68    214   0.59   0.61   0.52   0.03\n",
            "18225  TraU                 PF06834.14         45     2.80    308   0.59   0.52   0.47   0.02\n",
            "18226  TRAUB                PF08164.15        113     2.33     83   0.68   0.68   0.56   0.06\n",
            "18227  TraV                 PF09676.13         21     2.95    131   0.59   0.53   0.49   0.04\n",
            "18228  TraW_N               PF12477.11         11    11.00     30   1.61   1.69   1.58   0.21\n",
            "18229  TraX                 PF05857.14         10     1.78    218   0.59   0.58   0.51   0.06\n",
            "18230  TraY                 PF05509.14         24     8.15     49   1.13   1.16   1.12   0.08\n",
            "18231  Tra_M                PF05261.14          2     0.49    126   0.59   0.60   0.52   0.03\n",
            "18232  TrbC                 PF04956.16        107     7.91     99   0.59   0.63   0.51   0.09\n",
            "18233  TrbC_Ftype           PF09673.13         30     2.12    112   0.59   0.60   0.50   0.01\n",
            "18234  TrbE                 PF11100.11          1     0.57     66   0.85   0.85   0.78   0.11\n",
            "18235  TrbH                 PF07283.14          5     0.67    126   0.59   0.60   0.51   0.02\n",
            "18236  TrbI                 PF03743.17        169     7.60    187   0.59   0.62   0.47   0.03\n",
            "18237  TrbI_Ftype           PF09677.13          8     1.60    106   0.59   0.63   0.54   0.05\n",
            "18238  TrbK                 PF20084.2         107     3.01    109   0.59   0.57   0.51   0.05\n",
            "18239  TrbL                 PF04610.17        116     9.02    215   0.59   0.61   0.51   0.11\n",
            "18240  TrbL_2               PF19478.2           1     0.39    174   0.59   0.60   0.52   0.04\n",
            "18241  TrbL_3               PF19590.2           3     0.52    377   0.59   0.60   0.52   0.02\n",
            "18242  TrbM                 PF07424.14          7     0.82    156   0.59   0.54   0.51   0.03\n",
            "18243  TRC8_N               PF13705.9          33     1.78    489   0.59   0.58   0.52   0.06\n",
            "18244  TRCF                 PF03461.18        514     3.29     95   0.60   0.61   0.57   0.03\n",
            "18245  TrcR                 PF06242.14         41     1.03    139   0.59   0.60   0.54   0.03\n",
            "18246  Treacle              PF03546.17         10     1.00    532   0.59   0.64   0.52   0.11\n",
            "18247  Trefoil              PF00088.21        192     5.90     43   1.28   0.98   1.13   0.32\n",
            "18248  Trehalase            PF01204.21         19     1.57    512   0.59   0.55   0.51   0.02\n",
            "18249  Trehalase_Ca-bi      PF07492.14         45     5.93     30   1.80   1.86   1.77   0.15\n",
            "18250  Trehalose_PPase      PF02358.19         19     3.41    235   0.59   0.58   0.53   0.02\n",
            "18251  Trehalose_recp       PF06151.16          8     1.16    411   0.59   0.56   0.53   0.04\n",
            "18252  TREH_N               PF19291.2          40     0.87    212   0.59   0.59   0.53   0.01\n",
            "18253  Trep_dent_lipo       PF09710.13          4     0.75    397   0.59   0.55   0.50   0.04\n",
            "18254  Trep_Strep           PF09605.13         97     4.08    185   0.59   0.59   0.54   0.10\n",
            "18255  Treslin_N            PF15292.9          19     1.14    797   0.59   0.59   0.52   0.03\n",
            "18256  TRF                  PF08558.13         27     4.43    239   0.59   0.60   0.48   0.04\n",
            "18257  TrfA                 PF07042.14          2     0.42    282   0.59   0.57   0.52   0.01\n",
            "18258  TRH                  PF05438.15         13     0.93    221   0.59   0.57   0.50   0.05\n",
            "18259  TRI12                PF06609.16          2     0.42    599   0.59   0.60   0.52   0.02\n",
            "18260  Tri3                 PF07428.14          2     0.58    413   0.59   0.59   0.52   0.01\n",
            "18261  TRI5                 PF06330.14          3     0.56    362   0.59   0.55   0.52   0.02\n",
            "18262  TRI9                 PF08195.14          1     0.86     43   1.28   1.24   1.23   0.13\n",
            "18263  Triabin              PF03973.16          4     1.00    148   0.59   0.55   0.50   0.03\n",
            "18264  TRIC                 PF05197.16         34     1.36    193   0.59   0.58   0.55   0.04\n",
            "18265  Tricho_coat          PF05892.14          3     0.83    197   0.59   0.60   0.51   0.02\n",
            "18266  Tricorn_C1           PF14684.9          70     4.25     70   0.80   0.72   0.77   0.06\n",
            "18267  Tricorn_PDZ          PF14685.9          26     1.98     88   0.65   0.68   0.59   0.02\n",
            "18268  TRIF-NTD             PF17798.4           5     1.09    157   0.59   0.61   0.52   0.03\n",
            "18269  Trigger_C            PF05698.17         44     6.05    162   0.59   0.63   0.55   0.07\n",
            "18270  Trigger_N            PF05697.16        796     6.03    140   0.59   0.61   0.45   0.05\n",
            "18271  Trimer_CC            PF08954.14          7     1.52     52   1.06   1.13   1.03   0.11\n",
            "18272  TRIQK                PF15168.9           5     0.79     79   0.72   0.75   0.66   0.05\n",
            "18273  Tristanin_u2         PF16638.8          10     0.90    130   0.59   0.61   0.50   0.05\n",
            "18274  TrkA_C               PF02080.24        114    33.14     70   0.80   0.88   0.71   0.03\n",
            "18275  TrkA_N               PF02254.21        188     7.74    116   0.59   0.64   0.51   0.04\n",
            "18276  TrkA_TMD             PF18613.4           3     3.00     22   2.20   2.36   2.18   0.68\n",
            "18277  TrkH                 PF02386.19          9     2.61    502   0.59   0.59   0.52   0.03\n",
            "18278  TRL                  PF13146.9          25     3.91     77   0.73   0.73   0.67   0.04\n",
            "18279  TRM                  PF02005.19          9     1.16    376   0.59   0.58   0.50   0.02\n",
            "18280  Trm112p              PF03966.19         69     8.44     68   0.83   0.80   0.58   0.03\n",
            "18281  TRM13                PF05206.17         20     1.50    263   0.59   0.57   0.50   0.03\n",
            "18282  Trm56                PF01994.19         81     0.91    119   0.59   0.58   0.52   0.02\n",
            "18283  Trm5_N               PF18093.4          34    11.28     47   1.17   1.22   1.15   0.08\n",
            "18284  TrmB                 PF01978.22         45     5.61     68   0.83   0.88   0.77   0.04\n",
            "18285  TrmE_N               PF10396.12        843     2.32    117   0.59   0.61   0.46   0.02\n",
            "18286  TrmK                 PF04816.15         11     1.46    205   0.59   0.60   0.53   0.02\n",
            "18287  TrmO                 PF01980.19        357     1.62    119   0.59   0.59   0.50   0.01\n",
            "18288  TrmO_C               PF18389.4          54     3.56     65   0.86   0.82   0.80   0.04\n",
            "18289  tRNA-synt_1          PF00133.25         11     1.23    602   0.59   0.55   0.52   0.02\n",
            "18290  tRNA-synt_1b         PF00579.28         37     4.35    293   0.59   0.59   0.50   0.02\n",
            "18291  tRNA-synt_1c         PF00749.24         16     1.70    314   0.59   0.56   0.53   0.02\n",
            "18292  tRNA-synt_1c_C       PF03950.21        107     3.56    174   0.59   0.57   0.45   0.03\n",
            "18293  tRNA-synt_1d         PF00750.22          4     0.77    349   0.59   0.58   0.51   0.02\n",
            "18294  tRNA-synt_1e         PF01406.22         10     0.83    301   0.59   0.55   0.52   0.02\n",
            "18295  tRNA-synt_1f         PF01921.21         29     1.69    361   0.59   0.56   0.52   0.02\n",
            "18296  tRNA-synt_1g         PF09334.14         23     1.46    391   0.59   0.54   0.53   0.02\n",
            "18297  tRNA-synt_1_2        PF13603.9          65     1.49    185   0.59   0.57   0.52   0.02\n",
            "18298  tRNA-synt_2          PF00152.23         88     2.56    314   0.59   0.57   0.47   0.02\n",
            "18299  tRNA-synt_2b         PF00587.28          8     2.18    179   0.59   0.57   0.46   0.02\n",
            "18300  tRNA-synt_2c         PF01411.22         22     1.59    552   0.59   0.57   0.52   0.02\n",
            "18301  tRNA-synt_2d         PF01409.23         43     1.78    246   0.59   0.57   0.51   0.02\n",
            "18302  tRNA-synt_2e         PF02091.18        161     0.84    278   0.59   0.55   0.53   0.02\n",
            "18303  tRNA-synt_2_TM       PF16995.8          19     1.53    215   0.59   0.63   0.53   0.06\n",
            "18304  tRNA-synt_His        PF13393.9          68     3.38    309   0.59   0.62   0.53   0.02\n",
            "18305  tRNA-Thr_ED          PF08915.14         77     1.43    137   0.59   0.58   0.53   0.02\n",
            "18306  Trnau1ap             PF17654.4          38     2.29    118   0.59   0.51   0.43   0.07\n",
            "18307  tRNA_anti-codon      PF01336.28        540    21.06     76   0.74   0.77   0.61   0.04\n",
            "18308  tRNA_anti-like       PF12869.10          8     2.75    161   0.59   0.60   0.49   0.03\n",
            "18309  tRNA_anti_2          PF13742.9         634     3.41     95   0.60   0.60   0.46   0.02\n",
            "18310  tRNA_bind            PF01588.23         38     3.07     96   0.60   0.63   0.52   0.03\n",
            "18311  tRNA_bind_2          PF13725.9         112     1.75    235   0.59   0.59   0.53   0.04\n",
            "18312  tRNA_bind_3          PF17176.7          16     1.93    119   0.59   0.61   0.53   0.04\n",
            "18313  tRNA_bind_4          PF18490.4          16     1.10    162   0.59   0.61   0.52   0.04\n",
            "18314  tRNA_deacylase       PF04414.15         85     2.18    206   0.59   0.57   0.51   0.02\n",
            "18315  tRNA_edit            PF04073.18         63     8.48    123   0.59   0.63   0.52   0.02\n",
            "18316  tRNA_int_endo        PF01974.20         63     5.09     85   0.67   0.64   0.60   0.03\n",
            "18317  tRNA_int_endo_N      PF02778.17         43     6.44     67   0.84   0.88   0.75   0.04\n",
            "18318  tRNA_int_end_N2      PF12928.10        131     5.23     71   0.79   0.77   0.63   0.03\n",
            "18319  tRNA_lig_CPD         PF08302.14         75     1.97    253   0.59   0.57   0.49   0.03\n",
            "18320  tRNA_lig_kinase      PF08303.14          9     0.89    168   0.59   0.57   0.51   0.03\n",
            "18321  tRNA_m1G_MT          PF01746.24         15     3.15    195   0.59   0.58   0.51   0.02\n",
            "18322  tRNA_Me_trans        PF03054.19         18     1.09    202   0.59   0.57   0.52   0.02\n",
            "18323  tRNA_Me_trans_C      PF20258.1          18     2.65     77   0.73   0.73   0.68   0.02\n",
            "18324  tRNA_Me_trans_M      PF20259.1          17     2.45     66   0.85   0.84   0.79   0.04\n",
            "18325  tRNA_NucTran2_2      PF13735.9          22     3.17    149   0.59   0.60   0.53   0.04\n",
            "18326  tRNA_NucTransf2      PF09249.14         88     1.10    113   0.59   0.59   0.51   0.02\n",
            "18327  tRNA_SAD             PF07973.17        196    15.23     44   1.25   1.23   1.12   0.07\n",
            "18328  tRNA_synthFbeta      PF17759.4         213     4.99    215   0.59   0.60   0.52   0.01\n",
            "18329  tRNA_synt_1c_R1      PF04558.18        121     3.16    162   0.59   0.61   0.54   0.04\n",
            "18330  tRNA_synt_1c_R2      PF04557.18         66     2.24     94   0.61   0.62   0.49   0.07\n",
            "18331  tRNA_synt_2f         PF02092.20        419     2.17    534   0.59   0.61   0.51   0.02\n",
            "18332  tRNA_U5-meth_tr      PF05958.14          7     0.67    352   0.59   0.57   0.52   0.02\n",
            "18333  Trns_repr_metal      PF02583.20        575     4.95     77   0.73   0.76   0.64   0.07\n",
            "18334  Tropomodulin         PF03250.17         21     0.95    141   0.59   0.61   0.52   0.04\n",
            "18335  Tropomyosin          PF00261.23         17     1.66    237   0.59   0.64   0.55   0.12\n",
            "18336  Tropomyosin_1        PF12718.10         24     1.62    143   0.59   0.62   0.55   0.14\n",
            "18337  Troponin             PF00992.23         84     3.36    132   0.59   0.61   0.53   0.11\n",
            "18338  Troponin-I_N         PF11636.11          4     1.75     31   1.74   1.74   1.63   0.39\n",
            "18339  TROVE                PF05731.14         24     1.93    449   0.59   0.58   0.48   0.03\n",
            "18340  TRP                  PF06011.15        193     3.81    426   0.59   0.57   0.48   0.06\n",
            "18341  TrpBP                PF02081.18         26     0.89     69   0.82   0.84   0.76   0.05\n",
            "18342  TRPM_tetra           PF16519.8          16     2.74     56   0.99   1.03   0.97   0.12\n",
            "18343  TrpP                 PF17099.8           3     0.59    169   0.59   0.64   0.52   0.07\n",
            "18344  TRP_2                PF08344.14         41     2.07     63   0.89   0.82   0.84   0.08\n",
            "18345  Trp_dioxygenase      PF03301.16          5     0.95    346   0.59   0.58   0.52   0.02\n",
            "18346  Trp_DMAT             PF11991.11        102     5.11    363   0.59   0.56   0.45   0.01\n",
            "18347  Trp_halogenase       PF04820.17         23     1.81    454   0.59   0.57   0.52   0.01\n",
            "18348  Trp_leader1          PF08055.14          2     2.00     18   2.61   2.13   2.58   0.81\n",
            "18349  Trp_leader2          PF08056.14          1     0.84     41   1.34   1.14   1.29   0.21\n",
            "18350  TRP_N                PF14558.9         214     2.93    139   0.59   0.55   0.46   0.04\n",
            "18351  Trp_oprn_chp         PF09534.13         77     3.28    180   0.59   0.64   0.51   0.06\n",
            "18352  Trp_repressor        PF01371.22         17     2.04     88   0.65   0.67   0.61   0.05\n",
            "18353  Trp_ring             PF18669.4          17     2.61     50   1.11   1.12   1.04   0.12\n",
            "18354  Trp_syntA            PF00290.23         12     1.23    259   0.59   0.62   0.53   0.01\n",
            "18355  Trp_Tyr_perm         PF03222.16         13     1.56    394   0.59   0.61   0.54   0.05\n",
            "18356  Trs65                PF12735.10         55     3.04    313   0.59   0.60   0.47   0.02\n",
            "18357  TRSP                 PF12500.11         35     1.47    130   0.59   0.60   0.50   0.02\n",
            "18358  TruB-C_2             PF09157.14         56     7.54     58   0.96   1.02   0.90   0.04\n",
            "18359  TruB_C               PF09142.14         23     5.58     56   0.99   1.08   0.96   0.06\n",
            "18360  TruB_C_2             PF16198.8         182     4.69     64   0.88   0.91   0.74   0.04\n",
            "18361  TruB_N               PF01509.21        161     2.27    149   0.59   0.60   0.53   0.02\n",
            "18362  TruD                 PF01142.21         66     2.15    415   0.59   0.59   0.51   0.02\n",
            "18363  TrwB_AAD_bind        PF10412.12         14     1.84    387   0.59   0.59   0.53   0.01\n",
            "18364  TrwC                 PF08751.14        104     3.40    292   0.59   0.58   0.45   0.04\n",
            "18365  Trypan_glycop        PF00913.22         24     6.63    367   0.59   0.61   0.52   0.06\n",
            "18366  Trypan_glycop_C      PF10659.12         44     2.01    104   0.59   0.49   0.47   0.14\n",
            "18367  Trypan_PARP          PF05887.14          6     0.51    134   0.59   0.63   0.50   0.05\n",
            "18368  Trypco1              PF19493.2         273     7.19    100   0.59   0.64   0.45   0.03\n",
            "18369  Trypco2              PF19631.2          79     5.95     78   0.73   0.80   0.64   0.04\n",
            "18370  Trypsin              PF00089.29         70     5.66    221   0.59   0.56   0.45   0.02\n",
            "18371  Trypsin_2            PF13365.9         163    10.77    150   0.59   0.61   0.41   0.02\n",
            "18372  Tryp_alpha_amyl      PF00234.25        183     5.45     87   0.65   0.51   0.45   0.15\n",
            "18373  Tryp_FSAP            PF08248.14          2     2.00     12   2.66   2.40   2.63   1.12\n",
            "18374  Tryp_inh             PF17983.4           6     1.30     33   1.64   1.43   1.60   0.40\n",
            "18375  TryThrA_C            PF12319.11         41     2.47    216   0.59   0.44   0.56   0.17\n",
            "18376  TSA                  PF03249.16          1     0.39    510   0.59   0.60   0.51   0.01\n",
            "18377  TsaD                 PF00814.28        111     5.04    271   0.59   0.61   0.45   0.01\n",
            "18378  TsaE                 PF02367.20        622     2.13    129   0.59   0.59   0.43   0.01\n",
            "18379  TSC21                PF15217.9          17     0.71    170   0.59   0.55   0.53   0.02\n",
            "18380  TSC22                PF01166.21         19     2.21     57   0.98   1.00   0.94   0.10\n",
            "18381  Tsc35                PF15079.9           9     0.68    205   0.59   0.57   0.52   0.05\n",
            "18382  TSCPD                PF12637.10        153     1.36     74   0.76   0.72   0.69   0.07\n",
            "18383  Tsg                  PF04668.15         31     0.96    132   0.59   0.48   0.50   0.07\n",
            "18384  TSGA13               PF14994.9          10     0.70    274   0.59   0.57   0.53   0.03\n",
            "18385  TSGP1                PF07771.14          4     0.71    120   0.59   0.52   0.51   0.07\n",
            "18386  Tsi6                 PF18660.4          13     2.03     83   0.68   0.70   0.63   0.03\n",
            "18387  TSKS                 PF15358.9           3     0.68    532   0.59   0.61   0.51   0.03\n",
            "18388  TSLP                 PF15216.9           4     0.63    125   0.59   0.53   0.52   0.04\n",
            "18389  TSNAXIP1_N           PF15739.8          43     3.70    111   0.59   0.59   0.52   0.07\n",
            "18390  TSNR_N               PF04705.15          4     0.83    111   0.59   0.62   0.53   0.03\n",
            "18391  TSP1_ADAMTS          PF19030.3          21     2.39     55   1.01   0.78   0.94   0.23\n",
            "18392  TSP1_CCN             PF19035.3          33     2.24     44   1.25   1.00   1.20   0.31\n",
            "18393  TSP1_spondin         PF19028.3          80     2.55     53   1.05   0.81   0.99   0.26\n",
            "18394  TSP3_bac             PF18884.3          87    51.50     22   2.41   2.47   2.40   0.52\n",
            "18395  Tsp45I               PF06300.15          4     0.55    260   0.59   0.58   0.52   0.02\n",
            "18396  TSP9                 PF11493.11         14     1.17     81   0.70   0.75   0.60   0.04\n",
            "18397  TspO_MBR             PF03073.18        425     5.31    143   0.59   0.56   0.47   0.10\n",
            "18398  TSP_1                PF00090.22         24     4.80     49   1.13   0.90   1.06   0.26\n",
            "18399  TSP_3                PF02412.21         11     3.59     36   1.51   1.41   1.47   0.40\n",
            "18400  TSP_C                PF05735.15         26     1.07    198   0.59   0.54   0.53   0.02\n",
            "18401  TSP_NTD              PF17804.4          78     3.74    189   0.59   0.56   0.53   0.06\n",
            "18402  TSR                  PF18487.4          24     1.71     50   1.11   0.92   1.06   0.20\n",
            "18403  TssC                 PF17541.5          17     0.60    430   0.59   0.59   0.53   0.02\n",
            "18404  TSSC4                PF15264.9          13     1.27    126   0.59   0.57   0.49   0.04\n",
            "18405  TssD                 PF17642.4          76     3.28    128   0.59   0.56   0.54   0.03\n",
            "18406  TssN                 PF17555.5          20     1.94    274   0.59   0.56   0.54   0.05\n",
            "18407  TssO                 PF17561.5          23     3.80    148   0.59   0.57   0.55   0.06\n",
            "18408  TssR                 PF17643.4          17     2.30    746   0.59   0.58   0.54   0.04\n",
            "18409  TT1725               PF18324.4          78     1.20    110   0.59   0.58   0.53   0.02\n",
            "18410  TTC3_DZIP3_dom       PF19179.3          30     0.78    116   0.59   0.50   0.53   0.05\n",
            "18411  TTC5_OB              PF16669.8          34     2.64    115   0.59   0.59   0.53   0.03\n",
            "18412  TTC7_N               PF19440.2           9     1.04    386   0.59   0.61   0.53   0.03\n",
            "18413  TTc_toxin_rep        PF18807.4          21     2.86     45   1.22   1.23   1.20   0.06\n",
            "18414  TTD                  PF12148.11         34     1.19    155   0.59   0.56   0.52   0.03\n",
            "18415  Tti2                 PF10521.12         63     4.80    281   0.59   0.60   0.49   0.02\n",
            "18416  TTKRSYEDQ            PF10212.12          8     0.75    523   0.59   0.59   0.53   0.03\n",
            "18417  TTL                  PF03133.18         20     3.17    295   0.59   0.55   0.51   0.03\n",
            "18418  TTR-52               PF01060.26         95     2.94     81   0.70   0.64   0.60   0.03\n",
            "18419  TTRAP                PF14203.9          20     2.77     62   0.90   0.92   0.84   0.08\n",
            "18420  TTSSLRR              PF12468.11         13     5.15     54   1.03   0.94   0.93   0.09\n",
            "18421  TT_ORF1              PF02956.17         58     2.50    525   0.59   0.51   0.50   0.04\n",
            "18422  TT_ORF2              PF02957.18         17     2.77    122   0.59   0.54   0.50   0.05\n",
            "18423  TT_ORF2a             PF08197.14          1     0.75     49   1.13   1.14   1.07   0.08\n",
            "18424  Tub                  PF01167.21        127     2.08    263   0.59   0.56   0.45   0.02\n",
            "18425  TubC_N               PF18563.4          41     6.05     52   1.07   1.14   1.06   0.07\n",
            "18426  Tube                 PF17212.6           6     1.25    169   0.59   0.58   0.52   0.02\n",
            "18427  Tuberculin           PF12198.11          1     0.46     83   0.69   0.71   0.62   0.03\n",
            "18428  Tuberin              PF03542.19         14     1.27    351   0.59   0.58   0.53   0.04\n",
            "18429  Tubulin              PF00091.28         87     4.13    197   0.59   0.59   0.50   0.02\n",
            "18430  Tubulin-binding      PF00418.22         78     9.51     31   1.74   1.75   1.71   0.15\n",
            "18431  Tubulin_2            PF13809.9          41     4.46    351   0.59   0.60   0.46   0.01\n",
            "18432  Tubulin_3            PF14881.9          16     0.75    180   0.59   0.56   0.50   0.02\n",
            "18433  Tubulin_C            PF03953.20         34     1.79    126   0.59   0.55   0.53   0.03\n",
            "18434  Tub_N                PF16322.8          20     1.77    200   0.59   0.61   0.48   0.07\n",
            "18435  TUDOR                PF00567.27         44     7.00    122   0.59   0.54   0.53   0.02\n",
            "18436  Tudor-knot           PF11717.11         37     3.92     55   1.01   0.92   0.90   0.07\n",
            "18437  Tudor_1_RapA         PF18339.4          76     2.24     51   1.09   1.09   1.06   0.04\n",
            "18438  Tudor_2              PF18104.4          51    11.40     35   1.55   1.47   1.54   0.11\n",
            "18439  Tudor_3              PF18115.4          56     5.34     52   1.07   1.01   1.01   0.04\n",
            "18440  Tudor_4              PF18358.4          12     1.44     50   1.11   1.05   1.07   0.05\n",
            "18441  Tudor_5              PF18359.4          30     3.42     54   1.03   0.99   0.94   0.06\n",
            "18442  Tudor_FRX1           PF18336.4           7     1.30     49   1.13   1.10   1.09   0.04\n",
            "18443  Tudor_RapA           PF18337.4         128     4.88     62   0.90   0.91   0.86   0.03\n",
            "18444  TUG-UBL1             PF11470.11         97     7.91     65   0.86   0.89   0.82   0.04\n",
            "18445  Tugs                 PF17840.4          11     2.16     47   1.17   1.19   1.15   0.10\n",
            "18446  Tup_N                PF08581.13         51     2.47     77   0.73   0.74   0.70   0.11\n",
            "18447  Turandot             PF07240.14         10     2.27     81   0.70   0.71   0.64   0.02\n",
            "18448  TusA                 PF01206.20        132    11.84     70   0.80   0.81   0.76   0.03\n",
            "18449  TUSC2                PF15000.9          15     1.21    112   0.59   0.56   0.52   0.02\n",
            "18450  TUTase               PF19088.3          13     0.63    218   0.59   0.58   0.53   0.03\n",
            "18451  TUTF7_u4             PF16631.8           9     0.78     88   0.65   0.64   0.59   0.15\n",
            "18452  Tweety               PF04906.16         12     1.03    431   0.59   0.57   0.53   0.03\n",
            "18453  TxDE                 PF18711.4          20     3.22     55   1.01   1.05   0.98   0.04\n",
            "18454  Ty3_capsid           PF19259.2           5     1.48    198   0.59   0.55   0.54   0.03\n",
            "18455  TYA                  PF01021.22          6     0.67    384   0.59   0.57   0.53   0.02\n",
            "18456  TyeA                 PF09059.13         11     1.89     81   0.70   0.72   0.66   0.04\n",
            "18457  TylF                 PF05711.14         13     0.90    255   0.59   0.58   0.52   0.01\n",
            "18458  Tymo_45kd_70kd       PF03251.16          1     0.39    468   0.59   0.58   0.52   0.04\n",
            "18459  Tymo_coat            PF00983.21          3     0.76    179   0.59   0.59   0.51   0.03\n",
            "18460  Tyosinase_C          PF18132.4         229     4.08    122   0.59   0.58   0.47   0.02\n",
            "18461  TypeIII_RM_meth      PF12564.11         11     1.95     56   0.99   0.99   0.96   0.08\n",
            "18462  Type_III_YscG        PF09477.13          4     0.76    116   0.59   0.60   0.52   0.02\n",
            "18463  Type_III_YscX        PF09474.13         29     4.26    121   0.59   0.63   0.55   0.04\n",
            "18464  Type_ISP_C           PF18135.4          78     4.17    344   0.59   0.55   0.49   0.02\n",
            "18465  Tyr-DNA_phospho      PF06087.15        111     2.95    450   0.59   0.55   0.46   0.02\n",
            "18466  Tyrosinase           PF00264.23        120     3.97    222   0.59   0.48   0.42   0.05\n",
            "18467  TyrRSs_C             PF16714.8          45     1.38    120   0.59   0.60   0.54   0.02\n",
            "18468  Tyr_Deacylase        PF02580.19        799     2.36    144   0.59   0.61   0.49   0.02\n",
            "18469  TYW3                 PF02676.17        137     2.60    227   0.59   0.59   0.47   0.02\n",
            "18470  T_Ag_DNA_bind        PF02217.19         11     1.18     94   0.61   0.57   0.55   0.05\n",
            "18471  T_cell_tran_alt      PF15128.9           7     0.58     92   0.62   0.58   0.55   0.02\n",
            "18472  T_hemolysin          PF12261.11         32     1.77    174   0.59   0.58   0.55   0.02\n",
            "18473  U-box                PF04564.18         16     2.18     73   0.77   0.74   0.72   0.04\n",
            "18474  U1snRNP70_N          PF12220.11        128     4.25     92   0.62   0.61   0.53   0.07\n",
            "18475  U3snoRNP10           PF12397.11        144     7.80    116   0.59   0.64   0.46   0.06\n",
            "18476  U3_assoc_6           PF08640.14         97     3.71     86   0.66   0.66   0.60   0.08\n",
            "18477  U3_snoRNA_assoc      PF08297.14         66     4.63     90   0.63   0.66   0.51   0.08\n",
            "18478  U5_2-snRNA_bdg       PF10597.12         17     0.64    134   0.59   0.59   0.53   0.02\n",
            "18479  U6-snRNA_bdg         PF10596.12          9     0.77    159   0.59   0.55   0.52   0.02\n",
            "18480  U62_UL91             PF17442.5           3     0.96     65   0.86   0.88   0.82   0.06\n",
            "18481  U71                  PF17474.5           2     0.73     52   1.07   1.03   1.02   0.08\n",
            "18482  U79_P34              PF03064.19         14     1.29    135   0.59   0.57   0.52   0.03\n",
            "18483  U83                  PF17465.5           2     0.43     97   0.59   0.60   0.52   0.03\n",
            "18484  UAA                  PF08449.14         17     2.21    307   0.59   0.58   0.52   0.06\n",
            "18485  UAE_UbL              PF14732.9          83     4.68     87   0.65   0.68   0.54   0.05\n",
            "18486  UAF_Rrn10            PF05234.14          2     0.63    120   0.59   0.59   0.50   0.02\n",
            "18487  Ub-Mut7C             PF14451.9          11     1.66     81   0.70   0.72   0.66   0.02\n",
            "18488  Ub-RnfH              PF03658.17        149     2.23     83   0.68   0.70   0.63   0.02\n",
            "18489  UB2H                 PF14814.9          82     4.29     85   0.67   0.68   0.63   0.03\n",
            "18490  UBA                  PF00627.34         61    61.00     37   1.40   1.42   1.38   0.09\n",
            "18491  UBA2_C               PF16195.8           9     1.42     95   0.60   0.62   0.52   0.12\n",
            "18492  UBact                PF20513.1          16     2.07     43   1.28   1.26   1.23   0.18\n",
            "18493  UBA_2                PF08587.14         41     4.12     46   1.20   1.21   1.13   0.11\n",
            "18494  UBA_3                PF09288.13          6     1.78     55   1.01   1.07   0.97   0.07\n",
            "18495  UBA_4                PF14555.9         152    39.88     43   1.28   1.27   1.24   0.09\n",
            "18496  UBA_5                PF16577.8           2     0.66     62   0.90   0.90   0.84   0.01\n",
            "18497  UBA_6                PF18039.4           6     3.00     42   1.31   1.38   1.27   0.06\n",
            "18498  UBA_E1_SCCH          PF10585.12        210     2.72    256   0.59   0.55   0.48   0.04\n",
            "18499  UBD                  PF16455.8          50     1.12    104   0.59   0.56   0.50   0.02\n",
            "18500  UbiA                 PF01040.21        878    12.54    251   0.59   0.60   0.49   0.08\n",
            "18501  UbiD                 PF01977.19        397     4.99    406   0.59   0.59   0.47   0.01\n",
            "18502  Ubie_methyltran      PF01209.21          5     0.79    233   0.59   0.58   0.52   0.01\n",
            "18503  Ubiq-assoc           PF09145.13         10     1.96     44   1.25   1.27   1.21   0.09\n",
            "18504  Ubiq-Cytc-red_N      PF09165.13         20     3.42     75   0.75   0.82   0.71   0.07\n",
            "18505  ubiquitin            PF00240.26         60     9.61     72   0.78   0.81   0.73   0.05\n",
            "18506  Ubiquitin_2          PF14560.9          65     3.88     87   0.65   0.66   0.60   0.03\n",
            "18507  Ubiquitin_3          PF14836.9           4     0.82     88   0.65   0.62   0.57   0.03\n",
            "18508  Ubiquitin_4          PF18036.4          15     1.51     89   0.64   0.60   0.59   0.05\n",
            "18509  Ubiquitin_5          PF18037.4          14     1.17     98   0.59   0.62   0.52   0.05\n",
            "18510  Ubiq_cyt_C_chap      PF03981.15         61     6.67    141   0.59   0.59   0.51   0.03\n",
            "18511  UBM                  PF14377.9         161   161.00     35   1.28   1.35   1.25   0.19\n",
            "18512  UBN_AB               PF14075.9          25     2.80    212   0.59   0.57   0.52   0.04\n",
            "18513  UBX                  PF00789.23         19     4.02     81   0.70   0.71   0.62   0.03\n",
            "18514  UBZ_FAAP20           PF15750.8           8     2.26     35   1.55   1.43   1.53   0.19\n",
            "18515  UCH                  PF00443.32         54     3.46    278   0.59   0.55   0.40   0.03\n",
            "18516  UCH_1                PF13423.9         200     4.05    322   0.59   0.55   0.41   0.03\n",
            "18517  UCH_C                PF18031.4          31     6.66     46   1.20   1.18   1.18   0.11\n",
            "18518  UCH_N                PF16674.8          33     1.68    105   0.59   0.60   0.54   0.04\n",
            "18519  UCMA                 PF17085.8           3     0.55    134   0.59   0.56   0.52   0.05\n",
            "18520  UcrQ                 PF02939.19        101     3.44     78   0.73   0.67   0.69   0.03\n",
            "18521  UCR_14kD             PF02271.19        103     2.01    100   0.59   0.58   0.55   0.04\n",
            "18522  UCR_6-4kD            PF08997.13         14     1.81     55   1.01   0.92   0.97   0.08\n",
            "18523  UCR_Fe-S_N           PF10399.12         31     4.59     41   1.34   1.36   1.32   0.08\n",
            "18524  UCR_hinge            PF02320.19        112     3.40     66   0.85   0.73   0.79   0.14\n",
            "18525  UCR_TM               PF02921.17         42     4.55     66   0.85   0.85   0.81   0.05\n",
            "18526  UCR_UQCRX_QCR9       PF05365.15         97     4.81     53   1.05   0.97   1.03   0.09\n",
            "18527  uDENN                PF03456.21        145     8.03     57   0.98   0.89   0.71   0.06\n",
            "18528  UDG                  PF03167.22        147     9.08    155   0.59   0.58   0.41   0.01\n",
            "18529  UDI                  PF18880.3           2     0.73     82   0.69   0.70   0.63   0.03\n",
            "18530  UDP-g_GGTase         PF06427.14        155     1.57    109   0.59   0.59   0.51   0.01\n",
            "18531  UDPGP                PF01704.21         76     1.62    412   0.59   0.59   0.52   0.02\n",
            "18532  UDPGT                PF00201.21         13     1.01    499   0.59   0.57   0.53   0.01\n",
            "18533  UDPG_MGDP_dh         PF00984.22        586     3.26     94   0.61   0.61   0.56   0.02\n",
            "18534  UDPG_MGDP_dh_C       PF03720.18        152     4.55    105   0.59   0.59   0.50   0.02\n",
            "18535  UDPG_MGDP_dh_N       PF03721.17         19     1.64    187   0.59   0.61   0.52   0.02\n",
            "18536  Uds1                 PF15456.9          85     3.49    130   0.59   0.62   0.52   0.08\n",
            "18537  UEV                  PF05743.16         21     1.56    121   0.59   0.55   0.52   0.02\n",
            "18538  UFC1                 PF08694.14         26     0.59    155   0.59   0.54   0.52   0.02\n",
            "18539  UFD1                 PF03152.17         99     1.66    173   0.59   0.59   0.51   0.02\n",
            "18540  Ufd2P_core           PF10408.12        153     4.50    627   0.59   0.57   0.46   0.04\n",
            "18541  Ufm1                 PF03671.17         13     1.23     75   0.75   0.77   0.70   0.03\n",
            "18542  UIM                  PF02809.23        112   112.00     17   1.96   2.07   1.96   0.51\n",
            "18543  UK                   PF02512.17          1     0.39     96   0.60   0.56   0.53   0.03\n",
            "18544  UL11                 PF11094.11          3     1.81     39   1.40   1.34   1.28   0.10\n",
            "18545  UL141                PF16758.8           2     0.58    191   0.59   0.54   0.52   0.01\n",
            "18546  UL16                 PF17622.5           1     0.38    204   0.59   0.58   0.51   0.01\n",
            "18547  UL17                 PF17640.5           1     0.36    102   0.59   0.55   0.52   0.03\n",
            "18548  UL2                  PF08196.14          1     0.63     59   0.95   0.96   0.89   0.06\n",
            "18549  UL20                 PF17582.5           1     0.37    304   0.59   0.56   0.51   0.02\n",
            "18550  UL21a                PF17636.5           1     0.37    123   0.59   0.58   0.52   0.02\n",
            "18551  UL40                 PF10682.12          1     0.39    214   0.59   0.57   0.52   0.02\n",
            "18552  UL41A                PF17591.5           1     0.46     78   0.72   0.68   0.66   0.04\n",
            "18553  UL42                 PF17638.5           1     0.37    125   0.59   0.55   0.51   0.02\n",
            "18554  UL45                 PF05473.15         11     2.20    200   0.59   0.56   0.51   0.03\n",
            "18555  UL73_N               PF12522.11          5     4.35     27   1.98   2.17   1.96   0.59\n",
            "18556  UL97                 PF06734.15          2     0.61    182   0.59   0.57   0.52   0.03\n",
            "18557  ULD                  PF16534.8          17     1.08    109   0.59   0.60   0.53   0.03\n",
            "18558  Uma2                 PF05685.15        139     9.86    172   0.59   0.58   0.45   0.02\n",
            "18559  Umbravirus_LDM       PF04817.15          4     0.96    231   0.59   0.59   0.54   0.05\n",
            "18560  UME                  PF08064.16        115     4.10    102   0.59   0.58   0.54   0.04\n",
            "18561  UMP1                 PF05348.14         65     2.64    117   0.59   0.58   0.50   0.03\n",
            "18562  UMPH-1               PF05822.15          9     0.84    246   0.59   0.58   0.53   0.02\n",
            "18563  UnbV_ASPIC           PF07593.15        253     8.27     68   0.83   0.83   0.70   0.04\n",
            "18564  UNC-50               PF05216.16         89     1.61    221   0.59   0.55   0.52   0.07\n",
            "18565  UNC-79               PF14776.9           9     0.81    524   0.59   0.55   0.52   0.02\n",
            "18566  UNC-93               PF05978.19          9     1.36    379   0.59   0.59   0.54   0.05\n",
            "18567  UNC119_bdg           PF15435.9           7     0.56    198   0.59   0.59   0.52   0.02\n",
            "18568  UNC45-central        PF11701.11         95     5.82    153   0.59   0.61   0.51   0.04\n",
            "18569  UNC80                PF19424.2          16     0.82    707   0.59   0.57   0.51   0.02\n",
            "18570  UNC80_C              PF20262.1          16     0.79   1079   0.59   0.58   0.52   0.02\n",
            "18571  UNC80_N              PF15778.8          15     0.78    187   0.59   0.56   0.51   0.02\n",
            "18572  Unpaired             PF15972.8          10     1.01    273   0.59   0.55   0.50   0.05\n",
            "18573  Unstab_antitox       PF09720.13        141    21.82     54   1.03   1.08   0.98   0.12\n",
            "18574  UN_NPL4              PF11543.11          7     1.77     80   0.71   0.74   0.65   0.04\n",
            "18575  UPA                  PF17217.6          17     0.99    140   0.59   0.56   0.53   0.03\n",
            "18576  UPAR_LY6             PF00021.24         27     3.22     77   0.73   0.56   0.63   0.20\n",
            "18577  UPAR_LY6_2           PF16975.8          15     0.53    106   0.59   0.48   0.53   0.07\n",
            "18578  UPA_2                PF17809.4           4     0.76    131   0.59   0.58   0.52   0.02\n",
            "18579  UPF0004              PF00919.23        109     1.94     98   0.59   0.57   0.53   0.04\n",
            "18580  UPF0014              PF03649.16        155     2.57    242   0.59   0.62   0.56   0.07\n",
            "18581  UPF0016              PF01169.22        614     5.39     75   0.75   0.81   0.67   0.12\n",
            "18582  UPF0020              PF01170.21         13     2.40    197   0.59   0.59   0.51   0.01\n",
            "18583  UPF0029              PF01205.22        688     2.30    108   0.59   0.58   0.48   0.02\n",
            "18584  UPF0047              PF01894.20        618     2.33    115   0.59   0.58   0.46   0.03\n",
            "18585  UPF0058              PF01893.19         31     1.50     87   0.65   0.64   0.59   0.06\n",
            "18586  UPF0060              PF02694.18          4     0.62    107   0.59   0.59   0.52   0.07\n",
            "18587  UPF0093              PF03653.16         30     1.57    146   0.59   0.56   0.53   0.07\n",
            "18588  UPF0102              PF02021.20        780     3.18     79   0.72   0.73   0.52   0.05\n",
            "18589  UPF0113              PF03657.16         22     3.77     73   0.77   0.78   0.70   0.03\n",
            "18590  UPF0113_N            PF17833.4          28     3.13     83   0.68   0.70   0.58   0.04\n",
            "18591  UPF0114              PF03350.19        148     3.06    118   0.59   0.62   0.50   0.08\n",
            "18592  UPF0122              PF04297.17          9     1.08    101   0.59   0.59   0.54   0.07\n",
            "18593  UPF0128              PF03673.16          5     0.70    221   0.59   0.59   0.52   0.01\n",
            "18594  UPF0137              PF03677.16          5     0.86    237   0.59   0.60   0.52   0.03\n",
            "18595  UPF0146              PF03686.16          4     0.96    129   0.59   0.61   0.52   0.02\n",
            "18596  UPF0147              PF03685.16         52     1.64     81   0.70   0.72   0.66   0.05\n",
            "18597  UPF0149              PF03695.16        136     7.30    177   0.59   0.59   0.45   0.03\n",
            "18598  UPF0154              PF03672.16         55     3.32     60   0.93   0.90   0.90   0.08\n",
            "18599  UPF0158              PF03682.16          6     0.96    157   0.59   0.58   0.52   0.04\n",
            "18600  UPF0160              PF03690.16        184     1.96    319   0.59   0.56   0.51   0.02\n",
            "18601  UPF0164              PF03687.16          5     0.99    325   0.59   0.60   0.51   0.02\n",
            "18602  UPF0167              PF03691.17         15     1.31    176   0.59   0.49   0.51   0.04\n",
            "18603  UPF0172              PF03665.16         79     2.50    193   0.59   0.57   0.48   0.03\n",
            "18604  UPF0175              PF03683.16        102     7.31     75   0.75   0.80   0.72   0.04\n",
            "18605  UPF0176_N            PF17773.4          73     2.65     92   0.62   0.62   0.55   0.02\n",
            "18606  UPF0179              PF03684.16         57     1.63    140   0.59   0.53   0.52   0.06\n",
            "18607  UPF0180              PF03698.16         49     3.20     77   0.73   0.76   0.67   0.06\n",
            "18608  UPF0181              PF03701.17         25     2.08     50   1.11   1.12   1.06   0.11\n",
            "18609  UPF0182              PF03699.16        209     2.04    758   0.59   0.56   0.50   0.02\n",
            "18610  UPF0184              PF03670.16          3     0.95     83   0.68   0.70   0.63   0.07\n",
            "18611  UPF0193              PF05250.14         24     1.64    216   0.59   0.59   0.51   0.05\n",
            "18612  UPF0203              PF05254.15         56     2.00     69   0.82   0.67   0.73   0.12\n",
            "18613  UPF0220              PF05255.14         67     1.78    166   0.59   0.57   0.51   0.04\n",
            "18614  UPF0223              PF05256.15         49     1.65     86   0.66   0.62   0.62   0.06\n",
            "18615  UPF0225              PF17775.4         130     2.05     99   0.59   0.55   0.50   0.05\n",
            "18616  UPF0227              PF05728.15          9     1.48    187   0.59   0.58   0.53   0.01\n",
            "18617  UPF0228              PF05727.14         10     1.04    124   0.59   0.56   0.50   0.05\n",
            "18618  UPF0231              PF06062.14         33     1.68    121   0.59   0.56   0.53   0.04\n",
            "18619  UPF0236              PF06782.14          3     0.88    479   0.59   0.58   0.52   0.03\n",
            "18620  UPF0239              PF06783.14         14     1.10     88   0.65   0.65   0.58   0.03\n",
            "18621  UPF0240              PF06784.14         28     1.81    168   0.59   0.58   0.51   0.04\n",
            "18622  UPF0242              PF06785.14          6     1.49    194   0.59   0.60   0.50   0.06\n",
            "18623  UPF0253              PF06786.15          8     1.03     65   0.86   0.86   0.80   0.04\n",
            "18624  UPF0257              PF06788.16          5     0.73    235   0.59   0.58   0.53   0.02\n",
            "18625  UPF0259              PF06790.14          5     0.82    248   0.59   0.62   0.53   0.09\n",
            "18626  UPF0261              PF06792.14        119     1.91    404   0.59   0.61   0.52   0.02\n",
            "18627  UPF0262              PF06793.15         46     0.93    153   0.59   0.60   0.53   0.03\n",
            "18628  UPF0270              PF06794.15         57     3.04     69   0.81   0.84   0.76   0.04\n",
            "18629  UPF0300              PF08594.13         12     1.40    212   0.59   0.54   0.54   0.03\n",
            "18630  UPF0302              PF08864.13         31     2.09    105   0.59   0.57   0.55   0.04\n",
            "18631  UPF0370              PF13980.9           9     0.70     61   0.92   0.82   0.85   0.09\n",
            "18632  UPF0444              PF15475.9           7     0.78     91   0.63   0.69   0.55   0.07\n",
            "18633  UPF0449              PF15136.9          28     2.02    100   0.59   0.60   0.52   0.05\n",
            "18634  UPF0489              PF12640.10         15     2.56    175   0.59   0.54   0.48   0.03\n",
            "18635  UPF0492              PF15744.8           6     0.99    371   0.59   0.58   0.52   0.04\n",
            "18636  UPF0506              PF11703.11          3     0.95     59   0.95   0.71   0.89   0.27\n",
            "18637  UPF0515              PF15135.9           9     0.89    271   0.59   0.53   0.51   0.03\n",
            "18638  UPF0524              PF15823.8           9     0.51    239   0.59   0.55   0.52   0.03\n",
            "18639  UPF0542              PF15086.9           5     1.26     68   0.83   0.82   0.76   0.05\n",
            "18640  UPF0547              PF10571.12         13     9.10     26   2.06   1.74   2.02   0.44\n",
            "18641  UPF0552              PF10574.12         12     0.85    224   0.59   0.58   0.53   0.02\n",
            "18642  UPF0556              PF10572.12         20     1.44    132   0.59   0.55   0.49   0.03\n",
            "18643  UPF0560              PF10577.12         14     0.94    817   0.59   0.58   0.51   0.02\n",
            "18644  UPF0561              PF10573.12          3     0.51    120   0.59   0.57   0.52   0.03\n",
            "18645  UPF0564              PF10595.12          8     2.07    364   0.59   0.57   0.52   0.11\n",
            "18646  UPF0640              PF15114.9          14     1.08     67   0.84   0.79   0.78   0.05\n",
            "18647  UPF0669              PF17065.8           5     0.52    185   0.59   0.58   0.52   0.01\n",
            "18648  UPF0688              PF15772.8          15     1.17    233   0.59   0.59   0.53   0.06\n",
            "18649  UPF0697              PF15117.9           5     0.47     98   0.59   0.57   0.52   0.03\n",
            "18650  UPF0715              PF17094.8           4     0.73    115   0.59   0.58   0.52   0.12\n",
            "18651  UPF0728              PF15092.9          19     1.28     88   0.65   0.63   0.59   0.02\n",
            "18652  UPF0730              PF15827.8           3     1.14     46   1.20   1.11   1.14   0.12\n",
            "18653  UPF0731              PF14982.9           2     0.52     78   0.72   0.71   0.65   0.03\n",
            "18654  UPF0738              PF19785.2         128     1.82    119   0.59   0.58   0.53   0.04\n",
            "18655  UPF0758_N            PF20582.1         742     2.91     78   0.73   0.80   0.64   0.03\n",
            "18656  UPF0767              PF15990.8          18     1.18     83   0.68   0.68   0.61   0.02\n",
            "18657  UPF1_1B_dom          PF18141.4          17     1.41     93   0.61   0.59   0.54   0.02\n",
            "18658  UPF1_Zn_bind         PF09416.13         44     1.07    152   0.59   0.51   0.53   0.05\n",
            "18659  Upf2                 PF04050.17        108     4.04    133   0.59   0.61   0.45   0.08\n",
            "18660  UPRTase              PF14681.9          85     2.65    207   0.59   0.61   0.53   0.02\n",
            "18661  UpxZ                 PF06603.14         12     0.91    105   0.59   0.60   0.53   0.03\n",
            "18662  UQCC2_CBP6           PF20180.1         178     4.82     90   0.63   0.61   0.51   0.05\n",
            "18663  UQCC3                PF15141.9          20     1.50     87   0.65   0.67   0.58   0.03\n",
            "18664  UQ_con               PF00179.29         64     2.35    140   0.59   0.54   0.50   0.02\n",
            "18665  Urate_ox_N           PF06181.14         50     0.97    297   0.59   0.55   0.53   0.04\n",
            "18666  Urb2                 PF10441.12        106     5.22    206   0.59   0.59   0.46   0.03\n",
            "18667  Urease_alpha         PF00449.23         67     0.82    121   0.59   0.62   0.52   0.03\n",
            "18668  Urease_beta          PF00699.23        294     1.12     98   0.59   0.60   0.52   0.02\n",
            "18669  Urease_gamma         PF00547.21        148     1.19     99   0.59   0.63   0.54   0.04\n",
            "18670  Urease_linker        PF18473.4          13     4.43     34   1.59   1.61   1.58   0.17\n",
            "18671  UreD                 PF01774.20        355     5.10    213   0.59   0.60   0.43   0.01\n",
            "18672  UreE_C               PF05194.15         97     2.53     87   0.65   0.60   0.51   0.06\n",
            "18673  UreE_N               PF02814.18         72    12.16     62   0.90   0.97   0.87   0.06\n",
            "18674  UreF                 PF01730.19        289     5.27    133   0.59   0.61   0.49   0.05\n",
            "18675  Ureide_permease      PF07168.14         13     0.61    361   0.59   0.60   0.52   0.02\n",
            "18676  Ureidogly_lyase      PF04115.15         42     1.73    190   0.59   0.56   0.47   0.02\n",
            "18677  Uricase              PF01014.21        253     4.39    133   0.59   0.57   0.46   0.03\n",
            "18678  Urm1                 PF09138.14         18     0.89     96   0.60   0.61   0.52   0.03\n",
            "18679  URO-D                PF01208.20        121     5.35    345   0.59   0.59   0.50   0.01\n",
            "18680  Urocanase            PF01175.21        130     1.34    209   0.59   0.59   0.53   0.02\n",
            "18681  Urocanase_C          PF17392.5          79     1.01    196   0.59   0.58   0.53   0.02\n",
            "18682  Urocanase_N          PF17391.5          64     1.04    127   0.59   0.57   0.54   0.02\n",
            "18683  Uroplakin_II         PF07353.15          2     0.50    179   0.59   0.62   0.53   0.02\n",
            "18684  Urotensin_II         PF02083.18          4     4.00     12   3.06   2.56   3.05   0.75\n",
            "18685  US10                 PF17617.5           1     0.37    161   0.59   0.57   0.52   0.01\n",
            "18686  US2                  PF02476.18         15     2.05    126   0.59   0.55   0.49   0.02\n",
            "18687  US22                 PF02393.19         36     5.93    125   0.59   0.57   0.46   0.02\n",
            "18688  US30                 PF17624.5           1     0.38    282   0.59   0.58   0.51   0.01\n",
            "18689  US6                  PF17616.5           1     0.37    161   0.60   0.56   0.52   0.02\n",
            "18690  Use1                 PF09753.12         31     2.56    245   0.59   0.59   0.52   0.06\n",
            "18691  Usg                  PF06233.15         33     0.91     80   0.71   0.66   0.66   0.03\n",
            "18692  Usher                PF00577.23         17     1.95    549   0.59   0.58   0.53   0.03\n",
            "18693  Usher_TcfC           PF17271.5           4     1.01    420   0.59   0.58   0.52   0.03\n",
            "18694  Uso1_p115_C          PF04871.16         15     2.85    121   0.59   0.65   0.49   0.15\n",
            "18695  Uso1_p115_head       PF04869.17         74     2.81    314   0.59   0.59   0.49   0.03\n",
            "18696  Usp                  PF00582.29        159    16.59    141   0.59   0.64   0.53   0.03\n",
            "18697  USP19_linker         PF16602.8           7     0.59    121   0.59   0.58   0.50   0.05\n",
            "18698  USP47_C              PF19718.2           9     0.94    240   0.59   0.57   0.53   0.04\n",
            "18699  USP7_C2              PF14533.9         133     3.61    209   0.59   0.58   0.49   0.04\n",
            "18700  USP7_ICP0_bdg        PF12436.11        102     2.47    243   0.59   0.57   0.46   0.03\n",
            "18701  USP8_dimer           PF08969.14         18     2.38    114   0.59   0.58   0.53   0.06\n",
            "18702  USP8_interact        PF08941.13         18     0.84    179   0.59   0.57   0.52   0.02\n",
            "18703  UspA1_rep            PF18792.4          10    10.00     13   2.38   2.49   2.38   0.58\n",
            "18704  UspB                 PF10625.12          8     0.66    107   0.59   0.57   0.53   0.05\n",
            "18705  Ustilago_mating      PF05722.15          2     0.54    286   0.59   0.59   0.51   0.02\n",
            "18706  UstYa                PF11807.11         45     5.76    220   0.59   0.51   0.47   0.03\n",
            "18707  UT                   PF03253.17         61     3.47    293   0.59   0.62   0.54   0.07\n",
            "18708  Uteroglobin          PF01099.20         72     3.45     90   0.63   0.62   0.57   0.06\n",
            "18709  Utp11                PF03998.16        161     2.59    247   0.59   0.58   0.47   0.11\n",
            "18710  Utp12                PF04003.15        375     8.05    107   0.59   0.62   0.52   0.04\n",
            "18711  Utp13                PF08625.14        145     2.79    142   0.59   0.59   0.52   0.05\n",
            "18712  Utp14                PF04615.16        134     3.66    729   0.59   0.59   0.45   0.09\n",
            "18713  UTP15_C              PF09384.13        141     4.01    147   0.59   0.61   0.54   0.04\n",
            "18714  Utp21                PF04192.15        112     2.20    213   0.59   0.60   0.51   0.04\n",
            "18715  UTP25                PF06862.15         26     1.32    473   0.59   0.57   0.52   0.03\n",
            "18716  Utp8                 PF10395.12         24     2.43    690   0.59   0.60   0.48   0.03\n",
            "18717  UTRA                 PF07702.16        140    10.08    141   0.59   0.61   0.56   0.02\n",
            "18718  UVB_sens_prot        PF04884.17         61     2.09    241   0.59   0.60   0.53   0.03\n",
            "18719  UvdE                 PF03851.17          6     1.08    276   0.59   0.56   0.52   0.02\n",
            "18720  UVR                  PF02151.22         69    69.00     36   1.29   1.36   1.28   0.26\n",
            "18721  UvrA_DNA-bind        PF17755.4         381     4.25    110   0.59   0.58   0.53   0.02\n",
            "18722  UvrA_inter           PF17760.4          72     2.58    109   0.59   0.62   0.55   0.04\n",
            "18723  UvrB                 PF12344.11        623     4.25     43   1.28   1.22   1.26   0.12\n",
            "18724  UvrB_inter           PF17757.4         265     2.80     91   0.63   0.63   0.59   0.02\n",
            "18725  UvrC_RNaseH_dom      PF08459.14        665     2.42    161   0.59   0.60   0.48   0.02\n",
            "18726  UvrD-helicase        PF00580.24         36     3.16    315   0.59   0.59   0.48   0.04\n",
            "18727  UvrD_C               PF13361.9          18     2.70    348   0.59   0.59   0.46   0.04\n",
            "18728  UvrD_C_2             PF13538.9          66    12.28     52   1.07   1.05   1.04   0.05\n",
            "18729  UvsW-1               PF11637.11          2     1.18     56   1.00   1.02   0.94   0.07\n",
            "18730  UvsY                 PF11056.11          8     1.32    128   0.59   0.56   0.53   0.05\n",
            "18731  UxaC                 PF02614.17         17     0.90    464   0.59   0.55   0.53   0.02\n",
            "18732  UxaE                 PF16257.8          15     1.01    475   0.59   0.58   0.52   0.03\n",
            "18733  UXS1_N               PF11803.11          8     1.04     75   0.75   0.75   0.65   0.02\n",
            "18734  UxuA                 PF03786.16          8     0.87    352   0.59   0.56   0.52   0.01\n",
            "18735  V-ATPase_C           PF03223.18        113     1.99    370   0.59   0.58   0.52   0.03\n",
            "18736  V-ATPase_G           PF03179.18         25     1.48    105   0.59   0.62   0.55   0.10\n",
            "18737  V-ATPase_G_2         PF16999.8          12     1.35    104   0.59   0.65   0.55   0.11\n",
            "18738  V-ATPase_H_C         PF11698.11        114     1.97    117   0.59   0.56   0.51   0.04\n",
            "18739  V-ATPase_H_N         PF03224.17         31     3.60    315   0.59   0.60   0.50   0.04\n",
            "18740  V-set                PF07686.20         58     8.27    109   0.59   0.55   0.50   0.03\n",
            "18741  V-set_2              PF15910.8           7     0.75    113   0.59   0.54   0.53   0.03\n",
            "18742  V-set_CD47           PF08204.14         12     1.54    130   0.59   0.54   0.51   0.05\n",
            "18743  V-SNARE              PF05008.18         76    10.47     79   0.72   0.74   0.67   0.10\n",
            "18744  V-SNARE_C            PF12352.11         20     6.17     66   0.85   0.89   0.84   0.10\n",
            "18745  v110                 PF01639.20          4     0.65    104   0.59   0.46   0.51   0.06\n",
            "18746  V1R                  PF03402.17         15     1.37    292   0.59   0.57   0.54   0.06\n",
            "18747  V1R_C                PF08983.13          9     3.16     48   1.15   1.16   1.06   0.15\n",
            "18748  V4R                  PF02830.21         28     7.44     62   0.90   0.82   0.87   0.08\n",
            "18749  Vac14_Fab1_bd        PF12755.10         27     1.12     97   0.59   0.60   0.54   0.03\n",
            "18750  Vac14_Fig4_bd        PF11916.11         74     1.30    179   0.59   0.61   0.52   0.05\n",
            "18751  Vac17                PF17321.5          21     3.15    452   0.59   0.59   0.45   0.07\n",
            "18752  Vac7                 PF12751.10         31     1.14    379   0.59   0.59   0.49   0.03\n",
            "18753  VacA                 PF02691.18          2     0.45    981   0.59   0.60   0.51   0.02\n",
            "18754  VacA2                PF03077.17         13     2.16     60   0.93   0.95   0.89   0.10\n",
            "18755  Vac_Fusion           PF02346.19          7     1.92     56   0.99   1.01   0.96   0.09\n",
            "18756  Vac_ImportDeg        PF09783.12         79     1.33    162   0.59   0.53   0.47   0.03\n",
            "18757  VAD1-2               PF15310.9          28     1.93    249   0.59   0.59   0.52   0.06\n",
            "18758  Val_tRNA-synt_C      PF10458.12         21     3.34     66   0.85   0.93   0.83   0.11\n",
            "18759  Vanabin-2            PF11437.11          8     0.74     96   0.59   0.42   0.51   0.22\n",
            "18760  VanA_C               PF19112.3          44     6.45    195   0.59   0.55   0.51   0.02\n",
            "18761  Vanin_C              PF19018.3          74     4.01    167   0.59   0.52   0.47   0.04\n",
            "18762  VanW                 PF04294.16        397     2.56    130   0.59   0.58   0.53   0.02\n",
            "18763  VanY                 PF02557.20         43     1.88    133   0.59   0.54   0.51   0.04\n",
            "18764  VanZ                 PF04892.15        316     6.71    123   0.59   0.59   0.33   0.06\n",
            "18765  VAPB_antitox         PF02697.17         37     4.47     69   0.81   0.86   0.74   0.09\n",
            "18766  VapB_antitoxin       PF09957.12         27     8.03     47   1.17   1.27   1.12   0.12\n",
            "18767  VAR1                 PF05316.15         14     2.13    197   0.59   0.56   0.46   0.06\n",
            "18768  VARLMGL              PF14383.9          23     7.87     32   1.69   1.74   1.68   0.12\n",
            "18769  VAS1_LD              PF05827.15         26     3.03    140   0.59   0.53   0.46   0.05\n",
            "18770  Vasculin             PF15337.9           6     1.08     97   0.59   0.60   0.52   0.07\n",
            "18771  VasI                 PF11319.11         15     1.70    183   0.59   0.56   0.51   0.03\n",
            "18772  VasL                 PF12486.11          9     1.42    147   0.59   0.58   0.54   0.05\n",
            "18773  Vasohibin            PF14822.9          12     0.86    245   0.59   0.56   0.53   0.02\n",
            "18774  VASP_tetra           PF08776.14         18     4.06     36   1.51   1.59   1.50   0.27\n",
            "18775  VASt                 PF16016.8         315     7.61    151   0.59   0.54   0.41   0.03\n",
            "18776  VasX_N               PF20249.1          41     4.20    161   0.59   0.54   0.44   0.03\n",
            "18777  VATC                 PF18716.4          15     2.90     43   1.28   1.04   1.22   0.21\n",
            "18778  vATP-synt_AC39       PF01992.19        307    10.19    336   0.59   0.61   0.49   0.03\n",
            "18779  vATP-synt_E          PF01991.21         12     2.05    199   0.59   0.63   0.54   0.06\n",
            "18780  Vault                PF01505.21        124    15.03     43   1.28   1.27   1.17   0.03\n",
            "18781  Vault_2              PF17794.4           6     1.61     61   0.92   0.90   0.85   0.02\n",
            "18782  Vault_3              PF17795.4           5     1.18     62   0.90   0.93   0.86   0.03\n",
            "18783  Vault_4              PF17796.4           3     0.96     61   0.92   0.95   0.85   0.03\n",
            "18784  VbhA                 PF18495.4          40    17.52     47   1.17   1.24   1.17   0.13\n",
            "18785  VBS                  PF08913.13          4     0.70    125   0.59   0.62   0.53   0.04\n",
            "18786  VCH_CASS14           PF18315.4           2     0.66     97   0.59   0.61   0.53   0.02\n",
            "18787  VCIP135_N            PF19437.2          17     0.78    167   0.59   0.57   0.52   0.03\n",
            "18788  VCPO_N               PF17897.4           7     0.71    216   0.59   0.59   0.52   0.01\n",
            "18789  VCX_VCY              PF15231.9           2     0.58    129   0.59   0.62   0.51   0.08\n",
            "18790  VD10_N               PF08476.13          7     3.11     43   1.28   1.23   1.26   0.09\n",
            "18791  VDE                  PF07137.14         26     1.43    237   0.59   0.51   0.48   0.04\n",
            "18792  VEFS-Box             PF09733.12         26     1.56    137   0.59   0.54   0.53   0.04\n",
            "18793  VEG                  PF06257.14        100     2.36     73   0.77   0.78   0.74   0.05\n",
            "18794  VEGFR-2_TMD          PF17988.4          13     2.87     35   1.55   1.58   1.53   0.19\n",
            "18795  VEGF_C               PF14554.9           4     0.75     58   0.96   0.81   0.89   0.21\n",
            "18796  VEK-30               PF12107.11          8     8.00     17   2.61   2.65   2.60   0.77\n",
            "18797  Vel1p                PF10339.12          4     0.57    202   0.59   0.57   0.52   0.02\n",
            "18798  Velvet               PF11754.11        105     3.09    242   0.59   0.58   0.44   0.02\n",
            "18799  VERL                 PF11386.11          5     0.71     78   0.73   0.68   0.66   0.03\n",
            "18800  Vert_HS_TF           PF06546.14         14     1.40    275   0.59   0.61   0.51   0.04\n",
            "18801  Vert_IL3-reg_TF      PF06529.14         14     0.76    332   0.59   0.58   0.52   0.03\n",
            "18802  VESA1_N              PF12785.10         40     0.62    456   0.59   0.53   0.49   0.03\n",
            "18803  Vesiculo_matrix      PF06326.15          2     0.78    222   0.59   0.57   0.51   0.02\n",
            "18804  Vesi_VP2             PF05332.14          2     0.63    113   0.59   0.60   0.51   0.03\n",
            "18805  Vexin                PF15505.9           7     1.34     80   0.71   0.71   0.65   0.03\n",
            "18806  Vezatin              PF12632.10         29     4.82    282   0.59   0.60   0.50   0.04\n",
            "18807  VF530                PF09905.12        104     1.52     63   0.89   0.86   0.85   0.03\n",
            "18808  Vfa1                 PF08432.13         87     2.00    181   0.59   0.54   0.47   0.08\n",
            "18809  VGCC_alpha2          PF08473.14         13     0.70    431   0.59   0.55   0.52   0.02\n",
            "18810  VGCC_beta4Aa_N       PF12052.11         11     2.57     41   1.34   1.42   1.27   0.19\n",
            "18811  VGLL4                PF15245.9          12     1.08    222   0.59   0.56   0.50   0.04\n",
            "18812  VGPC1_C              PF16799.8           6     1.64     48   1.15   1.18   1.10   0.14\n",
            "18813  Vg_Tdu               PF07545.17         12     3.71     31   1.74   1.64   1.72   0.13\n",
            "18814  VHL                  PF01847.19          7     0.73     82   0.69   0.64   0.63   0.01\n",
            "18815  VHL_C                PF17211.6           6     1.37     49   1.13   1.16   1.09   0.10\n",
            "18816  VHP                  PF02209.22        186    13.40     36   1.51   1.44   1.51   0.19\n",
            "18817  Vhr1                 PF04001.16         20     0.76     91   0.63   0.62   0.57   0.06\n",
            "18818  VHS                  PF00790.22         15     2.35    141   0.59   0.59   0.54   0.03\n",
            "18819  Vicilin_N            PF04702.15          4     0.74    147   0.59   0.47   0.50   0.22\n",
            "18820  VID27                PF08553.13         50     0.81    356   0.59   0.59   0.53   0.02\n",
            "18821  VID27_N              PF17748.4          50     1.35    173   0.59   0.57   0.51   0.04\n",
            "18822  VID27_PH             PF17747.4          50     1.78    109   0.59   0.52   0.53   0.05\n",
            "18823  Vif                  PF00559.20          2     0.57    200   0.59   0.52   0.51   0.03\n",
            "18824  VIGSSK               PF14773.9          35     1.94     62   0.90   0.95   0.85   0.07\n",
            "18825  Vinculin             PF01044.22         10     1.29    969   0.59   0.61   0.52   0.04\n",
            "18826  Vint                 PF14623.9          34     1.27    167   0.59   0.56   0.49   0.02\n",
            "18827  VioE                 PF18234.4           6     0.57    182   0.59   0.55   0.53   0.01\n",
            "18828  Vip3A_N              PF12495.11         12     0.44    177   0.59   0.61   0.52   0.04\n",
            "18829  VipB                 PF05943.15         95     1.57    302   0.59   0.57   0.54   0.01\n",
            "18830  VipB_2               PF18945.3          95     1.29    112   0.59   0.57   0.55   0.02\n",
            "18831  Viral_alk_exo        PF01771.20          9     1.65    480   0.59   0.57   0.50   0.01\n",
            "18832  Viral_Beta_CD        PF04530.15          3     0.72    123   0.59   0.55   0.51   0.02\n",
            "18833  Viral_coat           PF00729.21         10     2.33    207   0.59   0.58   0.51   0.02\n",
            "18834  Viral_cys_rich       PF08008.15          3     0.83     83   0.69   0.64   0.62   0.02\n",
            "18835  Viral_DNA_bi         PF02236.20          3     1.19     79   0.72   0.71   0.65   0.02\n",
            "18836  Viral_DNA_bp         PF00747.20          8     1.50   1132   0.59   0.57   0.52   0.01\n",
            "18837  Viral_DNA_Zn_bi      PF03728.16          8     1.68     98   0.59   0.51   0.48   0.05\n",
            "18838  Viral_env_E26        PF11050.11          1     0.40    225   0.59   0.58   0.52   0.03\n",
            "18839  Viral_helicase1      PF01443.21         72     6.97    234   0.59   0.58   0.48   0.02\n",
            "18840  Viral_Hsp90          PF03225.17         17     3.70    512   0.59   0.57   0.52   0.03\n",
            "18841  Viral_NABP           PF05515.14          2     0.59    222   0.59   0.55   0.50   0.02\n",
            "18842  Viral_P18            PF04521.16          2     0.63    122   0.59   0.58   0.49   0.02\n",
            "18843  Viral_Rep            PF02407.19          4     0.93     82   0.69   0.64   0.61   0.03\n",
            "18844  VirArc_Nuclease      PF12187.11          3     0.68    190   0.59   0.57   0.51   0.03\n",
            "18845  VirB3                PF05101.16         59     5.60     84   0.68   0.64   0.60   0.07\n",
            "18846  VirB7                PF17413.5           8     2.86     35   1.55   1.49   1.53   0.09\n",
            "18847  VirB8                PF04335.16         42     6.31    212   0.59   0.59   0.52   0.03\n",
            "18848  VirC1                PF07015.14          4     0.64    231   0.59   0.61   0.53   0.02\n",
            "18849  VirC2                PF07181.14          2     0.49    202   0.59   0.61   0.52   0.02\n",
            "18850  VirD1                PF07328.14          2     0.46    142   0.59   0.63   0.52   0.02\n",
            "18851  VirDNA-topo-I_N      PF09266.13         10     1.67     58   0.96   0.96   0.92   0.04\n",
            "18852  VirE                 PF05272.14         60     2.51    217   0.59   0.57   0.52   0.03\n",
            "18853  VirE1                PF12189.11          2     0.71     63   0.89   0.89   0.84   0.04\n",
            "18854  VirE2                PF07229.15          1     0.39    556   0.59   0.58   0.52   0.01\n",
            "18855  VirE3                PF06661.14          1     0.40    312   0.59   0.60   0.52   0.03\n",
            "18856  VirE_N               PF08800.13         41     4.34    134   0.59   0.56   0.53   0.02\n",
            "18857  VirionAssem_T7       PF11653.11          1     0.43     99   0.59   0.65   0.52   0.06\n",
            "18858  VirJ                 PF06057.14         10     0.78    192   0.59   0.57   0.53   0.01\n",
            "18859  VirK                 PF06903.15         11     1.31     98   0.59   0.58   0.54   0.02\n",
            "18860  Virulence_fact       PF13769.9          35     2.28     81   0.70   0.70   0.65   0.04\n",
            "18861  Virulence_RhuM       PF13310.9         125     0.96    252   0.59   0.56   0.53   0.04\n",
            "18862  Virul_Fac            PF10139.12         13     1.28    877   0.59   0.59   0.53   0.02\n",
            "18863  Virul_fac_BrkB       PF03631.18         93     7.96    258   0.59   0.62   0.49   0.08\n",
            "18864  Vir_act_alpha_C      PF10400.12        191    11.30     77   0.73   0.74   0.63   0.07\n",
            "18865  VIR_N                PF15912.8           8     0.90    265   0.59   0.58   0.50   0.02\n",
            "18866  VIT                  PF08487.13        288     5.10    111   0.59   0.59   0.53   0.04\n",
            "18867  VIT1                 PF01988.22        474     4.21    215   0.59   0.64   0.48   0.03\n",
            "18868  VitD-bind_III        PF09164.13         11     1.30     65   0.86   0.78   0.82   0.08\n",
            "18869  Vitelline_membr      PF10542.12          7     1.75     38   1.43   1.31   1.37   0.28\n",
            "18870  Vitellogenin_N       PF01347.25         43     4.89    622   0.59   0.58   0.52   0.03\n",
            "18871  VitK2_biosynth       PF02621.17         84     4.28    261   0.59   0.60   0.51   0.01\n",
            "18872  VIT_2                PF13757.9           6     1.61     78   0.73   0.73   0.69   0.02\n",
            "18873  Vit_b-sht_shell      PF09175.13         30     2.15    167   0.59   0.56   0.55   0.02\n",
            "18874  Vit_open_b-sht       PF09172.14         98     8.83    303   0.59   0.59   0.49   0.02\n",
            "18875  VKG_Carbox           PF05090.17         49     1.69    435   0.59   0.53   0.52   0.04\n",
            "18876  VKOR                 PF07884.17        233     5.12    134   0.59   0.57   0.47   0.10\n",
            "18877  VlpA_repeat          PF04649.15          2     2.00     13   2.13   2.17   2.10   0.50\n",
            "18878  VLPT                 PF07122.14          2     1.88     30   1.79   1.82   1.76   0.22\n",
            "18879  Vma12                PF11712.11         91     2.83    139   0.59   0.60   0.44   0.03\n",
            "18880  VMA21                PF09446.13         46     5.15     64   0.88   0.88   0.82   0.16\n",
            "18881  VMAP-C               PF20028.2         154     9.53    228   0.59   0.57   0.32   0.04\n",
            "18882  VMAP-M0              PF19916.2          66     5.66    104   0.59   0.62   0.52   0.05\n",
            "18883  VMAP-M1              PF19963.2          30     5.29    115   0.59   0.58   0.47   0.05\n",
            "18884  VMAP-M12             PF20015.2           1     0.38    136   0.59   0.58   0.52   0.03\n",
            "18885  VMAP-M14             PF20018.2           3     0.59    132   0.59   0.58   0.52   0.04\n",
            "18886  VMAP-M15             PF20017.2           1     0.44     81   0.70   0.69   0.63   0.03\n",
            "18887  VMAP-M18             PF20021.2           1     0.39    109   0.59   0.59   0.52   0.02\n",
            "18888  VMAP-M19             PF20022.2           1     0.37    115   0.59   0.58   0.52   0.01\n",
            "18889  VMAP-M2              PF19965.2          28     3.76    217   0.59   0.64   0.50   0.04\n",
            "18890  VMAP-M20             PF20019.2           1     0.38    129   0.58   0.61   0.51   0.02\n",
            "18891  VMAP-M4              PF19966.2           3     0.79    107   0.59   0.62   0.53   0.02\n",
            "18892  VMAP-M5              PF19967.2           1     0.38    136   0.59   0.56   0.51   0.04\n",
            "18893  VMAP-M7              PF19968.2           2     0.61    160   0.59   0.60   0.52   0.02\n",
            "18894  VMAP-M8              PF19969.2           5     0.99     97   0.59   0.60   0.51   0.02\n",
            "18895  VMAP-M9              PF19970.2           1     0.39    135   0.59   0.59   0.51   0.04\n",
            "18896  Vmethyltransf        PF01660.20        122     7.58    346   0.59   0.54   0.45   0.02\n",
            "18897  Vmethyltransf_C      PF08456.13          3     0.89    230   0.59   0.59   0.51   0.02\n",
            "18898  VMR2                 PF17055.8           1     0.38    235   0.59   0.59   0.51   0.00\n",
            "18899  vMSA                 PF00695.22          2     0.60    400   0.59   0.57   0.50   0.02\n",
            "18900  Voldacs              PF03517.16         36     5.28    141   0.59   0.57   0.43   0.04\n",
            "18901  Voltage_CLC          PF00654.23        105     4.96    348   0.59   0.63   0.49   0.07\n",
            "18902  VOMI                 PF03762.20         36     1.51    167   0.59   0.50   0.49   0.05\n",
            "18903  VP1_VP3              PF06281.15          8     1.08     73   0.77   0.80   0.69   0.08\n",
            "18904  VP3-like             PF18965.3           3     0.89   1062   0.59   0.57   0.47   0.01\n",
            "18905  VP40                 PF07447.15          2     0.59    282   0.59   0.59   0.51   0.01\n",
            "18906  VP4_2                PF08935.13          2     0.78     84   0.68   0.68   0.61   0.07\n",
            "18907  VP4_haemagglut       PF00426.21          2     0.64    170   0.59   0.54   0.48   0.03\n",
            "18908  VP4_helical          PF17478.5           2     0.61    292   0.59   0.61   0.52   0.03\n",
            "18909  VP7                  PF00434.21          2     0.54    323   0.59   0.56   0.52   0.03\n",
            "18910  VP9                  PF09625.13          5     1.03     79   0.72   0.74   0.66   0.03\n",
            "18911  VPDSG-CTERM          PF18205.4          20    20.00     26   1.86   2.06   1.86   0.29\n",
            "18912  VPR                  PF00522.21         12     1.21     96   0.60   0.54   0.52   0.03\n",
            "18913  VPS11_C              PF12451.11         88    14.23     45   1.22   1.24   1.19   0.09\n",
            "18914  VPS13                PF16908.8         193     4.64    238   0.59   0.55   0.45   0.03\n",
            "18915  VPS13_C              PF16909.8          83     2.46    176   0.59   0.60   0.53   0.02\n",
            "18916  VPS13_mid_rpt        PF16910.8         121     3.37    236   0.59   0.60   0.53   0.03\n",
            "18917  Vps16_C              PF04840.15          4     1.05    320   0.59   0.60   0.53   0.03\n",
            "18918  Vps16_N              PF04841.16          5     1.30    410   0.59   0.58   0.50   0.02\n",
            "18919  Vps23_core           PF09454.13        101     6.83     64   0.88   0.90   0.84   0.05\n",
            "18920  Vps26                PF03643.18          8     0.84    275   0.59   0.58   0.50   0.02\n",
            "18921  VPS28                PF03997.15        100     1.88    190   0.59   0.59   0.51   0.04\n",
            "18922  Vps35                PF03635.20        113     2.07    728   0.59   0.58   0.49   0.04\n",
            "18923  Vps36-NZF-N          PF16988.8          29     2.76     65   0.86   0.73   0.81   0.11\n",
            "18924  Vps36_ESCRT-II       PF11605.11         19     2.40     92   0.62   0.62   0.56   0.03\n",
            "18925  VPS38                PF17649.4          11     1.38    424   0.59   0.58   0.53   0.04\n",
            "18926  Vps39_1              PF10366.12         20     1.75    108   0.59   0.59   0.52   0.04\n",
            "18927  Vps39_2              PF10367.12         18     1.94    109   0.59   0.56   0.55   0.05\n",
            "18928  Vps4_C               PF09336.13        121     4.29     61   0.92   0.88   0.88   0.03\n",
            "18929  Vps5                 PF09325.13         19     2.19    236   0.59   0.59   0.55   0.06\n",
            "18930  Vps51                PF08700.14         44     7.22     87   0.65   0.67   0.62   0.07\n",
            "18931  Vps52                PF04129.15          5     1.22    508   0.59   0.58   0.53   0.04\n",
            "18932  VPS53_C              PF16854.8          15     0.96    203   0.59   0.55   0.52   0.04\n",
            "18933  Vps53_N              PF04100.15          6     1.15    378   0.59   0.58   0.52   0.04\n",
            "18934  Vps54                PF07928.15        104     3.09    133   0.59   0.60   0.54   0.05\n",
            "18935  Vps54_N              PF10475.12          9     1.59    291   0.59   0.57   0.54   0.05\n",
            "18936  Vps55                PF04133.17        111     2.68    120   0.59   0.60   0.51   0.07\n",
            "18937  Vps62                PF06101.14         36     0.78    542   0.59   0.56   0.52   0.01\n",
            "18938  Vps8                 PF12816.10        137     2.84    199   0.59   0.58   0.50   0.03\n",
            "18939  VPS9                 PF02204.21         43     4.21    104   0.59   0.58   0.52   0.03\n",
            "18940  VpsR                 PF20161.2          30     2.10    121   0.59   0.57   0.54   0.02\n",
            "18941  Vpu                  PF00558.22         16     1.78     81   0.70   0.70   0.61   0.05\n",
            "18942  VP_N-CPKC            PF11475.11          1     1.00     32   1.51   1.36   1.47   0.16\n",
            "18943  VQ                   PF05678.17         92    52.77     28   1.92   1.92   1.92   0.13\n",
            "18944  VraX                 PF17412.5           2     0.87     55   1.01   0.97   0.96   0.05\n",
            "18945  VRP1                 PF03538.17          5     0.99    329   0.59   0.58   0.51   0.02\n",
            "18946  VRP3                 PF03536.18          4     0.59    240   0.59   0.57   0.53   0.01\n",
            "18947  VRR_NUC              PF08774.14         25     5.18    115   0.59   0.58   0.48   0.02\n",
            "18948  VSG_B                PF13206.9          55     5.87    353   0.59   0.56   0.49   0.06\n",
            "18949  VSP                  PF03302.16          6     0.57    397   0.59   0.52   0.51   0.09\n",
            "18950  Vsr                  PF03852.18          4     1.00     74   0.76   0.70   0.72   0.02\n",
            "18951  Vta1                 PF04652.19        133     4.16    145   0.59   0.58   0.50   0.05\n",
            "18952  Vta1_C               PF18097.4          13     6.33     38   1.44   1.50   1.43   0.14\n",
            "18953  VTC                  PF09359.13         41     3.46    275   0.59   0.56   0.46   0.04\n",
            "18954  Vut_1                PF02592.18        317     4.36    151   0.59   0.59   0.48   0.09\n",
            "18955  VWA                  PF00092.31        127     8.44    175   0.59   0.61   0.47   0.03\n",
            "18956  vWA-TerF-like        PF10138.12         18     1.12    200   0.59   0.58   0.52   0.02\n",
            "18957  Vwaint               PF14624.9          52     3.51     70   0.80   0.75   0.70   0.04\n",
            "18958  VWA_2                PF13519.9         174     9.11    108   0.59   0.61   0.46   0.03\n",
            "18959  VWA_3                PF13768.9          11     4.39    155   0.59   0.61   0.50   0.02\n",
            "18960  VWA_3_C              PF18571.4          12     2.72     47   1.17   1.27   1.15   0.10\n",
            "18961  VWA_CoxE             PF05762.17         12     2.50    221   0.59   0.61   0.51   0.03\n",
            "18962  VWA_N                PF08399.14         42     1.97    124   0.59   0.53   0.49   0.04\n",
            "18963  VWA_N2               PF16164.8           4     0.69     79   0.72   0.64   0.64   0.06\n",
            "18964  VWC                  PF00093.21         19     1.85     57   0.98   0.69   0.83   0.36\n",
            "18965  VWD                  PF00094.28         40     5.15    157   0.59   0.54   0.48   0.03\n",
            "18966  vWF_A                PF12450.11         87     1.35     94   0.61   0.58   0.55   0.04\n",
            "18967  V_ATPase_I           PF01496.22        229     1.69    816   0.59   0.56   0.51   0.03\n",
            "18968  V_ATPase_I_N         PF18670.4          13     1.23     90   0.63   0.71   0.59   0.04\n",
            "18969  V_ATPase_prox        PF18274.4           1     0.75     52   1.07   1.12   1.01   0.06\n",
            "18970  V_cholerae_RfbT      PF05575.14          2     0.38    286   0.59   0.56   0.51   0.02\n",
            "18971  W2                   PF02020.21        350     6.22     79   0.72   0.71   0.60   0.07\n",
            "18972  WaaY                 PF06176.14          3     0.48    229   0.59   0.58   0.52   0.03\n",
            "18973  WAC_Acf1_DNA_bd      PF10537.12         92     2.28    101   0.59   0.54   0.53   0.05\n",
            "18974  Waikav_capsid_1      PF12264.11          3     0.57    197   0.59   0.57   0.53   0.02\n",
            "18975  WAK                  PF08488.14         14     0.90    103   0.59   0.50   0.50   0.06\n",
            "18976  WAK_assoc            PF14380.9          48     2.28    105   0.59   0.48   0.48   0.10\n",
            "18977  WAP                  PF00095.24        419     6.70     43   1.28   0.97   1.10   0.46\n",
            "18978  Wap1                 PF16997.8          13     1.98    382   0.59   0.59   0.52   0.05\n",
            "18979  WAPL                 PF07814.16         60     5.17    355   0.59   0.62   0.49   0.04\n",
            "18980  WASH-4_N             PF14745.9          17     2.37    574   0.59   0.58   0.53   0.04\n",
            "18981  WASH-7_C             PF14746.9          30     1.56    175   0.59   0.57   0.53   0.04\n",
            "18982  WASH-7_mid           PF14744.9          33     1.83    346   0.59   0.57   0.52   0.04\n",
            "18983  WASH_WAHD            PF11945.11          9     1.37    292   0.59   0.59   0.51   0.03\n",
            "18984  WavE                 PF07507.14          5     1.05    305   0.59   0.56   0.51   0.01\n",
            "18985  Wax2_C               PF12076.11         42     1.16    164   0.59   0.55   0.53   0.02\n",
            "18986  WBP-1                PF11669.11         11     1.18    105   0.59   0.48   0.45   0.06\n",
            "18987  Wbp11                PF09429.13         79     4.86     78   0.73   0.76   0.68   0.17\n",
            "18988  WbqC                 PF08889.14         74     2.90    218   0.59   0.55   0.52   0.02\n",
            "18989  WBS28                PF15164.9          10     0.70    267   0.59   0.60   0.53   0.04\n",
            "18990  WBS_methylT          PF12589.11         45     2.29     82   0.69   0.68   0.57   0.11\n",
            "18991  WC-rich              PF20053.2          51     1.50    111   0.59   0.49   0.51   0.06\n",
            "18992  WcbI                 PF18588.4          67     4.68    205   0.59   0.57   0.51   0.01\n",
            "18993  WCCH                 PF03716.17         30    30.00     25   2.03   1.63   2.01   0.46\n",
            "18994  WCOB                 PF19913.2          44     2.78    122   0.59   0.57   0.50   0.03\n",
            "18995  WCOR413              PF05562.14         18     0.85    183   0.59   0.60   0.52   0.04\n",
            "18996  WD-like_fungi        PF20493.1          23     1.68    199   0.59   0.53   0.52   0.06\n",
            "18997  WD40                 PF00400.35       1465  1465.00     38   1.10   1.00   0.84   0.09\n",
            "18998  WD40_2               PF19056.3          24     1.22    489   0.59   0.59   0.51   0.02\n",
            "18999  WD40_3               PF15911.8          39     5.46     57   0.98   0.90   0.95   0.03\n",
            "19000  WD40_4               PF16300.8         175     9.77     44   1.25   1.19   1.22   0.05\n",
            "19001  WD40_alt             PF14077.9           7     1.57     48   1.15   1.18   1.11   0.06\n",
            "19002  WD40_like            PF17005.8           6     1.16    301   0.59   0.59   0.53   0.04\n",
            "19003  WDCP                 PF15390.9          27     1.11    688   0.59   0.59   0.53   0.02\n",
            "19004  WEF-hand             PF19914.2          45     3.11     88   0.65   0.66   0.54   0.07\n",
            "19005  WEMBL                PF05701.14         25     1.30    562   0.59   0.64   0.54   0.08\n",
            "19006  WES_acyltransf       PF03007.19         15     2.56    264   0.59   0.59   0.53   0.02\n",
            "19007  WGG                  PF10273.12        143     3.57     81   0.70   0.69   0.62   0.04\n",
            "19008  WGR                  PF05406.18        106     5.07     80   0.71   0.66   0.65   0.06\n",
            "19009  WG_beta_rep          PF14903.9         208   208.00     35   1.53   1.44   1.48   0.16\n",
            "19010  WH1                  PF00568.26          8     1.54    111   0.59   0.54   0.51   0.03\n",
            "19011  WH2                  PF02205.23         43    43.00     30   1.54   1.66   1.52   0.16\n",
            "19012  WHAMM-JMY_N          PF15920.8          13     1.60     50   1.11   1.03   1.05   0.06\n",
            "19013  Wheel                PF18972.3          25     1.33    115   0.59   0.57   0.50   0.03\n",
            "19014  WHEP-TRS             PF00458.23        158     6.90     53   1.05   1.15   1.00   0.13\n",
            "19015  WHH                  PF14414.9          21     8.13     43   1.28   1.13   1.16   0.14\n",
            "19016  Whi5                 PF08528.14         69    69.00     25   2.06   2.08   2.06   0.20\n",
            "19017  WhiA_N               PF10298.12         61     5.27     86   0.66   0.70   0.62   0.04\n",
            "19018  Whib                 PF02467.19         65     2.23     64   0.87   0.77   0.77   0.10\n",
            "19019  WHIM1                PF15612.9         105   105.00     46   1.11   1.12   1.06   0.05\n",
            "19020  Whirly               PF08536.14         22     1.23    137   0.59   0.59   0.52   0.02\n",
            "19021  WI12                 PF07107.14          4     0.69    109   0.59   0.58   0.51   0.03\n",
            "19022  WIF                  PF02019.21         25     1.64    132   0.59   0.56   0.52   0.02\n",
            "19023  WIYLD                PF10440.12         27     3.35     61   0.92   0.91   0.87   0.04\n",
            "19024  WKF                  PF10180.12        122     3.83     63   0.89   0.81   0.82   0.08\n",
            "19025  WLM                  PF08325.13         18     1.72    191   0.59   0.58   0.50   0.03\n",
            "19026  WND                  PF07861.14          3     0.42    239   0.59   0.58   0.52   0.04\n",
            "19027  wnt                  PF00110.22        173     1.35    309   0.59   0.49   0.50   0.08\n",
            "19028  Wound_ind            PF08186.14          4     1.52     44   1.25   1.24   1.12   0.08\n",
            "19029  WPP                  PF13943.9          24     0.94     99   0.59   0.61   0.52   0.03\n",
            "19030  WRC                  PF08879.13         28     3.50     43   1.28   1.06   1.26   0.25\n",
            "19031  WRKY                 PF03106.18        225     2.56     60   0.93   0.75   0.84   0.14\n",
            "19032  WRNPLPNID            PF15017.9          32     4.54     69   0.82   0.79   0.66   0.10\n",
            "19033  WRW                  PF10206.12         20     1.12    102   0.59   0.51   0.54   0.04\n",
            "19034  WSC                  PF01822.22        102     3.22     82   0.69   0.55   0.59   0.15\n",
            "19035  WSD                  PF15613.9          64     2.77     98   0.59   0.57   0.46   0.04\n",
            "19036  WSK                  PF03832.16         14    10.72     29   1.85   1.87   1.85   0.27\n",
            "19037  WSLR                 PF20023.2          60    60.00     39   1.38   1.38   1.34   0.09\n",
            "19038  WSN                  PF02206.21        165    10.05     66   0.85   0.92   0.84   0.07\n",
            "19039  WSS_VP               PF12175.11          3     0.88    201   0.59   0.61   0.49   0.03\n",
            "19040  WS_DGAT_C            PF06974.16        124     4.41    145   0.59   0.60   0.54   0.02\n",
            "19041  WT1                  PF02165.18         11     0.63    290   0.59   0.55   0.50   0.02\n",
            "19042  Wtap                 PF17098.8          35     3.44    155   0.59   0.62   0.52   0.07\n",
            "19043  WTF                  PF03303.16          7     0.91    221   0.59   0.57   0.50   0.03\n",
            "19044  WTX                  PF09422.13         14     1.31    479   0.59   0.59   0.50   0.04\n",
            "19045  WVELL                PF14043.9          16     1.11     74   0.76   0.73   0.71   0.04\n",
            "19046  WW                   PF00397.29        448   128.80     31   1.74   1.50   1.72   0.27\n",
            "19047  WWamide              PF08258.14          2     2.00      7   2.66   2.05   2.63   1.14\n",
            "19048  WWE                  PF02825.23        199     7.29     67   0.84   0.72   0.69   0.09\n",
            "19049  WW_1                 PF18507.4          10     2.47     27   1.98   1.85   1.96   0.19\n",
            "19050  WW_FCH_linker        PF16623.8           8     0.65     92   0.62   0.58   0.55   0.04\n",
            "19051  WW_like              PF17890.4           3     0.99     53   1.05   0.98   1.00   0.08\n",
            "19052  Wx5_PLAF3D7          PF09688.13         11     1.49    144   0.59   0.51   0.54   0.08\n",
            "19053  WXG100               PF06013.15         85    10.96     86   0.66   0.66   0.64   0.08\n",
            "19054  WxL                  PF13731.9          56     3.00    213   0.59   0.59   0.43   0.04\n",
            "19055  WXXGXW               PF12779.10         61    52.52     26   2.06   1.64   2.04   0.73\n",
            "19056  WYL                  PF13280.9         384    10.34    172   0.59   0.58   0.49   0.02\n",
            "19057  WYL_2                PF10902.11         41     4.34     74   0.76   0.71   0.71   0.04\n",
            "19058  WYL_3                PF18488.4          14     2.81     63   0.89   0.86   0.85   0.09\n",
            "19059  Wyosine_form         PF08608.15         70     6.40     63   0.89   0.86   0.81   0.02\n",
            "19060  Wza_C                PF18412.4          38     8.53     30   1.80   1.79   1.79   0.17\n",
            "19061  Wzt_C                PF14524.9         157    10.74    141   0.59   0.58   0.45   0.02\n",
            "19062  WzyE                 PF06899.14          4     0.46    447   0.59   0.59   0.52   0.05\n",
            "19063  Wzy_C                PF04932.18         75     8.53    154   0.59   0.60   0.48   0.05\n",
            "19064  Wzy_C_2              PF11846.11         25     4.18    188   0.59   0.58   0.54   0.03\n",
            "19065  Wzz                  PF02706.18         37     5.85     92   0.62   0.66   0.53   0.05\n",
            "19066  W_rich_C             PF07483.14         16     1.58    105   0.59   0.55   0.52   0.05\n",
            "19067  X                    PF00739.22          5     0.60    142   0.59   0.58   0.52   0.02\n",
            "19068  X8                   PF07983.16        186     2.95     76   0.74   0.63   0.66   0.09\n",
            "19069  XAF1_C               PF18608.4           8     1.65     54   1.03   0.94   0.99   0.08\n",
            "19070  Xan_ur_permease      PF00860.23         20     3.75    389   0.59   0.62   0.52   0.07\n",
            "19071  XAP5                 PF04921.17         90     1.41    254   0.59   0.56   0.47   0.04\n",
            "19072  XdhC_C               PF13478.9         566     3.65    125   0.59   0.63   0.47   0.02\n",
            "19073  XdhC_CoxI            PF02625.19        450    10.38     68   0.83   0.89   0.73   0.06\n",
            "19074  XendoU               PF09412.13         81     2.21    263   0.59   0.55   0.51   0.03\n",
            "19075  XET_C                PF06955.15        180     7.13     47   1.17   0.98   1.09   0.14\n",
            "19076  XFP                  PF03894.18        134     1.32    176   0.59   0.56   0.52   0.02\n",
            "19077  XFP_C                PF09363.13        236     1.42    202   0.59   0.57   0.53   0.02\n",
            "19078  XFP_N                PF09364.13          5     0.63    364   0.59   0.57   0.52   0.00\n",
            "19079  XG_FTase             PF03254.16          3     0.61    475   0.59   0.56   0.52   0.01\n",
            "19080  XH                   PF03469.17         63     1.60    131   0.59   0.56   0.53   0.04\n",
            "19081  XhlA                 PF10779.12         32     7.40     67   0.84   0.86   0.80   0.09\n",
            "19082  XhoI                 PF04555.16          5     0.60    191   0.59   0.58   0.52   0.01\n",
            "19083  Xin                  PF08043.15         56    56.00     16   2.70   2.55   2.70   0.23\n",
            "19084  XisH                 PF08814.13         69     1.21    134   0.59   0.59   0.54   0.03\n",
            "19085  XisI                 PF08869.14        106     1.47    110   0.59   0.56   0.53   0.04\n",
            "19086  XK-related           PF09815.12        116     4.95    339   0.59   0.54   0.50   0.08\n",
            "19087  XkdW                 PF09636.13          4     0.66    108   0.59   0.58   0.52   0.02\n",
            "19088  XLF                  PF09302.14         76     3.96    180   0.59   0.59   0.48   0.04\n",
            "19089  Xlink                PF00193.20        173     1.94     93   0.61   0.53   0.55   0.05\n",
            "19090  Xol-1_GHMP-like      PF09109.13          2     0.64    189   0.59   0.59   0.51   0.02\n",
            "19091  Xol-1_N              PF09108.13          2     0.56    160   0.59   0.59   0.52   0.02\n",
            "19092  XOO_2897-deam        PF14440.9           7     1.80    101   0.59   0.57   0.50   0.03\n",
            "19093  XPA_C                PF05181.15         54     3.05     51   1.09   1.04   1.04   0.08\n",
            "19094  XPA_N                PF01286.21         29     7.16     32   1.69   1.42   1.69   0.36\n",
            "19095  XPB_DRD              PF18458.4          93     6.58     57   0.98   0.99   0.92   0.05\n",
            "19096  XPC-binding          PF09280.14         88     5.76     57   0.98   0.99   0.97   0.13\n",
            "19097  XPG_I                PF00867.21        105     3.42     91   0.63   0.63   0.54   0.03\n",
            "19098  XPG_I_2              PF12813.10         19     2.73    249   0.59   0.59   0.49   0.02\n",
            "19099  XPG_N                PF00752.20         10     1.66    101   0.59   0.58   0.52   0.02\n",
            "19100  Xpo1                 PF08389.15         52     5.25    149   0.59   0.58   0.50   0.04\n",
            "19101  XRCC1_N              PF01834.19         33     1.47    148   0.59   0.56   0.53   0.03\n",
            "19102  XRCC4                PF06632.15         10     1.01    337   0.59   0.59   0.53   0.04\n",
            "19103  Xre-like-HTH         PF20432.1         146    81.73     63   0.89   1.00   0.82   0.07\n",
            "19104  Xre_MbcA_ParS_C      PF09722.13        165    20.18     51   1.09   1.12   1.01   0.05\n",
            "19105  XRN1_D1              PF18332.4          85     1.37    192   0.59   0.58   0.53   0.02\n",
            "19106  XRN1_D2_D3           PF18334.4          96     1.56     87   0.65   0.66   0.61   0.02\n",
            "19107  Xrn1_D3              PF18194.4         114     3.67     71   0.79   0.83   0.76   0.06\n",
            "19108  XRN1_DBM             PF18245.4           1     1.00     26   1.60   1.46   1.55   0.30\n",
            "19109  XRN_M                PF17846.4          10     1.10    443   0.59   0.56   0.52   0.02\n",
            "19110  XRN_N                PF03159.21        133     1.62    251   0.59   0.57   0.49   0.02\n",
            "19111  xRRM                 PF19977.2          39     1.31    120   0.59   0.56   0.52   0.03\n",
            "19112  XS                   PF03468.17         71     1.96    113   0.59   0.53   0.51   0.03\n",
            "19113  XTBD                 PF11952.11         38     2.73     85   0.67   0.62   0.59   0.04\n",
            "19114  Xylo_C               PF12529.11         29     2.12    182   0.59   0.55   0.52   0.02\n",
            "19115  XylR_N               PF06505.14         55     2.07    101   0.59   0.61   0.54   0.02\n",
            "19116  XYPPX                PF02162.20          7     7.00     15   3.41   3.23   3.29   1.80\n",
            "19117  Y1_Tnp               PF01797.19         88     3.32    122   0.59   0.51   0.51   0.05\n",
            "19118  Y2_Tnp               PF04986.16         28     2.27    183   0.59   0.55   0.51   0.03\n",
            "19119  YaaC                 PF14175.9          20     2.69    319   0.59   0.56   0.50   0.03\n",
            "19120  YabA                 PF06156.16         53     1.18    102   0.59   0.54   0.51   0.06\n",
            "19121  YABBY                PF04690.16         57     0.88    166   0.59   0.54   0.48   0.03\n",
            "19122  YabP                 PF07873.14        376     7.21     64   0.88   0.90   0.84   0.07\n",
            "19123  YAcAr                PF10686.12         59     5.01     67   0.84   0.82   0.76   0.04\n",
            "19124  YacG                 PF03884.17         38     2.00     52   1.07   0.89   1.02   0.13\n",
            "19125  YadA_anchor          PF03895.18         50     9.02     60   0.93   0.94   0.88   0.12\n",
            "19126  YadA_head            PF05658.17        337   337.00     27   1.83   2.00   1.81   0.47\n",
            "19127  YadA_stalk           PF05662.17        101    24.62     43   1.28   1.35   1.24   0.21\n",
            "19128  Yae1_N               PF09811.12         60    34.20     39   1.40   1.46   1.40   0.25\n",
            "19129  YaeQ                 PF07152.15        110     1.63    173   0.59   0.59   0.55   0.02\n",
            "19130  YAF2_RYBP            PF17219.6          14     4.22     33   1.64   1.67   1.61   0.15\n",
            "19131  YafO_toxin           PF13957.9           9     1.34    101   0.59   0.54   0.51   0.03\n",
            "19132  YafQ_toxin           PF15738.8          82     2.14     88   0.65   0.62   0.56   0.04\n",
            "19133  YaiA                 PF16362.8           2     0.75     63   0.89   0.86   0.83   0.05\n",
            "19134  YaiO                 PF19413.2          89     5.18    173   0.59   0.56   0.54   0.04\n",
            "19135  YajC                 PF02699.18        680     4.90     80   0.71   0.73   0.61   0.06\n",
            "19136  YARHG                PF13308.9          32     5.34     84   0.68   0.64   0.61   0.05\n",
            "19137  YbaB_DNA_bd          PF02575.19        649     6.30     90   0.63   0.68   0.57   0.08\n",
            "19138  YbaJ                 PF10757.12          4     0.57    118   0.59   0.56   0.52   0.03\n",
            "19139  YbbR                 PF07949.15        185    24.29     82   0.69   0.77   0.59   0.09\n",
            "19140  YbcO                 PF07102.15         10     1.15     92   0.62   0.57   0.55   0.04\n",
            "19141  YBD                  PF17725.4           5     0.77    201   0.59   0.57   0.50   0.03\n",
            "19142  YbeY                 PF02130.20        208     2.71    131   0.59   0.57   0.46   0.04\n",
            "19143  YbfN                 PF13982.9           4     0.61     88   0.65   0.60   0.59   0.05\n",
            "19144  YbgS                 PF13985.9           6     0.69    120   0.59   0.60   0.51   0.07\n",
            "19145  YbgT_YccB            PF08173.14         29     7.57     27   1.98   1.83   1.98   0.43\n",
            "19146  YbhQ                 PF11076.11          4     0.57    132   0.59   0.61   0.52   0.05\n",
            "19147  YbjM                 PF11045.11          8     0.93    117   0.59   0.59   0.53   0.06\n",
            "19148  YbjN                 PF10722.12        101    15.67    127   0.59   0.58   0.53   0.02\n",
            "19149  YbjQ_1               PF01906.20        101     2.10    104   0.59   0.62   0.54   0.05\n",
            "19150  YcaO                 PF02624.19        228     7.55    322   0.59   0.60   0.44   0.01\n",
            "19151  YcaO_C               PF18381.4          61     1.44    172   0.59   0.59   0.54   0.03\n",
            "19152  YcbB                 PF08664.13         34     1.65    136   0.59   0.59   0.52   0.04\n",
            "19153  YccF                 PF03733.16        396     8.17     51   1.09   1.03   1.04   0.24\n",
            "19154  YccJ                 PF13993.9           4     0.84     67   0.84   0.81   0.78   0.07\n",
            "19155  YccV-like            PF08755.14        134     2.98    101   0.59   0.53   0.46   0.03\n",
            "19156  YceD                 PF02620.20         79     6.44    121   0.59   0.58   0.50   0.04\n",
            "19157  YceG                 PF02618.19        761     3.38    276   0.59   0.59   0.41   0.02\n",
            "19158  YceG_bac             PF14266.9          24     1.95    483   0.59   0.58   0.52   0.03\n",
            "19159  YceI                 PF04264.16        729     6.55    146   0.59   0.60   0.30   0.03\n",
            "19160  Ycf1                 PF05758.15          5     0.54    945   0.59   0.57   0.51   0.02\n",
            "19161  Ycf15                PF10705.12          3     0.44     86   0.66   0.61   0.59   0.03\n",
            "19162  Ycf34                PF10718.12         25     0.92     77   0.73   0.59   0.66   0.07\n",
            "19163  Ycf4                 PF02392.19         51     0.94    178   0.59   0.61   0.53   0.02\n",
            "19164  Ycf54                PF10674.12         35     1.12     92   0.62   0.59   0.56   0.04\n",
            "19165  Ycf66_N              PF07444.14         37     2.37     76   0.74   0.78   0.71   0.08\n",
            "19166  Ycf70                PF17382.5           2     0.41     89   0.64   0.62   0.57   0.03\n",
            "19167  Ycf9                 PF01737.20         34     2.53     58   0.96   1.01   0.93   0.16\n",
            "19168  YCF90                PF17088.8           1     0.36    388   0.59   0.56   0.52   0.03\n",
            "19169  YcgL                 PF05166.16         63     1.83     73   0.77   0.75   0.73   0.03\n",
            "19170  YchF-GTPase_C        PF06071.16        353     1.68     84   0.68   0.64   0.62   0.04\n",
            "19171  YCII                 PF03795.17         26     4.28     95   0.60   0.61   0.51   0.02\n",
            "19172  YcxB                 PF14317.9         141    61.80     61   0.92   0.87   0.84   0.05\n",
            "19173  YdaS_antitoxin       PF15943.8          21     4.20     65   0.86   0.87   0.82   0.03\n",
            "19174  YdaT_toxin           PF06254.14          3     0.82     88   0.65   0.67   0.56   0.03\n",
            "19175  Ydc2-catalyt         PF09159.13         47     3.71    280   0.59   0.58   0.47   0.03\n",
            "19176  YdfZ                 PF14001.9           7     1.47     64   0.88   0.89   0.84   0.03\n",
            "19177  YDG                  PF18657.4          56     6.68     85   0.67   0.70   0.54   0.09\n",
            "19178  YdgH_BhsA-like       PF07338.16         82     8.21     55   1.01   1.03   0.94   0.06\n",
            "19179  ydhR                 PF08803.14         30     1.65     96   0.60   0.58   0.54   0.02\n",
            "19180  YdiH                 PF15930.8           2     0.72     62   0.90   0.93   0.84   0.04\n",
            "19181  YdjC                 PF04794.15        169     5.87    253   0.59   0.58   0.33   0.02\n",
            "19182  YdjM                 PF04307.17         91     5.43    173   0.59   0.58   0.43   0.05\n",
            "19183  YdjO                 PF14169.9          22     1.86     59   0.95   0.77   0.91   0.12\n",
            "19184  Ydr279_N             PF17745.4         113     5.94     79   0.72   0.72   0.54   0.03\n",
            "19185  Yeast-kill-tox       PF09207.14          2     0.39     87   0.65   0.55   0.58   0.06\n",
            "19186  Yeast_MT             PF11403.11          2     0.79     39   1.40   1.13   1.35   0.44\n",
            "19187  YEATS                PF03366.19        218     2.37     82   0.69   0.63   0.60   0.04\n",
            "19188  YebF                 PF13995.9           8     1.06     89   0.64   0.59   0.59   0.04\n",
            "19189  YebG                 PF07130.15         33     2.02     74   0.76   0.79   0.72   0.06\n",
            "19190  YebO                 PF13974.9          12     1.23     78   0.72   0.75   0.66   0.06\n",
            "19191  YecM                 PF06185.15         33     1.47    180   0.59   0.58   0.52   0.03\n",
            "19192  YecR                 PF13992.9           8     1.52     73   0.77   0.68   0.72   0.08\n",
            "19193  YedD                 PF13987.9           5     0.57    106   0.59   0.58   0.52   0.02\n",
            "19194  YegS_C               PF19279.2          38     1.38    158   0.59   0.62   0.52   0.02\n",
            "19195  YejG                 PF13989.9          10     0.73    106   0.59   0.60   0.53   0.02\n",
            "19196  YesK                 PF14150.9           8     2.10     81   0.70   0.73   0.65   0.13\n",
            "19197  YetA                 PF19501.2          50     1.28    867   0.59   0.55   0.52   0.02\n",
            "19198  YfaZ                 PF07437.14          3     0.76    180   0.59   0.61   0.53   0.02\n",
            "19199  YfbR-like            PF12917.10         83     1.69    182   0.59   0.57   0.53   0.03\n",
            "19200  YfbU                 PF03887.17         16     1.44    166   0.59   0.55   0.51   0.04\n",
            "19201  YfcL                 PF08891.14         26     1.41     83   0.68   0.70   0.63   0.04\n",
            "19202  YfdX                 PF10938.11         34     4.48    149   0.59   0.65   0.52   0.07\n",
            "19203  YfhD                 PF14151.9          13     3.13     59   0.95   0.94   0.88   0.15\n",
            "19204  YfhE                 PF14152.9          10     4.12     36   1.51   1.51   1.49   0.24\n",
            "19205  YfhO                 PF09586.13         34     3.95    840   0.59   0.57   0.51   0.04\n",
            "19206  YfiO                 PF13525.9          12     1.91    203   0.59   0.57   0.54   0.05\n",
            "19207  YfjS_YafY            PF20494.1           3     0.51    113   0.59   0.56   0.53   0.02\n",
            "19208  YfkB                 PF08756.13         27     0.65    151   0.59   0.55   0.53   0.02\n",
            "19209  YfkD                 PF14167.9          13     0.73    232   0.59   0.58   0.53   0.02\n",
            "19210  YflT                 PF11181.11         80     7.50    100   0.59   0.62   0.54   0.04\n",
            "19211  YfmQ                 PF10787.12          5     0.60    142   0.59   0.58   0.52   0.03\n",
            "19212  YfzA                 PF14118.9           3     0.62     90   0.63   0.60   0.56   0.05\n",
            "19213  YgaB                 PF14182.9           7     1.10     77   0.74   0.73   0.69   0.09\n",
            "19214  YgbA_NO              PF11756.11         43     0.82    107   0.59   0.50   0.52   0.06\n",
            "19215  YgbB                 PF02542.19        613     2.17    155   0.59   0.62   0.51   0.02\n",
            "19216  YggL_50S_bp          PF04320.17         28     1.51    102   0.59   0.58   0.52   0.03\n",
            "19217  YGGT                 PF02325.20        669    11.59     69   0.82   0.84   0.63   0.13\n",
            "19218  YgjP-like            PF01863.20         59     4.04    207   0.59   0.55   0.54   0.04\n",
            "19219  YhcG_C               PF06250.14         32     1.09    155   0.59   0.59   0.53   0.03\n",
            "19220  YhdB                 PF14148.9           9     0.91     71   0.79   0.74   0.74   0.08\n",
            "19221  YhdX                 PF17444.5           3     1.48     33   1.64   1.68   1.60   0.17\n",
            "19222  YhfC                 PF10086.12         10     1.67    223   0.59   0.62   0.53   0.05\n",
            "19223  YhfH                 PF14149.9          23     3.32     37   1.47   1.23   1.45   0.28\n",
            "19224  YhfT                 PF10797.12         14     0.91    422   0.59   0.64   0.53   0.05\n",
            "19225  YhfZ_C               PF14503.9          18     1.99    236   0.59   0.59   0.54   0.02\n",
            "19226  YhhN                 PF07947.17         62     4.03    184   0.59   0.60   0.50   0.09\n",
            "19227  YHS                  PF04945.16         11     5.40     46   1.20   1.06   1.13   0.11\n",
            "19228  YHYH                 PF14240.9          14     1.49    198   0.59   0.55   0.46   0.03\n",
            "19229  YhzD                 PF14120.9          15     1.86     61   0.92   0.92   0.87   0.05\n",
            "19230  YiaAB                PF05360.17        116     8.06     53   1.05   1.04   1.00   0.14\n",
            "19231  YibE_F               PF07907.14        268     4.12    241   0.59   0.64   0.55   0.09\n",
            "19232  YicC_N               PF03755.16        597     3.57    152   0.59   0.61   0.45   0.03\n",
            "19233  YidB                 PF20159.2          12     1.05    143   0.59   0.64   0.48   0.04\n",
            "19234  YidC_periplas        PF14849.9         149     4.53    283   0.59   0.57   0.47   0.02\n",
            "19235  YidD                 PF01809.21        649     2.27     67   0.84   0.74   0.76   0.06\n",
            "19236  YIEGIA               PF14045.9          36     1.21    282   0.59   0.61   0.54   0.02\n",
            "19237  YIF1                 PF03878.18        103     1.83    244   0.59   0.56   0.50   0.04\n",
            "19238  YihI                 PF04220.15         26     1.36    155   0.59   0.62   0.51   0.08\n",
            "19239  YiiD_C               PF09500.13         19     1.46    144   0.59   0.58   0.54   0.01\n",
            "19240  Yip1                 PF04893.20        213    20.70    173   0.59   0.62   0.42   0.13\n",
            "19241  Yippee-Mis18         PF03226.17         44     3.04    105   0.59   0.51   0.49   0.07\n",
            "19242  YitT_membrane        PF02588.18         54     3.30    206   0.59   0.62   0.55   0.08\n",
            "19243  YjbE                 PF11106.11          3     0.93     79   0.72   0.82   0.65   0.15\n",
            "19244  YjbF                 PF11102.11         21     3.22    190   0.59   0.58   0.51   0.02\n",
            "19245  YjbH                 PF06082.14         40     2.14    660   0.59   0.58   0.52   0.01\n",
            "19246  YjbR                 PF04237.16        492     7.62     90   0.63   0.60   0.41   0.02\n",
            "19247  YjbT                 PF17089.8           1     0.41     92   0.62   0.62   0.55   0.01\n",
            "19248  YjcB                 PF15940.8           5     0.70     90   0.64   0.62   0.56   0.05\n",
            "19249  YjcQ                 PF09639.13         10     1.60     96   0.60   0.60   0.54   0.04\n",
            "19250  YjcZ                 PF13990.9           3     0.79    272   0.59   0.59   0.53   0.02\n",
            "19251  YjcZ_2               PF09680.13         24     7.39     24   2.22   2.31   2.21   0.60\n",
            "19252  YjdM                 PF03831.17        152     1.99     69   0.82   0.86   0.75   0.07\n",
            "19253  YjdM_Zn_Ribbon       PF08274.15         27     8.60     30   1.80   1.52   1.66   0.36\n",
            "19254  YjeF_N               PF03853.18         49     4.27    171   0.59   0.62   0.48   0.02\n",
            "19255  YjeJ                 PF15922.8           4     0.71    283   0.59   0.59   0.53   0.02\n",
            "19256  YjfB_motility        PF14070.9          90    10.88     57   0.98   1.01   0.89   0.14\n",
            "19257  YjgF_endoribonc      PF14588.9           2     0.57    148   0.59   0.62   0.53   0.01\n",
            "19258  YjhX_toxin           PF09857.12         19     0.99     85   0.67   0.68   0.62   0.03\n",
            "19259  YJL171C_Tos1_C       PF10287.12        100     1.47    227   0.59   0.53   0.50   0.03\n",
            "19260  YJL171C_Tos1_N       PF10290.12         94     2.09     63   0.89   0.85   0.82   0.02\n",
            "19261  YjzC                 PF14168.9          25     2.48     55   1.01   0.94   0.93   0.08\n",
            "19262  Ykof                 PF07615.14          5     1.21     81   0.70   0.67   0.65   0.02\n",
            "19263  YkpC                 PF17447.5           5     1.85     42   1.31   1.38   1.27   0.10\n",
            "19264  YkuD                 PF03734.17        167    12.84    146   0.59   0.54   0.39   0.01\n",
            "19265  YkuD_2               PF13645.9          73     1.53    169   0.59   0.58   0.53   0.02\n",
            "19266  YkuI_C               PF10388.12          8     0.64    166   0.59   0.54   0.53   0.04\n",
            "19267  YkyA                 PF10368.12        106     4.62    186   0.59   0.60   0.56   0.12\n",
            "19268  YkyB                 PF14177.9         158     1.25    116   0.59   0.58   0.53   0.04\n",
            "19269  YL1                  PF05764.16        108     4.18    240   0.59   0.61   0.40   0.13\n",
            "19270  YL1_C                PF08265.14        181    24.33     29   1.85   1.77   1.84   0.15\n",
            "19271  YlaC                 PF10777.12         10     0.69    154   0.59   0.57   0.53   0.04\n",
            "19272  YlaH                 PF14036.9          24     1.56     76   0.74   0.77   0.68   0.14\n",
            "19273  YLATT                PF20303.1          35     1.75    108   0.59   0.62   0.52   0.04\n",
            "19274  YlbD_coat            PF14071.9          15     1.09    127   0.59   0.54   0.52   0.06\n",
            "19275  YlbE                 PF14003.9          16     1.38     61   0.92   0.82   0.88   0.08\n",
            "19276  YliH                 PF10799.11          3     0.54    126   0.59   0.59   0.52   0.03\n",
            "19277  YlmH_RBD             PF17774.4          72     1.25     84   0.68   0.67   0.63   0.02\n",
            "19278  YLP                  PF02757.20         26    26.00      9   2.79   2.82   2.79   0.70\n",
            "19279  YlqD                 PF11068.11         64     2.27    131   0.59   0.61   0.54   0.08\n",
            "19280  YlxR                 PF04296.16        514     4.63     75   0.75   0.74   0.66   0.05\n",
            "19281  YlzJ                 PF14035.9          52     4.77     65   0.86   0.85   0.82   0.03\n",
            "19282  YmaF                 PF12788.10         17     1.01     97   0.59   0.50   0.53   0.11\n",
            "19283  YmcE_antitoxin       PF15939.8           1     0.47     76   0.75   0.71   0.68   0.04\n",
            "19284  YmdB                 PF13277.9         301     1.80    253   0.59   0.60   0.52   0.02\n",
            "19285  YMF19                PF02326.18         23     2.96     86   0.66   0.62   0.58   0.06\n",
            "19286  YMF19_C              PF06449.14          2     0.89     51   1.09   1.07   1.02   0.08\n",
            "19287  YmgB                 PF10798.11         10     2.11     59   0.95   1.01   0.92   0.09\n",
            "19288  YmgD                 PF16456.8           2     0.62     82   0.69   0.71   0.63   0.03\n",
            "19289  YmzC                 PF14157.9           3     1.56     58   0.96   0.93   0.90   0.08\n",
            "19290  YndJ                 PF14158.9          17     2.13    261   0.59   0.62   0.53   0.07\n",
            "19291  YnfE                 PF17452.5           4     0.90     78   0.72   0.71   0.67   0.06\n",
            "19292  YniB                 PF14002.9          13     0.80    166   0.59   0.59   0.53   0.04\n",
            "19293  YoaP                 PF14268.9          44     6.56     44   1.25   1.22   1.22   0.05\n",
            "19294  YobH                 PF13996.9           5     0.68     70   0.81   0.83   0.75   0.06\n",
            "19295  YodL                 PF14191.9          32     1.44    100   0.59   0.55   0.51   0.04\n",
            "19296  YoeB_toxin           PF06769.17         13     0.90     80   0.71   0.65   0.66   0.03\n",
            "19297  YojJ                 PF10372.12          8     2.31     69   0.82   0.78   0.79   0.06\n",
            "19298  YokU                 PF14122.9           9     0.82     87   0.66   0.59   0.60   0.05\n",
            "19299  YolD                 PF08863.13         41     4.35     94   0.61   0.58   0.58   0.06\n",
            "19300  YonK                 PF09642.13          4     1.64     60   0.93   0.94   0.87   0.07\n",
            "19301  Yop-YscD_cpl         PF16697.8          19     2.44     94   0.61   0.64   0.55   0.02\n",
            "19302  Yop-YscD_ppl         PF16693.8           3     0.68    254   0.59   0.61   0.53   0.02\n",
            "19303  YopD                 PF05844.15          3     0.72    297   0.59   0.62   0.53   0.04\n",
            "19304  YopE                 PF03545.16          4     1.18     70   0.81   0.83   0.76   0.04\n",
            "19305  YopE_N               PF09020.13          1     0.43    126   0.59   0.61   0.52   0.03\n",
            "19306  YopH_N               PF09013.13          2     0.64    121   0.59   0.63   0.51   0.04\n",
            "19307  Yopt                 PF09467.13          1     0.57     71   0.79   0.83   0.73   0.04\n",
            "19308  YopX                 PF09643.13         14     2.59    129   0.59   0.56   0.50   0.03\n",
            "19309  YoqO                 PF14037.9           5     1.13    116   0.59   0.60   0.53   0.10\n",
            "19310  YorP                 PF09629.13          1     0.50     71   0.79   0.75   0.73   0.03\n",
            "19311  Yos1                 PF08571.13         82     1.64     80   0.71   0.76   0.61   0.07\n",
            "19312  Yos9_DD              PF17880.4          11     1.23    128   0.59   0.57   0.53   0.03\n",
            "19313  YozD                 PF14162.9          20     1.18     56   0.99   0.99   0.95   0.08\n",
            "19314  YozE_SAM_like        PF06855.15         49     3.82     66   0.85   0.76   0.80   0.08\n",
            "19315  YPEB                 PF14620.9          45     2.70    361   0.59   0.59   0.54   0.03\n",
            "19316  YpjP                 PF14005.9          18     1.07    134   0.59   0.54   0.53   0.04\n",
            "19317  YpM                  PF09144.13          2     0.38    117   0.59   0.58   0.52   0.02\n",
            "19318  YpmT                 PF17431.5           2     0.71     62   0.90   0.85   0.85   0.09\n",
            "19319  YppF                 PF14178.9           9     1.94     59   0.95   0.92   0.91   0.05\n",
            "19320  YppG                 PF14179.9          11     1.38    105   0.59   0.53   0.45   0.06\n",
            "19321  YpsA                 PF06908.14         62     1.84    168   0.59   0.56   0.54   0.03\n",
            "19322  YpzG                 PF14139.9          11     2.80     49   1.13   1.05   1.05   0.11\n",
            "19323  YpzI                 PF14140.9           9     1.66     42   1.31   1.33   1.27   0.17\n",
            "19324  YqaH                 PF17448.5           3     0.77     88   0.65   0.67   0.58   0.03\n",
            "19325  Yqai                 PF09466.13          3     1.38     66   0.85   0.82   0.76   0.04\n",
            "19326  YqaJ                 PF09588.13         17     3.05    148   0.59   0.55   0.44   0.02\n",
            "19327  YqbF                 PF14553.9           7     5.18     43   1.28   1.26   1.25   0.09\n",
            "19328  YqcI_YcgG            PF08892.14         60     2.27    211   0.59   0.53   0.53   0.02\n",
            "19329  YqeC                 PF19842.2         146     2.94    150   0.59   0.61   0.49   0.02\n",
            "19330  YqeY                 PF09424.13         39     1.45    143   0.59   0.62   0.54   0.05\n",
            "19331  YqfD                 PF06898.14        124     4.64    379   0.59   0.60   0.55   0.03\n",
            "19332  YqfQ                 PF14181.9           5     1.12    177   0.59   0.61   0.49   0.03\n",
            "19333  YqgB                 PF11036.11          2     1.16     43   1.28   1.27   1.23   0.07\n",
            "19334  YqgC                 PF17430.5           2     0.82     51   1.09   1.04   1.04   0.07\n",
            "19335  YqgF                 PF14639.9           4     0.84    150   0.59   0.61   0.52   0.02\n",
            "19336  YqhG                 PF11079.11         17     0.78    258   0.59   0.56   0.53   0.02\n",
            "19337  YqhR                 PF11085.11         11     0.66    165   0.59   0.57   0.53   0.04\n",
            "19338  YqjK                 PF13997.9          27     4.76     73   0.77   0.77   0.74   0.08\n",
            "19339  YqzE                 PF14038.9          20     2.52     54   1.03   0.97   0.99   0.07\n",
            "19340  YqzH                 PF14164.9           6     1.89     64   0.88   0.82   0.84   0.06\n",
            "19341  YqzL                 PF14006.9          15     4.00     45   1.22   1.17   1.17   0.09\n",
            "19342  YqzM                 PF14141.9          14     2.29     42   1.31   1.29   1.27   0.19\n",
            "19343  YrbL-PhoP_reg        PF10707.12          9     1.45    185   0.59   0.59   0.53   0.02\n",
            "19344  YrhC                 PF14143.9          14     2.29     72   0.78   0.80   0.75   0.06\n",
            "19345  YrhK                 PF14145.9          46     5.74     61   0.92   0.92   0.83   0.11\n",
            "19346  YrpD                 PF15493.9           5     0.80    216   0.59   0.58   0.51   0.03\n",
            "19347  YrvL                 PF14184.9          14     2.68    127   0.59   0.61   0.56   0.13\n",
            "19348  YrzK                 PF17449.5           3     1.30     54   1.03   0.99   0.99   0.08\n",
            "19349  YrzO                 PF14142.9           2     1.06     46   1.19   1.23   1.15   0.09\n",
            "19350  YsaB                 PF13983.9           3     0.71     75   0.75   0.70   0.70   0.04\n",
            "19351  Ysc84                PF04366.15        261     3.91    128   0.59   0.62   0.49   0.02\n",
            "19352  YscJ_FliF            PF01514.20        322     3.93    179   0.59   0.63   0.54   0.04\n",
            "19353  YscJ_FliF_C          PF08345.14        354     4.27    149   0.59   0.61   0.43   0.06\n",
            "19354  YscK                 PF06578.15          5     1.04    209   0.59   0.57   0.50   0.02\n",
            "19355  YscO                 PF07321.15         12     3.24    149   0.59   0.61   0.56   0.12\n",
            "19356  YscO-like            PF16789.8           5     1.06    161   0.59   0.60   0.54   0.10\n",
            "19357  YscW                 PF09619.13         35     2.80    107   0.59   0.62   0.53   0.03\n",
            "19358  YSIRK_signal         PF04650.20         45    32.96     26   2.06   2.15   2.05   0.24\n",
            "19359  Ytca                 PF17090.8           6     1.21     62   0.90   0.85   0.83   0.15\n",
            "19360  YtfJ_HI0045          PF09695.13         11     0.71    160   0.59   0.60   0.53   0.03\n",
            "19361  YTH                  PF04146.18        210     3.46    183   0.59   0.55   0.40   0.04\n",
            "19362  YtkA                 PF13115.9          55     7.98     86   0.66   0.64   0.57   0.04\n",
            "19363  Ytp1                 PF10355.12        110     1.38    277   0.59   0.55   0.50   0.03\n",
            "19364  YtpI                 PF14007.9          26     2.64     87   0.65   0.66   0.62   0.09\n",
            "19365  YTV                  PF07639.14         26     7.06     43   1.28   1.14   1.23   0.35\n",
            "19366  YtxC                 PF08812.14         83     3.07    217   0.59   0.58   0.53   0.05\n",
            "19367  YtxH                 PF12732.10        162     9.52     75   0.75   0.85   0.66   0.05\n",
            "19368  YtzH                 PF14165.9          16     1.36     86   0.66   0.64   0.61   0.06\n",
            "19369  YueH                 PF14166.9           9     1.75     79   0.72   0.71   0.66   0.05\n",
            "19370  YugN                 PF08868.13         20     0.99    130   0.59   0.57   0.53   0.02\n",
            "19371  YuiB                 PF14068.9          23     0.78    101   0.59   0.60   0.53   0.08\n",
            "19372  YukC                 PF10140.12         16     2.04    357   0.59   0.58   0.55   0.04\n",
            "19373  YukD                 PF08817.13        113     7.96     76   0.74   0.81   0.66   0.03\n",
            "19374  Yuri_gagarin         PF15934.8           3     0.58    234   0.59   0.58   0.52   0.05\n",
            "19375  YusW                 PF14039.9          19     2.67     92   0.62   0.63   0.56   0.06\n",
            "19376  YutD-like            PF06265.14         48     1.00     85   0.67   0.62   0.62   0.05\n",
            "19377  YuzL                 PF14115.9           9     2.98     41   1.34   1.39   1.29   0.23\n",
            "19378  YvbH_ext             PF11724.11         14     2.44     61   0.92   0.86   0.88   0.09\n",
            "19379  YvfG                 PF09628.13          1     0.59     67   0.84   0.82   0.78   0.05\n",
            "19380  YvrJ                 PF12841.10         45     8.51     36   1.51   1.61   1.50   0.09\n",
            "19381  YwcE                 PF17368.5           5     0.67     85   0.67   0.65   0.61   0.10\n",
            "19382  YWFCY                PF14293.9          37     1.80    144   0.59   0.60   0.55   0.06\n",
            "19383  YwhD                 PF08741.13         23     0.75    162   0.59   0.57   0.53   0.03\n",
            "19384  YwiC                 PF14256.9          65     3.51    126   0.59   0.59   0.53   0.09\n",
            "19385  YwpF                 PF14183.9          20     1.22    133   0.59   0.60   0.54   0.04\n",
            "19386  YwqJ-deaminase       PF14431.9           8     2.96    134   0.59   0.58   0.47   0.03\n",
            "19387  YxiJ                 PF14176.9           5     0.95    110   0.59   0.55   0.52   0.04\n",
            "19388  YycC                 PF14174.9           8     1.52     51   1.09   1.08   1.05   0.11\n",
            "19389  YycH                 PF07435.14          5     1.40    438   0.59   0.57   0.52   0.03\n",
            "19390  YycI                 PF09648.13         24     3.97    232   0.59   0.58   0.50   0.05\n",
            "19391  YyzF                 PF14116.9          22     3.10     48   1.15   1.04   1.09   0.15\n",
            "19392  Y_phosphatase        PF00102.30         90     3.39    235   0.59   0.53   0.49   0.03\n",
            "19393  Y_phosphatase2       PF03162.16          8     1.57    165   0.59   0.56   0.53   0.02\n",
            "19394  Y_phosphatase3       PF13350.9         211     3.54    234   0.59   0.59   0.46   0.02\n",
            "19395  Y_Y_Y                PF07495.16         75    14.25     66   0.85   0.79   0.78   0.07\n",
            "19396  z-alpha              PF02295.20          5     2.08     67   0.84   0.83   0.79   0.03\n",
            "19397  Z1                   PF10593.12         54     1.95    229   0.59   0.57   0.49   0.03\n",
            "19398  Zap1_zf2             PF18217.4          14     3.92     24   2.22   1.85   2.19   0.44\n",
            "19399  ZapA                 PF05164.16        372    10.51     88   0.65   0.70   0.54   0.06\n",
            "19400  ZapB                 PF06005.15         30     4.61     71   0.79   0.85   0.73   0.16\n",
            "19401  ZapC                 PF07126.15         26     1.57    169   0.59   0.55   0.54   0.03\n",
            "19402  ZapD                 PF07072.14         94     2.11    210   0.59   0.59   0.55   0.03\n",
            "19403  Zds_C                PF08632.13         11     1.32     52   1.07   1.03   1.03   0.08\n",
            "19404  Zea_mays_MuDR        PF05928.14          4     0.40    207   0.59   0.58   0.52   0.02\n",
            "19405  Zein                 PF01559.19          4     0.68    245   0.59   0.60   0.50   0.07\n",
            "19406  Zein-binding         PF04576.18         62     1.72     92   0.62   0.64   0.59   0.10\n",
            "19407  Zemlya               PF17646.4          11     1.86    106   0.59   0.62   0.54   0.05\n",
            "19408  Zeta_toxin           PF06414.15         16     3.25    201   0.59   0.60   0.51   0.02\n",
            "19409  zf-3CxxC             PF13695.9         128     3.12    100   0.59   0.46   0.42   0.09\n",
            "19410  zf-3CxxC_2           PF17180.7           3     0.81     74   0.76   0.62   0.69   0.09\n",
            "19411  zf-4CXXC_R1          PF10497.12         47     0.87     99   0.59   0.46   0.50   0.10\n",
            "19412  zf-A20               PF01754.19        117    56.00     24   2.22   1.85   2.18   0.55\n",
            "19413  zf-ACC               PF17848.4          31    20.12     26   2.06   1.70   2.06   0.44\n",
            "19414  zf-AD                PF07776.18         45     6.61     77   0.73   0.63   0.62   0.11\n",
            "19415  zf-AN1               PF01428.19        404     6.52     40   1.37   1.03   1.30   0.42\n",
            "19416  zf-ANAPC11           PF12861.10          5     0.55     85   0.67   0.53   0.60   0.06\n",
            "19417  zf-BED               PF02892.18         17     7.53     44   1.25   1.03   1.16   0.18\n",
            "19418  zf-B_box             PF00643.27         89    12.06     42   1.31   1.00   1.20   0.36\n",
            "19419  zf-C2H2              PF00096.29        159   159.00     23   1.65   1.32   1.56   0.38\n",
            "19420  zf-C2H2_10           PF16588.8          22     5.17     23   2.31   2.05   2.29   0.45\n",
            "19421  zf-C2H2_11           PF16622.8          11     3.67     29   1.85   1.64   1.83   0.30\n",
            "19422  zf-C2H2_12           PF18658.4          31     2.96     64   0.87   0.74   0.85   0.08\n",
            "19423  zf-C2H2_2            PF12756.10         95     2.22    101   0.59   0.47   0.49   0.08\n",
            "19424  zf-C2H2_3            PF13878.9         124    10.89     40   1.37   1.18   1.31   0.17\n",
            "19425  zf-C2H2_3rep         PF18868.4          11     0.59    126   0.59   0.52   0.52   0.04\n",
            "19426  zf-C2H2_4            PF13894.9         313   313.00     24   1.28   0.98   1.22   0.33\n",
            "19427  zf-C2H2_6            PF13912.9          26    26.00     27   1.81   1.57   1.80   0.29\n",
            "19428  zf-C2H2_7            PF15269.9           2     0.68     54   1.03   0.88   0.97   0.11\n",
            "19429  zf-C2H2_8            PF15909.8           1     0.37     98   0.59   0.53   0.51   0.04\n",
            "19430  zf-C2H2_9            PF16293.8           9     1.37     59   0.95   0.84   0.88   0.06\n",
            "19431  zf-C2H2_aberr        PF17017.8           6     0.86    176   0.59   0.51   0.52   0.05\n",
            "19432  zf-C2H2_assoc        PF16606.8           5     0.64     81   0.70   0.74   0.64   0.09\n",
            "19433  zf-C2H2_assoc2       PF16624.8           6     1.15     97   0.59   0.61   0.50   0.09\n",
            "19434  zf-C2H2_assoc3       PF16637.8           7     0.97     85   0.67   0.65   0.59   0.06\n",
            "19435  zf-C2H2_jaz          PF12171.11        111   111.00     27   1.82   1.53   1.81   0.37\n",
            "19436  zf-C2HC              PF01530.21         64     8.56     29   1.85   1.63   1.81   0.36\n",
            "19437  zf-C2HC5             PF06221.16         61     2.02     54   1.03   0.84   0.96   0.23\n",
            "19438  zf-C2HCIx2C          PF10782.12         15     1.89     58   0.96   0.86   0.91   0.12\n",
            "19439  zf-C2HC_2            PF13913.9          94    94.00     25   1.98   1.70   1.95   0.37\n",
            "19440  zf-C2HE              PF16278.8          99     6.58     63   0.89   0.82   0.79   0.09\n",
            "19441  zf-C3H1              PF10650.12         38    17.37     22   2.41   2.09   2.38   0.42\n",
            "19442  zf-C3H2C3            PF17122.8           3     1.75     35   1.55   1.25   1.51   0.35\n",
            "19443  zf-C3HC              PF07967.16         22     2.31    133   0.59   0.51   0.49   0.05\n",
            "19444  zf-C3Hc3H            PF13891.9          81     2.61     62   0.90   0.70   0.79   0.18\n",
            "19445  zf-C3HC4             PF00097.28         35     4.75     41   1.34   0.99   1.23   0.45\n",
            "19446  zf-C3HC4_2           PF13923.9           9     2.55     40   1.37   1.05   1.31   0.37\n",
            "19447  zf-C3HC4_3           PF13920.9         337     5.80     50   1.11   0.83   1.04   0.34\n",
            "19448  zf-C3HC4_4           PF15227.9           7     1.82     42   1.30   1.02   1.26   0.34\n",
            "19449  zf-C3HC4_5           PF17121.8           6     0.88     51   1.09   0.90   1.04   0.15\n",
            "19450  zf-C4                PF00105.21         26     0.89     70   0.80   0.64   0.75   0.16\n",
            "19451  zf-C4H2              PF10146.12         24     1.40    244   0.59   0.56   0.50   0.07\n",
            "19452  zf-C4pol             PF14260.9         215     3.09     70   0.80   0.62   0.65   0.21\n",
            "19453  zf-C4_ClpX           PF06689.16        300     8.76     39   1.40   1.23   1.36   0.25\n",
            "19454  zf-C4_Topoisom       PF01396.22         22     5.14     39   1.40   1.19   1.34   0.22\n",
            "19455  zf-C5HC2             PF02928.19        193     6.59     53   1.05   0.84   0.95   0.22\n",
            "19456  zf-C6H2              PF15801.8         106     1.98     47   1.17   0.91   1.10   0.26\n",
            "19457  zf-CCCH              PF00642.27         70    70.00     27   1.83   1.51   1.82   0.38\n",
            "19458  zf-CCCH_2            PF14608.9         156   156.00     19   2.28   1.85   2.15   0.66\n",
            "19459  zf-CCCH_3            PF15663.8           9     0.57    111   0.59   0.47   0.51   0.05\n",
            "19460  zf-CCCH_4            PF18044.4          16    16.00     22   2.30   1.90   2.23   0.49\n",
            "19461  zf-CCCH_6            PF18585.4          40     2.15     53   1.05   0.97   1.01   0.08\n",
            "19462  zf-CCCH_7            PF18586.4           3     0.92     90   0.63   0.60   0.57   0.03\n",
            "19463  zf-CCCH_8            PF18633.4           5     2.00     28   1.91   1.70   1.89   0.29\n",
            "19464  zf-CCHC              PF00098.26         79    79.00     18   2.27   1.85   2.27   0.58\n",
            "19465  zf-CCHC_2            PF13696.9          45    28.92     21   2.52   2.08   2.50   0.54\n",
            "19466  zf-CCHC_3            PF13917.9          19     4.95     41   1.34   1.15   1.23   0.21\n",
            "19467  zf-CCHC_4            PF14392.9           9     2.87     49   1.13   1.00   1.10   0.09\n",
            "19468  zf-CCHC_5            PF14787.9           5     2.47     36   1.51   1.35   1.48   0.22\n",
            "19469  zf-CCHC_6            PF15288.9          15     2.87     42   1.31   1.15   1.28   0.17\n",
            "19470  zf-CCHH              PF10283.12         67    14.73     26   2.06   1.71   2.01   0.43\n",
            "19471  zf-CDGSH             PF09360.13        175     5.88     48   1.15   1.03   1.02   0.10\n",
            "19472  zf-CGNR              PF11706.11        218     4.78     44   1.25   1.05   1.19   0.26\n",
            "19473  zf-CHC2              PF01807.23         17     0.93     98   0.59   0.54   0.54   0.03\n",
            "19474  zf-CHCC              PF10276.12         99     5.61     37   1.47   1.28   1.39   0.17\n",
            "19475  zf-CHY               PF05495.15        237     1.67     75   0.75   0.52   0.61   0.23\n",
            "19476  zf-CRD               PF17979.4          42     2.15    159   0.59   0.46   0.48   0.08\n",
            "19477  zf-CSL               PF05207.16         56     4.43     59   0.95   0.82   0.84   0.12\n",
            "19478  zf-CW                PF07496.18        150     5.71     48   1.15   0.90   1.02   0.23\n",
            "19479  zf-CXXC              PF02008.23         31     2.55     48   1.15   0.87   1.09   0.44\n",
            "19480  zf-DBF               PF07535.15         96     4.16     45   1.22   1.06   1.20   0.15\n",
            "19481  zf-Di19              PF05605.15         42     2.63     54   1.03   0.83   0.94   0.17\n",
            "19482  zf-DNA_Pol           PF08996.13        126     2.49    186   0.59   0.51   0.50   0.06\n",
            "19483  zf-DNL               PF05180.15         99     1.65     64   0.87   0.76   0.80   0.09\n",
            "19484  zf-Dof               PF02701.18         44     1.05     59   0.95   0.78   0.88   0.12\n",
            "19485  zf-dskA_traR         PF01258.20        133    86.91     36   1.51   1.36   1.46   0.26\n",
            "19486  zf-FCS               PF06467.17         39    23.96     41   1.34   1.07   1.30   0.28\n",
            "19487  zf-FLZ               PF04570.17         58     2.42     54   1.03   0.88   1.00   0.14\n",
            "19488  zf-FPG_IleRS         PF06827.17         78    78.00     30   1.77   1.48   1.75   0.35\n",
            "19489  zf-GRF               PF06839.15         12     2.46     45   1.22   0.99   1.15   0.19\n",
            "19490  zf-H2C2              PF09337.13         22     7.94     39   1.40   1.27   1.36   0.17\n",
            "19491  zf-H2C2_2            PF13465.9         104   104.00     26   1.58   1.32   1.48   0.29\n",
            "19492  zf-H2C2_5            PF13909.9          23    11.82     25   2.13   1.84   2.10   0.36\n",
            "19493  zf-H3C2              PF16721.8           1     0.39    102   0.59   0.55   0.51   0.02\n",
            "19494  zf-HC2               PF13490.9         572   572.00     35   1.53   1.43   1.49   0.25\n",
            "19495  zf-HC3               PF16827.8          10     1.54     67   0.84   0.70   0.76   0.08\n",
            "19496  zf-HC5HC2H           PF13771.9          84     1.95     90   0.63   0.47   0.54   0.14\n",
            "19497  zf-HC5HC2H_2         PF13832.9          28     1.25    109   0.59   0.45   0.50   0.10\n",
            "19498  ZF-HD_dimer          PF04770.15         54     1.44     55   1.01   0.85   0.95   0.14\n",
            "19499  zf-His_Me_endon      PF05551.14         11     1.97    131   0.59   0.49   0.48   0.07\n",
            "19500  zf-HIT               PF04438.19         87    12.24     30   1.80   1.43   1.72   0.57\n",
            "19501  zf-HYPF              PF07503.15        484     9.65     34   1.59   1.34   1.57   0.27\n",
            "19502  zf-IS66              PF13005.10        169    18.69     46   1.20   1.03   1.12   0.23\n",
            "19503  zf-ISL3              PF14690.9          60    16.63     47   1.17   0.96   1.13   0.21\n",
            "19504  zf-like              PF04071.15         24     0.99     82   0.69   0.53   0.61   0.10\n",
            "19505  zf-LITAF-like        PF10601.12        306     5.10     70   0.80   0.62   0.72   0.16\n",
            "19506  zf-LSD1              PF06943.15         77    15.77     25   2.13   1.86   2.10   0.51\n",
            "19507  zf-LYAR              PF08790.14         77     6.72     28   1.92   1.64   1.90   0.36\n",
            "19508  zf-met               PF12874.10        266   266.00     25   1.85   1.57   1.85   0.33\n",
            "19509  zf-met2              PF12907.10          8     2.93     38   1.43   1.24   1.40   0.20\n",
            "19510  zf-MIZ               PF02891.23         29     2.09     50   1.11   0.87   1.07   0.18\n",
            "19511  zf-Mss51             PF13824.9          74     2.65     57   0.98   0.81   0.88   0.13\n",
            "19512  zf-MYND              PF01753.21        126     5.64     40   1.37   1.01   1.26   0.47\n",
            "19513  zf-MYST              PF17772.4          30     2.85     55   1.01   0.84   0.97   0.10\n",
            "19514  zf-NADH-PPase        PF09297.14         43    36.28     32   1.69   1.44   1.67   0.30\n",
            "19515  zf-nanos             PF05741.16         68     2.83     54   1.03   0.82   0.96   0.18\n",
            "19516  zf-NF-X1             PF01422.20        103    12.20     19   2.77   2.18   2.69   1.12\n",
            "19517  zf-NOSIP             PF15906.8           2     0.52     75   0.76   0.71   0.69   0.04\n",
            "19518  zf-NPL4              PF05020.18         74     0.88    145   0.59   0.52   0.53   0.03\n",
            "19519  zf-Nse               PF11789.11         19     1.92     58   0.96   0.85   0.88   0.09\n",
            "19520  zf-P11               PF03854.17          3     0.94     50   1.11   0.93   1.06   0.17\n",
            "19521  zf-Paramyx-P         PF13008.10          6     1.81     45   1.22   0.96   1.19   0.27\n",
            "19522  zf-PARP              PF00645.21        182     3.56     82   0.69   0.61   0.57   0.07\n",
            "19523  zf-PHD-like          PF15446.9          47     0.83    169   0.59   0.49   0.52   0.08\n",
            "19524  zf-piccolo           PF05715.16         18     1.16     60   0.93   0.74   0.85   0.17\n",
            "19525  zf-primase           PF09329.14        122     5.66     46   1.20   1.04   1.14   0.16\n",
            "19526  zf-RAG1              PF10426.12          6     2.96     30   1.80   1.63   1.76   0.23\n",
            "19527  zf-RanBP             PF00641.21         24    21.07     30   1.80   1.46   1.77   0.42\n",
            "19528  zf-rbx1              PF12678.10         66     2.29     55   1.01   0.77   0.92   0.20\n",
            "19529  zf-ribbon_3          PF13248.9          24    24.00     27   1.67   1.41   1.66   0.38\n",
            "19530  zf-RING-like         PF08746.14        102     4.78     43   1.28   0.96   1.21   0.37\n",
            "19531  zf-RING_10           PF16685.8          16     0.93     70   0.80   0.66   0.74   0.14\n",
            "19532  zf-RING_11           PF17123.8         114     9.84     29   1.85   1.52   1.81   0.42\n",
            "19533  zf-RING_12           PF17976.4           5     0.68     73   0.78   0.65   0.72   0.07\n",
            "19534  zf-RING_13           PF17977.4           1     0.48     68   0.82   0.71   0.76   0.09\n",
            "19535  zf-RING_14           PF17978.4          17     0.64     91   0.62   0.53   0.56   0.06\n",
            "19536  zf-RING_15           PF16744.8           7     0.96     73   0.77   0.65   0.71   0.10\n",
            "19537  zf-RING_16           PF17120.8           9     1.52     58   0.96   0.73   0.90   0.22\n",
            "19538  zf-RING_2            PF13639.9         165     8.02     44   1.25   0.93   1.08   0.37\n",
            "19539  zf-RING_4            PF14570.9          45     2.25     48   1.15   0.88   1.09   0.27\n",
            "19540  zf-RING_5            PF14634.9          75     7.45     44   1.25   0.92   1.12   0.41\n",
            "19541  zf-RING_6            PF14835.9           1     0.50     65   0.87   0.78   0.80   0.07\n",
            "19542  zf-RING_7            PF02591.18        173    42.72     33   1.64   1.48   1.58   0.24\n",
            "19543  zf-RING_9            PF13901.9          99     1.55    205   0.59   0.48   0.51   0.08\n",
            "19544  zf-RING_UBOX         PF13445.9         484    23.16     40   1.37   1.10   1.16   0.31\n",
            "19545  zf-RNPHF             PF08080.15          3     1.30     36   1.51   1.39   1.47   0.21\n",
            "19546  zf-RRN7              PF11781.11         74    74.00     32   1.68   1.41   1.62   0.31\n",
            "19547  zf-RRPl_C4           PF17026.8          11     1.12    108   0.59   0.47   0.51   0.12\n",
            "19548  zf-RVT               PF13966.9          96     3.71     85   0.67   0.52   0.59   0.08\n",
            "19549  zf-SAP30             PF13866.9          13     0.85     72   0.78   0.72   0.72   0.09\n",
            "19550  zf-SCNM1             PF15803.8          17     4.42     27   1.98   1.81   1.97   0.26\n",
            "19551  zf-Sec23_Sec24       PF04810.18        300     9.75     39   1.40   1.11   1.33   0.26\n",
            "19552  zf-TAZ               PF02135.19        145     2.00     72   0.78   0.57   0.61   0.23\n",
            "19553  zf-tcix              PF14952.9          10     2.72     42   1.31   1.16   1.28   0.24\n",
            "19554  zf-TFIIB             PF13453.9          57    10.09     41   1.34   1.17   1.28   0.16\n",
            "19555  zf-TFIIIC            PF12660.10         20     1.68    101   0.59   0.51   0.46   0.06\n",
            "19556  zf-Tim10_DDP         PF02953.18        406     7.49     64   0.88   0.76   0.82   0.13\n",
            "19557  zf-TRAF              PF02176.21          9     1.87     60   0.93   0.73   0.87   0.19\n",
            "19558  zf-TRAF_2            PF15965.8          23     0.68     93   0.61   0.46   0.55   0.12\n",
            "19559  zf-trcl              PF13451.9          76     3.29     48   1.15   0.95   1.11   0.22\n",
            "19560  zf-TRM13_CCCH        PF11722.11         77    15.84     29   1.85   1.56   1.82   0.34\n",
            "19561  zf-U1                PF06220.15         11     3.18     38   1.43   1.19   1.40   0.20\n",
            "19562  zf-U11-48K           PF05253.15        198   198.00     25   2.00   1.72   2.00   0.33\n",
            "19563  zf-UBP               PF02148.22        147     4.16     64   0.88   0.65   0.73   0.17\n",
            "19564  zf-UBP_var           PF17807.4          22     3.18     64   0.88   0.79   0.84   0.05\n",
            "19565  zf-UBR               PF02207.23        269     2.54     70   0.80   0.59   0.67   0.22\n",
            "19566  zf-UDP               PF14569.9          26     1.09     78   0.72   0.64   0.66   0.08\n",
            "19567  zf-WRNIP1_ubi        PF18279.4           1     1.00     21   1.58   1.33   1.53   0.34\n",
            "19568  zf-XS                PF03470.17         12     3.11     43   1.28   1.19   1.21   0.12\n",
            "19569  zf-ZPR1              PF03367.16        289     2.69    162   0.59   0.56   0.49   0.04\n",
            "19570  Zfx_Zfy_act          PF04704.16          9     0.88    328   0.59   0.60   0.50   0.04\n",
            "19571  ZFYVE21_C            PF16696.8          12     1.19    123   0.59   0.59   0.49   0.03\n",
            "19572  zf_C2H2_10           PF18414.4           3     1.81     26   2.06   1.81   2.03   0.37\n",
            "19573  zf_C2H2_13           PF18508.4           7     1.92     42   1.31   1.20   1.25   0.11\n",
            "19574  zf_C2H2_6            PF18450.4           6     2.18     28   1.92   1.71   1.89   0.28\n",
            "19575  zf_C2H2_ZHX          PF18387.4          18     2.02     53   1.05   0.96   1.01   0.07\n",
            "19576  zf_C2HC_14           PF18574.4          17     6.51     33   1.64   1.39   1.63   0.29\n",
            "19577  zf_CCCH_4            PF18345.4         275   275.00     19   2.42   1.95   2.38   0.64\n",
            "19578  zf_CCCH_5            PF18384.4           3     1.31     40   1.36   1.09   1.33   0.26\n",
            "19579  zf_CopZ              PF18423.4          61     3.66     62   0.90   0.78   0.86   0.10\n",
            "19580  zf_Hakai             PF18408.4           7     1.84     32   1.69   1.53   1.61   0.20\n",
            "19581  zf_PR_Knuckle        PF18445.4           3     1.03     38   1.44   1.26   1.39   0.13\n",
            "19582  zf_Rg                PF17915.4           6     1.60     49   1.13   1.02   1.08   0.13\n",
            "19583  zf_UBZ               PF18439.4          72    15.53     32   1.69   1.47   1.68   0.23\n",
            "19584  zf_ZIC               PF18366.4          16     1.55     46   1.20   1.05   1.15   0.08\n",
            "19585  zinc-ribbons_6       PF07191.15         21     1.49     67   0.84   0.66   0.76   0.20\n",
            "19586  zinc-ribbon_6        PF10005.12         22     1.04     93   0.61   0.48   0.53   0.09\n",
            "19587  Zincin_1             PF06262.14         99     2.81     97   0.59   0.59   0.49   0.02\n",
            "19588  Zincin_2             PF10103.12        133     2.56    332   0.59   0.60   0.50   0.03\n",
            "19589  zinc_ribbon_10       PF10058.12         25     3.38     54   1.03   0.88   0.98   0.11\n",
            "19590  zinc_ribbon_11       PF11682.11          2     0.50    128   0.59   0.52   0.51   0.04\n",
            "19591  zinc_ribbon_12       PF11331.11         32     2.56     45   1.22   1.13   1.18   0.16\n",
            "19592  zinc_ribbon_13       PF09855.12         37     1.71     63   0.89   0.78   0.84   0.09\n",
            "19593  zinc_ribbon_15       PF17032.8          51     2.11     86   0.66   0.49   0.58   0.14\n",
            "19594  zinc_ribbon_16       PF17034.8           3     0.57    125   0.59   0.48   0.50   0.06\n",
            "19595  zinc_ribbon_2        PF13240.9         179   179.00     23   2.02   1.69   2.01   0.52\n",
            "19596  zinc_ribbon_4        PF13717.9          52    10.51     36   1.51   1.27   1.50   0.25\n",
            "19597  zinc_ribbon_5        PF13719.9           3     2.14     37   1.47   1.32   1.43   0.21\n",
            "19598  zinc_ribbon_6        PF14599.9         108     2.31     59   0.95   0.77   0.90   0.12\n",
            "19599  zinc_ribbon_9        PF14369.9          57    10.21     35   1.55   1.25   1.49   0.26\n",
            "19600  ZinT                 PF09223.14         34     0.78    181   0.59   0.53   0.53   0.04\n",
            "19601  Zip                  PF02535.25         37     5.70    333   0.59   0.60   0.50   0.03\n",
            "19602  ZIP4_domain          PF18292.4          28     2.36    171   0.59   0.57   0.51   0.05\n",
            "19603  ZipA_C               PF04354.16         88     3.14    128   0.59   0.59   0.54   0.03\n",
            "19604  ZirS_C               PF16583.8           2     0.46    145   0.59   0.59   0.52   0.02\n",
            "19605  Zmiz1_N              PF18028.4           3     0.64     99   0.59   0.56   0.53   0.03\n",
            "19606  Zn-C2H2_12           PF18112.4           7     5.59     27   1.98   1.71   1.97   0.30\n",
            "19607  zn-ribbon_14         PF16503.8          64     5.82     32   1.69   1.53   1.68   0.31\n",
            "19608  Zn-ribbon_8          PF09723.13        507    11.24     40   1.37   1.14   1.25   0.21\n",
            "19609  ZNRF_3_ecto          PF18212.4           5     0.73    105   0.59   0.61   0.53   0.03\n",
            "19610  ZnuA                 PF01297.20        945     5.39    247   0.59   0.59   0.44   0.03\n",
            "19611  Zn_clus              PF00172.21         29     6.22     40   1.37   1.08   1.29   0.45\n",
            "19612  Zn_dep_PLPC          PF00882.21        129    18.11    179   0.59   0.52   0.40   0.03\n",
            "19613  Zn_finger_prot       PF19242.2           3     0.63    268   0.59   0.55   0.50   0.04\n",
            "19614  Zn_peptidase         PF04228.16          5     0.69    290   0.59   0.57   0.52   0.02\n",
            "19615  Zn_peptidase_2       PF04298.15        300     1.80    218   0.59   0.61   0.54   0.04\n",
            "19616  Zn_protease          PF05618.14         10     2.11    138   0.59   0.60   0.51   0.03\n",
            "19617  Zn_ribbon_2          PF12674.10         90     2.07     82   0.69   0.57   0.61   0.08\n",
            "19618  Zn_ribbon_recom      PF13408.9         270    17.83     58   0.96   0.83   0.80   0.12\n",
            "19619  Zn_ribbon_SprT       PF17283.5          31    19.30     38   1.44   1.15   1.33   0.31\n",
            "19620  Zn_Tnp_IS1           PF03811.16          3     1.59     35   1.55   1.33   1.51   0.27\n",
            "19621  Zn_Tnp_IS1595        PF12760.10         30     6.39     46   1.20   0.96   1.09   0.22\n",
            "19622  Zn_Tnp_IS91          PF14319.9         247     3.14     91   0.63   0.49   0.56   0.09\n",
            "19623  Zona_pellucida       PF00100.26        337     7.27    259   0.59   0.51   0.35   0.05\n",
            "19624  ZoocinA_TRD          PF16775.8           4     0.67    106   0.59   0.55   0.51   0.03\n",
            "19625  Zot                  PF05707.15          7     1.98    194   0.59   0.57   0.46   0.02\n",
            "19626  ZT_dimer             PF16916.8         362     6.97     79   0.72   0.71   0.66   0.06\n",
            "19627  ZU5                  PF00791.23         61     2.85     97   0.59   0.59   0.54   0.01\n",
            "19628  Zw10                 PF06248.16          7     1.59    513   0.59   0.59   0.51   0.03\n",
            "19629  Zwilch               PF09817.12         15     2.12    574   0.59   0.58   0.49   0.02\n",
            "19630  Zwint                PF15556.9           7     0.63    252   0.59   0.60   0.52   0.06\n",
            "19631  ZYG-11_interact      PF05884.15          8     1.10    295   0.59   0.62   0.51   0.03\n",
            "19632  ZZ                   PF00569.20          7     3.44     45   1.22   0.95   1.18   0.26\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!hmmsearch --tblout output.out Pfam-A.hmm protein.faa"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "id": "B-KJbrW2QbfF",
        "outputId": "d305af9f-4050-4e57-af44-a38e8f78aa99"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mВыходные данные были обрезаны до нескольких последних строк (5000).\u001b[0m\n",
            "   1 ?   11.6   0.1   1.5e-05     0.093       1      14 [.      29      42 ..      29      49 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.6 bits;  conditional E-value: 1.5e-05\n",
            "           zf_Rg  1 MiksvfeklCPNCg 14\n",
            "                    Mi +v++++C NC \n",
            "  XP_004357565.1 29 MIMLVVKRSCANCT 42\n",
            "                    9************6 PP\n",
            "\n",
            ">> XP_004359870.1  uncharacterized protein DFA_01908 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.1   0.2     0.014        85       9      28 ..      80      99 ..      74     112 .. 0.84\n",
            "   2 ?   -2.6   0.1      0.43   2.6e+03      26      32 ..     416     422 ..     415     428 .. 0.85\n",
            "   3 ?    7.8   0.2   0.00023       1.4       8      26 ..     430     448 ..     424     454 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.1 bits;  conditional E-value: 0.014\n",
            "           zf_Rg  9 lCPNCgGdisdeRlekGLpC 28\n",
            "                     CP C  +is+  l + L+ \n",
            "  XP_004359870.1 80 KCPQCRVPISNGGLSRSLLA 99\n",
            "                    7*********9999988875 PP\n",
            "\n",
            "  == domain 2  score: -2.6 bits;  conditional E-value: 0.43\n",
            "           zf_Rg  26 LpCkrCl 32 \n",
            "                     L C++Cl\n",
            "  XP_004359870.1 416 LGCQSCL 422\n",
            "                     67****8 PP\n",
            "\n",
            "  == domain 3  score: 7.8 bits;  conditional E-value: 0.00023\n",
            "           zf_Rg   8 klCPNCgGdisdeRlekGL 26 \n",
            "                       CP C  +is+ Rl + L\n",
            "  XP_004359870.1 430 GTCPQCRTPISNGRLSRSL 448\n",
            "                     57*************9987 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (49 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       302  (0.0248867); expected 242.7 (0.02)\n",
            "Passed bias filter:                      181  (0.0149155); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        22  (0.00181294); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.08u 0.00s 00:00:00.08 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 6157.27\n",
            "//\n",
            "Query:       zf_UBZ  [M=32]\n",
            "Accession:   PF18439.4\n",
            "Description: Ubiquitin-Binding Zinc Finger\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (32 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       527  (0.0434281); expected 242.7 (0.02)\n",
            "Passed bias filter:                      325  (0.026782); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        49  (0.00403791); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000576844); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.01s 00:00:00.12 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 3218.07\n",
            "//\n",
            "Query:       zf_ZIC  [M=46]\n",
            "Accession:   PF18366.4\n",
            "Description: Zic proteins zinc finger domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (46 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       250  (0.0206016); expected 242.7 (0.02)\n",
            "Passed bias filter:                      210  (0.0173053); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        15  (0.00123609); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 5011.95\n",
            "//\n",
            "Query:       zinc-ribbons_6  [M=67]\n",
            "Accession:   PF07191.15\n",
            "Description: zinc-ribbons\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    0.00073   18.9  10.7     0.0015   17.9  10.7    1.5  1  XP_004355683.1  uncharacterized protein DFA_08186 [Cavenderia\n",
            "     0.0098   15.3   4.5       0.02   14.3   4.5    1.5  1  XP_004366868.1  MEK1 interacting protein 1 [Cavenderia fascic\n",
            "  ------ inclusion threshold ------\n",
            "      0.042   13.3   3.4      0.084   12.3   3.4    1.4  1  XP_004366193.1  myb domain-containing protein [Cavenderia fas\n",
            "      0.042   13.3   6.5        0.1   12.0   6.5    1.6  1  XP_004362359.1  uncharacterized protein DFA_02751 [Cavenderia\n",
            "      0.058   12.8   0.6       0.18   11.3   0.6    1.8  1  XP_004362193.1  uncharacterized protein DFA_06492 [Cavenderia\n",
            "       0.52    9.8  13.9       0.92    9.0  11.8    2.6  2  XP_004361325.1  uncharacterized protein DFA_05607 [Cavenderia\n",
            "        2.1    7.8   8.9        4.6    6.8   5.9    1.9  2  XP_004350883.1  methionyl-tRNA synthetase [Cavenderia fascicu\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004355683.1  uncharacterized protein DFA_08186 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.9  10.7   8.8e-07    0.0015       3      47 ..      38      94 ..      36      98 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.9 bits;  conditional E-value: 8.8e-07\n",
            "                    -SSS-S--EE-S.........SEEE-TTT--...EEEEEEE-SSS-S---EEEETTE CS\n",
            "  zinc-ribbons_6  3 CpeCqqelekrg.........eqyhCeaCqe...dlklealCpeCeeelevlkaCga 47\n",
            "                    C++C+  +e+r+            +C +C++       + lC++C ++l+v+ +C +\n",
            "  XP_004355683.1 38 CQDCHDTIEWRKkyrkykplsMAGKCAHCTQrkvTRAYHELCEDCVKKLKVCAKCHK 94\n",
            "                    *********9999******9988899999885445567889**************65 PP\n",
            "\n",
            ">> XP_004366868.1  MEK1 interacting protein 1 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   14.3   4.5   1.2e-05      0.02       2      40 ..     314     357 ..     297     362 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.3 bits;  conditional E-value: 1.2e-05\n",
            "                     --SSS-S--EE-S....SEEE-TTT--.EEEEEEE-SSS-S--- CS\n",
            "  zinc-ribbons_6   2 kCpeCqqelekrg....eqyhCeaCqe.dlklealCpeCeeele 40 \n",
            "                      Cp C +++ek +     +++C  C +  l+++++Cp C+ +l \n",
            "  XP_004366868.1 314 SCPICFEKMEKMSsttcGHVFCTDCIQaALTRRKVCPVCSTKLT 357\n",
            "                     7*******997776788999*****665999*********9997 PP\n",
            "\n",
            ">> XP_004366193.1  myb domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.3   3.4   4.8e-05     0.084      15      46 ..     356     389 ..     338     403 .. 0.68\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.3 bits;  conditional E-value: 4.8e-05\n",
            "                     SEEE-TTT--..EEEEEEE-SSS-S---EEEETT CS\n",
            "  zinc-ribbons_6  15 eqyhCeaCqe..dlklealCpeCeeelevlkaCg 46 \n",
            "                     e y+C+ C++   +  +  C+eC e ++ ++ C \n",
            "  XP_004366193.1 356 EGYKCDGCDQepIVGYRYKCEECIEDIDLCQSCH 389\n",
            "                     5577777777444566666777777777777774 PP\n",
            "\n",
            ">> XP_004362359.1  uncharacterized protein DFA_02751 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.0   6.5     6e-05       0.1      15      51 ..     843     879 ..     824     888 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.0 bits;  conditional E-value: 6e-05\n",
            "                     SEEE-TTT--EEEEEEE-SSS-S---EEEETTEEEE- CS\n",
            "  zinc-ribbons_6  15 eqyhCeaCqedlklealCpeCeeelevlkaCgavdyf 51 \n",
            "                     + y+C++C++ l +  lC  C+  +   kaC ++++f\n",
            "  XP_004362359.1 843 QLYRCHQCKQTLMKPLLCSGCKAVVYCSKACQKESWF 879\n",
            "                     569******************************9999 PP\n",
            "\n",
            ">> XP_004362193.1  uncharacterized protein DFA_06492 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.3   0.6    0.0001      0.18       2      28 ..     262     291 ..     261     295 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.3 bits;  conditional E-value: 0.0001\n",
            "                     --SSS-S--EE-S...SEEE-TTT--EEEE CS\n",
            "  zinc-ribbons_6   2 kCpeCqqelekrg...eqyhCeaCqedlkl 28 \n",
            "                      Cp+C++ +++++   +q++C +C++ +++\n",
            "  XP_004362193.1 262 ICPSCDRLVSWNQadqTQFYCTTCKQPFNI 291\n",
            "                     5*******99988888999*****998876 PP\n",
            "\n",
            ">> XP_004361325.1  uncharacterized protein DFA_05607 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.7   0.0       1.2   2.1e+03      49      63 ..     323     338 ..     322     341 .. 0.73\n",
            "   2 ?    9.0  11.8   0.00053      0.92       2      55 ..     820     885 ..     819     891 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.7 bits;  conditional E-value: 1.2\n",
            "                     EE--TTTT.----TTS CS\n",
            "  zinc-ribbons_6  49 dyfCnkcn.eLiSkkr 63 \n",
            "                     ++fC     +LiS +r\n",
            "  XP_004361325.1 323 GFFCRGASgSLISNER 338\n",
            "                     79**965427999876 PP\n",
            "\n",
            "  == domain 2  score: 9.0 bits;  conditional E-value: 0.00053\n",
            "                     --SSS-S--EE-S.......SEEE-TTT--EEE....EEEE-SSS-S---EE...EETTEEEE--TTT CS\n",
            "  zinc-ribbons_6   2 kCpeCqqelekrg.......eqyhCeaCqedlk....lealCpeCeeelevl...kaCgavdyfCnkc 55 \n",
            "                     +C  C + l+k++       +++hC+aC+++++    ++  C  Ce+ le       Cg    +C +c\n",
            "  XP_004361325.1 820 QCSGCAKTLTKNEphfkrknDKVHCDACEKEYQslkmRNRKCTSCEKILEQNgtrVICGQ--LYCPTC 885\n",
            "                     7999*9999988889999999*********77622223457********97522256988..899988 PP\n",
            "\n",
            ">> XP_004350883.1  methionyl-tRNA synthetase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.8   5.9    0.0026       4.6       3      36 ..     149     182 ..     132     184 .. 0.88\n",
            "   2 ?    6.0   8.9    0.0045       7.8       3      40 ..     149     186 ..     147     210 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.8 bits;  conditional E-value: 0.0026\n",
            "                     -SSS-S--EE-SSEEE-TTT--EEEEEEE-SSS- CS\n",
            "  zinc-ribbons_6   3 CpeCqqelekrgeqyhCeaCqedlklealCpeCe 36 \n",
            "                     C++C+  l  r  +  C +Cq +  +   C++C \n",
            "  XP_004350883.1 149 CEQCKIFLADRYVEGICPHCQYEDARGDQCDKCT 182\n",
            "                     9999999999999999999999999999999997 PP\n",
            "\n",
            "  == domain 2  score: 6.0 bits;  conditional E-value: 0.0045\n",
            "                     -SSS-S--EE-SSEEE-TTT--EEEEEEE-SSS-S--- CS\n",
            "  zinc-ribbons_6   3 CpeCqqelekrgeqyhCeaCqedlklealCpeCeeele 40 \n",
            "                     C++C+  l  r  +  C +Cq +  +   C++C + l+\n",
            "  XP_004350883.1 149 CEQCKIFLADRYVEGICPHCQYEDARGDQCDKCTKLLN 186\n",
            "                     *********************************99886 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (67 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       907  (0.0747425); expected 242.7 (0.02)\n",
            "Passed bias filter:                      377  (0.0310672); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        73  (0.00601566); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        12  (0.000988875); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               7  [number of targets reported over threshold]\n",
            "# CPU time: 0.16u 0.01s 00:00:00.17 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 5259.57\n",
            "//\n",
            "Query:       zinc-ribbon_6  [M=93]\n",
            "Accession:   PF10005.12\n",
            "Description: zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (93 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       944  (0.0777915); expected 242.7 (0.02)\n",
            "Passed bias filter:                      334  (0.0275237); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        25  (0.00206016); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 9328.12\n",
            "//\n",
            "Query:       Zincin_1  [M=97]\n",
            "Accession:   PF06262.14\n",
            "Description: Zincin-like metallopeptidase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (97 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       217  (0.0178822); expected 242.7 (0.02)\n",
            "Passed bias filter:                      179  (0.0147507); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        16  (0.0013185); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 10132.54\n",
            "//\n",
            "Query:       Zincin_2  [M=332]\n",
            "Accession:   PF10103.12\n",
            "Description: Zincin-like metallopeptidase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (332 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       501  (0.0412855); expected 242.7 (0.02)\n",
            "Passed bias filter:                      250  (0.0206016); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        14  (0.00115369); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.22u 0.01s 00:00:00.23 Elapsed: 00:00:00.13\n",
            "# Mc/sec: 16962.88\n",
            "//\n",
            "Query:       zinc_ribbon_10  [M=54]\n",
            "Accession:   PF10058.12\n",
            "Description: Predicted integral membrane zinc-ribbon metal-binding protein\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    7.8e-19   66.6   0.2    1.5e-18   65.7   0.2    1.5  1  XP_004361551.1  uncharacterized protein DFA_05834 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "      0.015   14.5   0.3       0.41    9.9   0.0    2.3  2  XP_004359231.1  Peptide methionine sulfoxide reductase [Caven\n",
            "      0.038   13.2   4.6       0.82    8.9   0.1    2.9  3  XP_004360352.1  RNA polymerase III [Cavenderia fasciculata]\n",
            "        0.3   10.3   4.5        6.7    6.0   0.5    3.0  3  XP_004359161.1  RNA polymerase III subunit [Cavenderia fascic\n",
            "          4    6.7   6.6         12    5.2   6.6    1.8  1  XP_004355145.1  myb domain-containing protein [Cavenderia fas\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004361551.1  uncharacterized protein DFA_05834 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   65.7   0.2   6.3e-22   1.5e-18       1      54 []     351     401 ..     351     401 .. 0.98\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 65.7 bits;  conditional E-value: 6.3e-22\n",
            "  zinc_ribbon_10   1 wfdrvldlLvGddetaeknryaLICskCrshnGlapkeeleevqyrCpnCgaln 54 \n",
            "                     w+d+v+d+L++d++   k+ ++LIC+kC+shnG++p+ e+++vq+rC +C+++n\n",
            "  XP_004361551.1 351 WLDKVVDYLISDGP---KYGSPLICKKCHSHNGYVPTGEISTVQFRCRFCQYFN 401\n",
            "                     9*************...*********************************9987 PP\n",
            "\n",
            ">> XP_004359231.1  Peptide methionine sulfoxide reductase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.6   0.0     0.032        78      35      52 ..      87     103 ..      81     105 .. 0.80\n",
            "   2 ?    9.9   0.0   0.00017      0.41      13      36 ..     134     157 ..     130     165 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.6 bits;  conditional E-value: 0.032\n",
            "  zinc_ribbon_10  35 apkeeleevqyrCpnCga 52 \n",
            "                     +  +++++  y C+ Cga\n",
            "  XP_004359231.1  87 Y-DKKFDDGVYGCAGCGA 103\n",
            "                     5.67788889******97 PP\n",
            "\n",
            "  == domain 2  score: 9.9 bits;  conditional E-value: 0.00017\n",
            "  zinc_ribbon_10  13 detaeknryaLICskCrshnGlap 36 \n",
            "                     d+++ + r  ++C++C  h G + \n",
            "  XP_004359231.1 134 DKSHGMVRTEIVCKNCGGHLGHVF 157\n",
            "                     5556677899***********885 PP\n",
            "\n",
            ">> XP_004360352.1  RNA polymerase III [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.0   0.2    0.0057        14      17      30 ..     192     205 ..     186     207 .. 0.83\n",
            "   2 ?    8.9   0.1   0.00034      0.82      19      38 ..     241     260 ..     225     268 .. 0.83\n",
            "   3 ?   -3.0   0.1       1.8   4.5e+03      37      48 ..     702     713 ..     700     714 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.0 bits;  conditional E-value: 0.0057\n",
            "  zinc_ribbon_10  17 eknryaLICskCrs 30 \n",
            "                      +n++ +IC++C +\n",
            "  XP_004360352.1 192 ITNILQMICKNCST 205\n",
            "                     68999*******87 PP\n",
            "\n",
            "  == domain 2  score: 8.9 bits;  conditional E-value: 0.00034\n",
            "  zinc_ribbon_10  19 nryaLICskCrshnGlapke 38 \n",
            "                     +r+  IC kC+s+nG ++k+\n",
            "  XP_004360352.1 241 CRKKKICPKCNSVNGPVKKV 260\n",
            "                     56666**********99743 PP\n",
            "\n",
            "  == domain 3  score: -3.0 bits;  conditional E-value: 1.8\n",
            "  zinc_ribbon_10  37 keeleevqyrCp 48 \n",
            "                     ++++++ +y Cp\n",
            "  XP_004360352.1 702 SRTYTKNTYMCP 713\n",
            "                     5689999****8 PP\n",
            "\n",
            ">> XP_004359161.1  RNA polymerase III subunit [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    3.3   0.1      0.02        48      40      51 ..       9      20 ..       2      32 .. 0.69\n",
            "   2 ?    1.5   0.0     0.071   1.7e+02      19      51 ..      55      65 ..      42      68 .. 0.52\n",
            "   3 ?    6.0   0.5    0.0028       6.7      35      52 ..      78      95 ..      73      97 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 3.3 bits;  conditional E-value: 0.02\n",
            "  zinc_ribbon_10 40 leevqyrCpnCg 51\n",
            "                      +++++C  C \n",
            "  XP_004359161.1  9 QGKLKFYCQTCP 20\n",
            "                    446677777774 PP\n",
            "\n",
            "  == domain 2  score: 1.5 bits;  conditional E-value: 0.071\n",
            "  zinc_ribbon_10 19 nryaLICskCrshnGlapkeeleevqyrCpnCg 51\n",
            "                    + + + C                      p C+\n",
            "  XP_004359161.1 55 QQVDIEC----------------------PTCK 65\n",
            "                    3444445......................5554 PP\n",
            "\n",
            "  == domain 3  score: 6.0 bits;  conditional E-value: 0.0028\n",
            "  zinc_ribbon_10 35 apkeeleevqyrCpnCga 52\n",
            "                    +p +e+++  y+C +C +\n",
            "  XP_004359161.1 78 YPADEPKTAYYKCTYCAT 95\n",
            "                    678899999*******75 PP\n",
            "\n",
            ">> XP_004355145.1  myb domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.2   6.6    0.0049        12      14      53 ..     580     619 ..     576     620 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.2 bits;  conditional E-value: 0.0049\n",
            "  zinc_ribbon_10  14 etaeknryaLICskCrshnGlapkeeleevqyrCpnCgal 53 \n",
            "                     ++ ++n +a+ C  C++ +  + +++ +  q++C +C+++\n",
            "  XP_004355145.1 580 NSVTTNDQAIACYSCAHLFHHVCVVSHSGQQWQCTYCNQI 619\n",
            "                     4568999************999999************986 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (54 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       597  (0.0491965); expected 242.7 (0.02)\n",
            "Passed bias filter:                      343  (0.0282653); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        35  (0.00288422); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         5  (0.000412031); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               5  [number of targets reported over threshold]\n",
            "# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 5633.93\n",
            "//\n",
            "Query:       zinc_ribbon_11  [M=128]\n",
            "Accession:   PF11682.11\n",
            "Description: Probable zinc-ribbon\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (128 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       490  (0.0403791); expected 242.7 (0.02)\n",
            "Passed bias filter:                      267  (0.0220025); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        15  (0.00123609); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 11765.59\n",
            "//\n",
            "Query:       zinc_ribbon_12  [M=45]\n",
            "Accession:   PF11331.11\n",
            "Description: Probable zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.056   12.7   0.1       0.12   11.7   0.1    1.5  1  XP_004358572.1  countin2 [Cavenderia fasciculata]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004358572.1  countin2 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.7   0.1   9.7e-06      0.12       3      15 ..      48      60 ..      47      62 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.7 bits;  conditional E-value: 9.7e-06\n",
            "  zinc_ribbon_12  3 GAPFvvCsnCfel 15\n",
            "                    GAPF++C+ C+ +\n",
            "  XP_004358572.1 48 GAPFTTCNDCVSI 60\n",
            "                    9*********976 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (45 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       497  (0.0409559); expected 242.7 (0.02)\n",
            "Passed bias filter:                      325  (0.026782); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        28  (0.00230738); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 5005.11\n",
            "//\n",
            "Query:       zinc_ribbon_13  [M=63]\n",
            "Accession:   PF09855.12\n",
            "Description: Nucleic-acid-binding protein containing Zn-ribbon domain (DUF2082)\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "     0.0074   15.9   0.4     0.0074   15.9   0.4    1.9  2  XP_004357594.1  RNA polymerase [Cavenderia fasciculata]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004357594.1  RNA polymerase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.9   0.2      0.11   1.3e+03       2       8 ..       7      13 ..       6      20 .. 0.67\n",
            "   2 !   15.9   0.4   6.1e-07    0.0074      36      57 ..      22      43 ..      20      50 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.9 bits;  conditional E-value: 0.11\n",
            "  zinc_ribbon_13  2 CpKCgnt 8 \n",
            "                    C  Cg++\n",
            "  XP_004357594.1  7 CGECGSE 13\n",
            "                    6667765 PP\n",
            "\n",
            "  == domain 2  score: 15.9 bits;  conditional E-value: 6.1e-07\n",
            "  zinc_ribbon_13 36 vsCknCgYtelYkketsegkni 57\n",
            "                    + Ck+C+Y  +Ykk+t + +ni\n",
            "  XP_004357594.1 22 IKCKDCTYRIMYKKRTDKSMNI 43\n",
            "                    79******************98 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (63 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       474  (0.0390606); expected 242.7 (0.02)\n",
            "Passed bias filter:                      259  (0.0213432); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        13  (0.00107128); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 5740.90\n",
            "//\n",
            "Query:       zinc_ribbon_15  [M=86]\n",
            "Accession:   PF17032.8\n",
            "Description: zinc-ribbon family\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    3.1e-09   36.9   0.9    3.1e-09   36.9   0.9    1.6  2  XP_004360973.1  uncharacterized protein DFA_05252 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "      0.025   14.8   3.7      0.056   13.7   3.7    1.7  1  XP_004351727.1  uncharacterized protein DFA_09827 [Cavenderia\n",
            "      0.042   14.1   0.6      0.042   14.1   0.6    2.1  2  XP_004359819.1  uncharacterized protein DFA_01854 [Cavenderia\n",
            "       0.65   10.3   4.4        3.1    8.1   1.9    2.3  2  XP_004359231.1  Peptide methionine sulfoxide reductase [Caven\n",
            "        2.6    8.3   9.2       0.32   11.2   0.6    3.5  4  XP_004361876.1  uncharacterized protein DFA_06163 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004360973.1  uncharacterized protein DFA_05252 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   36.9   0.9   1.3e-12   3.1e-09       2      51 ..      24      72 ..      23      95 .. 0.89\n",
            "   2 ?   -2.5   0.1       2.5   6.1e+03      39      51 ..     139     151 ..     122     167 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 36.9 bits;  conditional E-value: 1.3e-12\n",
            "  zinc_ribbon_15  2 lCprCgklskaelvkkkkvlklffipvfkwkkkkpylacskCkslleask 51\n",
            "                    +Cp+C++  +++l +++++ +lffip++k+kk +p+l+c  C+ +++  +\n",
            "  XP_004360973.1 24 VCPNCHN-RSVQLERNDSWAHLFFIPFCKVKKGSPFLHCAACGCTVPYVE 72\n",
            "                    7*****6.899********************************9988765 PP\n",
            "\n",
            "  == domain 2  score: -2.5 bits;  conditional E-value: 2.5\n",
            "  zinc_ribbon_15  39 acskCkslleask 51 \n",
            "                       ++ ++++e+++\n",
            "  XP_004360973.1 139 NYNDQSTFYESNN 151\n",
            "                     3333334444444 PP\n",
            "\n",
            ">> XP_004351727.1  uncharacterized protein DFA_09827 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.7   3.7   2.3e-05     0.056      64      85 ..      43      87 ..       5      88 .. 0.64\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.7 bits;  conditional E-value: 2.3e-05\n",
            "  zinc_ribbon_15 64 e.......................kkeCenCgkeveeefkyCpkC 85\n",
            "                                            +++C++C+ +ve++  +C+ C\n",
            "  XP_004351727.1 43 LfinshkqeytfkyststtnhkhhQRSCWSCKMNVEPKEFFCSIC 87\n",
            "                    244444444445555555566667********************9 PP\n",
            "\n",
            ">> XP_004359819.1  uncharacterized protein DFA_01854 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.7   0.4       0.7   1.7e+03      64      65 ..     120     145 ..      98     165 .. 0.49\n",
            "   2 ?   14.1   0.6   1.7e-05     0.042      64      86 .]     261     294 ..     232     294 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.7 bits;  conditional E-value: 0.7\n",
            "  zinc_ribbon_15  64 e........................k 65 \n",
            "                     +                        +\n",
            "  XP_004359819.1 120 KekkdqdkekggnwftkfmdrleseS 145\n",
            "                     11112233345555555555554441 PP\n",
            "\n",
            "  == domain 2  score: 14.1 bits;  conditional E-value: 1.7e-05\n",
            "  zinc_ribbon_15  64 e.........kkeCenCgkev..eeefkyCpkCg 86 \n",
            "                     +         k  C +Cg++v  +++ ++C++Cg\n",
            "  XP_004359819.1 261 TsyhstghsgKSFCTKCGTKVpvSSSASFCSSCG 294\n",
            "                     245556667799******9872256799*****9 PP\n",
            "\n",
            ">> XP_004359231.1  Peptide methionine sulfoxide reductase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.1   1.9    0.0013       3.1      14      86 .]       7     117 ..       4     117 .. 0.81\n",
            "   2 ?    2.3   0.0     0.082     2e+02      65      74 ..     143     152 ..     128     173 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.1 bits;  conditional E-value: 0.0013\n",
            "  zinc_ribbon_15  14 lvkkkkvlklffipvfkwkkkkpylacskCkslleaskelgkeiekgesse......................................kkeCen 70 \n",
            "                     ++ +++  +lf ++ fk+++++++l +++  s +++ ++++++ ++++  +                                      ++ C+ \n",
            "  XP_004359231.1   7 YKSNTSQTSLFSVEQFKYNNNNKVLLSQQTYSYYSTTSNNNNNNNNTTM-NlsdsewkvkltpeqfkvlrekgteraftgeydkkfddgVYGCAG 100\n",
            "                     567788899***************9999999999988744444443333.358899999999999999999999999999999999999999999 PP\n",
            "\n",
            "  zinc_ribbon_15  71 Cgkeveee.fkyCpkCg 86 \n",
            "                     Cg+ + ++ +k+ + Cg\n",
            "  XP_004359231.1 101 CGAPLYSSkSKFNSGCG 117\n",
            "                     99976544378877776 PP\n",
            "\n",
            "  == domain 2  score: 2.3 bits;  conditional E-value: 0.082\n",
            "  zinc_ribbon_15  65 kkeCenCgke 74 \n",
            "                     + +C+nCg +\n",
            "  XP_004359231.1 143 EIVCKNCGGH 152\n",
            "                     5556666544 PP\n",
            "\n",
            ">> XP_004361876.1  uncharacterized protein DFA_06163 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.2   0.4       0.5   1.2e+03      64      72 ..     201     217 ..     178     228 .. 0.61\n",
            "   2 ?   11.2   0.6   0.00013      0.32      64      86 .]     371     405 ..     330     405 .. 0.75\n",
            "   3 ?   -1.5   0.1       1.3   3.1e+03      54      75 ..     962     983 ..     942     988 .. 0.67\n",
            "   4 ?   -2.2   0.0         2     5e+03      25      50 ..    1377    1402 ..    1372    1408 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.2 bits;  conditional E-value: 0.5\n",
            "  zinc_ribbon_15  64 e........kkeCenCg 72 \n",
            "                     +        +k+C++C+\n",
            "  XP_004361876.1 201 KdfcvscgtQKSCNKCD 217\n",
            "                     13444455556666665 PP\n",
            "\n",
            "  == domain 2  score: 11.2 bits;  conditional E-value: 0.00013\n",
            "  zinc_ribbon_15  64 e............kkeCenCgkeveeefkyCpkCg 86 \n",
            "                     +            ++eC +C+++ ++  kyC++Cg\n",
            "  XP_004361876.1 371 NakstfggvvvstTNECIHCKHSNSKLAKYCTSCG 405\n",
            "                     044456667777799*******************9 PP\n",
            "\n",
            "  == domain 3  score: -1.5 bits;  conditional E-value: 1.3\n",
            "  zinc_ribbon_15  54 gkeiekgessekkeCenCgkev 75 \n",
            "                     +kei ++ ++ +++C +C+ ++\n",
            "  XP_004361876.1 962 KKEIPEHLNRYSHVCYQCKVSI 983\n",
            "                     3444444444499999999876 PP\n",
            "\n",
            "  == domain 4  score: -2.2 bits;  conditional E-value: 2\n",
            "  zinc_ribbon_15   25 fipvfkwkkkkpylacskCkslleas 50  \n",
            "                      f p+ +++ +k+++++ +C++l++  \n",
            "  XP_004361876.1 1377 FAPMSRTDVEKYMATTGRCSALVKVT 1402\n",
            "                      6788888999999******9998765 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (86 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1132  (0.0932839); expected 242.7 (0.02)\n",
            "Passed bias filter:                      523  (0.0430985); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        67  (0.00552122); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000576844); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               5  [number of targets reported over threshold]\n",
            "# CPU time: 0.16u 0.00s 00:00:00.16 Elapsed: 00:00:00.09\n",
            "# Mc/sec: 6174.88\n",
            "//\n",
            "Query:       zinc_ribbon_16  [M=125]\n",
            "Accession:   PF17034.8\n",
            "Description: Zinc-ribbon like family\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    1.1e-21   76.8  19.8    2.6e-21   75.5  19.8    1.7  1  XP_004361154.1  uncharacterized protein DFA_05435 [Cavenderia\n",
            "    6.1e-06   25.9  20.5      0.013   15.1  20.5    3.7  1  XP_004362142.1  uncharacterized protein DFA_06441 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004361154.1  uncharacterized protein DFA_05435 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   75.5  19.8   4.4e-25   2.6e-21       1     123 [.     924    1038 ..     924    1040 .] 0.88\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 75.5 bits;  conditional E-value: 4.4e-25\n",
            "  zinc_ribbon_16    1 cnfckksitpfsakektnilqaavssaGgakfldksksslcprcskplpkcavcglslGtsdvtnkdskrksvvkevssskkrfekwlsfcls 93  \n",
            "                      c +c+ s+ +f +        + v ++  ak   + k   cp+c+++lp+c++c + +    v+  d k+ +  +e  +  + f++w+++c +\n",
            "  XP_004361154.1  924 CGYCQNSF-SFESVTSG----SMVGRN--AKINARAKVPCCPHCKQQLPRCCLCMIQMNCM-VPSMDFKQSNKQQEWTEGAESFDEWFTWCQT 1008\n",
            "                      88888888.77666653....333332..3666678899******************9998.******************************* PP\n",
            "\n",
            "  zinc_ribbon_16   94 cghgahashakewfsthsicpvadcnceck 123 \n",
            "                      c hg ha+h ++wf+ h  cpv+dcnc+c \n",
            "  XP_004361154.1 1009 CRHGGHATHLSDWFKDHIECPVTDCNCKCS 1038\n",
            "                      *****************************6 PP\n",
            "\n",
            ">> XP_004362142.1  uncharacterized protein DFA_06441 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.1  20.5   2.2e-06     0.013      44     123 ..    1413    1478 ..    1396    1480 .. 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.1 bits;  conditional E-value: 2.2e-06\n",
            "  zinc_ribbon_16   44 cskplpkcavcglslGtsdvtnkdskrksvvkev..ssskkrfekwlsfclscghgahashakewfsthsicpvadcnceck 123 \n",
            "                      c++ l + +vc               ++  v+ v  s  +  +    sfc+scghg h++h k+wf + s cp   c+c+c \n",
            "  XP_004362142.1 1413 CQRTLNNNCVC---------------PNCKVYAVkcSVCHISVRGLSSFCISCGHGGHTDHLKQWFGKRSDCPT-GCKCKCV 1478\n",
            "                      44444444444...............2233333300344444556789*************************6.6999996 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (125 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       947  (0.0780387); expected 242.7 (0.02)\n",
            "Passed bias filter:                      382  (0.0314792); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        29  (0.00238978); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.14u 0.00s 00:00:00.14 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 9896.44\n",
            "//\n",
            "Query:       zinc_ribbon_2  [M=23]\n",
            "Accession:   PF13240.9\n",
            "Description: zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    0.00021   20.4   3.6      0.001   18.2   3.6    2.1  2  XP_004359819.1  uncharacterized protein DFA_01854 [Cavenderia\n",
            "     0.0047   16.1   6.7     0.0096   15.1   6.7    1.5  1  XP_004357883.1  uncharacterized protein DFA_00167 [Cavenderia\n",
            "     0.0075   15.5   0.6      0.018   14.2   0.6    1.7  1  XP_004352445.1  WD40 repeat-containing protein [Cavenderia fa\n",
            "  ------ inclusion threshold ------\n",
            "      0.027   13.7   3.0      0.053   12.8   3.0    1.5  1  XP_004357569.1  uncharacterized protein DFA_02085 [Cavenderia\n",
            "      0.084   12.1   3.7          3    7.2   0.4    3.0  3  XP_004358406.1  ubiquitin [Cavenderia fasciculata]\n",
            "       0.11   11.8   4.7       0.57    9.5   4.7    2.1  1  XP_004359013.1  uncharacterized protein DFA_01038 [Cavenderia\n",
            "       0.37   10.1   1.6       0.82    9.0   1.6    1.6  1  XP_004351179.1  pleckstrin domain-containing protein [Cavende\n",
            "       0.68    9.2   7.4        2.1    7.7   7.4    1.8  1  XP_004361350.1  uncharacterized protein DFA_05632 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004359819.1  uncharacterized protein DFA_01854 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   11.0   0.7   0.00013      0.19      15      22 ..     273     280 ..     270     281 .. 0.73\n",
            "   2 !   18.2   3.6   6.9e-07     0.001       1      21 [.     273     295 ..     273     297 .] 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.0 bits;  conditional E-value: 0.00013\n",
            "   zinc_ribbon_2  15 FCpkCGtp 22 \n",
            "                     FC+kCGt+\n",
            "  XP_004359819.1 273 FCTKCGTK 280\n",
            "                     88888887 PP\n",
            "\n",
            "  == domain 2  score: 18.2 bits;  conditional E-value: 6.9e-07\n",
            "   zinc_ribbon_2   1 fCpnCGaei.pd.nakFCpkCGt 21 \n",
            "                     fC++CG+++     a FC+ CG \n",
            "  XP_004359819.1 273 FCTKCGTKVpVSsSASFCSSCGD 295\n",
            "                     9********7446*********5 PP\n",
            "\n",
            ">> XP_004357883.1  uncharacterized protein DFA_00167 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.1   6.7   6.3e-06    0.0096       2      19 ..       4      21 ..       3      21 .. 0.98\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.1 bits;  conditional E-value: 6.3e-06\n",
            "   zinc_ribbon_2  2 CpnCGaeipdnakFCpkC 19\n",
            "                    Cp+C +e +++ak+C+kC\n",
            "  XP_004357883.1  4 CPECQTEYEKEAKYCFKC 21\n",
            "                    ****************** PP\n",
            "\n",
            ">> XP_004352445.1  WD40 repeat-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   14.2   0.6   1.2e-05     0.018       5      21 ..     770     786 ..     769     787 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.2 bits;  conditional E-value: 1.2e-05\n",
            "   zinc_ribbon_2   5 CGaeipdnakFCpkCGt 21 \n",
            "                     CG+ i ++a  C kCGt\n",
            "  XP_004352445.1 770 CGKGILPGAIVCEKCGT 786\n",
            "                     ****************9 PP\n",
            "\n",
            ">> XP_004357569.1  uncharacterized protein DFA_02085 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.8   3.0   3.5e-05     0.053       2      23 .]     331     353 ..     330     353 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.8 bits;  conditional E-value: 3.5e-05\n",
            "   zinc_ribbon_2   2 CpnCGaeipd.nakFCpkCGtpl 23 \n",
            "                     C  C +e +   a+FC  CGt+l\n",
            "  XP_004357569.1 331 CGSCRTEAKNlAANFCASCGTKL 353\n",
            "                     999999998889*********97 PP\n",
            "\n",
            ">> XP_004358406.1  ubiquitin [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.1   0.0      0.83   1.3e+03       6      13 ..      53      60 ..      53      62 .. 0.85\n",
            "   2 ?    7.2   0.4     0.002         3      16      21 ..     128     133 ..     126     135 .. 0.58\n",
            "   3 ?    4.6   0.1     0.013        20      15      21 ..     147     153 ..     142     160 .. 0.71\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.1 bits;  conditional E-value: 0.83\n",
            "   zinc_ribbon_2  6 Gaeipdna 13\n",
            "                    G++++d+ \n",
            "  XP_004358406.1 53 GKQLEDGR 60\n",
            "                    9*****95 PP\n",
            "\n",
            "  == domain 2  score: 7.2 bits;  conditional E-value: 0.002\n",
            "   zinc_ribbon_2  16 CpkCGt 21 \n",
            "                     C++CG+\n",
            "  XP_004358406.1 128 CTNCGA 133\n",
            "                     555555 PP\n",
            "\n",
            "  == domain 3  score: 4.6 bits;  conditional E-value: 0.013\n",
            "   zinc_ribbon_2  15 FCpkCGt 21 \n",
            "                     +C kC t\n",
            "  XP_004358406.1 147 YCGKCHT 153\n",
            "                     6666665 PP\n",
            "\n",
            ">> XP_004359013.1  uncharacterized protein DFA_01038 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.5   4.7   0.00038      0.57       1      21 [.     227     248 ..     227     248 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.5 bits;  conditional E-value: 0.00038\n",
            "   zinc_ribbon_2   1 fCpnCGa.eipdnakFCpkCGt 21 \n",
            "                     +Cp CG+ ++  n ++C +CG+\n",
            "  XP_004359013.1 227 VCPSCGTdKNAGNTRYCIRCGA 248\n",
            "                     5*****934566*********7 PP\n",
            "\n",
            ">> XP_004351179.1  pleckstrin domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.0   1.6   0.00054      0.82       2      23 .]       5      26 ..       5      26 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.0 bits;  conditional E-value: 0.00054\n",
            "   zinc_ribbon_2  2 CpnCGaeipdnakFCpkCGtpl 23\n",
            "                    C  C + +  +a+FC  CG  l\n",
            "  XP_004351179.1  5 CGSCAKVNAVTARFCVSCGNSL 26\n",
            "                    999****************875 PP\n",
            "\n",
            ">> XP_004361350.1  uncharacterized protein DFA_05632 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.7   7.4    0.0014       2.1       2      23 .]     387     407 ..     386     407 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.7 bits;  conditional E-value: 0.0014\n",
            "   zinc_ribbon_2   2 CpnCGaeipdnakFCpkCGtpl 23 \n",
            "                     C+ CG  +    +FC++CGt+l\n",
            "  XP_004361350.1 387 CTTCGQ-VNGKTPFCFNCGTKL 407\n",
            "                     899998.67788********97 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (23 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       658  (0.0542233); expected 242.7 (0.02)\n",
            "Passed bias filter:                      378  (0.0311496); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        55  (0.00453234); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        16  (0.0013185); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               8  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.01s 00:00:00.12 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 2404.67\n",
            "//\n",
            "Query:       zinc_ribbon_4  [M=36]\n",
            "Accession:   PF13717.9\n",
            "Description: zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    0.00019   20.7   2.7     0.0015   17.9   0.3    2.7  2  XP_004362339.1  uncharacterized protein DFA_02731 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "       0.46    9.9   4.8        6.5    6.2   0.1    2.8  2  XP_004354139.1  uncharacterized protein DFA_10231 [Cavenderia\n",
            "       0.83    9.1   6.1        7.4    6.0   0.4    3.1  3  XP_004359161.1  RNA polymerase III subunit [Cavenderia fascic\n",
            "          3    7.3   5.6       0.17   11.3   0.4    1.6  2  XP_004357883.1  uncharacterized protein DFA_00167 [Cavenderia\n",
            "        3.9    6.9   7.7        4.5    6.7   0.1    3.2  2  XP_004360282.1  FVYE domain-containing protein [Cavenderia fa\n",
            "        7.1    6.1   7.9        1.1    8.7   2.0    2.3  2  XP_004350066.1  uncharacterized protein DFA_11123 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004362339.1  uncharacterized protein DFA_02731 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.9   0.1     0.072   1.5e+02      25      32 ..     251     258 ..     245     261 .. 0.71\n",
            "   2 !   17.9   0.3   7.4e-07    0.0015       3      30 ..     291     318 ..     290     322 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.9 bits;  conditional E-value: 0.072\n",
            "   zinc_ribbon_4  25 kvrCskCg 32 \n",
            "                     ++rC+ C \n",
            "  XP_004362339.1 251 SIRCPTCV 258\n",
            "                     67888875 PP\n",
            "\n",
            "  == domain 2  score: 17.9 bits;  conditional E-value: 7.4e-07\n",
            "   zinc_ribbon_4   3 itCpsCkakyriddeklpakgvkvrCsk 30 \n",
            "                     itCp+C+++y +     + +g++v+Cs+\n",
            "  XP_004362339.1 291 ITCPHCNSRYVLGSQYAQYHGMSVKCSN 318\n",
            "                     9*************99**********97 PP\n",
            "\n",
            ">> XP_004354139.1  uncharacterized protein DFA_10231 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.2   0.1    0.0032       6.5      21      35 ..     446     460 ..     441     461 .. 0.88\n",
            "   2 ?    3.7   1.7      0.02        41      21      33 ..     474     486 ..     470     487 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.2 bits;  conditional E-value: 0.0032\n",
            "   zinc_ribbon_4  21 akgvkvrCskCghvF 35 \n",
            "                     ++++ ++C+ C+++F\n",
            "  XP_004354139.1 446 PNRQGIQCTTCQESF 460\n",
            "                     678889******999 PP\n",
            "\n",
            "  == domain 2  score: 3.7 bits;  conditional E-value: 0.02\n",
            "   zinc_ribbon_4  21 akgvkvrCskCgh 33 \n",
            "                     +k ++v+C+ C+ \n",
            "  XP_004354139.1 474 DKDGMVKCPVCQR 486\n",
            "                     56789******95 PP\n",
            "\n",
            ">> XP_004359161.1  RNA polymerase III subunit [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.0   0.1      0.14   2.7e+02       3      16 ..      14      27 ..       9      34 .. 0.69\n",
            "   2 ?    6.0   0.4    0.0037       7.4       2      12 ..      58      68 ..      58      69 .. 0.81\n",
            "   3 ?    3.7   0.1     0.019        39      27      36 .]      89      98 ..      82      98 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.0 bits;  conditional E-value: 0.14\n",
            "   zinc_ribbon_4  3 itCpsCkakyridd 16\n",
            "                    + C+ C   + i++\n",
            "  XP_004359161.1 14 FYCQTCPYFFGINN 27\n",
            "                    56777776666665 PP\n",
            "\n",
            "  == domain 2  score: 6.0 bits;  conditional E-value: 0.0037\n",
            "   zinc_ribbon_4  2 kitCpsCkaky 12\n",
            "                    +i+Cp Ck ++\n",
            "  XP_004359161.1 58 DIECPTCKVRR 68\n",
            "                    59*****9765 PP\n",
            "\n",
            "  == domain 3  score: 3.7 bits;  conditional E-value: 0.019\n",
            "   zinc_ribbon_4 27 rCskCghvFr 36\n",
            "                    +C+ C + +r\n",
            "  XP_004359161.1 89 KCTYCATRWR 98\n",
            "                    9******996 PP\n",
            "\n",
            ">> XP_004357883.1  uncharacterized protein DFA_00167 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.3   0.4   8.6e-05      0.17       3      13 ..       2      12 ..       1      14 [. 0.92\n",
            "   2 ?   -3.9   0.7       4.8   9.7e+03      28      35 ..      18      25 ..      18      26 .. 0.56\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.3 bits;  conditional E-value: 8.6e-05\n",
            "   zinc_ribbon_4  3 itCpsCkakyr 13\n",
            "                    itCp+C+++y+\n",
            "  XP_004357883.1  2 ITCPECQTEYE 12\n",
            "                    9*********6 PP\n",
            "\n",
            "  == domain 2  score: -3.9 bits;  conditional E-value: 4.8\n",
            "   zinc_ribbon_4 28 CskCghvF 35\n",
            "                    C kC+  F\n",
            "  XP_004357883.1 18 CFKCRIAF 25\n",
            "                    66776544 PP\n",
            "\n",
            ">> XP_004360282.1  FVYE domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.7   3.3      0.17   3.5e+02       3      14 ..      13      24 ..      12      38 .. 0.76\n",
            "   2 ?    6.7   0.1    0.0022       4.5       3      35 ..      37      69 ..      36      70 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.7 bits;  conditional E-value: 0.17\n",
            "   zinc_ribbon_4  3 itCpsCkakyri 14\n",
            "                    i+C  C++ ++i\n",
            "  XP_004360282.1 13 IECTGCQSPFSI 24\n",
            "                    889999998887 PP\n",
            "\n",
            "  == domain 2  score: 6.7 bits;  conditional E-value: 0.0022\n",
            "   zinc_ribbon_4  3 itCpsCkakyriddeklpakgvkvrCskCghvF 35\n",
            "                    i C  C+  y++ ++ l  +g+   C  C  +F\n",
            "  XP_004360282.1 37 IFCDPCSNHYTVLPAELGYSGAQRLCRVCHSSF 69\n",
            "                    67***************************9998 PP\n",
            "\n",
            ">> XP_004350066.1  uncharacterized protein DFA_11123 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.5   0.4      0.41   8.2e+02       4      10 ..     741     747 ..     740     748 .. 0.77\n",
            "   2 ?    8.7   2.0   0.00054       1.1       4      33 ..     780     806 ..     779     807 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.5 bits;  conditional E-value: 0.41\n",
            "   zinc_ribbon_4   4 tCpsCka 10 \n",
            "                     +Cp Ck+\n",
            "  XP_004350066.1 741 VCPTCKT 747\n",
            "                     5899987 PP\n",
            "\n",
            "  == domain 2  score: 8.7 bits;  conditional E-value: 0.00054\n",
            "   zinc_ribbon_4   4 tCpsCkakyriddek.lpakgvkvrCskCgh 33 \n",
            "                     +C  C+a+y+i+    +p    k +C+ C+h\n",
            "  XP_004350066.1 780 QCNFCQATYSISQPDiIP----KSTCPSCQH 806\n",
            "                     7***********954344....55******9 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (36 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       590  (0.0486197); expected 242.7 (0.02)\n",
            "Passed bias filter:                      343  (0.0282653); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        52  (0.00428513); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         9  (0.000741656); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               6  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 3934.18\n",
            "//\n",
            "Query:       zinc_ribbon_5  [M=37]\n",
            "Accession:   PF13719.9\n",
            "Description: zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.014   14.6   0.9      0.035   13.3   0.9    1.6  1  XP_004357678.1  uncharacterized protein DFA_02194 [Cavenderia\n",
            "       0.14   11.4   5.7      0.033   13.4   0.8    2.7  2  XP_004351843.1  uncharacterized protein DFA_09947 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004357678.1  uncharacterized protein DFA_02194 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.3   0.9   5.8e-06     0.035      14      32 ..     712     730 ..     708     732 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.3 bits;  conditional E-value: 5.8e-06\n",
            "   zinc_ribbon_5  14 lvdsklKksgkkVrCakCa 32 \n",
            "                     l+++ +K  + kV CakC \n",
            "  XP_004357678.1 712 LDTDLIKIKQHKVVCAKCK 730\n",
            "                     89****************6 PP\n",
            "\n",
            ">> XP_004351843.1  uncharacterized protein DFA_09947 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.4   0.8   5.4e-06     0.033       3      32 ..     346     374 ..     344     375 .. 0.87\n",
            "   2 ?   -3.1   0.4      0.81   4.9e+03      27      34 ..     482     489 ..     477     489 .. 0.68\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.4 bits;  conditional E-value: 5.4e-06\n",
            "   zinc_ribbon_5   3 irCpdCdtkFklvdsklKksgkkVrCakCa 32 \n",
            "                     i Cp+C++  k++  ++  +++ + C kC \n",
            "  XP_004351843.1 346 IPCPQCESEIKVDRLEIDRNQQ-ILCVKCN 374\n",
            "                     78***************99996.6788885 PP\n",
            "\n",
            "  == domain 2  score: -3.1 bits;  conditional E-value: 0.81\n",
            "   zinc_ribbon_5  27 rCakCaev 34 \n",
            "                     +C  C++v\n",
            "  XP_004351843.1 482 KCKICGSV 489\n",
            "                     67777776 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (37 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       501  (0.0412855); expected 242.7 (0.02)\n",
            "Passed bias filter:                      324  (0.0266996); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        21  (0.00173053); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 4393.69\n",
            "//\n",
            "Query:       zinc_ribbon_6  [M=59]\n",
            "Accession:   PF14599.9\n",
            "Description: Zinc-ribbon\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    4.4e-21   74.1   4.5    4.4e-21   74.1   4.5    4.5  4  XP_004354944.1  uncharacterized protein DFA_09088 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004354944.1  uncharacterized protein DFA_09088 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -3.2   6.8      0.49     6e+03      27      56 ..      59      95 ..      50      98 .. 0.66\n",
            "   2 ?   -6.9   8.5         1   1.2e+04      31      54 ..     105     127 ..      97     132 .. 0.68\n",
            "   3 ?   -2.7   0.6      0.33     4e+03      29      35 ..     185     191 ..     163     195 .. 0.56\n",
            "   4 !   74.1   4.5   3.6e-25   4.4e-21       2      59 .]     197     255 ..     196     255 .. 0.97\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -3.2 bits;  conditional E-value: 0.49\n",
            "   zinc_ribbon_6 27 kvkilCnDCekksevkfhvlglk.......CaeCgSY 56\n",
            "                    + +i+C  C++ + v+ + +g         Ca+C+ Y\n",
            "  XP_004354944.1 59 TKEIMCMKCQTVQPVAEKCIGCDiqfaryfCAHCKFY 95\n",
            "                    6689999999887776666654322333337777766 PP\n",
            "\n",
            "  == domain 2  score: -6.9 bits;  conditional E-value: 1\n",
            "   zinc_ribbon_6  31 lCnDCekksevkfhvlglkCaeCg 54 \n",
            "                     +C DC+  ++v  +   + C++C+\n",
            "  XP_004354944.1 105 HCDDCK-ICRVGERDSFIHCKRCN 127\n",
            "                     699997.45555555566666665 PP\n",
            "\n",
            "  == domain 3  score: -2.7 bits;  conditional E-value: 0.33\n",
            "   zinc_ribbon_6  29 kilCnDC 35 \n",
            "                     +i C  C\n",
            "  XP_004354944.1 185 TIQCPMC 191\n",
            "                     3334333 PP\n",
            "\n",
            "  == domain 4  score: 74.1 bits;  conditional E-value: 3.6e-25\n",
            "   zinc_ribbon_6   2 dmeayfraldaeiaeqpmpeeyknkkvkilCnDCekkse.vkfhvlglkCaeCgSYNTr 59 \n",
            "                     ++++y+ ++da+ia+qpmp+ y+ +++++lC DC+kks+ ++ h+lg+kC++C+SYNT+\n",
            "  XP_004354944.1 197 NLTEYWLRIDAQIARQPMPRMYAGSTCSVLCCDCDKKSMdIALHFLGNKCQHCNSYNTT 255\n",
            "                     7899**********************************99******************5 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (59 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       489  (0.0402967); expected 242.7 (0.02)\n",
            "Passed bias filter:                      213  (0.0175525); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        23  (0.00189534); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 7156.22\n",
            "//\n",
            "Query:       zinc_ribbon_9  [M=35]\n",
            "Accession:   PF14369.9\n",
            "Description: zinc-ribbon\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    3.7e-09   36.1   3.4    3.7e-09   36.1   3.4    2.2  2  XP_004366460.1  uncharacterized protein DFA_04050 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "       0.01   15.5   1.9       0.01   15.5   1.9    2.3  2  XP_004357039.1  RING zinc finger-containing protein [Cavender\n",
            "      0.064   13.0   0.4       0.13   12.0   0.4    1.5  1  XP_004362968.1  putative GATA-binding transcription factor [C\n",
            "       0.23   11.2   7.0       0.89    9.3   2.8    2.9  2  XP_004362377.1  uncharacterized protein DFA_02769 [Cavenderia\n",
            "       0.84    9.4   5.6         15    5.4   0.2    3.7  4  XP_004359874.1  uncharacterized protein DFA_01912 [Cavenderia\n",
            "        1.5    8.6   6.6       0.14   11.9   0.3    2.7  3  XP_004359316.1  6-phosphogluconate dehydrogenase [Cavenderia \n",
            "        1.6    8.5   5.2         20    5.0   0.1    3.1  3  XP_004358905.1  RNA polymerase I subunit [Cavenderia fascicul\n",
            "        2.9    7.7   8.0         21    5.0   0.1    3.7  4  XP_004355784.1  heat shock protein DnaJ family protein [Caven\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004366460.1  uncharacterized protein DFA_04050 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   36.1   3.4   2.4e-12   3.7e-09       3      34 ..      87     116 ..      85     117 .. 0.91\n",
            "   2 ?   -2.1   0.2       2.2   3.3e+03      22      28 ..     429     434 ..     424     435 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 36.1 bits;  conditional E-value: 2.4e-12\n",
            "   zinc_ribbon_9   3 yWCHqCdrfVsirpdedsdvlCPeCgggFvEe 34 \n",
            "                     yWCH+C++ ++++ + d+ v CP C + F+Ee\n",
            "  XP_004366460.1  87 YWCHHCKKYIQVGNP-DELV-CPTCESEFIEE 116\n",
            "                     ************865.8889.**********9 PP\n",
            "\n",
            "  == domain 2  score: -2.1 bits;  conditional E-value: 2.2\n",
            "   zinc_ribbon_9  22 vlCPeCg 28 \n",
            "                     + CP C+\n",
            "  XP_004366460.1 429 S-CPVCR 434\n",
            "                     4.**996 PP\n",
            "\n",
            ">> XP_004357039.1  RING zinc finger-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.3   0.1       1.2   1.9e+03      16      28 ..      59      70 ..      58      71 .. 0.78\n",
            "   2 ?   15.5   1.9   6.8e-06      0.01       3      33 ..      85     110 ..      84     112 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.3 bits;  conditional E-value: 1.2\n",
            "   zinc_ribbon_9 16 pdedsdvlCPeCg 28\n",
            "                    ++e++d+ C  C \n",
            "  XP_004357039.1 59 EEEEEDK-CSICL 70\n",
            "                    4689999.99996 PP\n",
            "\n",
            "  == domain 2  score: 15.5 bits;  conditional E-value: 6.8e-06\n",
            "   zinc_ribbon_9   3 yWCHqCdrfVsirpdedsdvlCPeCgggFvE 33 \n",
            "                     ++C++C     +++++ s + CP C+++F+ \n",
            "  XP_004357039.1  85 HFCYHCI----LQWSDLSLK-CPLCKSQFLS 110\n",
            "                     9******....66779999*.********96 PP\n",
            "\n",
            ">> XP_004362968.1  putative GATA-binding transcription factor [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.0   0.4   8.7e-05      0.13       2      31 ..     666     698 ..     665     699 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.0 bits;  conditional E-value: 8.7e-05\n",
            "   zinc_ribbon_9   2 ryWCHqCdrfVsirpdedsd...vlCPeCgggF 31 \n",
            "                     +++CH C    +++++  +d   +lC+ Cg +F\n",
            "  XP_004362968.1 666 TLFCHTCGTKSTPEWRRGPDgpaTLCNACGLAF 698\n",
            "                     79*******9999999888886449****9887 PP\n",
            "\n",
            ">> XP_004362377.1  uncharacterized protein DFA_02769 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.7   0.1     0.071   1.1e+02       2      18 ..      61      76 ..      60      81 .. 0.73\n",
            "   2 ?    9.3   2.8   0.00059      0.89       3      31 ..     190     213 ..     188     216 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.7 bits;  conditional E-value: 0.071\n",
            "   zinc_ribbon_9  2 ryWCHqCdrfVsirpde 18\n",
            "                    +y+C++ d +++i+ ++\n",
            "  XP_004362377.1 61 SYYCYDQD-SIRITQSN 76\n",
            "                    8****955.77777665 PP\n",
            "\n",
            "  == domain 2  score: 9.3 bits;  conditional E-value: 0.00059\n",
            "   zinc_ribbon_9   3 yWCHqCdrfVsirpdedsdvlCPeCgggF 31 \n",
            "                      +C +C  ++s    ++++v CP+C+ +F\n",
            "  XP_004362377.1 190 IFCFECLHRWS----KTATV-CPNCKADF 213\n",
            "                     69999996666....67889.******99 PP\n",
            "\n",
            ">> XP_004359874.1  uncharacterized protein DFA_01912 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.1   0.1     0.025        39      12      28 ..      14      29 ..      11      30 .. 0.61\n",
            "   2 ?    5.4   0.2    0.0099        15      17      28 ..      39      51 ..      32      53 .. 0.69\n",
            "   3 ?   -0.4   0.0      0.64   9.8e+02       8      24 ..      97     113 ..      96     115 .. 0.87\n",
            "   4 ?   -2.5   0.0         3   4.6e+03       7      19 ..     192     204 ..     192     206 .. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.1 bits;  conditional E-value: 0.025\n",
            "   zinc_ribbon_9 12 VsirpdedsdvlCPeCg 28\n",
            "                    V+ +++ d+  +CP C+\n",
            "  XP_004359874.1 14 VASQSELDAL-TCPICR 29\n",
            "                    4444444444.5****6 PP\n",
            "\n",
            "  == domain 2  score: 5.4 bits;  conditional E-value: 0.0099\n",
            "   zinc_ribbon_9 17 dedsdv.lCPeCg 28\n",
            "                    ++++++ +CP+C+\n",
            "  XP_004359874.1 39 SSSETTgTCPQCR 51\n",
            "                    34444578****7 PP\n",
            "\n",
            "  == domain 3  score: -0.4 bits;  conditional E-value: 0.64\n",
            "   zinc_ribbon_9   8 CdrfVsirpdedsdvlC 24 \n",
            "                     C+  V++++++d   +C\n",
            "  XP_004359874.1  97 CQEIVTVEISKDHKLIC 113\n",
            "                     9999****999888778 PP\n",
            "\n",
            "  == domain 4  score: -2.5 bits;  conditional E-value: 3\n",
            "   zinc_ribbon_9   7 qCdrfVsirpded 19 \n",
            "                     +C+++++ + +++\n",
            "  XP_004359874.1 192 ECKQSIRETNKKQ 204\n",
            "                     6999999776655 PP\n",
            "\n",
            ">> XP_004359316.1  6-phosphogluconate dehydrogenase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -3.1   0.0       4.7   7.1e+03      18      28 ..      43      53 ..      41      54 .. 0.72\n",
            "   2 ?   11.9   0.3     9e-05      0.14       2      14 ..      85      97 ..      84     107 .. 0.89\n",
            "   3 ?   -3.8   0.5       7.6   1.2e+04      19      28 ..     389     398 ..     388     399 .. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -3.1 bits;  conditional E-value: 4.7\n",
            "   zinc_ribbon_9 18 edsdvlCPeCg 28\n",
            "                    e+++ +C  Cg\n",
            "  XP_004359316.1 43 ETENWICLKCG 53\n",
            "                    455657****8 PP\n",
            "\n",
            "  == domain 2  score: 11.9 bits;  conditional E-value: 9e-05\n",
            "   zinc_ribbon_9  2 ryWCHqCdrfVsi 14\n",
            "                    + WC++Cd+ V++\n",
            "  XP_004359316.1 85 SVWCYECDAYVTA 97\n",
            "                    67*********97 PP\n",
            "\n",
            "  == domain 3  score: -3.8 bits;  conditional E-value: 7.6\n",
            "   zinc_ribbon_9  19 dsdvlCPeCg 28 \n",
            "                     d++++C +C \n",
            "  XP_004359316.1 389 DEIPTCQDCS 398\n",
            "                     5677999995 PP\n",
            "\n",
            ">> XP_004358905.1  RNA polymerase I subunit [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.0   0.1     0.013        20      16      29 ..     155     169 ..     151     170 .. 0.72\n",
            "   2 ?   -1.8   0.1       1.7   2.6e+03      24      27 ..     183     186 ..     174     187 .. 0.78\n",
            "   3 ?    4.5   0.1     0.019        29      21      30 ..     226     234 ..     215     234 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.0 bits;  conditional E-value: 0.013\n",
            "   zinc_ribbon_9  16 pdedsdv.lCPeCgg 29 \n",
            "                      ++++ + +CP C++\n",
            "  XP_004358905.1 155 NNQQTKAnFCPTCNS 169\n",
            "                     455555569****97 PP\n",
            "\n",
            "  == domain 2  score: -1.8 bits;  conditional E-value: 1.7\n",
            "   zinc_ribbon_9  24 CPeC 27 \n",
            "                     CP C\n",
            "  XP_004358905.1 183 CPTC 186\n",
            "                     **99 PP\n",
            "\n",
            "  == domain 3  score: 4.5 bits;  conditional E-value: 0.019\n",
            "   zinc_ribbon_9  21 dvlCPeCggg 30 \n",
            "                     ++ CPeCg g\n",
            "  XP_004358905.1 226 EK-CPECGHG 234\n",
            "                     46.****986 PP\n",
            "\n",
            ">> XP_004355784.1  heat shock protein DnaJ family protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.6   0.0      0.32   4.8e+02      20      29 ..     157     166 ..     151     168 .. 0.73\n",
            "   2 ?   -2.4   0.2       2.7   4.1e+03      22      29 ..     176     183 ..     172     184 .. 0.81\n",
            "   3 ?    5.0   0.1     0.014        21      23      29 ..     203     209 ..     195     211 .. 0.76\n",
            "   4 ?    4.5   0.1     0.018        28      18      29 ..     215     225 ..     212     228 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.6 bits;  conditional E-value: 0.32\n",
            "   zinc_ribbon_9  20 sdvlCPeCgg 29 \n",
            "                     +d++C  C+g\n",
            "  XP_004355784.1 157 RDIVCTTCNG 166\n",
            "                     4545****98 PP\n",
            "\n",
            "  == domain 2  score: -2.4 bits;  conditional E-value: 2.7\n",
            "   zinc_ribbon_9  22 vlCPeCgg 29 \n",
            "                     ++C  C+g\n",
            "  XP_004355784.1 176 SICTKCKG 183\n",
            "                     57****98 PP\n",
            "\n",
            "  == domain 3  score: 5.0 bits;  conditional E-value: 0.014\n",
            "   zinc_ribbon_9  23 lCPeCgg 29 \n",
            "                     +CP+C+g\n",
            "  XP_004355784.1 203 ACPQCHG 209\n",
            "                     5****98 PP\n",
            "\n",
            "  == domain 4  score: 4.5 bits;  conditional E-value: 0.018\n",
            "   zinc_ribbon_9  18 edsdvlCPeCgg 29 \n",
            "                     +++d+ CP C+g\n",
            "  XP_004355784.1 215 KEEDK-CPKCKG 225\n",
            "                     67999.****97 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (35 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       682  (0.0562011); expected 242.7 (0.02)\n",
            "Passed bias filter:                      423  (0.0348578); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        57  (0.00469716); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        12  (0.000988875); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               8  [number of targets reported over threshold]\n",
            "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 2998.08\n",
            "//\n",
            "Query:       ZinT  [M=181]\n",
            "Accession:   PF09223.14\n",
            "Description: ZinT (YodA) periplasmic lipocalin-like zinc-recruitment\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (181 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       588  (0.0484549); expected 242.7 (0.02)\n",
            "Passed bias filter:                      185  (0.0152452); expected 242.7 (0.02)\n",
            "Passed Vit filter:                         8  (0.00065925); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.13u 0.01s 00:00:00.14 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 16002.51\n",
            "//\n",
            "Query:       Zip  [M=333]\n",
            "Accession:   PF02535.25\n",
            "Description: ZIP Zinc transporter\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    2.5e-52  177.6   3.5    1.6e-51  174.9   3.5    1.9  1  XP_004360230.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "    6.1e-50  169.8   4.1    7.8e-50  169.4   4.1    1.1  1  XP_004360276.1  uncharacterized protein DFA_04547 [Cavenderia\n",
            "      9e-50  169.2  11.7    1.1e-49  169.0  11.7    1.0  1  XP_004367068.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "    4.9e-48  163.5  14.9    4.9e-48  163.5  14.9    1.3  1  XP_004367069.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "    3.3e-37  127.9   6.0    3.3e-37  127.9   6.0    2.9  1  XP_004367397.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "    6.6e-35  120.3   9.2      1e-33  116.4   9.2    2.2  1  XP_004354513.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "    0.00051   18.8   1.5    0.00062   18.5   1.5    1.1  1  XP_004350816.1  cytochrome c oxidase assembly protein [Cavend\n",
            "     0.0012   17.6   0.2     0.0012   17.6   0.2    2.8  1  XP_004357740.1  acetylglutamate kinase [Cavenderia fasciculat\n",
            "     0.0016   17.2   0.0     0.0016   17.2   0.0    1.2  1  XP_004361557.1  uncharacterized protein DFA_05840 [Cavenderia\n",
            "     0.0017   17.1   0.2     0.0017   17.1   0.2    3.0  2  XP_004361315.1  uncharacterized protein DFA_05597 [Cavenderia\n",
            "     0.0054   15.4   0.6     0.0089   14.7   0.6    1.3  1  XP_004358720.1  uncharacterized protein DFA_00735 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "       0.01   14.5   0.4      0.016   13.9   0.4    1.3  1  XP_004354180.1  uncharacterized protein DFA_10273 [Cavenderia\n",
            "      0.016   13.9   1.9      0.019   13.6   1.9    1.1  1  XP_004358408.1  uncharacterized protein DFA_00419 [Cavenderia\n",
            "      0.019   13.7   1.7       0.02   13.6   1.7    1.2  1  XP_004350550.1  RhoGAP domain-containing protein [Cavenderia \n",
            "      0.022   13.4   3.2      0.029   13.1   2.4    1.7  1  XP_004357612.1  uncharacterized protein DFA_02128 [Cavenderia\n",
            "      0.023   13.4   0.6      0.034   12.8   0.6    1.2  1  XP_004361841.1  uncharacterized protein DFA_06128 [Cavenderia\n",
            "      0.026   13.2   1.6      0.039   12.6   1.6    1.3  1  XP_004352581.1  uncharacterized protein DFA_09153 [Cavenderia\n",
            "      0.028   13.1   4.3      0.063   11.9   4.3    1.7  1  XP_004357413.1  uncharacterized protein DFA_02691 [Cavenderia\n",
            "      0.033   12.8   0.5      0.045   12.4   0.5    1.2  1  XP_004356110.1  cyclin-like F-box containing protein [Cavende\n",
            "      0.036   12.7   1.5        4.3    5.9   1.5    2.9  1  XP_004363105.1  uncharacterized protein DFA_03502 [Cavenderia\n",
            "      0.037   12.7   0.0      0.083   11.5   0.0    1.6  1  XP_004354958.1  glycosyltransferase [Cavenderia fasciculata]\n",
            "      0.049   12.3   2.3       0.65    8.6   0.1    2.2  2  XP_004352197.1  uncharacterized protein DFA_09541 [Cavenderia\n",
            "      0.055   12.1   4.0      0.064   11.9   4.0    1.2  1  XP_004366334.1  uncharacterized protein DFA_03924 [Cavenderia\n",
            "      0.059   12.0   0.4      0.089   11.4   0.4    1.2  1  XP_004362039.1  SAP DNA-binding domain-containing protein [Ca\n",
            "       0.06   12.0   6.1       0.69    8.5   6.1    2.5  2  XP_004357421.1  phosphatidylinositol glycan [Cavenderia fasci\n",
            "      0.067   11.9   0.8        0.1   11.2   0.8    1.2  1  XP_004362965.1  uncharacterized protein DFA_03360 [Cavenderia\n",
            "      0.087   11.5   1.5       0.12   11.0   1.5    1.2  1  XP_004362041.1  SAP DNA-binding domain-containing protein [Ca\n",
            "       0.11   11.1   2.5       0.16   10.6   2.5    1.3  1  XP_004350243.1  uncharacterized protein DFA_11300 [Cavenderia\n",
            "       0.12   11.0   6.8       0.12   11.0   6.8    1.3  1  XP_004356149.1  uncharacterized protein DFA_08661 [Cavenderia\n",
            "       0.16   10.6   6.0       0.23   10.1   6.0    1.2  1  XP_004351373.1  Phytochelatins synthase [Cavenderia fascicula\n",
            "       0.16   10.6   0.0       0.16   10.6   0.0    2.7  2  XP_004355729.1  uncharacterized protein DFA_08235 [Cavenderia\n",
            "       0.22   10.1   2.4       0.29    9.7   2.4    1.2  1  XP_004361946.1  uncharacterized protein DFA_06234 [Cavenderia\n",
            "       0.24   10.0   2.8        3.7    6.1   0.0    2.2  2  XP_004363094.1  uncharacterized protein DFA_03491 [Cavenderia\n",
            "       0.26    9.9   0.0       0.26    9.9   0.0    1.9  1  XP_004356331.1  putative transmembrane protein [Cavenderia fa\n",
            "        0.3    9.7   0.0        0.3    9.7   0.0    2.7  1  XP_004356130.1  putative RNA helicase [Cavenderia fasciculata\n",
            "       0.31    9.6   1.9       0.41    9.3   1.9    1.1  1  XP_004358095.1  uncharacterized protein DFA_06849, partial [C\n",
            "       0.32    9.6   2.7       0.33    9.6   2.7    1.1  1  XP_004362018.1  uncharacterized protein DFA_06313 [Cavenderia\n",
            "       0.36    9.5   4.8       0.76    8.4   4.8    1.5  1  XP_004354628.1  uncharacterized protein DFA_09274 [Cavenderia\n",
            "        0.4    9.3   6.4       0.72    8.5   6.4    1.4  1  XP_004351053.1  uncharacterized protein DFA_12114 [Cavenderia\n",
            "        0.4    9.3   7.9       0.69    8.5   7.9    1.3  1  XP_004350295.1  uncharacterized protein DFA_11352 [Cavenderia\n",
            "       0.42    9.2   4.5       0.45    9.1   4.5    1.1  1  XP_004352044.1  DNA-3-methyladenine glycosylase I [Cavenderia\n",
            "       0.45    9.1   1.5       0.66    8.6   1.5    1.2  1  XP_004351325.1  uncharacterized protein DFA_10682 [Cavenderia\n",
            "       0.45    9.1   1.5       0.66    8.6   1.5    1.2  1  XP_004358641.1  uncharacterized protein DFA_00656 [Cavenderia\n",
            "       0.47    9.1   7.9       0.25    9.9   2.7    2.3  2  XP_004355377.1  uncharacterized protein DFA_07884 [Cavenderia\n",
            "       0.58    8.8  10.9       0.33    9.6   4.9    2.3  2  XP_004359704.1  uncharacterized protein DFA_01739 [Cavenderia\n",
            "       0.71    8.5   9.0       0.89    8.2   9.0    1.1  1  XP_004360802.1  uncharacterized protein DFA_05081 [Cavenderia\n",
            "       0.76    8.4   2.6       0.83    8.3   2.6    1.1  1  XP_004353824.1  uncharacterized protein DFA_12187 [Cavenderia\n",
            "       0.77    8.4   8.2        1.1    7.8   8.2    1.4  1  XP_004359443.1  MADS-box transcription factor [Cavenderia fas\n",
            "       0.79    8.3   2.9       0.75    8.4   2.9    1.1  1  XP_004350327.1  uncharacterized protein DFA_11384 [Cavenderia\n",
            "        0.8    8.3   5.2        1.2    7.8   5.2    1.2  1  XP_004362344.1  UAS domain-containing protein [Cavenderia fas\n",
            "       0.81    8.3  11.6        1.6    7.3  11.6    1.6  1  XP_004354593.1  uncharacterized protein DFA_09239 [Cavenderia\n",
            "          1    8.0   6.4        1.4    7.5   6.4    1.3  1  XP_004363124.1  uncharacterized protein DFA_03521 [Cavenderia\n",
            "        1.1    7.9   4.2        1.8    7.2   4.2    1.2  1  XP_004359491.1  WD40-like domain-containing protein [Cavender\n",
            "        1.1    7.9   3.2        1.4    7.5   3.2    1.2  1  XP_004359733.1  SET domain-containing protein [Cavenderia fas\n",
            "        1.4    7.5   4.4        2.2    6.9   4.4    1.2  1  XP_004366718.1  RhoGAP domain-containing protein [Cavenderia \n",
            "        1.4    7.5   3.6        1.9    7.1   3.6    1.2  1  XP_004366832.1  uncharacterized protein DFA_02447 [Cavenderia\n",
            "        1.4    7.5   3.5        1.6    7.3   3.5    1.1  1  XP_004358926.1  uncharacterized protein DFA_00951 [Cavenderia\n",
            "        1.5    7.4  17.3      0.061   12.0   8.1    2.3  2  XP_004354601.1  uncharacterized protein DFA_09247 [Cavenderia\n",
            "        1.6    7.3   5.4        3.1    6.4   5.4    1.5  1  XP_004367084.1  uncharacterized protein DFA_07218 [Cavenderia\n",
            "        1.6    7.3  10.7          2    7.0  10.7    1.2  1  XP_004366934.1  uncharacterized protein DFA_07061 [Cavenderia\n",
            "        1.6    7.3   6.2        2.9    6.5   6.2    1.3  1  XP_004359218.1  leucine-rich repeat-containing protein [Caven\n",
            "        1.7    7.2   5.2        4.3    5.9   5.2    1.6  1  XP_004351983.1  uncharacterized protein DFA_10097 [Cavenderia\n",
            "        2.1    6.9   5.3        3.1    6.4   5.3    1.2  1  XP_004350793.1  uncharacterized protein DFA_11848 [Cavenderia\n",
            "        2.3    6.8  11.5        4.1    6.0  11.5    1.4  1  XP_004350944.1  uncharacterized protein DFA_12005 [Cavenderia\n",
            "        2.5    6.7  18.1        3.3    6.3   4.4    2.4  2  XP_004358866.1  putative RNA helicase [Cavenderia fasciculata\n",
            "        2.6    6.6  26.9       0.23   10.1   3.9    2.6  2  XP_004356887.1  cyclin [Cavenderia fasciculata]\n",
            "        2.9    6.5   5.1        4.7    5.8   5.1    1.3  1  XP_004359196.1  extracellular signal-regulated protein kinase\n",
            "        3.2    6.3   4.9        3.8    6.1   4.9    1.3  1  XP_004359778.1  N2227-like domain-containing protein [Cavende\n",
            "        3.9    6.0   6.5        5.6    5.5   6.5    1.4  1  XP_004360810.1  protein kinase 3 [Cavenderia fasciculata]\n",
            "          4    6.0   7.2        5.8    5.5   7.2    1.1  1  XP_004366559.1  uncharacterized protein DFA_04150 [Cavenderia\n",
            "          4    6.0   6.6        5.5    5.6   6.6    1.2  1  XP_004355277.1  UPF0551 family protein [Cavenderia fasciculat\n",
            "        4.7    5.8   5.5         11    4.5   5.5    1.6  1  XP_004350733.1  kinetochore protein Ndc80 [Cavenderia fascicu\n",
            "        4.7    5.8  10.5        1.6    7.3   5.1    2.2  2  XP_004355651.1  uncharacterized protein DFA_08151 [Cavenderia\n",
            "        5.2    5.6  12.3         10    4.6  12.3    1.4  1  XP_004358977.1  pleckstrin domain-containing protein [Cavende\n",
            "        5.8    5.5   6.4         14    4.2   6.4    1.6  1  XP_004367225.1  putative mediator complex subunit 12 [Cavende\n",
            "        5.9    5.4   8.8       0.17   10.5   1.7    1.9  1  XP_004356343.1  uncharacterized protein DFA_08860 [Cavenderia\n",
            "        6.3    5.4   8.5         11    4.6   8.5    1.5  1  XP_004350761.1  transcription initiation factor TFIID subunit\n",
            "        6.4    5.3  10.3       0.49    9.0   3.8    2.2  2  XP_004350810.1  HEAT repeat-containing protein [Cavenderia fa\n",
            "        6.5    5.3   6.7          9    4.8   6.7    1.1  1  XP_004357732.1  NPR2 family protein [Cavenderia fasciculata]\n",
            "        6.9    5.2   7.0        8.4    4.9   7.0    1.3  1  XP_004355597.1  NUC153 domain-containing protein [Cavenderia \n",
            "        7.4    5.1   5.5        7.2    5.2   5.5    1.0  1  XP_004366330.1  uncharacterized protein DFA_03920 [Cavenderia\n",
            "        8.7    4.9   9.1         10    4.7   9.1    1.2  1  XP_004359062.1  uncharacterized protein DFA_01087 [Cavenderia\n",
            "          9    4.9   7.3         14    4.2   7.3    1.5  1  XP_004362923.1  uncharacterized protein DFA_03318 [Cavenderia\n",
            "        9.3    4.8   7.4        1.6    7.3   4.1    1.4  1  XP_004350745.1  WD40 repeat-containing protein [Cavenderia fa\n",
            "        9.4    4.8   5.9         13    4.3   5.9    1.2  1  XP_004366514.1  uncharacterized protein DFA_04104 [Cavenderia\n",
            "        9.6    4.8   6.9        2.5    6.7   1.7    2.1  2  XP_004367319.1  uncharacterized protein DFA_07460 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004360230.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  174.9   3.5   1.2e-53   1.6e-51      32     333 .]       6     379 ..       1     379 [. 0.77\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 174.9 bits;  conditional E-value: 1.2e-53\n",
            "             Zip  32 lsklllslllafavGvllgtaflhLlPealealeesqleeehthplagllvllGflllllveklltmivek.....ksshkektpkkassssses 121\n",
            "                      s+++lsl   f++G+++g a+lhL+ +++ +l++  +      p a l+++ Gfl+++++e +l+++ e+     +s      + + +++ ++ \n",
            "  XP_004360230.1   6 NSNKWLSLCNSFSGGIFFGAALLHLFVDSISQLQNVISY-----PIAPLCLCAGFLITFFLELFLHIYLERhpkllNS------NINNNNNNNNT 89 \n",
            "                     6899*******************************8753.....***************************6655522......22222222222 PP\n",
            "\n",
            "             Zip 122 aakksvsndkveeseek.................................................................edhvaeadtdads 151\n",
            "                     ++++++  ++++   ++                                                                              \n",
            "  XP_004360230.1  90 SSSSTILHHQHQHINNNnqstlnfgntnsnttfnflnsptnnrrsmildrpstpnivntlvftpppnhrtnvdndnectpmlS------------ 172\n",
            "                     22222222222222222233444566666666677777777777778888888888888887777665442222222222221............ PP\n",
            "\n",
            "             Zip 152 keelrngkqeksesekeqas...................................eeedkeklaslkvraiilllglilHsfidGLAlGvsssss 211\n",
            "                        ++n++ + ++++ke +s                                   ++++++ + +  +  +il++gl++Hs+++GLA+G++s++ \n",
            "  XP_004360230.1 173 ---DHNNNNNNDDDDKEDESsdsgdyipssnhssiyfqdalhlsstsiqqqqqevTKQETTLTNQNYLLPFILVFGLSVHSLFEGLAFGLQSTVP 264\n",
            "                     ...2222222222222222244577777777788899***********999999666666666666777************************** PP\n",
            "\n",
            "             Zip 212 tgislfiAvllHelpeelglgaiLlkaglsrkkavllnllfaltaplGiliGilvrsisessgaeivsgillalsaGtflYvalvellphEllne 306\n",
            "                     +++ l++Av+ H+l++ ++lg+  +    s  k +l+ ++f++++p+G+ iGi+   ++ s+  ++v +il++l++GtflY++lve++p+E++++\n",
            "  XP_004360230.1 265 KILDLLVAVFSHKLLASFALGVSTIIYSRSLVKMILILFIFSISSPIGASIGIV--LVDYSQIGNVVPPILQGLASGTFLYISLVEIIPKEIKSD 357\n",
            "                     ***********************99999999***********************..888999********************************* PP\n",
            "\n",
            "             Zip 307 kgssslkqsllqllglllGfalmllla 333\n",
            "                     +     ++ + + l+lllG  lm+++a\n",
            "  XP_004360230.1 358 S-----ENVFIKCLLLLLGWSLMAVIA 379\n",
            "                     9.....69999**************97 PP\n",
            "\n",
            ">> XP_004360276.1  uncharacterized protein DFA_04547 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  169.4   4.1   5.5e-52   7.8e-50       6     332 ..       8     396 ..       3     397 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 169.4 bits;  conditional E-value: 5.5e-52\n",
            "             Zip   6 laiiaililsllgllgpllvpkiskl...lsklllslllafavGvllgtaflhLlPealealeesqleeehthplagllvllGflllllvekllt 97 \n",
            "                     ++i++ +++s +++l+p+ ++k+ k+    +++++ l  +f +G++l+ af+hLl e+ e+l+e  ++     pla++++ +Gf++++++ek+ +\n",
            "  XP_004360276.1   8 VLILLSILISGFPGLIPYALDKYVKIkhsKKEYVIGLGNCFVAGIFLSIAFVHLLNESSEKLSEVNHHI----PLAFIMCPVGFVVVFMIEKIFN 98 \n",
            "                     45666677888899999999999999999999********************************97655....********************** PP\n",
            "\n",
            "             Zip  98 mivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas..................... 171\n",
            "                      + ++  +h++   + ++++++++++ k+ +n+ +ees+   +++++  +    +++  ++++++++ +++ ++                     \n",
            "  XP_004360276.1  99 CFEGH--NHGHSHGGSKKNKKNKNKKNKKNNNNLLEESTTIYKQQSGSSGYIILESSNLDEQKQPKSVSPKFPHspsygtfqditidtsrppspf 191\n",
            "                     88777..4444444444444444444444455555556665555555555555555555555555555555444899999*************** PP\n",
            "\n",
            "             Zip 172 ................................................eeedkeklaslkvraiilllglilHsfidGLAlGvssssstgislfi 218\n",
            "                                                                     ++ d++ ++      iil++ l++Hs+i G+ lGv s+ + + +lfi\n",
            "  XP_004360276.1 192 pqspnlsssssssiptttititqdietptgdehlpmikediiitlendKSTDSSTTSLKGPTPIILVFILSIHSIISGFTLGVESNYDIIYPLFI 286\n",
            "                     *******************************************99765444444444455669******************************** PP\n",
            "\n",
            "             Zip 219 AvllHelpeelglgaiLlkaglsrkkavllnllfaltaplGiliGilvrsisessgaeivsgillalsaGtflYvalvellphEllnekgssslk 313\n",
            "                      ++ H+ +e+++lg+ L++++ s    + l+ l++lt plGi++G++   +s s  +  +++++la+s+Gtf+Y+al+++l  E+ ++  ++s  \n",
            "  XP_004360276.1 287 GIISHKWLEAMSLGVSLVRNKSSFYETLKLVSLYSLTEPLGIVLGVA---ASVSVASTTATSLVLAFSSGTFIYIALMDILVVEFNESI-KQSRL 377\n",
            "                     ***********************************************...9******************************99999999.57888 PP\n",
            "\n",
            "             Zip 314 qsllqllglllGfalmlll 332\n",
            "                     ++l +l++ ++Gf+ m++l\n",
            "  XP_004360276.1 378 TKLVKLAFCIIGFLGMSAL 396\n",
            "                     899*************987 PP\n",
            "\n",
            ">> XP_004367068.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  169.0  11.7   7.5e-52   1.1e-49       2     333 .]      56     389 ..      55     389 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 169.0 bits;  conditional E-value: 7.5e-52\n",
            "             Zip   2 llkilaiiaililsllgllgpllvpkiskllsklllslllafavGvllgtaflhLlPealealeesqleee.hth......plagllvllGflll 89 \n",
            "                      +ki++i+ i++l+ll+ ++p+++ + +   +  lls+ ++fa+Gv+l+ +f h+lP a e++ +    ++ +++      p+a  ++++ +l+l\n",
            "  XP_004367068.1  56 NAKIGLIVGIFFLTLLSSYIPFILGRAKVKGFITLLSIGTCFAGGVILAGGFNHILPGAEESFTSYF--DQvAPEnkyrefPFAATIAIFTLLVL 148\n",
            "                     589**********************99999*******************************999853..222444667889************** PP\n",
            "\n",
            "             Zip  90 llveklltmivek..ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas...eeedkekl 179\n",
            "                       ++kl+     +  k++++ + +++a++++ + +++   ++ ++ +++ + d++ +++ ++ ++    +  q++++++++ ++   +e+ + k \n",
            "  XP_004367068.1 149 VAIDKLIIEGGFQgeKGHNHMNLSSHADNQHHHTNTHA--PDLEFGQESSS-DEEDSHGATPGNPDGALAPPQHSHGHAHSGKHdelHEKGNGKS 240\n",
            "                     *****988643335577777777777777777777766..44454444443.5556788888888888899999999999988743333333333 PP\n",
            "\n",
            "             Zip 180 aslkvr.....aiilllglilHsfidGLAlGvssssstgislfiAvllHelpeelglgaiLlkaglsrkkavllnllfaltaplGiliGilvrsi 269\n",
            "                     +    +     a+++l++l++Hs+ dGL lG+  s++ +  l++Avl H+++ +++lg++++ a++s  +  l+  + a+++plGi+iG++v+s+\n",
            "  XP_004367068.1 241 H----VantgqAWLFLVALSIHSILDGLGLGAETSKDGFYGLLVAVLAHKMLDGFALGVPIYFANFSTLQTALSLAFCAAMTPLGIGIGMAVTSV 331\n",
            "                     3....235568***********************************************************************************9 PP\n",
            "\n",
            "             Zip 270 sessgaeivsgillalsaGtflYvalvellphEllnekgssslkqsllqllglllGfalmllla 333\n",
            "                      ++s   +++gi+l++++G f Y++l el+p  l+++          l+l++++lG a ++++a\n",
            "  XP_004367068.1 332 YNGSSGHLAEGIILGVTCGSFFYISLIELIPSGLCQPG------WLRLKLAMVFLGWACLSVIA 389\n",
            "                     999****************************7777777......57799**********99996 PP\n",
            "\n",
            ">> XP_004367069.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  163.5  14.9   3.4e-50   4.9e-48       2     333 .]      23     348 ..      22     348 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 163.5 bits;  conditional E-value: 3.4e-50\n",
            "             Zip   2 llkilaiiaililsllgllgpllvpkiskllsklllslllafavGvllgtaflhLlPealealeesqleeehth.........plagllvllGfl 87 \n",
            "                      +k+++i+ i++l+l + ++p+++ + +      +l+++++fa+Gv+l+ +f h+  +a ea+ e   + +            p+   ++++ +l\n",
            "  XP_004367069.1  23 NAKVGLIVGIFFLTLGSSYIPFIMGRAKVKNLLTILAVFTCFAGGVILAGGFSHIPHHAEEAFAEYFADID--PegqneylhfPFSMTIAIFTLL 115\n",
            "                     57999********************999999***********************99999888877433222..0233556557888888888888 PP\n",
            "\n",
            "             Zip  88 llllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklasl 182\n",
            "                     +l  ++kl      + ++ ++ ++  +++++++++++++ +++++ e e   +    +++dad+++++ +g+++k+++ +     ++ + k+++ \n",
            "  XP_004367069.1 116 VLIAIDKLFIEGGFQGEKGHNHMNLSSHADTQHKHTSNHTPDVEMAEVETPGN---PDGGDADHGHSHGHGHSGKHDELH-----DDGNGKTHQA 202\n",
            "                     88888887665333323333333333444444455555555555555555433...455555555555555554444444.....4456777788 PP\n",
            "\n",
            "             Zip 183 kvr.aiilllglilHsfidGLAlGvssssstgislfiAvllHelpeelglgaiLlkaglsrkkavllnllfaltaplGiliGilvrsisessgae 276\n",
            "                     +v  a+++l++l++Hs+ dGL lG+  s++ +  l++Avl H+l+ +++lg++++ a++s+ +  ++ ++ a+++plGi+iG++v+s+ e+ g  \n",
            "  XP_004367069.1 203 NVGqAWLFLVALSIHSILDGLGLGAETSKDGFYGLLVAVLAHKLLDGFALGVPMYFAKFSVLQTAMALVFCAAMTPLGIGIGMAVTSVYEGAGGH 297\n",
            "                     8878***********************************************************************************9999**** PP\n",
            "\n",
            "             Zip 277 ivsgillalsaGtflYvalvellphEllnekgssslkqsllqllglllGfalmllla 333\n",
            "                     +++gi+l++++G f+Y++l el+p  l+++          l+l++++lG al++++a\n",
            "  XP_004367069.1 298 LAEGIILSVTCGSFIYISLIELIPSGLCQPG------WLRLKLAMVFLGWALLAIIA 348\n",
            "                     ************************7777777......57799************997 PP\n",
            "\n",
            ">> XP_004367397.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  127.9   6.0   2.4e-39   3.3e-37       4     324 ..     228     527 ..     222     536 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 127.9 bits;  conditional E-value: 2.4e-39\n",
            "             Zip   4 kilaiiaililsllgllgpllvpkiskllsklllslllafavGvllgtaflhLlPealealeesqleeehthplagllvllGflllllveklltm 98 \n",
            "                     ++++ii+ l + + gl+++++      ++s ++l ++l f++Gv+++ +f++LlPe+ ++++            a ++++ G++++ lv kl+  \n",
            "  XP_004367397.1 228 MLMCIISGLSTAIGGLFILFMK-----SPSYKFLGYMLGFSSGVMIYISFMDLLPESAAEIGFLP---------ANGFFFAGIIFFALVLKLVPE 308\n",
            "                     4444454444444444444444.....48999***************************999865.........778999**************9 PP\n",
            "\n",
            "             Zip  99 ivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvraiilllgl 193\n",
            "                     + e+   ++ ++++++++ss es+   ++ n+++++s+  ++++  ++t ++s+ + +  kq+++++++++++e++++ ++a lk  + +  lg+\n",
            "  XP_004367397.1 309 ADEH---EHGPSEETHEHSSVESSNTTTHTNSSSTTSTVVKRKNGADKTPPQSPPSSATTKQQQQSKQTKKPKEKKKQMSDAYLKNLGVVTALGI 400\n",
            "                     8766...7777777777777778888888888888888888888899999999999999999999999999999999999999************ PP\n",
            "\n",
            "             Zip 194 ilHsfidGLAlG..vssssstgislfiAvllHelpeelglgaiLlkaglsrkkavllnllfaltaplGili.Gilvrsisessgaeivsgillal 285\n",
            "                     +lH+f++G+A+   + +s + g++l++A+  H++pe++++ a+++ a  s++ a+ ++ll++l  p G++i G++       ++  +v+++ la \n",
            "  XP_004367397.1 401 SLHNFPEGVAVYlsCLKSMQVGLPLMLAIAAHNIPEGMAVAAPIYTATGSKWVAFKYCLLSGLCEPFGAIIfGFF---FRDYMTPYLVQAM-LAA 491\n",
            "                     **********9988******************************************************7653999...5556666666665.566 PP\n",
            "\n",
            "             Zip 286 saGtflYvalvellphEllnekgssslkqsllqllglll 324\n",
            "                      aG+++ +a  el+p+ l++ +     + ++++++g+++\n",
            "  XP_004367397.1 492 VAGIMIFMAIKELIPATLKYID---VNGATISNFVGMFV 527\n",
            "                     79*************9999888...45677777777775 PP\n",
            "\n",
            ">> XP_004354513.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  116.4   9.2   7.2e-36     1e-33       4     328 ..       7     417 ..       4     419 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 116.4 bits;  conditional E-value: 7.2e-36\n",
            "             Zip   4 kilaiiaililsllgllgpllvpkiskl................lsklllslllafavGvllgtaflhLlPealealeesqleeehth....... 75 \n",
            "                      ila ia +++s l+l++ +l ++++++                 s++ +++lla+++G l++ a l LlPea++   +s + +           \n",
            "  XP_004354513.1   7 SILAGIAPILSSSLPLFFFYLQRRQQQNkkdsssssnsnhstsvGSSNQYNILLALSAGLLFSVATLELLPEAIGLAIQSTSTQPSSSppppqts 101\n",
            "                     58899999999999999999999999999************9999999**************************976666655533347888899 PP\n",
            "\n",
            "             Zip  76 ..................plagllvllGflllllveklltmivek...............ksshkektpkkasssssesaakksvsndkveesee 137\n",
            "                                         +   v  Gf+ll +++ l +   ++               +           ++++s   ++++++ ++ ++++ \n",
            "  XP_004354513.1 102 etkhnhddddqgdnesslRIPMYGVASGFILLIILDLLFSDPNGGvhghshsnpiisskdN-----------DHHHSVDIKHHHHHHNNNSYNST 185\n",
            "                     999999***********9777778899***********99766667766444444444432...........44444333333333333333333 PP\n",
            "\n",
            "             Zip 138 k.............................................edhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvrai 187\n",
            "                     +                                              d++   + ++++  +  n++ ++s+s+++    + + +  ++ ++ ++\n",
            "  XP_004354513.1 186 NllvpqedhledsfelgsplmvnsnkkddndqddpssnsstpnrplSDNDIIIKNNNNNNNNNNNNNNSSSSSSSTL---STNIQTQSKTSTLSL 277\n",
            "                     33444444444444444444444444444444444444455555545555555555555555555555555555554...3467777789999** PP\n",
            "\n",
            "             Zip 188 illlglilHsfidGLAlG..vssssstgislfiAvllHelpeelglgaiLlkaglsrkkav..llnllfaltaplGiliGilvrsisessgaeiv 278\n",
            "                     + l++l +Hsf+dG+ ++   s+s s g  + +A+++H+lp ++ + +iLl+ + +        + ++ a+++p+G ++  +     ++  ++ +\n",
            "  XP_004354513.1 278 TTLIALAIHSFVDGMVISgaYSASPSIGARVALAIVIHKLPDGFVMSSILLSKNQKIFGYHpfASMIFIACMTPIGSFVSSF---LFNGIIQSST 369\n",
            "                     *****************99*********************************999987766779999***************...5566788999 PP\n",
            "\n",
            "             Zip 279 sgillalsaGtflYvalvellphEllnekgssslkqs..llqllglllGfal 328\n",
            "                      + +l++ aGtf+ ++++e+lp Ell++++   lk+s  l +l++  lGf++\n",
            "  XP_004354513.1 370 ISFILGFGAGTFIFITTCEILP-ELLHNSK---LKKStlLISLVVGYLGFVV 417\n",
            "                     **********************.******4...4443003344444556654 PP\n",
            "\n",
            ">> XP_004350816.1  cytochrome c oxidase assembly protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   18.5   1.5   4.4e-06   0.00062      94     170 ..      59     131 ..      50     219 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 18.5 bits;  conditional E-value: 4.4e-06\n",
            "             Zip  94 klltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     ++l++++++++++k++t++++++++ +++++k    ++++++e +++++++++ ++++k ++++++ +  ++e +++\n",
            "  XP_004350816.1  59 QILRYFSTENNNNKNTTENNKNNNTVNKNDQK----EESTSEESNNNNEQQQNQQNEHKYKQQRRSNKEYSEELKEK 131\n",
            "                     67888888855555555555555555544444....44444444444444444444433333333332222222221 PP\n",
            "\n",
            ">> XP_004357740.1  acetylglutamate kinase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.6   0.2   8.5e-06    0.0012      62     177 ..     205     327 ..     190     463 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.6 bits;  conditional E-value: 8.5e-06\n",
            "             Zip  62 ealeesqleee.hth...plagllvllGflllllveklltmivek......ksshkektpkkasssssesaakksvsndkveeseekedhvaead 146\n",
            "                     ea ee+q+++   t+    ++ l+++++fl l++vekllt + ++      k ++k+++p+++ ss ++++++k+v++ k+ +++++ ++ + ++\n",
            "  XP_004357740.1 205 EADEEKQQQGAiLTFsaiFYKLLIFVISFLPLFVVEKLLTFF-AWvqlellKAKAKNINPDQTFSSLKKKRKAKKVKKLKTLTNSQS-KVSTTKK 297\n",
            "                     45555555554455588899********************76.66699***999999999999999999999999999999999987.4466677 PP\n",
            "\n",
            "             Zip 147 tdadskeelrngkqeksesekeqaseeedke 177\n",
            "                      + +++  ++  ++e++ ++++++s +++++\n",
            "  XP_004357740.1 298 FNLNTPLIHSTVVEEPPNEQSSSSS-SSSST 327\n",
            "                     7777777777777777766666655.22222 PP\n",
            "\n",
            ">> XP_004361557.1  uncharacterized protein DFA_05840 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.2   0.0   1.1e-05    0.0016     107     171 ..      53     117 ..      31     312 .. 0.77\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.2 bits;  conditional E-value: 1.1e-05\n",
            "             Zip 107 kektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                      + ++k+a+ ++   ++ +sv+n++ +++ee +++++e++++++ kee+ ++++ek+e+e++++ \n",
            "  XP_004361557.1  53 YKYQQKHANLNYYSFYSTSSVNNNNNKKNEEEKEDKEEDKEEDKDKEEEEEKEEEKNEEEECHEI 117\n",
            "                     34456666666667777778888888877777777777777777777777777777777777654 PP\n",
            "\n",
            ">> XP_004361315.1  uncharacterized protein DFA_05597 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.1   0.2   1.2e-05    0.0017      76     172 ..     318     427 ..     234     494 .. 0.69\n",
            "   2 ?   -0.2   0.2       2.2   3.1e+02      79     153 ..     568     636 ..     542     671 .. 0.55\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.1 bits;  conditional E-value: 1.2e-05\n",
            "             Zip  76 plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     ++  +  ++G  ++  + ++++++++k    + +++++ +ssss ++   + +nd +++++ +++++++ + +++ +++++++++e +e++ +++\n",
            "  XP_004361315.1 318 LVFTGHSMGGTSATTATLRAMELVYKK---SNINSSSNDHSSSSSNNNNIENNNDDTTTNTTNHNNNVNNENNNNIDSQQQQQQEEVEENNVHKN 409\n",
            "                     555566677777788888888888888...66667777777777777788888888888888889999998888888888888888888888776 PP\n",
            "\n",
            "             Zip 171 s................e 172\n",
            "                     +                 \n",
            "  XP_004361315.1 410 NntittlgqnnrlelkcF 427\n",
            "                     645555555555555550 PP\n",
            "\n",
            "  == domain 2  score: -0.2 bits;  conditional E-value: 2.2\n",
            "             Zip  79 gllvllGflllllveklltmivek.......ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadske 153\n",
            "                      l+ l  f l++ +++ + +++++       +             ++ +    ++++++ +++s    ++  ++++++++++\n",
            "  XP_004361315.1 568 HLFKLYNFRLMFAADHSIDLYYQRvmrqtrpN-------------HKINIIHFNTCNKCLTDNSGLPINNPNNHQSHQNNSQ 636\n",
            "                     55566667788999999999999944422220.............2222222222222222222222222222222222111 PP\n",
            "\n",
            ">> XP_004358720.1  uncharacterized protein DFA_00735 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   14.7   0.6   6.3e-05    0.0089      56     172 ..      90     212 ..      89     286 .. 0.76\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.7 bits;  conditional E-value: 6.3e-05\n",
            "             Zip  56 LlPealealeesqleeehth....plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaead 146\n",
            "                     L P ++++l+  + +++h+      +  ++vl+ f   + ++k +  i +     ++ ++k+++++ +++ ak++  n++ +++++  +++++++\n",
            "  XP_004358720.1  90 LTPTSQGKLKIDCRHANHMLkgldQYSHVWVLFWFHNNYRIDKNTDVITPP---DEDRINKQKQKQLENKIAKDQLLNNNNQQEQQIDQEQEQEE 181\n",
            "                     6789999999999999877799999999********************999...77788888888888888888877777777777755544444 PP\n",
            "\n",
            "             Zip 147 tdadskeelrngkqeksesekeqas.......e 172\n",
            "                      d +  e + n+ +e +++e++q++       +\n",
            "  XP_004358720.1 182 NDIE--EDEENEDGEDDGEESQQTKednivssS 212\n",
            "                     4444..444444444444444444422443331 PP\n",
            "\n",
            ">> XP_004354180.1  uncharacterized protein DFA_10273 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.9   0.4   0.00011     0.016     108     172 ..     473     592 ..     469     663 .] 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.9 bits;  conditional E-value: 0.00011\n",
            "             Zip 108 ektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas............................... 171\n",
            "                     e++p+++sssss++++++s ++++++++  + +++ +  ++++ +++  ng+++ ++s+++ +s                               \n",
            "  XP_004354180.1 473 EPIPSNSSSSSSNNDSSSSNNDSSSSNNDSSSSNNDSSSSSNNGSSSSNNGSSSNNGSSSNDSSssdvtcklsrsgklrveypgdapinctskgi 567\n",
            "                     57899999999999999999999999999999999999999999999999999999999999885555555555555555555555555555555 PP\n",
            "\n",
            "             Zip 172 ........................e 172\n",
            "                                              \n",
            "  XP_004354180.1 568 secnapggfkclgllsssltqcfaP 592\n",
            "                     5555555555555555555555550 PP\n",
            "\n",
            ">> XP_004358408.1  uncharacterized protein DFA_00419 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.6   1.9   0.00014     0.019      90     174 ..      93     172 ..      69     245 .. 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.6 bits;  conditional E-value: 0.00014\n",
            "             Zip  90 llveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseee 174\n",
            "                     ++++  ++ +++     +++t+ +++++ ++s+ +++ +n++ +++ ++++++ + + ++++  +  n++ + + ++ + ++ + \n",
            "  XP_004358408.1  93 YILDTDIKALYSP----NTNTNTNTNTNNTKSGGGGTGNNNNKNDNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNKN-DN 172\n",
            "                     4445444444444....56666666666666666666666555555555555444444444444444444443333333332.11 PP\n",
            "\n",
            ">> XP_004350550.1  RhoGAP domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.6   1.7   0.00014      0.02     103     178 ..      38     111 ..      15     193 .. 0.64\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.6 bits;  conditional E-value: 0.00014\n",
            "             Zip 103 ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkek 178\n",
            "                     +++ +ek+++k ++s+   +a +   +d++e++   ed++++ + d+d +e+ +++++e+++ e++q  e+ ++e+\n",
            "  XP_004350550.1  38 EEEDGEKEEDKVEQSDDDDEARSGEASDEIEDED--EDNQESGQDDDDDDENSAAEEEEQEGGEENQDGEDGEEEE 111\n",
            "                     2222333333333333333333333333333333..4444555555555666666666666666666543333333 PP\n",
            "\n",
            ">> XP_004357612.1  uncharacterized protein DFA_02128 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.1   2.4    0.0002     0.029      75     176 ..     260     357 ..     179     377 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.1 bits;  conditional E-value: 0.0002\n",
            "             Zip  75 h....plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqekses 165\n",
            "                          pl  ++++++++l++++  + +m+        ++ + +++ ++   + + s++ ++ e+++ k++ ++ +  +++++ ++ n+k++  ++\n",
            "  XP_004357612.1 260 SrsvfPLGVFCLVISLFLFYFF--CYHMWLVV-----KNKTTNETFKYGDLKEQISINRVEQENQSLKHELDELESINNQNTNNQNNNKKKNGKQ 347\n",
            "                     0444477788888888888875..55555555.....3344444444555555555555555555554333333333333222222222222222 PP\n",
            "\n",
            "             Zip 166 ekeqaseeedk 176\n",
            "                     +++q++ + ++\n",
            "  XP_004357612.1 348 QQQQQQ-TNNN 357\n",
            "                     222211.1111 PP\n",
            "\n",
            ">> XP_004361841.1  uncharacterized protein DFA_06128 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.8   0.6   0.00024     0.034      91     172 ..     150     263 ..     147     303 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.8 bits;  conditional E-value: 0.00024\n",
            "             Zip  91 lveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhva.eadtdadskeelrngkqeksesekeqas............. 171\n",
            "                     ++e++l  +++ +++++++t+++ ++++++ + k+  ++d+ e+++++++++    + +++++++  n+++++++s+k++ s             \n",
            "  XP_004361841.1 150 IAERYLLGYIDNENNNNNNTNNTNNTNTNNTKIKNNFKDDNNENNTNNNNNNGiLNNVNKKTTTTNNNNNTSSTSSSKTK-Siiirlpliwgkgd 243\n",
            "                     57888888888888899999999999999999999999999888888755443034444444444444444444443333.35777777777766 PP\n",
            "\n",
            "             Zip 172 ...................e 172\n",
            "                                        +\n",
            "  XP_004361841.1 244 nvlkhlcnlantlrwvwigG 263\n",
            "                     66555555555555555554 PP\n",
            "\n",
            ">> XP_004352581.1  uncharacterized protein DFA_09153 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.6   1.6   0.00028     0.039     109     172 ..     117     188 ..      99     229 .. 0.42\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.6 bits;  conditional E-value: 0.00028\n",
            "             Zip 109 ktpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas............e 172\n",
            "                      ++++++++s+ ++  +s ++++ +++e+ ++++++++ ++d+k    +++ +  +++k+  +             \n",
            "  XP_004352581.1 117 RNNEQKNNNSNSNDNDNSSNDNNNQDTEQQQQQQQQQQQQQDNKRIYTEEELS--SQSKQ--KvaaicndlkieyR 188\n",
            "                     23333333333333333333333333333333222222222222222222211..11111..02234566655551 PP\n",
            "\n",
            ">> XP_004357413.1  uncharacterized protein DFA_02691 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.9   4.3   0.00045     0.063       9     180 ..     145     346 ..     137     397 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.9 bits;  conditional E-value: 0.00045\n",
            "             Zip   9 iaililsllgllgpllvpkiskl.......lsklllslllafavGvllgtaflhLlPealealeesqleeehth....plagllvllGflllllv 92 \n",
            "                     +++++  ++++   +l++ i+k+         +l+++ l+++++   +    + +   ++  + +  ++++ +h    +  ++ +l G +++  v\n",
            "  XP_004357413.1 145 LLVFFGIFAVVNYFILYRQIKKYetkmrsdNMSLRVRKLKILSAALVVTIVCVLIRDLVYYVIIKVPMDHNLRHlssfITFFFEALQGVIIMIAV 239\n",
            "                     44444444444455555555555677777766666666666666544444444444445556666666666666677777777788888888877 PP\n",
            "\n",
            "             Zip  93 eklltmivek...ksshkektpkkasssssesaakksvsndkveeseekedhvaead...tdadskeelrngkqeksesekeqas.......... 171\n",
            "                      +  ++++     + ++++ t + + ss+s+s+++ s ++ ++ +  ++ + v+ ++    ++d+++  ++++qeks++++ +as          \n",
            "  XP_004357413.1 240 SERPSYYFLFrnvNPHRGDSTASSHVSSESNSQSVASKNMATTITIDDGASSVTVQHpniNNNDKSHVVHQQQQEKSSHIALTASnlvewsdded 334\n",
            "                     7777777766799999999999999**************99999999998877777633345667777888888889999888889999999988 PP\n",
            "\n",
            "             Zip 172 ...eeedkekla 180\n",
            "                        ++edk++  \n",
            "  XP_004357413.1 335 edeDDEDKQDGN 346\n",
            "                     776333333333 PP\n",
            "\n",
            ">> XP_004356110.1  cyclin-like F-box containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.4   0.5   0.00032     0.045      51     176 ..       9     130 ..       8     186 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.4 bits;  conditional E-value: 0.00032\n",
            "             Zip  51 taflhLlPealealeesqleeehthplagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaea 145\n",
            "                     t+++ +l + +ea+++  l+++       ++ l+  + l+++  + t+  +++s+++ kt+ ++++++++s++ ++ s+++ +++ +   +++  \n",
            "  XP_004356110.1   9 TSIVVVLTHTMEAIKQLFLKSTG--QRNLIITLVTSVSLFIITCIYTYRLKSSSRQQSKTTTTTNDEHKQSSTISTPSKNNNNNNNQ--HVASTN 99 \n",
            "                     66777777777777777776642..34444455555555555666666666644444444444444444444444444333333222..344445 PP\n",
            "\n",
            "             Zip 146 dtdadskeelrngkqeksesekeqaseeedk 176\n",
            "                     ++ a+s+++ ++ + eks  +++q+++++ +\n",
            "  XP_004356110.1 100 ESAASSSTTPKKIVNEKSPAKPQQQQQQQVT 130\n",
            "                     5555556666666666666666665533333 PP\n",
            "\n",
            ">> XP_004363105.1  uncharacterized protein DFA_03502 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.9   1.5     0.031       4.3     171     172 ..      91     290 ..       5     347 .. 0.45\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.9 bits;  conditional E-value: 0.031\n",
            "             Zip 171 s.............................................................................................. 171\n",
            "                     +                                                                                              \n",
            "  XP_004363105.1  91 Qrieklkekknlkqkeklerqqiltnnnnnninnnninrdikndilltnnnnnndgststihtskntplvalstkqqpeiiepplpsdqhmeeee 185\n",
            "                     13333333332222222222233333333444444444444444444444444444444444444444444444444444444444444444333 PP\n",
            "\n",
            "             Zip 172 ............................................................................................... 171\n",
            "                                                                                                                    \n",
            "  XP_004363105.1 186 edhqghdhhghfkkspiklrardklrrirtpnvpppkgvnivddpfieldsvdsrymededqaekrrilgtrkyetayikidyylehiniffsvv 280\n",
            "                     33333333333333355555555555555555555555555555555555555555555555555555555555555555555555555555555 PP\n",
            "\n",
            "             Zip 172 .........e 172\n",
            "                              +\n",
            "  XP_004363105.1 281 cygiyiaisY 290\n",
            "                     5555555554 PP\n",
            "\n",
            ">> XP_004354958.1  glycosyltransferase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.5   0.0   0.00059     0.083      75     135 ..     211     285 ..     131     297 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.5 bits;  conditional E-value: 0.00059\n",
            "             Zip  75 h..............plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkvees 135\n",
            "                     +              ++a l +++G l++ l+ +++++i + +s+ + k++++++++ ++++  +s +n+++ + \n",
            "  XP_004354958.1 211 IenlalhgihnrlthLFASLPLMIGPLIIVLTIEIISIIRNYNSNSNSKSNQPTTNNNNNNNNNNSNNNSESGNG 285\n",
            "                     044444444444555**********99999999999999999666666666666666666666665555555433 PP\n",
            "\n",
            ">> XP_004352197.1  uncharacterized protein DFA_09541 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.6   0.1    0.0046      0.65      76     150 ..      45     116 ..       7     123 .. 0.78\n",
            "   2 ?    3.4   0.3      0.17        24     100     151 ..     310     358 ..     290     366 .] 0.36\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.6 bits;  conditional E-value: 0.0046\n",
            "             Zip  76 plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdad 150\n",
            "                     +  ++l   +++l  +v ++++ + ++     ++t+ + +++++++ ++++ s++++++s+ +++  + +d++++\n",
            "  XP_004352197.1  45 IISFGLWRYSIFLPRFVQNIISSFTPT---PTTTTTTTNNNNHNNNITQQRSSSSTTTTSTTNTNTSTTTDSNQP 116\n",
            "                     455555667899999999999999999...999999999999999999999999999999999887777777665 PP\n",
            "\n",
            "  == domain 2  score: 3.4 bits;  conditional E-value: 0.17\n",
            "             Zip 100 vekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdads 151\n",
            "                     +++    +++++ + ++s ++  ++++ + ++ + +  +++++++++ ++++\n",
            "  XP_004352197.1 310 FSS---SNRNESINNQNSNNYYTTSGTRNTNNNNINNTNNSNNPSQQQQQQQ 358\n",
            "                     333...2233333333333333333333333333333333333333222222 PP\n",
            "\n",
            ">> XP_004366334.1  uncharacterized protein DFA_03924 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.9   4.0   0.00045     0.064      79     168 ..       9     102 ..       2     180 .. 0.63\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.9 bits;  conditional E-value: 0.00045\n",
            "             Zip  79 gllvllGflllllveklltmivek....ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqekseseke 168\n",
            "                      ll+l+G+  + ++ ++   +v+k    ++++++   ++ s+ ss+s++++s s++++++s+ + +  + ++++ +s+++  ++kq+ ++++  \n",
            "  XP_004366334.1   9 ILLLLIGIACIATITSARDVVVNKqsglHQNNNNHHGHDPSAASSYSYSSSSSSSSSSSSSSSSSSSTTTKKSSSSSTTKNYKQKQQNPGNQVG 102\n",
            "                     5788999999999999999999997555333333333333333444444444444444444444444444444444444444444444444433 PP\n",
            "\n",
            ">> XP_004362039.1  SAP DNA-binding domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.4   0.4   0.00063     0.089      69     168 ..     252     348 ..     238     464 .. 0.64\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.4 bits;  conditional E-value: 0.00063\n",
            "             Zip  69 leeehth..........plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadske 153\n",
            "                     + ++++h          +++   v+ G+ ll  ++  ++ i ++     +++ ++++++ss++++ +  ++d  +e+ ++         d+d +e\n",
            "  XP_004362039.1 252 TPGHIRHiisrdwelfkVFPRSSVIAGAGLLRYIDPKTEVITKS-----NDSDDQEDNNSSHNDDDDDNQDDDDDEDDDD--------DDEDDEE 333\n",
            "                     44554445666666666777777777777777777777766666.....3333333333333333333222222222222........2222222 PP\n",
            "\n",
            "             Zip 154 elrngkqekseseke 168\n",
            "                      + +++qe +++++e\n",
            "  XP_004362039.1 334 DDDEQEQEVEQEDEE 348\n",
            "                     222222222222221 PP\n",
            "\n",
            ">> XP_004357421.1  phosphatidylinositol glycan [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.5   6.1    0.0049      0.69     154     199 ..      16      60 ..       5      64 .. 0.62\n",
            "   2 ?   -0.4   0.0       2.6   3.7e+02      81     114 ..     702     731 ..     693     745 .. 0.67\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.5 bits;  conditional E-value: 0.0049\n",
            "             Zip 154 elrngkqeksesekeqaseeedkeklaslkvraiilllglilHsfi 199\n",
            "                     + +  ++++++s+++++s+ +d++k  + k+  +i++lgli+Hs++\n",
            "  XP_004357421.1  16 HSHSHSSHHHHSSHHHSSHGQDEKKRFNGKI-LFIVVLGLIFHSVF 60 \n",
            "                     3344444555555555554445555555544.67899*******98 PP\n",
            "\n",
            "  == domain 2  score: -0.4 bits;  conditional E-value: 2.6\n",
            "             Zip  81 lvllGflllllveklltmivekksshkektpkka 114\n",
            "                     lv+lG +l++ + +++ ++v+      +++  + \n",
            "  XP_004357421.1 702 LVCLGGILIFAIGSFTIYYVNN----RTNWATTV 731\n",
            "                     5788888888899999999888....23333333 PP\n",
            "\n",
            ">> XP_004362965.1  uncharacterized protein DFA_03360 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.2   0.8   0.00072       0.1     102     167 ..     355     424 ..     313     430 .. 0.50\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.2 bits;  conditional E-value: 0.00072\n",
            "             Zip 102 k.......ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesek 167\n",
            "                     +       ++s ++ ++ + s+sss s++++s s++++++++++ ++   +++++ds ++ +n+k+ +s++e+\n",
            "  XP_004362965.1 355 RfkivsrtDKSSGRRSTTTRSNSSSSSNSSSSSSSSSSNGNKDNSNN---KNSNNDSNSNSSNSKSMSSSKEE 424\n",
            "                     15555555445555555555555555555555555555555555443...33333333333333333222222 PP\n",
            "\n",
            ">> XP_004362041.1  SAP DNA-binding domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.0   1.5   0.00085      0.12      76     170 ..      89     184 ..      49     300 .. 0.69\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.0 bits;  conditional E-value: 0.00085\n",
            "             Zip  76 plagllvllGflllllveklltmivekksshkektpkk.asssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeq 169\n",
            "                     +l+  +v+ G   l +v++ ++ + e+++++++k++++  ++++ e++ +++ +n+++++++++++++ + + ++++  +  n++ + + ++ + \n",
            "  XP_004362041.1  89 LLPSSTVIAGVGNLRFVDQPVNDLNEQEKQEDQKEEEDyDDEEEDEEEEEEEEENKNFQPQSKNNSNNINNNNNNNNNNNNNNNNNNNNNNSGNN 183\n",
            "                     56667788888888899999999988833333333333033344455666677778888888887777777776666666666555555544444 PP\n",
            "\n",
            "             Zip 170 a 170\n",
            "                     +\n",
            "  XP_004362041.1 184 N 184\n",
            "                     3 PP\n",
            "\n",
            ">> XP_004350243.1  uncharacterized protein DFA_11300 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.6   2.5    0.0011      0.16     101     179 ..      88     166 ..      41     187 .. 0.51\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.6 bits;  conditional E-value: 0.0011\n",
            "             Zip 101 ekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas....eeedkekl 179\n",
            "                     + +++h++ +++  ++s+s s + +  +++++++s+  +   +++   ++ +ee+ +++q+ s ++ ++++    +   +++l\n",
            "  XP_004350243.1  88 H-HHHHQNSITSRNNQSTSISTPPQPSPCSSSTTSTPTTTTSSKDTYFDKDDEEEEEENQDRSNNTRKRKKivnaK---TTQL 166\n",
            "                     1.34444444444444444444333333333333333333333333344444444444444444444444443332...2222 PP\n",
            "\n",
            ">> XP_004356149.1  uncharacterized protein DFA_08661 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.0   6.8   0.00086      0.12     121     177 ..     102     144 ..      24     221 .. 0.69\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.0 bits;  conditional E-value: 0.00086\n",
            "             Zip 121 saakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedke 177\n",
            "                      ++k+ +++d +++s+ kedh ++              k ++++++++  ++++++e\n",
            "  XP_004356149.1 102 REKKSGHHDDHHKDSHHKEDHHHK--------------KDDHHDDGHHDDHHDHHHE 144\n",
            "                     233333333333333333332222..............1122222222211111111 PP\n",
            "\n",
            ">> XP_004351373.1  Phytochelatins synthase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.1   6.0    0.0016      0.23      94     176 ..     219     298 ..     204     415 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.1 bits;  conditional E-value: 0.0016\n",
            "             Zip  94 klltmivekksshkektpkkasssss......esaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedk 176\n",
            "                     +l+ +++++        ++ +s+sss      + +   s +n+ +++++ +    +++++++ +++++ n++ + ++++++q+ +++++\n",
            "  XP_004351373.1 219 ELTPHVHHR------CKSSSSSASSSlltiqhHRHHLISNNNKRHYSQQCC---SVQHNHSHGHSHSHGNSHAHANGHNHNQQPQQQQQ 298\n",
            "                     333333333......222222222223333323333333333333333333...23444444444444444444444444444311111 PP\n",
            "\n",
            ">> XP_004355729.1  uncharacterized protein DFA_08235 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.6   0.0    0.0011      0.16     128     172 ..     178     221 ..     167     326 .. 0.75\n",
            "   2 ?    0.0   3.5       1.9   2.7e+02     115     155 ..     505     545 ..     479     553 .] 0.41\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.6 bits;  conditional E-value: 0.0011\n",
            "             Zip 128 sndkveeseekedhvaeadtdadskeelrngkqeksesekeqase 172\n",
            "                      +d+++++eeke+++++++ ++ ++e++ ++k+e +e++ke +++\n",
            "  XP_004355729.1 178 RADQIKQQEEKEQKEQQQQNEQ-KEENKEEEKEEEEEDNKENKQD 221\n",
            "                     5677777777766444444333.3444444444444444443332 PP\n",
            "\n",
            "  == domain 2  score: 0.0 bits;  conditional E-value: 1.9\n",
            "             Zip 115 sssssesaakksvsndkveeseekedhvaeadtdadskeel 155\n",
            "                     ++++++++++   + ++ +++ ++++++ + + ++++  ++\n",
            "  XP_004355729.1 505 QQEKEKEKDGANLMTNNNNNNNNNNNNNNNNNNNNTTNNTQ 545\n",
            "                     22222222222222222222222222222222222222222 PP\n",
            "\n",
            ">> XP_004361946.1  uncharacterized protein DFA_06234 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.7   2.4    0.0021      0.29     106     182 ..      50     114 ..      27     225 .. 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.7 bits;  conditional E-value: 0.0021\n",
            "             Zip 106 hkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklasl 182\n",
            "                     h++++++++++ +++++ +++ ++ + +e e+ e+            e++ ++k+e +++e eq++ e+++ ++++ \n",
            "  XP_004361946.1  50 HNRIIEGQERERQERERQERERQERERQERERQER------------EQQEREKEEREKHEREQQEREKEERQPKTA 114\n",
            "                     44444555555555555555444444444444433............333333333333333333322222222222 PP\n",
            "\n",
            ">> XP_004363094.1  uncharacterized protein DFA_03491 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.1   0.0     0.026       3.7      77     171 ..     108     242 ..      91     343 .. 0.77\n",
            "   2 ?    1.4   3.1      0.71     1e+02      99     161 ..     389     441 ..     376     455 .. 0.48\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.1 bits;  conditional E-value: 0.026\n",
            "             Zip  77 lagllvllGflllllveklltmivek........ksshkektpkkasssss................................esaakksvsndk 131\n",
            "                     ++ l++  G+l ++  e++++ + ++        ++s++++++k+a++ +                                   ++ +  +nd \n",
            "  XP_004363094.1 108 FPLLFIYSGLLCFMRWESMVKRWNNGdiydyrriYNSQQRDIHKNANARKYdlitgffdhlsfdqdsvkeekgviflptlnqdDDDDDDDDDNDH 202\n",
            "                     67777888888888888888888888888888886666666666666666667777777777777777766666666666666666777777778 PP\n",
            "\n",
            "             Zip 132 veeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +++s+++e+ + ++d d++++ e  +g++e +e+e+ q++\n",
            "  XP_004363094.1 203 LQDSSDDEELNISKDDDKEEDDEIFDGEEEEDEEEDYQPH 242\n",
            "                     8888888888888888888888888888888888777766 PP\n",
            "\n",
            "  == domain 2  score: 1.4 bits;  conditional E-value: 0.71\n",
            "             Zip  99 ivek...ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqe 161\n",
            "                     i++k   +++++++++ k +++++ +++ k+ +n+++++se+ ++             ++ ++++ \n",
            "  XP_004363094.1 389 IKSKqpkENDKSTNIKVKLQDNEKDKKKRKRDNNNNIKKSEKVKS-------------TTPKQVKT 441\n",
            "                     333333334444444444444444444444444444444444433.............32222222 PP\n",
            "\n",
            ">> XP_004356331.1  putative transmembrane protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.9   0.0    0.0019      0.26     134     205 ..      53     125 ..      32     136 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.9 bits;  conditional E-value: 0.0019\n",
            "             Zip 134 eseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvr......aiilllglilHsfidGLAlG 205\n",
            "                     ++ + +++ + + td+++++e+++++q+++++++++     ++  +++   r       + + +++ +   i G+++G\n",
            "  XP_004356331.1  53 NNNNTFKNYRYYSTDHNQQQEQQQQQQQQQQQQQQR-----KENVYTKADWRltganrDWNNYIAFLIGAAISGISFG 125\n",
            "                     222233333334444443333333333333333333.....33333333333333444*******************9 PP\n",
            "\n",
            ">> XP_004356130.1  putative RNA helicase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.7   0.0    0.0021       0.3     108     209 ..     158     335 ..     120     341 .. 0.71\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.7 bits;  conditional E-value: 0.0021\n",
            "             Zip 108 ektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas............................... 171\n",
            "                      + ++ ++++++e++ ++s s+++vee ee e++++++ +++++ +                 s                               \n",
            "  XP_004356130.1 158 SDDEDIKDNEEKEQDESDSDSDSEVEEVEEVEEEEQKVPKNNNTNK-----------------Sipkvkvepldtlktlesnkrmkkevvmdlpt 235\n",
            "                     2333344444444444455555555555555444444444444444.................14444444444444455589999999****** PP\n",
            "\n",
            "             Zip 172 ..............................................................eeedkeklaslkvraiilllglilHsfidGLAl 204\n",
            "                                                                                   + + +  ++++ ++  + +l+l  Hs+++ LA \n",
            "  XP_004356130.1 236 feelhlsrplqkavaklgytqptpiqakaiplilngkdilasattgsgktaafilpilerllYRDATHRVSRVLIVLPTRELALQCHSVFESLAQ 330\n",
            "                     *********************************************************9999888888888888888999999*******999998 PP\n",
            "\n",
            "             Zip 205 Gvsss 209\n",
            "                     ++ ++\n",
            "  XP_004356130.1 331 FTNVQ 335\n",
            "                     75444 PP\n",
            "\n",
            ">> XP_004358095.1  uncharacterized protein DFA_06849, partial [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.3   1.9    0.0029      0.41     101     172 ..      70     143 ..      44     208 .. 0.53\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.3 bits;  conditional E-value: 0.0029\n",
            "             Zip 101 ekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas.......e 172\n",
            "                     e+ ++ k++ ++k+++++++ ++ +s +++++ee+ee+  ++ e+++++++ +     k  +++      +       +\n",
            "  XP_004358095.1  70 ES-KQPKKDKKEKKETNKEKGKDDDSDEESSHEEDEEGYVVEDEEQHTDKKRKASTPPKPTTKKAP----AkkaaakdK 143\n",
            "                     22.333444444444444455555555555555555554443333333332222222222222222....122233330 PP\n",
            "\n",
            ">> XP_004362018.1  uncharacterized protein DFA_06313 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.6   2.7    0.0024      0.33     114     163 ..       9      60 ..       1     172 [. 0.60\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.6 bits;  conditional E-value: 0.0024\n",
            "             Zip 114 asssssesaakksvsndkveeseek..edhvaeadtdadskeelrngkqeks 163\n",
            "                     +++++++++++k+ + +++ ++++   ++ +++++ ++++++++++++q+k+\n",
            "  XP_004362018.1   9 QQKKQKQKQKQKQKQIQQLIKKKKQkkKQLQQQQQQQQQQQQQKQKQQQQKQ 60 \n",
            "                     1111111111111111111111111001111111111111111111111111 PP\n",
            "\n",
            ">> XP_004354628.1  uncharacterized protein DFA_09274 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.4   4.8    0.0054      0.76     114     156 ..     102     145 ..      36     203 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.4 bits;  conditional E-value: 0.0054\n",
            "             Zip 114 asssssesaakksvsnd.kveeseekedhvaeadtdadskeelr 156\n",
            "                     ++++ +++++k++ +++ k e++e+++ h+ ++++++++ke++r\n",
            "  XP_004354628.1 102 KNDNNNKKEKKEKKEKKeKKEKKEKNHHHQDQHKMKKTKKEKER 145\n",
            "                     11111111111111111011111111112222222222222221 PP\n",
            "\n",
            ">> XP_004351053.1  uncharacterized protein DFA_12114 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.5   6.4    0.0051      0.72     126     180 ..     338     392 ..     323     490 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.5 bits;  conditional E-value: 0.0051\n",
            "             Zip 126 svsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekla 180\n",
            "                     +v+++k ee+ee e++++e++++++++ee+ ++++ek+e+e+e+++eee++e++ \n",
            "  XP_004351053.1 338 EVEKKKEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEKEEEEEEEEEEEEEEEEIR 392\n",
            "                     4444444444444444445555555555555555555555555544444444443 PP\n",
            "\n",
            ">> XP_004350295.1  uncharacterized protein DFA_11352 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.5   7.9    0.0049      0.69     101     205 ..      34     154 ..      22     175 .. 0.48\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.5 bits;  conditional E-value: 0.0049\n",
            "             Zip 101 ek.ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas............eeedkeklasl 182\n",
            "                     ++ ks  k +++++ + sss s++++s sn+k ++s+ ke++++++ ++ +++++++  k+++++ ++++++              +++   +++\n",
            "  XP_004350295.1  34 SSsKSTSKSTSSNNNNPSSSSSSTSSSKSNKKKKNSKSKEKEKTSTTSTTSTSTSTSTVKSSSPSTQPSKKKpaaaaapvplkpVAQREVTFSNI 128\n",
            "                     33222333333333333444444555555555555555555555555555555555444444444444444466677777777644444444444 PP\n",
            "\n",
            "             Zip 183 kvr...aiilllglilHsfidGLAlG 205\n",
            "                       +   + + +      s+++GL++ \n",
            "  XP_004350295.1 129 VFVaqpVVVATATTAATSIPEGLSTT 154\n",
            "                     44422233333334455666666666 PP\n",
            "\n",
            ">> XP_004352044.1  DNA-3-methyladenine glycosylase I [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.1   4.5    0.0032      0.45     112     167 ..      34      89 ..       9     178 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.1 bits;  conditional E-value: 0.0032\n",
            "             Zip 112 kkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesek 167\n",
            "                       +++ +s++++k+++++d+ ++s e+e++++e+++++ +++e+ ++++e+s+++ \n",
            "  XP_004352044.1  34 TLKQNTRSKKKVKQEIEKDTESSSDESEQVDKEDEESDVEESEEEEEEEESSDEKV 89 \n",
            "                     45677788888888888888888888888888888888777777777777776654 PP\n",
            "\n",
            ">> XP_004351325.1  uncharacterized protein DFA_10682 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.6   1.5    0.0047      0.66     101     167 ..     291     356 ..     265     427 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.6 bits;  conditional E-value: 0.0047\n",
            "             Zip 101 ekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesek 167\n",
            "                     e+ ++ k++ ++k+++++++ ++ +s +++++ee+ee+  ++ e+++++++ +     k  +++  +\n",
            "  XP_004351325.1 291 ES-KQPKKDKKEKKETNKEKGKDDDSDEESSHEEDEEGYVVEDEEQHTDKKRKASTPPKPTTKKAPA 356\n",
            "                     22.3334444444444444555555555555555555544443333333322222222222222222 PP\n",
            "\n",
            ">> XP_004358641.1  uncharacterized protein DFA_00656 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.6   1.5    0.0047      0.66     101     167 ..     291     356 ..     265     427 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.6 bits;  conditional E-value: 0.0047\n",
            "             Zip 101 ekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesek 167\n",
            "                     e+ ++ k++ ++k+++++++ ++ +s +++++ee+ee+  ++ e+++++++ +     k  +++  +\n",
            "  XP_004358641.1 291 ES-KQPKKDKKEKKETNKEKGKDDDSDEESSHEEDEEGYVVEDEEQHTDKKRKASTPPKPTTKKAPA 356\n",
            "                     22.3334444444444444555555555555555555544443333333322222222222222222 PP\n",
            "\n",
            ">> XP_004355377.1  uncharacterized protein DFA_07884 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.9   2.7    0.0018      0.25      76     179 ..      60     160 ..      31     175 .. 0.74\n",
            "   2 ?    1.5   0.1      0.65        91     103     163 ..     316     343 ..     262     462 .. 0.63\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.9 bits;  conditional E-value: 0.0018\n",
            "             Zip  76 plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     +l    +  + ++   v+k++t+ +++  s ++ t + +sss++++++ +s +n++ +++++  +  ++ + d++ +++  +++ ++++ +ke +\n",
            "  XP_004355377.1  60 VLYLSSCAASTFICPNVNKTTTIGKST-FSFSTSTASSSSSSYTKKHYNNSGNNKQYTSNQKYDSRYKQRKYDDQRQQQPLRNEDHQDKPRKEDE 153\n",
            "                     556667777888888888888877776.99999999999***********************988777777777777776666666666666654 PP\n",
            "\n",
            "             Zip 171 seeedkekl 179\n",
            "                     +  +++e+ \n",
            "  XP_004355377.1 154 H--REEEEE 160\n",
            "                     4..444443 PP\n",
            "\n",
            "  == domain 2  score: 1.5 bits;  conditional E-value: 0.65\n",
            "             Zip 103 ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeks 163\n",
            "                     +++++++ +kk+ss++++ +                                  n ++++s\n",
            "  XP_004355377.1 316 NKNNNQTKKKKSSSEDQTTN---------------------------------TNISSSSS 343\n",
            "                     11111111111111111111.................................12222222 PP\n",
            "\n",
            ">> XP_004359704.1  uncharacterized protein DFA_01739 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.6   4.9    0.0023      0.33     119     179 ..     143     201 ..      75     269 .. 0.56\n",
            "   2 ?    2.5   0.3      0.34        47     113     169 ..     550     591 ..     522     667 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.6 bits;  conditional E-value: 0.0023\n",
            "             Zip 119 sesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekl 179\n",
            "                     ++++  ++  +++ +++++ ++++++++ ++++++++++++q++++++++q++  +++++ \n",
            "  XP_004359704.1 143 QQQQCLERLAQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQ--QQQQQQ 201\n",
            "                     22222233333333333332222222222222222222222222222222211..111111 PP\n",
            "\n",
            "  == domain 2  score: 2.5 bits;  conditional E-value: 0.34\n",
            "             Zip 113 kasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeq 169\n",
            "                     +++++++ +++k +++n+++e +++               +++++++q++++ + ++\n",
            "  XP_004359704.1 550 QQDQQQKDCSTKMEIDNCQMEIDSQ---------------HHHQQQQQQQQQFNGKH 591\n",
            "                     2233333333333333333332222...............22222222222222222 PP\n",
            "\n",
            ">> XP_004360802.1  uncharacterized protein DFA_05081 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.2   9.0    0.0063      0.89      88     171 ..      35     127 ..      31     144 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.2 bits;  conditional E-value: 0.0063\n",
            "             Zip  88 llllveklltmivek.........ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +ll+ +k+ +m+ +k         + ++ ++ ++++++++++++ ++  +n++ ++  ++++++++++ ++ ++ ++++++ +++ ++ +q+ \n",
            "  XP_004360802.1  35 ALLFRDKTREMFDDKpsnknnnnnNTNQYNNHNNNQYNNRNNNNNQYNGYNNNNQYYNNNNNNNSQYNNNRYNTNNQQQQQNNTNNNNNNQQV 127\n",
            "                     667777777887777655555544445555555566666666666666666666666666666666666666666666666666666655543 PP\n",
            "\n",
            ">> XP_004353824.1  uncharacterized protein DFA_12187 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.3   2.6    0.0059      0.83     105     164 ..      65     125 ..      37     164 .. 0.58\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.3 bits;  conditional E-value: 0.0059\n",
            "             Zip 105 shkektpkkasssssesaakksvsn.dkveeseekedhvaeadtdadskeelrngkqekse 164\n",
            "                     +h +++ ++a +++s++ ++++  + + +ee+ee e+++ ++++ +d++ee++  k++k+ \n",
            "  XP_004353824.1  65 QHRRISLEQAINNTSTKVKQRHDGQeEDSEEEEEEEENDDQEEEYDDKEEEEQYYKENKQY 125\n",
            "                     4445554544444444433333333033333334444455555555555554444444333 PP\n",
            "\n",
            ">> XP_004359443.1  MADS-box transcription factor [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.8   8.2    0.0081       1.1     109     184 ..     153     223 ..      74     251 .. 0.57\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.8 bits;  conditional E-value: 0.0081\n",
            "             Zip 109 ktpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkv 184\n",
            "                      tp +++s++++++ ++++ +++ +++ +  + +++a+ +a    ++++++q ++++ ++q+++++++++++++++\n",
            "  XP_004359443.1 153 RTPTNTTSHHQAQQQQQQAAQQQAQQQAQQ-QAAQQAQQQA----QQQAQQQAHQQQMQQQQQQQQQHPSPTQQAI 223\n",
            "                     333334444443333333333333333332.1111111111....1222222222222222222222222222222 PP\n",
            "\n",
            ">> XP_004350327.1  uncharacterized protein DFA_11384 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.4   2.9    0.0053      0.75     134     171 ..      18      55 ..       8     105 .. 0.55\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.4 bits;  conditional E-value: 0.0053\n",
            "             Zip 134 eseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     e e++e++++e++++++ +ee+ ++++e +e+e+e+++\n",
            "  XP_004350327.1  18 EIEDGEEEEEEEEEEEEDGEEEEEEEEEEEEEEEEEED 55 \n",
            "                     33333333333333333333333333333333333222 PP\n",
            "\n",
            ">> XP_004362344.1  UAS domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.8   5.2    0.0083       1.2     102     170 ..     274     346 ..     261     402 .. 0.56\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.8 bits;  conditional E-value: 0.0083\n",
            "             Zip 102 k......ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     +      +++ k++  +++++++ e+a + s +  + +++++++d++ e+  d++ ++ ++++++e++e+e+ q+\n",
            "  XP_004362344.1 274 QpldetdSKDVKRQKRHTTEEEELEKAIQLSLQGATNQNKQDDQDEQDEQ--DQQDEQDEQEEQEEQDEQEELQE 346\n",
            "                     25544222222333333333333344444444444444444443333333..33333333333333344444442 PP\n",
            "\n",
            ">> XP_004354593.1  uncharacterized protein DFA_09239 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.3  11.6     0.012       1.6     100     172 ..      62     159 ..      27     263 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.3 bits;  conditional E-value: 0.012\n",
            "             Zip 100 vekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas....................... 171\n",
            "                     +++++ h+ +t+++ ++++++++ +++++   ++++  + ++ +++++++  + ++ n++  +s+s+++++s                       \n",
            "  XP_004354593.1  62 INQDHIHHVNTQSHNHHHHNHNNHHHHHHHMDMNDMNMNMNNTHHQQHNNHDQYNHLNDHYTSSSSSSSSSSspcrnnnnnnnnnnnnnnqpsli 156\n",
            "                     33378888899999999999999999999999999999888888888877777777777777766666666656666555555555555444444 PP\n",
            "\n",
            "             Zip 172 ..e 172\n",
            "                       +\n",
            "  XP_004354593.1 157 laS 159\n",
            "                     440 PP\n",
            "\n",
            ">> XP_004363124.1  uncharacterized protein DFA_03521 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.5   6.4    0.0099       1.4      98     171 ..     215     288 ..     208     377 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.5 bits;  conditional E-value: 0.0099\n",
            "             Zip  98 mivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +i++kk+  ++++++++++sss+s ++ + +n++ +++ + ++  ++      +++++ ++k+e +++ ke+++\n",
            "  XP_004363124.1 215 NIHKKKQPDSKIQEEEQNDSSSPSTTSTTTNNNNNNNNNDINSITESISNVTIQEQKEEEQKEEENDKPKEKEE 288\n",
            "                     44555666777778888888888888888888777777777554333333332222222222222222222222 PP\n",
            "\n",
            ">> XP_004359491.1  WD40-like domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.2   4.2     0.012       1.8     147     202 ..     526     579 ..     505     603 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.2 bits;  conditional E-value: 0.012\n",
            "             Zip 147 tdadskeelrngkqeksesekeqaseeedkeklaslkvraiilllglilHsfidGL 202\n",
            "                      +++s ++  n++ + ++s+++q  +++++++ ++++  + i   +++  s+idGL\n",
            "  XP_004359491.1 526 QEPNSNSNNNNNNNNNKSSKHQQ--HNHHHQQQHNHNHNQTIPATAVGGESIIDGL 579\n",
            "                     33333444444444444444444..4444455555555555555556666666655 PP\n",
            "\n",
            ">> XP_004359733.1  SET domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.5   3.2      0.01       1.4     108     179 ..      97     153 ..      68     232 .. 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.5 bits;  conditional E-value: 0.01\n",
            "             Zip 108 ektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekl 179\n",
            "                       ++k++++s++ +++k s +++k++e+ e e               +++++     +++ +++eee++e+ \n",
            "  XP_004359733.1  97 GANSKPKKKSKKDKKKKTSKKEKKTDEQGEAE---------------EQQQEDAAAVDATVKEEEEEEEEDG 153\n",
            "                     22222222222233333333333333322222...............2222222222222222222223333 PP\n",
            "\n",
            ">> XP_004366718.1  RhoGAP domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.9   4.4     0.015       2.2     126     175 ..     478     523 ..     449     575 .. 0.56\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.9 bits;  conditional E-value: 0.015\n",
            "             Zip 126 svsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeed 175\n",
            "                     + + + +e +++k    + +++++++ke+++++k+ek+++ekeq++++++\n",
            "  XP_004366718.1 478 QEERERIERDQKK----KDKKGEKEQKEKEKREKKEKEKKEKEQKKKDKS 523\n",
            "                     1111112222222....112233333333333333333333333332222 PP\n",
            "\n",
            ">> XP_004366832.1  uncharacterized protein DFA_02447 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.1   3.6     0.014       1.9      75     171 ..      39     164 ..      25     167 .] 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.1 bits;  conditional E-value: 0.014\n",
            "             Zip  75 hplagllvllGflllllveklltmivek.............................ksshkektpkkasssss..esaakksvsndkveeseek 138\n",
            "                     h+ +   + l +l+ + +++l  m+ ++                             k  ++    +k++++s    s+a+k++ ++++ ++e  \n",
            "  XP_004366832.1  39 HLTPLQSAKLNILVAYSLNSLFFMYLQTqgvsphnhkvkpeldrirhyitklqslskK--ESGVADEKEQKPSMtiDSEASKRIIQHSLASNEYI 131\n",
            "                     4555555555566666666666666666555555555777777777777766666432..22222223333322336667777777888888877 PP\n",
            "\n",
            "             Zip 139 edhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     ++++ + + + ++ ++  n++  ++++e+eq++\n",
            "  XP_004366832.1 132 NNNNNNNNNNTNNNTNNNNNNNFPPKKETEQKK 164\n",
            "                     777777777777777777777777777777754 PP\n",
            "\n",
            ">> XP_004358926.1  uncharacterized protein DFA_00951 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.3   3.5     0.011       1.6      92     147 ..      31     104 ..      23     109 .] 0.37\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.3 bits;  conditional E-value: 0.011\n",
            "             Zip  92 veklltmivek..................ksshkektpkkasssssesaakksvsndkveeseekedhvaeadt 147\n",
            "                     + ++++ +v+k                  +++ +++++ +++++ +++++++  +  ++++s+ ++++++++ +\n",
            "  XP_004358926.1  31 MKHIVSCLVSKkapvitslsfnnnivgnqNNNSNDNNNIEENKNGNNNDDQNNNNTTTTTTSTINNNNDTKDTK 104\n",
            "                     56666666666333333333333333332333333333333333333333333333333333333222222221 PP\n",
            "\n",
            ">> XP_004354601.1  uncharacterized protein DFA_09247 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.0   8.1   0.00043     0.061     103     175 ..      69     141 ..      39     216 .. 0.74\n",
            "   2 ?    0.2   1.8       1.6   2.3e+02      93     170 ..     930    1015 ..     905    1045 .. 0.46\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.0 bits;  conditional E-value: 0.00043\n",
            "             Zip 103 ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeed 175\n",
            "                     +ss +  +++  ++++ + + ++s +n++ +++ ++++++ + + ++++  +  n+ ++ + ++++ + ++ +\n",
            "  XP_004354601.1  69 SSSSSPVNNNINNHHHHHHRNSYSRNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNGKHNHNHNHNNNRSSMS 141\n",
            "                     4555666677778888888888888888888888877776666666665555555555555555544332222 PP\n",
            "\n",
            "  == domain 2  score: 0.2 bits;  conditional E-value: 1.6\n",
            "             Zip   93 eklltmivek.................ksshkektpkkasssssesaakksvsndkveeseek...............edhvaeadtdadske 153 \n",
            "                      e+  ++ +++                 ++++++++pk +s+ +s         ++++++s+                 +              \n",
            "  XP_004354601.1  930 EHSKKYQKSTkssitvqsnslasltlnNNNNHQIQPKGSSNTKS---------SSNITPSKYFkslyykflggnykdyN-------------- 999 \n",
            "                      44444444444444444422323333333333333333333333.........33333333333333333333333332.............. PP\n",
            "\n",
            "             Zip  154 elrngkqeksesekeqa 170 \n",
            "                        +n++ + ++++++ +\n",
            "  XP_004354601.1 1000 -SHNNHHHGHHNHHNIS 1015\n",
            "                      .3333333333333222 PP\n",
            "\n",
            ">> XP_004367084.1  uncharacterized protein DFA_07218 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.4   5.4     0.022       3.1      96     169 ..      25      95 ..      12     237 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.4 bits;  conditional E-value: 0.022\n",
            "             Zip  96 ltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeq 169\n",
            "                     l  +   ++++++ ++++++++++++++++s s++ + +++ k+++v++ +++++ k ++ +++ + + ++++ \n",
            "  XP_004367084.1  25 LLFF---NQQQQQYSSQPSTAKTNTKSSSSSSSSSALGSTKTKKQNVVQNKSNDPLKRSKNDNNNHNHNHNHSL 95 \n",
            "                     4444...3446777777777777888888888888888888899999999999998888888888888888764 PP\n",
            "\n",
            ">> XP_004366934.1  uncharacterized protein DFA_07061 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.0  10.7     0.014         2     101     172 ..      45     127 ..       8     158 .. 0.50\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.0 bits;  conditional E-value: 0.014\n",
            "             Zip 101 ekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas..............e 172\n",
            "                     +    +++ ++k+++++++es+  ++ ++++++  + + ++v+e+d++++ +ee+ ++++e++e+e+e+++              +\n",
            "  XP_004366934.1  45 DL---QSKSNKKEEEEEEEESDEDEQEKKVTSKLGKINIKQVKEEDEEEEDEEEEEEEEEEHQEEEEEEEElsefeeieqdwisdQ 127\n",
            "                     32...222222223333333333333333333333333444444444444444444444444444443333455666666666552 PP\n",
            "\n",
            ">> XP_004359218.1  leucine-rich repeat-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.5   6.2     0.021       2.9     105     206 ..     417     516 ..     375     524 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.5 bits;  conditional E-value: 0.021\n",
            "             Zip 105 shkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvraiilllglilHsfi 199\n",
            "                      ++++++++ +++ ++++  +  +nd+ + +  +++++  ++t+ ++++++++  ++++  +++++  + ++ + ++ +   ii+  ++  + +i\n",
            "  XP_004359218.1 417 TNNNNNNNNNNNNNENNENNDNNDNDNDT-TATNNNNDNTTATTTPTTTTTAATATPTPTTTTTTST-NNNNNNNSENEKSPIIVWQSIYPDLII 509\n",
            "                     12222222222222222222222222222.2333455556666666777777777777766666654.555555555555566666667777778 PP\n",
            "\n",
            "             Zip 200 dGLAlGv 206\n",
            "                     dGL lG+\n",
            "  XP_004359218.1 510 DGLYLGC 516\n",
            "                     8888874 PP\n",
            "\n",
            ">> XP_004351983.1  uncharacterized protein DFA_10097 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.9   5.2      0.03       4.3     103     186 ..    1453    1540 ..    1431    1610 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.9 bits;  conditional E-value: 0.03\n",
            "             Zip  103 ksshkektpkkasssssesaakksvsndkveeseek..........edhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvr 185 \n",
            "                      +++++e+ +++++++++e+++++s +n+k +++++k          ++++ + + ++++  +  n       ++ + ++++ ++    s+++r\n",
            "  XP_004351983.1 1453 NDKENEEDEDDEDDEEEEEDDEESKENQKKTGKKRKkkkitniikkKKKNNNNNNNNNNNNNNNN------NNNNNNNNNNNNNNLSGSQSTR 1539\n",
            "                      55555666666666666666666666666666665533332222222211111111111111111......1111111122233333444444 PP\n",
            "\n",
            "             Zip  186 a 186 \n",
            "                       \n",
            "  XP_004351983.1 1540 D 1540\n",
            "                      4 PP\n",
            "\n",
            ">> XP_004350793.1  uncharacterized protein DFA_11848 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.4   5.3     0.022       3.1     129     171 ..     403     445 ..     342     461 .. 0.57\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.4 bits;  conditional E-value: 0.022\n",
            "             Zip 129 ndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     ++  e++e+ ed+++e++++++++ee+ +++++ +e+e+e+++\n",
            "  XP_004350793.1 403 EVVDEDDEDEEDEEEEEEEEEEEEEEDEEEEEDEEEEEDEEEA 445\n",
            "                     3333333333333333333333333333333333333322222 PP\n",
            "\n",
            ">> XP_004350944.1  uncharacterized protein DFA_12005 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.0  11.5     0.029       4.1     139     172 ..      30      70 ..       5     121 .. 0.67\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.0 bits;  conditional E-value: 0.029\n",
            "             Zip 139 edhvaeadtdadskeelrngkqeksesekeqas.......e 172\n",
            "                       + +++++++++ ++++n + +++++++++ +       +\n",
            "  XP_004350944.1  30 LADHPSHHHNHPHNHNHHNHHDHHHHHHPHHHHphydhhhH 70 \n",
            "                     33445556666666666666666666666655543332211 PP\n",
            "\n",
            ">> XP_004358866.1  putative RNA helicase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.3   4.4     0.024       3.3      99     170 ..     136     206 ..     101     284 .. 0.62\n",
            "   2 ?    5.9   5.8     0.031       4.4     114     170 ..     454     510 ..     425     592 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.3 bits;  conditional E-value: 0.024\n",
            "             Zip  99 ivekksshkektpkkasssssesaakk..svsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     +ve+    ++++++k+sssss+++ k+    ++ + ee+e    ++++++++a +k ++++++q+ + ++ +++\n",
            "  XP_004358866.1 136 VVEE--TTSDTESEKKSSSSSKKNNKQgiVLHEREKEEKEGAIVDAKHKNKKA-KKGNQQQQQQQEDNKNRNSE 206\n",
            "                     4444..33333333333333333333300333333333333333333333333.33333333333333333322 PP\n",
            "\n",
            "  == domain 2  score: 5.9 bits;  conditional E-value: 0.031\n",
            "             Zip 114 asssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     ++++ ++ + +++v++d+ e++ee ed ++e++ ++d +ee+ +++ e +e+e+e +\n",
            "  XP_004358866.1 454 HYKKGQNDEEQEEVEQDEDEDQEEQEDAEEEEQQEEDGEEEEEEQVDEDQEEEEEME 510\n",
            "                     444444555555555555555555555554444444444444444444333333333 PP\n",
            "\n",
            ">> XP_004356887.1  cyclin [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.9  15.1      0.52        73     104     172 ..      75     150 ..      36     212 .. 0.55\n",
            "   2 ?   10.1   3.9    0.0016      0.23     106     164 ..     272     330 ..     256     422 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.9 bits;  conditional E-value: 0.52\n",
            "             Zip 104 sshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas........e 172\n",
            "                     +++++k++++++++ ++++ k +  n++++++ ++++++ + + ++++  +  n++ + + ++k+ ++        +\n",
            "  XP_004356887.1  75 GNKNNKQSPQHNNNNNNNNNKLTN-NSNITNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNKKNNKklksqsnsS 150\n",
            "                     333334444444444433333333.3333333333222222222222222222222222222222211122222230 PP\n",
            "\n",
            "  == domain 2  score: 10.1 bits;  conditional E-value: 0.0016\n",
            "             Zip 106 hkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqekse 164\n",
            "                     h+++++++ +++ ++++  +  +n++ +++ ++++++ + + ++++  +  n++ + + \n",
            "  XP_004356887.1 272 HNNNNNNNNNNNNNNNNNNNNSNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNDPN 330\n",
            "                     22222222222222222222222222222222222222222222222222222222222 PP\n",
            "\n",
            ">> XP_004359196.1  extracellular signal-regulated protein kinase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.8   5.1     0.034       4.7     113     169 ..      46     100 ..      20     171 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.8 bits;  conditional E-value: 0.034\n",
            "             Zip 113 kasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeq 169\n",
            "                      +++++++++a+++++++  ++ ++   ++ +++ ++ +++ + +++q++++ +++q\n",
            "  XP_004359196.1  46 LQQQQQAQQQAQQQAHQQAQQQAQQQ--QQIQQQQQQAHQQAQLQQQQQQQQAQQQQ 100\n",
            "                     11222222222222221111111111..11111111111111111111111111111 PP\n",
            "\n",
            ">> XP_004359778.1  N2227-like domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.1   4.9     0.027       3.8     104     159 ..       4      56 ..       1     169 [. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.1 bits;  conditional E-value: 0.027\n",
            "             Zip 104 sshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngk 159\n",
            "                     +sh+++ ++ +s+s+s+s++++s+++d  +++ ++   + + d +++++ee+ +++\n",
            "  XP_004359778.1   4 HSHSHDHSHSHSHSHSHSHDHSSHNCDDGDNDGDD---DISIDDQDNEDEEETDEN 56 \n",
            "                     33555555555555555555555444444333332...333333333333333333 PP\n",
            "\n",
            ">> XP_004360810.1  protein kinase 3 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.5   6.5      0.04       5.6     108     178 ..     186     253 ..     142     405 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.5 bits;  conditional E-value: 0.04\n",
            "             Zip 108 ektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkek 178\n",
            "                     ++ + + +++ss   +  s+s++++++s+ k+d+++e++ +++ ++e  ++ +  +e ekeq++   ++++\n",
            "  XP_004360810.1 186 QIANTNNNQPSSLYLSNASISSNSSNGSQSKKDKKKEKEREKEREKEREKEREREKEREKEQKK---QQQQ 253\n",
            "                     4444555555554444444444444444444333333322222222222222222222222211...1122 PP\n",
            "\n",
            ">> XP_004366559.1  uncharacterized protein DFA_04150 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.5   7.2     0.041       5.8     103     162 ..      55     117 ..      25     190 .. 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.5 bits;  conditional E-value: 0.041\n",
            "             Zip 103 ksshkektpkkasssssesaakksvsndkvee...seekedhvaeadtdadskeelrngkqek 162\n",
            "                     k++++++++++++++++++++ ++ ++++ ++   +++k+dh+++++ +++s  ++ n++ +k\n",
            "  XP_004366559.1  55 KHNQQQQQQQQQQQQQQHKDRDSHPEEENNQQdilKKRKRDHQQQQQNNNNSNDNDNNSSNNK 117\n",
            "                     333333333333333333333333333322221113333444444444444444444444333 PP\n",
            "\n",
            ">> XP_004355277.1  UPF0551 family protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.6   6.6     0.039       5.5     106     183 ..      77     154 ..      47     209 .. 0.60\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.6 bits;  conditional E-value: 0.039\n",
            "             Zip 106 hkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslk 183\n",
            "                     +++++pk++ +++ +s+ + +++ +   ++ + +++ +++++++  +e + n  q+ +  +++++s+++++e+ +++k\n",
            "  XP_004355277.1  77 KQQPQPKQQPKQHGHSHGGVECDGNHGVPKPQPKQQPKQHGHSHGGVECDGNHGQNNTAAASAAPSSSQQQEQQSNSK 154\n",
            "                     344555555555555555554444444445555677777777777777777777776666666666444444433333 PP\n",
            "\n",
            ">> XP_004350733.1  kinetochore protein Ndc80 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.5   5.5      0.08        11      86     172 ..     592     686 ..     589     730 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.5 bits;  conditional E-value: 0.08\n",
            "             Zip  86 flllllveklltmivek..ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas......e 172\n",
            "                     f+ ++ +ek+ ++i e+  k+   e+++  ++++s++ ++  s ++++++ se+k ++ +++  +++ k  +  ++ ek+ + k++++      +\n",
            "  XP_004350733.1 592 FIYFFCLEKIKQYIEEQieKKMSMEEENTIKENESKQLDDLLSDNKQQLKRSEQKLSNLENVSQREKDKNDKELKQYEKKFELKNRENemlkeqT 686\n",
            "                     67889999999999888986666677777778888888999999999999999996655554333322222222222222222222222233221 PP\n",
            "\n",
            ">> XP_004355651.1  uncharacterized protein DFA_08151 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.3   5.1     0.012       1.6      76     176 ..      12     104 ..       2     137 .. 0.50\n",
            "   2 ?    1.0   0.2      0.97   1.4e+02     144     177 ..     674     707 ..     654     727 .. 0.44\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.3 bits;  conditional E-value: 0.012\n",
            "             Zip  76 plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                       +++++l ++ l+l +  ++++  +++++     +  +sssss +++++s +n++ + + + ++  ++ +       ++ n++ ++s+ +k+ +\n",
            "  XP_004355651.1  12 QQQEIIALSSLSLVLATTDMVRYKQTSNHNYYSSFSFSSSSSSSSPSSSSSFNNHTNNHNNNHNN--HNNH-------NTNNNENSTSSFKKSIK 97 \n",
            "                     56889999999999999999998555533333333333333333333333333333333333222..2222.......22222222222222222 PP\n",
            "\n",
            "             Zip 171 s.eeedk 176\n",
            "                     s +++++\n",
            "  XP_004355651.1  98 SfKHHNH 104\n",
            "                     2122333 PP\n",
            "\n",
            "  == domain 2  score: 1.0 bits;  conditional E-value: 0.97\n",
            "             Zip 144 eadtdadskeelrngkqeksesekeqaseeedke 177\n",
            "                      ++t++d + +   ++ +++ s+++q++e+e++e\n",
            "  XP_004355651.1 674 TTKTTKDGSISNPIDSNHHHLSSQQQEQEQEQQE 707\n",
            "                     2222222222222222222222222222111111 PP\n",
            "\n",
            ">> XP_004358977.1  pleckstrin domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.6  12.3     0.074        10     107     171 ..     181     245 ..     156     302 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.6 bits;  conditional E-value: 0.074\n",
            "             Zip 107 kektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +  tp+  s+sss +++kk+ +++k  e+e+ke++++e +++++ ++e +++k++ +e eke+++\n",
            "  XP_004358977.1 181 SSFTPSSGSNSSSSKDKKKEKEKEKEREKERKEEKEKERKEEKEREKERKEEKEKEKEREKEKEK 245\n",
            "                     44455555555555555555555555555555443333333333222222222222222222222 PP\n",
            "\n",
            ">> XP_004367225.1  putative mediator complex subunit 12 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.2   6.4       0.1        14     107     171 ..      23      85 ..       9     163 .. 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.2 bits;  conditional E-value: 0.1\n",
            "             Zip 107 kektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     ++++++++++++++++++++ ++ +++++++ ++  +++++++ + ++++ ++q++++++++q++\n",
            "  XP_004367225.1  23 QQQQQQQQQQQQQHQHRVQQLQQMQIQQMQQQQQ--QQQHMQQLQMQQQQMQQQQQQQQQQMQQQ 85 \n",
            "                     3445555555555555555555555555555433..34444444444444444444444444433 PP\n",
            "\n",
            ">> XP_004356343.1  uncharacterized protein DFA_08860 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.5   1.7    0.0012      0.17     116     192 ..     541     614 ..     490     632 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.5 bits;  conditional E-value: 0.0012\n",
            "             Zip 116 ssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvraiilllg 192\n",
            "                     ++++e +++ + ++++++++ee e+ ++ ++++++++ee++++++e s  +++q  +++  ek+ ++ + ++ +++g\n",
            "  XP_004356343.1 541 EKTKEDDTVTTQPEEPIKKEEETEK-KELVEENKPKEEEEKKKEEEESTTTQQQ--QQDVVEKIVTITIYTLSFTFG 614\n",
            "                     3344566666666666666666544.4555566666666666666666666665..567888888888888888877 PP\n",
            "\n",
            ">> XP_004350761.1  transcription initiation factor TFIID subunit [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.6   8.5     0.075        11     103     171 ..      38     106 ..       6     189 .. 0.60\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.6 bits;  conditional E-value: 0.075\n",
            "             Zip 103 ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     ++++++++  + +s  +e + ++  +++  ++++e+ ++++e++ ++ +k+ l n++ + ++++++ ++\n",
            "  XP_004350761.1  38 NGNNNNNNRYDNDSTMKEYNEQDNEDEHHNNNNKENGKKKEEYKVSNITKTALLNNNNNNHSNGSNNTA 106\n",
            "                     122222222222222222222222223333333333444455555555555555555555555555554 PP\n",
            "\n",
            ">> XP_004350810.1  HEAT repeat-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.0   0.5       3.8   5.3e+02     128     170 ..     363     403 ..     310     461 .. 0.57\n",
            "   2 ?    9.0   3.8    0.0035      0.49     107     171 ..     959    1021 ..     909    1139 .. 0.68\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.0 bits;  conditional E-value: 3.8\n",
            "             Zip 128 sndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     +++++ e ee e++++ ++ +++  ee+ ++k   +e+e++++\n",
            "  XP_004350810.1 363 EEENIVELEEIEKDGEDEQDEQE--EEEEDKKVIVEEEEDQRN 403\n",
            "                     33333333333222222222222..222222222222222222 PP\n",
            "\n",
            "  == domain 2  score: 9.0 bits;  conditional E-value: 0.0035\n",
            "             Zip  107 kektpkkasssssesaakksvsndkveeseek...edhvaeadtdadskeelrngkqeksesekeqas 171 \n",
            "                      +++++kk+++++++++  ++++ d+v e++e    e +++++d+d++s+ee+ ++++e   +e++++s\n",
            "  XP_004350810.1  959 QRESEKKKNQPKPQQD--QEMKVDQVVEEKEEekkEVEGQQEDSDDESDEEEEDEEEE---EEEKKES 1021\n",
            "                      3333333333333333..3333333333333300033344444444444444433333...3333333 PP\n",
            "\n",
            ">> XP_004357732.1  NPR2 family protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.8   6.7     0.064         9     115     171 ..     129     185 ..      90     236 .. 0.53\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.8 bits;  conditional E-value: 0.064\n",
            "             Zip 115 sssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +++s++++   + ++++ +++++ ++++ ++++++++++++++++q++++s+++++ \n",
            "  XP_004357732.1 129 QQHSTNQQILLQQQQQQQQQQQQIQQQQMQQQMQQQQQQQQQQQQQQQQQSQQQSTT 185\n",
            "                     222222222333333333333333333333333333333333222222222222222 PP\n",
            "\n",
            ">> XP_004355597.1  NUC153 domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.9   7.0     0.059       8.4     109     172 ..     177     245 ..     108     322 .. 0.69\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.9 bits;  conditional E-value: 0.059\n",
            "             Zip 109 ktpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas......e 172\n",
            "                      ++ k++++ ++ a+ +s++++ +++ +++ + +++++++++++++++n++ +k+ ++ ++ +      e\n",
            "  XP_004355597.1 177 SQEFKKHNPAAKLAKLDSIYKEHFQNVQDDDE-DENEEEEEEKSTTKKNNNNKKNNNNNNRMEidsdedE 245\n",
            "                     34445556666666666666666666666533.2333333333333333333333333222222222220 PP\n",
            "\n",
            ">> XP_004366330.1  uncharacterized protein DFA_03920 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.2   5.5     0.051       7.2     109     180 ..      34      92 ..       3      98 .] 0.51\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.2 bits;  conditional E-value: 0.051\n",
            "             Zip 109 ktpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekla 180\n",
            "                     +  ++++++++ ++ ++             ed ++++d d+d++ e++ +  +++++e+eq++eee++e+  \n",
            "  XP_004366330.1  34 DDDDDEDEDEEDEEEGE-------------EDYDVGYDVDDDQEVEDAVDAFPPEDEEDEQEEEEEEEEHRG 92 \n",
            "                     33333333333333333.............445555555555555555555555555555554444443333 PP\n",
            "\n",
            ">> XP_004359062.1  uncharacterized protein DFA_01087 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.7   9.1     0.073        10     114     178 ..      49     114 ..      31     194 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.7 bits;  conditional E-value: 0.073\n",
            "             Zip 114 asssssesaakksvsndkveeseek.edhvaeadtdadskeelrngkqeksesekeqaseeedkek 178\n",
            "                     +++ +++++ + + ++++++++++   ++++++++ a ++++l++++q+++++++ q++++  +++\n",
            "  XP_004359062.1  49 QQQYQQYQQQQLHQQQQQLQQQQQLqLQQQQQQQHAAAQQAQLQQQQQHHHHQQQLQQQHMNGSSS 114\n",
            "                     333344444444444444444444314555566666677777777777777777777664444333 PP\n",
            "\n",
            ">> XP_004362923.1  uncharacterized protein DFA_03318 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.2   7.3       0.1        14     122     181 ..     175     230 ..     134     348 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.2 bits;  conditional E-value: 0.1\n",
            "             Zip 122 aakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklas 181\n",
            "                     ++ ++ s +++++   + +    +d  + +kee+++++q++++++++ + ++ ++e++++\n",
            "  XP_004362923.1 175 STTSTTSTSSTTTVDTGSS----DDVIDATKEEQQQQQQQQQQQQQQDNTNNNQEEEISN 230\n",
            "                     2222222222222222222....2222222222222222222222222222222222222 PP\n",
            "\n",
            ">> XP_004350745.1  WD40 repeat-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.3   4.1     0.012       1.6      93     180 ..       3      87 ..       1     179 [. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.3 bits;  conditional E-value: 0.012\n",
            "             Zip  93 eklltmivekksshkektpkkasssssesaakksvsndkve.eseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekla 180\n",
            "                      k+ ++i+e++++  ++ + ++++s  + ++ k++sn++ + +++e ed+++ ++ + ++k ++ ++++++++ +++ ++    + k +\n",
            "  XP_004350745.1   3 SKINHLIKERQQHPTQTRNIDHYTSIIQINRDKEMSNNNNNnNKREVEDQESLENQSSRKKVKEDESSTSTTSTTTTNKK----NVKKQ 87 \n",
            "                     57889999998888888888888887766666666555554045556777777766666666666666666666655544....22222 PP\n",
            "\n",
            ">> XP_004366514.1  uncharacterized protein DFA_04104 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.3   5.9     0.092        13     134     179 ..     158     198 ..     110     254 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.3 bits;  conditional E-value: 0.092\n",
            "             Zip 134 eseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekl 179\n",
            "                     +++e   +++e + +++++ee++n++q  +e+e+     ++d+++ \n",
            "  XP_004366514.1 158 DEREE--EEEEIQDSNQEEEEHHNSSQYDEEEEEDD---DDDESHN 198\n",
            "                     22221..11112112222222222222211111111...1111111 PP\n",
            "\n",
            ">> XP_004367319.1  uncharacterized protein DFA_07460 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.7   1.7     0.018       2.5     106     171 ..      15      80 ..       1     205 [. 0.63\n",
            "   2 ?    0.3   0.3       1.5   2.1e+02     126     172 ..    4935    4996 ..    4898    5064 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.7 bits;  conditional E-value: 0.018\n",
            "             Zip 106 hkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +ke+ + + ++ ++ s+++ s +++ +++s+  ++++++++ ++++  + +++++++++s+++ ++\n",
            "  XP_004367319.1  15 AKETDHADIQTTDTSSSSSPSSHSNASTDSSALQQQQQQQQQEDNNNNNNKKEEEKQEKSIQQDNQ 80 \n",
            "                     333333333333334444444444444444444444444444444445555555555555554433 PP\n",
            "\n",
            "  == domain 2  score: 0.3 bits;  conditional E-value: 1.5\n",
            "             Zip  126 svsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas.................e 172 \n",
            "                        + +k++e+ee e  +++++ d++ +e l  ++++ +++e+++++                 e\n",
            "  XP_004367319.1 4935 NDHTKKLKEKEEEE--KTKQQLDESDEEVLTIEEKQGEGEEEQEKKnpelnfkyydnnycfldE 4996\n",
            "                      33333444444433..333444444444444444444444444433445555555555555542 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (333 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      3525  (0.290482); expected 242.7 (0.02)\n",
            "Passed bias filter:                     3134  (0.258261); expected 242.7 (0.02)\n",
            "Passed Vit filter:                      1619  (0.133416); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                      1090  (0.0898228); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):              86  [number of targets reported over threshold]\n",
            "# CPU time: 14.64u 0.05s 00:00:14.69 Elapsed: 00:00:13.29\n",
            "# Mc/sec: 169.46\n",
            "//\n",
            "Query:       ZIP4_domain  [M=171]\n",
            "Accession:   PF18292.4\n",
            "Description: Zinc transporter ZIP4 domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "       0.07   12.4   0.0       0.07   12.4   0.0    3.0  3  XP_004361063.1  uncharacterized protein DFA_05344 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004361063.1  uncharacterized protein DFA_05344 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -4.5   1.8      0.86     1e+04      76     133 ..      51      83 ..      30     105 .. 0.52\n",
            "   2 ?   -2.4   2.9      0.19   2.4e+03      89     124 ..     265     303 ..     250     334 .. 0.55\n",
            "   3 ?   12.4   0.0   5.8e-06      0.07      90     150 ..     707     766 ..     687     775 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -4.5 bits;  conditional E-value: 0.86\n",
            "                     HHHHHHHHHHHH...........HSTTTHHHH.HHHHHHHHHHC---------B-HHH CS\n",
            "     ZIP4_domain  76 waseadhllyklyldsllslrqdedsealsqselerlLqriqqhyrvsnsqaCvdaeq 133\n",
            "                     ++s++++ l +l  +  l+++++e++e ++                          +q\n",
            "  XP_004361063.1  51 NKSTSSSSLISLLIKRRLEVEEKEEKESIK-------------------------HHQ 83 \n",
            "                     233333333333333333444444333333.........................333 PP\n",
            "\n",
            "  == domain 2  score: -2.4 bits;  conditional E-value: 0.19\n",
            "                     ..........HSTTTHHH...H.HHHHHHHHHHC----- CS\n",
            "     ZIP4_domain  89 ldsllslrqdedsealsq...selerlLqriqqhyrvsn 124\n",
            "                      ++l +++q+++++ l +   ++l +lL++ q++ +++n\n",
            "  XP_004361063.1 265 SENLSKFQQQQQQQPLHHastGNLHSLLNQSQNNHNHNN 303\n",
            "                     344555555555555544000445555555553333333 PP\n",
            "\n",
            "  == domain 3  score: 12.4 bits;  conditional E-value: 5.8e-06\n",
            "                     .........HSTTTHHHH.HHHHHHHHH.HC---------B-HHHHHHHCSSTT----HHH. CS\n",
            "     ZIP4_domain  90 dsllslrqdedsealsqselerlLqriq.qhyrvsnsqaCvdaeqlleeaniagaesspgad 150\n",
            "                       ll +++d+++e l +++l++l ++iq +  + ++ q+C++++ + ++ +  + +ss ga+\n",
            "  XP_004361063.1 707 SHLLIYNNDKEEELLHKKSLSDL-TSIQlTPLTKEGYQNCFTVSFVYTKNET-SPSSSGGAS 766\n",
            "                     57999***************997.5666599999999***********9997.788888885 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (171 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       797  (0.0656778); expected 242.7 (0.02)\n",
            "Passed bias filter:                      259  (0.0213432); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        17  (0.00140091); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.15u 0.00s 00:00:00.15 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 13477.29\n",
            "//\n",
            "Query:       ZipA_C  [M=128]\n",
            "Accession:   PF04354.16\n",
            "Description: ZipA, C-terminal FtsZ-binding domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (128 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       289  (0.0238154); expected 242.7 (0.02)\n",
            "Passed bias filter:                      192  (0.015822); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        11  (0.000906469); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 13988.77\n",
            "//\n",
            "Query:       ZirS_C  [M=145]\n",
            "Accession:   PF16583.8\n",
            "Description: Zinc-regulated secreted antivirulence protein C-terminal domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.011   14.8   0.3      0.014   14.4   0.3    1.2  1  XP_004361966.1  uncharacterized protein DFA_06254 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004361966.1  uncharacterized protein DFA_06254 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   14.4   0.3   1.1e-06     0.014      55     111 ..      47     103 ..      26     112 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.4 bits;  conditional E-value: 1.1e-06\n",
            "                     ----B-TT-EEEEEEE-BBTTS-B------B--EE-S-SSSEEEE---EE-SSSSEE CS\n",
            "          ZirS_C  55 dkavlspgesatittlvkdvdgnpvnevminkavarenlkglwdigplkkevvpgky 111\n",
            "                        v s    at      d+d n    +++  +++++n     ++ p+ k+ +pg+ \n",
            "  XP_004361966.1  47 VVPVSSGSSGATTFNYNIDIDSNDFYGIYVGTTTSKNNKCNNNNFNPIDKGLGPGHV 103\n",
            "                     445555555677778889*************************************75 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (145 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       485  (0.039967); expected 242.7 (0.02)\n",
            "Passed bias filter:                      218  (0.0179646); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        12  (0.000988875); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 10949.07\n",
            "//\n",
            "Query:       Zmiz1_N  [M=99]\n",
            "Accession:   PF18028.4\n",
            "Description: Zmiz1 N-terminal tetratricopeptide repeat domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (99 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       371  (0.0305727); expected 242.7 (0.02)\n",
            "Passed bias filter:                      229  (0.018871); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        13  (0.00107128); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.01s 00:00:00.12 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 10112.51\n",
            "//\n",
            "Query:       Zn-C2H2_12  [M=27]\n",
            "Accession:   PF18112.4\n",
            "Description: Autophagy receptor zinc finger-C2H2 domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    4.7e-08   32.7   5.3       0.32   10.8   0.1    3.9  3  XP_004350244.1  uncharacterized protein DFA_11301 [Cavenderia\n",
            "     0.0068   16.2   0.1      0.017   14.9   0.1    1.7  1  XP_004350243.1  uncharacterized protein DFA_11300 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "      0.025   14.4   7.4        3.4    7.6   0.4    4.1  3  XP_004350436.1  C2H2-type zinc finger-containing protein [Cav\n",
            "      0.027   14.3   0.1      0.073   12.9   0.1    1.7  1  XP_004350270.1  uncharacterized protein DFA_11327 [Cavenderia\n",
            "      0.083   12.7   0.1       0.22   11.3   0.1    1.7  1  XP_004360378.1  Zinc finger protein [Cavenderia fasciculata]\n",
            "       0.71    9.7   3.1       0.72    9.7   0.5    2.4  2  XP_004350269.1  uncharacterized protein DFA_11326 [Cavenderia\n",
            "        2.6    7.9   5.0       0.93    9.3   1.6    2.1  2  XP_004355866.1  uncharacterized protein DFA_08377 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004350244.1  uncharacterized protein DFA_11301 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   10.8   0.1   0.00018      0.32       3      14 ..      20      31 ..      19      36 .. 0.93\n",
            "   2 !   10.8   0.1   0.00018      0.32       3      14 ..    1154    1165 ..    1153    1170 .. 0.93\n",
            "   3 !   10.8   0.1   0.00018      0.32       3      14 ..    1791    1802 ..    1790    1807 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.8 bits;  conditional E-value: 0.00018\n",
            "      Zn-C2H2_12  3 CPlCsevFskev 14\n",
            "                    CPlC+e Fsk+v\n",
            "  XP_004350244.1 20 CPLCKEDFSKDV 31\n",
            "                    **********98 PP\n",
            "\n",
            "  == domain 2  score: 10.8 bits;  conditional E-value: 0.00018\n",
            "      Zn-C2H2_12    3 CPlCsevFskev 14  \n",
            "                      CPlC+e Fsk+v\n",
            "  XP_004350244.1 1154 CPLCKEDFSKDV 1165\n",
            "                      **********98 PP\n",
            "\n",
            "  == domain 3  score: 10.8 bits;  conditional E-value: 0.00018\n",
            "      Zn-C2H2_12    3 CPlCsevFskev 14  \n",
            "                      CPlC+e Fsk+v\n",
            "  XP_004350244.1 1791 CPLCKEDFSKDV 1802\n",
            "                      **********98 PP\n",
            "\n",
            ">> XP_004350243.1  uncharacterized protein DFA_11300 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   14.9   0.1   9.7e-06     0.017       3      14 ..      25      36 ..      24      43 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.9 bits;  conditional E-value: 9.7e-06\n",
            "      Zn-C2H2_12  3 CPlCsevFskev 14\n",
            "                    CPlC+e Fsk+v\n",
            "  XP_004350243.1 25 CPLCKEDFSKDV 36\n",
            "                    **********98 PP\n",
            "\n",
            ">> XP_004350436.1  C2H2-type zinc finger-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.6   0.4     0.002       3.4       3      22 ..     165     187 ..     164     187 .. 0.67\n",
            "   2 ?    0.7   0.2      0.27   4.7e+02       6      13 ..     219     226 ..     216     234 .. 0.78\n",
            "   3 ?    7.2   0.1    0.0025       4.4       3      16 ..     375     388 ..     374     394 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.6 bits;  conditional E-value: 0.002\n",
            "      Zn-C2H2_12   3 CPlCsevFs...kevdqdeFeeH 22 \n",
            "                     CP+C e F+   + v+  +Fe H\n",
            "  XP_004350436.1 165 CPMCLEEFKgdrSAVSSHLFEQH 187\n",
            "                     ********833334666666666 PP\n",
            "\n",
            "  == domain 2  score: 0.7 bits;  conditional E-value: 0.27\n",
            "      Zn-C2H2_12   6 CsevFske 13 \n",
            "                     C+++F+++\n",
            "  XP_004350436.1 219 CEKIFKSQ 226\n",
            "                     99***975 PP\n",
            "\n",
            "  == domain 3  score: 7.2 bits;  conditional E-value: 0.0025\n",
            "      Zn-C2H2_12   3 CPlCsevFskevdq 16 \n",
            "                     CP+C+   + e d \n",
            "  XP_004350436.1 375 CPMCGLKHQDEADL 388\n",
            "                     *****988877665 PP\n",
            "\n",
            ">> XP_004350270.1  uncharacterized protein DFA_11327 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.9   0.1   4.2e-05     0.073       3      14 ..      20      31 ..      19      37 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.9 bits;  conditional E-value: 4.2e-05\n",
            "      Zn-C2H2_12  3 CPlCsevFskev 14\n",
            "                    CPlC+e Fsk+v\n",
            "  XP_004350270.1 20 CPLCKEDFSKDV 31\n",
            "                    **********98 PP\n",
            "\n",
            ">> XP_004360378.1  Zinc finger protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.3   0.1   0.00013      0.22       1      11 [.    1201    1211 ..    1201    1220 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.3 bits;  conditional E-value: 0.00013\n",
            "      Zn-C2H2_12    1 KeCPlCsevFs 11  \n",
            "                      K+CPlCs++F \n",
            "  XP_004360378.1 1201 KTCPLCSKSFN 1211\n",
            "                      89********6 PP\n",
            "\n",
            ">> XP_004350269.1  uncharacterized protein DFA_11326 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.7   0.5   0.00041      0.72       1      13 [.      18      30 ..      18      33 .. 0.88\n",
            "   2 ?   -2.2   0.1       2.2   3.8e+03       4       9 ..     586     591 ..     585     591 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.7 bits;  conditional E-value: 0.00041\n",
            "      Zn-C2H2_12  1 KeCPlCsevFske 13\n",
            "                    K CPlC+e +s e\n",
            "  XP_004350269.1 18 KLCPLCKEDYSNE 30\n",
            "                    78********986 PP\n",
            "\n",
            "  == domain 2  score: -2.2 bits;  conditional E-value: 2.2\n",
            "      Zn-C2H2_12   4 PlCsev 9  \n",
            "                     P+Cs++\n",
            "  XP_004350269.1 586 PICSKI 591\n",
            "                     9***96 PP\n",
            "\n",
            ">> XP_004355866.1  uncharacterized protein DFA_08377 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -3.6   0.1       5.9     1e+04      19      26 ..     410     417 ..     407     417 .. 0.82\n",
            "   2 ?    9.3   1.6   0.00054      0.93       3      27 .]     535     556 ..     534     556 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -3.6 bits;  conditional E-value: 5.9\n",
            "      Zn-C2H2_12  19 FeeHVEsH 26 \n",
            "                     ++ H  +H\n",
            "  XP_004355866.1 410 LQQHLQQH 417\n",
            "                     78999999 PP\n",
            "\n",
            "  == domain 2  score: 9.3 bits;  conditional E-value: 0.00054\n",
            "      Zn-C2H2_12   3 CPlCsevFskevdqdeFeeHVEsHF 27 \n",
            "                     CP+C   F +e + +    HV sHF\n",
            "  XP_004355866.1 535 CPMCTRKFATEPEIN---QHVNSHF 556\n",
            "                     **********98765...7*****9 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (27 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       394  (0.0324681); expected 242.7 (0.02)\n",
            "Passed bias filter:                      293  (0.024145); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        53  (0.00436753); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000576844); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               7  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 3080.19\n",
            "//\n",
            "Query:       zn-ribbon_14  [M=32]\n",
            "Accession:   PF16503.8\n",
            "Description: Zinc-ribbon\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    9.3e-13   46.9   6.7    9.3e-13   46.9   6.7    3.3  5  XP_004356186.1  ATP-binding domain-containing protein 3 [Cave\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004356186.1  ATP-binding domain-containing protein 3 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -3.9   0.3      0.56   6.9e+03       3       7 ..      33      37 ..      32      38 .. 0.71\n",
            "   2 ?   -1.6   0.1      0.11   1.3e+03      12      18 ..      50      56 ..      48      57 .. 0.78\n",
            "   3 ?    0.3   0.1     0.028   3.4e+02       1       8 [.     166     173 ..     166     179 .. 0.83\n",
            "   4 ?   -3.6   0.0      0.47   5.7e+03      21      31 ..     275     285 ..     274     285 .. 0.84\n",
            "   5 !   46.9   6.7   7.7e-17   9.3e-13       1      32 []     308     339 ..     308     339 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -3.9 bits;  conditional E-value: 0.56\n",
            "    zn-ribbon_14  3 CerCG 7 \n",
            "                    Ce C \n",
            "  XP_004356186.1 33 CELCK 37\n",
            "                    88884 PP\n",
            "\n",
            "  == domain 2  score: -1.6 bits;  conditional E-value: 0.11\n",
            "    zn-ribbon_14 12 qkiCkAC 18\n",
            "                    q iCk C\n",
            "  XP_004356186.1 50 QMICKEC 56\n",
            "                    7799988 PP\n",
            "\n",
            "  == domain 3  score: 0.3 bits;  conditional E-value: 0.028\n",
            "    zn-ribbon_14   1 gkCerCGy 8  \n",
            "                     ++C+ CG \n",
            "  XP_004356186.1 166 NNCTFCGV 173\n",
            "                     69*****7 PP\n",
            "\n",
            "  == domain 4  score: -3.6 bits;  conditional E-value: 0.47\n",
            "    zn-ribbon_14  21 LegLnkgrpki 31 \n",
            "                     L+ L++ rp i\n",
            "  XP_004356186.1 275 LKDLEAVRPSI 285\n",
            "                     999**999976 PP\n",
            "\n",
            "  == domain 5  score: 46.9 bits;  conditional E-value: 7.7e-17\n",
            "    zn-ribbon_14   1 gkCerCGylSSqkiCkACvLLegLnkgrpkia 32 \n",
            "                     ++C+ CGy++S++iC+AC +L++Ln+g++k+a\n",
            "  XP_004356186.1 308 RNCNLCGYICSNEICMACDFLKNLNSGKAKYA 339\n",
            "                     59****************************85 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (32 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       363  (0.0299135); expected 242.7 (0.02)\n",
            "Passed bias filter:                      211  (0.0173877); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        15  (0.00123609); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 3281.24\n",
            "//\n",
            "Query:       Zn-ribbon_8  [M=40]\n",
            "Accession:   PF09723.13\n",
            "Description: Zinc ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "      0.001   18.6   2.3      0.001   18.6   2.3    2.7  2  XP_004350709.1  uncharacterized protein DFA_11762 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "      0.025   14.2   1.5      0.025   14.2   1.5    5.8  7  XP_004359993.1  LIM domain protein [Cavenderia fasciculata]\n",
            "      0.036   13.7   0.5       0.11   12.2   0.5    1.8  1  XP_004360556.1  MCM family protein [Cavenderia fasciculata]\n",
            "      0.082   12.5   0.4       0.15   11.7   0.4    1.4  1  XP_004358406.1  ubiquitin [Cavenderia fasciculata]\n",
            "        0.1   12.2  13.0        2.8    7.6   0.7    4.5  4  XP_004356292.1  uncharacterized protein DFA_08808 [Cavenderia\n",
            "       0.17   11.5   0.6       0.34   10.5   0.6    1.4  1  XP_004362968.1  putative GATA-binding transcription factor [C\n",
            "       0.24   11.0   1.8        1.1    9.0   0.2    2.4  2  XP_004362557.1  leucyl-tRNA synthetase [Cavenderia fasciculat\n",
            "       0.48   10.1   3.4         19    4.9   0.1    3.4  4  XP_004355856.1  ZPR1-type zinc finger-containing protein [Cav\n",
            "       0.63    9.7   8.5        1.9    8.2   1.3    3.4  3  XP_004350735.1  uncharacterized protein DFA_11790 [Cavenderia\n",
            "        1.3    8.7   8.5          8    6.2   4.6    2.2  2  XP_004356125.1  uncharacterized protein DFA_08637 [Cavenderia\n",
            "        1.8    8.2   6.7       0.44   10.2   0.7    2.6  2  XP_004350066.1  uncharacterized protein DFA_11123 [Cavenderia\n",
            "          4    7.1   7.5        8.2    6.1   7.5    1.5  1  XP_004366572.1  Putative transposase [Cavenderia fasciculata]\n",
            "        4.2    7.0   7.8        2.9    7.5   0.5    3.0  2  XP_004355683.1  uncharacterized protein DFA_08186 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004350709.1  uncharacterized protein DFA_11762 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.7   1.7       2.5   2.3e+03       7      34 ..     278     284 ..     271     291 .. 0.70\n",
            "   2 !   18.6   2.3   1.1e-06     0.001       6      33 ..     299     327 ..     298     330 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.7 bits;  conditional E-value: 2.5\n",
            "     Zn-ribbon_8   7 rCeeCgheFevlqkmsdapltaCPeCgs 34 \n",
            "                                           C eC++\n",
            "  XP_004350709.1 278 D---------------------CSECHA 284\n",
            "                     4.....................555554 PP\n",
            "\n",
            "  == domain 2  score: 18.6 bits;  conditional E-value: 1.1e-06\n",
            "     Zn-ribbon_8   6 YrCeeCgheFevlqkms.dapltaCPeCg 33 \n",
            "                     + C++C ++F +l  +  d++++ CP+C \n",
            "  XP_004350709.1 299 FACKKCSKVFRKLINAEfDESDEYCPHCD 327\n",
            "                     89***************9999999****6 PP\n",
            "\n",
            ">> XP_004359993.1  LIM domain protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.5   0.2      0.51   4.8e+02      29      34 ..       8      13 ..       2      19 .. 0.64\n",
            "   2 ?    1.7   0.8      0.21   1.9e+02       8      32 ..       8      35 ..       6      35 .. 0.62\n",
            "   3 ?    2.9   6.7      0.09        84       8      32 ..      35      59 ..      33      79 .. 0.74\n",
            "   4 ?   -0.5   1.7         1   9.6e+02       7      16 ..      66      75 ..      66      93 .. 0.52\n",
            "   5 ?   14.2   1.5   2.7e-05     0.025       6      38 ..      92     123 ..      86     125 .. 0.78\n",
            "   6 ?   -2.8   1.1       5.4   5.1e+03       6      12 ..     137     144 ..     135     148 .. 0.75\n",
            "   7 ?    4.1   1.8     0.037        34       6      34 ..     164     195 ..     163     196 .] 0.76\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.5 bits;  conditional E-value: 0.51\n",
            "     Zn-ribbon_8 29 CPeCgs 34\n",
            "                    C++C++\n",
            "  XP_004359993.1  8 CKRCQK 13\n",
            "                    666654 PP\n",
            "\n",
            "  == domain 2  score: 1.7 bits;  conditional E-value: 0.21\n",
            "     Zn-ribbon_8  8 CeeCgheFe...vlqkmsdapltaCPeC 32\n",
            "                    C++C++  +    ++ m+ +   +C  C\n",
            "  XP_004359993.1  8 CKRCQKNIDsgeSIHAMDADWHSDCWGC 35\n",
            "                    9999999885555555553334556555 PP\n",
            "\n",
            "  == domain 3  score: 2.9 bits;  conditional E-value: 0.09\n",
            "     Zn-ribbon_8  8 CeeCgheFevlqkmsdapltaCPeC 32\n",
            "                    C eC++ Fe     +   l+ C+ C\n",
            "  XP_004359993.1 35 CAECHTAFETQSFYKVQGLPICKNC 59\n",
            "                    *********5444445556678887 PP\n",
            "\n",
            "  == domain 4  score: -0.5 bits;  conditional E-value: 1\n",
            "     Zn-ribbon_8  7 rCeeCgheFe 16\n",
            "                    +C++C +   \n",
            "  XP_004359993.1 66 KCKKCSQPIA 75\n",
            "                    5777775544 PP\n",
            "\n",
            "  == domain 5  score: 14.2 bits;  conditional E-value: 2.7e-05\n",
            "     Zn-ribbon_8   6 YrCeeCgheFevlqkmsdapltaCPeCgsevrr 38 \n",
            "                     + C +C++  ++   ++d  ++ CP+C+ +++ \n",
            "  XP_004359993.1  92 FTCSNCNKNIDANFFIKD-TNILCPQCNDQLKE 123\n",
            "                     66*********8876665.4566****988775 PP\n",
            "\n",
            "  == domain 6  score: -2.8 bits;  conditional E-value: 5.4\n",
            "     Zn-ribbon_8   6 YrCee.Cg 12 \n",
            "                     Y C++ C+\n",
            "  XP_004359993.1 137 YVCKKpCN 144\n",
            "                     78887776 PP\n",
            "\n",
            "  == domain 7  score: 4.1 bits;  conditional E-value: 0.037\n",
            "     Zn-ribbon_8   6 YrCeeCgheFe.vlqkms.daplta.CPeCgs 34 \n",
            "                     ++C +C++e   ++ + s  +++   C +C+s\n",
            "  XP_004359993.1 164 FKCVKCHNEIKgEFYTLSdVKSKNYvCMACNS 195\n",
            "                     89*****9999977777754455666999986 PP\n",
            "\n",
            ">> XP_004360556.1  MCM family protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.2   0.5   0.00011      0.11       5      30 ..     211     240 ..     209     245 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.2 bits;  conditional E-value: 0.00011\n",
            "     Zn-ribbon_8   5 eYrCeeCgheFevlqkms.dap...lta.CP 30 \n",
            "                     e+ C +Cgh+F++   +  + +   l++ CP\n",
            "  XP_004360556.1 211 EFACLKCGHRFDAQIDFEqQ-NnfsLPKkCP 240\n",
            "                     8***********66555533.2355666688 PP\n",
            "\n",
            ">> XP_004358406.1  ubiquitin [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.7   0.4   0.00016      0.15       7      39 ..     127     158 ..     113     158 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.7 bits;  conditional E-value: 0.00016\n",
            "     Zn-ribbon_8   7 rCeeCgheFevlqkmsdaplta.CPeCgsevrrv 39 \n",
            "                     +C++Cg      q    +p  + C +C++++ +v\n",
            "  XP_004358406.1 127 ECTNCGAGVFMAQH--ANPARQyCGKCHTTLVKV 158\n",
            "                     6*****88776664..4455555*****988875 PP\n",
            "\n",
            ">> XP_004356292.1  uncharacterized protein DFA_08808 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.2   0.4      0.81   7.6e+02      26      34 ..     183     192 ..     168     196 .. 0.64\n",
            "   2 ?    6.2   0.1    0.0081       7.5      25      38 ..     285     299 ..     277     300 .. 0.76\n",
            "   3 ?    7.6   0.7    0.0031       2.8       8      38 ..     307     346 ..     304     348 .. 0.78\n",
            "   4 ?    3.6   0.0     0.054        50       8      23 ..     355     369 ..     352     374 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.2 bits;  conditional E-value: 0.81\n",
            "     Zn-ribbon_8  26 lta.CPeCgs 34 \n",
            "                      t+ C +C s\n",
            "  XP_004356292.1 183 TTKdCSSCSS 192\n",
            "                     4455999987 PP\n",
            "\n",
            "  == domain 2  score: 6.2 bits;  conditional E-value: 0.0081\n",
            "     Zn-ribbon_8  25 pltaCPeCgs.evrr 38 \n",
            "                     ++++CP C+  ++rr\n",
            "  XP_004356292.1 285 DKPTCPNCHViATRR 299\n",
            "                     4678*****988887 PP\n",
            "\n",
            "  == domain 3  score: 7.6 bits;  conditional E-value: 0.0031\n",
            "     Zn-ribbon_8   8 CeeCgheFevlqkms..dap.......ltaCPeCgsevrr 38 \n",
            "                     C++C + F++++k +   +p        ++C +C ++++r\n",
            "  XP_004356292.1 307 CKNCVKSFTPFRKGKkyNDPnssnpslAPQCGRCSTQATR 346\n",
            "                     **********999986444489999955569999886655 PP\n",
            "\n",
            "  == domain 4  score: 3.6 bits;  conditional E-value: 0.054\n",
            "     Zn-ribbon_8   8 CeeCgheFevlqkmsd 23 \n",
            "                     C++C++ F++ q +s+\n",
            "  XP_004356292.1 355 CKSCKKPFTPNQ-FSE 369\n",
            "                     *********999.555 PP\n",
            "\n",
            ">> XP_004362968.1  putative GATA-binding transcription factor [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.5   0.6   0.00037      0.34       8      36 ..     669     698 ..     667     702 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.5 bits;  conditional E-value: 0.00037\n",
            "     Zn-ribbon_8   8 CeeCgheFe.vlqkmsdapltaCPeCgsev 36 \n",
            "                     C+ Cg++ + ++++  d p t C +Cg + \n",
            "  XP_004362968.1 669 CHTCGTKSTpEWRRGPDGPATLCNACGLAF 698\n",
            "                     *********7777766***999****9754 PP\n",
            "\n",
            ">> XP_004362557.1  leucyl-tRNA synthetase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.2   0.1       0.6   5.6e+02       7      34 ..     475     481 ..     470     485 .. 0.60\n",
            "   2 ?    9.0   0.2    0.0011       1.1      23      38 ..     511     526 ..     501     527 .. 0.76\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.2 bits;  conditional E-value: 0.6\n",
            "     Zn-ribbon_8   7 rCeeCgheFevlqkmsdapltaCPeCgs 34 \n",
            "                     +C++Cg                     +\n",
            "  XP_004362557.1 475 HCNSCG---------------------E 481\n",
            "                     455555.....................4 PP\n",
            "\n",
            "  == domain 2  score: 9.0 bits;  conditional E-value: 0.0011\n",
            "     Zn-ribbon_8  23 dapltaCPeCgsevrr 38 \n",
            "                     d ++ +CP+Cg++++r\n",
            "  XP_004362557.1 511 DWKNCKCPKCGGNAQR 526\n",
            "                     445556*******998 PP\n",
            "\n",
            ">> XP_004355856.1  ZPR1-type zinc finger-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.8   0.0       2.6   2.4e+03      29      37 ..      33      42 ..      14      45 .. 0.67\n",
            "   2 ?    4.9   0.1     0.021        19       6      20 ..      60      74 ..      58      80 .. 0.85\n",
            "   3 ?   -1.6   0.0       2.2   2.1e+03      26      35 ..     266     277 ..     259     282 .. 0.66\n",
            "   4 ?    3.7   0.0      0.05        47       6      19 ..     297     310 ..     295     317 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.8 bits;  conditional E-value: 2.6\n",
            "     Zn-ribbon_8 29 CPeCgs.evr 37\n",
            "                    C  C++   +\n",
            "  XP_004355856.1 33 CMNCHEdGIT 42\n",
            "                    8888775555 PP\n",
            "\n",
            "  == domain 2  score: 4.9 bits;  conditional E-value: 0.021\n",
            "     Zn-ribbon_8  6 YrCeeCgheFevlqk 20\n",
            "                    ++Ce Cg +  ++q+\n",
            "  XP_004355856.1 60 FECEHCGFKSSEVQS 74\n",
            "                    99*****99888886 PP\n",
            "\n",
            "  == domain 3  score: -1.6 bits;  conditional E-value: 2.2\n",
            "     Zn-ribbon_8  26 lta.CPeCgs.e 35 \n",
            "                     l++ C  Cg+  \n",
            "  XP_004355856.1 266 LPQsCVFCGEmG 277\n",
            "                     566688888755 PP\n",
            "\n",
            "  == domain 4  score: 3.7 bits;  conditional E-value: 0.05\n",
            "     Zn-ribbon_8   6 YrCeeCgheFevlq 19 \n",
            "                     + Ce+Cg +  ++ \n",
            "  XP_004355856.1 297 FSCENCGYKSNEIK 310\n",
            "                     89*****8877765 PP\n",
            "\n",
            ">> XP_004350735.1  uncharacterized protein DFA_11790 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.4   0.0      0.13   1.2e+02      21      36 ..      76      92 ..      69      96 .. 0.70\n",
            "   2 ?    8.2   1.3     0.002       1.9       8      32 ..     103     125 ..      99     132 .. 0.81\n",
            "   3 ?    1.6   0.4      0.22   2.1e+02      27      35 ..     137     146 ..     127     150 .. 0.69\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.4 bits;  conditional E-value: 0.13\n",
            "     Zn-ribbon_8 21 msdapltaCPeCgs.ev 36\n",
            "                     s ++++ C eCg  + \n",
            "  XP_004350735.1 76 TSTPNEKICIECGDsKD 92\n",
            "                    55666667****98665 PP\n",
            "\n",
            "  == domain 2  score: 8.2 bits;  conditional E-value: 0.002\n",
            "     Zn-ribbon_8   8 CeeCgheFevlqkmsdaplta.CPeC 32 \n",
            "                     C++ +++Fe++++++   +++ C  C\n",
            "  XP_004350735.1 103 CKDSKERFEAFHQSD---KITkCNTC 125\n",
            "                     9**********9755...44449998 PP\n",
            "\n",
            "  == domain 3  score: 1.6 bits;  conditional E-value: 0.22\n",
            "     Zn-ribbon_8  27 ta.CPeCgse 35 \n",
            "                     t+ C++C  +\n",
            "  XP_004350735.1 137 TTqCKDCRNT 146\n",
            "                     4449999654 PP\n",
            "\n",
            ">> XP_004356125.1  uncharacterized protein DFA_08637 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.6   0.1     0.027        25      27      36 ..       7      16 ..       1      17 [. 0.77\n",
            "   2 ?    6.2   4.6    0.0085         8       7      34 ..      36      63 ..      29      66 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.6 bits;  conditional E-value: 0.027\n",
            "     Zn-ribbon_8 27 taCPeCgsev 36\n",
            "                    +aCP C ++v\n",
            "  XP_004356125.1  7 EACPVCEKKV 16\n",
            "                    67****8765 PP\n",
            "\n",
            "  == domain 2  score: 6.2 bits;  conditional E-value: 0.0085\n",
            "     Zn-ribbon_8  7 rCeeCgheFevlqkmsdapltaCPeCgs 34\n",
            "                     C++C++  +      +++++ C++C +\n",
            "  XP_004356125.1 36 VCTACKKSLDSTNLCEKDSKPYCKACYG 63\n",
            "                    4*********888777777788****76 PP\n",
            "\n",
            ">> XP_004350066.1  uncharacterized protein DFA_11123 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.2   0.7      0.86     8e+02       6      34 ..     740     747 ..     732     762 .. 0.73\n",
            "   2 ?   10.2   0.7   0.00047      0.44       7      34 ..     780     806 ..     779     809 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.2 bits;  conditional E-value: 0.86\n",
            "     Zn-ribbon_8   6 YrCeeCgheFevlqkmsdapltaCPeCgs 34 \n",
            "                     Y                      CP C++\n",
            "  XP_004350066.1 740 YV---------------------CPTCKT 747\n",
            "                     44.....................666644 PP\n",
            "\n",
            "  == domain 2  score: 10.2 bits;  conditional E-value: 0.00047\n",
            "     Zn-ribbon_8   7 rCeeCgheFevlqkmsdapltaCPeCgs 34 \n",
            "                     +C+ C+ ++   q+ +  p+ +CP+C++\n",
            "  XP_004350066.1 780 QCNFCQATYSISQP-DIIPKSTCPSCQH 806\n",
            "                     6*********9996.6667899****85 PP\n",
            "\n",
            ">> XP_004366572.1  Putative transposase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.1   7.5    0.0088       8.2       5      34 ..     165     198 ..     165     202 .. 0.77\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.1 bits;  conditional E-value: 0.0088\n",
            "     Zn-ribbon_8   5 eYrCeeCgheFe....vlqkms.daplta.CPeCgs 34 \n",
            "                     e++C++C+ +F+    +l+    + ++   C++C++\n",
            "  XP_004366572.1 165 EFKCKSCHYQFDqgetRLV--CvECKDLSlCKDCHQ 198\n",
            "                     8***********9966444..446667777****98 PP\n",
            "\n",
            ">> XP_004355683.1  uncharacterized protein DFA_08186 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.5   0.5    0.0031       2.9       8      38 ..      38      73 ..      35      75 .. 0.74\n",
            "   2 ?    2.0   1.5      0.17   1.6e+02       8      24 ..      89     105 ..      75     109 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.5 bits;  conditional E-value: 0.0031\n",
            "     Zn-ribbon_8  8 CeeCgheFe...vlqkmsdap..lta.CPeCgs.evrr 38\n",
            "                    C++C+ + e   + +k +  p  +   C +C + +v+r\n",
            "  XP_004355683.1 38 CQDCHDTIEwrkKYRKYK--PlsMAGkCAHCTQrKVTR 73\n",
            "                    *********665444444..2446666******99999 PP\n",
            "\n",
            "  == domain 2  score: 2.0 bits;  conditional E-value: 0.17\n",
            "     Zn-ribbon_8   8 CeeCgheFevlqkmsda 24 \n",
            "                     C +C++  e + +++d+\n",
            "  XP_004355683.1  89 CAKCHKAKEIVAEITDP 105\n",
            "                     77777777777766665 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (40 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       630  (0.0519159); expected 242.7 (0.02)\n",
            "Passed bias filter:                      426  (0.0351051); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        83  (0.00683972); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        20  (0.00164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):              13  [number of targets reported over threshold]\n",
            "# CPU time: 0.14u 0.00s 00:00:00.14 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 3476.03\n",
            "//\n",
            "Query:       ZNRF_3_ecto  [M=105]\n",
            "Accession:   PF18212.4\n",
            "Description: ZNRF-3 Ectodomain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.075   12.5   0.0       0.21   11.1   0.0    1.6  1  XP_004362966.1  3-hydroxyacyl-CoA dehydrogenase type II [Cave\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004362966.1  3-hydroxyacyl-CoA dehydrogenase type II [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.1   0.0   1.7e-05      0.21      45      68 ..      25      48 ..      17      59 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.1 bits;  conditional E-value: 1.7e-05\n",
            "     ZNRF_3_ecto 45 alerGatavifdisddaeaadqlr 68\n",
            "                      erGa+ vifd+ + +e  ++l \n",
            "  XP_004362966.1 25 LHERGANVVIFDVNNAPELEKELG 48\n",
            "                    579**************9999885 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (105 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       217  (0.0178822); expected 242.7 (0.02)\n",
            "Passed bias filter:                      163  (0.0134322); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        10  (0.000824063); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.09\n",
            "# Mc/sec: 7507.27\n",
            "//\n",
            "Query:       ZnuA  [M=247]\n",
            "Accession:   PF01297.20\n",
            "Description: Zinc-uptake complex component A periplasmic\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.028   13.5  10.4      0.051   12.6   0.8    2.8  2  XP_004355084.1  Lis-interacting protein [Cavenderia fascicula\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004355084.1  Lis-interacting protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.6   0.8   4.2e-06     0.051     107     150 ..      26      69 ..      17      72 .. 0.88\n",
            "   2 ?    3.7   3.4    0.0023        28     112     192 ..      70     150 ..      68     188 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.6 bits;  conditional E-value: 4.2e-06\n",
            "                     GGHHHHHHHHHHHHHHHHHHHHHHHHHHTTSSTCEGEEEESSST CS\n",
            "                     xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx RF\n",
            "            ZnuA 107 enaeayeanaaaykekLeeldaeireelasipekkrklvtthda 150\n",
            "                     ++ +++e+n+++y+e  ++l++e+++el ++++k+  lv++h  \n",
            "  XP_004355084.1  26 QEVDDLEQNFNEYQEFSKQLEEEMETELRACEKKHTDLVSQHAR 69 \n",
            "                     455789************************************75 PP\n",
            "\n",
            "  == domain 2  score: 3.7 bits;  conditional E-value: 0.0023\n",
            "                     HHHHHHHHHHHHHHHHHHHHHHHTTSSTCEGEEEESSSTTHHHHHHTT-EEEEE..ESSSSSSSSSHHHHHHHHHHHHHTT-S CS\n",
            "                     xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx..xxxxxxxxxxxxxxxxxxxxxxxxxxx RF\n",
            "            ZnuA 112 yeanaaaykekLeeldaeireelasipekkrklvtthdafgylakaygleqaal..sgvstesepsaadiaklieeikeegvk 192\n",
            "                     +++ +++ ++kL+ + +e  + +as++++ +kl  +h+  + l+++  leq     +  +  s++s++d+++  +++ ee+v \n",
            "  XP_004355084.1  70 LKNDYENIQDKLNGVSRESSSLIASLQDEVDKL--KHFKQSLLGDKRKLEQENDtlERRERASSASVQDLSEKLDRVMEENVW 150\n",
            "                     56789999*****************99998665..888889999999999776622445667788888888888888888875 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (247 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       803  (0.0661722); expected 242.7 (0.02)\n",
            "Passed bias filter:                      286  (0.0235682); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        25  (0.00206016); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.21u 0.01s 00:00:00.22 Elapsed: 00:00:00.11\n",
            "# Mc/sec: 14105.01\n",
            "//\n",
            "Query:       Zn_clus  [M=40]\n",
            "Accession:   PF00172.21\n",
            "Description: Fungal Zn(2)-Cys(6) binuclear cluster domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    2.8e-07   30.0  12.8    8.2e-07   28.5  12.8    1.9  1  XP_004355461.1  transcriptional regulatory protein [Cavenderi\n",
            "    0.00012   21.6  10.9    0.00028   20.4  10.9    1.7  1  XP_004358441.1  transcriptional regulatory protein [Cavenderi\n",
            "    0.00055   19.5   4.5     0.0009   18.8   4.5    1.3  1  XP_004360064.1  uncharacterized protein DFA_04331 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004355461.1  transcriptional regulatory protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   28.5  12.8     2e-10   8.2e-07       1      38 [.      34      72 ..      34      73 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 28.5 bits;  conditional E-value: 2e-10\n",
            "                    SS-HHHHHHTS--STSSSS-HHHHHTT..HHHCCCCTCSC CS\n",
            "         Zn_clus  1 haCdnCrkkKvKCdakkPaCsnCkkln..leCtfysessk 38\n",
            "                    +aC nCr+ Kv Cd+++P C++C k++   +C + ++++k\n",
            "  XP_004355461.1 34 RACVNCRQSKVACDQQRP-CTRCTKHGieDKCHDVPRKTK 72\n",
            "                    69****************.********999**99888875 PP\n",
            "\n",
            ">> XP_004358441.1  transcriptional regulatory protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   20.4  10.9   6.9e-08   0.00028       1      39 [.      10      49 ..      10      50 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 20.4 bits;  conditional E-value: 6.9e-08\n",
            "                    SS-HHHHHHTS--STSSSS-HHHHHTT..HHHCCCCTCSCE CS\n",
            "         Zn_clus  1 haCdnCrkkKvKCdakkPaCsnCkkln..leCtfysesskr 39\n",
            "                    +aC nCr  Kv Cd  +P C++C+kl     C + ++++++\n",
            "  XP_004358441.1 10 RACLNCRNSKVACDPSRP-CTRCVKLDmgHSCYDLPRKKRE 49\n",
            "                    69****************.******9888999988888875 PP\n",
            "\n",
            ">> XP_004360064.1  uncharacterized protein DFA_04331 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   18.8   4.5   2.2e-07    0.0009       1      37 [.      13      49 ..      13      52 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 18.8 bits;  conditional E-value: 2.2e-07\n",
            "                    SS-HHHHHHTS--STSSSS-HHHHHTT..HHHCCCCTCS CS\n",
            "         Zn_clus  1 haCdnCrkkKvKCdakkPaCsnCkkln..leCtfysess 37\n",
            "                    ++C  C k +  Cd+++P C+nC+ ++    C  y+e+ \n",
            "  XP_004360064.1 13 RSCFLCQKEHLECDEQTP-CGNCALKGtpQMCF-YTERL 49\n",
            "                    59****************.******99666787.77665 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (40 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       837  (0.068974); expected 242.7 (0.02)\n",
            "Passed bias filter:                      295  (0.0243098); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        14  (0.00115369); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         3  (0.000247219); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 3366.06\n",
            "//\n",
            "Query:       Zn_dep_PLPC  [M=179]\n",
            "Accession:   PF00882.21\n",
            "Description: Zinc dependent phospholipase C\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    1.3e-21   76.8   0.2    3.2e-21   75.6   0.2    1.6  1  XP_004360529.1  glycosylphosphatidylinositol phospholipase D \n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004360529.1  glycosylphosphatidylinositol phospholipase D [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   75.6   0.2   2.6e-25   3.2e-21       1     151 [.      46     206 ..      46     235 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 75.6 bits;  conditional E-value: 2.6e-25\n",
            "                     HHHHHHHHHHH..HH....HHH-..HHHTHHHHHHHHHGGGG..-TTGGG--BTTTB-HHHHS-HHHHHHHHHHHHHHHHHC........T-HHH CS\n",
            "     Zn_dep_PLPC   1 tHltlashala..vlpalkkrrk..lklnkaalllGsvaPDleylftsshsykthtagllgedhhweetgkfllllikklrely...ksegslkl 88 \n",
            "                     tH  +a++a +  +++++++  +  ++++++ +++G+ +PD +y++ ++ ++ + +        hw   ++fl+++  +l+++y   +se   kl\n",
            "  XP_004360529.1  46 THNEVARRAFNfsSFQDYPQF-QtlIPQYLSSFQAGASFPDWGYDC-GNSGNESES-------AHW---PPFLRNATVYLKKTYpqpWSEAGAKL 128\n",
            "                     799999999997755545443.2589******************88.888888888.......9**...**************9***9999**** PP\n",
            "\n",
            "                     HHHHHHHHHHHHHHHTSH..............HTT--TTT-THHHHHHHHHHHHGGGG.......--S--SS-TTSHH CS\n",
            "     Zn_dep_PLPC  89 aafllGvlsHyvaDvftH.......g..stlgeneipsyrllqHgsyelgldflllkyv......krdaaeaktllls 151\n",
            "                     a fl+GvlsH+vaD+++H       g  + ++ + ++s++  +H++ ++g++++l +++      +r+  ++++ +++\n",
            "  XP_004360529.1 129 AVFLMGVLSHQVADISWHsiggiheGliRAMAGQDFNSDYSIAHTNADVGGEYVLAYNYdlswlvDRWDVPISDVKNI 206\n",
            "                     *********************9988356999999******************99988889977773333333333222 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (179 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       519  (0.0427689); expected 242.7 (0.02)\n",
            "Passed bias filter:                      237  (0.0195303); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        16  (0.0013185); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.15u 0.01s 00:00:00.16 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 14552.07\n",
            "//\n",
            "Query:       Zn_finger_prot  [M=268]\n",
            "Accession:   PF19242.2\n",
            "Description: Putative zinc finger protein\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.043   13.0   0.5      0.046   12.9   0.5    1.1  1  XP_004351250.1  uncharacterized protein DFA_11002 [Cavenderia\n",
            "       0.88    8.7   7.5        1.7    7.8   7.5    1.5  1  XP_004354380.1  uncharacterized protein DFA_10480 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004351250.1  uncharacterized protein DFA_11002 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.9   0.5   7.6e-06     0.046     147     198 ..      43      94 ..      27      97 .] 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.9 bits;  conditional E-value: 7.6e-06\n",
            "  Zn_finger_prot 147 raaatplnsktklnvakPnkvkqekqvvkrdevqkPtrlclsvlkktkcffg 198\n",
            "                     +  ++  n+++++ +   n  +q++q  + +  q+P  l ls+l +t+ f  \n",
            "  XP_004351250.1  43 HNHNHRVNQQHQQHQIITNSQQQQQQQQQLQIQQRPIFLPLSILHNTRPFSI 94 \n",
            "                     556778889999999999999999999899999**************99865 PP\n",
            "\n",
            ">> XP_004354380.1  uncharacterized protein DFA_10480 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.8   7.5   0.00027       1.7      25      83 ..     368     427 ..     344     477 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.8 bits;  conditional E-value: 0.00027\n",
            "  Zn_finger_prot  25 yesddeseeedfydryepeeeefdddeyqlfvdeedd.erkddPkkitkvikPisllk.sP 83 \n",
            "                      e +de eeed y+ ye + +e++++e +  v  ed+ e +++   ++ ++k +sl++  P\n",
            "  XP_004354380.1 368 DEDEDEDEEEDEYE-YEYDGDEYEEEEEEELVVGEDQvEIRNNLLLLNEIVKVVSLSNiYP 427\n",
            "                     34556677777776.4547788888888887777777577777777799999998887434 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (268 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      2600  (0.214256); expected 242.7 (0.02)\n",
            "Passed bias filter:                      337  (0.0277709); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        51  (0.00420272); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         9  (0.000741656); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.33u 0.00s 00:00:00.33 Elapsed: 00:00:00.18\n",
            "# Mc/sec: 9844.69\n",
            "//\n",
            "Query:       Zn_peptidase  [M=290]\n",
            "Accession:   PF04228.16\n",
            "Description: Putative neutral zinc metallopeptidase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "       0.11   11.5   0.1       0.33    9.8   0.2    1.7  2  XP_004366821.1  uncharacterized protein DFA_02436 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004366821.1  uncharacterized protein DFA_02436 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.6   0.0     0.017   2.1e+02      45      92 ..     263     310 ..     259     320 .. 0.62\n",
            "   2 ?    9.8   0.2   2.7e-05      0.33       8     124 ..     283     402 ..     277     415 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.6 bits;  conditional E-value: 0.017\n",
            "    Zn_peptidase  45 lvgllfGvdlssllglgqeespaeaaaqsqeaaaeeaedeaaefvsvi 92 \n",
            "                     l ++++ +d  s+ g ++++ +++ ++ +++ + ++   +  ef+++i\n",
            "  XP_004366821.1 263 LNSTYYSIDWKSIFGRANNSNNQNINNNQNNNNQNSEMVKVVEFARSI 310\n",
            "                     56789**********655555554444333333333334555666665 PP\n",
            "\n",
            "  == domain 2  score: 9.8 bits;  conditional E-value: 2.7e-05\n",
            "    Zn_peptidase   8 esdnvedrrassggedaGGgg...rlslggkgGllllilvlvgllfGvdlssllglgqeespaeaaaqsqeaaaeeaedeaaefvsviladteev 99 \n",
            "                     + +n+++ ++ +++++        ++s+++++      lvl+ l f   ++ +l l+   ++  a++q+q++++ +  ++ +  +++i  d + +\n",
            "  XP_004366821.1 283 NNQNINNNQNNNNQNSEMVKVvefARSISKRCDRFDKKLVLACLSFPSHPTYMLVLEMAAKNTTANSQQQQQQSLSPSNKGKLTTKSIKEDEKYM 377\n",
            "                     56677777777766663332235578888888888889999999999999998886555555555555555555553344444567777888889 PP\n",
            "\n",
            "    Zn_peptidase 100 Weqlfaklqrqyqkpklvlfsrvvr 124\n",
            "                     W++++ +++   q p + +f+  ++\n",
            "  XP_004366821.1 378 WHKYLIDHHSALQFPNISIFKYFIH 402\n",
            "                     ****************999987766 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (290 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1074  (0.0885043); expected 242.7 (0.02)\n",
            "Passed bias filter:                      493  (0.0406263); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        46  (0.00379069); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.29u 0.00s 00:00:00.28 Elapsed: 00:00:00.15\n",
            "# Mc/sec: 12392.90\n",
            "//\n",
            "Query:       Zn_peptidase_2  [M=218]\n",
            "Accession:   PF04298.15\n",
            "Description: Putative neutral zinc metallopeptidase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.027   13.5   0.1      0.032   13.3   0.1    1.2  1  XP_004366310.1  uncharacterized protein DFA_03900 [Cavenderia\n",
            "       0.11   11.5   0.5       0.19   10.7   0.2    1.7  1  XP_004367187.1  uncharacterized protein DFA_07324 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004366310.1  uncharacterized protein DFA_03900 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.3   0.1   5.3e-06     0.032       2      70 ..       4      79 ..       3     104 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.3 bits;  conditional E-value: 5.3e-06\n",
            "  Zn_peptidase_2  2 lillivllllsllaqlkvkstfkkyskvrsss.......gltGaevArklldenglsdVkveeveGeltDhYdprk 70\n",
            "                    +i+l +++ll ++a ++ + ++  +++ rs+         +   e Arkll++n+l++V+v     + +  Y+p+ \n",
            "  XP_004366310.1  4 IIILAIIVLLGMVACIQAEDAYFAFRTHRSTLdfvfkltDVKKIEHARKLLSDNDLQEVHVMGRIRKTQKDYNPHY 79\n",
            "                    5777888888888888888888888877776422221114566789***************866557777799975 PP\n",
            "\n",
            ">> XP_004367187.1  uncharacterized protein DFA_07324 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.7   0.2   3.2e-05      0.19      83     102 ..     117     136 ..      98     217 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.7 bits;  conditional E-value: 3.2e-05\n",
            "  Zn_peptidase_2  83 rsvaAvavAAHEvGHAiQha 102\n",
            "                      s+ A++v  HE+GHAi   \n",
            "  XP_004367187.1 117 LSLWAIGVSVHEFGHAIVAV 136\n",
            "                     5899************9655 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (218 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       440  (0.0362588); expected 242.7 (0.02)\n",
            "Passed bias filter:                      352  (0.029007); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        25  (0.00206016); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.17u 0.01s 00:00:00.18 Elapsed: 00:00:00.09\n",
            "# Mc/sec: 15083.69\n",
            "//\n",
            "Query:       Zn_protease  [M=138]\n",
            "Accession:   PF05618.14\n",
            "Description: Putative ATP-dependant zinc protease\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (138 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       335  (0.0276061); expected 242.7 (0.02)\n",
            "Passed bias filter:                      192  (0.015822); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        12  (0.000988875); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 14418.03\n",
            "//\n",
            "Query:       Zn_ribbon_2  [M=82]\n",
            "Accession:   PF12674.10\n",
            "Description: Putative zinc ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.043   13.9   1.5       0.15   12.2   0.2    2.2  2  XP_004359533.1  cellular retinaldehyde-binding/triple functio\n",
            "       0.78    9.9   7.2       0.21   11.7   0.2    3.1  3  XP_004363093.1  uncharacterized protein DFA_03490 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004359533.1  cellular retinaldehyde-binding/triple function domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.0   0.1      0.16   9.7e+02      44      70 ..      16      38 ..       6      53 .. 0.49\n",
            "   2 ?   12.2   0.2   2.4e-05      0.15      12      76 ..     111     176 ..     104     182 .. 0.67\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.0 bits;  conditional E-value: 0.16\n",
            "     Zn_ribbon_2 44 leemiefcvkklkeekkelteeeakkl 70\n",
            "                    l+e+ + + k++ +    +t+eea+kl\n",
            "  XP_004359533.1 16 LQEVKDLVNKNMTD----ATKEEAEKL 38\n",
            "                    34444444444444....444444443 PP\n",
            "\n",
            "  == domain 2  score: 12.2 bits;  conditional E-value: 2.4e-05\n",
            "     Zn_ribbon_2  12 eeelgteadgsksedYCkyCykdGe.ftepdltleemiefcvkklkeekkelt.eeeakkllkkllp 76 \n",
            "                      ++ gt++d ++ + YC y ++ G+ + +p ++   +i  + ++  ++  +++  + + ++++k++p\n",
            "  XP_004359533.1 111 RKHDGTNRDIDEAQRYCVYVMESGKqMLKPGIETCTLIFDMSSFSTKNM-DYPlVKFMVDMFQKYYP 176\n",
            "                     5667******************996366777776666655555444443.33202335555555555 PP\n",
            "\n",
            ">> XP_004363093.1  uncharacterized protein DFA_03490 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -4.1   0.0         2   1.2e+04      17      29 ..      90     102 ..      89     107 .. 0.82\n",
            "   2 ?   11.7   0.2   3.4e-05      0.21       2      67 ..     461     525 ..     460     535 .. 0.72\n",
            "   3 ?    0.5   0.6      0.11   6.8e+02      17      48 ..    1050    1084 ..    1024    1093 .. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -4.1 bits;  conditional E-value: 2\n",
            "     Zn_ribbon_2  17 teadgsksedYCk 29 \n",
            "                     t++ g ks  YCk\n",
            "  XP_004363093.1  90 TNSYGGKSFAYCK 102\n",
            "                     6788999999**8 PP\n",
            "\n",
            "  == domain 2  score: 11.7 bits;  conditional E-value: 3.4e-05\n",
            "     Zn_ribbon_2   2 CqsCGmPl...kdeeelgteadgsksedYCkyCykdGeftepdltleemiefcvkklkeekkelteeea 67 \n",
            "                     C++CG  l   ++e+ + + +dg++ + YC  C   G+f +   +l+++ ++  +k+ +  k+l++e++\n",
            "  XP_004363093.1 461 CKTCGRGLmdfEREKVILKRKDGERIQLYCAECR-SGTFLN---KLKNTSSHSKQKIYSFFKNLDSEDV 525\n",
            "                     *****7774436677789***************5.688887...5666666666665555555555555 PP\n",
            "\n",
            "  == domain 3  score: 0.5 bits;  conditional E-value: 0.11\n",
            "     Zn_ribbon_2   17 teadgsksedYCkyCyk...dGeftepdltleemi 48  \n",
            "                       e+d+  +++ C+ C k   +Ge++++ +++ e i\n",
            "  XP_004363093.1 1050 FEKDKAAKQQQCTKCDKqplHGEYVSKRIQMFENI 1084\n",
            "                      47788888888888887566688888777776655 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (82 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1031  (0.0849609); expected 242.7 (0.02)\n",
            "Passed bias filter:                      499  (0.0411207); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        42  (0.00346106); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.14u 0.00s 00:00:00.14 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 7380.40\n",
            "//\n",
            "Query:       Zn_ribbon_recom  [M=58]\n",
            "Accession:   PF13408.9\n",
            "Description: Recombinase zinc beta ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "       0.48   10.4   5.6       0.39   10.7   3.2    2.2  1  XP_004357934.1  DNA topoisomerase III [Cavenderia fasciculata\n",
            "        1.2    9.1   3.5       0.67   10.0   0.3    2.3  2  XP_004354785.1  RNA polymerase I [Cavenderia fasciculata]\n",
            "          3    7.9   8.0        9.2    6.3   8.0    1.8  1  XP_004350226.1  uncharacterized protein DFA_11283 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004357934.1  DNA topoisomerase III [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.7   3.2   9.6e-05      0.39       7      37 ..     644     674 ..     640     703 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.7 bits;  conditional E-value: 9.6e-05\n",
            "  Zn_ribbon_recom   7 rCgeCGspmtrkkrkgkrryYrCstyrrkgk 37 \n",
            "                      +Cg+C++ m  kk++g r    Cst++++ +\n",
            "   XP_004357934.1 644 KCGKCNNLMDLKKTNGDRKILNCSTCNTSYN 674\n",
            "                      7*************************99876 PP\n",
            "\n",
            ">> XP_004354785.1  RNA polymerase I [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -2.9   0.2       1.7   6.9e+03      25      44 ..     559     577 ..     552     583 .. 0.69\n",
            "   2 ?   10.0   0.3   0.00017      0.67       5      24 ..     946     965 ..     943     981 .. 0.67\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -2.9 bits;  conditional E-value: 1.7\n",
            "  Zn_ribbon_recom  25 ryYrCstyrrkgkkCsnrsi 44 \n",
            "                        Y+C+ +++ + + +++s+\n",
            "   XP_004354785.1 559 NMYQCQMAKQTM-GTPMHSY 577\n",
            "                      569999888777.7777744 PP\n",
            "\n",
            "  == domain 2  score: 10.0 bits;  conditional E-value: 0.00017\n",
            "  Zn_ribbon_recom   5 llrCgeCGspmtrkkrkgkr 24 \n",
            "                      +++C +CGs++t+ ++ +++\n",
            "   XP_004354785.1 946 MMVCRHCGSTLTVATKPSGQ 965\n",
            "                      69***********7766333 PP\n",
            "\n",
            ">> XP_004350226.1  uncharacterized protein DFA_11283 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.3   8.0    0.0023       9.2       6      43 ..     512     550 ..     509     557 .. 0.71\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.3 bits;  conditional E-value: 0.0023\n",
            "  Zn_ribbon_recom   6 lrCgeCGspmtrkkrkgkrryYrCstyrrkgk..kCsnrs 43 \n",
            "                      + Cg+C+ + + ++  gk+  ++C++++++ +  +C  +s\n",
            "   XP_004350226.1 512 VTCGICNGKISTNWG-GKKQVFHCNNCKEYYHngECVEQS 550\n",
            "                      78********99977.777666666666555433686555 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (58 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       727  (0.0599094); expected 242.7 (0.02)\n",
            "Passed bias filter:                      431  (0.0355171); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        43  (0.00354347); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000576844); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.13u 0.01s 00:00:00.14 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 5225.33\n",
            "//\n",
            "Query:       Zn_ribbon_SprT  [M=38]\n",
            "Accession:   PF17283.5\n",
            "Description: SprT-like zinc ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.048   13.1   0.1      0.048   13.1   0.1    6.1  5  XP_004359993.1  LIM domain protein [Cavenderia fasciculata]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004359993.1  LIM domain protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.4   0.3    0.0089   1.1e+02      26      32 ..       7      13 ..       6      16 .. 0.80\n",
            "   2 ?   -5.3   9.1         1   1.2e+04       7      33 ..      35      73 ..      29      77 .. 0.65\n",
            "   3 ?   13.1   0.1     4e-06     0.048       6      35 ..      93     122 ..      92     126 .. 0.89\n",
            "   4 ?    0.4   3.3     0.038   4.6e+02       7      29 ..     114     141 ..     113     147 .. 0.74\n",
            "   5 ?    0.6   2.7     0.033     4e+02       7      15 ..     166     174 ..     164     195 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.4 bits;  conditional E-value: 0.0089\n",
            "  Zn_ribbon_SprT 26 rCrrCkg 32\n",
            "                    +C+rC+ \n",
            "  XP_004359993.1  7 ICKRCQK 13\n",
            "                    7999985 PP\n",
            "\n",
            "  == domain 2  score: -5.3 bits;  conditional E-value: 1\n",
            "  Zn_ribbon_SprT  7 CpsCgreytrhlrsrrhn............yrCrrCkgt 33\n",
            "                    C +C++ +  ++  +++               C++C ++\n",
            "  XP_004359993.1 35 CAECHTAFETQSFYKVQGlpickncylkkqSKCKKCSQP 73\n",
            "                    666666666666666655655556666666779999876 PP\n",
            "\n",
            "  == domain 3  score: 13.1 bits;  conditional E-value: 4e-06\n",
            "  Zn_ribbon_SprT   6 rCpsCgreytrhlrsrrhnyrCrrCkgtLk 35 \n",
            "                     +C +C +++++++ ++  n  C +C  +Lk\n",
            "  XP_004359993.1  93 TCSNCNKNIDANFFIKDTNILCPQCNDQLK 122\n",
            "                     7**************66699******8887 PP\n",
            "\n",
            "  == domain 4  score: 0.4 bits;  conditional E-value: 0.038\n",
            "  Zn_ribbon_SprT   7 CpsCgreytrhlrsrrhn.....yrCrr 29 \n",
            "                     Cp C ++ + + + ++ +     y+C++\n",
            "  XP_004359993.1 114 CPQCNDQLKEQTKKQIGKelgncYVCKK 141\n",
            "                     8888877777766677778888888886 PP\n",
            "\n",
            "  == domain 5  score: 0.6 bits;  conditional E-value: 0.033\n",
            "  Zn_ribbon_SprT   7 CpsCgreyt 15 \n",
            "                     C++C++e++\n",
            "  XP_004359993.1 166 CVKCHNEIK 174\n",
            "                     888887765 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (38 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       651  (0.0536465); expected 242.7 (0.02)\n",
            "Passed bias filter:                      322  (0.0265348); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        38  (0.00313144); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         3  (0.000247219); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 4107.35\n",
            "//\n",
            "Query:       Zn_Tnp_IS1  [M=35]\n",
            "Accession:   PF03811.16\n",
            "Description: InsA N-terminal domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.018   14.2   3.0      0.041   13.1   3.0    1.6  1  XP_004360866.1  histone deacetylase family protein [Cavenderi\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004360866.1  histone deacetylase family protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.1   3.0   3.4e-06     0.041       7      34 ..      32      60 ..      29      61 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.1 bits;  conditional E-value: 3.4e-06\n",
            "      Zn_Tnp_IS1  7 hCPrCsstd.VyRhGkstagHeRfrCrdC 34\n",
            "                    hCPr +      R+Gk  +g +R+rC+ C\n",
            "  XP_004360866.1 32 HCPRIEDVFySQRKGKTNKGAQRWRCKAC 60\n",
            "                    999987655567****************9 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (35 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       375  (0.0309023); expected 242.7 (0.02)\n",
            "Passed bias filter:                      222  (0.0182942); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        18  (0.00148331); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 4186.89\n",
            "//\n",
            "Query:       Zn_Tnp_IS1595  [M=46]\n",
            "Accession:   PF12760.10\n",
            "Description: Transposase zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.036   13.4   5.9       0.79    9.1   0.3    2.7  3  XP_004358905.1  RNA polymerase I subunit [Cavenderia fascicul\n",
            "       0.04   13.3  22.0      0.037   13.4   3.9    3.5  3  XP_004356292.1  uncharacterized protein DFA_08808 [Cavenderia\n",
            "      0.088   12.2   4.0       0.15   11.5   4.0    1.4  1  XP_004362940.1  S60 ribosomal protein L37A [Cavenderia fascic\n",
            "       0.16   11.4   1.6       0.36   10.2   1.6    1.5  1  XP_004362193.1  uncharacterized protein DFA_06492 [Cavenderia\n",
            "        1.9    7.9   6.7       0.56    9.6   0.3    2.3  2  XP_004355086.1  PH domain-containing protein [Cavenderia fasc\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004358905.1  RNA polymerase I subunit [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.9   1.0    0.0033       8.1      19      45 ..     163     188 ..     157     189 .. 0.83\n",
            "   2 ?    1.8   0.1     0.065   1.6e+02      17      27 ..     179     189 ..     175     195 .. 0.76\n",
            "   3 ?    9.1   0.3   0.00033      0.79      20      43 ..     227     259 ..     221     261 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.9 bits;  conditional E-value: 0.0033\n",
            "   Zn_Tnp_IS1595  19 pvCPhCgsreayristrglfqCksCrk 45 \n",
            "                     + CP C s  +y  + +++  C  C++\n",
            "  XP_004358905.1 163 F-CPTCNSLLSYVSEFATRIDCPTCSY 188\n",
            "                     4.*****99888888889999999998 PP\n",
            "\n",
            "  == domain 2  score: 1.8 bits;  conditional E-value: 0.065\n",
            "   Zn_Tnp_IS1595  17 ngpvCPhCgsr 27 \n",
            "                     ++  CP C ++\n",
            "  XP_004358905.1 179 TRIDCPTCSYT 189\n",
            "                     5677****876 PP\n",
            "\n",
            "  == domain 3  score: 9.1 bits;  conditional E-value: 0.00033\n",
            "   Zn_Tnp_IS1595  20 vCPhCgsreayrist.........rglfqCksC 43 \n",
            "                     +CP Cg+ + y+ ++            + C +C\n",
            "  XP_004358905.1 227 KCPECGHGKMYFKTAqtrsadegqTIFYDCMKC 259\n",
            "                     6********999998788888776667777777 PP\n",
            "\n",
            ">> XP_004356292.1  uncharacterized protein DFA_08808 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.0   0.1     0.054   1.3e+02      20      38 ..     186     204 ..     180     205 .. 0.83\n",
            "   2 ?   13.4   3.9   1.5e-05     0.037      17      45 ..     285     312 ..     282     313 .. 0.87\n",
            "   3 ?    5.9   5.5    0.0033       8.1      19      45 ..     335     360 ..     329     361 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.0 bits;  conditional E-value: 0.054\n",
            "   Zn_Tnp_IS1595  20 vCPhCgsreayristrglf 38 \n",
            "                      C++C s+++ + s+r+ +\n",
            "  XP_004356292.1 186 DCSSCSSSSSSSSSHRKDY 204\n",
            "                     5*******99999999876 PP\n",
            "\n",
            "  == domain 2  score: 13.4 bits;  conditional E-value: 1.5e-05\n",
            "   Zn_Tnp_IS1595  17 ngpvCPhCgsreayristrglfqCksCrk 45 \n",
            "                     ++p CP+C++   +r    +++ Ck C k\n",
            "  XP_004356292.1 285 DKPTCPNCHVIATRRHDK-RRWWCKNCVK 312\n",
            "                     5799*****998888887.********65 PP\n",
            "\n",
            "  == domain 3  score: 5.9 bits;  conditional E-value: 0.0033\n",
            "   Zn_Tnp_IS1595  19 pvCPhCgsreayristrglfqCksCrk 45 \n",
            "                     p+C +C   +a r  + +++ CksC+k\n",
            "  XP_004356292.1 335 PQCGRCST-QATRKHDKRRWWCKSCKK 360\n",
            "                     66777773.44444447********97 PP\n",
            "\n",
            ">> XP_004362940.1  S60 ribosomal protein L37A [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.5   4.0   6.1e-05      0.15      19      46 .]      37      63 ..      29      63 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.5 bits;  conditional E-value: 6.1e-05\n",
            "   Zn_Tnp_IS1595 19 pvCPhCgsreayristrglfqCksCrkq 46\n",
            "                      C++Cg++++ r+  +g + C+ Cr++\n",
            "  XP_004362940.1 37 YTCSFCGKDSVKRTC-VGIWSCSGCRRT 63\n",
            "                    57*****88777777.6********985 PP\n",
            "\n",
            ">> XP_004362193.1  uncharacterized protein DFA_06492 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.2   1.6   0.00015      0.36      18      44 ..     260     286 ..     256     287 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.2 bits;  conditional E-value: 0.00015\n",
            "   Zn_Tnp_IS1595  18 gpvCPhCgsreayristrglfqCksCr 44 \n",
            "                     +p CP+C+   ++  ++ ++f C  C+\n",
            "  XP_004362193.1 260 EPICPSCDRLVSWNQADQTQFYCTTCK 286\n",
            "                     489*****999999999*********8 PP\n",
            "\n",
            ">> XP_004355086.1  PH domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.4   1.5      0.18   4.4e+02      37      45 ..    1112    1120 ..    1103    1121 .. 0.85\n",
            "   2 ?    9.6   0.3   0.00023      0.56       6      34 ..    1224    1252 ..    1223    1256 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.4 bits;  conditional E-value: 0.18\n",
            "   Zn_Tnp_IS1595   37 lfqCksCrk 45  \n",
            "                       ++C++C+k\n",
            "  XP_004355086.1 1112 CWKCSKCSK 1120\n",
            "                      69*****97 PP\n",
            "\n",
            "  == domain 2  score: 9.6 bits;  conditional E-value: 0.00023\n",
            "   Zn_Tnp_IS1595    6 aralleklrWpng.pvCPhCgsreayrist 34  \n",
            "                      a+++l++ +++++ p CP C  +++++ist\n",
            "  XP_004355086.1 1224 ALKYLCESCYNDTcPICPTCN-ESVITIST 1252\n",
            "                      789*****************9.66666665 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (46 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       609  (0.0501854); expected 242.7 (0.02)\n",
            "Passed bias filter:                      341  (0.0281005); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        61  (0.00502678); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        11  (0.000906469); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               5  [number of targets reported over threshold]\n",
            "# CPU time: 0.12u 0.01s 00:00:00.13 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 3936.51\n",
            "//\n",
            "Query:       Zn_Tnp_IS91  [M=91]\n",
            "Accession:   PF14319.9\n",
            "Description: Transposase zinc-binding domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "     0.0061   16.0   1.1     0.0061   16.0   1.1    1.8  2  XP_004367206.1  uncharacterized protein DFA_07344 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "          1    8.9   5.2        1.5    8.4   0.5    2.6  2  XP_004361876.1  uncharacterized protein DFA_06163 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004367206.1  uncharacterized protein DFA_07344 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.5   1.3      0.28   1.7e+03      42      56 ..       8      22 ..       2      44 .. 0.55\n",
            "   2 !   16.0   1.1     1e-06    0.0061       9      55 ..      44      88 ..      36      92 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.5 bits;  conditional E-value: 0.28\n",
            "     Zn_Tnp_IS91 42 CeeCghekivpfsCk 56\n",
            "                    C++ g+   ++f C \n",
            "  XP_004367206.1  8 CSSTGEYCNSMFGCY 22\n",
            "                    444444333333332 PP\n",
            "\n",
            "  == domain 2  score: 16.0 bits;  conditional E-value: 1e-06\n",
            "     Zn_Tnp_IS91  9 flekkeklspsqkkavekilkCrtaalGfaryeCeeCghekivpfsC 55\n",
            "                    +++   ++++ + ++ ++  +C++ + G ++ +C eC + + + ++C\n",
            "  XP_004367206.1 44 YRNV-YQVNQIIYQEWHSTSECSDIEPGIHTSQCGECYE-NSFINTC 88\n",
            "                    5555.67899999************************86.5555666 PP\n",
            "\n",
            ">> XP_004361876.1  uncharacterized protein DFA_06163 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.4   0.5   0.00024       1.5      49      81 ..     201     234 ..     184     242 .. 0.83\n",
            "   2 ?    1.6   0.4     0.032     2e+02      25      66 ..     385     407 ..     368     424 .. 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.4 bits;  conditional E-value: 0.00024\n",
            "     Zn_Tnp_IS91  49 kivpfsCksrh.CpsCqkketeewlekqqeellp 81 \n",
            "                     k ++ sC+ +  C++C +  te    kqq ++l+\n",
            "  XP_004361876.1 201 KDFCVSCGTQKsCNKCDHVDTEIITFKQQVKVLE 234\n",
            "                     667899998877*************999999987 PP\n",
            "\n",
            "  == domain 2  score: 1.6 bits;  conditional E-value: 0.032\n",
            "     Zn_Tnp_IS91  25 ekilkCrtaalGfaryeCeeCghekivpfsCksrhCpsCqkk 66 \n",
            "                     ++               C +C+    + +s   + C+sC+k+\n",
            "  XP_004361876.1 385 NE---------------CIHCK----HSNSKLAKYCTSCGKE 407\n",
            "                     33...............33333....3445555556666554 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (91 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       912  (0.0751545); expected 242.7 (0.02)\n",
            "Passed bias filter:                      336  (0.0276885); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        37  (0.00304903); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         4  (0.000329625); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.15u 0.00s 00:00:00.15 Elapsed: 00:00:00.09\n",
            "# Mc/sec: 6711.78\n",
            "//\n",
            "Query:       Zona_pellucida  [M=259]\n",
            "Accession:   PF00100.26\n",
            "Description: Zona pellucida-like domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (259 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1316  (0.108447); expected 242.7 (0.02)\n",
            "Passed bias filter:                      139  (0.0114545); expected 242.7 (0.02)\n",
            "Passed Vit filter:                         3  (0.000247219); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.19u 0.00s 00:00:00.19 Elapsed: 00:00:00.11\n",
            "# Mc/sec: 14995.70\n",
            "//\n",
            "Query:       ZoocinA_TRD  [M=106]\n",
            "Accession:   PF16775.8\n",
            "Description: Target recognition domain of lytic exoenzyme\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (106 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       405  (0.0333745); expected 242.7 (0.02)\n",
            "Passed bias filter:                      213  (0.0175525); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        12  (0.000988875); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 12058.30\n",
            "//\n",
            "Query:       Zot  [M=194]\n",
            "Accession:   PF05707.15\n",
            "Description: Zonular occludens toxin (Zot)\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.044   12.9   0.6        1.9    7.5   0.1    3.2  2  XP_004356293.1  putative ATPase [Cavenderia fasciculata]\n",
            "       0.11   11.6   0.0       0.22   10.6   0.0    1.4  1  XP_004355492.1  uncharacterized protein DFA_07989 [Cavenderia\n",
            "       0.12   11.5   0.3        1.9    7.6   0.0    2.2  2  XP_004358974.1  AAA ATPase domain-containing protein [Cavende\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004356293.1  putative ATPase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.2   0.1     0.042   1.7e+02      82     143 ..     383     450 ..     353     487 .. 0.76\n",
            "   2 ?    7.5   0.1   0.00047       1.9      56     106 ..     636     686 ..     604     700 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.2 bits;  conditional E-value: 0.042\n",
            "             Zot  82 allviDEcqeifpvrakegrkd.....pevi.naftehRhlgwDivlltqdiskihkevralaekavk 143\n",
            "                     ++ vi    eifp+  + ++k       +++ n   +    +  i+ +t ++ ++ + vr l +++ k\n",
            "  XP_004356293.1 383 SVFVIENIDEIFPNLENDDTKSqeeslTGMMmNIMEKNNSQNTFIIAMTSKLRNVSNLVRSLFDEQIK 450\n",
            "                     67788888888888777766665555534443666666777788888999999999999888777655 PP\n",
            "\n",
            "  == domain 2  score: 7.5 bits;  conditional E-value: 0.00047\n",
            "             Zot  56 rliridnpegaetlkefakwatidkgallviDEcqeifpvrakegrkdpev 106\n",
            "                     ++++ +  e+++ l+ ++k+a+ +  ++++iDE q if +r   g + +++\n",
            "  XP_004356293.1 636 EIVQGEVGESEKMLADIFKRAVQAAPSIVFIDEIQAIFGQRDSSGSHGKNI 686\n",
            "                     3444446788889999999999999*************9998866555555 PP\n",
            "\n",
            ">> XP_004355492.1  uncharacterized protein DFA_07989 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.6   0.0   5.4e-05      0.22       5      91 ..      57     145 ..      53     160 .. 0.60\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.6 bits;  conditional E-value: 5.4e-05\n",
            "             Zot   5 vtGapGsgKtsgavkkdllpalk.egRkiitnidGlr.lerieqvlgekvstarlirid.npegaetlkefakwatid..kgallviDEcqe 91 \n",
            "                     v G+pG+gKt+ a++  ++   k +   +++n  G++   +i+  l+++   ++  ++  + e  +++++  +    +  k   +++DE+  \n",
            "  XP_004355492.1  57 VSGQPGTGKTLTALNR-INSIPKrKCTTLFFNCMGMQdPANIYTELHNTLCKPKKKKTPgQSEMVSKIQQTICDP--EesKMFCVILDEVDS 145\n",
            "                     68************98.88866615566778*****988888874444333222222222222222222222222..123555566777765 PP\n",
            "\n",
            ">> XP_004358974.1  AAA ATPase domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.6   0.0     0.032   1.3e+02       4      22 ..     171     191 ..     170     209 .. 0.73\n",
            "   2 ?    7.6   0.0   0.00046       1.9      64     102 ..     210     248 ..     196     292 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.6 bits;  conditional E-value: 0.032\n",
            "             Zot   4 lvtGapGsgKts..gavkkdl 22 \n",
            "                     l +G+pG+gK++  +av  ++\n",
            "  XP_004358974.1 171 LLYGPPGTGKSYlaKAVATEI 191\n",
            "                     679********8223555555 PP\n",
            "\n",
            "  == domain 2  score: 7.6 bits;  conditional E-value: 0.00046\n",
            "             Zot  64 egaetlkefakwatidkgallviDEcqeifpvrakegrk 102\n",
            "                     ++++++k+++  a+ +k+++++iDE   + ++r   + +\n",
            "  XP_004358974.1 210 DSEKLVKQLFEMAREKKNSVIFIDEIDSLCSTRNDSESE 248\n",
            "                     7788899999999988***********999998876444 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (194 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       493  (0.0406263); expected 242.7 (0.02)\n",
            "Passed bias filter:                      303  (0.0249691); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        29  (0.00238978); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         3  (0.000247219); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.16u 0.01s 00:00:00.17 Elapsed: 00:00:00.09\n",
            "# Mc/sec: 14326.45\n",
            "//\n",
            "Query:       ZT_dimer  [M=79]\n",
            "Accession:   PF16916.8\n",
            "Description: Dimerisation domain of Zinc Transporter\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    3.3e-13   49.0   0.0    8.1e-13   47.8   0.0    1.6  1  XP_004361148.1  putative cation efflux pump [Cavenderia fasci\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004361148.1  putative cation efflux pump [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   47.8   0.0   6.7e-17   8.1e-13       2      77 ..     341     416 ..     340     417 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 47.8 bits;  conditional E-value: 6.7e-17\n",
            "                     ...HHHHHHHHHTT.-TT-EEEEEEEEEECTTEEEEEEEEEE-TTS-HHHHHHHHHHHHHHHHHH-TTEEEEEEEEE CS\n",
            "        ZT_dimer   2 peeeeeeIreiies.vpgvkgvhdlrtrksGnkifvdlhievdgdlsvkeaHeiadeieerlreelpevedvliHve 77 \n",
            "                     p+e +++++ i++s  p+v +v+ +r+ ++Gn+ fv+++i++++d++++++H+i ++++ +l+ +l+eve+ ++Hv+\n",
            "  XP_004361148.1 341 PPEFLQRLTYIAMShSPEVLKVDTCRAFHVGNNFFVEVDIVLPPDMPLQKSHDIGESLQIKLE-SLDEVERAFVHVD 416\n",
            "                     67789999999964499*********************************************5.899********97 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (79 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       357  (0.029419); expected 242.7 (0.02)\n",
            "Passed bias filter:                      272  (0.0224145); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        12  (0.000988875); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 8133.23\n",
            "//\n",
            "Query:       ZU5  [M=97]\n",
            "Accession:   PF00791.23\n",
            "Description: ZU5 domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (97 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       234  (0.0192831); expected 242.7 (0.02)\n",
            "Passed bias filter:                      182  (0.0149979); expected 242.7 (0.02)\n",
            "Passed Vit filter:                         8  (0.00065925); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 12082.85\n",
            "//\n",
            "Query:       Zw10  [M=513]\n",
            "Accession:   PF06248.16\n",
            "Description: Centromere/kinetochore Zw10\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    6.2e-33  113.7  21.0    2.5e-30  105.2  21.0    3.0  1  XP_004357742.1  centrosomal protein [Cavenderia fasciculata]\n",
            "     0.0023   16.2   0.6     0.0028   15.9   0.6    1.2  1  XP_004358831.1  uncharacterized protein DFA_00850 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004357742.1  centrosomal protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  105.2  21.0   4.1e-34   2.5e-30      56     512 ..     160     575 ..     107     576 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 105.2 bits;  conditional E-value: 4.1e-34\n",
            "            Zw10  56 ladldasveelaelkqqlkekevvlellehlkkieealeeleealkekdylkaaklleelkellealke...rkgqdvk.ilkaLsaeltvlsen 146\n",
            "                        +++      +l + l++ e  l+++   ++++  ++ +e+a ke d ++ +  l ++ +++++      +  q+++ il +++ +l+  +++\n",
            "  XP_004357742.1 160 VGRIRSNISYDQQLRKRLESIESTLVMVDYFEEYDGYVKAYEKAIKENDIVQSTRHLVKMETMVNIGIPdslQSMQPLPrILPSIKDQLKKRKNQ 254\n",
            "                     33444444444567888889999999**************************************997651114556666778888********** PP\n",
            "\n",
            "            Zw10 147 lvykLseewdrlvlwklpkskvklegdksnlkvklkLsteeselsevleAlavlgeleekLkafgqkllknvlkplivspslkveVekvdkeeg. 240\n",
            "                     l  +L ee+ ++++     +             +l++s +  ++ + + Al++lg l   + + ++++ k++++pl    +   +  k++ + + \n",
            "  XP_004357742.1 255 LKTQLLEEFTNALVI----N-------------HLEISLARDNIYQLIGALEQLGLLYFIVSSISNQIYKSIIQPLLSTVQARLSLTKNESQ-Pt 331\n",
            "                     ***********9443....3.............478888999***********************************999999999996654.34 PP\n",
            "\n",
            "            Zw10 241 .......vtlslessqktkgeksepskvYskvlevlkvllskLvssksqeklgkvslveilGdliwpelsellikdclvksiPetsskledyqev 328\n",
            "                              l+l ++ ++k ++s+ +++++ + ++++++ +++         ++ +l+e +G  iw ++s  +i++cl + iP + s+l  +q +\n",
            "  XP_004357742.1 332 idiskngNHLKLTIDIQEKNKISTIDNLFNILKNMFEFIYNHIFG-------KENRLMERFGTDIWSDISTTIISNCLKNIIPDDPSQLSLFQSI 419\n",
            "                     6777777789999999999999999999999999999**999933.......5689**************************************9 PP\n",
            "\n",
            "            Zw10 329 ieqteeFeeaLkeleflskdetdllkyaedvnvhFankkcqdvlvkarkLlksdldntVkvtpdetavkvlglksklkipaeleakeiseelkle 423\n",
            "                       q + Fee+L +++ +++ + + + +++++++hFa kk  ++l+kar ++  d  +   +      + v+g                       \n",
            "  XP_004357742.1 420 --QSKPFEEYLVQIGLINSTQQSITEFINNIESHFAEKKRHNLLAKARHIIMTDKFDS--ID---SSESVIGES--------------------- 486\n",
            "                     ..6789*******************************************998832222..21...222233322..................... PP\n",
            "\n",
            "            Zw10 424 neetlseltfsfprCrvSesakklmdLaektlkeattssaetavqLfltvrnilelyvdvvPtkhkenleslpqlsalfhNnCmYishl 512\n",
            "                     ++++ s +++ f    + + a+ l+++ +  l e  +ss  +  +++   r+i+ ++     ++hk++++++p l+ l++N C+++sh \n",
            "  XP_004357742.1 487 QQNNGSLQLYYFTPRSIFKNANTLIQMCKTALYESLQSSPLCSTKIYQGCRDIFDMFRFMYIKYHKDKIKTVPLLACLYYNSCLFLSHS 575\n",
            "                     2233334445555558999*********************************************************************5 PP\n",
            "\n",
            ">> XP_004358831.1  uncharacterized protein DFA_00850 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.9   0.6   4.6e-07    0.0028      15      73 ..      55     112 ..      41     129 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.9 bits;  conditional E-value: 4.6e-07\n",
            "            Zw10  15 tivkkYsefleslssteellekveslskeiddllekvekevladldasveelaelkqql 73 \n",
            "                     ti+k +s    ++++t++++++ +slske++  ++   ++  ++ld+  + ++e++q++\n",
            "  XP_004358831.1  55 TISKTFSAHELNVTQTDDMIHQMSSLSKELSSQINLLRGS-FQALDGINNSVTEINQNA 112\n",
            "                     799************************************9.667777777777777765 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (513 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       947  (0.0780387); expected 242.7 (0.02)\n",
            "Passed bias filter:                      146  (0.0120313); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        14  (0.00115369); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.31u 0.00s 00:00:00.31 Elapsed: 00:00:00.16\n",
            "# Mc/sec: 20841.57\n",
            "//\n",
            "Query:       Zwilch  [M=574]\n",
            "Accession:   PF09817.12\n",
            "Description: RZZ complex, subunit zwilch\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (574 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       942  (0.0776267); expected 242.7 (0.02)\n",
            "Passed bias filter:                      123  (0.010136); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        10  (0.000824063); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.29u 0.00s 00:00:00.28 Elapsed: 00:00:00.16\n",
            "# Mc/sec: 23686.18\n",
            "//\n",
            "Query:       Zwint  [M=252]\n",
            "Accession:   PF15556.9\n",
            "Description: ZW10 interactor\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (252 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1734  (0.142892); expected 242.7 (0.02)\n",
            "Passed bias filter:                      216  (0.0177998); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        36  (0.00296663); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.24u 0.01s 00:00:00.25 Elapsed: 00:00:00.12\n",
            "# Mc/sec: 13404.89\n",
            "//\n",
            "Query:       ZYG-11_interact  [M=295]\n",
            "Accession:   PF05884.15\n",
            "Description: Interactor of ZYG-11\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "       0.22   10.3   5.4       0.29   10.0   5.4    1.2  1  XP_004363104.1  uncharacterized protein DFA_03501 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004363104.1  uncharacterized protein DFA_03501 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.0   5.4   2.4e-05      0.29      19      78 ..      64     125 ..      52     172 .. 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.0 bits;  conditional E-value: 2.4e-05\n",
            "  ZYG-11_interact  19 vknegseeeavirlvirtysklgrdleqave.krlkklpeegerlleql.lssifsqkskds 78 \n",
            "                       ++e+++e+ +i+++++ ++ l +d+++  + ++ +k +e   + l+ + + ++ ++k k++\n",
            "   XP_004363104.1  64 DNVEDAQEKEAIAMAMKDMETLEKDVKEKKDiQEQQKQQEYEAQRLKRIeQDQLEEEKAKEQ 125\n",
            "                      4689************************7776333333333333333331223333333333 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (295 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1096  (0.0903173); expected 242.7 (0.02)\n",
            "Passed bias filter:                      682  (0.0562011); expected 242.7 (0.02)\n",
            "Passed Vit filter:                       117  (0.00964153); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        25  (0.00206016); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.51u 0.01s 00:00:00.52 Elapsed: 00:00:00.30\n",
            "# Mc/sec: 6590.21\n",
            "//\n",
            "Query:       ZZ  [M=45]\n",
            "Accession:   PF00569.20\n",
            "Description: Zinc finger, ZZ type\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    4.3e-11   41.9   6.7    7.5e-11   41.1   6.7    1.4  1  XP_004366193.1  myb domain-containing protein [Cavenderia fas\n",
            "    1.4e-10   40.2   9.7      3e-10   39.2   9.7    1.5  1  XP_004358632.1  ZZ-type zinc finger-containing protein [Caven\n",
            "    1.1e-09   37.3   3.7    2.1e-09   36.5   3.7    1.5  1  XP_004358587.1  ZZ-type zinc finger-containing protein [Caven\n",
            "    5.9e-09   35.1   8.2    5.9e-09   35.1   8.2    3.7  3  XP_004351748.1  uncharacterized protein DFA_09851 [Cavenderia\n",
            "    2.9e-08   32.9   6.9    5.8e-08   31.9   6.9    1.4  1  XP_004350198.1  myb domain-containing protein [Cavenderia fas\n",
            "    1.8e-07   30.3  32.0    0.00056   19.1   6.7    3.8  3  XP_004355615.1  uncharacterized protein DFA_08113 [Cavenderia\n",
            "    9.5e-05   21.6   7.1    9.5e-05   21.6   7.1    2.3  2  XP_004352141.1  uncharacterized protein DFA_09485 [Cavenderia\n",
            "    0.00042   19.5   6.9      0.001   18.3   6.9    1.7  1  XP_004352615.1  uncharacterized protein DFA_09189 [Cavenderia\n",
            "     0.0018   17.5   8.5     0.0058   15.9   8.5    1.9  1  XP_004360876.1  uncharacterized protein DFA_05155 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004366193.1  myb domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   41.1   6.7   5.5e-14   7.5e-11       1      45 []     354     400 ..     354     400 .. 0.95\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 41.1 bits;  conditional E-value: 5.5e-14\n",
            "                     SSSEEBETTTSSBBECSCEEEESSS.TSSEECHHHHHT.GCBTTCCC CS\n",
            "              ZZ   1 ihkkytCdgCseapligvRyhclrc.sdYDLCqsCfst.hkagkhkm 45 \n",
            "                     ih++y+CdgC + p++g Ry+c  c +d DLCqsC  +  k g hk+\n",
            "  XP_004366193.1 354 IHEGYKCDGCDQEPIVGYRYKCEECiEDIDLCQSCHDSyEKIGSHKS 400\n",
            "                     69***********************88***********999999985 PP\n",
            "\n",
            ">> XP_004358632.1  ZZ-type zinc finger-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   39.2   9.7   2.2e-13     3e-10       5      44 ..     220     259 ..     217     260 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 39.2 bits;  conditional E-value: 2.2e-13\n",
            "                     EBETTTSSBBECSCEEEESSSTSSEECHHHHHT.GCBTTCC CS\n",
            "              ZZ   5 ytCdgCseapligvRyhclrcsdYDLCqsCfst.hkagkhk 44 \n",
            "                     + Cd+C +  +ig Ry+c +c++YDLCq C+   +  ++h+\n",
            "  XP_004358632.1 220 ALCDICHK-QIIGIRYKCKVCPNYDLCQDCKEGgKHPADHE 259\n",
            "                     67******.**********************9977777775 PP\n",
            "\n",
            ">> XP_004358587.1  ZZ-type zinc finger-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   36.5   3.7   1.5e-12   2.1e-09       2      37 ..     207     241 ..     206     252 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 36.5 bits;  conditional E-value: 1.5e-12\n",
            "                     SSEEBETTTSSBBECSCEEEESSSTSSEECHHHHHT CS\n",
            "              ZZ   2 hkkytCdgCseapligvRyhclrcsdYDLCqsCfst 37 \n",
            "                     h ++ CdgC    + g R +c++c++YDLC sC+ +\n",
            "  XP_004358587.1 207 HPGVVCDGCNM-GIFGIRHKCAVCPNYDLCESCKEK 241\n",
            "                     66789******.*********************987 PP\n",
            "\n",
            ">> XP_004351748.1  uncharacterized protein DFA_09851 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   35.1   8.2   4.4e-12   5.9e-09       4      43 ..    2981    3021 ..    2978    3023 .. 0.93\n",
            "   2 ?   -4.4   4.2         9   1.2e+04       9      28 ..    3764    3781 ..    3761    3783 .. 0.72\n",
            "   3 ?    4.8   1.2     0.012        17       4      20 ..    3788    3804 ..    3786    3808 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 35.1 bits;  conditional E-value: 4.4e-12\n",
            "                      EEBETTTSSBBECSCEEEESSSTSSEECHHHHHT.GCBTTC CS\n",
            "              ZZ    4 kytCdgCseapligvRyhclrcsdYDLCqsCfst.hkagkh 43  \n",
            "                       ++Cd+Cs  p+ + Ryhc +c d+DLC+ C+ +   +++h\n",
            "  XP_004351748.1 2981 HFSCDKCSVYPIQQKRYHCEVCGDFDLCSDCYLKiTATKEH 3021\n",
            "                      589******************************98888888 PP\n",
            "\n",
            "  == domain 2  score: -4.4 bits;  conditional E-value: 9\n",
            "                      TTSSBBECSCEEEESSSTSS CS\n",
            "              ZZ    9 gCseapligvRyhclrcsdY 28  \n",
            "                      +C++    ++ ++c +c++ \n",
            "  XP_004351748.1 3764 ICKH--CHENAFQCKQCRNI 3781\n",
            "                      4777..44577899999885 PP\n",
            "\n",
            "  == domain 3  score: 4.8 bits;  conditional E-value: 0.012\n",
            "                      EEBETTTSSBBECSCEE CS\n",
            "              ZZ    4 kytCdgCseapligvRy 20  \n",
            "                      ++ C++C+++ ++++ y\n",
            "  XP_004351748.1 3788 GFLCNECGFCKHARFDY 3804\n",
            "                      678***********988 PP\n",
            "\n",
            ">> XP_004350198.1  myb domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   31.9   6.9   4.3e-11   5.8e-08       4      37 ..     225     258 ..     223     267 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 31.9 bits;  conditional E-value: 4.3e-11\n",
            "                     EEBETTTSSBBECSCEEEESSSTSSEECHHHHHT CS\n",
            "              ZZ   4 kytCdgCseapligvRyhclrcsdYDLCqsCfst 37 \n",
            "                     +y+Cd C ++ ++ v  +c +c+d+DLC +Cfs \n",
            "  XP_004350198.1 225 QYHCDYCQKDLSNVVKIKCSVCEDFDLCLECFSV 258\n",
            "                     8*******************************85 PP\n",
            "\n",
            ">> XP_004355615.1  uncharacterized protein DFA_08113 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   19.1   6.7   4.2e-07   0.00056       7      37 ..     247     277 ..     244     283 .. 0.89\n",
            "   2 !   11.1   4.2   0.00014      0.18       4      25 ..     315     335 ..     313     341 .. 0.93\n",
            "   3 !   11.3   5.2   0.00011      0.15       2      42 ..     441     481 .]     440     481 .] 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 19.1 bits;  conditional E-value: 4.2e-07\n",
            "                     ETTTSSBBECSCEEEESSSTSSEECHHHHHT CS\n",
            "              ZZ   7 CdgCseapligvRyhclrcsdYDLCqsCfst 37 \n",
            "                     Cd Cs+       +hc +c ++DLC sC  +\n",
            "  XP_004355615.1 247 CDYCSKGYGTSEQFHCSSCGNFDLCRSCIEK 277\n",
            "                     ******999999****************876 PP\n",
            "\n",
            "  == domain 2  score: 11.1 bits;  conditional E-value: 0.00014\n",
            "                     EEBETTTSSBBECSCEEEESSS CS\n",
            "              ZZ   4 kytCdgCseapligvRyhclrc 25 \n",
            "                     ++ Cd+C +  + gv+yhc++c\n",
            "  XP_004355615.1 315 EARCDKCLS-NVKGVVYHCTIC 335\n",
            "                     678****99.9**********9 PP\n",
            "\n",
            "  == domain 3  score: 11.3 bits;  conditional E-value: 0.00011\n",
            "                     SSEEBETTTSSB..BECSCEEEESSSTSSEECHHHHHTGCBTT CS\n",
            "              ZZ   2 hkkytCdgCsea..pligvRyhclrcsdYDLCqsCfsthkagk 42 \n",
            "                     h + +Cd+C +   ++  + yh+   ++YD C++C+ +h+++k\n",
            "  XP_004355615.1 441 HSEWICDVCRSRygATEITFYHS--PNNYDVCVGCYYKHFQKK 481\n",
            "                     78899****98422344455775..57***********99985 PP\n",
            "\n",
            ">> XP_004352141.1  uncharacterized protein DFA_09485 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   21.6   7.1   7.1e-08   9.5e-05       8      40 ..       3      36 ..       1      41 [. 0.79\n",
            "   2 ?   -0.7   0.3      0.65   8.7e+02       4      12 ..     150     158 ..     146     162 .. 0.68\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 21.6 bits;  conditional E-value: 7.1e-08\n",
            "                    TTTSSBBECSCEEEESSSTSSEECHHHHHT.GCB CS\n",
            "              ZZ  8 dgCseapligvRyhclrcsdYDLCqsCfst.hka 40\n",
            "                    ++C ++   gv y+c+ c++  LC+ C++  +k+\n",
            "  XP_004352141.1  3 NTCTACGDTGVKYKCTTCDNVSLCILCYNDpSKH 36\n",
            "                    45555555*********************95544 PP\n",
            "\n",
            "  == domain 2  score: -0.7 bits;  conditional E-value: 0.65\n",
            "                     EEBETTTSS CS\n",
            "              ZZ   4 kytCdgCse 12 \n",
            "                      y+C  Cs \n",
            "  XP_004352141.1 150 TYKCPYCST 158\n",
            "                     577777775 PP\n",
            "\n",
            ">> XP_004352615.1  uncharacterized protein DFA_09189 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   18.3   6.9   7.6e-07     0.001       5      40 ..      30      67 ..      27      72 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 18.3 bits;  conditional E-value: 7.6e-07\n",
            "                    EBETTTS.SBBECSCEEEESSSTSSEECHHHHHT.GCB CS\n",
            "              ZZ  5 ytCdgCs.eapligvRyhclrcsdYDLCqsCfst.hka 40\n",
            "                    + C+gCs +  + + Ry c+ c   DLC +C++  +k+\n",
            "  XP_004352615.1 30 FMCNGCSyNFKHGEPRYVCTECAGIDLCKKCYKRdQKY 67\n",
            "                    78****943478899*****************985443 PP\n",
            "\n",
            ">> XP_004360876.1  uncharacterized protein DFA_05155 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.9   8.5   4.3e-06    0.0058       2      38 ..     756     793 ..     755     797 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.9 bits;  conditional E-value: 4.3e-06\n",
            "                     SSEEBETTTSSBBECSCEEEESSSTSSEECHHHHHT.G CS\n",
            "              ZZ   2 hkkytCdgCseapligvRyhclrcsdYDLCqsCfst.h 38 \n",
            "                     h + tC +C + p+ig+ y c +c  +  C+sC  + +\n",
            "  XP_004360876.1 756 HRGTTCTVCTQSPIIGTFYYCSVCTGFTFCDSCRAQqS 793\n",
            "                     7789*****************************98833 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (45 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       729  (0.0600742); expected 242.7 (0.02)\n",
            "Passed bias filter:                      285  (0.0234858); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        36  (0.00296663); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         9  (0.000741656); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               9  [number of targets reported over threshold]\n",
            "# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 4529.01\n",
            "//\n",
            "[ok]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "полный поиск"
      ],
      "metadata": {
        "id": "gpcFmHOdb_o4"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "U8MiRfA9cPSF",
        "outputId": "57701ab5-e017-42b3-b67c-74b81d4bbd0b"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mВыходные данные были обрезаны до нескольких последних строк (5000).\u001b[0m\n",
            "   1 ?   11.6   0.1   1.5e-05     0.093       1      14 [.      29      42 ..      29      49 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.6 bits;  conditional E-value: 1.5e-05\n",
            "           zf_Rg  1 MiksvfeklCPNCg 14\n",
            "                    Mi +v++++C NC \n",
            "  XP_004357565.1 29 MIMLVVKRSCANCT 42\n",
            "                    9************6 PP\n",
            "\n",
            ">> XP_004359870.1  uncharacterized protein DFA_01908 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.1   0.2     0.014        85       9      28 ..      80      99 ..      74     112 .. 0.84\n",
            "   2 ?   -2.6   0.1      0.43   2.6e+03      26      32 ..     416     422 ..     415     428 .. 0.85\n",
            "   3 ?    7.8   0.2   0.00023       1.4       8      26 ..     430     448 ..     424     454 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.1 bits;  conditional E-value: 0.014\n",
            "           zf_Rg  9 lCPNCgGdisdeRlekGLpC 28\n",
            "                     CP C  +is+  l + L+ \n",
            "  XP_004359870.1 80 KCPQCRVPISNGGLSRSLLA 99\n",
            "                    7*********9999988875 PP\n",
            "\n",
            "  == domain 2  score: -2.6 bits;  conditional E-value: 0.43\n",
            "           zf_Rg  26 LpCkrCl 32 \n",
            "                     L C++Cl\n",
            "  XP_004359870.1 416 LGCQSCL 422\n",
            "                     67****8 PP\n",
            "\n",
            "  == domain 3  score: 7.8 bits;  conditional E-value: 0.00023\n",
            "           zf_Rg   8 klCPNCgGdisdeRlekGL 26 \n",
            "                       CP C  +is+ Rl + L\n",
            "  XP_004359870.1 430 GTCPQCRTPISNGRLSRSL 448\n",
            "                     57*************9987 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (49 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       302  (0.0248867); expected 242.7 (0.02)\n",
            "Passed bias filter:                      181  (0.0149155); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        22  (0.00181294); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 3069.07\n",
            "//\n",
            "Query:       zf_UBZ  [M=32]\n",
            "Accession:   PF18439.4\n",
            "Description: Ubiquitin-Binding Zinc Finger\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (32 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       527  (0.0434281); expected 242.7 (0.02)\n",
            "Passed bias filter:                      325  (0.026782); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        49  (0.00403791); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000576844); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 2094.87\n",
            "//\n",
            "Query:       zf_ZIC  [M=46]\n",
            "Accession:   PF18366.4\n",
            "Description: Zic proteins zinc finger domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (46 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       250  (0.0206016); expected 242.7 (0.02)\n",
            "Passed bias filter:                      210  (0.0173053); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        15  (0.00123609); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.09\n",
            "# Mc/sec: 3397.21\n",
            "//\n",
            "Query:       zinc-ribbons_6  [M=67]\n",
            "Accession:   PF07191.15\n",
            "Description: zinc-ribbons\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    0.00073   18.9  10.7     0.0015   17.9  10.7    1.5  1  XP_004355683.1  uncharacterized protein DFA_08186 [Cavenderia\n",
            "     0.0098   15.3   4.5       0.02   14.3   4.5    1.5  1  XP_004366868.1  MEK1 interacting protein 1 [Cavenderia fascic\n",
            "  ------ inclusion threshold ------\n",
            "      0.042   13.3   3.4      0.084   12.3   3.4    1.4  1  XP_004366193.1  myb domain-containing protein [Cavenderia fas\n",
            "      0.042   13.3   6.5        0.1   12.0   6.5    1.6  1  XP_004362359.1  uncharacterized protein DFA_02751 [Cavenderia\n",
            "      0.058   12.8   0.6       0.18   11.3   0.6    1.8  1  XP_004362193.1  uncharacterized protein DFA_06492 [Cavenderia\n",
            "       0.52    9.8  13.9       0.92    9.0  11.8    2.6  2  XP_004361325.1  uncharacterized protein DFA_05607 [Cavenderia\n",
            "        2.1    7.8   8.9        4.6    6.8   5.9    1.9  2  XP_004350883.1  methionyl-tRNA synthetase [Cavenderia fascicu\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004355683.1  uncharacterized protein DFA_08186 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.9  10.7   8.8e-07    0.0015       3      47 ..      38      94 ..      36      98 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.9 bits;  conditional E-value: 8.8e-07\n",
            "                    -SSS-S--EE-S.........SEEE-TTT--...EEEEEEE-SSS-S---EEEETTE CS\n",
            "  zinc-ribbons_6  3 CpeCqqelekrg.........eqyhCeaCqe...dlklealCpeCeeelevlkaCga 47\n",
            "                    C++C+  +e+r+            +C +C++       + lC++C ++l+v+ +C +\n",
            "  XP_004355683.1 38 CQDCHDTIEWRKkyrkykplsMAGKCAHCTQrkvTRAYHELCEDCVKKLKVCAKCHK 94\n",
            "                    *********9999******9988899999885445567889**************65 PP\n",
            "\n",
            ">> XP_004366868.1  MEK1 interacting protein 1 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   14.3   4.5   1.2e-05      0.02       2      40 ..     314     357 ..     297     362 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.3 bits;  conditional E-value: 1.2e-05\n",
            "                     --SSS-S--EE-S....SEEE-TTT--.EEEEEEE-SSS-S--- CS\n",
            "  zinc-ribbons_6   2 kCpeCqqelekrg....eqyhCeaCqe.dlklealCpeCeeele 40 \n",
            "                      Cp C +++ek +     +++C  C +  l+++++Cp C+ +l \n",
            "  XP_004366868.1 314 SCPICFEKMEKMSsttcGHVFCTDCIQaALTRRKVCPVCSTKLT 357\n",
            "                     7*******997776788999*****665999*********9997 PP\n",
            "\n",
            ">> XP_004366193.1  myb domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.3   3.4   4.8e-05     0.084      15      46 ..     356     389 ..     338     403 .. 0.68\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.3 bits;  conditional E-value: 4.8e-05\n",
            "                     SEEE-TTT--..EEEEEEE-SSS-S---EEEETT CS\n",
            "  zinc-ribbons_6  15 eqyhCeaCqe..dlklealCpeCeeelevlkaCg 46 \n",
            "                     e y+C+ C++   +  +  C+eC e ++ ++ C \n",
            "  XP_004366193.1 356 EGYKCDGCDQepIVGYRYKCEECIEDIDLCQSCH 389\n",
            "                     5577777777444566666777777777777774 PP\n",
            "\n",
            ">> XP_004362359.1  uncharacterized protein DFA_02751 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.0   6.5     6e-05       0.1      15      51 ..     843     879 ..     824     888 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.0 bits;  conditional E-value: 6e-05\n",
            "                     SEEE-TTT--EEEEEEE-SSS-S---EEEETTEEEE- CS\n",
            "  zinc-ribbons_6  15 eqyhCeaCqedlklealCpeCeeelevlkaCgavdyf 51 \n",
            "                     + y+C++C++ l +  lC  C+  +   kaC ++++f\n",
            "  XP_004362359.1 843 QLYRCHQCKQTLMKPLLCSGCKAVVYCSKACQKESWF 879\n",
            "                     569******************************9999 PP\n",
            "\n",
            ">> XP_004362193.1  uncharacterized protein DFA_06492 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.3   0.6    0.0001      0.18       2      28 ..     262     291 ..     261     295 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.3 bits;  conditional E-value: 0.0001\n",
            "                     --SSS-S--EE-S...SEEE-TTT--EEEE CS\n",
            "  zinc-ribbons_6   2 kCpeCqqelekrg...eqyhCeaCqedlkl 28 \n",
            "                      Cp+C++ +++++   +q++C +C++ +++\n",
            "  XP_004362193.1 262 ICPSCDRLVSWNQadqTQFYCTTCKQPFNI 291\n",
            "                     5*******99988888999*****998876 PP\n",
            "\n",
            ">> XP_004361325.1  uncharacterized protein DFA_05607 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.7   0.0       1.2   2.1e+03      49      63 ..     323     338 ..     322     341 .. 0.73\n",
            "   2 ?    9.0  11.8   0.00053      0.92       2      55 ..     820     885 ..     819     891 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.7 bits;  conditional E-value: 1.2\n",
            "                     EE--TTTT.----TTS CS\n",
            "  zinc-ribbons_6  49 dyfCnkcn.eLiSkkr 63 \n",
            "                     ++fC     +LiS +r\n",
            "  XP_004361325.1 323 GFFCRGASgSLISNER 338\n",
            "                     79**965427999876 PP\n",
            "\n",
            "  == domain 2  score: 9.0 bits;  conditional E-value: 0.00053\n",
            "                     --SSS-S--EE-S.......SEEE-TTT--EEE....EEEE-SSS-S---EE...EETTEEEE--TTT CS\n",
            "  zinc-ribbons_6   2 kCpeCqqelekrg.......eqyhCeaCqedlk....lealCpeCeeelevl...kaCgavdyfCnkc 55 \n",
            "                     +C  C + l+k++       +++hC+aC+++++    ++  C  Ce+ le       Cg    +C +c\n",
            "  XP_004361325.1 820 QCSGCAKTLTKNEphfkrknDKVHCDACEKEYQslkmRNRKCTSCEKILEQNgtrVICGQ--LYCPTC 885\n",
            "                     7999*9999988889999999*********77622223457********97522256988..899988 PP\n",
            "\n",
            ">> XP_004350883.1  methionyl-tRNA synthetase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.8   5.9    0.0026       4.6       3      36 ..     149     182 ..     132     184 .. 0.88\n",
            "   2 ?    6.0   8.9    0.0045       7.8       3      40 ..     149     186 ..     147     210 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.8 bits;  conditional E-value: 0.0026\n",
            "                     -SSS-S--EE-SSEEE-TTT--EEEEEEE-SSS- CS\n",
            "  zinc-ribbons_6   3 CpeCqqelekrgeqyhCeaCqedlklealCpeCe 36 \n",
            "                     C++C+  l  r  +  C +Cq +  +   C++C \n",
            "  XP_004350883.1 149 CEQCKIFLADRYVEGICPHCQYEDARGDQCDKCT 182\n",
            "                     9999999999999999999999999999999997 PP\n",
            "\n",
            "  == domain 2  score: 6.0 bits;  conditional E-value: 0.0045\n",
            "                     -SSS-S--EE-SSEEE-TTT--EEEEEEE-SSS-S--- CS\n",
            "  zinc-ribbons_6   3 CpeCqqelekrgeqyhCeaCqedlklealCpeCeeele 40 \n",
            "                     C++C+  l  r  +  C +Cq +  +   C++C + l+\n",
            "  XP_004350883.1 149 CEQCKIFLADRYVEGICPHCQYEDARGDQCDKCTKLLN 186\n",
            "                     *********************************99886 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (67 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       907  (0.0747425); expected 242.7 (0.02)\n",
            "Passed bias filter:                      377  (0.0310672); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        73  (0.00601566); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        12  (0.000988875); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               7  [number of targets reported over threshold]\n",
            "# CPU time: 0.16u 0.00s 00:00:00.16 Elapsed: 00:00:00.17\n",
            "# Mc/sec: 2642.01\n",
            "//\n",
            "Query:       zinc-ribbon_6  [M=93]\n",
            "Accession:   PF10005.12\n",
            "Description: zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (93 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       944  (0.0777915); expected 242.7 (0.02)\n",
            "Passed bias filter:                      334  (0.0275237); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        25  (0.00206016); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.12\n",
            "# Mc/sec: 5153.79\n",
            "//\n",
            "Query:       Zincin_1  [M=97]\n",
            "Accession:   PF06262.14\n",
            "Description: Zincin-like metallopeptidase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (97 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       217  (0.0178822); expected 242.7 (0.02)\n",
            "Passed bias filter:                      179  (0.0147507); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        16  (0.0013185); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.11\n",
            "# Mc/sec: 5854.04\n",
            "//\n",
            "Query:       Zincin_2  [M=332]\n",
            "Accession:   PF10103.12\n",
            "Description: Zincin-like metallopeptidase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (332 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       501  (0.0412855); expected 242.7 (0.02)\n",
            "Passed bias filter:                      250  (0.0206016); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        14  (0.00115369); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.22u 0.00s 00:00:00.22 Elapsed: 00:00:00.23\n",
            "# Mc/sec: 9671.14\n",
            "//\n",
            "Query:       zinc_ribbon_10  [M=54]\n",
            "Accession:   PF10058.12\n",
            "Description: Predicted integral membrane zinc-ribbon metal-binding protein\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    7.8e-19   66.6   0.2    1.5e-18   65.7   0.2    1.5  1  XP_004361551.1  uncharacterized protein DFA_05834 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "      0.015   14.5   0.3       0.41    9.9   0.0    2.3  2  XP_004359231.1  Peptide methionine sulfoxide reductase [Caven\n",
            "      0.038   13.2   4.6       0.82    8.9   0.1    2.9  3  XP_004360352.1  RNA polymerase III [Cavenderia fasciculata]\n",
            "        0.3   10.3   4.5        6.7    6.0   0.5    3.0  3  XP_004359161.1  RNA polymerase III subunit [Cavenderia fascic\n",
            "          4    6.7   6.6         12    5.2   6.6    1.8  1  XP_004355145.1  myb domain-containing protein [Cavenderia fas\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004361551.1  uncharacterized protein DFA_05834 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   65.7   0.2   6.3e-22   1.5e-18       1      54 []     351     401 ..     351     401 .. 0.98\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 65.7 bits;  conditional E-value: 6.3e-22\n",
            "  zinc_ribbon_10   1 wfdrvldlLvGddetaeknryaLICskCrshnGlapkeeleevqyrCpnCgaln 54 \n",
            "                     w+d+v+d+L++d++   k+ ++LIC+kC+shnG++p+ e+++vq+rC +C+++n\n",
            "  XP_004361551.1 351 WLDKVVDYLISDGP---KYGSPLICKKCHSHNGYVPTGEISTVQFRCRFCQYFN 401\n",
            "                     9*************...*********************************9987 PP\n",
            "\n",
            ">> XP_004359231.1  Peptide methionine sulfoxide reductase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.6   0.0     0.032        78      35      52 ..      87     103 ..      81     105 .. 0.80\n",
            "   2 ?    9.9   0.0   0.00017      0.41      13      36 ..     134     157 ..     130     165 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.6 bits;  conditional E-value: 0.032\n",
            "  zinc_ribbon_10  35 apkeeleevqyrCpnCga 52 \n",
            "                     +  +++++  y C+ Cga\n",
            "  XP_004359231.1  87 Y-DKKFDDGVYGCAGCGA 103\n",
            "                     5.67788889******97 PP\n",
            "\n",
            "  == domain 2  score: 9.9 bits;  conditional E-value: 0.00017\n",
            "  zinc_ribbon_10  13 detaeknryaLICskCrshnGlap 36 \n",
            "                     d+++ + r  ++C++C  h G + \n",
            "  XP_004359231.1 134 DKSHGMVRTEIVCKNCGGHLGHVF 157\n",
            "                     5556677899***********885 PP\n",
            "\n",
            ">> XP_004360352.1  RNA polymerase III [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.0   0.2    0.0057        14      17      30 ..     192     205 ..     186     207 .. 0.83\n",
            "   2 ?    8.9   0.1   0.00034      0.82      19      38 ..     241     260 ..     225     268 .. 0.83\n",
            "   3 ?   -3.0   0.1       1.8   4.5e+03      37      48 ..     702     713 ..     700     714 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.0 bits;  conditional E-value: 0.0057\n",
            "  zinc_ribbon_10  17 eknryaLICskCrs 30 \n",
            "                      +n++ +IC++C +\n",
            "  XP_004360352.1 192 ITNILQMICKNCST 205\n",
            "                     68999*******87 PP\n",
            "\n",
            "  == domain 2  score: 8.9 bits;  conditional E-value: 0.00034\n",
            "  zinc_ribbon_10  19 nryaLICskCrshnGlapke 38 \n",
            "                     +r+  IC kC+s+nG ++k+\n",
            "  XP_004360352.1 241 CRKKKICPKCNSVNGPVKKV 260\n",
            "                     56666**********99743 PP\n",
            "\n",
            "  == domain 3  score: -3.0 bits;  conditional E-value: 1.8\n",
            "  zinc_ribbon_10  37 keeleevqyrCp 48 \n",
            "                     ++++++ +y Cp\n",
            "  XP_004360352.1 702 SRTYTKNTYMCP 713\n",
            "                     5689999****8 PP\n",
            "\n",
            ">> XP_004359161.1  RNA polymerase III subunit [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    3.3   0.1      0.02        48      40      51 ..       9      20 ..       2      32 .. 0.69\n",
            "   2 ?    1.5   0.0     0.071   1.7e+02      19      51 ..      55      65 ..      42      68 .. 0.52\n",
            "   3 ?    6.0   0.5    0.0028       6.7      35      52 ..      78      95 ..      73      97 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 3.3 bits;  conditional E-value: 0.02\n",
            "  zinc_ribbon_10 40 leevqyrCpnCg 51\n",
            "                      +++++C  C \n",
            "  XP_004359161.1  9 QGKLKFYCQTCP 20\n",
            "                    446677777774 PP\n",
            "\n",
            "  == domain 2  score: 1.5 bits;  conditional E-value: 0.071\n",
            "  zinc_ribbon_10 19 nryaLICskCrshnGlapkeeleevqyrCpnCg 51\n",
            "                    + + + C                      p C+\n",
            "  XP_004359161.1 55 QQVDIEC----------------------PTCK 65\n",
            "                    3444445......................5554 PP\n",
            "\n",
            "  == domain 3  score: 6.0 bits;  conditional E-value: 0.0028\n",
            "  zinc_ribbon_10 35 apkeeleevqyrCpnCga 52\n",
            "                    +p +e+++  y+C +C +\n",
            "  XP_004359161.1 78 YPADEPKTAYYKCTYCAT 95\n",
            "                    678899999*******75 PP\n",
            "\n",
            ">> XP_004355145.1  myb domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.2   6.6    0.0049        12      14      53 ..     580     619 ..     576     620 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.2 bits;  conditional E-value: 0.0049\n",
            "  zinc_ribbon_10  14 etaeknryaLICskCrshnGlapkeeleevqyrCpnCgal 53 \n",
            "                     ++ ++n +a+ C  C++ +  + +++ +  q++C +C+++\n",
            "  XP_004355145.1 580 NSVTTNDQAIACYSCAHLFHHVCVVSHSGQQWQCTYCNQI 619\n",
            "                     4568999************999999************986 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (54 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       597  (0.0491965); expected 242.7 (0.02)\n",
            "Passed bias filter:                      343  (0.0282653); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        35  (0.00288422); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         5  (0.000412031); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               5  [number of targets reported over threshold]\n",
            "# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.12\n",
            "# Mc/sec: 3032.19\n",
            "//\n",
            "Query:       zinc_ribbon_11  [M=128]\n",
            "Accession:   PF11682.11\n",
            "Description: Probable zinc-ribbon\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (128 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       490  (0.0403791); expected 242.7 (0.02)\n",
            "Passed bias filter:                      267  (0.0220025); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        15  (0.00123609); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.12u 0.01s 00:00:00.13 Elapsed: 00:00:00.13\n",
            "# Mc/sec: 6600.78\n",
            "//\n",
            "Query:       zinc_ribbon_12  [M=45]\n",
            "Accession:   PF11331.11\n",
            "Description: Probable zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.056   12.7   0.1       0.12   11.7   0.1    1.5  1  XP_004358572.1  countin2 [Cavenderia fasciculata]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004358572.1  countin2 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.7   0.1   9.7e-06      0.12       3      15 ..      48      60 ..      47      62 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.7 bits;  conditional E-value: 9.7e-06\n",
            "  zinc_ribbon_12  3 GAPFvvCsnCfel 15\n",
            "                    GAPF++C+ C+ +\n",
            "  XP_004358572.1 48 GAPFTTCNDCVSI 60\n",
            "                    9*********976 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (45 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       497  (0.0409559); expected 242.7 (0.02)\n",
            "Passed bias filter:                      325  (0.026782); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        28  (0.00230738); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.11\n",
            "# Mc/sec: 2710.74\n",
            "//\n",
            "Query:       zinc_ribbon_13  [M=63]\n",
            "Accession:   PF09855.12\n",
            "Description: Nucleic-acid-binding protein containing Zn-ribbon domain (DUF2082)\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "     0.0074   15.9   0.4     0.0074   15.9   0.4    1.9  2  XP_004357594.1  RNA polymerase [Cavenderia fasciculata]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004357594.1  RNA polymerase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.9   0.2      0.11   1.3e+03       2       8 ..       7      13 ..       6      20 .. 0.67\n",
            "   2 !   15.9   0.4   6.1e-07    0.0074      36      57 ..      22      43 ..      20      50 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.9 bits;  conditional E-value: 0.11\n",
            "  zinc_ribbon_13  2 CpKCgnt 8 \n",
            "                    C  Cg++\n",
            "  XP_004357594.1  7 CGECGSE 13\n",
            "                    6667765 PP\n",
            "\n",
            "  == domain 2  score: 15.9 bits;  conditional E-value: 6.1e-07\n",
            "  zinc_ribbon_13 36 vsCknCgYtelYkketsegkni 57\n",
            "                    + Ck+C+Y  +Ykk+t + +ni\n",
            "  XP_004357594.1 22 IKCKDCTYRIMYKKRTDKSMNI 43\n",
            "                    79******************98 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (63 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       474  (0.0390606); expected 242.7 (0.02)\n",
            "Passed bias filter:                      259  (0.0213432); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        13  (0.00107128); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 5009.71\n",
            "//\n",
            "Query:       zinc_ribbon_15  [M=86]\n",
            "Accession:   PF17032.8\n",
            "Description: zinc-ribbon family\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    3.1e-09   36.9   0.9    3.1e-09   36.9   0.9    1.6  2  XP_004360973.1  uncharacterized protein DFA_05252 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "      0.025   14.8   3.7      0.056   13.7   3.7    1.7  1  XP_004351727.1  uncharacterized protein DFA_09827 [Cavenderia\n",
            "      0.042   14.1   0.6      0.042   14.1   0.6    2.1  2  XP_004359819.1  uncharacterized protein DFA_01854 [Cavenderia\n",
            "       0.65   10.3   4.4        3.1    8.1   1.9    2.3  2  XP_004359231.1  Peptide methionine sulfoxide reductase [Caven\n",
            "        2.6    8.3   9.2       0.32   11.2   0.6    3.5  4  XP_004361876.1  uncharacterized protein DFA_06163 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004360973.1  uncharacterized protein DFA_05252 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   36.9   0.9   1.3e-12   3.1e-09       2      51 ..      24      72 ..      23      95 .. 0.89\n",
            "   2 ?   -2.5   0.1       2.5   6.1e+03      39      51 ..     139     151 ..     122     167 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 36.9 bits;  conditional E-value: 1.3e-12\n",
            "  zinc_ribbon_15  2 lCprCgklskaelvkkkkvlklffipvfkwkkkkpylacskCkslleask 51\n",
            "                    +Cp+C++  +++l +++++ +lffip++k+kk +p+l+c  C+ +++  +\n",
            "  XP_004360973.1 24 VCPNCHN-RSVQLERNDSWAHLFFIPFCKVKKGSPFLHCAACGCTVPYVE 72\n",
            "                    7*****6.899********************************9988765 PP\n",
            "\n",
            "  == domain 2  score: -2.5 bits;  conditional E-value: 2.5\n",
            "  zinc_ribbon_15  39 acskCkslleask 51 \n",
            "                       ++ ++++e+++\n",
            "  XP_004360973.1 139 NYNDQSTFYESNN 151\n",
            "                     3333334444444 PP\n",
            "\n",
            ">> XP_004351727.1  uncharacterized protein DFA_09827 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.7   3.7   2.3e-05     0.056      64      85 ..      43      87 ..       5      88 .. 0.64\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.7 bits;  conditional E-value: 2.3e-05\n",
            "  zinc_ribbon_15 64 e.......................kkeCenCgkeveeefkyCpkC 85\n",
            "                                            +++C++C+ +ve++  +C+ C\n",
            "  XP_004351727.1 43 LfinshkqeytfkyststtnhkhhQRSCWSCKMNVEPKEFFCSIC 87\n",
            "                    244444444445555555566667********************9 PP\n",
            "\n",
            ">> XP_004359819.1  uncharacterized protein DFA_01854 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.7   0.4       0.7   1.7e+03      64      65 ..     120     145 ..      98     165 .. 0.49\n",
            "   2 ?   14.1   0.6   1.7e-05     0.042      64      86 .]     261     294 ..     232     294 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.7 bits;  conditional E-value: 0.7\n",
            "  zinc_ribbon_15  64 e........................k 65 \n",
            "                     +                        +\n",
            "  XP_004359819.1 120 KekkdqdkekggnwftkfmdrleseS 145\n",
            "                     11112233345555555555554441 PP\n",
            "\n",
            "  == domain 2  score: 14.1 bits;  conditional E-value: 1.7e-05\n",
            "  zinc_ribbon_15  64 e.........kkeCenCgkev..eeefkyCpkCg 86 \n",
            "                     +         k  C +Cg++v  +++ ++C++Cg\n",
            "  XP_004359819.1 261 TsyhstghsgKSFCTKCGTKVpvSSSASFCSSCG 294\n",
            "                     245556667799******9872256799*****9 PP\n",
            "\n",
            ">> XP_004359231.1  Peptide methionine sulfoxide reductase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.1   1.9    0.0013       3.1      14      86 .]       7     117 ..       4     117 .. 0.81\n",
            "   2 ?    2.3   0.0     0.082     2e+02      65      74 ..     143     152 ..     128     173 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.1 bits;  conditional E-value: 0.0013\n",
            "  zinc_ribbon_15  14 lvkkkkvlklffipvfkwkkkkpylacskCkslleaskelgkeiekgesse......................................kkeCen 70 \n",
            "                     ++ +++  +lf ++ fk+++++++l +++  s +++ ++++++ ++++  +                                      ++ C+ \n",
            "  XP_004359231.1   7 YKSNTSQTSLFSVEQFKYNNNNKVLLSQQTYSYYSTTSNNNNNNNNTTM-NlsdsewkvkltpeqfkvlrekgteraftgeydkkfddgVYGCAG 100\n",
            "                     567788899***************9999999999988744444443333.358899999999999999999999999999999999999999999 PP\n",
            "\n",
            "  zinc_ribbon_15  71 Cgkeveee.fkyCpkCg 86 \n",
            "                     Cg+ + ++ +k+ + Cg\n",
            "  XP_004359231.1 101 CGAPLYSSkSKFNSGCG 117\n",
            "                     99976544378877776 PP\n",
            "\n",
            "  == domain 2  score: 2.3 bits;  conditional E-value: 0.082\n",
            "  zinc_ribbon_15  65 kkeCenCgke 74 \n",
            "                     + +C+nCg +\n",
            "  XP_004359231.1 143 EIVCKNCGGH 152\n",
            "                     5556666544 PP\n",
            "\n",
            ">> XP_004361876.1  uncharacterized protein DFA_06163 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.2   0.4       0.5   1.2e+03      64      72 ..     201     217 ..     178     228 .. 0.61\n",
            "   2 ?   11.2   0.6   0.00013      0.32      64      86 .]     371     405 ..     330     405 .. 0.75\n",
            "   3 ?   -1.5   0.1       1.3   3.1e+03      54      75 ..     962     983 ..     942     988 .. 0.67\n",
            "   4 ?   -2.2   0.0         2     5e+03      25      50 ..    1377    1402 ..    1372    1408 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.2 bits;  conditional E-value: 0.5\n",
            "  zinc_ribbon_15  64 e........kkeCenCg 72 \n",
            "                     +        +k+C++C+\n",
            "  XP_004361876.1 201 KdfcvscgtQKSCNKCD 217\n",
            "                     13444455556666665 PP\n",
            "\n",
            "  == domain 2  score: 11.2 bits;  conditional E-value: 0.00013\n",
            "  zinc_ribbon_15  64 e............kkeCenCgkeveeefkyCpkCg 86 \n",
            "                     +            ++eC +C+++ ++  kyC++Cg\n",
            "  XP_004361876.1 371 NakstfggvvvstTNECIHCKHSNSKLAKYCTSCG 405\n",
            "                     044456667777799*******************9 PP\n",
            "\n",
            "  == domain 3  score: -1.5 bits;  conditional E-value: 1.3\n",
            "  zinc_ribbon_15  54 gkeiekgessekkeCenCgkev 75 \n",
            "                     +kei ++ ++ +++C +C+ ++\n",
            "  XP_004361876.1 962 KKEIPEHLNRYSHVCYQCKVSI 983\n",
            "                     3444444444499999999876 PP\n",
            "\n",
            "  == domain 4  score: -2.2 bits;  conditional E-value: 2\n",
            "  zinc_ribbon_15   25 fipvfkwkkkkpylacskCkslleas 50  \n",
            "                      f p+ +++ +k+++++ +C++l++  \n",
            "  XP_004361876.1 1377 FAPMSRTDVEKYMATTGRCSALVKVT 1402\n",
            "                      6788888999999******9998765 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (86 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1132  (0.0932839); expected 242.7 (0.02)\n",
            "Passed bias filter:                      523  (0.0430985); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        67  (0.00552122); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000576844); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               5  [number of targets reported over threshold]\n",
            "# CPU time: 0.15u 0.01s 00:00:00.16 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 6849.93\n",
            "//\n",
            "Query:       zinc_ribbon_16  [M=125]\n",
            "Accession:   PF17034.8\n",
            "Description: Zinc-ribbon like family\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    1.1e-21   76.8  19.8    2.6e-21   75.5  19.8    1.7  1  XP_004361154.1  uncharacterized protein DFA_05435 [Cavenderia\n",
            "    6.1e-06   25.9  20.5      0.013   15.1  20.5    3.7  1  XP_004362142.1  uncharacterized protein DFA_06441 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004361154.1  uncharacterized protein DFA_05435 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   75.5  19.8   4.4e-25   2.6e-21       1     123 [.     924    1038 ..     924    1040 .] 0.88\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 75.5 bits;  conditional E-value: 4.4e-25\n",
            "  zinc_ribbon_16    1 cnfckksitpfsakektnilqaavssaGgakfldksksslcprcskplpkcavcglslGtsdvtnkdskrksvvkevssskkrfekwlsfcls 93  \n",
            "                      c +c+ s+ +f +        + v ++  ak   + k   cp+c+++lp+c++c + +    v+  d k+ +  +e  +  + f++w+++c +\n",
            "  XP_004361154.1  924 CGYCQNSF-SFESVTSG----SMVGRN--AKINARAKVPCCPHCKQQLPRCCLCMIQMNCM-VPSMDFKQSNKQQEWTEGAESFDEWFTWCQT 1008\n",
            "                      88888888.77666653....333332..3666678899******************9998.******************************* PP\n",
            "\n",
            "  zinc_ribbon_16   94 cghgahashakewfsthsicpvadcnceck 123 \n",
            "                      c hg ha+h ++wf+ h  cpv+dcnc+c \n",
            "  XP_004361154.1 1009 CRHGGHATHLSDWFKDHIECPVTDCNCKCS 1038\n",
            "                      *****************************6 PP\n",
            "\n",
            ">> XP_004362142.1  uncharacterized protein DFA_06441 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.1  20.5   2.2e-06     0.013      44     123 ..    1413    1478 ..    1396    1480 .. 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.1 bits;  conditional E-value: 2.2e-06\n",
            "  zinc_ribbon_16   44 cskplpkcavcglslGtsdvtnkdskrksvvkev..ssskkrfekwlsfclscghgahashakewfsthsicpvadcnceck 123 \n",
            "                      c++ l + +vc               ++  v+ v  s  +  +    sfc+scghg h++h k+wf + s cp   c+c+c \n",
            "  XP_004362142.1 1413 CQRTLNNNCVC---------------PNCKVYAVkcSVCHISVRGLSSFCISCGHGGHTDHLKQWFGKRSDCPT-GCKCKCV 1478\n",
            "                      44444444444...............2233333300344444556789*************************6.6999996 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (125 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       947  (0.0780387); expected 242.7 (0.02)\n",
            "Passed bias filter:                      382  (0.0314792); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        29  (0.00238978); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.14u 0.00s 00:00:00.14 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 9966.51\n",
            "//\n",
            "Query:       zinc_ribbon_2  [M=23]\n",
            "Accession:   PF13240.9\n",
            "Description: zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    0.00021   20.4   3.6      0.001   18.2   3.6    2.1  2  XP_004359819.1  uncharacterized protein DFA_01854 [Cavenderia\n",
            "     0.0047   16.1   6.7     0.0096   15.1   6.7    1.5  1  XP_004357883.1  uncharacterized protein DFA_00167 [Cavenderia\n",
            "     0.0075   15.5   0.6      0.018   14.2   0.6    1.7  1  XP_004352445.1  WD40 repeat-containing protein [Cavenderia fa\n",
            "  ------ inclusion threshold ------\n",
            "      0.027   13.7   3.0      0.053   12.8   3.0    1.5  1  XP_004357569.1  uncharacterized protein DFA_02085 [Cavenderia\n",
            "      0.084   12.1   3.7          3    7.2   0.4    3.0  3  XP_004358406.1  ubiquitin [Cavenderia fasciculata]\n",
            "       0.11   11.8   4.7       0.57    9.5   4.7    2.1  1  XP_004359013.1  uncharacterized protein DFA_01038 [Cavenderia\n",
            "       0.37   10.1   1.6       0.82    9.0   1.6    1.6  1  XP_004351179.1  pleckstrin domain-containing protein [Cavende\n",
            "       0.68    9.2   7.4        2.1    7.7   7.4    1.8  1  XP_004361350.1  uncharacterized protein DFA_05632 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004359819.1  uncharacterized protein DFA_01854 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   11.0   0.7   0.00013      0.19      15      22 ..     273     280 ..     270     281 .. 0.73\n",
            "   2 !   18.2   3.6   6.9e-07     0.001       1      21 [.     273     295 ..     273     297 .] 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.0 bits;  conditional E-value: 0.00013\n",
            "   zinc_ribbon_2  15 FCpkCGtp 22 \n",
            "                     FC+kCGt+\n",
            "  XP_004359819.1 273 FCTKCGTK 280\n",
            "                     88888887 PP\n",
            "\n",
            "  == domain 2  score: 18.2 bits;  conditional E-value: 6.9e-07\n",
            "   zinc_ribbon_2   1 fCpnCGaei.pd.nakFCpkCGt 21 \n",
            "                     fC++CG+++     a FC+ CG \n",
            "  XP_004359819.1 273 FCTKCGTKVpVSsSASFCSSCGD 295\n",
            "                     9********7446*********5 PP\n",
            "\n",
            ">> XP_004357883.1  uncharacterized protein DFA_00167 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.1   6.7   6.3e-06    0.0096       2      19 ..       4      21 ..       3      21 .. 0.98\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.1 bits;  conditional E-value: 6.3e-06\n",
            "   zinc_ribbon_2  2 CpnCGaeipdnakFCpkC 19\n",
            "                    Cp+C +e +++ak+C+kC\n",
            "  XP_004357883.1  4 CPECQTEYEKEAKYCFKC 21\n",
            "                    ****************** PP\n",
            "\n",
            ">> XP_004352445.1  WD40 repeat-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   14.2   0.6   1.2e-05     0.018       5      21 ..     770     786 ..     769     787 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.2 bits;  conditional E-value: 1.2e-05\n",
            "   zinc_ribbon_2   5 CGaeipdnakFCpkCGt 21 \n",
            "                     CG+ i ++a  C kCGt\n",
            "  XP_004352445.1 770 CGKGILPGAIVCEKCGT 786\n",
            "                     ****************9 PP\n",
            "\n",
            ">> XP_004357569.1  uncharacterized protein DFA_02085 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.8   3.0   3.5e-05     0.053       2      23 .]     331     353 ..     330     353 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.8 bits;  conditional E-value: 3.5e-05\n",
            "   zinc_ribbon_2   2 CpnCGaeipd.nakFCpkCGtpl 23 \n",
            "                     C  C +e +   a+FC  CGt+l\n",
            "  XP_004357569.1 331 CGSCRTEAKNlAANFCASCGTKL 353\n",
            "                     999999998889*********97 PP\n",
            "\n",
            ">> XP_004358406.1  ubiquitin [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.1   0.0      0.83   1.3e+03       6      13 ..      53      60 ..      53      62 .. 0.85\n",
            "   2 ?    7.2   0.4     0.002         3      16      21 ..     128     133 ..     126     135 .. 0.58\n",
            "   3 ?    4.6   0.1     0.013        20      15      21 ..     147     153 ..     142     160 .. 0.71\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.1 bits;  conditional E-value: 0.83\n",
            "   zinc_ribbon_2  6 Gaeipdna 13\n",
            "                    G++++d+ \n",
            "  XP_004358406.1 53 GKQLEDGR 60\n",
            "                    9*****95 PP\n",
            "\n",
            "  == domain 2  score: 7.2 bits;  conditional E-value: 0.002\n",
            "   zinc_ribbon_2  16 CpkCGt 21 \n",
            "                     C++CG+\n",
            "  XP_004358406.1 128 CTNCGA 133\n",
            "                     555555 PP\n",
            "\n",
            "  == domain 3  score: 4.6 bits;  conditional E-value: 0.013\n",
            "   zinc_ribbon_2  15 FCpkCGt 21 \n",
            "                     +C kC t\n",
            "  XP_004358406.1 147 YCGKCHT 153\n",
            "                     6666665 PP\n",
            "\n",
            ">> XP_004359013.1  uncharacterized protein DFA_01038 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.5   4.7   0.00038      0.57       1      21 [.     227     248 ..     227     248 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.5 bits;  conditional E-value: 0.00038\n",
            "   zinc_ribbon_2   1 fCpnCGa.eipdnakFCpkCGt 21 \n",
            "                     +Cp CG+ ++  n ++C +CG+\n",
            "  XP_004359013.1 227 VCPSCGTdKNAGNTRYCIRCGA 248\n",
            "                     5*****934566*********7 PP\n",
            "\n",
            ">> XP_004351179.1  pleckstrin domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.0   1.6   0.00054      0.82       2      23 .]       5      26 ..       5      26 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.0 bits;  conditional E-value: 0.00054\n",
            "   zinc_ribbon_2  2 CpnCGaeipdnakFCpkCGtpl 23\n",
            "                    C  C + +  +a+FC  CG  l\n",
            "  XP_004351179.1  5 CGSCAKVNAVTARFCVSCGNSL 26\n",
            "                    999****************875 PP\n",
            "\n",
            ">> XP_004361350.1  uncharacterized protein DFA_05632 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.7   7.4    0.0014       2.1       2      23 .]     387     407 ..     386     407 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.7 bits;  conditional E-value: 0.0014\n",
            "   zinc_ribbon_2   2 CpnCGaeipdnakFCpkCGtpl 23 \n",
            "                     C+ CG  +    +FC++CGt+l\n",
            "  XP_004361350.1 387 CTTCGQ-VNGKTPFCFNCGTKL 407\n",
            "                     899998.67788********97 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (23 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       658  (0.0542233); expected 242.7 (0.02)\n",
            "Passed bias filter:                      378  (0.0311496); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        55  (0.00453234); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        16  (0.0013185); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               8  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 2506.49\n",
            "//\n",
            "Query:       zinc_ribbon_4  [M=36]\n",
            "Accession:   PF13717.9\n",
            "Description: zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    0.00019   20.7   2.7     0.0015   17.9   0.3    2.7  2  XP_004362339.1  uncharacterized protein DFA_02731 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "       0.46    9.9   4.8        6.5    6.2   0.1    2.8  2  XP_004354139.1  uncharacterized protein DFA_10231 [Cavenderia\n",
            "       0.83    9.1   6.1        7.4    6.0   0.4    3.1  3  XP_004359161.1  RNA polymerase III subunit [Cavenderia fascic\n",
            "          3    7.3   5.6       0.17   11.3   0.4    1.6  2  XP_004357883.1  uncharacterized protein DFA_00167 [Cavenderia\n",
            "        3.9    6.9   7.7        4.5    6.7   0.1    3.2  2  XP_004360282.1  FVYE domain-containing protein [Cavenderia fa\n",
            "        7.1    6.1   7.9        1.1    8.7   2.0    2.3  2  XP_004350066.1  uncharacterized protein DFA_11123 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004362339.1  uncharacterized protein DFA_02731 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.9   0.1     0.072   1.5e+02      25      32 ..     251     258 ..     245     261 .. 0.71\n",
            "   2 !   17.9   0.3   7.4e-07    0.0015       3      30 ..     291     318 ..     290     322 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.9 bits;  conditional E-value: 0.072\n",
            "   zinc_ribbon_4  25 kvrCskCg 32 \n",
            "                     ++rC+ C \n",
            "  XP_004362339.1 251 SIRCPTCV 258\n",
            "                     67888875 PP\n",
            "\n",
            "  == domain 2  score: 17.9 bits;  conditional E-value: 7.4e-07\n",
            "   zinc_ribbon_4   3 itCpsCkakyriddeklpakgvkvrCsk 30 \n",
            "                     itCp+C+++y +     + +g++v+Cs+\n",
            "  XP_004362339.1 291 ITCPHCNSRYVLGSQYAQYHGMSVKCSN 318\n",
            "                     9*************99**********97 PP\n",
            "\n",
            ">> XP_004354139.1  uncharacterized protein DFA_10231 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.2   0.1    0.0032       6.5      21      35 ..     446     460 ..     441     461 .. 0.88\n",
            "   2 ?    3.7   1.7      0.02        41      21      33 ..     474     486 ..     470     487 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.2 bits;  conditional E-value: 0.0032\n",
            "   zinc_ribbon_4  21 akgvkvrCskCghvF 35 \n",
            "                     ++++ ++C+ C+++F\n",
            "  XP_004354139.1 446 PNRQGIQCTTCQESF 460\n",
            "                     678889******999 PP\n",
            "\n",
            "  == domain 2  score: 3.7 bits;  conditional E-value: 0.02\n",
            "   zinc_ribbon_4  21 akgvkvrCskCgh 33 \n",
            "                     +k ++v+C+ C+ \n",
            "  XP_004354139.1 474 DKDGMVKCPVCQR 486\n",
            "                     56789******95 PP\n",
            "\n",
            ">> XP_004359161.1  RNA polymerase III subunit [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.0   0.1      0.14   2.7e+02       3      16 ..      14      27 ..       9      34 .. 0.69\n",
            "   2 ?    6.0   0.4    0.0037       7.4       2      12 ..      58      68 ..      58      69 .. 0.81\n",
            "   3 ?    3.7   0.1     0.019        39      27      36 .]      89      98 ..      82      98 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.0 bits;  conditional E-value: 0.14\n",
            "   zinc_ribbon_4  3 itCpsCkakyridd 16\n",
            "                    + C+ C   + i++\n",
            "  XP_004359161.1 14 FYCQTCPYFFGINN 27\n",
            "                    56777776666665 PP\n",
            "\n",
            "  == domain 2  score: 6.0 bits;  conditional E-value: 0.0037\n",
            "   zinc_ribbon_4  2 kitCpsCkaky 12\n",
            "                    +i+Cp Ck ++\n",
            "  XP_004359161.1 58 DIECPTCKVRR 68\n",
            "                    59*****9765 PP\n",
            "\n",
            "  == domain 3  score: 3.7 bits;  conditional E-value: 0.019\n",
            "   zinc_ribbon_4 27 rCskCghvFr 36\n",
            "                    +C+ C + +r\n",
            "  XP_004359161.1 89 KCTYCATRWR 98\n",
            "                    9******996 PP\n",
            "\n",
            ">> XP_004357883.1  uncharacterized protein DFA_00167 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.3   0.4   8.6e-05      0.17       3      13 ..       2      12 ..       1      14 [. 0.92\n",
            "   2 ?   -3.9   0.7       4.8   9.7e+03      28      35 ..      18      25 ..      18      26 .. 0.56\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.3 bits;  conditional E-value: 8.6e-05\n",
            "   zinc_ribbon_4  3 itCpsCkakyr 13\n",
            "                    itCp+C+++y+\n",
            "  XP_004357883.1  2 ITCPECQTEYE 12\n",
            "                    9*********6 PP\n",
            "\n",
            "  == domain 2  score: -3.9 bits;  conditional E-value: 4.8\n",
            "   zinc_ribbon_4 28 CskCghvF 35\n",
            "                    C kC+  F\n",
            "  XP_004357883.1 18 CFKCRIAF 25\n",
            "                    66776544 PP\n",
            "\n",
            ">> XP_004360282.1  FVYE domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.7   3.3      0.17   3.5e+02       3      14 ..      13      24 ..      12      38 .. 0.76\n",
            "   2 ?    6.7   0.1    0.0022       4.5       3      35 ..      37      69 ..      36      70 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.7 bits;  conditional E-value: 0.17\n",
            "   zinc_ribbon_4  3 itCpsCkakyri 14\n",
            "                    i+C  C++ ++i\n",
            "  XP_004360282.1 13 IECTGCQSPFSI 24\n",
            "                    889999998887 PP\n",
            "\n",
            "  == domain 2  score: 6.7 bits;  conditional E-value: 0.0022\n",
            "   zinc_ribbon_4  3 itCpsCkakyriddeklpakgvkvrCskCghvF 35\n",
            "                    i C  C+  y++ ++ l  +g+   C  C  +F\n",
            "  XP_004360282.1 37 IFCDPCSNHYTVLPAELGYSGAQRLCRVCHSSF 69\n",
            "                    67***************************9998 PP\n",
            "\n",
            ">> XP_004350066.1  uncharacterized protein DFA_11123 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.5   0.4      0.41   8.2e+02       4      10 ..     741     747 ..     740     748 .. 0.77\n",
            "   2 ?    8.7   2.0   0.00054       1.1       4      33 ..     780     806 ..     779     807 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.5 bits;  conditional E-value: 0.41\n",
            "   zinc_ribbon_4   4 tCpsCka 10 \n",
            "                     +Cp Ck+\n",
            "  XP_004350066.1 741 VCPTCKT 747\n",
            "                     5899987 PP\n",
            "\n",
            "  == domain 2  score: 8.7 bits;  conditional E-value: 0.00054\n",
            "   zinc_ribbon_4   4 tCpsCkakyriddek.lpakgvkvrCskCgh 33 \n",
            "                     +C  C+a+y+i+    +p    k +C+ C+h\n",
            "  XP_004350066.1 780 QCNFCQATYSISQPDiIP----KSTCPSCQH 806\n",
            "                     7***********954344....55******9 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (36 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       590  (0.0486197); expected 242.7 (0.02)\n",
            "Passed bias filter:                      343  (0.0282653); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        52  (0.00428513); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         9  (0.000741656); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               6  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 3711.91\n",
            "//\n",
            "Query:       zinc_ribbon_5  [M=37]\n",
            "Accession:   PF13719.9\n",
            "Description: zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.014   14.6   0.9      0.035   13.3   0.9    1.6  1  XP_004357678.1  uncharacterized protein DFA_02194 [Cavenderia\n",
            "       0.14   11.4   5.7      0.033   13.4   0.8    2.7  2  XP_004351843.1  uncharacterized protein DFA_09947 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004357678.1  uncharacterized protein DFA_02194 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.3   0.9   5.8e-06     0.035      14      32 ..     712     730 ..     708     732 .. 0.94\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.3 bits;  conditional E-value: 5.8e-06\n",
            "   zinc_ribbon_5  14 lvdsklKksgkkVrCakCa 32 \n",
            "                     l+++ +K  + kV CakC \n",
            "  XP_004357678.1 712 LDTDLIKIKQHKVVCAKCK 730\n",
            "                     89****************6 PP\n",
            "\n",
            ">> XP_004351843.1  uncharacterized protein DFA_09947 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.4   0.8   5.4e-06     0.033       3      32 ..     346     374 ..     344     375 .. 0.87\n",
            "   2 ?   -3.1   0.4      0.81   4.9e+03      27      34 ..     482     489 ..     477     489 .. 0.68\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.4 bits;  conditional E-value: 5.4e-06\n",
            "   zinc_ribbon_5   3 irCpdCdtkFklvdsklKksgkkVrCakCa 32 \n",
            "                     i Cp+C++  k++  ++  +++ + C kC \n",
            "  XP_004351843.1 346 IPCPQCESEIKVDRLEIDRNQQ-ILCVKCN 374\n",
            "                     78***************99996.6788885 PP\n",
            "\n",
            "  == domain 2  score: -3.1 bits;  conditional E-value: 0.81\n",
            "   zinc_ribbon_5  27 rCakCaev 34 \n",
            "                     +C  C++v\n",
            "  XP_004351843.1 482 KCKICGSV 489\n",
            "                     67777776 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (37 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       501  (0.0412855); expected 242.7 (0.02)\n",
            "Passed bias filter:                      324  (0.0266996); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        21  (0.00173053); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 4539.08\n",
            "//\n",
            "Query:       zinc_ribbon_6  [M=59]\n",
            "Accession:   PF14599.9\n",
            "Description: Zinc-ribbon\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    4.4e-21   74.1   4.5    4.4e-21   74.1   4.5    4.5  4  XP_004354944.1  uncharacterized protein DFA_09088 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004354944.1  uncharacterized protein DFA_09088 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -3.2   6.8      0.49     6e+03      27      56 ..      59      95 ..      50      98 .. 0.66\n",
            "   2 ?   -6.9   8.5         1   1.2e+04      31      54 ..     105     127 ..      97     132 .. 0.68\n",
            "   3 ?   -2.7   0.6      0.33     4e+03      29      35 ..     185     191 ..     163     195 .. 0.56\n",
            "   4 !   74.1   4.5   3.6e-25   4.4e-21       2      59 .]     197     255 ..     196     255 .. 0.97\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -3.2 bits;  conditional E-value: 0.49\n",
            "   zinc_ribbon_6 27 kvkilCnDCekksevkfhvlglk.......CaeCgSY 56\n",
            "                    + +i+C  C++ + v+ + +g         Ca+C+ Y\n",
            "  XP_004354944.1 59 TKEIMCMKCQTVQPVAEKCIGCDiqfaryfCAHCKFY 95\n",
            "                    6689999999887776666654322333337777766 PP\n",
            "\n",
            "  == domain 2  score: -6.9 bits;  conditional E-value: 1\n",
            "   zinc_ribbon_6  31 lCnDCekksevkfhvlglkCaeCg 54 \n",
            "                     +C DC+  ++v  +   + C++C+\n",
            "  XP_004354944.1 105 HCDDCK-ICRVGERDSFIHCKRCN 127\n",
            "                     699997.45555555566666665 PP\n",
            "\n",
            "  == domain 3  score: -2.7 bits;  conditional E-value: 0.33\n",
            "   zinc_ribbon_6  29 kilCnDC 35 \n",
            "                     +i C  C\n",
            "  XP_004354944.1 185 TIQCPMC 191\n",
            "                     3334333 PP\n",
            "\n",
            "  == domain 4  score: 74.1 bits;  conditional E-value: 3.6e-25\n",
            "   zinc_ribbon_6   2 dmeayfraldaeiaeqpmpeeyknkkvkilCnDCekkse.vkfhvlglkCaeCgSYNTr 59 \n",
            "                     ++++y+ ++da+ia+qpmp+ y+ +++++lC DC+kks+ ++ h+lg+kC++C+SYNT+\n",
            "  XP_004354944.1 197 NLTEYWLRIDAQIARQPMPRMYAGSTCSVLCCDCDKKSMdIALHFLGNKCQHCNSYNTT 255\n",
            "                     7899**********************************99******************5 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (59 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       489  (0.0402967); expected 242.7 (0.02)\n",
            "Passed bias filter:                      213  (0.0175525); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        23  (0.00189534); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 7270.44\n",
            "//\n",
            "Query:       zinc_ribbon_9  [M=35]\n",
            "Accession:   PF14369.9\n",
            "Description: zinc-ribbon\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    3.7e-09   36.1   3.4    3.7e-09   36.1   3.4    2.2  2  XP_004366460.1  uncharacterized protein DFA_04050 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "       0.01   15.5   1.9       0.01   15.5   1.9    2.3  2  XP_004357039.1  RING zinc finger-containing protein [Cavender\n",
            "      0.064   13.0   0.4       0.13   12.0   0.4    1.5  1  XP_004362968.1  putative GATA-binding transcription factor [C\n",
            "       0.23   11.2   7.0       0.89    9.3   2.8    2.9  2  XP_004362377.1  uncharacterized protein DFA_02769 [Cavenderia\n",
            "       0.84    9.4   5.6         15    5.4   0.2    3.7  4  XP_004359874.1  uncharacterized protein DFA_01912 [Cavenderia\n",
            "        1.5    8.6   6.6       0.14   11.9   0.3    2.7  3  XP_004359316.1  6-phosphogluconate dehydrogenase [Cavenderia \n",
            "        1.6    8.5   5.2         20    5.0   0.1    3.1  3  XP_004358905.1  RNA polymerase I subunit [Cavenderia fascicul\n",
            "        2.9    7.7   8.0         21    5.0   0.1    3.7  4  XP_004355784.1  heat shock protein DnaJ family protein [Caven\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004366460.1  uncharacterized protein DFA_04050 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   36.1   3.4   2.4e-12   3.7e-09       3      34 ..      87     116 ..      85     117 .. 0.91\n",
            "   2 ?   -2.1   0.2       2.2   3.3e+03      22      28 ..     429     434 ..     424     435 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 36.1 bits;  conditional E-value: 2.4e-12\n",
            "   zinc_ribbon_9   3 yWCHqCdrfVsirpdedsdvlCPeCgggFvEe 34 \n",
            "                     yWCH+C++ ++++ + d+ v CP C + F+Ee\n",
            "  XP_004366460.1  87 YWCHHCKKYIQVGNP-DELV-CPTCESEFIEE 116\n",
            "                     ************865.8889.**********9 PP\n",
            "\n",
            "  == domain 2  score: -2.1 bits;  conditional E-value: 2.2\n",
            "   zinc_ribbon_9  22 vlCPeCg 28 \n",
            "                     + CP C+\n",
            "  XP_004366460.1 429 S-CPVCR 434\n",
            "                     4.**996 PP\n",
            "\n",
            ">> XP_004357039.1  RING zinc finger-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.3   0.1       1.2   1.9e+03      16      28 ..      59      70 ..      58      71 .. 0.78\n",
            "   2 ?   15.5   1.9   6.8e-06      0.01       3      33 ..      85     110 ..      84     112 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.3 bits;  conditional E-value: 1.2\n",
            "   zinc_ribbon_9 16 pdedsdvlCPeCg 28\n",
            "                    ++e++d+ C  C \n",
            "  XP_004357039.1 59 EEEEEDK-CSICL 70\n",
            "                    4689999.99996 PP\n",
            "\n",
            "  == domain 2  score: 15.5 bits;  conditional E-value: 6.8e-06\n",
            "   zinc_ribbon_9   3 yWCHqCdrfVsirpdedsdvlCPeCgggFvE 33 \n",
            "                     ++C++C     +++++ s + CP C+++F+ \n",
            "  XP_004357039.1  85 HFCYHCI----LQWSDLSLK-CPLCKSQFLS 110\n",
            "                     9******....66779999*.********96 PP\n",
            "\n",
            ">> XP_004362968.1  putative GATA-binding transcription factor [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.0   0.4   8.7e-05      0.13       2      31 ..     666     698 ..     665     699 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.0 bits;  conditional E-value: 8.7e-05\n",
            "   zinc_ribbon_9   2 ryWCHqCdrfVsirpdedsd...vlCPeCgggF 31 \n",
            "                     +++CH C    +++++  +d   +lC+ Cg +F\n",
            "  XP_004362968.1 666 TLFCHTCGTKSTPEWRRGPDgpaTLCNACGLAF 698\n",
            "                     79*******9999999888886449****9887 PP\n",
            "\n",
            ">> XP_004362377.1  uncharacterized protein DFA_02769 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.7   0.1     0.071   1.1e+02       2      18 ..      61      76 ..      60      81 .. 0.73\n",
            "   2 ?    9.3   2.8   0.00059      0.89       3      31 ..     190     213 ..     188     216 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.7 bits;  conditional E-value: 0.071\n",
            "   zinc_ribbon_9  2 ryWCHqCdrfVsirpde 18\n",
            "                    +y+C++ d +++i+ ++\n",
            "  XP_004362377.1 61 SYYCYDQD-SIRITQSN 76\n",
            "                    8****955.77777665 PP\n",
            "\n",
            "  == domain 2  score: 9.3 bits;  conditional E-value: 0.00059\n",
            "   zinc_ribbon_9   3 yWCHqCdrfVsirpdedsdvlCPeCgggF 31 \n",
            "                      +C +C  ++s    ++++v CP+C+ +F\n",
            "  XP_004362377.1 190 IFCFECLHRWS----KTATV-CPNCKADF 213\n",
            "                     69999996666....67889.******99 PP\n",
            "\n",
            ">> XP_004359874.1  uncharacterized protein DFA_01912 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.1   0.1     0.025        39      12      28 ..      14      29 ..      11      30 .. 0.61\n",
            "   2 ?    5.4   0.2    0.0099        15      17      28 ..      39      51 ..      32      53 .. 0.69\n",
            "   3 ?   -0.4   0.0      0.64   9.8e+02       8      24 ..      97     113 ..      96     115 .. 0.87\n",
            "   4 ?   -2.5   0.0         3   4.6e+03       7      19 ..     192     204 ..     192     206 .. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.1 bits;  conditional E-value: 0.025\n",
            "   zinc_ribbon_9 12 VsirpdedsdvlCPeCg 28\n",
            "                    V+ +++ d+  +CP C+\n",
            "  XP_004359874.1 14 VASQSELDAL-TCPICR 29\n",
            "                    4444444444.5****6 PP\n",
            "\n",
            "  == domain 2  score: 5.4 bits;  conditional E-value: 0.0099\n",
            "   zinc_ribbon_9 17 dedsdv.lCPeCg 28\n",
            "                    ++++++ +CP+C+\n",
            "  XP_004359874.1 39 SSSETTgTCPQCR 51\n",
            "                    34444578****7 PP\n",
            "\n",
            "  == domain 3  score: -0.4 bits;  conditional E-value: 0.64\n",
            "   zinc_ribbon_9   8 CdrfVsirpdedsdvlC 24 \n",
            "                     C+  V++++++d   +C\n",
            "  XP_004359874.1  97 CQEIVTVEISKDHKLIC 113\n",
            "                     9999****999888778 PP\n",
            "\n",
            "  == domain 4  score: -2.5 bits;  conditional E-value: 3\n",
            "   zinc_ribbon_9   7 qCdrfVsirpded 19 \n",
            "                     +C+++++ + +++\n",
            "  XP_004359874.1 192 ECKQSIRETNKKQ 204\n",
            "                     6999999776655 PP\n",
            "\n",
            ">> XP_004359316.1  6-phosphogluconate dehydrogenase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -3.1   0.0       4.7   7.1e+03      18      28 ..      43      53 ..      41      54 .. 0.72\n",
            "   2 ?   11.9   0.3     9e-05      0.14       2      14 ..      85      97 ..      84     107 .. 0.89\n",
            "   3 ?   -3.8   0.5       7.6   1.2e+04      19      28 ..     389     398 ..     388     399 .. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -3.1 bits;  conditional E-value: 4.7\n",
            "   zinc_ribbon_9 18 edsdvlCPeCg 28\n",
            "                    e+++ +C  Cg\n",
            "  XP_004359316.1 43 ETENWICLKCG 53\n",
            "                    455657****8 PP\n",
            "\n",
            "  == domain 2  score: 11.9 bits;  conditional E-value: 9e-05\n",
            "   zinc_ribbon_9  2 ryWCHqCdrfVsi 14\n",
            "                    + WC++Cd+ V++\n",
            "  XP_004359316.1 85 SVWCYECDAYVTA 97\n",
            "                    67*********97 PP\n",
            "\n",
            "  == domain 3  score: -3.8 bits;  conditional E-value: 7.6\n",
            "   zinc_ribbon_9  19 dsdvlCPeCg 28 \n",
            "                     d++++C +C \n",
            "  XP_004359316.1 389 DEIPTCQDCS 398\n",
            "                     5677999995 PP\n",
            "\n",
            ">> XP_004358905.1  RNA polymerase I subunit [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.0   0.1     0.013        20      16      29 ..     155     169 ..     151     170 .. 0.72\n",
            "   2 ?   -1.8   0.1       1.7   2.6e+03      24      27 ..     183     186 ..     174     187 .. 0.78\n",
            "   3 ?    4.5   0.1     0.019        29      21      30 ..     226     234 ..     215     234 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.0 bits;  conditional E-value: 0.013\n",
            "   zinc_ribbon_9  16 pdedsdv.lCPeCgg 29 \n",
            "                      ++++ + +CP C++\n",
            "  XP_004358905.1 155 NNQQTKAnFCPTCNS 169\n",
            "                     455555569****97 PP\n",
            "\n",
            "  == domain 2  score: -1.8 bits;  conditional E-value: 1.7\n",
            "   zinc_ribbon_9  24 CPeC 27 \n",
            "                     CP C\n",
            "  XP_004358905.1 183 CPTC 186\n",
            "                     **99 PP\n",
            "\n",
            "  == domain 3  score: 4.5 bits;  conditional E-value: 0.019\n",
            "   zinc_ribbon_9  21 dvlCPeCggg 30 \n",
            "                     ++ CPeCg g\n",
            "  XP_004358905.1 226 EK-CPECGHG 234\n",
            "                     46.****986 PP\n",
            "\n",
            ">> XP_004355784.1  heat shock protein DnaJ family protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.6   0.0      0.32   4.8e+02      20      29 ..     157     166 ..     151     168 .. 0.73\n",
            "   2 ?   -2.4   0.2       2.7   4.1e+03      22      29 ..     176     183 ..     172     184 .. 0.81\n",
            "   3 ?    5.0   0.1     0.014        21      23      29 ..     203     209 ..     195     211 .. 0.76\n",
            "   4 ?    4.5   0.1     0.018        28      18      29 ..     215     225 ..     212     228 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.6 bits;  conditional E-value: 0.32\n",
            "   zinc_ribbon_9  20 sdvlCPeCgg 29 \n",
            "                     +d++C  C+g\n",
            "  XP_004355784.1 157 RDIVCTTCNG 166\n",
            "                     4545****98 PP\n",
            "\n",
            "  == domain 2  score: -2.4 bits;  conditional E-value: 2.7\n",
            "   zinc_ribbon_9  22 vlCPeCgg 29 \n",
            "                     ++C  C+g\n",
            "  XP_004355784.1 176 SICTKCKG 183\n",
            "                     57****98 PP\n",
            "\n",
            "  == domain 3  score: 5.0 bits;  conditional E-value: 0.014\n",
            "   zinc_ribbon_9  23 lCPeCgg 29 \n",
            "                     +CP+C+g\n",
            "  XP_004355784.1 203 ACPQCHG 209\n",
            "                     5****98 PP\n",
            "\n",
            "  == domain 4  score: 4.5 bits;  conditional E-value: 0.018\n",
            "   zinc_ribbon_9  18 edsdvlCPeCgg 29 \n",
            "                     +++d+ CP C+g\n",
            "  XP_004355784.1 215 KEEDK-CPKCKG 225\n",
            "                     67999.****97 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (35 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       682  (0.0562011); expected 242.7 (0.02)\n",
            "Passed bias filter:                      423  (0.0348578); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        57  (0.00469716); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        12  (0.000988875); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               8  [number of targets reported over threshold]\n",
            "# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 3189.39\n",
            "//\n",
            "Query:       ZinT  [M=181]\n",
            "Accession:   PF09223.14\n",
            "Description: ZinT (YodA) periplasmic lipocalin-like zinc-recruitment\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (181 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       588  (0.0484549); expected 242.7 (0.02)\n",
            "Passed bias filter:                      185  (0.0152452); expected 242.7 (0.02)\n",
            "Passed Vit filter:                         8  (0.00065925); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 17167.73\n",
            "//\n",
            "Query:       Zip  [M=333]\n",
            "Accession:   PF02535.25\n",
            "Description: ZIP Zinc transporter\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    2.5e-52  177.6   3.5    1.6e-51  174.9   3.5    1.9  1  XP_004360230.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "    6.1e-50  169.8   4.1    7.8e-50  169.4   4.1    1.1  1  XP_004360276.1  uncharacterized protein DFA_04547 [Cavenderia\n",
            "      9e-50  169.2  11.7    1.1e-49  169.0  11.7    1.0  1  XP_004367068.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "    4.9e-48  163.5  14.9    4.9e-48  163.5  14.9    1.3  1  XP_004367069.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "    3.3e-37  127.9   6.0    3.3e-37  127.9   6.0    2.9  1  XP_004367397.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "    6.6e-35  120.3   9.2      1e-33  116.4   9.2    2.2  1  XP_004354513.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "    0.00051   18.8   1.5    0.00062   18.5   1.5    1.1  1  XP_004350816.1  cytochrome c oxidase assembly protein [Cavend\n",
            "     0.0012   17.6   0.2     0.0012   17.6   0.2    2.8  1  XP_004357740.1  acetylglutamate kinase [Cavenderia fasciculat\n",
            "     0.0016   17.2   0.0     0.0016   17.2   0.0    1.2  1  XP_004361557.1  uncharacterized protein DFA_05840 [Cavenderia\n",
            "     0.0017   17.1   0.2     0.0017   17.1   0.2    3.0  2  XP_004361315.1  uncharacterized protein DFA_05597 [Cavenderia\n",
            "     0.0054   15.4   0.6     0.0089   14.7   0.6    1.3  1  XP_004358720.1  uncharacterized protein DFA_00735 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "       0.01   14.5   0.4      0.016   13.9   0.4    1.3  1  XP_004354180.1  uncharacterized protein DFA_10273 [Cavenderia\n",
            "      0.016   13.9   1.9      0.019   13.6   1.9    1.1  1  XP_004358408.1  uncharacterized protein DFA_00419 [Cavenderia\n",
            "      0.019   13.7   1.7       0.02   13.6   1.7    1.2  1  XP_004350550.1  RhoGAP domain-containing protein [Cavenderia \n",
            "      0.022   13.4   3.2      0.029   13.1   2.4    1.7  1  XP_004357612.1  uncharacterized protein DFA_02128 [Cavenderia\n",
            "      0.023   13.4   0.6      0.034   12.8   0.6    1.2  1  XP_004361841.1  uncharacterized protein DFA_06128 [Cavenderia\n",
            "      0.026   13.2   1.6      0.039   12.6   1.6    1.3  1  XP_004352581.1  uncharacterized protein DFA_09153 [Cavenderia\n",
            "      0.028   13.1   4.3      0.063   11.9   4.3    1.7  1  XP_004357413.1  uncharacterized protein DFA_02691 [Cavenderia\n",
            "      0.033   12.8   0.5      0.045   12.4   0.5    1.2  1  XP_004356110.1  cyclin-like F-box containing protein [Cavende\n",
            "      0.036   12.7   1.5        4.3    5.9   1.5    2.9  1  XP_004363105.1  uncharacterized protein DFA_03502 [Cavenderia\n",
            "      0.037   12.7   0.0      0.083   11.5   0.0    1.6  1  XP_004354958.1  glycosyltransferase [Cavenderia fasciculata]\n",
            "      0.049   12.3   2.3       0.65    8.6   0.1    2.2  2  XP_004352197.1  uncharacterized protein DFA_09541 [Cavenderia\n",
            "      0.055   12.1   4.0      0.064   11.9   4.0    1.2  1  XP_004366334.1  uncharacterized protein DFA_03924 [Cavenderia\n",
            "      0.059   12.0   0.4      0.089   11.4   0.4    1.2  1  XP_004362039.1  SAP DNA-binding domain-containing protein [Ca\n",
            "       0.06   12.0   6.1       0.69    8.5   6.1    2.5  2  XP_004357421.1  phosphatidylinositol glycan [Cavenderia fasci\n",
            "      0.067   11.9   0.8        0.1   11.2   0.8    1.2  1  XP_004362965.1  uncharacterized protein DFA_03360 [Cavenderia\n",
            "      0.087   11.5   1.5       0.12   11.0   1.5    1.2  1  XP_004362041.1  SAP DNA-binding domain-containing protein [Ca\n",
            "       0.11   11.1   2.5       0.16   10.6   2.5    1.3  1  XP_004350243.1  uncharacterized protein DFA_11300 [Cavenderia\n",
            "       0.12   11.0   6.8       0.12   11.0   6.8    1.3  1  XP_004356149.1  uncharacterized protein DFA_08661 [Cavenderia\n",
            "       0.16   10.6   6.0       0.23   10.1   6.0    1.2  1  XP_004351373.1  Phytochelatins synthase [Cavenderia fascicula\n",
            "       0.16   10.6   0.0       0.16   10.6   0.0    2.7  2  XP_004355729.1  uncharacterized protein DFA_08235 [Cavenderia\n",
            "       0.22   10.1   2.4       0.29    9.7   2.4    1.2  1  XP_004361946.1  uncharacterized protein DFA_06234 [Cavenderia\n",
            "       0.24   10.0   2.8        3.7    6.1   0.0    2.2  2  XP_004363094.1  uncharacterized protein DFA_03491 [Cavenderia\n",
            "       0.26    9.9   0.0       0.26    9.9   0.0    1.9  1  XP_004356331.1  putative transmembrane protein [Cavenderia fa\n",
            "        0.3    9.7   0.0        0.3    9.7   0.0    2.7  1  XP_004356130.1  putative RNA helicase [Cavenderia fasciculata\n",
            "       0.31    9.6   1.9       0.41    9.3   1.9    1.1  1  XP_004358095.1  uncharacterized protein DFA_06849, partial [C\n",
            "       0.32    9.6   2.7       0.33    9.6   2.7    1.1  1  XP_004362018.1  uncharacterized protein DFA_06313 [Cavenderia\n",
            "       0.36    9.5   4.8       0.76    8.4   4.8    1.5  1  XP_004354628.1  uncharacterized protein DFA_09274 [Cavenderia\n",
            "        0.4    9.3   6.4       0.72    8.5   6.4    1.4  1  XP_004351053.1  uncharacterized protein DFA_12114 [Cavenderia\n",
            "        0.4    9.3   7.9       0.69    8.5   7.9    1.3  1  XP_004350295.1  uncharacterized protein DFA_11352 [Cavenderia\n",
            "       0.42    9.2   4.5       0.45    9.1   4.5    1.1  1  XP_004352044.1  DNA-3-methyladenine glycosylase I [Cavenderia\n",
            "       0.45    9.1   1.5       0.66    8.6   1.5    1.2  1  XP_004351325.1  uncharacterized protein DFA_10682 [Cavenderia\n",
            "       0.45    9.1   1.5       0.66    8.6   1.5    1.2  1  XP_004358641.1  uncharacterized protein DFA_00656 [Cavenderia\n",
            "       0.47    9.1   7.9       0.25    9.9   2.7    2.3  2  XP_004355377.1  uncharacterized protein DFA_07884 [Cavenderia\n",
            "       0.58    8.8  10.9       0.33    9.6   4.9    2.3  2  XP_004359704.1  uncharacterized protein DFA_01739 [Cavenderia\n",
            "       0.71    8.5   9.0       0.89    8.2   9.0    1.1  1  XP_004360802.1  uncharacterized protein DFA_05081 [Cavenderia\n",
            "       0.76    8.4   2.6       0.83    8.3   2.6    1.1  1  XP_004353824.1  uncharacterized protein DFA_12187 [Cavenderia\n",
            "       0.77    8.4   8.2        1.1    7.8   8.2    1.4  1  XP_004359443.1  MADS-box transcription factor [Cavenderia fas\n",
            "       0.79    8.3   2.9       0.75    8.4   2.9    1.1  1  XP_004350327.1  uncharacterized protein DFA_11384 [Cavenderia\n",
            "        0.8    8.3   5.2        1.2    7.8   5.2    1.2  1  XP_004362344.1  UAS domain-containing protein [Cavenderia fas\n",
            "       0.81    8.3  11.6        1.6    7.3  11.6    1.6  1  XP_004354593.1  uncharacterized protein DFA_09239 [Cavenderia\n",
            "          1    8.0   6.4        1.4    7.5   6.4    1.3  1  XP_004363124.1  uncharacterized protein DFA_03521 [Cavenderia\n",
            "        1.1    7.9   4.2        1.8    7.2   4.2    1.2  1  XP_004359491.1  WD40-like domain-containing protein [Cavender\n",
            "        1.1    7.9   3.2        1.4    7.5   3.2    1.2  1  XP_004359733.1  SET domain-containing protein [Cavenderia fas\n",
            "        1.4    7.5   4.4        2.2    6.9   4.4    1.2  1  XP_004366718.1  RhoGAP domain-containing protein [Cavenderia \n",
            "        1.4    7.5   3.6        1.9    7.1   3.6    1.2  1  XP_004366832.1  uncharacterized protein DFA_02447 [Cavenderia\n",
            "        1.4    7.5   3.5        1.6    7.3   3.5    1.1  1  XP_004358926.1  uncharacterized protein DFA_00951 [Cavenderia\n",
            "        1.5    7.4  17.3      0.061   12.0   8.1    2.3  2  XP_004354601.1  uncharacterized protein DFA_09247 [Cavenderia\n",
            "        1.6    7.3   5.4        3.1    6.4   5.4    1.5  1  XP_004367084.1  uncharacterized protein DFA_07218 [Cavenderia\n",
            "        1.6    7.3  10.7          2    7.0  10.7    1.2  1  XP_004366934.1  uncharacterized protein DFA_07061 [Cavenderia\n",
            "        1.6    7.3   6.2        2.9    6.5   6.2    1.3  1  XP_004359218.1  leucine-rich repeat-containing protein [Caven\n",
            "        1.7    7.2   5.2        4.3    5.9   5.2    1.6  1  XP_004351983.1  uncharacterized protein DFA_10097 [Cavenderia\n",
            "        2.1    6.9   5.3        3.1    6.4   5.3    1.2  1  XP_004350793.1  uncharacterized protein DFA_11848 [Cavenderia\n",
            "        2.3    6.8  11.5        4.1    6.0  11.5    1.4  1  XP_004350944.1  uncharacterized protein DFA_12005 [Cavenderia\n",
            "        2.5    6.7  18.1        3.3    6.3   4.4    2.4  2  XP_004358866.1  putative RNA helicase [Cavenderia fasciculata\n",
            "        2.6    6.6  26.9       0.23   10.1   3.9    2.6  2  XP_004356887.1  cyclin [Cavenderia fasciculata]\n",
            "        2.9    6.5   5.1        4.7    5.8   5.1    1.3  1  XP_004359196.1  extracellular signal-regulated protein kinase\n",
            "        3.2    6.3   4.9        3.8    6.1   4.9    1.3  1  XP_004359778.1  N2227-like domain-containing protein [Cavende\n",
            "        3.9    6.0   6.5        5.6    5.5   6.5    1.4  1  XP_004360810.1  protein kinase 3 [Cavenderia fasciculata]\n",
            "          4    6.0   7.2        5.8    5.5   7.2    1.1  1  XP_004366559.1  uncharacterized protein DFA_04150 [Cavenderia\n",
            "          4    6.0   6.6        5.5    5.6   6.6    1.2  1  XP_004355277.1  UPF0551 family protein [Cavenderia fasciculat\n",
            "        4.7    5.8   5.5         11    4.5   5.5    1.6  1  XP_004350733.1  kinetochore protein Ndc80 [Cavenderia fascicu\n",
            "        4.7    5.8  10.5        1.6    7.3   5.1    2.2  2  XP_004355651.1  uncharacterized protein DFA_08151 [Cavenderia\n",
            "        5.2    5.6  12.3         10    4.6  12.3    1.4  1  XP_004358977.1  pleckstrin domain-containing protein [Cavende\n",
            "        5.8    5.5   6.4         14    4.2   6.4    1.6  1  XP_004367225.1  putative mediator complex subunit 12 [Cavende\n",
            "        5.9    5.4   8.8       0.17   10.5   1.7    1.9  1  XP_004356343.1  uncharacterized protein DFA_08860 [Cavenderia\n",
            "        6.3    5.4   8.5         11    4.6   8.5    1.5  1  XP_004350761.1  transcription initiation factor TFIID subunit\n",
            "        6.4    5.3  10.3       0.49    9.0   3.8    2.2  2  XP_004350810.1  HEAT repeat-containing protein [Cavenderia fa\n",
            "        6.5    5.3   6.7          9    4.8   6.7    1.1  1  XP_004357732.1  NPR2 family protein [Cavenderia fasciculata]\n",
            "        6.9    5.2   7.0        8.4    4.9   7.0    1.3  1  XP_004355597.1  NUC153 domain-containing protein [Cavenderia \n",
            "        7.4    5.1   5.5        7.2    5.2   5.5    1.0  1  XP_004366330.1  uncharacterized protein DFA_03920 [Cavenderia\n",
            "        8.7    4.9   9.1         10    4.7   9.1    1.2  1  XP_004359062.1  uncharacterized protein DFA_01087 [Cavenderia\n",
            "          9    4.9   7.3         14    4.2   7.3    1.5  1  XP_004362923.1  uncharacterized protein DFA_03318 [Cavenderia\n",
            "        9.3    4.8   7.4        1.6    7.3   4.1    1.4  1  XP_004350745.1  WD40 repeat-containing protein [Cavenderia fa\n",
            "        9.4    4.8   5.9         13    4.3   5.9    1.2  1  XP_004366514.1  uncharacterized protein DFA_04104 [Cavenderia\n",
            "        9.6    4.8   6.9        2.5    6.7   1.7    2.1  2  XP_004367319.1  uncharacterized protein DFA_07460 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004360230.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  174.9   3.5   1.2e-53   1.6e-51      32     333 .]       6     379 ..       1     379 [. 0.77\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 174.9 bits;  conditional E-value: 1.2e-53\n",
            "             Zip  32 lsklllslllafavGvllgtaflhLlPealealeesqleeehthplagllvllGflllllveklltmivek.....ksshkektpkkassssses 121\n",
            "                      s+++lsl   f++G+++g a+lhL+ +++ +l++  +      p a l+++ Gfl+++++e +l+++ e+     +s      + + +++ ++ \n",
            "  XP_004360230.1   6 NSNKWLSLCNSFSGGIFFGAALLHLFVDSISQLQNVISY-----PIAPLCLCAGFLITFFLELFLHIYLERhpkllNS------NINNNNNNNNT 89 \n",
            "                     6899*******************************8753.....***************************6655522......22222222222 PP\n",
            "\n",
            "             Zip 122 aakksvsndkveeseek.................................................................edhvaeadtdads 151\n",
            "                     ++++++  ++++   ++                                                                              \n",
            "  XP_004360230.1  90 SSSSTILHHQHQHINNNnqstlnfgntnsnttfnflnsptnnrrsmildrpstpnivntlvftpppnhrtnvdndnectpmlS------------ 172\n",
            "                     22222222222222222233444566666666677777777777778888888888888887777665442222222222221............ PP\n",
            "\n",
            "             Zip 152 keelrngkqeksesekeqas...................................eeedkeklaslkvraiilllglilHsfidGLAlGvsssss 211\n",
            "                        ++n++ + ++++ke +s                                   ++++++ + +  +  +il++gl++Hs+++GLA+G++s++ \n",
            "  XP_004360230.1 173 ---DHNNNNNNDDDDKEDESsdsgdyipssnhssiyfqdalhlsstsiqqqqqevTKQETTLTNQNYLLPFILVFGLSVHSLFEGLAFGLQSTVP 264\n",
            "                     ...2222222222222222244577777777788899***********999999666666666666777************************** PP\n",
            "\n",
            "             Zip 212 tgislfiAvllHelpeelglgaiLlkaglsrkkavllnllfaltaplGiliGilvrsisessgaeivsgillalsaGtflYvalvellphEllne 306\n",
            "                     +++ l++Av+ H+l++ ++lg+  +    s  k +l+ ++f++++p+G+ iGi+   ++ s+  ++v +il++l++GtflY++lve++p+E++++\n",
            "  XP_004360230.1 265 KILDLLVAVFSHKLLASFALGVSTIIYSRSLVKMILILFIFSISSPIGASIGIV--LVDYSQIGNVVPPILQGLASGTFLYISLVEIIPKEIKSD 357\n",
            "                     ***********************99999999***********************..888999********************************* PP\n",
            "\n",
            "             Zip 307 kgssslkqsllqllglllGfalmllla 333\n",
            "                     +     ++ + + l+lllG  lm+++a\n",
            "  XP_004360230.1 358 S-----ENVFIKCLLLLLGWSLMAVIA 379\n",
            "                     9.....69999**************97 PP\n",
            "\n",
            ">> XP_004360276.1  uncharacterized protein DFA_04547 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  169.4   4.1   5.5e-52   7.8e-50       6     332 ..       8     396 ..       3     397 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 169.4 bits;  conditional E-value: 5.5e-52\n",
            "             Zip   6 laiiaililsllgllgpllvpkiskl...lsklllslllafavGvllgtaflhLlPealealeesqleeehthplagllvllGflllllvekllt 97 \n",
            "                     ++i++ +++s +++l+p+ ++k+ k+    +++++ l  +f +G++l+ af+hLl e+ e+l+e  ++     pla++++ +Gf++++++ek+ +\n",
            "  XP_004360276.1   8 VLILLSILISGFPGLIPYALDKYVKIkhsKKEYVIGLGNCFVAGIFLSIAFVHLLNESSEKLSEVNHHI----PLAFIMCPVGFVVVFMIEKIFN 98 \n",
            "                     45666677888899999999999999999999********************************97655....********************** PP\n",
            "\n",
            "             Zip  98 mivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas..................... 171\n",
            "                      + ++  +h++   + ++++++++++ k+ +n+ +ees+   +++++  +    +++  ++++++++ +++ ++                     \n",
            "  XP_004360276.1  99 CFEGH--NHGHSHGGSKKNKKNKNKKNKKNNNNLLEESTTIYKQQSGSSGYIILESSNLDEQKQPKSVSPKFPHspsygtfqditidtsrppspf 191\n",
            "                     88777..4444444444444444444444455555556665555555555555555555555555555555444899999*************** PP\n",
            "\n",
            "             Zip 172 ................................................eeedkeklaslkvraiilllglilHsfidGLAlGvssssstgislfi 218\n",
            "                                                                     ++ d++ ++      iil++ l++Hs+i G+ lGv s+ + + +lfi\n",
            "  XP_004360276.1 192 pqspnlsssssssiptttititqdietptgdehlpmikediiitlendKSTDSSTTSLKGPTPIILVFILSIHSIISGFTLGVESNYDIIYPLFI 286\n",
            "                     *******************************************99765444444444455669******************************** PP\n",
            "\n",
            "             Zip 219 AvllHelpeelglgaiLlkaglsrkkavllnllfaltaplGiliGilvrsisessgaeivsgillalsaGtflYvalvellphEllnekgssslk 313\n",
            "                      ++ H+ +e+++lg+ L++++ s    + l+ l++lt plGi++G++   +s s  +  +++++la+s+Gtf+Y+al+++l  E+ ++  ++s  \n",
            "  XP_004360276.1 287 GIISHKWLEAMSLGVSLVRNKSSFYETLKLVSLYSLTEPLGIVLGVA---ASVSVASTTATSLVLAFSSGTFIYIALMDILVVEFNESI-KQSRL 377\n",
            "                     ***********************************************...9******************************99999999.57888 PP\n",
            "\n",
            "             Zip 314 qsllqllglllGfalmlll 332\n",
            "                     ++l +l++ ++Gf+ m++l\n",
            "  XP_004360276.1 378 TKLVKLAFCIIGFLGMSAL 396\n",
            "                     899*************987 PP\n",
            "\n",
            ">> XP_004367068.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  169.0  11.7   7.5e-52   1.1e-49       2     333 .]      56     389 ..      55     389 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 169.0 bits;  conditional E-value: 7.5e-52\n",
            "             Zip   2 llkilaiiaililsllgllgpllvpkiskllsklllslllafavGvllgtaflhLlPealealeesqleee.hth......plagllvllGflll 89 \n",
            "                      +ki++i+ i++l+ll+ ++p+++ + +   +  lls+ ++fa+Gv+l+ +f h+lP a e++ +    ++ +++      p+a  ++++ +l+l\n",
            "  XP_004367068.1  56 NAKIGLIVGIFFLTLLSSYIPFILGRAKVKGFITLLSIGTCFAGGVILAGGFNHILPGAEESFTSYF--DQvAPEnkyrefPFAATIAIFTLLVL 148\n",
            "                     589**********************99999*******************************999853..222444667889************** PP\n",
            "\n",
            "             Zip  90 llveklltmivek..ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas...eeedkekl 179\n",
            "                       ++kl+     +  k++++ + +++a++++ + +++   ++ ++ +++ + d++ +++ ++ ++    +  q++++++++ ++   +e+ + k \n",
            "  XP_004367068.1 149 VAIDKLIIEGGFQgeKGHNHMNLSSHADNQHHHTNTHA--PDLEFGQESSS-DEEDSHGATPGNPDGALAPPQHSHGHAHSGKHdelHEKGNGKS 240\n",
            "                     *****988643335577777777777777777777766..44454444443.5556788888888888899999999999988743333333333 PP\n",
            "\n",
            "             Zip 180 aslkvr.....aiilllglilHsfidGLAlGvssssstgislfiAvllHelpeelglgaiLlkaglsrkkavllnllfaltaplGiliGilvrsi 269\n",
            "                     +    +     a+++l++l++Hs+ dGL lG+  s++ +  l++Avl H+++ +++lg++++ a++s  +  l+  + a+++plGi+iG++v+s+\n",
            "  XP_004367068.1 241 H----VantgqAWLFLVALSIHSILDGLGLGAETSKDGFYGLLVAVLAHKMLDGFALGVPIYFANFSTLQTALSLAFCAAMTPLGIGIGMAVTSV 331\n",
            "                     3....235568***********************************************************************************9 PP\n",
            "\n",
            "             Zip 270 sessgaeivsgillalsaGtflYvalvellphEllnekgssslkqsllqllglllGfalmllla 333\n",
            "                      ++s   +++gi+l++++G f Y++l el+p  l+++          l+l++++lG a ++++a\n",
            "  XP_004367068.1 332 YNGSSGHLAEGIILGVTCGSFFYISLIELIPSGLCQPG------WLRLKLAMVFLGWACLSVIA 389\n",
            "                     999****************************7777777......57799**********99996 PP\n",
            "\n",
            ">> XP_004367069.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  163.5  14.9   3.4e-50   4.9e-48       2     333 .]      23     348 ..      22     348 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 163.5 bits;  conditional E-value: 3.4e-50\n",
            "             Zip   2 llkilaiiaililsllgllgpllvpkiskllsklllslllafavGvllgtaflhLlPealealeesqleeehth.........plagllvllGfl 87 \n",
            "                      +k+++i+ i++l+l + ++p+++ + +      +l+++++fa+Gv+l+ +f h+  +a ea+ e   + +            p+   ++++ +l\n",
            "  XP_004367069.1  23 NAKVGLIVGIFFLTLGSSYIPFIMGRAKVKNLLTILAVFTCFAGGVILAGGFSHIPHHAEEAFAEYFADID--PegqneylhfPFSMTIAIFTLL 115\n",
            "                     57999********************999999***********************99999888877433222..0233556557888888888888 PP\n",
            "\n",
            "             Zip  88 llllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklasl 182\n",
            "                     +l  ++kl      + ++ ++ ++  +++++++++++++ +++++ e e   +    +++dad+++++ +g+++k+++ +     ++ + k+++ \n",
            "  XP_004367069.1 116 VLIAIDKLFIEGGFQGEKGHNHMNLSSHADTQHKHTSNHTPDVEMAEVETPGN---PDGGDADHGHSHGHGHSGKHDELH-----DDGNGKTHQA 202\n",
            "                     88888887665333323333333333444444455555555555555555433...455555555555555554444444.....4456777788 PP\n",
            "\n",
            "             Zip 183 kvr.aiilllglilHsfidGLAlGvssssstgislfiAvllHelpeelglgaiLlkaglsrkkavllnllfaltaplGiliGilvrsisessgae 276\n",
            "                     +v  a+++l++l++Hs+ dGL lG+  s++ +  l++Avl H+l+ +++lg++++ a++s+ +  ++ ++ a+++plGi+iG++v+s+ e+ g  \n",
            "  XP_004367069.1 203 NVGqAWLFLVALSIHSILDGLGLGAETSKDGFYGLLVAVLAHKLLDGFALGVPMYFAKFSVLQTAMALVFCAAMTPLGIGIGMAVTSVYEGAGGH 297\n",
            "                     8878***********************************************************************************9999**** PP\n",
            "\n",
            "             Zip 277 ivsgillalsaGtflYvalvellphEllnekgssslkqsllqllglllGfalmllla 333\n",
            "                     +++gi+l++++G f+Y++l el+p  l+++          l+l++++lG al++++a\n",
            "  XP_004367069.1 298 LAEGIILSVTCGSFIYISLIELIPSGLCQPG------WLRLKLAMVFLGWALLAIIA 348\n",
            "                     ************************7777777......57799************997 PP\n",
            "\n",
            ">> XP_004367397.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  127.9   6.0   2.4e-39   3.3e-37       4     324 ..     228     527 ..     222     536 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 127.9 bits;  conditional E-value: 2.4e-39\n",
            "             Zip   4 kilaiiaililsllgllgpllvpkiskllsklllslllafavGvllgtaflhLlPealealeesqleeehthplagllvllGflllllveklltm 98 \n",
            "                     ++++ii+ l + + gl+++++      ++s ++l ++l f++Gv+++ +f++LlPe+ ++++            a ++++ G++++ lv kl+  \n",
            "  XP_004367397.1 228 MLMCIISGLSTAIGGLFILFMK-----SPSYKFLGYMLGFSSGVMIYISFMDLLPESAAEIGFLP---------ANGFFFAGIIFFALVLKLVPE 308\n",
            "                     4444454444444444444444.....48999***************************999865.........778999**************9 PP\n",
            "\n",
            "             Zip  99 ivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvraiilllgl 193\n",
            "                     + e+   ++ ++++++++ss es+   ++ n+++++s+  ++++  ++t ++s+ + +  kq+++++++++++e++++ ++a lk  + +  lg+\n",
            "  XP_004367397.1 309 ADEH---EHGPSEETHEHSSVESSNTTTHTNSSSTTSTVVKRKNGADKTPPQSPPSSATTKQQQQSKQTKKPKEKKKQMSDAYLKNLGVVTALGI 400\n",
            "                     8766...7777777777777778888888888888888888888899999999999999999999999999999999999999************ PP\n",
            "\n",
            "             Zip 194 ilHsfidGLAlG..vssssstgislfiAvllHelpeelglgaiLlkaglsrkkavllnllfaltaplGili.Gilvrsisessgaeivsgillal 285\n",
            "                     +lH+f++G+A+   + +s + g++l++A+  H++pe++++ a+++ a  s++ a+ ++ll++l  p G++i G++       ++  +v+++ la \n",
            "  XP_004367397.1 401 SLHNFPEGVAVYlsCLKSMQVGLPLMLAIAAHNIPEGMAVAAPIYTATGSKWVAFKYCLLSGLCEPFGAIIfGFF---FRDYMTPYLVQAM-LAA 491\n",
            "                     **********9988******************************************************7653999...5556666666665.566 PP\n",
            "\n",
            "             Zip 286 saGtflYvalvellphEllnekgssslkqsllqllglll 324\n",
            "                      aG+++ +a  el+p+ l++ +     + ++++++g+++\n",
            "  XP_004367397.1 492 VAGIMIFMAIKELIPATLKYID---VNGATISNFVGMFV 527\n",
            "                     79*************9999888...45677777777775 PP\n",
            "\n",
            ">> XP_004354513.1  zinc/iron permease [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  116.4   9.2   7.2e-36     1e-33       4     328 ..       7     417 ..       4     419 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 116.4 bits;  conditional E-value: 7.2e-36\n",
            "             Zip   4 kilaiiaililsllgllgpllvpkiskl................lsklllslllafavGvllgtaflhLlPealealeesqleeehth....... 75 \n",
            "                      ila ia +++s l+l++ +l ++++++                 s++ +++lla+++G l++ a l LlPea++   +s + +           \n",
            "  XP_004354513.1   7 SILAGIAPILSSSLPLFFFYLQRRQQQNkkdsssssnsnhstsvGSSNQYNILLALSAGLLFSVATLELLPEAIGLAIQSTSTQPSSSppppqts 101\n",
            "                     58899999999999999999999999999************9999999**************************976666655533347888899 PP\n",
            "\n",
            "             Zip  76 ..................plagllvllGflllllveklltmivek...............ksshkektpkkasssssesaakksvsndkveesee 137\n",
            "                                         +   v  Gf+ll +++ l +   ++               +           ++++s   ++++++ ++ ++++ \n",
            "  XP_004354513.1 102 etkhnhddddqgdnesslRIPMYGVASGFILLIILDLLFSDPNGGvhghshsnpiisskdN-----------DHHHSVDIKHHHHHHNNNSYNST 185\n",
            "                     999999***********9777778899***********99766667766444444444432...........44444333333333333333333 PP\n",
            "\n",
            "             Zip 138 k.............................................edhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvrai 187\n",
            "                     +                                              d++   + ++++  +  n++ ++s+s+++    + + +  ++ ++ ++\n",
            "  XP_004354513.1 186 NllvpqedhledsfelgsplmvnsnkkddndqddpssnsstpnrplSDNDIIIKNNNNNNNNNNNNNNSSSSSSSTL---STNIQTQSKTSTLSL 277\n",
            "                     33444444444444444444444444444444444444455555545555555555555555555555555555554...3467777789999** PP\n",
            "\n",
            "             Zip 188 illlglilHsfidGLAlG..vssssstgislfiAvllHelpeelglgaiLlkaglsrkkav..llnllfaltaplGiliGilvrsisessgaeiv 278\n",
            "                     + l++l +Hsf+dG+ ++   s+s s g  + +A+++H+lp ++ + +iLl+ + +        + ++ a+++p+G ++  +     ++  ++ +\n",
            "  XP_004354513.1 278 TTLIALAIHSFVDGMVISgaYSASPSIGARVALAIVIHKLPDGFVMSSILLSKNQKIFGYHpfASMIFIACMTPIGSFVSSF---LFNGIIQSST 369\n",
            "                     *****************99*********************************999987766779999***************...5566788999 PP\n",
            "\n",
            "             Zip 279 sgillalsaGtflYvalvellphEllnekgssslkqs..llqllglllGfal 328\n",
            "                      + +l++ aGtf+ ++++e+lp Ell++++   lk+s  l +l++  lGf++\n",
            "  XP_004354513.1 370 ISFILGFGAGTFIFITTCEILP-ELLHNSK---LKKStlLISLVVGYLGFVV 417\n",
            "                     **********************.******4...4443003344444556654 PP\n",
            "\n",
            ">> XP_004350816.1  cytochrome c oxidase assembly protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   18.5   1.5   4.4e-06   0.00062      94     170 ..      59     131 ..      50     219 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 18.5 bits;  conditional E-value: 4.4e-06\n",
            "             Zip  94 klltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     ++l++++++++++k++t++++++++ +++++k    ++++++e +++++++++ ++++k ++++++ +  ++e +++\n",
            "  XP_004350816.1  59 QILRYFSTENNNNKNTTENNKNNNTVNKNDQK----EESTSEESNNNNEQQQNQQNEHKYKQQRRSNKEYSEELKEK 131\n",
            "                     67888888855555555555555555544444....44444444444444444444433333333332222222221 PP\n",
            "\n",
            ">> XP_004357740.1  acetylglutamate kinase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.6   0.2   8.5e-06    0.0012      62     177 ..     205     327 ..     190     463 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.6 bits;  conditional E-value: 8.5e-06\n",
            "             Zip  62 ealeesqleee.hth...plagllvllGflllllveklltmivek......ksshkektpkkasssssesaakksvsndkveeseekedhvaead 146\n",
            "                     ea ee+q+++   t+    ++ l+++++fl l++vekllt + ++      k ++k+++p+++ ss ++++++k+v++ k+ +++++ ++ + ++\n",
            "  XP_004357740.1 205 EADEEKQQQGAiLTFsaiFYKLLIFVISFLPLFVVEKLLTFF-AWvqlellKAKAKNINPDQTFSSLKKKRKAKKVKKLKTLTNSQS-KVSTTKK 297\n",
            "                     45555555554455588899********************76.66699***999999999999999999999999999999999987.4466677 PP\n",
            "\n",
            "             Zip 147 tdadskeelrngkqeksesekeqaseeedke 177\n",
            "                      + +++  ++  ++e++ ++++++s +++++\n",
            "  XP_004357740.1 298 FNLNTPLIHSTVVEEPPNEQSSSSS-SSSST 327\n",
            "                     7777777777777777766666655.22222 PP\n",
            "\n",
            ">> XP_004361557.1  uncharacterized protein DFA_05840 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.2   0.0   1.1e-05    0.0016     107     171 ..      53     117 ..      31     312 .. 0.77\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.2 bits;  conditional E-value: 1.1e-05\n",
            "             Zip 107 kektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                      + ++k+a+ ++   ++ +sv+n++ +++ee +++++e++++++ kee+ ++++ek+e+e++++ \n",
            "  XP_004361557.1  53 YKYQQKHANLNYYSFYSTSSVNNNNNKKNEEEKEDKEEDKEEDKDKEEEEEKEEEKNEEEECHEI 117\n",
            "                     34456666666667777778888888877777777777777777777777777777777777654 PP\n",
            "\n",
            ">> XP_004361315.1  uncharacterized protein DFA_05597 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   17.1   0.2   1.2e-05    0.0017      76     172 ..     318     427 ..     234     494 .. 0.69\n",
            "   2 ?   -0.2   0.2       2.2   3.1e+02      79     153 ..     568     636 ..     542     671 .. 0.55\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 17.1 bits;  conditional E-value: 1.2e-05\n",
            "             Zip  76 plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     ++  +  ++G  ++  + ++++++++k    + +++++ +ssss ++   + +nd +++++ +++++++ + +++ +++++++++e +e++ +++\n",
            "  XP_004361315.1 318 LVFTGHSMGGTSATTATLRAMELVYKK---SNINSSSNDHSSSSSNNNNIENNNDDTTTNTTNHNNNVNNENNNNIDSQQQQQQEEVEENNVHKN 409\n",
            "                     555566677777788888888888888...66667777777777777788888888888888889999998888888888888888888888776 PP\n",
            "\n",
            "             Zip 171 s................e 172\n",
            "                     +                 \n",
            "  XP_004361315.1 410 NntittlgqnnrlelkcF 427\n",
            "                     645555555555555550 PP\n",
            "\n",
            "  == domain 2  score: -0.2 bits;  conditional E-value: 2.2\n",
            "             Zip  79 gllvllGflllllveklltmivek.......ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadske 153\n",
            "                      l+ l  f l++ +++ + +++++       +             ++ +    ++++++ +++s    ++  ++++++++++\n",
            "  XP_004361315.1 568 HLFKLYNFRLMFAADHSIDLYYQRvmrqtrpN-------------HKINIIHFNTCNKCLTDNSGLPINNPNNHQSHQNNSQ 636\n",
            "                     55566667788999999999999944422220.............2222222222222222222222222222222222111 PP\n",
            "\n",
            ">> XP_004358720.1  uncharacterized protein DFA_00735 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   14.7   0.6   6.3e-05    0.0089      56     172 ..      90     212 ..      89     286 .. 0.76\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.7 bits;  conditional E-value: 6.3e-05\n",
            "             Zip  56 LlPealealeesqleeehth....plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaead 146\n",
            "                     L P ++++l+  + +++h+      +  ++vl+ f   + ++k +  i +     ++ ++k+++++ +++ ak++  n++ +++++  +++++++\n",
            "  XP_004358720.1  90 LTPTSQGKLKIDCRHANHMLkgldQYSHVWVLFWFHNNYRIDKNTDVITPP---DEDRINKQKQKQLENKIAKDQLLNNNNQQEQQIDQEQEQEE 181\n",
            "                     6789999999999999877799999999********************999...77788888888888888888877777777777755544444 PP\n",
            "\n",
            "             Zip 147 tdadskeelrngkqeksesekeqas.......e 172\n",
            "                      d +  e + n+ +e +++e++q++       +\n",
            "  XP_004358720.1 182 NDIE--EDEENEDGEDDGEESQQTKednivssS 212\n",
            "                     4444..444444444444444444422443331 PP\n",
            "\n",
            ">> XP_004354180.1  uncharacterized protein DFA_10273 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.9   0.4   0.00011     0.016     108     172 ..     473     592 ..     469     663 .] 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.9 bits;  conditional E-value: 0.00011\n",
            "             Zip 108 ektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas............................... 171\n",
            "                     e++p+++sssss++++++s ++++++++  + +++ +  ++++ +++  ng+++ ++s+++ +s                               \n",
            "  XP_004354180.1 473 EPIPSNSSSSSSNNDSSSSNNDSSSSNNDSSSSNNDSSSSSNNGSSSSNNGSSSNNGSSSNDSSssdvtcklsrsgklrveypgdapinctskgi 567\n",
            "                     57899999999999999999999999999999999999999999999999999999999999885555555555555555555555555555555 PP\n",
            "\n",
            "             Zip 172 ........................e 172\n",
            "                                              \n",
            "  XP_004354180.1 568 secnapggfkclgllsssltqcfaP 592\n",
            "                     5555555555555555555555550 PP\n",
            "\n",
            ">> XP_004358408.1  uncharacterized protein DFA_00419 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.6   1.9   0.00014     0.019      90     174 ..      93     172 ..      69     245 .. 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.6 bits;  conditional E-value: 0.00014\n",
            "             Zip  90 llveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseee 174\n",
            "                     ++++  ++ +++     +++t+ +++++ ++s+ +++ +n++ +++ ++++++ + + ++++  +  n++ + + ++ + ++ + \n",
            "  XP_004358408.1  93 YILDTDIKALYSP----NTNTNTNTNTNNTKSGGGGTGNNNNKNDNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNKN-DN 172\n",
            "                     4445444444444....56666666666666666666666555555555555444444444444444444443333333332.11 PP\n",
            "\n",
            ">> XP_004350550.1  RhoGAP domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.6   1.7   0.00014      0.02     103     178 ..      38     111 ..      15     193 .. 0.64\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.6 bits;  conditional E-value: 0.00014\n",
            "             Zip 103 ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkek 178\n",
            "                     +++ +ek+++k ++s+   +a +   +d++e++   ed++++ + d+d +e+ +++++e+++ e++q  e+ ++e+\n",
            "  XP_004350550.1  38 EEEDGEKEEDKVEQSDDDDEARSGEASDEIEDED--EDNQESGQDDDDDDENSAAEEEEQEGGEENQDGEDGEEEE 111\n",
            "                     2222333333333333333333333333333333..4444555555555666666666666666666543333333 PP\n",
            "\n",
            ">> XP_004357612.1  uncharacterized protein DFA_02128 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.1   2.4    0.0002     0.029      75     176 ..     260     357 ..     179     377 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.1 bits;  conditional E-value: 0.0002\n",
            "             Zip  75 h....plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqekses 165\n",
            "                          pl  ++++++++l++++  + +m+        ++ + +++ ++   + + s++ ++ e+++ k++ ++ +  +++++ ++ n+k++  ++\n",
            "  XP_004357612.1 260 SrsvfPLGVFCLVISLFLFYFF--CYHMWLVV-----KNKTTNETFKYGDLKEQISINRVEQENQSLKHELDELESINNQNTNNQNNNKKKNGKQ 347\n",
            "                     0444477788888888888875..55555555.....3344444444555555555555555555554333333333333222222222222222 PP\n",
            "\n",
            "             Zip 166 ekeqaseeedk 176\n",
            "                     +++q++ + ++\n",
            "  XP_004357612.1 348 QQQQQQ-TNNN 357\n",
            "                     222211.1111 PP\n",
            "\n",
            ">> XP_004361841.1  uncharacterized protein DFA_06128 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.8   0.6   0.00024     0.034      91     172 ..     150     263 ..     147     303 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.8 bits;  conditional E-value: 0.00024\n",
            "             Zip  91 lveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhva.eadtdadskeelrngkqeksesekeqas............. 171\n",
            "                     ++e++l  +++ +++++++t+++ ++++++ + k+  ++d+ e+++++++++    + +++++++  n+++++++s+k++ s             \n",
            "  XP_004361841.1 150 IAERYLLGYIDNENNNNNNTNNTNNTNTNNTKIKNNFKDDNNENNTNNNNNNGiLNNVNKKTTTTNNNNNTSSTSSSKTK-Siiirlpliwgkgd 243\n",
            "                     57888888888888899999999999999999999999999888888755443034444444444444444444443333.35777777777766 PP\n",
            "\n",
            "             Zip 172 ...................e 172\n",
            "                                        +\n",
            "  XP_004361841.1 244 nvlkhlcnlantlrwvwigG 263\n",
            "                     66555555555555555554 PP\n",
            "\n",
            ">> XP_004352581.1  uncharacterized protein DFA_09153 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.6   1.6   0.00028     0.039     109     172 ..     117     188 ..      99     229 .. 0.42\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.6 bits;  conditional E-value: 0.00028\n",
            "             Zip 109 ktpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas............e 172\n",
            "                      ++++++++s+ ++  +s ++++ +++e+ ++++++++ ++d+k    +++ +  +++k+  +             \n",
            "  XP_004352581.1 117 RNNEQKNNNSNSNDNDNSSNDNNNQDTEQQQQQQQQQQQQQDNKRIYTEEELS--SQSKQ--KvaaicndlkieyR 188\n",
            "                     23333333333333333333333333333333222222222222222222211..11111..02234566655551 PP\n",
            "\n",
            ">> XP_004357413.1  uncharacterized protein DFA_02691 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.9   4.3   0.00045     0.063       9     180 ..     145     346 ..     137     397 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.9 bits;  conditional E-value: 0.00045\n",
            "             Zip   9 iaililsllgllgpllvpkiskl.......lsklllslllafavGvllgtaflhLlPealealeesqleeehth....plagllvllGflllllv 92 \n",
            "                     +++++  ++++   +l++ i+k+         +l+++ l+++++   +    + +   ++  + +  ++++ +h    +  ++ +l G +++  v\n",
            "  XP_004357413.1 145 LLVFFGIFAVVNYFILYRQIKKYetkmrsdNMSLRVRKLKILSAALVVTIVCVLIRDLVYYVIIKVPMDHNLRHlssfITFFFEALQGVIIMIAV 239\n",
            "                     44444444444455555555555677777766666666666666544444444444445556666666666666677777777788888888877 PP\n",
            "\n",
            "             Zip  93 eklltmivek...ksshkektpkkasssssesaakksvsndkveeseekedhvaead...tdadskeelrngkqeksesekeqas.......... 171\n",
            "                      +  ++++     + ++++ t + + ss+s+s+++ s ++ ++ +  ++ + v+ ++    ++d+++  ++++qeks++++ +as          \n",
            "  XP_004357413.1 240 SERPSYYFLFrnvNPHRGDSTASSHVSSESNSQSVASKNMATTITIDDGASSVTVQHpniNNNDKSHVVHQQQQEKSSHIALTASnlvewsdded 334\n",
            "                     7777777766799999999999999**************99999999998877777633345667777888888889999888889999999988 PP\n",
            "\n",
            "             Zip 172 ...eeedkekla 180\n",
            "                        ++edk++  \n",
            "  XP_004357413.1 335 edeDDEDKQDGN 346\n",
            "                     776333333333 PP\n",
            "\n",
            ">> XP_004356110.1  cyclin-like F-box containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.4   0.5   0.00032     0.045      51     176 ..       9     130 ..       8     186 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.4 bits;  conditional E-value: 0.00032\n",
            "             Zip  51 taflhLlPealealeesqleeehthplagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaea 145\n",
            "                     t+++ +l + +ea+++  l+++       ++ l+  + l+++  + t+  +++s+++ kt+ ++++++++s++ ++ s+++ +++ +   +++  \n",
            "  XP_004356110.1   9 TSIVVVLTHTMEAIKQLFLKSTG--QRNLIITLVTSVSLFIITCIYTYRLKSSSRQQSKTTTTTNDEHKQSSTISTPSKNNNNNNNQ--HVASTN 99 \n",
            "                     66777777777777777776642..34444455555555555666666666644444444444444444444444444333333222..344445 PP\n",
            "\n",
            "             Zip 146 dtdadskeelrngkqeksesekeqaseeedk 176\n",
            "                     ++ a+s+++ ++ + eks  +++q+++++ +\n",
            "  XP_004356110.1 100 ESAASSSTTPKKIVNEKSPAKPQQQQQQQVT 130\n",
            "                     5555556666666666666666665533333 PP\n",
            "\n",
            ">> XP_004363105.1  uncharacterized protein DFA_03502 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.9   1.5     0.031       4.3     171     172 ..      91     290 ..       5     347 .. 0.45\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.9 bits;  conditional E-value: 0.031\n",
            "             Zip 171 s.............................................................................................. 171\n",
            "                     +                                                                                              \n",
            "  XP_004363105.1  91 Qrieklkekknlkqkeklerqqiltnnnnnninnnninrdikndilltnnnnnndgststihtskntplvalstkqqpeiiepplpsdqhmeeee 185\n",
            "                     13333333332222222222233333333444444444444444444444444444444444444444444444444444444444444444333 PP\n",
            "\n",
            "             Zip 172 ............................................................................................... 171\n",
            "                                                                                                                    \n",
            "  XP_004363105.1 186 edhqghdhhghfkkspiklrardklrrirtpnvpppkgvnivddpfieldsvdsrymededqaekrrilgtrkyetayikidyylehiniffsvv 280\n",
            "                     33333333333333355555555555555555555555555555555555555555555555555555555555555555555555555555555 PP\n",
            "\n",
            "             Zip 172 .........e 172\n",
            "                              +\n",
            "  XP_004363105.1 281 cygiyiaisY 290\n",
            "                     5555555554 PP\n",
            "\n",
            ">> XP_004354958.1  glycosyltransferase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.5   0.0   0.00059     0.083      75     135 ..     211     285 ..     131     297 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.5 bits;  conditional E-value: 0.00059\n",
            "             Zip  75 h..............plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkvees 135\n",
            "                     +              ++a l +++G l++ l+ +++++i + +s+ + k++++++++ ++++  +s +n+++ + \n",
            "  XP_004354958.1 211 IenlalhgihnrlthLFASLPLMIGPLIIVLTIEIISIIRNYNSNSNSKSNQPTTNNNNNNNNNNSNNNSESGNG 285\n",
            "                     044444444444555**********99999999999999999666666666666666666666665555555433 PP\n",
            "\n",
            ">> XP_004352197.1  uncharacterized protein DFA_09541 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.6   0.1    0.0046      0.65      76     150 ..      45     116 ..       7     123 .. 0.78\n",
            "   2 ?    3.4   0.3      0.17        24     100     151 ..     310     358 ..     290     366 .] 0.36\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.6 bits;  conditional E-value: 0.0046\n",
            "             Zip  76 plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdad 150\n",
            "                     +  ++l   +++l  +v ++++ + ++     ++t+ + +++++++ ++++ s++++++s+ +++  + +d++++\n",
            "  XP_004352197.1  45 IISFGLWRYSIFLPRFVQNIISSFTPT---PTTTTTTTNNNNHNNNITQQRSSSSTTTTSTTNTNTSTTTDSNQP 116\n",
            "                     455555667899999999999999999...999999999999999999999999999999999887777777665 PP\n",
            "\n",
            "  == domain 2  score: 3.4 bits;  conditional E-value: 0.17\n",
            "             Zip 100 vekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdads 151\n",
            "                     +++    +++++ + ++s ++  ++++ + ++ + +  +++++++++ ++++\n",
            "  XP_004352197.1 310 FSS---SNRNESINNQNSNNYYTTSGTRNTNNNNINNTNNSNNPSQQQQQQQ 358\n",
            "                     333...2233333333333333333333333333333333333333222222 PP\n",
            "\n",
            ">> XP_004366334.1  uncharacterized protein DFA_03924 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.9   4.0   0.00045     0.064      79     168 ..       9     102 ..       2     180 .. 0.63\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.9 bits;  conditional E-value: 0.00045\n",
            "             Zip  79 gllvllGflllllveklltmivek....ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqekseseke 168\n",
            "                      ll+l+G+  + ++ ++   +v+k    ++++++   ++ s+ ss+s++++s s++++++s+ + +  + ++++ +s+++  ++kq+ ++++  \n",
            "  XP_004366334.1   9 ILLLLIGIACIATITSARDVVVNKqsglHQNNNNHHGHDPSAASSYSYSSSSSSSSSSSSSSSSSSSTTTKKSSSSSTTKNYKQKQQNPGNQVG 102\n",
            "                     5788999999999999999999997555333333333333333444444444444444444444444444444444444444444444444433 PP\n",
            "\n",
            ">> XP_004362039.1  SAP DNA-binding domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.4   0.4   0.00063     0.089      69     168 ..     252     348 ..     238     464 .. 0.64\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.4 bits;  conditional E-value: 0.00063\n",
            "             Zip  69 leeehth..........plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadske 153\n",
            "                     + ++++h          +++   v+ G+ ll  ++  ++ i ++     +++ ++++++ss++++ +  ++d  +e+ ++         d+d +e\n",
            "  XP_004362039.1 252 TPGHIRHiisrdwelfkVFPRSSVIAGAGLLRYIDPKTEVITKS-----NDSDDQEDNNSSHNDDDDDNQDDDDDEDDDD--------DDEDDEE 333\n",
            "                     44554445666666666777777777777777777777766666.....3333333333333333333222222222222........2222222 PP\n",
            "\n",
            "             Zip 154 elrngkqekseseke 168\n",
            "                      + +++qe +++++e\n",
            "  XP_004362039.1 334 DDDEQEQEVEQEDEE 348\n",
            "                     222222222222221 PP\n",
            "\n",
            ">> XP_004357421.1  phosphatidylinositol glycan [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.5   6.1    0.0049      0.69     154     199 ..      16      60 ..       5      64 .. 0.62\n",
            "   2 ?   -0.4   0.0       2.6   3.7e+02      81     114 ..     702     731 ..     693     745 .. 0.67\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.5 bits;  conditional E-value: 0.0049\n",
            "             Zip 154 elrngkqeksesekeqaseeedkeklaslkvraiilllglilHsfi 199\n",
            "                     + +  ++++++s+++++s+ +d++k  + k+  +i++lgli+Hs++\n",
            "  XP_004357421.1  16 HSHSHSSHHHHSSHHHSSHGQDEKKRFNGKI-LFIVVLGLIFHSVF 60 \n",
            "                     3344444555555555554445555555544.67899*******98 PP\n",
            "\n",
            "  == domain 2  score: -0.4 bits;  conditional E-value: 2.6\n",
            "             Zip  81 lvllGflllllveklltmivekksshkektpkka 114\n",
            "                     lv+lG +l++ + +++ ++v+      +++  + \n",
            "  XP_004357421.1 702 LVCLGGILIFAIGSFTIYYVNN----RTNWATTV 731\n",
            "                     5788888888899999999888....23333333 PP\n",
            "\n",
            ">> XP_004362965.1  uncharacterized protein DFA_03360 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.2   0.8   0.00072       0.1     102     167 ..     355     424 ..     313     430 .. 0.50\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.2 bits;  conditional E-value: 0.00072\n",
            "             Zip 102 k.......ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesek 167\n",
            "                     +       ++s ++ ++ + s+sss s++++s s++++++++++ ++   +++++ds ++ +n+k+ +s++e+\n",
            "  XP_004362965.1 355 RfkivsrtDKSSGRRSTTTRSNSSSSSNSSSSSSSSSSNGNKDNSNN---KNSNNDSNSNSSNSKSMSSSKEE 424\n",
            "                     15555555445555555555555555555555555555555555443...33333333333333333222222 PP\n",
            "\n",
            ">> XP_004362041.1  SAP DNA-binding domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.0   1.5   0.00085      0.12      76     170 ..      89     184 ..      49     300 .. 0.69\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.0 bits;  conditional E-value: 0.00085\n",
            "             Zip  76 plagllvllGflllllveklltmivekksshkektpkk.asssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeq 169\n",
            "                     +l+  +v+ G   l +v++ ++ + e+++++++k++++  ++++ e++ +++ +n+++++++++++++ + + ++++  +  n++ + + ++ + \n",
            "  XP_004362041.1  89 LLPSSTVIAGVGNLRFVDQPVNDLNEQEKQEDQKEEEDyDDEEEDEEEEEEEEENKNFQPQSKNNSNNINNNNNNNNNNNNNNNNNNNNNNSGNN 183\n",
            "                     56667788888888899999999988833333333333033344455666677778888888887777777776666666666555555544444 PP\n",
            "\n",
            "             Zip 170 a 170\n",
            "                     +\n",
            "  XP_004362041.1 184 N 184\n",
            "                     3 PP\n",
            "\n",
            ">> XP_004350243.1  uncharacterized protein DFA_11300 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.6   2.5    0.0011      0.16     101     179 ..      88     166 ..      41     187 .. 0.51\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.6 bits;  conditional E-value: 0.0011\n",
            "             Zip 101 ekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas....eeedkekl 179\n",
            "                     + +++h++ +++  ++s+s s + +  +++++++s+  +   +++   ++ +ee+ +++q+ s ++ ++++    +   +++l\n",
            "  XP_004350243.1  88 H-HHHHQNSITSRNNQSTSISTPPQPSPCSSSTTSTPTTTTSSKDTYFDKDDEEEEEENQDRSNNTRKRKKivnaK---TTQL 166\n",
            "                     1.34444444444444444444333333333333333333333333344444444444444444444444443332...2222 PP\n",
            "\n",
            ">> XP_004356149.1  uncharacterized protein DFA_08661 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.0   6.8   0.00086      0.12     121     177 ..     102     144 ..      24     221 .. 0.69\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.0 bits;  conditional E-value: 0.00086\n",
            "             Zip 121 saakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedke 177\n",
            "                      ++k+ +++d +++s+ kedh ++              k ++++++++  ++++++e\n",
            "  XP_004356149.1 102 REKKSGHHDDHHKDSHHKEDHHHK--------------KDDHHDDGHHDDHHDHHHE 144\n",
            "                     233333333333333333332222..............1122222222211111111 PP\n",
            "\n",
            ">> XP_004351373.1  Phytochelatins synthase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.1   6.0    0.0016      0.23      94     176 ..     219     298 ..     204     415 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.1 bits;  conditional E-value: 0.0016\n",
            "             Zip  94 klltmivekksshkektpkkasssss......esaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedk 176\n",
            "                     +l+ +++++        ++ +s+sss      + +   s +n+ +++++ +    +++++++ +++++ n++ + ++++++q+ +++++\n",
            "  XP_004351373.1 219 ELTPHVHHR------CKSSSSSASSSlltiqhHRHHLISNNNKRHYSQQCC---SVQHNHSHGHSHSHGNSHAHANGHNHNQQPQQQQQ 298\n",
            "                     333333333......222222222223333323333333333333333333...23444444444444444444444444444311111 PP\n",
            "\n",
            ">> XP_004355729.1  uncharacterized protein DFA_08235 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.6   0.0    0.0011      0.16     128     172 ..     178     221 ..     167     326 .. 0.75\n",
            "   2 ?    0.0   3.5       1.9   2.7e+02     115     155 ..     505     545 ..     479     553 .] 0.41\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.6 bits;  conditional E-value: 0.0011\n",
            "             Zip 128 sndkveeseekedhvaeadtdadskeelrngkqeksesekeqase 172\n",
            "                      +d+++++eeke+++++++ ++ ++e++ ++k+e +e++ke +++\n",
            "  XP_004355729.1 178 RADQIKQQEEKEQKEQQQQNEQ-KEENKEEEKEEEEEDNKENKQD 221\n",
            "                     5677777777766444444333.3444444444444444443332 PP\n",
            "\n",
            "  == domain 2  score: 0.0 bits;  conditional E-value: 1.9\n",
            "             Zip 115 sssssesaakksvsndkveeseekedhvaeadtdadskeel 155\n",
            "                     ++++++++++   + ++ +++ ++++++ + + ++++  ++\n",
            "  XP_004355729.1 505 QQEKEKEKDGANLMTNNNNNNNNNNNNNNNNNNNNTTNNTQ 545\n",
            "                     22222222222222222222222222222222222222222 PP\n",
            "\n",
            ">> XP_004361946.1  uncharacterized protein DFA_06234 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.7   2.4    0.0021      0.29     106     182 ..      50     114 ..      27     225 .. 0.66\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.7 bits;  conditional E-value: 0.0021\n",
            "             Zip 106 hkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklasl 182\n",
            "                     h++++++++++ +++++ +++ ++ + +e e+ e+            e++ ++k+e +++e eq++ e+++ ++++ \n",
            "  XP_004361946.1  50 HNRIIEGQERERQERERQERERQERERQERERQER------------EQQEREKEEREKHEREQQEREKEERQPKTA 114\n",
            "                     44444555555555555555444444444444433............333333333333333333322222222222 PP\n",
            "\n",
            ">> XP_004363094.1  uncharacterized protein DFA_03491 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.1   0.0     0.026       3.7      77     171 ..     108     242 ..      91     343 .. 0.77\n",
            "   2 ?    1.4   3.1      0.71     1e+02      99     161 ..     389     441 ..     376     455 .. 0.48\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.1 bits;  conditional E-value: 0.026\n",
            "             Zip  77 lagllvllGflllllveklltmivek........ksshkektpkkasssss................................esaakksvsndk 131\n",
            "                     ++ l++  G+l ++  e++++ + ++        ++s++++++k+a++ +                                   ++ +  +nd \n",
            "  XP_004363094.1 108 FPLLFIYSGLLCFMRWESMVKRWNNGdiydyrriYNSQQRDIHKNANARKYdlitgffdhlsfdqdsvkeekgviflptlnqdDDDDDDDDDNDH 202\n",
            "                     67777888888888888888888888888888886666666666666666667777777777777777766666666666666666777777778 PP\n",
            "\n",
            "             Zip 132 veeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +++s+++e+ + ++d d++++ e  +g++e +e+e+ q++\n",
            "  XP_004363094.1 203 LQDSSDDEELNISKDDDKEEDDEIFDGEEEEDEEEDYQPH 242\n",
            "                     8888888888888888888888888888888888777766 PP\n",
            "\n",
            "  == domain 2  score: 1.4 bits;  conditional E-value: 0.71\n",
            "             Zip  99 ivek...ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqe 161\n",
            "                     i++k   +++++++++ k +++++ +++ k+ +n+++++se+ ++             ++ ++++ \n",
            "  XP_004363094.1 389 IKSKqpkENDKSTNIKVKLQDNEKDKKKRKRDNNNNIKKSEKVKS-------------TTPKQVKT 441\n",
            "                     333333334444444444444444444444444444444444433.............32222222 PP\n",
            "\n",
            ">> XP_004356331.1  putative transmembrane protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.9   0.0    0.0019      0.26     134     205 ..      53     125 ..      32     136 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.9 bits;  conditional E-value: 0.0019\n",
            "             Zip 134 eseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvr......aiilllglilHsfidGLAlG 205\n",
            "                     ++ + +++ + + td+++++e+++++q+++++++++     ++  +++   r       + + +++ +   i G+++G\n",
            "  XP_004356331.1  53 NNNNTFKNYRYYSTDHNQQQEQQQQQQQQQQQQQQR-----KENVYTKADWRltganrDWNNYIAFLIGAAISGISFG 125\n",
            "                     222233333334444443333333333333333333.....33333333333333444*******************9 PP\n",
            "\n",
            ">> XP_004356130.1  putative RNA helicase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.7   0.0    0.0021       0.3     108     209 ..     158     335 ..     120     341 .. 0.71\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.7 bits;  conditional E-value: 0.0021\n",
            "             Zip 108 ektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas............................... 171\n",
            "                      + ++ ++++++e++ ++s s+++vee ee e++++++ +++++ +                 s                               \n",
            "  XP_004356130.1 158 SDDEDIKDNEEKEQDESDSDSDSEVEEVEEVEEEEQKVPKNNNTNK-----------------Sipkvkvepldtlktlesnkrmkkevvmdlpt 235\n",
            "                     2333344444444444455555555555555444444444444444.................14444444444444455589999999****** PP\n",
            "\n",
            "             Zip 172 ..............................................................eeedkeklaslkvraiilllglilHsfidGLAl 204\n",
            "                                                                                   + + +  ++++ ++  + +l+l  Hs+++ LA \n",
            "  XP_004356130.1 236 feelhlsrplqkavaklgytqptpiqakaiplilngkdilasattgsgktaafilpilerllYRDATHRVSRVLIVLPTRELALQCHSVFESLAQ 330\n",
            "                     *********************************************************9999888888888888888999999*******999998 PP\n",
            "\n",
            "             Zip 205 Gvsss 209\n",
            "                     ++ ++\n",
            "  XP_004356130.1 331 FTNVQ 335\n",
            "                     75444 PP\n",
            "\n",
            ">> XP_004358095.1  uncharacterized protein DFA_06849, partial [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.3   1.9    0.0029      0.41     101     172 ..      70     143 ..      44     208 .. 0.53\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.3 bits;  conditional E-value: 0.0029\n",
            "             Zip 101 ekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas.......e 172\n",
            "                     e+ ++ k++ ++k+++++++ ++ +s +++++ee+ee+  ++ e+++++++ +     k  +++      +       +\n",
            "  XP_004358095.1  70 ES-KQPKKDKKEKKETNKEKGKDDDSDEESSHEEDEEGYVVEDEEQHTDKKRKASTPPKPTTKKAP----AkkaaakdK 143\n",
            "                     22.333444444444444455555555555555555554443333333332222222222222222....122233330 PP\n",
            "\n",
            ">> XP_004362018.1  uncharacterized protein DFA_06313 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.6   2.7    0.0024      0.33     114     163 ..       9      60 ..       1     172 [. 0.60\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.6 bits;  conditional E-value: 0.0024\n",
            "             Zip 114 asssssesaakksvsndkveeseek..edhvaeadtdadskeelrngkqeks 163\n",
            "                     +++++++++++k+ + +++ ++++   ++ +++++ ++++++++++++q+k+\n",
            "  XP_004362018.1   9 QQKKQKQKQKQKQKQIQQLIKKKKQkkKQLQQQQQQQQQQQQQKQKQQQQKQ 60 \n",
            "                     1111111111111111111111111001111111111111111111111111 PP\n",
            "\n",
            ">> XP_004354628.1  uncharacterized protein DFA_09274 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.4   4.8    0.0054      0.76     114     156 ..     102     145 ..      36     203 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.4 bits;  conditional E-value: 0.0054\n",
            "             Zip 114 asssssesaakksvsnd.kveeseekedhvaeadtdadskeelr 156\n",
            "                     ++++ +++++k++ +++ k e++e+++ h+ ++++++++ke++r\n",
            "  XP_004354628.1 102 KNDNNNKKEKKEKKEKKeKKEKKEKNHHHQDQHKMKKTKKEKER 145\n",
            "                     11111111111111111011111111112222222222222221 PP\n",
            "\n",
            ">> XP_004351053.1  uncharacterized protein DFA_12114 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.5   6.4    0.0051      0.72     126     180 ..     338     392 ..     323     490 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.5 bits;  conditional E-value: 0.0051\n",
            "             Zip 126 svsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekla 180\n",
            "                     +v+++k ee+ee e++++e++++++++ee+ ++++ek+e+e+e+++eee++e++ \n",
            "  XP_004351053.1 338 EVEKKKEEEEEEEEEEEEEEEEEEEEEEEEEAEEEEKEEEEEEEEEEEEEEEEIR 392\n",
            "                     4444444444444444445555555555555555555555555544444444443 PP\n",
            "\n",
            ">> XP_004350295.1  uncharacterized protein DFA_11352 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.5   7.9    0.0049      0.69     101     205 ..      34     154 ..      22     175 .. 0.48\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.5 bits;  conditional E-value: 0.0049\n",
            "             Zip 101 ek.ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas............eeedkeklasl 182\n",
            "                     ++ ks  k +++++ + sss s++++s sn+k ++s+ ke++++++ ++ +++++++  k+++++ ++++++              +++   +++\n",
            "  XP_004350295.1  34 SSsKSTSKSTSSNNNNPSSSSSSTSSSKSNKKKKNSKSKEKEKTSTTSTTSTSTSTSTVKSSSPSTQPSKKKpaaaaapvplkpVAQREVTFSNI 128\n",
            "                     33222333333333333444444555555555555555555555555555555555444444444444444466677777777644444444444 PP\n",
            "\n",
            "             Zip 183 kvr...aiilllglilHsfidGLAlG 205\n",
            "                       +   + + +      s+++GL++ \n",
            "  XP_004350295.1 129 VFVaqpVVVATATTAATSIPEGLSTT 154\n",
            "                     44422233333334455666666666 PP\n",
            "\n",
            ">> XP_004352044.1  DNA-3-methyladenine glycosylase I [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.1   4.5    0.0032      0.45     112     167 ..      34      89 ..       9     178 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.1 bits;  conditional E-value: 0.0032\n",
            "             Zip 112 kkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesek 167\n",
            "                       +++ +s++++k+++++d+ ++s e+e++++e+++++ +++e+ ++++e+s+++ \n",
            "  XP_004352044.1  34 TLKQNTRSKKKVKQEIEKDTESSSDESEQVDKEDEESDVEESEEEEEEEESSDEKV 89 \n",
            "                     45677788888888888888888888888888888888777777777777776654 PP\n",
            "\n",
            ">> XP_004351325.1  uncharacterized protein DFA_10682 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.6   1.5    0.0047      0.66     101     167 ..     291     356 ..     265     427 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.6 bits;  conditional E-value: 0.0047\n",
            "             Zip 101 ekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesek 167\n",
            "                     e+ ++ k++ ++k+++++++ ++ +s +++++ee+ee+  ++ e+++++++ +     k  +++  +\n",
            "  XP_004351325.1 291 ES-KQPKKDKKEKKETNKEKGKDDDSDEESSHEEDEEGYVVEDEEQHTDKKRKASTPPKPTTKKAPA 356\n",
            "                     22.3334444444444444555555555555555555544443333333322222222222222222 PP\n",
            "\n",
            ">> XP_004358641.1  uncharacterized protein DFA_00656 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.6   1.5    0.0047      0.66     101     167 ..     291     356 ..     265     427 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.6 bits;  conditional E-value: 0.0047\n",
            "             Zip 101 ekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesek 167\n",
            "                     e+ ++ k++ ++k+++++++ ++ +s +++++ee+ee+  ++ e+++++++ +     k  +++  +\n",
            "  XP_004358641.1 291 ES-KQPKKDKKEKKETNKEKGKDDDSDEESSHEEDEEGYVVEDEEQHTDKKRKASTPPKPTTKKAPA 356\n",
            "                     22.3334444444444444555555555555555555544443333333322222222222222222 PP\n",
            "\n",
            ">> XP_004355377.1  uncharacterized protein DFA_07884 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.9   2.7    0.0018      0.25      76     179 ..      60     160 ..      31     175 .. 0.74\n",
            "   2 ?    1.5   0.1      0.65        91     103     163 ..     316     343 ..     262     462 .. 0.63\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.9 bits;  conditional E-value: 0.0018\n",
            "             Zip  76 plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     +l    +  + ++   v+k++t+ +++  s ++ t + +sss++++++ +s +n++ +++++  +  ++ + d++ +++  +++ ++++ +ke +\n",
            "  XP_004355377.1  60 VLYLSSCAASTFICPNVNKTTTIGKST-FSFSTSTASSSSSSYTKKHYNNSGNNKQYTSNQKYDSRYKQRKYDDQRQQQPLRNEDHQDKPRKEDE 153\n",
            "                     556667777888888888888877776.99999999999***********************988777777777777776666666666666654 PP\n",
            "\n",
            "             Zip 171 seeedkekl 179\n",
            "                     +  +++e+ \n",
            "  XP_004355377.1 154 H--REEEEE 160\n",
            "                     4..444443 PP\n",
            "\n",
            "  == domain 2  score: 1.5 bits;  conditional E-value: 0.65\n",
            "             Zip 103 ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeks 163\n",
            "                     +++++++ +kk+ss++++ +                                  n ++++s\n",
            "  XP_004355377.1 316 NKNNNQTKKKKSSSEDQTTN---------------------------------TNISSSSS 343\n",
            "                     11111111111111111111.................................12222222 PP\n",
            "\n",
            ">> XP_004359704.1  uncharacterized protein DFA_01739 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.6   4.9    0.0023      0.33     119     179 ..     143     201 ..      75     269 .. 0.56\n",
            "   2 ?    2.5   0.3      0.34        47     113     169 ..     550     591 ..     522     667 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.6 bits;  conditional E-value: 0.0023\n",
            "             Zip 119 sesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekl 179\n",
            "                     ++++  ++  +++ +++++ ++++++++ ++++++++++++q++++++++q++  +++++ \n",
            "  XP_004359704.1 143 QQQQCLERLAQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQ--QQQQQQ 201\n",
            "                     22222233333333333332222222222222222222222222222222211..111111 PP\n",
            "\n",
            "  == domain 2  score: 2.5 bits;  conditional E-value: 0.34\n",
            "             Zip 113 kasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeq 169\n",
            "                     +++++++ +++k +++n+++e +++               +++++++q++++ + ++\n",
            "  XP_004359704.1 550 QQDQQQKDCSTKMEIDNCQMEIDSQ---------------HHHQQQQQQQQQFNGKH 591\n",
            "                     2233333333333333333332222...............22222222222222222 PP\n",
            "\n",
            ">> XP_004360802.1  uncharacterized protein DFA_05081 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.2   9.0    0.0063      0.89      88     171 ..      35     127 ..      31     144 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.2 bits;  conditional E-value: 0.0063\n",
            "             Zip  88 llllveklltmivek.........ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +ll+ +k+ +m+ +k         + ++ ++ ++++++++++++ ++  +n++ ++  ++++++++++ ++ ++ ++++++ +++ ++ +q+ \n",
            "  XP_004360802.1  35 ALLFRDKTREMFDDKpsnknnnnnNTNQYNNHNNNQYNNRNNNNNQYNGYNNNNQYYNNNNNNNSQYNNNRYNTNNQQQQQNNTNNNNNNQQV 127\n",
            "                     667777777887777655555544445555555566666666666666666666666666666666666666666666666666666655543 PP\n",
            "\n",
            ">> XP_004353824.1  uncharacterized protein DFA_12187 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.3   2.6    0.0059      0.83     105     164 ..      65     125 ..      37     164 .. 0.58\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.3 bits;  conditional E-value: 0.0059\n",
            "             Zip 105 shkektpkkasssssesaakksvsn.dkveeseekedhvaeadtdadskeelrngkqekse 164\n",
            "                     +h +++ ++a +++s++ ++++  + + +ee+ee e+++ ++++ +d++ee++  k++k+ \n",
            "  XP_004353824.1  65 QHRRISLEQAINNTSTKVKQRHDGQeEDSEEEEEEEENDDQEEEYDDKEEEEQYYKENKQY 125\n",
            "                     4445554544444444433333333033333334444455555555555554444444333 PP\n",
            "\n",
            ">> XP_004359443.1  MADS-box transcription factor [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.8   8.2    0.0081       1.1     109     184 ..     153     223 ..      74     251 .. 0.57\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.8 bits;  conditional E-value: 0.0081\n",
            "             Zip 109 ktpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkv 184\n",
            "                      tp +++s++++++ ++++ +++ +++ +  + +++a+ +a    ++++++q ++++ ++q+++++++++++++++\n",
            "  XP_004359443.1 153 RTPTNTTSHHQAQQQQQQAAQQQAQQQAQQ-QAAQQAQQQA----QQQAQQQAHQQQMQQQQQQQQQHPSPTQQAI 223\n",
            "                     333334444443333333333333333332.1111111111....1222222222222222222222222222222 PP\n",
            "\n",
            ">> XP_004350327.1  uncharacterized protein DFA_11384 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.4   2.9    0.0053      0.75     134     171 ..      18      55 ..       8     105 .. 0.55\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.4 bits;  conditional E-value: 0.0053\n",
            "             Zip 134 eseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     e e++e++++e++++++ +ee+ ++++e +e+e+e+++\n",
            "  XP_004350327.1  18 EIEDGEEEEEEEEEEEEDGEEEEEEEEEEEEEEEEEED 55 \n",
            "                     33333333333333333333333333333333333222 PP\n",
            "\n",
            ">> XP_004362344.1  UAS domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.8   5.2    0.0083       1.2     102     170 ..     274     346 ..     261     402 .. 0.56\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.8 bits;  conditional E-value: 0.0083\n",
            "             Zip 102 k......ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     +      +++ k++  +++++++ e+a + s +  + +++++++d++ e+  d++ ++ ++++++e++e+e+ q+\n",
            "  XP_004362344.1 274 QpldetdSKDVKRQKRHTTEEEELEKAIQLSLQGATNQNKQDDQDEQDEQ--DQQDEQDEQEEQEEQDEQEELQE 346\n",
            "                     25544222222333333333333344444444444444444443333333..33333333333333344444442 PP\n",
            "\n",
            ">> XP_004354593.1  uncharacterized protein DFA_09239 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.3  11.6     0.012       1.6     100     172 ..      62     159 ..      27     263 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.3 bits;  conditional E-value: 0.012\n",
            "             Zip 100 vekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas....................... 171\n",
            "                     +++++ h+ +t+++ ++++++++ +++++   ++++  + ++ +++++++  + ++ n++  +s+s+++++s                       \n",
            "  XP_004354593.1  62 INQDHIHHVNTQSHNHHHHNHNNHHHHHHHMDMNDMNMNMNNTHHQQHNNHDQYNHLNDHYTSSSSSSSSSSspcrnnnnnnnnnnnnnnqpsli 156\n",
            "                     33378888899999999999999999999999999999888888888877777777777777766666666656666555555555555444444 PP\n",
            "\n",
            "             Zip 172 ..e 172\n",
            "                       +\n",
            "  XP_004354593.1 157 laS 159\n",
            "                     440 PP\n",
            "\n",
            ">> XP_004363124.1  uncharacterized protein DFA_03521 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.5   6.4    0.0099       1.4      98     171 ..     215     288 ..     208     377 .. 0.65\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.5 bits;  conditional E-value: 0.0099\n",
            "             Zip  98 mivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +i++kk+  ++++++++++sss+s ++ + +n++ +++ + ++  ++      +++++ ++k+e +++ ke+++\n",
            "  XP_004363124.1 215 NIHKKKQPDSKIQEEEQNDSSSPSTTSTTTNNNNNNNNNDINSITESISNVTIQEQKEEEQKEEENDKPKEKEE 288\n",
            "                     44555666777778888888888888888888777777777554333333332222222222222222222222 PP\n",
            "\n",
            ">> XP_004359491.1  WD40-like domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.2   4.2     0.012       1.8     147     202 ..     526     579 ..     505     603 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.2 bits;  conditional E-value: 0.012\n",
            "             Zip 147 tdadskeelrngkqeksesekeqaseeedkeklaslkvraiilllglilHsfidGL 202\n",
            "                      +++s ++  n++ + ++s+++q  +++++++ ++++  + i   +++  s+idGL\n",
            "  XP_004359491.1 526 QEPNSNSNNNNNNNNNKSSKHQQ--HNHHHQQQHNHNHNQTIPATAVGGESIIDGL 579\n",
            "                     33333444444444444444444..4444455555555555555556666666655 PP\n",
            "\n",
            ">> XP_004359733.1  SET domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.5   3.2      0.01       1.4     108     179 ..      97     153 ..      68     232 .. 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.5 bits;  conditional E-value: 0.01\n",
            "             Zip 108 ektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekl 179\n",
            "                       ++k++++s++ +++k s +++k++e+ e e               +++++     +++ +++eee++e+ \n",
            "  XP_004359733.1  97 GANSKPKKKSKKDKKKKTSKKEKKTDEQGEAE---------------EQQQEDAAAVDATVKEEEEEEEEDG 153\n",
            "                     22222222222233333333333333322222...............2222222222222222222223333 PP\n",
            "\n",
            ">> XP_004366718.1  RhoGAP domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.9   4.4     0.015       2.2     126     175 ..     478     523 ..     449     575 .. 0.56\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.9 bits;  conditional E-value: 0.015\n",
            "             Zip 126 svsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeed 175\n",
            "                     + + + +e +++k    + +++++++ke+++++k+ek+++ekeq++++++\n",
            "  XP_004366718.1 478 QEERERIERDQKK----KDKKGEKEQKEKEKREKKEKEKKEKEQKKKDKS 523\n",
            "                     1111112222222....112233333333333333333333333332222 PP\n",
            "\n",
            ">> XP_004366832.1  uncharacterized protein DFA_02447 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.1   3.6     0.014       1.9      75     171 ..      39     164 ..      25     167 .] 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.1 bits;  conditional E-value: 0.014\n",
            "             Zip  75 hplagllvllGflllllveklltmivek.............................ksshkektpkkasssss..esaakksvsndkveeseek 138\n",
            "                     h+ +   + l +l+ + +++l  m+ ++                             k  ++    +k++++s    s+a+k++ ++++ ++e  \n",
            "  XP_004366832.1  39 HLTPLQSAKLNILVAYSLNSLFFMYLQTqgvsphnhkvkpeldrirhyitklqslskK--ESGVADEKEQKPSMtiDSEASKRIIQHSLASNEYI 131\n",
            "                     4555555555566666666666666666555555555777777777777766666432..22222223333322336667777777888888877 PP\n",
            "\n",
            "             Zip 139 edhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     ++++ + + + ++ ++  n++  ++++e+eq++\n",
            "  XP_004366832.1 132 NNNNNNNNNNTNNNTNNNNNNNFPPKKETEQKK 164\n",
            "                     777777777777777777777777777777754 PP\n",
            "\n",
            ">> XP_004358926.1  uncharacterized protein DFA_00951 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.3   3.5     0.011       1.6      92     147 ..      31     104 ..      23     109 .] 0.37\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.3 bits;  conditional E-value: 0.011\n",
            "             Zip  92 veklltmivek..................ksshkektpkkasssssesaakksvsndkveeseekedhvaeadt 147\n",
            "                     + ++++ +v+k                  +++ +++++ +++++ +++++++  +  ++++s+ ++++++++ +\n",
            "  XP_004358926.1  31 MKHIVSCLVSKkapvitslsfnnnivgnqNNNSNDNNNIEENKNGNNNDDQNNNNTTTTTTSTINNNNDTKDTK 104\n",
            "                     56666666666333333333333333332333333333333333333333333333333333333222222221 PP\n",
            "\n",
            ">> XP_004354601.1  uncharacterized protein DFA_09247 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.0   8.1   0.00043     0.061     103     175 ..      69     141 ..      39     216 .. 0.74\n",
            "   2 ?    0.2   1.8       1.6   2.3e+02      93     170 ..     930    1015 ..     905    1045 .. 0.46\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.0 bits;  conditional E-value: 0.00043\n",
            "             Zip 103 ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeed 175\n",
            "                     +ss +  +++  ++++ + + ++s +n++ +++ ++++++ + + ++++  +  n+ ++ + ++++ + ++ +\n",
            "  XP_004354601.1  69 SSSSSPVNNNINNHHHHHHRNSYSRNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNGKHNHNHNHNNNRSSMS 141\n",
            "                     4555666677778888888888888888888888877776666666665555555555555555544332222 PP\n",
            "\n",
            "  == domain 2  score: 0.2 bits;  conditional E-value: 1.6\n",
            "             Zip   93 eklltmivek.................ksshkektpkkasssssesaakksvsndkveeseek...............edhvaeadtdadske 153 \n",
            "                      e+  ++ +++                 ++++++++pk +s+ +s         ++++++s+                 +              \n",
            "  XP_004354601.1  930 EHSKKYQKSTkssitvqsnslasltlnNNNNHQIQPKGSSNTKS---------SSNITPSKYFkslyykflggnykdyN-------------- 999 \n",
            "                      44444444444444444422323333333333333333333333.........33333333333333333333333332.............. PP\n",
            "\n",
            "             Zip  154 elrngkqeksesekeqa 170 \n",
            "                        +n++ + ++++++ +\n",
            "  XP_004354601.1 1000 -SHNNHHHGHHNHHNIS 1015\n",
            "                      .3333333333333222 PP\n",
            "\n",
            ">> XP_004367084.1  uncharacterized protein DFA_07218 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.4   5.4     0.022       3.1      96     169 ..      25      95 ..      12     237 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.4 bits;  conditional E-value: 0.022\n",
            "             Zip  96 ltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeq 169\n",
            "                     l  +   ++++++ ++++++++++++++++s s++ + +++ k+++v++ +++++ k ++ +++ + + ++++ \n",
            "  XP_004367084.1  25 LLFF---NQQQQQYSSQPSTAKTNTKSSSSSSSSSALGSTKTKKQNVVQNKSNDPLKRSKNDNNNHNHNHNHSL 95 \n",
            "                     4444...3446777777777777888888888888888888899999999999998888888888888888764 PP\n",
            "\n",
            ">> XP_004366934.1  uncharacterized protein DFA_07061 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.0  10.7     0.014         2     101     172 ..      45     127 ..       8     158 .. 0.50\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.0 bits;  conditional E-value: 0.014\n",
            "             Zip 101 ekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas..............e 172\n",
            "                     +    +++ ++k+++++++es+  ++ ++++++  + + ++v+e+d++++ +ee+ ++++e++e+e+e+++              +\n",
            "  XP_004366934.1  45 DL---QSKSNKKEEEEEEEESDEDEQEKKVTSKLGKINIKQVKEEDEEEEDEEEEEEEEEEHQEEEEEEEElsefeeieqdwisdQ 127\n",
            "                     32...222222223333333333333333333333333444444444444444444444444444443333455666666666552 PP\n",
            "\n",
            ">> XP_004359218.1  leucine-rich repeat-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.5   6.2     0.021       2.9     105     206 ..     417     516 ..     375     524 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.5 bits;  conditional E-value: 0.021\n",
            "             Zip 105 shkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvraiilllglilHsfi 199\n",
            "                      ++++++++ +++ ++++  +  +nd+ + +  +++++  ++t+ ++++++++  ++++  +++++  + ++ + ++ +   ii+  ++  + +i\n",
            "  XP_004359218.1 417 TNNNNNNNNNNNNNENNENNDNNDNDNDT-TATNNNNDNTTATTTPTTTTTAATATPTPTTTTTTST-NNNNNNNSENEKSPIIVWQSIYPDLII 509\n",
            "                     12222222222222222222222222222.2333455556666666777777777777766666654.555555555555566666667777778 PP\n",
            "\n",
            "             Zip 200 dGLAlGv 206\n",
            "                     dGL lG+\n",
            "  XP_004359218.1 510 DGLYLGC 516\n",
            "                     8888874 PP\n",
            "\n",
            ">> XP_004351983.1  uncharacterized protein DFA_10097 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.9   5.2      0.03       4.3     103     186 ..    1453    1540 ..    1431    1610 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.9 bits;  conditional E-value: 0.03\n",
            "             Zip  103 ksshkektpkkasssssesaakksvsndkveeseek..........edhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvr 185 \n",
            "                      +++++e+ +++++++++e+++++s +n+k +++++k          ++++ + + ++++  +  n       ++ + ++++ ++    s+++r\n",
            "  XP_004351983.1 1453 NDKENEEDEDDEDDEEEEEDDEESKENQKKTGKKRKkkkitniikkKKKNNNNNNNNNNNNNNNN------NNNNNNNNNNNNNNLSGSQSTR 1539\n",
            "                      55555666666666666666666666666666665533332222222211111111111111111......1111111122233333444444 PP\n",
            "\n",
            "             Zip  186 a 186 \n",
            "                       \n",
            "  XP_004351983.1 1540 D 1540\n",
            "                      4 PP\n",
            "\n",
            ">> XP_004350793.1  uncharacterized protein DFA_11848 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.4   5.3     0.022       3.1     129     171 ..     403     445 ..     342     461 .. 0.57\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.4 bits;  conditional E-value: 0.022\n",
            "             Zip 129 ndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     ++  e++e+ ed+++e++++++++ee+ +++++ +e+e+e+++\n",
            "  XP_004350793.1 403 EVVDEDDEDEEDEEEEEEEEEEEEEEDEEEEEDEEEEEDEEEA 445\n",
            "                     3333333333333333333333333333333333333322222 PP\n",
            "\n",
            ">> XP_004350944.1  uncharacterized protein DFA_12005 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.0  11.5     0.029       4.1     139     172 ..      30      70 ..       5     121 .. 0.67\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.0 bits;  conditional E-value: 0.029\n",
            "             Zip 139 edhvaeadtdadskeelrngkqeksesekeqas.......e 172\n",
            "                       + +++++++++ ++++n + +++++++++ +       +\n",
            "  XP_004350944.1  30 LADHPSHHHNHPHNHNHHNHHDHHHHHHPHHHHphydhhhH 70 \n",
            "                     33445556666666666666666666666655543332211 PP\n",
            "\n",
            ">> XP_004358866.1  putative RNA helicase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.3   4.4     0.024       3.3      99     170 ..     136     206 ..     101     284 .. 0.62\n",
            "   2 ?    5.9   5.8     0.031       4.4     114     170 ..     454     510 ..     425     592 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.3 bits;  conditional E-value: 0.024\n",
            "             Zip  99 ivekksshkektpkkasssssesaakk..svsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     +ve+    ++++++k+sssss+++ k+    ++ + ee+e    ++++++++a +k ++++++q+ + ++ +++\n",
            "  XP_004358866.1 136 VVEE--TTSDTESEKKSSSSSKKNNKQgiVLHEREKEEKEGAIVDAKHKNKKA-KKGNQQQQQQQEDNKNRNSE 206\n",
            "                     4444..33333333333333333333300333333333333333333333333.33333333333333333322 PP\n",
            "\n",
            "  == domain 2  score: 5.9 bits;  conditional E-value: 0.031\n",
            "             Zip 114 asssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     ++++ ++ + +++v++d+ e++ee ed ++e++ ++d +ee+ +++ e +e+e+e +\n",
            "  XP_004358866.1 454 HYKKGQNDEEQEEVEQDEDEDQEEQEDAEEEEQQEEDGEEEEEEQVDEDQEEEEEME 510\n",
            "                     444444555555555555555555555554444444444444444444333333333 PP\n",
            "\n",
            ">> XP_004356887.1  cyclin [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.9  15.1      0.52        73     104     172 ..      75     150 ..      36     212 .. 0.55\n",
            "   2 ?   10.1   3.9    0.0016      0.23     106     164 ..     272     330 ..     256     422 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.9 bits;  conditional E-value: 0.52\n",
            "             Zip 104 sshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas........e 172\n",
            "                     +++++k++++++++ ++++ k +  n++++++ ++++++ + + ++++  +  n++ + + ++k+ ++        +\n",
            "  XP_004356887.1  75 GNKNNKQSPQHNNNNNNNNNKLTN-NSNITNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNKKNNKklksqsnsS 150\n",
            "                     333334444444444433333333.3333333333222222222222222222222222222222211122222230 PP\n",
            "\n",
            "  == domain 2  score: 10.1 bits;  conditional E-value: 0.0016\n",
            "             Zip 106 hkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqekse 164\n",
            "                     h+++++++ +++ ++++  +  +n++ +++ ++++++ + + ++++  +  n++ + + \n",
            "  XP_004356887.1 272 HNNNNNNNNNNNNNNNNNNNNSNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNDPN 330\n",
            "                     22222222222222222222222222222222222222222222222222222222222 PP\n",
            "\n",
            ">> XP_004359196.1  extracellular signal-regulated protein kinase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.8   5.1     0.034       4.7     113     169 ..      46     100 ..      20     171 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.8 bits;  conditional E-value: 0.034\n",
            "             Zip 113 kasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeq 169\n",
            "                      +++++++++a+++++++  ++ ++   ++ +++ ++ +++ + +++q++++ +++q\n",
            "  XP_004359196.1  46 LQQQQQAQQQAQQQAHQQAQQQAQQQ--QQIQQQQQQAHQQAQLQQQQQQQQAQQQQ 100\n",
            "                     11222222222222221111111111..11111111111111111111111111111 PP\n",
            "\n",
            ">> XP_004359778.1  N2227-like domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.1   4.9     0.027       3.8     104     159 ..       4      56 ..       1     169 [. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.1 bits;  conditional E-value: 0.027\n",
            "             Zip 104 sshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngk 159\n",
            "                     +sh+++ ++ +s+s+s+s++++s+++d  +++ ++   + + d +++++ee+ +++\n",
            "  XP_004359778.1   4 HSHSHDHSHSHSHSHSHSHDHSSHNCDDGDNDGDD---DISIDDQDNEDEEETDEN 56 \n",
            "                     33555555555555555555555444444333332...333333333333333333 PP\n",
            "\n",
            ">> XP_004360810.1  protein kinase 3 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.5   6.5      0.04       5.6     108     178 ..     186     253 ..     142     405 .. 0.70\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.5 bits;  conditional E-value: 0.04\n",
            "             Zip 108 ektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkek 178\n",
            "                     ++ + + +++ss   +  s+s++++++s+ k+d+++e++ +++ ++e  ++ +  +e ekeq++   ++++\n",
            "  XP_004360810.1 186 QIANTNNNQPSSLYLSNASISSNSSNGSQSKKDKKKEKEREKEREKEREKEREREKEREKEQKK---QQQQ 253\n",
            "                     4444555555554444444444444444444333333322222222222222222222222211...1122 PP\n",
            "\n",
            ">> XP_004366559.1  uncharacterized protein DFA_04150 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.5   7.2     0.041       5.8     103     162 ..      55     117 ..      25     190 .. 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.5 bits;  conditional E-value: 0.041\n",
            "             Zip 103 ksshkektpkkasssssesaakksvsndkvee...seekedhvaeadtdadskeelrngkqek 162\n",
            "                     k++++++++++++++++++++ ++ ++++ ++   +++k+dh+++++ +++s  ++ n++ +k\n",
            "  XP_004366559.1  55 KHNQQQQQQQQQQQQQQHKDRDSHPEEENNQQdilKKRKRDHQQQQQNNNNSNDNDNNSSNNK 117\n",
            "                     333333333333333333333333333322221113333444444444444444444444333 PP\n",
            "\n",
            ">> XP_004355277.1  UPF0551 family protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.6   6.6     0.039       5.5     106     183 ..      77     154 ..      47     209 .. 0.60\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.6 bits;  conditional E-value: 0.039\n",
            "             Zip 106 hkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslk 183\n",
            "                     +++++pk++ +++ +s+ + +++ +   ++ + +++ +++++++  +e + n  q+ +  +++++s+++++e+ +++k\n",
            "  XP_004355277.1  77 KQQPQPKQQPKQHGHSHGGVECDGNHGVPKPQPKQQPKQHGHSHGGVECDGNHGQNNTAAASAAPSSSQQQEQQSNSK 154\n",
            "                     344555555555555555554444444445555677777777777777777777776666666666444444433333 PP\n",
            "\n",
            ">> XP_004350733.1  kinetochore protein Ndc80 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.5   5.5      0.08        11      86     172 ..     592     686 ..     589     730 .. 0.61\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.5 bits;  conditional E-value: 0.08\n",
            "             Zip  86 flllllveklltmivek..ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas......e 172\n",
            "                     f+ ++ +ek+ ++i e+  k+   e+++  ++++s++ ++  s ++++++ se+k ++ +++  +++ k  +  ++ ek+ + k++++      +\n",
            "  XP_004350733.1 592 FIYFFCLEKIKQYIEEQieKKMSMEEENTIKENESKQLDDLLSDNKQQLKRSEQKLSNLENVSQREKDKNDKELKQYEKKFELKNRENemlkeqT 686\n",
            "                     67889999999999888986666677777778888888999999999999999996655554333322222222222222222222222233221 PP\n",
            "\n",
            ">> XP_004355651.1  uncharacterized protein DFA_08151 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.3   5.1     0.012       1.6      76     176 ..      12     104 ..       2     137 .. 0.50\n",
            "   2 ?    1.0   0.2      0.97   1.4e+02     144     177 ..     674     707 ..     654     727 .. 0.44\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.3 bits;  conditional E-value: 0.012\n",
            "             Zip  76 plagllvllGflllllveklltmivekksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                       +++++l ++ l+l +  ++++  +++++     +  +sssss +++++s +n++ + + + ++  ++ +       ++ n++ ++s+ +k+ +\n",
            "  XP_004355651.1  12 QQQEIIALSSLSLVLATTDMVRYKQTSNHNYYSSFSFSSSSSSSSPSSSSSFNNHTNNHNNNHNN--HNNH-------NTNNNENSTSSFKKSIK 97 \n",
            "                     56889999999999999999998555533333333333333333333333333333333333222..2222.......22222222222222222 PP\n",
            "\n",
            "             Zip 171 s.eeedk 176\n",
            "                     s +++++\n",
            "  XP_004355651.1  98 SfKHHNH 104\n",
            "                     2122333 PP\n",
            "\n",
            "  == domain 2  score: 1.0 bits;  conditional E-value: 0.97\n",
            "             Zip 144 eadtdadskeelrngkqeksesekeqaseeedke 177\n",
            "                      ++t++d + +   ++ +++ s+++q++e+e++e\n",
            "  XP_004355651.1 674 TTKTTKDGSISNPIDSNHHHLSSQQQEQEQEQQE 707\n",
            "                     2222222222222222222222222222111111 PP\n",
            "\n",
            ">> XP_004358977.1  pleckstrin domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.6  12.3     0.074        10     107     171 ..     181     245 ..     156     302 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.6 bits;  conditional E-value: 0.074\n",
            "             Zip 107 kektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +  tp+  s+sss +++kk+ +++k  e+e+ke++++e +++++ ++e +++k++ +e eke+++\n",
            "  XP_004358977.1 181 SSFTPSSGSNSSSSKDKKKEKEKEKEREKERKEEKEKERKEEKEREKERKEEKEKEKEREKEKEK 245\n",
            "                     44455555555555555555555555555555443333333333222222222222222222222 PP\n",
            "\n",
            ">> XP_004367225.1  putative mediator complex subunit 12 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.2   6.4       0.1        14     107     171 ..      23      85 ..       9     163 .. 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.2 bits;  conditional E-value: 0.1\n",
            "             Zip 107 kektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     ++++++++++++++++++++ ++ +++++++ ++  +++++++ + ++++ ++q++++++++q++\n",
            "  XP_004367225.1  23 QQQQQQQQQQQQQHQHRVQQLQQMQIQQMQQQQQ--QQQHMQQLQMQQQQMQQQQQQQQQQMQQQ 85 \n",
            "                     3445555555555555555555555555555433..34444444444444444444444444433 PP\n",
            "\n",
            ">> XP_004356343.1  uncharacterized protein DFA_08860 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.5   1.7    0.0012      0.17     116     192 ..     541     614 ..     490     632 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.5 bits;  conditional E-value: 0.0012\n",
            "             Zip 116 ssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklaslkvraiilllg 192\n",
            "                     ++++e +++ + ++++++++ee e+ ++ ++++++++ee++++++e s  +++q  +++  ek+ ++ + ++ +++g\n",
            "  XP_004356343.1 541 EKTKEDDTVTTQPEEPIKKEEETEK-KELVEENKPKEEEEKKKEEEESTTTQQQ--QQDVVEKIVTITIYTLSFTFG 614\n",
            "                     3344566666666666666666544.4555566666666666666666666665..567888888888888888877 PP\n",
            "\n",
            ">> XP_004350761.1  transcription initiation factor TFIID subunit [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.6   8.5     0.075        11     103     171 ..      38     106 ..       6     189 .. 0.60\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.6 bits;  conditional E-value: 0.075\n",
            "             Zip 103 ksshkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     ++++++++  + +s  +e + ++  +++  ++++e+ ++++e++ ++ +k+ l n++ + ++++++ ++\n",
            "  XP_004350761.1  38 NGNNNNNNRYDNDSTMKEYNEQDNEDEHHNNNNKENGKKKEEYKVSNITKTALLNNNNNNHSNGSNNTA 106\n",
            "                     122222222222222222222222223333333333444455555555555555555555555555554 PP\n",
            "\n",
            ">> XP_004350810.1  HEAT repeat-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.0   0.5       3.8   5.3e+02     128     170 ..     363     403 ..     310     461 .. 0.57\n",
            "   2 ?    9.0   3.8    0.0035      0.49     107     171 ..     959    1021 ..     909    1139 .. 0.68\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.0 bits;  conditional E-value: 3.8\n",
            "             Zip 128 sndkveeseekedhvaeadtdadskeelrngkqeksesekeqa 170\n",
            "                     +++++ e ee e++++ ++ +++  ee+ ++k   +e+e++++\n",
            "  XP_004350810.1 363 EEENIVELEEIEKDGEDEQDEQE--EEEEDKKVIVEEEEDQRN 403\n",
            "                     33333333333222222222222..222222222222222222 PP\n",
            "\n",
            "  == domain 2  score: 9.0 bits;  conditional E-value: 0.0035\n",
            "             Zip  107 kektpkkasssssesaakksvsndkveeseek...edhvaeadtdadskeelrngkqeksesekeqas 171 \n",
            "                      +++++kk+++++++++  ++++ d+v e++e    e +++++d+d++s+ee+ ++++e   +e++++s\n",
            "  XP_004350810.1  959 QRESEKKKNQPKPQQD--QEMKVDQVVEEKEEekkEVEGQQEDSDDESDEEEEDEEEE---EEEKKES 1021\n",
            "                      3333333333333333..3333333333333300033344444444444444433333...3333333 PP\n",
            "\n",
            ">> XP_004357732.1  NPR2 family protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.8   6.7     0.064         9     115     171 ..     129     185 ..      90     236 .. 0.53\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.8 bits;  conditional E-value: 0.064\n",
            "             Zip 115 sssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +++s++++   + ++++ +++++ ++++ ++++++++++++++++q++++s+++++ \n",
            "  XP_004357732.1 129 QQHSTNQQILLQQQQQQQQQQQQIQQQQMQQQMQQQQQQQQQQQQQQQQQSQQQSTT 185\n",
            "                     222222222333333333333333333333333333333333222222222222222 PP\n",
            "\n",
            ">> XP_004355597.1  NUC153 domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.9   7.0     0.059       8.4     109     172 ..     177     245 ..     108     322 .. 0.69\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.9 bits;  conditional E-value: 0.059\n",
            "             Zip 109 ktpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas......e 172\n",
            "                      ++ k++++ ++ a+ +s++++ +++ +++ + +++++++++++++++n++ +k+ ++ ++ +      e\n",
            "  XP_004355597.1 177 SQEFKKHNPAAKLAKLDSIYKEHFQNVQDDDE-DENEEEEEEKSTTKKNNNNKKNNNNNNRMEidsdedE 245\n",
            "                     34445556666666666666666666666533.2333333333333333333333333222222222220 PP\n",
            "\n",
            ">> XP_004366330.1  uncharacterized protein DFA_03920 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.2   5.5     0.051       7.2     109     180 ..      34      92 ..       3      98 .] 0.51\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.2 bits;  conditional E-value: 0.051\n",
            "             Zip 109 ktpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekla 180\n",
            "                     +  ++++++++ ++ ++             ed ++++d d+d++ e++ +  +++++e+eq++eee++e+  \n",
            "  XP_004366330.1  34 DDDDDEDEDEEDEEEGE-------------EDYDVGYDVDDDQEVEDAVDAFPPEDEEDEQEEEEEEEEHRG 92 \n",
            "                     33333333333333333.............445555555555555555555555555555554444443333 PP\n",
            "\n",
            ">> XP_004359062.1  uncharacterized protein DFA_01087 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.7   9.1     0.073        10     114     178 ..      49     114 ..      31     194 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.7 bits;  conditional E-value: 0.073\n",
            "             Zip 114 asssssesaakksvsndkveeseek.edhvaeadtdadskeelrngkqeksesekeqaseeedkek 178\n",
            "                     +++ +++++ + + ++++++++++   ++++++++ a ++++l++++q+++++++ q++++  +++\n",
            "  XP_004359062.1  49 QQQYQQYQQQQLHQQQQQLQQQQQLqLQQQQQQQHAAAQQAQLQQQQQHHHHQQQLQQQHMNGSSS 114\n",
            "                     333344444444444444444444314555566666677777777777777777777664444333 PP\n",
            "\n",
            ">> XP_004362923.1  uncharacterized protein DFA_03318 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.2   7.3       0.1        14     122     181 ..     175     230 ..     134     348 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.2 bits;  conditional E-value: 0.1\n",
            "             Zip 122 aakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqaseeedkeklas 181\n",
            "                     ++ ++ s +++++   + +    +d  + +kee+++++q++++++++ + ++ ++e++++\n",
            "  XP_004362923.1 175 STTSTTSTSSTTTVDTGSS----DDVIDATKEEQQQQQQQQQQQQQQDNTNNNQEEEISN 230\n",
            "                     2222222222222222222....2222222222222222222222222222222222222 PP\n",
            "\n",
            ">> XP_004350745.1  WD40 repeat-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.3   4.1     0.012       1.6      93     180 ..       3      87 ..       1     179 [. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.3 bits;  conditional E-value: 0.012\n",
            "             Zip  93 eklltmivekksshkektpkkasssssesaakksvsndkve.eseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekla 180\n",
            "                      k+ ++i+e++++  ++ + ++++s  + ++ k++sn++ + +++e ed+++ ++ + ++k ++ ++++++++ +++ ++    + k +\n",
            "  XP_004350745.1   3 SKINHLIKERQQHPTQTRNIDHYTSIIQINRDKEMSNNNNNnNKREVEDQESLENQSSRKKVKEDESSTSTTSTTTTNKK----NVKKQ 87 \n",
            "                     57889999998888888888888887766666666555554045556777777766666666666666666666655544....22222 PP\n",
            "\n",
            ">> XP_004366514.1  uncharacterized protein DFA_04104 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.3   5.9     0.092        13     134     179 ..     158     198 ..     110     254 .. 0.54\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.3 bits;  conditional E-value: 0.092\n",
            "             Zip 134 eseekedhvaeadtdadskeelrngkqeksesekeqaseeedkekl 179\n",
            "                     +++e   +++e + +++++ee++n++q  +e+e+     ++d+++ \n",
            "  XP_004366514.1 158 DEREE--EEEEIQDSNQEEEEHHNSSQYDEEEEEDD---DDDESHN 198\n",
            "                     22221..11112112222222222222211111111...1111111 PP\n",
            "\n",
            ">> XP_004367319.1  uncharacterized protein DFA_07460 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.7   1.7     0.018       2.5     106     171 ..      15      80 ..       1     205 [. 0.63\n",
            "   2 ?    0.3   0.3       1.5   2.1e+02     126     172 ..    4935    4996 ..    4898    5064 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.7 bits;  conditional E-value: 0.018\n",
            "             Zip 106 hkektpkkasssssesaakksvsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas 171\n",
            "                     +ke+ + + ++ ++ s+++ s +++ +++s+  ++++++++ ++++  + +++++++++s+++ ++\n",
            "  XP_004367319.1  15 AKETDHADIQTTDTSSSSSPSSHSNASTDSSALQQQQQQQQQEDNNNNNNKKEEEKQEKSIQQDNQ 80 \n",
            "                     333333333333334444444444444444444444444444444445555555555555554433 PP\n",
            "\n",
            "  == domain 2  score: 0.3 bits;  conditional E-value: 1.5\n",
            "             Zip  126 svsndkveeseekedhvaeadtdadskeelrngkqeksesekeqas.................e 172 \n",
            "                        + +k++e+ee e  +++++ d++ +e l  ++++ +++e+++++                 e\n",
            "  XP_004367319.1 4935 NDHTKKLKEKEEEE--KTKQQLDESDEEVLTIEEKQGEGEEEQEKKnpelnfkyydnnycfldE 4996\n",
            "                      33333444444433..333444444444444444444444444433445555555555555542 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (333 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      3525  (0.290482); expected 242.7 (0.02)\n",
            "Passed bias filter:                     3134  (0.258261); expected 242.7 (0.02)\n",
            "Passed Vit filter:                      1619  (0.133416); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                      1090  (0.0898228); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):              86  [number of targets reported over threshold]\n",
            "# CPU time: 12.66u 0.02s 00:00:12.68 Elapsed: 00:00:06.59\n",
            "# Mc/sec: 341.57\n",
            "//\n",
            "Query:       ZIP4_domain  [M=171]\n",
            "Accession:   PF18292.4\n",
            "Description: Zinc transporter ZIP4 domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "       0.07   12.4   0.0       0.07   12.4   0.0    3.0  3  XP_004361063.1  uncharacterized protein DFA_05344 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004361063.1  uncharacterized protein DFA_05344 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -4.5   1.8      0.86     1e+04      76     133 ..      51      83 ..      30     105 .. 0.52\n",
            "   2 ?   -2.4   2.9      0.19   2.4e+03      89     124 ..     265     303 ..     250     334 .. 0.55\n",
            "   3 ?   12.4   0.0   5.8e-06      0.07      90     150 ..     707     766 ..     687     775 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -4.5 bits;  conditional E-value: 0.86\n",
            "                     HHHHHHHHHHHH...........HSTTTHHHH.HHHHHHHHHHC---------B-HHH CS\n",
            "     ZIP4_domain  76 waseadhllyklyldsllslrqdedsealsqselerlLqriqqhyrvsnsqaCvdaeq 133\n",
            "                     ++s++++ l +l  +  l+++++e++e ++                          +q\n",
            "  XP_004361063.1  51 NKSTSSSSLISLLIKRRLEVEEKEEKESIK-------------------------HHQ 83 \n",
            "                     233333333333333333444444333333.........................333 PP\n",
            "\n",
            "  == domain 2  score: -2.4 bits;  conditional E-value: 0.19\n",
            "                     ..........HSTTTHHH...H.HHHHHHHHHHC----- CS\n",
            "     ZIP4_domain  89 ldsllslrqdedsealsq...selerlLqriqqhyrvsn 124\n",
            "                      ++l +++q+++++ l +   ++l +lL++ q++ +++n\n",
            "  XP_004361063.1 265 SENLSKFQQQQQQQPLHHastGNLHSLLNQSQNNHNHNN 303\n",
            "                     344555555555555544000445555555553333333 PP\n",
            "\n",
            "  == domain 3  score: 12.4 bits;  conditional E-value: 5.8e-06\n",
            "                     .........HSTTTHHHH.HHHHHHHHH.HC---------B-HHHHHHHCSSTT----HHH. CS\n",
            "     ZIP4_domain  90 dsllslrqdedsealsqselerlLqriq.qhyrvsnsqaCvdaeqlleeaniagaesspgad 150\n",
            "                       ll +++d+++e l +++l++l ++iq +  + ++ q+C++++ + ++ +  + +ss ga+\n",
            "  XP_004361063.1 707 SHLLIYNNDKEEELLHKKSLSDL-TSIQlTPLTKEGYQNCFTVSFVYTKNET-SPSSSGGAS 766\n",
            "                     57999***************997.5666599999999***********9997.788888885 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (171 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       797  (0.0656778); expected 242.7 (0.02)\n",
            "Passed bias filter:                      259  (0.0213432); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        17  (0.00140091); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.15u 0.00s 00:00:00.15 Elapsed: 00:00:00.09\n",
            "# Mc/sec: 11929.48\n",
            "//\n",
            "Query:       ZipA_C  [M=128]\n",
            "Accession:   PF04354.16\n",
            "Description: ZipA, C-terminal FtsZ-binding domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (128 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       289  (0.0238154); expected 242.7 (0.02)\n",
            "Passed bias filter:                      192  (0.015822); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        11  (0.000906469); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.01s 00:00:00.12 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 14179.45\n",
            "//\n",
            "Query:       ZirS_C  [M=145]\n",
            "Accession:   PF16583.8\n",
            "Description: Zinc-regulated secreted antivirulence protein C-terminal domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.011   14.8   0.3      0.014   14.4   0.3    1.2  1  XP_004361966.1  uncharacterized protein DFA_06254 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004361966.1  uncharacterized protein DFA_06254 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   14.4   0.3   1.1e-06     0.014      55     111 ..      47     103 ..      26     112 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.4 bits;  conditional E-value: 1.1e-06\n",
            "                     ----B-TT-EEEEEEE-BBTTS-B------B--EE-S-SSSEEEE---EE-SSSSEE CS\n",
            "          ZirS_C  55 dkavlspgesatittlvkdvdgnpvnevminkavarenlkglwdigplkkevvpgky 111\n",
            "                        v s    at      d+d n    +++  +++++n     ++ p+ k+ +pg+ \n",
            "  XP_004361966.1  47 VVPVSSGSSGATTFNYNIDIDSNDFYGIYVGTTTSKNNKCNNNNFNPIDKGLGPGHV 103\n",
            "                     445555555677778889*************************************75 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (145 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       485  (0.039967); expected 242.7 (0.02)\n",
            "Passed bias filter:                      218  (0.0179646); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        12  (0.000988875); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 11509.06\n",
            "//\n",
            "Query:       Zmiz1_N  [M=99]\n",
            "Accession:   PF18028.4\n",
            "Description: Zmiz1 N-terminal tetratricopeptide repeat domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (99 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       371  (0.0305727); expected 242.7 (0.02)\n",
            "Passed bias filter:                      229  (0.018871); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        13  (0.00107128); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 10694.52\n",
            "//\n",
            "Query:       Zn-C2H2_12  [M=27]\n",
            "Accession:   PF18112.4\n",
            "Description: Autophagy receptor zinc finger-C2H2 domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    4.7e-08   32.7   5.3       0.32   10.8   0.1    3.9  3  XP_004350244.1  uncharacterized protein DFA_11301 [Cavenderia\n",
            "     0.0068   16.2   0.1      0.017   14.9   0.1    1.7  1  XP_004350243.1  uncharacterized protein DFA_11300 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "      0.025   14.4   7.4        3.4    7.6   0.4    4.1  3  XP_004350436.1  C2H2-type zinc finger-containing protein [Cav\n",
            "      0.027   14.3   0.1      0.073   12.9   0.1    1.7  1  XP_004350270.1  uncharacterized protein DFA_11327 [Cavenderia\n",
            "      0.083   12.7   0.1       0.22   11.3   0.1    1.7  1  XP_004360378.1  Zinc finger protein [Cavenderia fasciculata]\n",
            "       0.71    9.7   3.1       0.72    9.7   0.5    2.4  2  XP_004350269.1  uncharacterized protein DFA_11326 [Cavenderia\n",
            "        2.6    7.9   5.0       0.93    9.3   1.6    2.1  2  XP_004355866.1  uncharacterized protein DFA_08377 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004350244.1  uncharacterized protein DFA_11301 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   10.8   0.1   0.00018      0.32       3      14 ..      20      31 ..      19      36 .. 0.93\n",
            "   2 !   10.8   0.1   0.00018      0.32       3      14 ..    1154    1165 ..    1153    1170 .. 0.93\n",
            "   3 !   10.8   0.1   0.00018      0.32       3      14 ..    1791    1802 ..    1790    1807 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.8 bits;  conditional E-value: 0.00018\n",
            "      Zn-C2H2_12  3 CPlCsevFskev 14\n",
            "                    CPlC+e Fsk+v\n",
            "  XP_004350244.1 20 CPLCKEDFSKDV 31\n",
            "                    **********98 PP\n",
            "\n",
            "  == domain 2  score: 10.8 bits;  conditional E-value: 0.00018\n",
            "      Zn-C2H2_12    3 CPlCsevFskev 14  \n",
            "                      CPlC+e Fsk+v\n",
            "  XP_004350244.1 1154 CPLCKEDFSKDV 1165\n",
            "                      **********98 PP\n",
            "\n",
            "  == domain 3  score: 10.8 bits;  conditional E-value: 0.00018\n",
            "      Zn-C2H2_12    3 CPlCsevFskev 14  \n",
            "                      CPlC+e Fsk+v\n",
            "  XP_004350244.1 1791 CPLCKEDFSKDV 1802\n",
            "                      **********98 PP\n",
            "\n",
            ">> XP_004350243.1  uncharacterized protein DFA_11300 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   14.9   0.1   9.7e-06     0.017       3      14 ..      25      36 ..      24      43 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 14.9 bits;  conditional E-value: 9.7e-06\n",
            "      Zn-C2H2_12  3 CPlCsevFskev 14\n",
            "                    CPlC+e Fsk+v\n",
            "  XP_004350243.1 25 CPLCKEDFSKDV 36\n",
            "                    **********98 PP\n",
            "\n",
            ">> XP_004350436.1  C2H2-type zinc finger-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.6   0.4     0.002       3.4       3      22 ..     165     187 ..     164     187 .. 0.67\n",
            "   2 ?    0.7   0.2      0.27   4.7e+02       6      13 ..     219     226 ..     216     234 .. 0.78\n",
            "   3 ?    7.2   0.1    0.0025       4.4       3      16 ..     375     388 ..     374     394 .. 0.79\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.6 bits;  conditional E-value: 0.002\n",
            "      Zn-C2H2_12   3 CPlCsevFs...kevdqdeFeeH 22 \n",
            "                     CP+C e F+   + v+  +Fe H\n",
            "  XP_004350436.1 165 CPMCLEEFKgdrSAVSSHLFEQH 187\n",
            "                     ********833334666666666 PP\n",
            "\n",
            "  == domain 2  score: 0.7 bits;  conditional E-value: 0.27\n",
            "      Zn-C2H2_12   6 CsevFske 13 \n",
            "                     C+++F+++\n",
            "  XP_004350436.1 219 CEKIFKSQ 226\n",
            "                     99***975 PP\n",
            "\n",
            "  == domain 3  score: 7.2 bits;  conditional E-value: 0.0025\n",
            "      Zn-C2H2_12   3 CPlCsevFskevdq 16 \n",
            "                     CP+C+   + e d \n",
            "  XP_004350436.1 375 CPMCGLKHQDEADL 388\n",
            "                     *****988877665 PP\n",
            "\n",
            ">> XP_004350270.1  uncharacterized protein DFA_11327 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.9   0.1   4.2e-05     0.073       3      14 ..      20      31 ..      19      37 .. 0.92\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.9 bits;  conditional E-value: 4.2e-05\n",
            "      Zn-C2H2_12  3 CPlCsevFskev 14\n",
            "                    CPlC+e Fsk+v\n",
            "  XP_004350270.1 20 CPLCKEDFSKDV 31\n",
            "                    **********98 PP\n",
            "\n",
            ">> XP_004360378.1  Zinc finger protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.3   0.1   0.00013      0.22       1      11 [.    1201    1211 ..    1201    1220 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.3 bits;  conditional E-value: 0.00013\n",
            "      Zn-C2H2_12    1 KeCPlCsevFs 11  \n",
            "                      K+CPlCs++F \n",
            "  XP_004360378.1 1201 KTCPLCSKSFN 1211\n",
            "                      89********6 PP\n",
            "\n",
            ">> XP_004350269.1  uncharacterized protein DFA_11326 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    9.7   0.5   0.00041      0.72       1      13 [.      18      30 ..      18      33 .. 0.88\n",
            "   2 ?   -2.2   0.1       2.2   3.8e+03       4       9 ..     586     591 ..     585     591 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 9.7 bits;  conditional E-value: 0.00041\n",
            "      Zn-C2H2_12  1 KeCPlCsevFske 13\n",
            "                    K CPlC+e +s e\n",
            "  XP_004350269.1 18 KLCPLCKEDYSNE 30\n",
            "                    78********986 PP\n",
            "\n",
            "  == domain 2  score: -2.2 bits;  conditional E-value: 2.2\n",
            "      Zn-C2H2_12   4 PlCsev 9  \n",
            "                     P+Cs++\n",
            "  XP_004350269.1 586 PICSKI 591\n",
            "                     9***96 PP\n",
            "\n",
            ">> XP_004355866.1  uncharacterized protein DFA_08377 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -3.6   0.1       5.9     1e+04      19      26 ..     410     417 ..     407     417 .. 0.82\n",
            "   2 ?    9.3   1.6   0.00054      0.93       3      27 .]     535     556 ..     534     556 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -3.6 bits;  conditional E-value: 5.9\n",
            "      Zn-C2H2_12  19 FeeHVEsH 26 \n",
            "                     ++ H  +H\n",
            "  XP_004355866.1 410 LQQHLQQH 417\n",
            "                     78999999 PP\n",
            "\n",
            "  == domain 2  score: 9.3 bits;  conditional E-value: 0.00054\n",
            "      Zn-C2H2_12   3 CPlCsevFskevdqdeFeeHVEsHF 27 \n",
            "                     CP+C   F +e + +    HV sHF\n",
            "  XP_004355866.1 535 CPMCTRKFATEPEIN---QHVNSHF 556\n",
            "                     **********98765...7*****9 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (27 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       394  (0.0324681); expected 242.7 (0.02)\n",
            "Passed bias filter:                      293  (0.024145); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        53  (0.00436753); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000576844); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               7  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 3273.78\n",
            "//\n",
            "Query:       zn-ribbon_14  [M=32]\n",
            "Accession:   PF16503.8\n",
            "Description: Zinc-ribbon\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    9.3e-13   46.9   6.7    9.3e-13   46.9   6.7    3.3  5  XP_004356186.1  ATP-binding domain-containing protein 3 [Cave\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004356186.1  ATP-binding domain-containing protein 3 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -3.9   0.3      0.56   6.9e+03       3       7 ..      33      37 ..      32      38 .. 0.71\n",
            "   2 ?   -1.6   0.1      0.11   1.3e+03      12      18 ..      50      56 ..      48      57 .. 0.78\n",
            "   3 ?    0.3   0.1     0.028   3.4e+02       1       8 [.     166     173 ..     166     179 .. 0.83\n",
            "   4 ?   -3.6   0.0      0.47   5.7e+03      21      31 ..     275     285 ..     274     285 .. 0.84\n",
            "   5 !   46.9   6.7   7.7e-17   9.3e-13       1      32 []     308     339 ..     308     339 .. 0.96\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -3.9 bits;  conditional E-value: 0.56\n",
            "    zn-ribbon_14  3 CerCG 7 \n",
            "                    Ce C \n",
            "  XP_004356186.1 33 CELCK 37\n",
            "                    88884 PP\n",
            "\n",
            "  == domain 2  score: -1.6 bits;  conditional E-value: 0.11\n",
            "    zn-ribbon_14 12 qkiCkAC 18\n",
            "                    q iCk C\n",
            "  XP_004356186.1 50 QMICKEC 56\n",
            "                    7799988 PP\n",
            "\n",
            "  == domain 3  score: 0.3 bits;  conditional E-value: 0.028\n",
            "    zn-ribbon_14   1 gkCerCGy 8  \n",
            "                     ++C+ CG \n",
            "  XP_004356186.1 166 NNCTFCGV 173\n",
            "                     69*****7 PP\n",
            "\n",
            "  == domain 4  score: -3.6 bits;  conditional E-value: 0.47\n",
            "    zn-ribbon_14  21 LegLnkgrpki 31 \n",
            "                     L+ L++ rp i\n",
            "  XP_004356186.1 275 LKDLEAVRPSI 285\n",
            "                     999**999976 PP\n",
            "\n",
            "  == domain 5  score: 46.9 bits;  conditional E-value: 7.7e-17\n",
            "    zn-ribbon_14   1 gkCerCGylSSqkiCkACvLLegLnkgrpkia 32 \n",
            "                     ++C+ CGy++S++iC+AC +L++Ln+g++k+a\n",
            "  XP_004356186.1 308 RNCNLCGYICSNEICMACDFLKNLNSGKAKYA 339\n",
            "                     59****************************85 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (32 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       363  (0.0299135); expected 242.7 (0.02)\n",
            "Passed bias filter:                      211  (0.0173877); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        15  (0.00123609); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 3557.90\n",
            "//\n",
            "Query:       Zn-ribbon_8  [M=40]\n",
            "Accession:   PF09723.13\n",
            "Description: Zinc ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "      0.001   18.6   2.3      0.001   18.6   2.3    2.7  2  XP_004350709.1  uncharacterized protein DFA_11762 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "      0.025   14.2   1.5      0.025   14.2   1.5    5.8  7  XP_004359993.1  LIM domain protein [Cavenderia fasciculata]\n",
            "      0.036   13.7   0.5       0.11   12.2   0.5    1.8  1  XP_004360556.1  MCM family protein [Cavenderia fasciculata]\n",
            "      0.082   12.5   0.4       0.15   11.7   0.4    1.4  1  XP_004358406.1  ubiquitin [Cavenderia fasciculata]\n",
            "        0.1   12.2  13.0        2.8    7.6   0.7    4.5  4  XP_004356292.1  uncharacterized protein DFA_08808 [Cavenderia\n",
            "       0.17   11.5   0.6       0.34   10.5   0.6    1.4  1  XP_004362968.1  putative GATA-binding transcription factor [C\n",
            "       0.24   11.0   1.8        1.1    9.0   0.2    2.4  2  XP_004362557.1  leucyl-tRNA synthetase [Cavenderia fasciculat\n",
            "       0.48   10.1   3.4         19    4.9   0.1    3.4  4  XP_004355856.1  ZPR1-type zinc finger-containing protein [Cav\n",
            "       0.63    9.7   8.5        1.9    8.2   1.3    3.4  3  XP_004350735.1  uncharacterized protein DFA_11790 [Cavenderia\n",
            "        1.3    8.7   8.5          8    6.2   4.6    2.2  2  XP_004356125.1  uncharacterized protein DFA_08637 [Cavenderia\n",
            "        1.8    8.2   6.7       0.44   10.2   0.7    2.6  2  XP_004350066.1  uncharacterized protein DFA_11123 [Cavenderia\n",
            "          4    7.1   7.5        8.2    6.1   7.5    1.5  1  XP_004366572.1  Putative transposase [Cavenderia fasciculata]\n",
            "        4.2    7.0   7.8        2.9    7.5   0.5    3.0  2  XP_004355683.1  uncharacterized protein DFA_08186 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004350709.1  uncharacterized protein DFA_11762 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.7   1.7       2.5   2.3e+03       7      34 ..     278     284 ..     271     291 .. 0.70\n",
            "   2 !   18.6   2.3   1.1e-06     0.001       6      33 ..     299     327 ..     298     330 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.7 bits;  conditional E-value: 2.5\n",
            "     Zn-ribbon_8   7 rCeeCgheFevlqkmsdapltaCPeCgs 34 \n",
            "                                           C eC++\n",
            "  XP_004350709.1 278 D---------------------CSECHA 284\n",
            "                     4.....................555554 PP\n",
            "\n",
            "  == domain 2  score: 18.6 bits;  conditional E-value: 1.1e-06\n",
            "     Zn-ribbon_8   6 YrCeeCgheFevlqkms.dapltaCPeCg 33 \n",
            "                     + C++C ++F +l  +  d++++ CP+C \n",
            "  XP_004350709.1 299 FACKKCSKVFRKLINAEfDESDEYCPHCD 327\n",
            "                     89***************9999999****6 PP\n",
            "\n",
            ">> XP_004359993.1  LIM domain protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.5   0.2      0.51   4.8e+02      29      34 ..       8      13 ..       2      19 .. 0.64\n",
            "   2 ?    1.7   0.8      0.21   1.9e+02       8      32 ..       8      35 ..       6      35 .. 0.62\n",
            "   3 ?    2.9   6.7      0.09        84       8      32 ..      35      59 ..      33      79 .. 0.74\n",
            "   4 ?   -0.5   1.7         1   9.6e+02       7      16 ..      66      75 ..      66      93 .. 0.52\n",
            "   5 ?   14.2   1.5   2.7e-05     0.025       6      38 ..      92     123 ..      86     125 .. 0.78\n",
            "   6 ?   -2.8   1.1       5.4   5.1e+03       6      12 ..     137     144 ..     135     148 .. 0.75\n",
            "   7 ?    4.1   1.8     0.037        34       6      34 ..     164     195 ..     163     196 .] 0.76\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.5 bits;  conditional E-value: 0.51\n",
            "     Zn-ribbon_8 29 CPeCgs 34\n",
            "                    C++C++\n",
            "  XP_004359993.1  8 CKRCQK 13\n",
            "                    666654 PP\n",
            "\n",
            "  == domain 2  score: 1.7 bits;  conditional E-value: 0.21\n",
            "     Zn-ribbon_8  8 CeeCgheFe...vlqkmsdapltaCPeC 32\n",
            "                    C++C++  +    ++ m+ +   +C  C\n",
            "  XP_004359993.1  8 CKRCQKNIDsgeSIHAMDADWHSDCWGC 35\n",
            "                    9999999885555555553334556555 PP\n",
            "\n",
            "  == domain 3  score: 2.9 bits;  conditional E-value: 0.09\n",
            "     Zn-ribbon_8  8 CeeCgheFevlqkmsdapltaCPeC 32\n",
            "                    C eC++ Fe     +   l+ C+ C\n",
            "  XP_004359993.1 35 CAECHTAFETQSFYKVQGLPICKNC 59\n",
            "                    *********5444445556678887 PP\n",
            "\n",
            "  == domain 4  score: -0.5 bits;  conditional E-value: 1\n",
            "     Zn-ribbon_8  7 rCeeCgheFe 16\n",
            "                    +C++C +   \n",
            "  XP_004359993.1 66 KCKKCSQPIA 75\n",
            "                    5777775544 PP\n",
            "\n",
            "  == domain 5  score: 14.2 bits;  conditional E-value: 2.7e-05\n",
            "     Zn-ribbon_8   6 YrCeeCgheFevlqkmsdapltaCPeCgsevrr 38 \n",
            "                     + C +C++  ++   ++d  ++ CP+C+ +++ \n",
            "  XP_004359993.1  92 FTCSNCNKNIDANFFIKD-TNILCPQCNDQLKE 123\n",
            "                     66*********8876665.4566****988775 PP\n",
            "\n",
            "  == domain 6  score: -2.8 bits;  conditional E-value: 5.4\n",
            "     Zn-ribbon_8   6 YrCee.Cg 12 \n",
            "                     Y C++ C+\n",
            "  XP_004359993.1 137 YVCKKpCN 144\n",
            "                     78887776 PP\n",
            "\n",
            "  == domain 7  score: 4.1 bits;  conditional E-value: 0.037\n",
            "     Zn-ribbon_8   6 YrCeeCgheFe.vlqkms.daplta.CPeCgs 34 \n",
            "                     ++C +C++e   ++ + s  +++   C +C+s\n",
            "  XP_004359993.1 164 FKCVKCHNEIKgEFYTLSdVKSKNYvCMACNS 195\n",
            "                     89*****9999977777754455666999986 PP\n",
            "\n",
            ">> XP_004360556.1  MCM family protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.2   0.5   0.00011      0.11       5      30 ..     211     240 ..     209     245 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.2 bits;  conditional E-value: 0.00011\n",
            "     Zn-ribbon_8   5 eYrCeeCgheFevlqkms.dap...lta.CP 30 \n",
            "                     e+ C +Cgh+F++   +  + +   l++ CP\n",
            "  XP_004360556.1 211 EFACLKCGHRFDAQIDFEqQ-NnfsLPKkCP 240\n",
            "                     8***********66555533.2355666688 PP\n",
            "\n",
            ">> XP_004358406.1  ubiquitin [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.7   0.4   0.00016      0.15       7      39 ..     127     158 ..     113     158 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.7 bits;  conditional E-value: 0.00016\n",
            "     Zn-ribbon_8   7 rCeeCgheFevlqkmsdaplta.CPeCgsevrrv 39 \n",
            "                     +C++Cg      q    +p  + C +C++++ +v\n",
            "  XP_004358406.1 127 ECTNCGAGVFMAQH--ANPARQyCGKCHTTLVKV 158\n",
            "                     6*****88776664..4455555*****988875 PP\n",
            "\n",
            ">> XP_004356292.1  uncharacterized protein DFA_08808 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.2   0.4      0.81   7.6e+02      26      34 ..     183     192 ..     168     196 .. 0.64\n",
            "   2 ?    6.2   0.1    0.0081       7.5      25      38 ..     285     299 ..     277     300 .. 0.76\n",
            "   3 ?    7.6   0.7    0.0031       2.8       8      38 ..     307     346 ..     304     348 .. 0.78\n",
            "   4 ?    3.6   0.0     0.054        50       8      23 ..     355     369 ..     352     374 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.2 bits;  conditional E-value: 0.81\n",
            "     Zn-ribbon_8  26 lta.CPeCgs 34 \n",
            "                      t+ C +C s\n",
            "  XP_004356292.1 183 TTKdCSSCSS 192\n",
            "                     4455999987 PP\n",
            "\n",
            "  == domain 2  score: 6.2 bits;  conditional E-value: 0.0081\n",
            "     Zn-ribbon_8  25 pltaCPeCgs.evrr 38 \n",
            "                     ++++CP C+  ++rr\n",
            "  XP_004356292.1 285 DKPTCPNCHViATRR 299\n",
            "                     4678*****988887 PP\n",
            "\n",
            "  == domain 3  score: 7.6 bits;  conditional E-value: 0.0031\n",
            "     Zn-ribbon_8   8 CeeCgheFevlqkms..dap.......ltaCPeCgsevrr 38 \n",
            "                     C++C + F++++k +   +p        ++C +C ++++r\n",
            "  XP_004356292.1 307 CKNCVKSFTPFRKGKkyNDPnssnpslAPQCGRCSTQATR 346\n",
            "                     **********999986444489999955569999886655 PP\n",
            "\n",
            "  == domain 4  score: 3.6 bits;  conditional E-value: 0.054\n",
            "     Zn-ribbon_8   8 CeeCgheFevlqkmsd 23 \n",
            "                     C++C++ F++ q +s+\n",
            "  XP_004356292.1 355 CKSCKKPFTPNQ-FSE 369\n",
            "                     *********999.555 PP\n",
            "\n",
            ">> XP_004362968.1  putative GATA-binding transcription factor [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.5   0.6   0.00037      0.34       8      36 ..     669     698 ..     667     702 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.5 bits;  conditional E-value: 0.00037\n",
            "     Zn-ribbon_8   8 CeeCgheFe.vlqkmsdapltaCPeCgsev 36 \n",
            "                     C+ Cg++ + ++++  d p t C +Cg + \n",
            "  XP_004362968.1 669 CHTCGTKSTpEWRRGPDGPATLCNACGLAF 698\n",
            "                     *********7777766***999****9754 PP\n",
            "\n",
            ">> XP_004362557.1  leucyl-tRNA synthetase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.2   0.1       0.6   5.6e+02       7      34 ..     475     481 ..     470     485 .. 0.60\n",
            "   2 ?    9.0   0.2    0.0011       1.1      23      38 ..     511     526 ..     501     527 .. 0.76\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.2 bits;  conditional E-value: 0.6\n",
            "     Zn-ribbon_8   7 rCeeCgheFevlqkmsdapltaCPeCgs 34 \n",
            "                     +C++Cg                     +\n",
            "  XP_004362557.1 475 HCNSCG---------------------E 481\n",
            "                     455555.....................4 PP\n",
            "\n",
            "  == domain 2  score: 9.0 bits;  conditional E-value: 0.0011\n",
            "     Zn-ribbon_8  23 dapltaCPeCgsevrr 38 \n",
            "                     d ++ +CP+Cg++++r\n",
            "  XP_004362557.1 511 DWKNCKCPKCGGNAQR 526\n",
            "                     445556*******998 PP\n",
            "\n",
            ">> XP_004355856.1  ZPR1-type zinc finger-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.8   0.0       2.6   2.4e+03      29      37 ..      33      42 ..      14      45 .. 0.67\n",
            "   2 ?    4.9   0.1     0.021        19       6      20 ..      60      74 ..      58      80 .. 0.85\n",
            "   3 ?   -1.6   0.0       2.2   2.1e+03      26      35 ..     266     277 ..     259     282 .. 0.66\n",
            "   4 ?    3.7   0.0      0.05        47       6      19 ..     297     310 ..     295     317 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.8 bits;  conditional E-value: 2.6\n",
            "     Zn-ribbon_8 29 CPeCgs.evr 37\n",
            "                    C  C++   +\n",
            "  XP_004355856.1 33 CMNCHEdGIT 42\n",
            "                    8888775555 PP\n",
            "\n",
            "  == domain 2  score: 4.9 bits;  conditional E-value: 0.021\n",
            "     Zn-ribbon_8  6 YrCeeCgheFevlqk 20\n",
            "                    ++Ce Cg +  ++q+\n",
            "  XP_004355856.1 60 FECEHCGFKSSEVQS 74\n",
            "                    99*****99888886 PP\n",
            "\n",
            "  == domain 3  score: -1.6 bits;  conditional E-value: 2.2\n",
            "     Zn-ribbon_8  26 lta.CPeCgs.e 35 \n",
            "                     l++ C  Cg+  \n",
            "  XP_004355856.1 266 LPQsCVFCGEmG 277\n",
            "                     566688888755 PP\n",
            "\n",
            "  == domain 4  score: 3.7 bits;  conditional E-value: 0.05\n",
            "     Zn-ribbon_8   6 YrCeeCgheFevlq 19 \n",
            "                     + Ce+Cg +  ++ \n",
            "  XP_004355856.1 297 FSCENCGYKSNEIK 310\n",
            "                     89*****8877765 PP\n",
            "\n",
            ">> XP_004350735.1  uncharacterized protein DFA_11790 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.4   0.0      0.13   1.2e+02      21      36 ..      76      92 ..      69      96 .. 0.70\n",
            "   2 ?    8.2   1.3     0.002       1.9       8      32 ..     103     125 ..      99     132 .. 0.81\n",
            "   3 ?    1.6   0.4      0.22   2.1e+02      27      35 ..     137     146 ..     127     150 .. 0.69\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.4 bits;  conditional E-value: 0.13\n",
            "     Zn-ribbon_8 21 msdapltaCPeCgs.ev 36\n",
            "                     s ++++ C eCg  + \n",
            "  XP_004350735.1 76 TSTPNEKICIECGDsKD 92\n",
            "                    55666667****98665 PP\n",
            "\n",
            "  == domain 2  score: 8.2 bits;  conditional E-value: 0.002\n",
            "     Zn-ribbon_8   8 CeeCgheFevlqkmsdaplta.CPeC 32 \n",
            "                     C++ +++Fe++++++   +++ C  C\n",
            "  XP_004350735.1 103 CKDSKERFEAFHQSD---KITkCNTC 125\n",
            "                     9**********9755...44449998 PP\n",
            "\n",
            "  == domain 3  score: 1.6 bits;  conditional E-value: 0.22\n",
            "     Zn-ribbon_8  27 ta.CPeCgse 35 \n",
            "                     t+ C++C  +\n",
            "  XP_004350735.1 137 TTqCKDCRNT 146\n",
            "                     4449999654 PP\n",
            "\n",
            ">> XP_004356125.1  uncharacterized protein DFA_08637 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    4.6   0.1     0.027        25      27      36 ..       7      16 ..       1      17 [. 0.77\n",
            "   2 ?    6.2   4.6    0.0085         8       7      34 ..      36      63 ..      29      66 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 4.6 bits;  conditional E-value: 0.027\n",
            "     Zn-ribbon_8 27 taCPeCgsev 36\n",
            "                    +aCP C ++v\n",
            "  XP_004356125.1  7 EACPVCEKKV 16\n",
            "                    67****8765 PP\n",
            "\n",
            "  == domain 2  score: 6.2 bits;  conditional E-value: 0.0085\n",
            "     Zn-ribbon_8  7 rCeeCgheFevlqkmsdapltaCPeCgs 34\n",
            "                     C++C++  +      +++++ C++C +\n",
            "  XP_004356125.1 36 VCTACKKSLDSTNLCEKDSKPYCKACYG 63\n",
            "                    4*********888777777788****76 PP\n",
            "\n",
            ">> XP_004350066.1  uncharacterized protein DFA_11123 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.2   0.7      0.86     8e+02       6      34 ..     740     747 ..     732     762 .. 0.73\n",
            "   2 ?   10.2   0.7   0.00047      0.44       7      34 ..     780     806 ..     779     809 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.2 bits;  conditional E-value: 0.86\n",
            "     Zn-ribbon_8   6 YrCeeCgheFevlqkmsdapltaCPeCgs 34 \n",
            "                     Y                      CP C++\n",
            "  XP_004350066.1 740 YV---------------------CPTCKT 747\n",
            "                     44.....................666644 PP\n",
            "\n",
            "  == domain 2  score: 10.2 bits;  conditional E-value: 0.00047\n",
            "     Zn-ribbon_8   7 rCeeCgheFevlqkmsdapltaCPeCgs 34 \n",
            "                     +C+ C+ ++   q+ +  p+ +CP+C++\n",
            "  XP_004350066.1 780 QCNFCQATYSISQP-DIIPKSTCPSCQH 806\n",
            "                     6*********9996.6667899****85 PP\n",
            "\n",
            ">> XP_004366572.1  Putative transposase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.1   7.5    0.0088       8.2       5      34 ..     165     198 ..     165     202 .. 0.77\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.1 bits;  conditional E-value: 0.0088\n",
            "     Zn-ribbon_8   5 eYrCeeCgheFe....vlqkms.daplta.CPeCgs 34 \n",
            "                     e++C++C+ +F+    +l+    + ++   C++C++\n",
            "  XP_004366572.1 165 EFKCKSCHYQFDqgetRLV--CvECKDLSlCKDCHQ 198\n",
            "                     8***********9966444..446667777****98 PP\n",
            "\n",
            ">> XP_004355683.1  uncharacterized protein DFA_08186 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.5   0.5    0.0031       2.9       8      38 ..      38      73 ..      35      75 .. 0.74\n",
            "   2 ?    2.0   1.5      0.17   1.6e+02       8      24 ..      89     105 ..      75     109 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.5 bits;  conditional E-value: 0.0031\n",
            "     Zn-ribbon_8  8 CeeCgheFe...vlqkmsdap..lta.CPeCgs.evrr 38\n",
            "                    C++C+ + e   + +k +  p  +   C +C + +v+r\n",
            "  XP_004355683.1 38 CQDCHDTIEwrkKYRKYK--PlsMAGkCAHCTQrKVTR 73\n",
            "                    *********665444444..2446666******99999 PP\n",
            "\n",
            "  == domain 2  score: 2.0 bits;  conditional E-value: 0.17\n",
            "     Zn-ribbon_8   8 CeeCgheFevlqkmsda 24 \n",
            "                     C +C++  e + +++d+\n",
            "  XP_004355683.1  89 CAKCHKAKEIVAEITDP 105\n",
            "                     77777777777766665 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (40 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       630  (0.0519159); expected 242.7 (0.02)\n",
            "Passed bias filter:                      426  (0.0351051); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        83  (0.00683972); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        20  (0.00164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):              13  [number of targets reported over threshold]\n",
            "# CPU time: 0.14u 0.01s 00:00:00.15 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 3321.09\n",
            "//\n",
            "Query:       ZNRF_3_ecto  [M=105]\n",
            "Accession:   PF18212.4\n",
            "Description: ZNRF-3 Ectodomain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.075   12.5   0.0       0.21   11.1   0.0    1.6  1  XP_004362966.1  3-hydroxyacyl-CoA dehydrogenase type II [Cave\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004362966.1  3-hydroxyacyl-CoA dehydrogenase type II [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.1   0.0   1.7e-05      0.21      45      68 ..      25      48 ..      17      59 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.1 bits;  conditional E-value: 1.7e-05\n",
            "     ZNRF_3_ecto 45 alerGatavifdisddaeaadqlr 68\n",
            "                      erGa+ vifd+ + +e  ++l \n",
            "  XP_004362966.1 25 LHERGANVVIFDVNNAPELEKELG 48\n",
            "                    579**************9999885 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (105 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       217  (0.0178822); expected 242.7 (0.02)\n",
            "Passed bias filter:                      163  (0.0134322); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        10  (0.000824063); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 12408.52\n",
            "//\n",
            "Query:       ZnuA  [M=247]\n",
            "Accession:   PF01297.20\n",
            "Description: Zinc-uptake complex component A periplasmic\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.028   13.5  10.4      0.051   12.6   0.8    2.8  2  XP_004355084.1  Lis-interacting protein [Cavenderia fascicula\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004355084.1  Lis-interacting protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.6   0.8   4.2e-06     0.051     107     150 ..      26      69 ..      17      72 .. 0.88\n",
            "   2 ?    3.7   3.4    0.0023        28     112     192 ..      70     150 ..      68     188 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.6 bits;  conditional E-value: 4.2e-06\n",
            "                     GGHHHHHHHHHHHHHHHHHHHHHHHHHHTTSSTCEGEEEESSST CS\n",
            "                     xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx RF\n",
            "            ZnuA 107 enaeayeanaaaykekLeeldaeireelasipekkrklvtthda 150\n",
            "                     ++ +++e+n+++y+e  ++l++e+++el ++++k+  lv++h  \n",
            "  XP_004355084.1  26 QEVDDLEQNFNEYQEFSKQLEEEMETELRACEKKHTDLVSQHAR 69 \n",
            "                     455789************************************75 PP\n",
            "\n",
            "  == domain 2  score: 3.7 bits;  conditional E-value: 0.0023\n",
            "                     HHHHHHHHHHHHHHHHHHHHHHHTTSSTCEGEEEESSSTTHHHHHHTT-EEEEE..ESSSSSSSSSHHHHHHHHHHHHHTT-S CS\n",
            "                     xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx..xxxxxxxxxxxxxxxxxxxxxxxxxxx RF\n",
            "            ZnuA 112 yeanaaaykekLeeldaeireelasipekkrklvtthdafgylakaygleqaal..sgvstesepsaadiaklieeikeegvk 192\n",
            "                     +++ +++ ++kL+ + +e  + +as++++ +kl  +h+  + l+++  leq     +  +  s++s++d+++  +++ ee+v \n",
            "  XP_004355084.1  70 LKNDYENIQDKLNGVSRESSSLIASLQDEVDKL--KHFKQSLLGDKRKLEQENDtlERRERASSASVQDLSEKLDRVMEENVW 150\n",
            "                     56789999*****************99998665..888889999999999776622445667788888888888888888875 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (247 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       803  (0.0661722); expected 242.7 (0.02)\n",
            "Passed bias filter:                      286  (0.0235682); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        25  (0.00206016); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.20u 0.00s 00:00:00.20 Elapsed: 00:00:00.11\n",
            "# Mc/sec: 14399.73\n",
            "//\n",
            "Query:       Zn_clus  [M=40]\n",
            "Accession:   PF00172.21\n",
            "Description: Fungal Zn(2)-Cys(6) binuclear cluster domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    2.8e-07   30.0  12.8    8.2e-07   28.5  12.8    1.9  1  XP_004355461.1  transcriptional regulatory protein [Cavenderi\n",
            "    0.00012   21.6  10.9    0.00028   20.4  10.9    1.7  1  XP_004358441.1  transcriptional regulatory protein [Cavenderi\n",
            "    0.00055   19.5   4.5     0.0009   18.8   4.5    1.3  1  XP_004360064.1  uncharacterized protein DFA_04331 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004355461.1  transcriptional regulatory protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   28.5  12.8     2e-10   8.2e-07       1      38 [.      34      72 ..      34      73 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 28.5 bits;  conditional E-value: 2e-10\n",
            "                    SS-HHHHHHTS--STSSSS-HHHHHTT..HHHCCCCTCSC CS\n",
            "         Zn_clus  1 haCdnCrkkKvKCdakkPaCsnCkkln..leCtfysessk 38\n",
            "                    +aC nCr+ Kv Cd+++P C++C k++   +C + ++++k\n",
            "  XP_004355461.1 34 RACVNCRQSKVACDQQRP-CTRCTKHGieDKCHDVPRKTK 72\n",
            "                    69****************.********999**99888875 PP\n",
            "\n",
            ">> XP_004358441.1  transcriptional regulatory protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   20.4  10.9   6.9e-08   0.00028       1      39 [.      10      49 ..      10      50 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 20.4 bits;  conditional E-value: 6.9e-08\n",
            "                    SS-HHHHHHTS--STSSSS-HHHHHTT..HHHCCCCTCSCE CS\n",
            "         Zn_clus  1 haCdnCrkkKvKCdakkPaCsnCkkln..leCtfysesskr 39\n",
            "                    +aC nCr  Kv Cd  +P C++C+kl     C + ++++++\n",
            "  XP_004358441.1 10 RACLNCRNSKVACDPSRP-CTRCVKLDmgHSCYDLPRKKRE 49\n",
            "                    69****************.******9888999988888875 PP\n",
            "\n",
            ">> XP_004360064.1  uncharacterized protein DFA_04331 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   18.8   4.5   2.2e-07    0.0009       1      37 [.      13      49 ..      13      52 .. 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 18.8 bits;  conditional E-value: 2.2e-07\n",
            "                    SS-HHHHHHTS--STSSSS-HHHHHTT..HHHCCCCTCS CS\n",
            "         Zn_clus  1 haCdnCrkkKvKCdakkPaCsnCkkln..leCtfysess 37\n",
            "                    ++C  C k +  Cd+++P C+nC+ ++    C  y+e+ \n",
            "  XP_004360064.1 13 RSCFLCQKEHLECDEQTP-CGNCALKGtpQMCF-YTERL 49\n",
            "                    59****************.******99666787.77665 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (40 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       837  (0.068974); expected 242.7 (0.02)\n",
            "Passed bias filter:                      295  (0.0243098); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        14  (0.00115369); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         3  (0.000247219); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 4713.73\n",
            "//\n",
            "Query:       Zn_dep_PLPC  [M=179]\n",
            "Accession:   PF00882.21\n",
            "Description: Zinc dependent phospholipase C\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    1.3e-21   76.8   0.2    3.2e-21   75.6   0.2    1.6  1  XP_004360529.1  glycosylphosphatidylinositol phospholipase D \n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004360529.1  glycosylphosphatidylinositol phospholipase D [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   75.6   0.2   2.6e-25   3.2e-21       1     151 [.      46     206 ..      46     235 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 75.6 bits;  conditional E-value: 2.6e-25\n",
            "                     HHHHHHHHHHH..HH....HHH-..HHHTHHHHHHHHHGGGG..-TTGGG--BTTTB-HHHHS-HHHHHHHHHHHHHHHHHC........T-HHH CS\n",
            "     Zn_dep_PLPC   1 tHltlashala..vlpalkkrrk..lklnkaalllGsvaPDleylftsshsykthtagllgedhhweetgkfllllikklrely...ksegslkl 88 \n",
            "                     tH  +a++a +  +++++++  +  ++++++ +++G+ +PD +y++ ++ ++ + +        hw   ++fl+++  +l+++y   +se   kl\n",
            "  XP_004360529.1  46 THNEVARRAFNfsSFQDYPQF-QtlIPQYLSSFQAGASFPDWGYDC-GNSGNESES-------AHW---PPFLRNATVYLKKTYpqpWSEAGAKL 128\n",
            "                     799999999997755545443.2589******************88.888888888.......9**...**************9***9999**** PP\n",
            "\n",
            "                     HHHHHHHHHHHHHHHTSH..............HTT--TTT-THHHHHHHHHHHHGGGG.......--S--SS-TTSHH CS\n",
            "     Zn_dep_PLPC  89 aafllGvlsHyvaDvftH.......g..stlgeneipsyrllqHgsyelgldflllkyv......krdaaeaktllls 151\n",
            "                     a fl+GvlsH+vaD+++H       g  + ++ + ++s++  +H++ ++g++++l +++      +r+  ++++ +++\n",
            "  XP_004360529.1 129 AVFLMGVLSHQVADISWHsiggiheGliRAMAGQDFNSDYSIAHTNADVGGEYVLAYNYdlswlvDRWDVPISDVKNI 206\n",
            "                     *********************9988356999999******************99988889977773333333333222 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (179 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       519  (0.0427689); expected 242.7 (0.02)\n",
            "Passed bias filter:                      237  (0.0195303); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        16  (0.0013185); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.14u 0.00s 00:00:00.14 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 13941.18\n",
            "//\n",
            "Query:       Zn_finger_prot  [M=268]\n",
            "Accession:   PF19242.2\n",
            "Description: Putative zinc finger protein\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.043   13.0   0.5      0.046   12.9   0.5    1.1  1  XP_004351250.1  uncharacterized protein DFA_11002 [Cavenderia\n",
            "       0.88    8.7   7.5        1.7    7.8   7.5    1.5  1  XP_004354380.1  uncharacterized protein DFA_10480 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004351250.1  uncharacterized protein DFA_11002 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   12.9   0.5   7.6e-06     0.046     147     198 ..      43      94 ..      27      97 .] 0.86\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 12.9 bits;  conditional E-value: 7.6e-06\n",
            "  Zn_finger_prot 147 raaatplnsktklnvakPnkvkqekqvvkrdevqkPtrlclsvlkktkcffg 198\n",
            "                     +  ++  n+++++ +   n  +q++q  + +  q+P  l ls+l +t+ f  \n",
            "  XP_004351250.1  43 HNHNHRVNQQHQQHQIITNSQQQQQQQQQLQIQQRPIFLPLSILHNTRPFSI 94 \n",
            "                     556778889999999999999999999899999**************99865 PP\n",
            "\n",
            ">> XP_004354380.1  uncharacterized protein DFA_10480 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    7.8   7.5   0.00027       1.7      25      83 ..     368     427 ..     344     477 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 7.8 bits;  conditional E-value: 0.00027\n",
            "  Zn_finger_prot  25 yesddeseeedfydryepeeeefdddeyqlfvdeedd.erkddPkkitkvikPisllk.sP 83 \n",
            "                      e +de eeed y+ ye + +e++++e +  v  ed+ e +++   ++ ++k +sl++  P\n",
            "  XP_004354380.1 368 DEDEDEDEEEDEYE-YEYDGDEYEEEEEEELVVGEDQvEIRNNLLLLNEIVKVVSLSNiYP 427\n",
            "                     34556677777776.4547788888888887777777577777777799999998887434 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (268 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      2600  (0.214256); expected 242.7 (0.02)\n",
            "Passed bias filter:                      337  (0.0277709); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        51  (0.00420272); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         9  (0.000741656); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.32u 0.00s 00:00:00.32 Elapsed: 00:00:00.17\n",
            "# Mc/sec: 10401.57\n",
            "//\n",
            "Query:       Zn_peptidase  [M=290]\n",
            "Accession:   PF04228.16\n",
            "Description: Putative neutral zinc metallopeptidase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "       0.11   11.5   0.1       0.33    9.8   0.2    1.7  2  XP_004366821.1  uncharacterized protein DFA_02436 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004366821.1  uncharacterized protein DFA_02436 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.6   0.0     0.017   2.1e+02      45      92 ..     263     310 ..     259     320 .. 0.62\n",
            "   2 ?    9.8   0.2   2.7e-05      0.33       8     124 ..     283     402 ..     277     415 .. 0.72\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.6 bits;  conditional E-value: 0.017\n",
            "    Zn_peptidase  45 lvgllfGvdlssllglgqeespaeaaaqsqeaaaeeaedeaaefvsvi 92 \n",
            "                     l ++++ +d  s+ g ++++ +++ ++ +++ + ++   +  ef+++i\n",
            "  XP_004366821.1 263 LNSTYYSIDWKSIFGRANNSNNQNINNNQNNNNQNSEMVKVVEFARSI 310\n",
            "                     56789**********655555554444333333333334555666665 PP\n",
            "\n",
            "  == domain 2  score: 9.8 bits;  conditional E-value: 2.7e-05\n",
            "    Zn_peptidase   8 esdnvedrrassggedaGGgg...rlslggkgGllllilvlvgllfGvdlssllglgqeespaeaaaqsqeaaaeeaedeaaefvsviladteev 99 \n",
            "                     + +n+++ ++ +++++        ++s+++++      lvl+ l f   ++ +l l+   ++  a++q+q++++ +  ++ +  +++i  d + +\n",
            "  XP_004366821.1 283 NNQNINNNQNNNNQNSEMVKVvefARSISKRCDRFDKKLVLACLSFPSHPTYMLVLEMAAKNTTANSQQQQQQSLSPSNKGKLTTKSIKEDEKYM 377\n",
            "                     56677777777766663332235578888888888889999999999999998886555555555555555555553344444567777888889 PP\n",
            "\n",
            "    Zn_peptidase 100 Weqlfaklqrqyqkpklvlfsrvvr 124\n",
            "                     W++++ +++   q p + +f+  ++\n",
            "  XP_004366821.1 378 WHKYLIDHHSALQFPNISIFKYFIH 402\n",
            "                     ****************999987766 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (290 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1074  (0.0885043); expected 242.7 (0.02)\n",
            "Passed bias filter:                      493  (0.0406263); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        46  (0.00379069); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.28u 0.01s 00:00:00.29 Elapsed: 00:00:00.17\n",
            "# Mc/sec: 11371.05\n",
            "//\n",
            "Query:       Zn_peptidase_2  [M=218]\n",
            "Accession:   PF04298.15\n",
            "Description: Putative neutral zinc metallopeptidase\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.027   13.5   0.1      0.032   13.3   0.1    1.2  1  XP_004366310.1  uncharacterized protein DFA_03900 [Cavenderia\n",
            "       0.11   11.5   0.5       0.19   10.7   0.2    1.7  1  XP_004367187.1  uncharacterized protein DFA_07324 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004366310.1  uncharacterized protein DFA_03900 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.3   0.1   5.3e-06     0.032       2      70 ..       4      79 ..       3     104 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.3 bits;  conditional E-value: 5.3e-06\n",
            "  Zn_peptidase_2  2 lillivllllsllaqlkvkstfkkyskvrsss.......gltGaevArklldenglsdVkveeveGeltDhYdprk 70\n",
            "                    +i+l +++ll ++a ++ + ++  +++ rs+         +   e Arkll++n+l++V+v     + +  Y+p+ \n",
            "  XP_004366310.1  4 IIILAIIVLLGMVACIQAEDAYFAFRTHRSTLdfvfkltDVKKIEHARKLLSDNDLQEVHVMGRIRKTQKDYNPHY 79\n",
            "                    5777888888888888888888888877776422221114566789***************866557777799975 PP\n",
            "\n",
            ">> XP_004367187.1  uncharacterized protein DFA_07324 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.7   0.2   3.2e-05      0.19      83     102 ..     117     136 ..      98     217 .. 0.74\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.7 bits;  conditional E-value: 3.2e-05\n",
            "  Zn_peptidase_2  83 rsvaAvavAAHEvGHAiQha 102\n",
            "                      s+ A++v  HE+GHAi   \n",
            "  XP_004367187.1 117 LSLWAIGVSVHEFGHAIVAV 136\n",
            "                     5899************9655 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (218 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       440  (0.0362588); expected 242.7 (0.02)\n",
            "Passed bias filter:                      352  (0.029007); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        25  (0.00206016); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.17u 0.00s 00:00:00.17 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 16495.97\n",
            "//\n",
            "Query:       Zn_protease  [M=138]\n",
            "Accession:   PF05618.14\n",
            "Description: Putative ATP-dependant zinc protease\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (138 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       335  (0.0276061); expected 242.7 (0.02)\n",
            "Passed bias filter:                      192  (0.015822); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        12  (0.000988875); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 14852.97\n",
            "//\n",
            "Query:       Zn_ribbon_2  [M=82]\n",
            "Accession:   PF12674.10\n",
            "Description: Putative zinc ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.043   13.9   1.5       0.15   12.2   0.2    2.2  2  XP_004359533.1  cellular retinaldehyde-binding/triple functio\n",
            "       0.78    9.9   7.2       0.21   11.7   0.2    3.1  3  XP_004363093.1  uncharacterized protein DFA_03490 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004359533.1  cellular retinaldehyde-binding/triple function domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -0.0   0.1      0.16   9.7e+02      44      70 ..      16      38 ..       6      53 .. 0.49\n",
            "   2 ?   12.2   0.2   2.4e-05      0.15      12      76 ..     111     176 ..     104     182 .. 0.67\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -0.0 bits;  conditional E-value: 0.16\n",
            "     Zn_ribbon_2 44 leemiefcvkklkeekkelteeeakkl 70\n",
            "                    l+e+ + + k++ +    +t+eea+kl\n",
            "  XP_004359533.1 16 LQEVKDLVNKNMTD----ATKEEAEKL 38\n",
            "                    34444444444444....444444443 PP\n",
            "\n",
            "  == domain 2  score: 12.2 bits;  conditional E-value: 2.4e-05\n",
            "     Zn_ribbon_2  12 eeelgteadgsksedYCkyCykdGe.ftepdltleemiefcvkklkeekkelt.eeeakkllkkllp 76 \n",
            "                      ++ gt++d ++ + YC y ++ G+ + +p ++   +i  + ++  ++  +++  + + ++++k++p\n",
            "  XP_004359533.1 111 RKHDGTNRDIDEAQRYCVYVMESGKqMLKPGIETCTLIFDMSSFSTKNM-DYPlVKFMVDMFQKYYP 176\n",
            "                     5667******************996366777776666655555444443.33202335555555555 PP\n",
            "\n",
            ">> XP_004363093.1  uncharacterized protein DFA_03490 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -4.1   0.0         2   1.2e+04      17      29 ..      90     102 ..      89     107 .. 0.82\n",
            "   2 ?   11.7   0.2   3.4e-05      0.21       2      67 ..     461     525 ..     460     535 .. 0.72\n",
            "   3 ?    0.5   0.6      0.11   6.8e+02      17      48 ..    1050    1084 ..    1024    1093 .. 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -4.1 bits;  conditional E-value: 2\n",
            "     Zn_ribbon_2  17 teadgsksedYCk 29 \n",
            "                     t++ g ks  YCk\n",
            "  XP_004363093.1  90 TNSYGGKSFAYCK 102\n",
            "                     6788999999**8 PP\n",
            "\n",
            "  == domain 2  score: 11.7 bits;  conditional E-value: 3.4e-05\n",
            "     Zn_ribbon_2   2 CqsCGmPl...kdeeelgteadgsksedYCkyCykdGeftepdltleemiefcvkklkeekkelteeea 67 \n",
            "                     C++CG  l   ++e+ + + +dg++ + YC  C   G+f +   +l+++ ++  +k+ +  k+l++e++\n",
            "  XP_004363093.1 461 CKTCGRGLmdfEREKVILKRKDGERIQLYCAECR-SGTFLN---KLKNTSSHSKQKIYSFFKNLDSEDV 525\n",
            "                     *****7774436677789***************5.688887...5666666666665555555555555 PP\n",
            "\n",
            "  == domain 3  score: 0.5 bits;  conditional E-value: 0.11\n",
            "     Zn_ribbon_2   17 teadgsksedYCkyCyk...dGeftepdltleemi 48  \n",
            "                       e+d+  +++ C+ C k   +Ge++++ +++ e i\n",
            "  XP_004363093.1 1050 FEKDKAAKQQQCTKCDKqplHGEYVSKRIQMFENI 1084\n",
            "                      47788888888888887566688888777776655 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (82 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1031  (0.0849609); expected 242.7 (0.02)\n",
            "Passed bias filter:                      499  (0.0411207); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        42  (0.00346106); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.14u 0.00s 00:00:00.14 Elapsed: 00:00:00.07\n",
            "# Mc/sec: 7876.37\n",
            "//\n",
            "Query:       Zn_ribbon_recom  [M=58]\n",
            "Accession:   PF13408.9\n",
            "Description: Recombinase zinc beta ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "       0.48   10.4   5.6       0.39   10.7   3.2    2.2  1  XP_004357934.1  DNA topoisomerase III [Cavenderia fasciculata\n",
            "        1.2    9.1   3.5       0.67   10.0   0.3    2.3  2  XP_004354785.1  RNA polymerase I [Cavenderia fasciculata]\n",
            "          3    7.9   8.0        9.2    6.3   8.0    1.8  1  XP_004350226.1  uncharacterized protein DFA_11283 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004357934.1  DNA topoisomerase III [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.7   3.2   9.6e-05      0.39       7      37 ..     644     674 ..     640     703 .. 0.80\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.7 bits;  conditional E-value: 9.6e-05\n",
            "  Zn_ribbon_recom   7 rCgeCGspmtrkkrkgkrryYrCstyrrkgk 37 \n",
            "                      +Cg+C++ m  kk++g r    Cst++++ +\n",
            "   XP_004357934.1 644 KCGKCNNLMDLKKTNGDRKILNCSTCNTSYN 674\n",
            "                      7*************************99876 PP\n",
            "\n",
            ">> XP_004354785.1  RNA polymerase I [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -2.9   0.2       1.7   6.9e+03      25      44 ..     559     577 ..     552     583 .. 0.69\n",
            "   2 ?   10.0   0.3   0.00017      0.67       5      24 ..     946     965 ..     943     981 .. 0.67\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -2.9 bits;  conditional E-value: 1.7\n",
            "  Zn_ribbon_recom  25 ryYrCstyrrkgkkCsnrsi 44 \n",
            "                        Y+C+ +++ + + +++s+\n",
            "   XP_004354785.1 559 NMYQCQMAKQTM-GTPMHSY 577\n",
            "                      569999888777.7777744 PP\n",
            "\n",
            "  == domain 2  score: 10.0 bits;  conditional E-value: 0.00017\n",
            "  Zn_ribbon_recom   5 llrCgeCGspmtrkkrkgkr 24 \n",
            "                      +++C +CGs++t+ ++ +++\n",
            "   XP_004354785.1 946 MMVCRHCGSTLTVATKPSGQ 965\n",
            "                      69***********7766333 PP\n",
            "\n",
            ">> XP_004350226.1  uncharacterized protein DFA_11283 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    6.3   8.0    0.0023       9.2       6      43 ..     512     550 ..     509     557 .. 0.71\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 6.3 bits;  conditional E-value: 0.0023\n",
            "  Zn_ribbon_recom   6 lrCgeCGspmtrkkrkgkrryYrCstyrrkgk..kCsnrs 43 \n",
            "                      + Cg+C+ + + ++  gk+  ++C++++++ +  +C  +s\n",
            "   XP_004350226.1 512 VTCGICNGKISTNWG-GKKQVFHCNNCKEYYHngECVEQS 550\n",
            "                      78********99977.777666666666555433686555 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (58 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       727  (0.0599094); expected 242.7 (0.02)\n",
            "Passed bias filter:                      431  (0.0355171); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        43  (0.00354347); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         7  (0.000576844); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 4864.49\n",
            "//\n",
            "Query:       Zn_ribbon_SprT  [M=38]\n",
            "Accession:   PF17283.5\n",
            "Description: SprT-like zinc ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.048   13.1   0.1      0.048   13.1   0.1    6.1  5  XP_004359993.1  LIM domain protein [Cavenderia fasciculata]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004359993.1  LIM domain protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.4   0.3    0.0089   1.1e+02      26      32 ..       7      13 ..       6      16 .. 0.80\n",
            "   2 ?   -5.3   9.1         1   1.2e+04       7      33 ..      35      73 ..      29      77 .. 0.65\n",
            "   3 ?   13.1   0.1     4e-06     0.048       6      35 ..      93     122 ..      92     126 .. 0.89\n",
            "   4 ?    0.4   3.3     0.038   4.6e+02       7      29 ..     114     141 ..     113     147 .. 0.74\n",
            "   5 ?    0.6   2.7     0.033     4e+02       7      15 ..     166     174 ..     164     195 .. 0.62\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.4 bits;  conditional E-value: 0.0089\n",
            "  Zn_ribbon_SprT 26 rCrrCkg 32\n",
            "                    +C+rC+ \n",
            "  XP_004359993.1  7 ICKRCQK 13\n",
            "                    7999985 PP\n",
            "\n",
            "  == domain 2  score: -5.3 bits;  conditional E-value: 1\n",
            "  Zn_ribbon_SprT  7 CpsCgreytrhlrsrrhn............yrCrrCkgt 33\n",
            "                    C +C++ +  ++  +++               C++C ++\n",
            "  XP_004359993.1 35 CAECHTAFETQSFYKVQGlpickncylkkqSKCKKCSQP 73\n",
            "                    666666666666666655655556666666779999876 PP\n",
            "\n",
            "  == domain 3  score: 13.1 bits;  conditional E-value: 4e-06\n",
            "  Zn_ribbon_SprT   6 rCpsCgreytrhlrsrrhnyrCrrCkgtLk 35 \n",
            "                     +C +C +++++++ ++  n  C +C  +Lk\n",
            "  XP_004359993.1  93 TCSNCNKNIDANFFIKDTNILCPQCNDQLK 122\n",
            "                     7**************66699******8887 PP\n",
            "\n",
            "  == domain 4  score: 0.4 bits;  conditional E-value: 0.038\n",
            "  Zn_ribbon_SprT   7 CpsCgreytrhlrsrrhn.....yrCrr 29 \n",
            "                     Cp C ++ + + + ++ +     y+C++\n",
            "  XP_004359993.1 114 CPQCNDQLKEQTKKQIGKelgncYVCKK 141\n",
            "                     8888877777766677778888888886 PP\n",
            "\n",
            "  == domain 5  score: 0.6 bits;  conditional E-value: 0.033\n",
            "  Zn_ribbon_SprT   7 CpsCgreyt 15 \n",
            "                     C++C++e++\n",
            "  XP_004359993.1 166 CVKCHNEIK 174\n",
            "                     888887765 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (38 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       651  (0.0536465); expected 242.7 (0.02)\n",
            "Passed bias filter:                      322  (0.0265348); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        38  (0.00313144); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         3  (0.000247219); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.11u 0.00s 00:00:00.11 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 4155.08\n",
            "//\n",
            "Query:       Zn_Tnp_IS1  [M=35]\n",
            "Accession:   PF03811.16\n",
            "Description: InsA N-terminal domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.018   14.2   3.0      0.041   13.1   3.0    1.6  1  XP_004360866.1  histone deacetylase family protein [Cavenderi\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004360866.1  histone deacetylase family protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   13.1   3.0   3.4e-06     0.041       7      34 ..      32      60 ..      29      61 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 13.1 bits;  conditional E-value: 3.4e-06\n",
            "      Zn_Tnp_IS1  7 hCPrCsstd.VyRhGkstagHeRfrCrdC 34\n",
            "                    hCPr +      R+Gk  +g +R+rC+ C\n",
            "  XP_004360866.1 32 HCPRIEDVFySQRKGKTNKGAQRWRCKAC 60\n",
            "                    999987655567****************9 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (35 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       375  (0.0309023); expected 242.7 (0.02)\n",
            "Passed bias filter:                      222  (0.0182942); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        18  (0.00148331); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 4281.84\n",
            "//\n",
            "Query:       Zn_Tnp_IS1595  [M=46]\n",
            "Accession:   PF12760.10\n",
            "Description: Transposase zinc-ribbon domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.036   13.4   5.9       0.79    9.1   0.3    2.7  3  XP_004358905.1  RNA polymerase I subunit [Cavenderia fascicul\n",
            "       0.04   13.3  22.0      0.037   13.4   3.9    3.5  3  XP_004356292.1  uncharacterized protein DFA_08808 [Cavenderia\n",
            "      0.088   12.2   4.0       0.15   11.5   4.0    1.4  1  XP_004362940.1  S60 ribosomal protein L37A [Cavenderia fascic\n",
            "       0.16   11.4   1.6       0.36   10.2   1.6    1.5  1  XP_004362193.1  uncharacterized protein DFA_06492 [Cavenderia\n",
            "        1.9    7.9   6.7       0.56    9.6   0.3    2.3  2  XP_004355086.1  PH domain-containing protein [Cavenderia fasc\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004358905.1  RNA polymerase I subunit [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    5.9   1.0    0.0033       8.1      19      45 ..     163     188 ..     157     189 .. 0.83\n",
            "   2 ?    1.8   0.1     0.065   1.6e+02      17      27 ..     179     189 ..     175     195 .. 0.76\n",
            "   3 ?    9.1   0.3   0.00033      0.79      20      43 ..     227     259 ..     221     261 .. 0.81\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 5.9 bits;  conditional E-value: 0.0033\n",
            "   Zn_Tnp_IS1595  19 pvCPhCgsreayristrglfqCksCrk 45 \n",
            "                     + CP C s  +y  + +++  C  C++\n",
            "  XP_004358905.1 163 F-CPTCNSLLSYVSEFATRIDCPTCSY 188\n",
            "                     4.*****99888888889999999998 PP\n",
            "\n",
            "  == domain 2  score: 1.8 bits;  conditional E-value: 0.065\n",
            "   Zn_Tnp_IS1595  17 ngpvCPhCgsr 27 \n",
            "                     ++  CP C ++\n",
            "  XP_004358905.1 179 TRIDCPTCSYT 189\n",
            "                     5677****876 PP\n",
            "\n",
            "  == domain 3  score: 9.1 bits;  conditional E-value: 0.00033\n",
            "   Zn_Tnp_IS1595  20 vCPhCgsreayrist.........rglfqCksC 43 \n",
            "                     +CP Cg+ + y+ ++            + C +C\n",
            "  XP_004358905.1 227 KCPECGHGKMYFKTAqtrsadegqTIFYDCMKC 259\n",
            "                     6********999998788888776667777777 PP\n",
            "\n",
            ">> XP_004356292.1  uncharacterized protein DFA_08808 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    2.0   0.1     0.054   1.3e+02      20      38 ..     186     204 ..     180     205 .. 0.83\n",
            "   2 ?   13.4   3.9   1.5e-05     0.037      17      45 ..     285     312 ..     282     313 .. 0.87\n",
            "   3 ?    5.9   5.5    0.0033       8.1      19      45 ..     335     360 ..     329     361 .. 0.73\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 2.0 bits;  conditional E-value: 0.054\n",
            "   Zn_Tnp_IS1595  20 vCPhCgsreayristrglf 38 \n",
            "                      C++C s+++ + s+r+ +\n",
            "  XP_004356292.1 186 DCSSCSSSSSSSSSHRKDY 204\n",
            "                     5*******99999999876 PP\n",
            "\n",
            "  == domain 2  score: 13.4 bits;  conditional E-value: 1.5e-05\n",
            "   Zn_Tnp_IS1595  17 ngpvCPhCgsreayristrglfqCksCrk 45 \n",
            "                     ++p CP+C++   +r    +++ Ck C k\n",
            "  XP_004356292.1 285 DKPTCPNCHVIATRRHDK-RRWWCKNCVK 312\n",
            "                     5799*****998888887.********65 PP\n",
            "\n",
            "  == domain 3  score: 5.9 bits;  conditional E-value: 0.0033\n",
            "   Zn_Tnp_IS1595  19 pvCPhCgsreayristrglfqCksCrk 45 \n",
            "                     p+C +C   +a r  + +++ CksC+k\n",
            "  XP_004356292.1 335 PQCGRCST-QATRKHDKRRWWCKSCKK 360\n",
            "                     66777773.44444447********97 PP\n",
            "\n",
            ">> XP_004362940.1  S60 ribosomal protein L37A [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   11.5   4.0   6.1e-05      0.15      19      46 .]      37      63 ..      29      63 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 11.5 bits;  conditional E-value: 6.1e-05\n",
            "   Zn_Tnp_IS1595 19 pvCPhCgsreayristrglfqCksCrkq 46\n",
            "                      C++Cg++++ r+  +g + C+ Cr++\n",
            "  XP_004362940.1 37 YTCSFCGKDSVKRTC-VGIWSCSGCRRT 63\n",
            "                    57*****88777777.6********985 PP\n",
            "\n",
            ">> XP_004362193.1  uncharacterized protein DFA_06492 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.2   1.6   0.00015      0.36      18      44 ..     260     286 ..     256     287 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.2 bits;  conditional E-value: 0.00015\n",
            "   Zn_Tnp_IS1595  18 gpvCPhCgsreayristrglfqCksCr 44 \n",
            "                     +p CP+C+   ++  ++ ++f C  C+\n",
            "  XP_004362193.1 260 EPICPSCDRLVSWNQADQTQFYCTTCK 286\n",
            "                     489*****999999999*********8 PP\n",
            "\n",
            ">> XP_004355086.1  PH domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    0.4   1.5      0.18   4.4e+02      37      45 ..    1112    1120 ..    1103    1121 .. 0.85\n",
            "   2 ?    9.6   0.3   0.00023      0.56       6      34 ..    1224    1252 ..    1223    1256 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 0.4 bits;  conditional E-value: 0.18\n",
            "   Zn_Tnp_IS1595   37 lfqCksCrk 45  \n",
            "                       ++C++C+k\n",
            "  XP_004355086.1 1112 CWKCSKCSK 1120\n",
            "                      69*****97 PP\n",
            "\n",
            "  == domain 2  score: 9.6 bits;  conditional E-value: 0.00023\n",
            "   Zn_Tnp_IS1595    6 aralleklrWpng.pvCPhCgsreayrist 34  \n",
            "                      a+++l++ +++++ p CP C  +++++ist\n",
            "  XP_004355086.1 1224 ALKYLCESCYNDTcPICPTCN-ESVITIST 1252\n",
            "                      789*****************9.66666665 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (46 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       609  (0.0501854); expected 242.7 (0.02)\n",
            "Passed bias filter:                      341  (0.0281005); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        61  (0.00502678); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        11  (0.000906469); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               5  [number of targets reported over threshold]\n",
            "# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 4525.34\n",
            "//\n",
            "Query:       Zn_Tnp_IS91  [M=91]\n",
            "Accession:   PF14319.9\n",
            "Description: Transposase zinc-binding domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "     0.0061   16.0   1.1     0.0061   16.0   1.1    1.8  2  XP_004367206.1  uncharacterized protein DFA_07344 [Cavenderia\n",
            "  ------ inclusion threshold ------\n",
            "          1    8.9   5.2        1.5    8.4   0.5    2.6  2  XP_004361876.1  uncharacterized protein DFA_06163 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004367206.1  uncharacterized protein DFA_07344 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   -1.5   1.3      0.28   1.7e+03      42      56 ..       8      22 ..       2      44 .. 0.55\n",
            "   2 !   16.0   1.1     1e-06    0.0061       9      55 ..      44      88 ..      36      92 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: -1.5 bits;  conditional E-value: 0.28\n",
            "     Zn_Tnp_IS91 42 CeeCghekivpfsCk 56\n",
            "                    C++ g+   ++f C \n",
            "  XP_004367206.1  8 CSSTGEYCNSMFGCY 22\n",
            "                    444444333333332 PP\n",
            "\n",
            "  == domain 2  score: 16.0 bits;  conditional E-value: 1e-06\n",
            "     Zn_Tnp_IS91  9 flekkeklspsqkkavekilkCrtaalGfaryeCeeCghekivpfsC 55\n",
            "                    +++   ++++ + ++ ++  +C++ + G ++ +C eC + + + ++C\n",
            "  XP_004367206.1 44 YRNV-YQVNQIIYQEWHSTSECSDIEPGIHTSQCGECYE-NSFINTC 88\n",
            "                    5555.67899999************************86.5555666 PP\n",
            "\n",
            ">> XP_004361876.1  uncharacterized protein DFA_06163 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    8.4   0.5   0.00024       1.5      49      81 ..     201     234 ..     184     242 .. 0.83\n",
            "   2 ?    1.6   0.4     0.032     2e+02      25      66 ..     385     407 ..     368     424 .. 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 8.4 bits;  conditional E-value: 0.00024\n",
            "     Zn_Tnp_IS91  49 kivpfsCksrh.CpsCqkketeewlekqqeellp 81 \n",
            "                     k ++ sC+ +  C++C +  te    kqq ++l+\n",
            "  XP_004361876.1 201 KDFCVSCGTQKsCNKCDHVDTEIITFKQQVKVLE 234\n",
            "                     667899998877*************999999987 PP\n",
            "\n",
            "  == domain 2  score: 1.6 bits;  conditional E-value: 0.032\n",
            "     Zn_Tnp_IS91  25 ekilkCrtaalGfaryeCeeCghekivpfsCksrhCpsCqkk 66 \n",
            "                     ++               C +C+    + +s   + C+sC+k+\n",
            "  XP_004361876.1 385 NE---------------CIHCK----HSNSKLAKYCTSCGKE 407\n",
            "                     33...............33333....3445555556666554 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (91 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       912  (0.0751545); expected 242.7 (0.02)\n",
            "Passed bias filter:                      336  (0.0276885); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        37  (0.00304903); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         4  (0.000329625); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.13u 0.00s 00:00:00.13 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 7308.42\n",
            "//\n",
            "Query:       Zona_pellucida  [M=259]\n",
            "Accession:   PF00100.26\n",
            "Description: Zona pellucida-like domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (259 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1316  (0.108447); expected 242.7 (0.02)\n",
            "Passed bias filter:                      139  (0.0114545); expected 242.7 (0.02)\n",
            "Passed Vit filter:                         3  (0.000247219); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.19u 0.01s 00:00:00.20 Elapsed: 00:00:00.10\n",
            "# Mc/sec: 17257.85\n",
            "//\n",
            "Query:       ZoocinA_TRD  [M=106]\n",
            "Accession:   PF16775.8\n",
            "Description: Target recognition domain of lytic exoenzyme\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (106 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       405  (0.0333745); expected 242.7 (0.02)\n",
            "Passed bias filter:                      213  (0.0175525); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        12  (0.000988875); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.10u 0.00s 00:00:00.10 Elapsed: 00:00:00.05\n",
            "# Mc/sec: 12784.67\n",
            "//\n",
            "Query:       Zot  [M=194]\n",
            "Accession:   PF05707.15\n",
            "Description: Zonular occludens toxin (Zot)\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "      0.044   12.9   0.6        1.9    7.5   0.1    3.2  2  XP_004356293.1  putative ATPase [Cavenderia fasciculata]\n",
            "       0.11   11.6   0.0       0.22   10.6   0.0    1.4  1  XP_004355492.1  uncharacterized protein DFA_07989 [Cavenderia\n",
            "       0.12   11.5   0.3        1.9    7.6   0.0    2.2  2  XP_004358974.1  AAA ATPase domain-containing protein [Cavende\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004356293.1  putative ATPase [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.2   0.1     0.042   1.7e+02      82     143 ..     383     450 ..     353     487 .. 0.76\n",
            "   2 ?    7.5   0.1   0.00047       1.9      56     106 ..     636     686 ..     604     700 .. 0.78\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.2 bits;  conditional E-value: 0.042\n",
            "             Zot  82 allviDEcqeifpvrakegrkd.....pevi.naftehRhlgwDivlltqdiskihkevralaekavk 143\n",
            "                     ++ vi    eifp+  + ++k       +++ n   +    +  i+ +t ++ ++ + vr l +++ k\n",
            "  XP_004356293.1 383 SVFVIENIDEIFPNLENDDTKSqeeslTGMMmNIMEKNNSQNTFIIAMTSKLRNVSNLVRSLFDEQIK 450\n",
            "                     67788888888888777766665555534443666666777788888999999999999888777655 PP\n",
            "\n",
            "  == domain 2  score: 7.5 bits;  conditional E-value: 0.00047\n",
            "             Zot  56 rliridnpegaetlkefakwatidkgallviDEcqeifpvrakegrkdpev 106\n",
            "                     ++++ +  e+++ l+ ++k+a+ +  ++++iDE q if +r   g + +++\n",
            "  XP_004356293.1 636 EIVQGEVGESEKMLADIFKRAVQAAPSIVFIDEIQAIFGQRDSSGSHGKNI 686\n",
            "                     3444446788889999999999999*************9998866555555 PP\n",
            "\n",
            ">> XP_004355492.1  uncharacterized protein DFA_07989 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.6   0.0   5.4e-05      0.22       5      91 ..      57     145 ..      53     160 .. 0.60\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.6 bits;  conditional E-value: 5.4e-05\n",
            "             Zot   5 vtGapGsgKtsgavkkdllpalk.egRkiitnidGlr.lerieqvlgekvstarlirid.npegaetlkefakwatid..kgallviDEcqe 91 \n",
            "                     v G+pG+gKt+ a++  ++   k +   +++n  G++   +i+  l+++   ++  ++  + e  +++++  +    +  k   +++DE+  \n",
            "  XP_004355492.1  57 VSGQPGTGKTLTALNR-INSIPKrKCTTLFFNCMGMQdPANIYTELHNTLCKPKKKKTPgQSEMVSKIQQTICDP--EesKMFCVILDEVDS 145\n",
            "                     68************98.88866615566778*****988888874444333222222222222222222222222..123555566777765 PP\n",
            "\n",
            ">> XP_004358974.1  AAA ATPase domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?    1.6   0.0     0.032   1.3e+02       4      22 ..     171     191 ..     170     209 .. 0.73\n",
            "   2 ?    7.6   0.0   0.00046       1.9      64     102 ..     210     248 ..     196     292 .. 0.84\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 1.6 bits;  conditional E-value: 0.032\n",
            "             Zot   4 lvtGapGsgKts..gavkkdl 22 \n",
            "                     l +G+pG+gK++  +av  ++\n",
            "  XP_004358974.1 171 LLYGPPGTGKSYlaKAVATEI 191\n",
            "                     679********8223555555 PP\n",
            "\n",
            "  == domain 2  score: 7.6 bits;  conditional E-value: 0.00046\n",
            "             Zot  64 egaetlkefakwatidkgallviDEcqeifpvrakegrk 102\n",
            "                     ++++++k+++  a+ +k+++++iDE   + ++r   + +\n",
            "  XP_004358974.1 210 DSEKLVKQLFEMAREKKNSVIFIDEIDSLCSTRNDSESE 248\n",
            "                     7788899999999988***********999998876444 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (194 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       493  (0.0406263); expected 242.7 (0.02)\n",
            "Passed bias filter:                      303  (0.0249691); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        29  (0.00238978); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         3  (0.000247219); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               3  [number of targets reported over threshold]\n",
            "# CPU time: 0.16u 0.00s 00:00:00.16 Elapsed: 00:00:00.08\n",
            "# Mc/sec: 15338.26\n",
            "//\n",
            "Query:       ZT_dimer  [M=79]\n",
            "Accession:   PF16916.8\n",
            "Description: Dimerisation domain of Zinc Transporter\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    3.3e-13   49.0   0.0    8.1e-13   47.8   0.0    1.6  1  XP_004361148.1  putative cation efflux pump [Cavenderia fasci\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004361148.1  putative cation efflux pump [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   47.8   0.0   6.7e-17   8.1e-13       2      77 ..     341     416 ..     340     417 .. 0.93\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 47.8 bits;  conditional E-value: 6.7e-17\n",
            "                     ...HHHHHHHHHTT.-TT-EEEEEEEEEECTTEEEEEEEEEE-TTS-HHHHHHHHHHHHHHHHHH-TTEEEEEEEEE CS\n",
            "        ZT_dimer   2 peeeeeeIreiies.vpgvkgvhdlrtrksGnkifvdlhievdgdlsvkeaHeiadeieerlreelpevedvliHve 77 \n",
            "                     p+e +++++ i++s  p+v +v+ +r+ ++Gn+ fv+++i++++d++++++H+i ++++ +l+ +l+eve+ ++Hv+\n",
            "  XP_004361148.1 341 PPEFLQRLTYIAMShSPEVLKVDTCRAFHVGNNFFVEVDIVLPPDMPLQKSHDIGESLQIKLE-SLDEVERAFVHVD 416\n",
            "                     67789999999964499*********************************************5.899********97 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (79 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       357  (0.029419); expected 242.7 (0.02)\n",
            "Passed bias filter:                      272  (0.0224145); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        12  (0.000988875); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         1  (8.24063e-05); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.01s 00:00:00.10 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 8565.29\n",
            "//\n",
            "Query:       ZU5  [M=97]\n",
            "Accession:   PF00791.23\n",
            "Description: ZU5 domain\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (97 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       234  (0.0192831); expected 242.7 (0.02)\n",
            "Passed bias filter:                      182  (0.0149979); expected 242.7 (0.02)\n",
            "Passed Vit filter:                         8  (0.00065925); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.09u 0.00s 00:00:00.09 Elapsed: 00:00:00.06\n",
            "# Mc/sec: 10343.15\n",
            "//\n",
            "Query:       Zw10  [M=513]\n",
            "Accession:   PF06248.16\n",
            "Description: Centromere/kinetochore Zw10\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    6.2e-33  113.7  21.0    2.5e-30  105.2  21.0    3.0  1  XP_004357742.1  centrosomal protein [Cavenderia fasciculata]\n",
            "     0.0023   16.2   0.6     0.0028   15.9   0.6    1.2  1  XP_004358831.1  uncharacterized protein DFA_00850 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004357742.1  centrosomal protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !  105.2  21.0   4.1e-34   2.5e-30      56     512 ..     160     575 ..     107     576 .. 0.83\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 105.2 bits;  conditional E-value: 4.1e-34\n",
            "            Zw10  56 ladldasveelaelkqqlkekevvlellehlkkieealeeleealkekdylkaaklleelkellealke...rkgqdvk.ilkaLsaeltvlsen 146\n",
            "                        +++      +l + l++ e  l+++   ++++  ++ +e+a ke d ++ +  l ++ +++++      +  q+++ il +++ +l+  +++\n",
            "  XP_004357742.1 160 VGRIRSNISYDQQLRKRLESIESTLVMVDYFEEYDGYVKAYEKAIKENDIVQSTRHLVKMETMVNIGIPdslQSMQPLPrILPSIKDQLKKRKNQ 254\n",
            "                     33444444444567888889999999**************************************997651114556666778888********** PP\n",
            "\n",
            "            Zw10 147 lvykLseewdrlvlwklpkskvklegdksnlkvklkLsteeselsevleAlavlgeleekLkafgqkllknvlkplivspslkveVekvdkeeg. 240\n",
            "                     l  +L ee+ ++++     +             +l++s +  ++ + + Al++lg l   + + ++++ k++++pl    +   +  k++ + + \n",
            "  XP_004357742.1 255 LKTQLLEEFTNALVI----N-------------HLEISLARDNIYQLIGALEQLGLLYFIVSSISNQIYKSIIQPLLSTVQARLSLTKNESQ-Pt 331\n",
            "                     ***********9443....3.............478888999***********************************999999999996654.34 PP\n",
            "\n",
            "            Zw10 241 .......vtlslessqktkgeksepskvYskvlevlkvllskLvssksqeklgkvslveilGdliwpelsellikdclvksiPetsskledyqev 328\n",
            "                              l+l ++ ++k ++s+ +++++ + ++++++ +++         ++ +l+e +G  iw ++s  +i++cl + iP + s+l  +q +\n",
            "  XP_004357742.1 332 idiskngNHLKLTIDIQEKNKISTIDNLFNILKNMFEFIYNHIFG-------KENRLMERFGTDIWSDISTTIISNCLKNIIPDDPSQLSLFQSI 419\n",
            "                     6777777789999999999999999999999999999**999933.......5689**************************************9 PP\n",
            "\n",
            "            Zw10 329 ieqteeFeeaLkeleflskdetdllkyaedvnvhFankkcqdvlvkarkLlksdldntVkvtpdetavkvlglksklkipaeleakeiseelkle 423\n",
            "                       q + Fee+L +++ +++ + + + +++++++hFa kk  ++l+kar ++  d  +   +      + v+g                       \n",
            "  XP_004357742.1 420 --QSKPFEEYLVQIGLINSTQQSITEFINNIESHFAEKKRHNLLAKARHIIMTDKFDS--ID---SSESVIGES--------------------- 486\n",
            "                     ..6789*******************************************998832222..21...222233322..................... PP\n",
            "\n",
            "            Zw10 424 neetlseltfsfprCrvSesakklmdLaektlkeattssaetavqLfltvrnilelyvdvvPtkhkenleslpqlsalfhNnCmYishl 512\n",
            "                     ++++ s +++ f    + + a+ l+++ +  l e  +ss  +  +++   r+i+ ++     ++hk++++++p l+ l++N C+++sh \n",
            "  XP_004357742.1 487 QQNNGSLQLYYFTPRSIFKNANTLIQMCKTALYESLQSSPLCSTKIYQGCRDIFDMFRFMYIKYHKDKIKTVPLLACLYYNSCLFLSHS 575\n",
            "                     2233334445555558999*********************************************************************5 PP\n",
            "\n",
            ">> XP_004358831.1  uncharacterized protein DFA_00850 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.9   0.6   4.6e-07    0.0028      15      73 ..      55     112 ..      41     129 .. 0.85\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.9 bits;  conditional E-value: 4.6e-07\n",
            "            Zw10  15 tivkkYsefleslssteellekveslskeiddllekvekevladldasveelaelkqql 73 \n",
            "                     ti+k +s    ++++t++++++ +slske++  ++   ++  ++ld+  + ++e++q++\n",
            "  XP_004358831.1  55 TISKTFSAHELNVTQTDDMIHQMSSLSKELSSQINLLRGS-FQALDGINNSVTEINQNA 112\n",
            "                     799************************************9.667777777777777765 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (513 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       947  (0.0780387); expected 242.7 (0.02)\n",
            "Passed bias filter:                      146  (0.0120313); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        14  (0.00115369); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         2  (0.000164813); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               2  [number of targets reported over threshold]\n",
            "# CPU time: 0.29u 0.00s 00:00:00.28 Elapsed: 00:00:00.15\n",
            "# Mc/sec: 21940.63\n",
            "//\n",
            "Query:       Zwilch  [M=574]\n",
            "Accession:   PF09817.12\n",
            "Description: RZZ complex, subunit zwilch\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (574 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       942  (0.0776267); expected 242.7 (0.02)\n",
            "Passed bias filter:                      123  (0.010136); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        10  (0.000824063); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.28u 0.01s 00:00:00.29 Elapsed: 00:00:00.16\n",
            "# Mc/sec: 24043.58\n",
            "//\n",
            "Query:       Zwint  [M=252]\n",
            "Accession:   PF15556.9\n",
            "Description: ZW10 interactor\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  -------- -----------\n",
            "\n",
            "   [No hits detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            "\n",
            "   [No targets detected that satisfy reporting thresholds]\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (252 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1734  (0.142892); expected 242.7 (0.02)\n",
            "Passed bias filter:                      216  (0.0177998); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        36  (0.00296663); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         0  (0); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               0  [number of targets reported over threshold]\n",
            "# CPU time: 0.23u 0.00s 00:00:00.23 Elapsed: 00:00:00.12\n",
            "# Mc/sec: 13972.43\n",
            "//\n",
            "Query:       ZYG-11_interact  [M=295]\n",
            "Accession:   PF05884.15\n",
            "Description: Interactor of ZYG-11\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "  ------ inclusion threshold ------\n",
            "       0.22   10.3   5.4       0.29   10.0   5.4    1.2  1  XP_004363104.1  uncharacterized protein DFA_03501 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004363104.1  uncharacterized protein DFA_03501 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 ?   10.0   5.4   2.4e-05      0.29      19      78 ..      64     125 ..      52     172 .. 0.59\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 10.0 bits;  conditional E-value: 2.4e-05\n",
            "  ZYG-11_interact  19 vknegseeeavirlvirtysklgrdleqave.krlkklpeegerlleql.lssifsqkskds 78 \n",
            "                       ++e+++e+ +i+++++ ++ l +d+++  + ++ +k +e   + l+ + + ++ ++k k++\n",
            "   XP_004363104.1  64 DNVEDAQEKEAIAMAMKDMETLEKDVKEKKDiQEQQKQQEYEAQRLKRIeQDQLEEEKAKEQ 125\n",
            "                      4689************************7776333333333333333331223333333333 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (295 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                      1096  (0.0903173); expected 242.7 (0.02)\n",
            "Passed bias filter:                      682  (0.0562011); expected 242.7 (0.02)\n",
            "Passed Vit filter:                       117  (0.00964153); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                        25  (0.00206016); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               1  [number of targets reported over threshold]\n",
            "# CPU time: 0.50u 0.00s 00:00:00.50 Elapsed: 00:00:00.28\n",
            "# Mc/sec: 6929.36\n",
            "//\n",
            "Query:       ZZ  [M=45]\n",
            "Accession:   PF00569.20\n",
            "Description: Zinc finger, ZZ type\n",
            "Scores for complete sequences (score includes all domains):\n",
            "   --- full sequence ---   --- best 1 domain ---    -#dom-\n",
            "    E-value  score  bias    E-value  score  bias    exp  N  Sequence       Description\n",
            "    ------- ------ -----    ------- ------ -----   ---- --  --------       -----------\n",
            "    4.3e-11   41.9   6.7    7.5e-11   41.1   6.7    1.4  1  XP_004366193.1  myb domain-containing protein [Cavenderia fas\n",
            "    1.4e-10   40.2   9.7      3e-10   39.2   9.7    1.5  1  XP_004358632.1  ZZ-type zinc finger-containing protein [Caven\n",
            "    1.1e-09   37.3   3.7    2.1e-09   36.5   3.7    1.5  1  XP_004358587.1  ZZ-type zinc finger-containing protein [Caven\n",
            "    5.9e-09   35.1   8.2    5.9e-09   35.1   8.2    3.7  3  XP_004351748.1  uncharacterized protein DFA_09851 [Cavenderia\n",
            "    2.9e-08   32.9   6.9    5.8e-08   31.9   6.9    1.4  1  XP_004350198.1  myb domain-containing protein [Cavenderia fas\n",
            "    1.8e-07   30.3  32.0    0.00056   19.1   6.7    3.8  3  XP_004355615.1  uncharacterized protein DFA_08113 [Cavenderia\n",
            "    9.5e-05   21.6   7.1    9.5e-05   21.6   7.1    2.3  2  XP_004352141.1  uncharacterized protein DFA_09485 [Cavenderia\n",
            "    0.00042   19.5   6.9      0.001   18.3   6.9    1.7  1  XP_004352615.1  uncharacterized protein DFA_09189 [Cavenderia\n",
            "     0.0018   17.5   8.5     0.0058   15.9   8.5    1.9  1  XP_004360876.1  uncharacterized protein DFA_05155 [Cavenderia\n",
            "\n",
            "\n",
            "Domain annotation for each sequence (and alignments):\n",
            ">> XP_004366193.1  myb domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   41.1   6.7   5.5e-14   7.5e-11       1      45 []     354     400 ..     354     400 .. 0.95\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 41.1 bits;  conditional E-value: 5.5e-14\n",
            "                     SSSEEBETTTSSBBECSCEEEESSS.TSSEECHHHHHT.GCBTTCCC CS\n",
            "              ZZ   1 ihkkytCdgCseapligvRyhclrc.sdYDLCqsCfst.hkagkhkm 45 \n",
            "                     ih++y+CdgC + p++g Ry+c  c +d DLCqsC  +  k g hk+\n",
            "  XP_004366193.1 354 IHEGYKCDGCDQEPIVGYRYKCEECiEDIDLCQSCHDSyEKIGSHKS 400\n",
            "                     69***********************88***********999999985 PP\n",
            "\n",
            ">> XP_004358632.1  ZZ-type zinc finger-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   39.2   9.7   2.2e-13     3e-10       5      44 ..     220     259 ..     217     260 .. 0.89\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 39.2 bits;  conditional E-value: 2.2e-13\n",
            "                     EBETTTSSBBECSCEEEESSSTSSEECHHHHHT.GCBTTCC CS\n",
            "              ZZ   5 ytCdgCseapligvRyhclrcsdYDLCqsCfst.hkagkhk 44 \n",
            "                     + Cd+C +  +ig Ry+c +c++YDLCq C+   +  ++h+\n",
            "  XP_004358632.1 220 ALCDICHK-QIIGIRYKCKVCPNYDLCQDCKEGgKHPADHE 259\n",
            "                     67******.**********************9977777775 PP\n",
            "\n",
            ">> XP_004358587.1  ZZ-type zinc finger-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   36.5   3.7   1.5e-12   2.1e-09       2      37 ..     207     241 ..     206     252 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 36.5 bits;  conditional E-value: 1.5e-12\n",
            "                     SSEEBETTTSSBBECSCEEEESSSTSSEECHHHHHT CS\n",
            "              ZZ   2 hkkytCdgCseapligvRyhclrcsdYDLCqsCfst 37 \n",
            "                     h ++ CdgC    + g R +c++c++YDLC sC+ +\n",
            "  XP_004358587.1 207 HPGVVCDGCNM-GIFGIRHKCAVCPNYDLCESCKEK 241\n",
            "                     66789******.*********************987 PP\n",
            "\n",
            ">> XP_004351748.1  uncharacterized protein DFA_09851 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   35.1   8.2   4.4e-12   5.9e-09       4      43 ..    2981    3021 ..    2978    3023 .. 0.93\n",
            "   2 ?   -4.4   4.2         9   1.2e+04       9      28 ..    3764    3781 ..    3761    3783 .. 0.72\n",
            "   3 ?    4.8   1.2     0.012        17       4      20 ..    3788    3804 ..    3786    3808 .. 0.87\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 35.1 bits;  conditional E-value: 4.4e-12\n",
            "                      EEBETTTSSBBECSCEEEESSSTSSEECHHHHHT.GCBTTC CS\n",
            "              ZZ    4 kytCdgCseapligvRyhclrcsdYDLCqsCfst.hkagkh 43  \n",
            "                       ++Cd+Cs  p+ + Ryhc +c d+DLC+ C+ +   +++h\n",
            "  XP_004351748.1 2981 HFSCDKCSVYPIQQKRYHCEVCGDFDLCSDCYLKiTATKEH 3021\n",
            "                      589******************************98888888 PP\n",
            "\n",
            "  == domain 2  score: -4.4 bits;  conditional E-value: 9\n",
            "                      TTSSBBECSCEEEESSSTSS CS\n",
            "              ZZ    9 gCseapligvRyhclrcsdY 28  \n",
            "                      +C++    ++ ++c +c++ \n",
            "  XP_004351748.1 3764 ICKH--CHENAFQCKQCRNI 3781\n",
            "                      4777..44577899999885 PP\n",
            "\n",
            "  == domain 3  score: 4.8 bits;  conditional E-value: 0.012\n",
            "                      EEBETTTSSBBECSCEE CS\n",
            "              ZZ    4 kytCdgCseapligvRy 20  \n",
            "                      ++ C++C+++ ++++ y\n",
            "  XP_004351748.1 3788 GFLCNECGFCKHARFDY 3804\n",
            "                      678***********988 PP\n",
            "\n",
            ">> XP_004350198.1  myb domain-containing protein [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   31.9   6.9   4.3e-11   5.8e-08       4      37 ..     225     258 ..     223     267 .. 0.91\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 31.9 bits;  conditional E-value: 4.3e-11\n",
            "                     EEBETTTSSBBECSCEEEESSSTSSEECHHHHHT CS\n",
            "              ZZ   4 kytCdgCseapligvRyhclrcsdYDLCqsCfst 37 \n",
            "                     +y+Cd C ++ ++ v  +c +c+d+DLC +Cfs \n",
            "  XP_004350198.1 225 QYHCDYCQKDLSNVVKIKCSVCEDFDLCLECFSV 258\n",
            "                     8*******************************85 PP\n",
            "\n",
            ">> XP_004355615.1  uncharacterized protein DFA_08113 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   19.1   6.7   4.2e-07   0.00056       7      37 ..     247     277 ..     244     283 .. 0.89\n",
            "   2 !   11.1   4.2   0.00014      0.18       4      25 ..     315     335 ..     313     341 .. 0.93\n",
            "   3 !   11.3   5.2   0.00011      0.15       2      42 ..     441     481 .]     440     481 .] 0.75\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 19.1 bits;  conditional E-value: 4.2e-07\n",
            "                     ETTTSSBBECSCEEEESSSTSSEECHHHHHT CS\n",
            "              ZZ   7 CdgCseapligvRyhclrcsdYDLCqsCfst 37 \n",
            "                     Cd Cs+       +hc +c ++DLC sC  +\n",
            "  XP_004355615.1 247 CDYCSKGYGTSEQFHCSSCGNFDLCRSCIEK 277\n",
            "                     ******999999****************876 PP\n",
            "\n",
            "  == domain 2  score: 11.1 bits;  conditional E-value: 0.00014\n",
            "                     EEBETTTSSBBECSCEEEESSS CS\n",
            "              ZZ   4 kytCdgCseapligvRyhclrc 25 \n",
            "                     ++ Cd+C +  + gv+yhc++c\n",
            "  XP_004355615.1 315 EARCDKCLS-NVKGVVYHCTIC 335\n",
            "                     678****99.9**********9 PP\n",
            "\n",
            "  == domain 3  score: 11.3 bits;  conditional E-value: 0.00011\n",
            "                     SSEEBETTTSSB..BECSCEEEESSSTSSEECHHHHHTGCBTT CS\n",
            "              ZZ   2 hkkytCdgCsea..pligvRyhclrcsdYDLCqsCfsthkagk 42 \n",
            "                     h + +Cd+C +   ++  + yh+   ++YD C++C+ +h+++k\n",
            "  XP_004355615.1 441 HSEWICDVCRSRygATEITFYHS--PNNYDVCVGCYYKHFQKK 481\n",
            "                     78899****98422344455775..57***********99985 PP\n",
            "\n",
            ">> XP_004352141.1  uncharacterized protein DFA_09485 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   21.6   7.1   7.1e-08   9.5e-05       8      40 ..       3      36 ..       1      41 [. 0.79\n",
            "   2 ?   -0.7   0.3      0.65   8.7e+02       4      12 ..     150     158 ..     146     162 .. 0.68\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 21.6 bits;  conditional E-value: 7.1e-08\n",
            "                    TTTSSBBECSCEEEESSSTSSEECHHHHHT.GCB CS\n",
            "              ZZ  8 dgCseapligvRyhclrcsdYDLCqsCfst.hka 40\n",
            "                    ++C ++   gv y+c+ c++  LC+ C++  +k+\n",
            "  XP_004352141.1  3 NTCTACGDTGVKYKCTTCDNVSLCILCYNDpSKH 36\n",
            "                    45555555*********************95544 PP\n",
            "\n",
            "  == domain 2  score: -0.7 bits;  conditional E-value: 0.65\n",
            "                     EEBETTTSS CS\n",
            "              ZZ   4 kytCdgCse 12 \n",
            "                      y+C  Cs \n",
            "  XP_004352141.1 150 TYKCPYCST 158\n",
            "                     577777775 PP\n",
            "\n",
            ">> XP_004352615.1  uncharacterized protein DFA_09189 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   18.3   6.9   7.6e-07     0.001       5      40 ..      30      67 ..      27      72 .. 0.82\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 18.3 bits;  conditional E-value: 7.6e-07\n",
            "                    EBETTTS.SBBECSCEEEESSSTSSEECHHHHHT.GCB CS\n",
            "              ZZ  5 ytCdgCs.eapligvRyhclrcsdYDLCqsCfst.hka 40\n",
            "                    + C+gCs +  + + Ry c+ c   DLC +C++  +k+\n",
            "  XP_004352615.1 30 FMCNGCSyNFKHGEPRYVCTECAGIDLCKKCYKRdQKY 67\n",
            "                    78****943478899*****************985443 PP\n",
            "\n",
            ">> XP_004360876.1  uncharacterized protein DFA_05155 [Cavenderia fasciculata]\n",
            "   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc\n",
            " ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----\n",
            "   1 !   15.9   8.5   4.3e-06    0.0058       2      38 ..     756     793 ..     755     797 .. 0.90\n",
            "\n",
            "  Alignments for each domain:\n",
            "  == domain 1  score: 15.9 bits;  conditional E-value: 4.3e-06\n",
            "                     SSEEBETTTSSBBECSCEEEESSSTSSEECHHHHHT.G CS\n",
            "              ZZ   2 hkkytCdgCseapligvRyhclrcsdYDLCqsCfst.h 38 \n",
            "                     h + tC +C + p+ig+ y c +c  +  C+sC  + +\n",
            "  XP_004360876.1 756 HRGTTCTVCTQSPIIGTFYYCSVCTGFTFCDSCRAQqS 793\n",
            "                     7789*****************************98833 PP\n",
            "\n",
            "\n",
            "\n",
            "Internal pipeline statistics summary:\n",
            "-------------------------------------\n",
            "Query model(s):                            1  (45 nodes)\n",
            "Target sequences:                      12135  (6767248 residues searched)\n",
            "Passed MSV filter:                       729  (0.0600742); expected 242.7 (0.02)\n",
            "Passed bias filter:                      285  (0.0234858); expected 242.7 (0.02)\n",
            "Passed Vit filter:                        36  (0.00296663); expected 12.1 (0.001)\n",
            "Passed Fwd filter:                         9  (0.000741656); expected 0.1 (1e-05)\n",
            "Initial search space (Z):              12135  [actual number of targets]\n",
            "Domain search space  (domZ):               9  [number of targets reported over threshold]\n",
            "# CPU time: 0.14u 0.01s 00:00:00.15 Elapsed: 00:00:00.13\n",
            "# Mc/sec: 2250.48\n",
            "//\n",
            "[ok]\n"
          ]
        }
      ],
      "source": [
        "!hmmsearch --tblout test.out Pfam-A.hmm protein.faa"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "отфильтровываем"
      ],
      "metadata": {
        "id": "B3L6dANWcHec"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "dataset = {}\n",
        "with open(\"families.csv\", \"r\") as q:\n",
        "  while Q := q.readline():\n",
        "    Q = Q.rstrip()\n",
        "    if not \";\" in Q:\n",
        "      arr = Q.split(\", \")\n",
        "      for a in arr:\n",
        "        p = a.split(\" \")\n",
        "        if not dataset.get(p[1] if len(p) > 1 else p[0]):\n",
        "          dataset.update({p[1] if len(p) > 1 else p[0]: 1})\n",
        "    else:\n",
        "      arr = Q.split(\";\")\n",
        "      for a in arr:\n",
        "        if not dataset.get(a):\n",
        "          dataset.update({a: 1})\n",
        "print(dataset)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "JiCRSSTCxyDe",
        "outputId": "065ba45a-7b1f-42d0-8fc1-d5ad0bb21549",
        "collapsed": true
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{'approved': 1, 'modification,APOB_mRNA_editosome,\"mRNA': 1, 'complementation': 1, 'PF00076': 1, 'PF14259': 1, 'ACINU,#,RNA modification,#,mRNA,Apoptotic chromatin condensation inducer in the nucleus (Acinus),ACINU_HUMAN,PF16294': 1, 'PF02037,ACINU_MOUSE,Alternative splicing': 1, 'remodeling': 1, 'nBAF': 1, 'npBAF': 1, 'PBAF': 1, 'EPAFB': 1, 'bBAF': 1, 'BRM-BRG1': 1, 'NuA4': 1, 'complex\",#,\"actin': 1, 'PF00022': 1, 'SWI/SNF_Brg1(I)': 1, 'SWI/SNF_Brg1(II)': 1, 'SWI/SNF_Brm': 1, 'SWI/SNF-like_EPAFa': 1, 'WINAC': 1, 'Ino80': 1, 'complex': 1, '6A,ACL6A_HUMAN,Actin': 1, 'BRM-BRG1\",#,actin-like': 1, 'remodeling,#,\"H2A': 1, 'H3': 1, 'actin-related': 1, 'remodeling,Ino80,#,ARP5': 1, 'modification': 1, 'PB010803': 1, 'PB021724': 1, 'PB029916': 1, 'PB008048': 1, 'PF13465': 1, 'modification,APOB_mRNA_editosome,\"ssDNA': 1, 'cytidine': 1, 'TF\",#,\"H3K4': 1, 'H3K4me3': 1, 'motif\",autoimmune': 1, 'PB009104': 1, 'PF01342': 1, 'PF03172': 1, 'ALKBH1,\"C': 1, 'modification\",#,\"m3C': 1, 'of': 1, 'demethylation\"': 1, 'modification,#,H2A,\"alkB': 1, 'repair': 1, 'modification,#,m6A': 1, 'demethylase\",ALKB5_HUMAN,PF13532,ALKB5_MOUSE,RNA': 1, 'PB101142': 1, 'PB012624': 1, 'PB018156': 1, 'PB081366': 1, 'A\",AN32A_HUMAN,LRR_9': 1, 'chaperone,#,\"H3': 1, '(leucine-rich)': 1, 'B\",AN32B_HUMAN,\"LRR_4': 1, 'PB030686': 1, 'chaperone': 1, 'E\",AN32E_HUMAN,\"LRR_4': 1, 'PF13855': 1, 'PB038005': 1, 'PB094319': 1, 'modification,#,H2AX,\"amyloid': 1, 'B': 1, '1': 1, 'PB023321': 1, 'PF00397': 1, 'PB021616': 1, 'modification\",APOB_mRNA_editosome,\"ssDNA': 1, 'mRNA': 1, 'polypeptide': 1, 'editing\"': 1, 'modification\",#,\"ssDNA': 1, 'polypeptide-like': 1, 'PB038491': 1, 'PF08210': 1, 'PF00383': 1, 'BRM-BRG1\",DNA': 1, 'PF12031': 1, 'PB016906': 1, 'PB001854': 1, 'BRM-BRG1\",\"H2BK120': 1, 'motif\",AT': 1, 'PB060945': 1, 'PB007257': 1, 'PF01388': 1, 'PB011884': 1, 'PB014804': 1, 'PB000649': 1, 'PB006603': 1, 'PF02257': 1, 'PB001988': 1, 'PF08169': 1, 'PF11717': 1, 'PB015110': 1, 'hydrocarbon': 1, 'PF00989': 1, 'PF14598': 1, 'PB002781': 1, 'modification,#,#,\"arrestin': 1, '1\",ARRB1_HUMAN,\"Arrestin_C': 1, 'PF00339': 1, 'function': 1, 'small': 1, 'homeotic)-like': 1, 'PF00439': 1, 'PF00628': 1, 'PB014208': 1, 'PB018246': 1, 'PB025586': 1, 'PB004663': 1, 'PB006310': 1, 'PF00856': 1, 'Menin-associated_HMT': 1, 'MLL-HCF': 1, 'CHD8': 1, 'MLL2/3': 1, 'MLL1,2': 1, 'MLL4/WBP7': 1, 'MLL3,4\",#,\"ash2': 1, 'PB012412': 1, 'PF00622': 1, 'group': 1, 'PF05066': 1, 'PF13922': 1, 'PB004087': 1, 'sex': 1, 'protein': 1, 'PB017129': 1, 'PB020113': 1, 'PB027498': 1, 'PB034336': 1, 'PB006874': 1, 'remodeling,#,#,\"ATPase': 1, 'domain': 1, 'PB011064': 1, 'PB023116': 1, 'PB040477': 1, 'H2A': 1, 'H2B': 1, 'and': 1, 'PB130965': 1, 'TF\",#,\"H2B': 1, 'H4': 1, 'motif\",activating': 1, 'activator\"': 1, 'PF02260': 1, 'PF00454': 1, 'PB010220': 1, 'PB013450': 1, 'PB016049': 1, 'PB017604': 1, 'PB003386': 1, 'PF11640': 1, 'PB015857': 1, 'PB017853': 1, 'PB020071': 1, 'PB021066': 1, 'PB021726': 1, 'PB022672': 1, 'PF08064': 1, 'remodeling,#,\"H3K9me2': 1, 'H3K9me3': 1, 'thalassemia/mental': 1, 'PB012326': 1, 'PB014952': 1, 'PB016104': 1, 'PB016400': 1, 'PB017163': 1, 'PB019720': 1, 'PB024253': 1, 'PB003534': 1, 'PF00176': 1, 'PB023658': 1, 'PB006735': 1, 'PF08313': 1, 'PF08209': 1, 'PF00069': 1, 'kinase': 1, 'remodeling,#,H3K27me3,bromo': 1, 'associated': 1, 'PB014567': 1, 'PB016218': 1, 'PB004660': 1, 'BARD1,\"H2AXub': 1, 'H2Aub': 1, 'H2Bub': 1, 'H3ub': 1, 'BRCA1-A\",\"H2AX': 1, 'PF00533': 1, 'PB030184': 1, 'PB032178': 1, 'PB032432': 1, 'PB033025': 1, 'PF14835': 1, 'chaperone,\"ACF': 1, 'adjacent': 1, 'PF02791': 1, 'PB018021': 1, 'PB018567': 1, 'PB019398': 1, 'PF10537': 1, 'PF15612': 1, 'WINAC\",\"H2AXT142': 1, 'PF15613': 1, 'PF15614': 1, 'BAZ2A,\"H4K5': 1, 'H4K8': 1, 'motif\",\"bromodomain': 1, 'PF01429': 1, 'PB014139': 1, 'PB001702': 1, 'PB025722': 1, 'PB003124': 1, 'PB032186': 1, 'PB006766': 1, 'H2Aac': 1, 'H2Bac': 1, 'H3ac': 1, 'PB010350': 1, 'PB012038': 1, 'PB013066': 1, 'PB001587': 1, 'PB017078': 1, 'PB017427': 1, 'PB009375': 1, 'PB011277': 1, 'PB026758': 1, 'PB027995': 1, 'PB030654': 1, 'PB005207': 1, 'PB001109': 1, 'PB016739': 1, 'PB020248': 1, 'PB031169': 1, 'PB008912': 1, 'ring': 1, 'remodeling,NuRF,#,bromodomain': 1, 'PB014861': 1, 'PB026081': 1, 'PB000911': 1, 'PB094218': 1, 'TF\",\"BRCC': 1, 'motif,\"breast': 1, 'onset\",BRCA1_HUMAN,\"BRCT': 1, 'PF12820': 1, 'PF04873': 1, 'PB017893': 1, 'PF00097': 1, 'methylation': 1, 'ubiquitination': 1, 'activator': 1, 'repressor\"': 1, 'cancer': 1, 'onset\",BRCA2_HUMAN,\"BRCA-2_OB1': 1, 'PF09104': 1, 'PF09169': 1, 'PF00634': 1, 'PB013057': 1, 'PB047717': 1, 'PB008258': 1, 'PF09121': 1, 'BRCC3,\"H2A': 1, 'BRCA1-A\",\"H2AK63': 1, '3\",BRCC3_HUMAN,JAB': 1, 'containing': 1, 'PF10513': 1, 'PF13831': 1, 'PF00855': 1, 'PF13832': 1, 'H3K14me2': 1, 'H4K5ac': 1, 'PB012618': 1, 'PB014336': 1, 'H3K14': 1, 'H4K5': 1, 'PB014896': 1, 'PB015118': 1, 'PB018611': 1, 'PB024632': 1, 'PB029595': 1, 'PB035798': 1, 'PB000361': 1, 'H3K14ac': 1, 'PF12024': 1, 'PB001543': 1, 'PB064177': 1, 'complex\",#,bromodomain': 1, 'PB019676': 1, 'PB055933': 1, 'PB008260': 1, 'H4K8ac\",\"bromodomain': 1, 'PB011430': 1, 'BRCC': 1, 'remodeling,mSin3A,#,breast': 1, 'PB005327': 1, 'PB018973': 1, 'PB022078': 1, 'PB027858': 1, 'PB000071': 1, 'PB008184': 1, 'PF00400': 1, 'PB012397': 1, 'PB002569': 1, 'PB010468': 1, 'PB010943': 1, 'C14orf169,\"H3K4me2': 1, 'H3K4': 1, 'H3K4me1': 1, 'PF08007': 1, 'PB041208': 1, 'PB047541': 1, '17': 1, 'CARM1,\"H3R17me': 1, 'PF05185': 1, 'PB010255': 1, 'PB013059': 1, 'modification,WMM,A': 1, 'homolog': 1, 'PF01393': 1, 'PB001400': 1, 'PB016393': 1, 'PB026041': 1, 'PB008537': 1, 'PB041557': 1, 'PB049897': 1, 'remodeling,#,#,cell': 1, 'PF09079': 1, 'CDK2,\"H1Sph': 1, 'Y-linked': 1, 'PF00385': 1, 'PF00378': 1, 'PB051834': 1, 'PB014969': 1, '2\",CDYL2_HUMAN,\"Chromo': 1, 'CERF\",\"H2A': 1, 'eye': 1, '2\",CECR2_HUMAN,\"Bromodomain': 1, 'PB013010': 1, 'PB018245': 1, 'PB020164': 1, 'modification,#,mRNA,CUGBP': 1, 'modification,#,C,centromere': 1, 'PF15620': 1, 'PF15622': 1, 'remodeling,CAF-1,\"H3': 1, 'assembly': 1, 'A': 1, 'PF15539': 1, 'PF15557': 1, 'PF12253': 1, 'remodeling,\"WINAC': 1, 'CAF-1\",\"H3': 1, 'remodeling,#,#,chromodomain': 1, 'PF13907': 1, 'PF00271': 1, 'PB000182': 1, 'PB091833': 1, 'PB002492': 1, 'PB019320': 1, 'PB000761': 1, 'remodeling,NuRD,#,chromodomain': 1, 'PF08073': 1, 'PF06461': 1, 'PF06465': 1, 'remodeling,#,\"H3K27me3': 1, 'helicase': 1, 'PB020089': 1, 'PB018962': 1, 'PB023619': 1, 'PB071010': 1, 'PB019729': 1, 'PB019848': 1, 'PB002227': 1, 'PB002565': 1, 'PB006146': 1, 'remodeling,\"CHD8': 1, 'PB010410': 1, 'PB012479': 1, 'PB013819': 1, 'PB020709': 1, 'PB006192': 1, 'PB007386': 1, 'PB008182': 1, 'PB008545': 1, 'PB001107': 1, 'PB015117': 1, 'PB024836': 1, 'chaperone,CHRAC,#,chromatin': 1, 'PB015051': 1, 'PF10197': 1, 'PB016354': 1, 'PB011694': 1, 'PB000387': 1, 'PF00433': 1, 'phosphorylation\"': 1, 'nucleotide-sensitive': 1, 'PB077655': 1, 'PF03517': 1, 'circadian': 1, 'PB003081': 1, 'PB007827': 1, 'PB008264': 1, 'PF02210': 1, 'PB003557': 1, 'PB079172': 1, 'PF12661': 1, 'PF09030': 1, 'PF06001': 1, 'PF08214': 1, 'PF02172': 1, 'PB002669': 1, 'PB000817': 1, 'PF00569': 1, 'PF02135': 1, 'modification,RING2-FBRS,#,\"casein': 1, 'PB006449': 1, 'PB008520': 1, 'remodeling,LSD-CoREST,#,C-terminal': 1, 'PF02826': 1, 'CTBP2,\"H3K9me': 1, 'motif,CCCTC-binding': 1, 'PF00096': 1, 'PF13894': 1, 'remodeling,#,#,CCCTC-binding': 1, 'PB059982': 1, 'polymerase': 1, 'PF00515': 1, 'PF07719': 1, 'CUL1,\"H3K9': 1, 'H3K36': 1, 'H3K36me3': 1, '1,CUL1_HUMAN,\"Cullin': 1, 'PF10557': 1, '4A,CUL4A_HUMAN,\"Cullin': 1, '4B,CUL4B_HUMAN,\"Cullin': 1, 'TF\",COMPASS,\"CG': 1, 'motif\",CXXC': 1, 'PF02008': 1, 'PF12269': 1, 'PF03178': 1, 'PF10433': 1, 'PB000431': 1, 'DDX17,#,RNA modification,#,mRNA,Probable ATP-dependent RNA helicase DDX17 (EC 3.6.4.13) (DEAD box protein 17) (DEAD box protein p72) (DEAD box protein p82) (RNA-dependent helicase p72),DDX17_HUMAN,PF00270': 1, 'PF00271,DDX17_MOUSE,Alternative splicing': 1, 'modification,B-WICH,#,DEAD': 1, 'PF08152': 1, 'PB047445': 1, 'DDX5,#,RNA modification,#,RNA,Probable ATP-dependent RNA helicase DDX5 (EC 3.6.4.13) (DEAD box protein 5) (RNA helicase p68),DDX5_HUMAN,PF00270': 1, 'PF08061,DDX5_MOUSE,Alternative splicing': 1, 'modification,#,#,DEAD': 1, 'remodeling,B-WICH,H4,DEK': 1, 'PB010960': 1, 'PF02037': 1, 'DHX9,#,RNA modification,#,mRNA,ATP-dependent RNA helicase A (EC 3.6.4.13) (DEAH box protein 9) (DExH-box helicase 9) (Leukophysin) (LKP) (Nuclear DNA helicase II) (NDH II) (RNA helicase A),DHX9_HUMAN,PF00270': 1, 'PF00035': 1, 'PF04408': 1, 'PF07717,DHX9_MOUSE,Alternative splicing': 1, 'remodeling,\"NuA4': 1, 'methyltransferase': 1, 'C': 1, '1\",DNJC1_HUMAN,\"DnaJ': 1, 'PF00249': 1, 'PB008255': 1, 'deacetylation\"': 1, '2\",DNJC2_HUMAN,\"DnaJ': 1, 'PB024524': 1, 'PB030652': 1, 'modification,#,#,DND': 1, 'modification,#,dhC,DNA': 1, 'PF06464': 1, 'PF00145': 1, 'PF12047': 1, 'PB041249': 1, 'H2B\",\"deoxynucleotidyltransferase': 1, 'terminal': 1, 'PB030738': 1, 'BRM-BRG1\",#,\"D4': 1, 'PB077905': 1, 'PF14051': 1, 'remodeling,SWI/SNF': 1, 'remodeling,\"BAF': 1, 'BRM-BRG1\",\"H3': 1, 'H4\",\"D4': 1, '3\",DPF3_HUMAN,\"PHD': 1, 'MLL3,4\",#,dpy-30': 1, 'PB028037': 1, 'chaperone,ATAC,#,\"down-regulator': 1, '(negative': 1, 'ubiquitin': 1, 'PF13639': 1, 'PB005525': 1, 'PB005636': 1, 'PB008501': 1, 'PB008729': 1, 'PB006684': 1, 'modification,#,Lys165': 1, 'K36,Protein': 1, 'modification,#,Lys55': 1, 'EHMT1,\"H3K9me1': 1, 'PF13637': 1, 'PB028560': 1, 'PF05033': 1, 'EHMT2,\"H3K9me1': 1, 'PB009355': 1, 'PB007728': 1, 'EIF4A3,#,RNA modification,#,mRNA,\"Eukaryotic initiation factor 4A-III (eIF-4A-III) (eIF4A-III) (EC 3.6.4.13) (ATP-dependent RNA helicase DDX48) (ATP-dependent RNA helicase eIF4A-3) (DEAD box protein 48) (Eukaryotic initiation factor 4A-like NUK-34) (Eukaryotic translation initiation factor 4A isoform 3) (Nuclear matrix protein 265) (NMP 265) (hNMP 265) [Cleaved into: Eukaryotic initiation factor 4A-III, N-terminally processed]\",IF4A3_HUMAN,PF00270': 1, 'PF00271,IF4A3_MOUSE,Alternative splicing': 1, 'acetyltransferase': 1, 'PB013823': 1, 'PF04055': 1, 'binding': 1, 'PB015526': 1, 'complex\",#,E1A': 1, 'PB018422': 1, 'Piccolo_NuA4': 1, 'complex\",#,enhancer': 1, 'PF06752': 1, 'remodeling,#,#,enhancer': 1, 'PB018248': 1, 'PB019406': 1, 'PF14843': 1, 'PB010445': 1, 'PF07714': 1, 'PF01030': 1, 'remodeling,B-WICH,#,excision': 1, 'PB058020': 1, 'modification\",RNA': 1, 'PF10447': 1, 'PF01138': 1, 'PF03725': 1, 'PB003150': 1, 'EZH1,\"H3K27me1': 1, 'H3K27me2': 1, 'EZH2,\"H3K27me1': 1, 'protein,BRCA1-A,#,\"family': 1, 'A\",F175A_HUMAN,Pfam-B_8506': 1, 'B\",F175B_HUMAN,Pfam-B_24421': 1, 'modification,RING2-FBRS,#,fibrosin,FBRS_HUMAN,\"Auts2': 1, 'PB058582': 1, 'modification,RING2-FBRS,#,fibrosin-like': 1, 'PB017807': 1, 'PB187339': 1, 'PB052362': 1, 'motif,forkhead': 1, 'PF08430': 1, 'PF09354': 1, 'FTO,\"hm6A': 1, 'f6A': 1, 'demethylation': 1, 'arrest': 1, 'PF01248': 1, 'PB013145': 1, 'PB017231': 1, 'zinc': 1, 'remodeling,#,#,growth': 1, 'PF03535': 1, 'PF03446': 1, 'coiled-coil': 1, 'PB041097': 1, 'PB049843': 1, '4': 1, 'PF12657': 1, 'PF12660': 1, 'remodeling,\"NSL': 1, 'COMPASS': 1, 'cell': 1, 'PF13415': 1, 'PB049758': 1, 'PF13854': 1, 'acetylation\"': 1, 'NuRD': 1, 'BHC': 1, 'MeCP1': 1, 'mSin3A': 1, 'HDAC': 1, 'RING2-L3MBTL2': 1, 'CREST-BRG1': 1, 'LSD-CoREST\",\"H3': 1, 'deacetylase': 1, 'PB022185': 1, 'HDAC10,\"H2AK': 1, 'H2BK': 1, 'H3K': 1, 'H2BKac': 1, 'H3Kac': 1, 'HDAC11,\"H2AK': 1, 'HDAC2,\"H2AK': 1, 'LSD-CoREST\",\"H2AKac': 1, 'PB046198': 1, 'HDAC3,\"H2AK': 1, 'PB003137': 1, 'HDAC4,\"H2AK': 1, 'PF12203': 1, 'PF00850': 1, 'HDAC5,\"H2AK': 1, 'HDAC6,\"H2AK': 1, 'PF02148': 1, 'HDAC7,\"H2AK': 1, 'PB005922': 1, 'HDAC8,\"H2AK': 1, 'PB015628': 1, 'HDAC9,\"H3K': 1, 'PB005353': 1, 'motif,hepatoma-derived': 1, 'H4K20me3': 1, 'like': 1, 'PB032406': 1, 'remodeling,#,#,\"helicase': 1, 'subunit': 1, 'TF\",#,\"H4': 1, 'motif\",histone': 1, 'PF07569': 1, 'PB018560': 1, 'interacting': 1, 'chaperone,#,H4,Holliday': 1, 'PF12346': 1, 'PB000631': 1, 'PF10384': 1, 'synthetase': 1, 'PF03099': 1, 'PB033359': 1, 'PB035799': 1, 'remodeling,\"BHC': 1, 'mobility': 1, 'PB010576': 1, 'remodeling,#,#,high': 1, 'PF09011': 1, 'remodeling,#,H1,high': 1, 'HNRNPU,#,RNA modification,#,mRNA,Heterogeneous nuclear ribonucleoprotein U (hnRNP U) (GRIP120) (Nuclear p120 ribonucleoprotein) (Scaffold-attachment factor A) (SAF-A) (p120) (pp120),HNRPU_HUMAN,PF02037': 1, 'PF00622,HNRPU_MOUSE,Alternative splicing': 1, 'HNRPL,#,RNA modification,#,mRNA,Heterogeneous nuclear ribonucleoprotein L (hnRNP L),HNRPL_HUMAN,PF00076': 1, 'PF11835,HNRPL_MOUSE,Alternative splicing': 1, 'HNRPM,#,RNA modification,#,mRNA,Heterogeneous nuclear ribonucleoprotein M (hnRNP M),HNRPM_HUMAN,PF11532': 1, 'PF00076,HNRPM_MOUSE,Alternative splicing': 1, 'remodeling,#,#,\"heterochromatin': 1, 'growth': 1, 'PB040014': 1, 'PB080075': 1, 'ubiquitination\"': 1, 'shock': 1, 'PF06012': 1, 'PF06025': 1, 'PF00632': 1, 'PB013571': 1, 'PB015798': 1, 'PB020712': 1, 'PB006870': 1, 'PF00627': 1, 'PF02825': 1, 'protein,Pol2': 1, 'complex-associated': 1, 'PB008846': 1, 'motif,IKAROS': 1, '1\",ING1_HUMAN,\"ING': 1, 'PB000837': 1, '2\",ING2_HUMAN,\"ING': 1, 'Piccolo_NuA4\",\"H2A': 1, '3\",ING3_HUMAN,\"ING': 1, '4\",ING4_HUMAN,\"ING': 1, 'MOZ/MORF\",\"H3K4me3': 1, '5\",ING5_HUMAN,\"ING': 1, 'remodeling,Ino80,#,INO80': 1, 'PB016799': 1, 'PF04438': 1, 'PF13891': 1, 'JADE1,\"H3ac': 1, 'family': 1, 'JADE2,\"H3ac': 1, 'PB023296': 1, 'JADE3,\"H3ac': 1, 'PF00017': 1, 'H3K9\",\"jumonji': 1, 'rich': 1, 'PF02373': 1, 'PF02375': 1, 'PB000728': 1, 'PF02928': 1, 'PB014856': 1, 'PB015350': 1, 'PB016501': 1, 'PB017329': 1, 'PB019453': 1, 'PB002770': 1, 'PB063706': 1, 'JMJD6,\"H3R2': 1, 'PB022969': 1, 'regulatory': 1, 'PB014811': 1, 'PB019390': 1, 'PB002194': 1, 'PB006892': 1, 'PB010092': 1, 'PB008995': 1, 'SAGA': 1, 'ATAC': 1, 'PF06466': 1, 'PB010717': 1, 'PB358817': 1, 'KAT5,\"H2AK5ac': 1, 'H4K8ac': 1, 'H4K12ac': 1, 'Piccolo_NuA4\",\"H2AK5': 1, 'H4K5,H4K8': 1, 'H4K12': 1, 'KAT6A,\"H3ac': 1, 'PB019382': 1, 'PB021350': 1, 'PB025985': 1, 'PB003468': 1, 'PB006470': 1, 'PF01853': 1, 'PF01530': 1, 'KAT8,\"H2Aac': 1, 'MLL4/WBP7\",\"H2A': 1, 'KDM1A,\"H3K4': 1, 'SCL\",\"H3K4me1': 1, 'H3K4me2': 1, '(K)-specific': 1, 'PF04433': 1, 'PF07496': 1, 'PB011167': 1, 'PB006618': 1, 'PB009841': 1, 'KDM2B,\"H3K4': 1, 'PF00646': 1, 'PB023456': 1, 'PB033113': 1, 'PB014772': 1, 'PB017027': 1, 'PB006147': 1, 'PB007111': 1, 'PB016202': 1, 'PB016670': 1, 'PB019299': 1, 'PB022020': 1, 'PB003525': 1, 'PB038652': 1, 'KDM4A,\"H3K4me2': 1, 'KDM4B,\"H3K9me1': 1, 'PB012059': 1, 'KDM4C,\"H3K9me1': 1, 'PB010616': 1, 'PB014467': 1, 'PB005109': 1, 'KDM4D,\"H3K9me1': 1, 'PB045485': 1, 'PB084013': 1, 'PF08429': 1, 'KDM5C,\"H3K4me2': 1, 'PB012821': 1, 'MLL3,4\",H3K27me2.': 1, 'PB000604': 1, 'PB000972': 1, 'PF13431': 1, 'PF13181': 1, 'PB005108': 1, 'PB000992': 1, 'KDM7A,\"H3K9': 1, 'H3K27': 1, 'PB018122': 1, 'PB013686': 1, 'PB001060': 1, 'remodeling,#,#,kelch-like': 1, 'PF00651': 1, 'PF01344': 1, 'KHDRBS1,#,RNA modification,#,mRNA,\"KH domain-containing, RNA-binding, signal transduction-associated protein 1 (GAP-associated tyrosine phosphoprotein p62) (Src-associated in mitosis 68 kDa protein) (Sam68) (p21 Ras GTPase-activating protein-associated p62) (p68)\",KHDR1_HUMAN,PF00013': 1, 'PF16274': 1, 'PF16568,KHDR1_MOUSE,Alternative splicing': 1, 'MLL1,2\",H3K4,lysine': 1, 'PF05964': 1, 'PB011202': 1, 'PF13771': 1, 'MLL3,4\",H3K4,lysine': 1, 'PB025060': 1, 'PB025524': 1, 'PB026440': 1, 'PB027348': 1, 'PB003353': 1, 'PB006332': 1, 'PB085509': 1, 'PB016694': 1, 'PB019991': 1, 'PB000043': 1, 'PB049476': 1, 'PB026489': 1, 'PB029065': 1, 'PB002976': 1, 'PB033020': 1, 'KMT2E,\"H3K4me1': 1, 'PB013611': 1, 'PB019470': 1, 'PB019601': 1, 'PB020626': 1, 'PB022469': 1, 'PB024586': 1, 'PB007010': 1, 'PB007713': 1, 'PF00536': 1, 'H3K9': 1, '2': 1, 'PB029190': 1, 'LAS1L,\"H3K4me': 1, 'H3Ac': 1, 'H4Ac': 1, 'MLL4/WBP7\",\"H3K4': 1, '(S.': 1, 'PB028535': 1, 'PB017842': 1, 'PB020153': 1, 'PB006431': 1, 'remodeling,#,\"H3K9me3': 1, 'repeats': 1, 'PB028625': 1, 'PB039252': 1, 'modification,#,mRNA,Protein': 1, 'remodeling,#,#,mitogen-activated': 1, 'MASTL,\"H1p': 1, 'PB008409': 1, 'TF\",\"CHD8': 1, 'motif,MYC': 1, 'acetylation': 1, 'remodeling,#,#,MYC-associated': 1, 'TF\",#,\"mCG': 1, 'motif\",methyl-CpG': 1, 'PB012306': 1, 'PB009691': 1, 'TF\",\"NuRD': 1, 'MeCP1\",\"mCG': 1, 'PF14048': 1, 'modification,#,\"G:T': 1, 'PB047325': 1, 'remodeling,#,#,methyl-CpG': 1, 'PB019258': 1, 'PB041495': 1, 'PB051737': 1, 'PB005417': 1, 'PB007705': 1, 'PB000276': 1, 'modification,#,mRNA,Muscleblind-like': 1, 'PB045320': 1, 'MCRS1,\"H4K5ac': 1, 'NSL': 1, 'MLL4/WBP7\",\"H4K5': 1, 'PF13325': 1, 'PB001811': 1, 'MEAF6,\"H2Aac': 1, 'MOZ/MORF\",\"H2A': 1, 'factor': 1, 'motif\",methyl': 1, 'endocrine': 1, 'modification,#,U6': 1, 'modification,#,Hsp70,methyltransferase': 1, 'modification\",#,\"A': 1, 'snRNA\",methyltransferase': 1, 'methylation\"': 1, 'TF\",\"RING2-L3MBTL2': 1, 'motif,\"MGA': 1, 'dimerization': 1, 'PB011518': 1, 'PB024878': 1, 'PB000591': 1, 'PB007320': 1, 'PB080122': 1, 'PB008542': 1, 'PF00907': 1, 'MGEA5,\"H3K14ac': 1, 'expressed': 1, 'PB015148': 1, 'remodeling,Mis18,CENPA,MIS18': 1, 'MIS18BP1,#,Chromatin remodeling,Mis18,CENPA,MIS18 binding protein 1,M18BP_HUMAN,PF00249': 1, ' PF09133,M18BP_MOUSE,#': 1, 'MLLT1,#,Chromatin remodeling cofactor,\"SWI/SNF-like_EPAFa, SWI/SNF-like EPAFB\",#,\"myeloid/lymphoid or mixed-lineage leukemia (trithorax homolog, Drosophila)': 1, ' translocated to, 1\",ENL_HUMAN,YEATS PF03366 29-112,#,#': 1, 'MLLT10,H3K79me3,Histone modification write cofactor,#,H3K79,\"myeloid/lymphoid or mixed-lineage leukemia (trithorax homolog, Drosophila)': 1, ' translocated to, 10\",AF10_HUMAN,\"PHD_2 PF13831 36-72, Pfam-B_12689 PB012689 541-619, Pfam-B_12729 PB012729 821-879, Pfam-B_4074 PB004074 234-538, Pfam-B_6933 PB006933 721-819, Pfam-B_9291 PB009291 621-719, zf-HC5HC2H_2 PF13832 80-197\",AF10_MOUSE,Histone methylation': 1, 'MLLT6,H3K79me3,Histone modification write cofactor,#,H3K79,\"myeloid/lymphoid or mixed-lineage leukemia (trithorax homolog, Drosophila)': 1, ' translocated to, 6\",AF17_HUMAN,\"Pfam-B_16181 PB016181 791-1091, Pfam-B_3448 PB003448 551-789, zf-HC5HC2H_2 PF13832 63-180\",#,Histone methylation': 1, 'PB026706': 1, 'phosphoprotein': 1, 'MPND,H2A,Histone modification erase,#,H2A-Ub (K119),MPN domain-containing,MPND_HUMAN,PF01398': 1, ' PF18755,MPND_MOUSE,Histone ubiquitination': 1, 'PF05188': 1, 'PF05192': 1, 'PF05190': 1, 'PF00488': 1, 'PB041610': 1, 'PB070275': 1, 'PB019348': 1, 'PB020099': 1, 'PB022844': 1, 'modification,#,#,macrophage': 1, 'PF00024': 1, 'PF00089': 1, 'PF01448': 1, 'PF00320': 1, '2\",MTA2_HUMAN,\"BAH': 1, 'PB003593': 1, '3\",MTA3_HUMAN,\"BAH': 1, 'PB011869': 1, 'PB022870': 1, 'K3K79me': 1, 'PF06017': 1, 'PF00063': 1, 'PB030622': 1, 'NAA60,\"H4K20ac': 1, 'H4K79ac': 1, 'H4K79': 1, '60': 1, 'catalytic': 1, 'chaperone,#,#,nucleosome': 1, 'remodeling,#,H1,nuclear': 1, 'NAT10,ac4C,RNA modification,#,C of mRNA,N-acetyltransferase 10,NAT10_HUMAN,PF08351': 1, 'PF13718': 1, 'PF05127': 1, 'PF13725,NAT10_MOUSE,RNA acetylation': 1, 'PF13725': 1, 'remodeling,#,H2AX,nibrin,NBN_HUMAN,\"FHA': 1, 'PF08599': 1, 'chaperone,#,\"DNA': 1, 'motif\",nucleolin,NUCL_HUMAN,\"Pfam-B_12581': 1, 'receptor': 1, 'PF08815': 1, 'PF08832': 1, 'PB010879': 1, 'PB014009': 1, 'MLL3,4\",#,nuclear': 1, 'PB014375': 1, 'PB003537': 1, 'PB004054': 1, 'PB000670': 1, 'PB007837': 1, 'PB001539': 1, 'NEK6,\"H1ph': 1, 'PF00415': 1, 'motif,\"nuclear': 1, 'PF00808': 1, 'PB017851': 1, 'modification,#,#,\"nuclear': 1, 'PB008767': 1, 'PF12830': 1, 'PB000338': 1, 'PB004450': 1, 'PB005032': 1, 'PB022168': 1, 'PB002429': 1, 'PB050064': 1, 'motif,neuronal': 1, '(nucleolar': 1, 'PF03066': 1, 'chaperone,#,\"H2A': 1, '2,NPM2_HUMAN,TSP_1': 1, 'NSD1,\"H3K36me': 1, 'PB007228': 1, 'kinetochore': 1, 'modification,#,mRNA,Nuclear': 1, 'modification,#,C': 1, 'NTMT1,\"Km1/m2/m3': 1, 'Rm': 1, 'CENP-B': 1, 'Xaa-Pro-Lys': 1, 'NUP98,#,RNA modification,#,mRNA,Nuclear pore complex protein Nup98-Nup96 (EC 3.4.21.-) [Cleaved into: Nuclear pore complex protein Nup98 (98 kDa nucleoporin) (Nucleoporin Nup98) (Nup98)': 1, ' Nuclear pore complex protein Nup96 (96 kDa nucleoporin) (Nucleoporin Nup96) (Nup96)],NUP98_HUMAN,PF04096': 1, 'PF12110,NUP98_MOUSE,Alternative splicing': 1, 'PF13414': 1, 'remodeling,Mis18,CENPA,Opa': 1, 'PADI1,\"H2ARci': 1, 'H3Rci': 1, 'modification,#,\"H2AR': 1, 'H3R': 1, 'arginine': 1, 'I\",PADI1_HUMAN,\"PAD': 1, 'PF08527': 1, 'PF08526': 1, 'PADI2,\"H2ARci': 1, 'II\",PADI2_HUMAN,\"PAD': 1, 'PADI3,\"H2ARci': 1, 'III\",PADI3_HUMAN,\"PAD': 1, 'PADI4,\"H2ARci': 1, 'H3R2ci': 1, 'H3R8ci': 1, 'H3R17ci': 1, 'H3R26ci': 1, 'H3R2': 1, 'H3R8': 1, 'H3R17': 1, 'H3R26': 1, 'IV\",PADI4_HUMAN,\"PAD': 1, 'H3\",\"Paf1': 1, 'MLL3,4\",#,PAXIP1': 1, 'remodeling,#,H3K9,poly': 1, 'PB014007': 1, 'remodeling,#,H1,poly': 1, 'PF08063': 1, 'PF00644': 1, 'PF02877': 1, 'PF05406': 1, 'PF00645': 1, '3\",PARP3_HUMAN,\"PARP': 1, 'MLL3,4\",H3K4,PAX': 1, 'PF12738': 1, 'PB013490': 1, 'PB016838': 1, 'BRM-BRG1\",H3,polybromo': 1, 'PF00505': 1, 'PF00705': 1, 'PB013776': 1, 'MLL4/WBP7\",\"H1': 1, 'H4\",\"proline': 1, 'PF08167': 1, 'PB005821': 1, 'PF06467': 1, 'PB006832': 1, 'PB017091': 1, 'PB019383': 1, 'PB019396': 1, 'PB018040': 1, 'PB024844': 1, 'remodeling,\"npBAF': 1, 'BRM-BRG1\",#,PHD': 1, 'PB001316': 1, 'PB028476': 1, 'PHF14,\"H2Aac': 1, 'finger': 1, 'PB016803': 1, 'PB019012': 1, 'PB031053': 1, 'PB003847': 1, 'PB021846': 1, 'PB023453': 1, 'PB037699': 1, 'PB015568': 1, 'PB007050': 1, 'PHF8,\"H3K9': 1, 'H4K20': 1, 'H3K9me2': 1, 'H4K20me1': 1, 'H3K36me2': 1, 'PB022520': 1, 'PB005629': 1, 'PB015784': 1, 'PB060538': 1, 'PB002782': 1, 'PB034294': 1, 'PF02171': 1, 'PKM,\"H3S10ph': 1, 'H3S28ph': 1, 'H2BS32ph': 1, 'H3S28': 1, 'PF00224': 1, 'PF02887': 1, 'PB033979': 1, 'PF03221': 1, 'PB010413': 1, 'PB010981': 1, 'PB005179': 1, 'PB006754': 1, 'chaperone,CHRAC,#,\"polymerase': 1, '3': 1, 'subunit\",DPOE3_HUMAN,\"CBFD_NFYB_HMF': 1, 'PB035841': 1, 'remodeling,#,#,\"protein': 1, 'dependent': 1, 'PF00481': 1, 'isozyme\",PP2AA_HUMAN,Metallophos': 1, 'subunit\",PP4C_HUMAN,Metallophos': 1, 'modification,#,mRNA,Polyglutamine-binding': 1, 'ZNF': 1, 'modification,#,mC,PR': 1, 'TF\",#,\"H3K9': 1, 'motif\",PR': 1, 'PF13912': 1, 'PF12171': 1, 'PF12874': 1, 'PRDM6,\"H3R2me1': 1, 'H3R2me2': 1, 'PF09514': 1, 'AMP-activated': 1, 'PB005182': 1, 'PF00130': 1, 'PF00168': 1, 'modification,#,#,\"protein': 1, 'PRKDC,\"H2AXS139ph': 1, 'DNA-activated': 1, 'polypeptide\",PRKDC_HUMAN,\"FAT': 1, 'PF08163': 1, 'PB014440': 1, 'PB001565': 1, 'PB009125': 1, 'PRMT1,\"H4R3me1': 1, 'PF14604': 1, 'PRMT5,\"H3R8me': 1, 'PB018578': 1, 'PRMT6,\"H2AR3me': 1, 'H4R3me': 1, 'PRMT7,\"H2A': 1, 'PF06325': 1, 'processing': 1, 'PF01798': 1, 'PB023523': 1, 'PF09785': 1, 'remodeling,#,#,PC4': 1, 'PTBP1,#,RNA modification,#,mRNA,Polypyrimidine tract-binding protein 1 (PTB) (57 kDa RNA-binding protein PPTB-1) (Heterogeneous nuclear ribonucleoprotein I) (hnRNP I),PTBP1_HUMAN,PF00076': 1, 'PF11835,PTBP1_MOUSE,Alternative splicing': 1, 'modification,#,mRNA,Poly(U)-binding-splicing': 1, 'PF08423': 1, 'remodeling,#,#,RAD54': 1, 'PB010171': 1, 'remodeling,#,#,RAD54-like': 1, 'PB014570': 1, 'PF08658': 1, 'PB002830': 1, 'motif\",recombination': 1, 'PF12940': 1, 'PF10426': 1, 'PF13341': 1, 'remodeling,#,#,retinoic': 1, 'PB027361': 1, 'PB006090': 1, 'RARA,\"H3K4me': 1, 'acid': 1, 'PF00104': 1, 'PF00105': 1, '1,RB_HUMAN,\"DUF3452': 1, 'PF01858': 1, 'PF01857': 1, 'PF08934': 1, 'chaperone,\"NuRF': 1, 'PRC2': 1, 'RBBP5,\"H3K4me1': 1, 'MLL3,4\",H3K4,retinoblastoma': 1, 'RBFOX1,#,RNA modification,#,mRNA,\"RNA-binding motif protein, X chromosome (Glycoprotein p43) (Heterogeneous nuclear ribonucleoprotein G) (hnRNP G) [Cleaved into: RNA-binding motif protein, X chromosome, N-terminally processed]\",RBMX_HUMAN,PF08081': 1, 'PF00076,RBMX_MOUSE,Alternative splicing': 1, 'modification,#,mRNA,Splicing': 1, 'RBM15,m6A,RNA modification,WMM,A of mRNA,RNA-binding protein 15 (One-twenty two protein 1) (RNA-binding motif protein 15),RBM15_HUMAN,PF00076': 1, 'PF07744,RBM15_MOUSE,RNA methylation': 1, 'RBM15B,m6A,RNA modification,WMM,A of mRNA,Putative RNA-binding protein 15B (One-twenty two protein 3) (HsOTT3) (HuOTT3) (RNA-binding motif protein 15B),RB15B_HUMAN,PF00076': 1, 'PF07744,RB15B_MOUSE,RNA methylation': 1, 'RBM17,#,RNA modification,#,mRNA,Splicing factor 45 (45 kDa-splicing factor) (RNA-binding motif protein 17),SPF45_HUMAN,PF01585': 1, 'PF00076,SPF45_MOUSE,Alternative splicing': 1, 'modification,#,mRNA,RNA-binding': 1, 'RBM25,#,RNA modification,#,RNA,RNA-binding protein 25 (Arg/Glu/Asp-rich protein of 120 kDa) (RED120) (Protein S164) (RNA-binding motif protein 25) (RNA-binding region-containing protein 7),RBM25_HUMAN,PF01480': 1, 'PF00076,RBM25_MOUSE,Alternative splicing': 1, 'RBM4,#,RNA modification,#,RNA,RNA-binding protein 4 (Lark homolog) (hLark) (RNA-binding motif protein 4) (RNA-binding motif protein 4a),RBM4_HUMAN,PF00076': 1, 'PF00098,RBM4_MOUSE,Alternative splicing': 1, 'RBM5,#,RNA modification,#,mRNA,RNA-binding protein 5 (Protein G15) (Putative tumor suppressor LUCA15) (RNA-binding motif protein 5) (Renal carcinoma antigen NY-REN-9),RBM5_HUMAN,PF01585': 1, 'PF17780': 1, 'PF00641,RBM5_MOUSE,Alternative splicing': 1, 'modification,#,snRNA,RNA-binding': 1, 'splicing\"': 1, 'RBMY1A1,#,RNA modification,#,mRNA,\"RNA-binding motif protein, Y chromosome, family 1 member A1 (RNA-binding motif protein 1) (RNA-binding motif protein 2) (Y chromosome RNA recognition motif 1) (hRBMY)\",RBY1A_HUMAN,PF08081': 1, 'PF00076,RBY1A_MOUSE,Alternative splicing': 1, 'RBX1,\"H3ub': 1, 'SCL': 1, 'corepressor': 1, 'motif,RE1-silencing': 1, 'PB009727': 1, 'RFOX1,#,RNA modification,#,mRNA,RNA binding protein fox-1 homolog 1 (Ataxin-2-binding protein 1) (Fox-1 homolog A) (Hexaribonucleotide-binding protein 1),RFOX1_HUMAN,PF12414': 1, 'PF00076,RFOX1_MOUSE,Alternative splicing': 1, 'BCOR': 1, 'PB013488': 1, 'PB023468': 1, 'modification,#,#,RecQ': 1, 'PB000227': 1, 'RNF168,\"H2AK13ub': 1, 'H2AK15ub': 1, 'H2AXK13ub': 1, 'H2AK15': 1, 'H2AXK13': 1, 'PF13923': 1, 'RING2-FBRS': 1, 'PB014933': 1, 'PB006844': 1, 'PB009159': 1, 'RNF8,\"H2AK63ub': 1, 'H2AXK63ub': 1, 'H2AK48ub': 1, 'H2AXK63': 1, 'H2AK48': 1, '90kDa': 1, '3\",KS6A3_HUMAN,\"Pkinase': 1, 'RPS6KA4,\"H3S10ph': 1, '4\",KS6A4_HUMAN,\"Pkinase': 1, 'RPS6KA5,\"H2AS1ph': 1, 'H3S10ph': 1, 'H3S10': 1, '5\",KS6A5_HUMAN,\"Pkinase': 1, 'modification,#,mt--mRNA,Mitochondrial': 1, '(yeast)\",RRP8_HUMAN,Methyltransf_8': 1, 'modification,#,mRNA,Serine/Arginine-related': 1, 'SWR': 1, 'AAA': 1, 'PF00641': 1, 'remodeling,#,#,scaffold': 1, 'PB010398': 1, 'PB013351': 1, 'PB017122': 1, 'PB018714': 1, 'PB000237': 1, 'PF06487': 1, 'PF15476': 1, 'complex\",#,\"Sin3A-associated': 1, 'PF13867': 1, 'PF13866': 1, 'PF00046': 1, 'PB009146': 1, 'PB008103': 1, 'PF02820': 1, 'PB002161': 1, 'PB034931': 1, 'specific': 1, 'modification,#,#,SET': 1, 'PB024657': 1, 'PB024930': 1, 'PB041492': 1, 'PB001367': 1, 'PB014013': 1, 'PB022486': 1, 'PB002796': 1, 'PB003607': 1, 'PB067824': 1, 'PF08236': 1, 'PF09273': 1, '1\",SETB1_HUMAN,\"MBD': 1, 'PB023747': 1, 'PB027578': 1, '2\",SETB2_HUMAN,\"MBD': 1, 'SETMAR,\"H3K4me': 1, 'PF01359': 1, 'modification,B-WICH,#,\"splicing': 1, 'PB001720': 1, 'PF08920': 1, 'modification,#,#,\"splicing': 1, 'PB019576': 1, 'SFMBT2,\"H3': 1, 'TF\",#,\"H3K9me2': 1, 'H3K27me3': 1, 'H4K20me2': 1, 'with': 1, 'SFSWAP,#,RNA modification,#,mRNA,\"Splicing factor, suppressor of white-apricot homolog (Splicing factor, arginine/serine-rich 8) (Suppressor of white apricot protein homolog)\",SFSWA_HUMAN,PF09750': 1, 'PF01805,SFSWA_MOUSE,Alternative splicing': 1, 'PF00538': 1, 'PB011163': 1, 'TF\",\"SWI/SNF_Brg1(I)': 1, 'complex\",DNA': 1, 'PB010250': 1, 'PB010714': 1, 'PB008471': 1, 'PF08295': 1, 'motif,SIN3': 1, 'SIRT1,\"H1K26': 1, 'H3K9ac': 1, '1,SIR1_HUMAN,\"Pfam-B_28511': 1, 'PB005913': 1, 'PB008042': 1, 'PF02146': 1, 'SIRT2,\"H3K18': 1, 'H3K56': 1, 'H4K16': 1, 'H3K56ac': 1, 'H4K16ac': 1, '2,SIR2_HUMAN,SIR2': 1, 'SIRT6,\"H3K9': 1, '6,SIR6_HUMAN,SIR2': 1, 'PF03931': 1, 'modification,#,mRNA,Pre-mRNA-splicing': 1, 'CERF': 1, 'related': 1, 'a': 1, '1\",SMCA1_HUMAN,\"DBINO': 1, 'PF09110': 1, 'PF09111': 1, 'TF\",\"BAF': 1, 'motif\",\"SWI/SNF': 1, '2\",SMCA2_HUMAN,\"BRK': 1, 'PF07529': 1, 'PB017897': 1, 'PB002366': 1, 'PF08880': 1, 'PF14619': 1, 'CREST-BRG1\",\"H3': 1, '4\",SMCA4_HUMAN,\"BRK': 1, 'remodeling,\"ACF': 1, 'B-WICH': 1, 'RSF': 1, 'CHRAC': 1, '5\",SMCA5_HUMAN,\"DBINO': 1, 'PB036230': 1, 'remodeling,#,#,\"SWI/SNF-related': 1, 'actin-dependent': 1, 'DEAD/H': 1, 'PB000295': 1, 'remodeling,#,#,\"SWI/SNF': 1, 'a-like': 1, 'PB007889': 1, 'BRM-BRG1\",H3K56,\"SWI/SNF': 1, 'b': 1, '1\",SNF5_HUMAN,\"Pfam-B_9879': 1, 'PF04855': 1, 'BRM-BRG1\",#,\"SWI/SNF': 1, 'c': 1, '1\",SMRC1_HUMAN,\"Myb_DNA-binding': 1, '2\",SMRC2_HUMAN,\"Myb_DNA-binding': 1, 'PB001849': 1, 'd': 1, '1\",SMRD1_HUMAN,\"Pfam-B_13759': 1, 'PB003130': 1, 'PF02201': 1, '2\",SMRD2_HUMAN,\"Pfam-B_13705': 1, 'PB007929': 1, '3\",SMRD3_HUMAN,\"Pfam-B_18257': 1, 'e': 1, '1\",SMCE1_HUMAN,\"HMG_box': 1, 'PB014199': 1, 'PB000334': 1, 'PF04802': 1, 'PF01753': 1, 'SMYD2,\"H3K4me': 1, 'SMYD3,\"H3K4me2': 1, 'H3K5me2': 1, 'PB050317': 1, 'motif,Sp1': 1, 'PB007620': 1, 'PB182547': 1, 'PB041802': 1, 'nuclear': 1, 'PB017673': 1, 'PB017985': 1, 'PB002825': 1, 'PF13893': 1, 'PF07744': 1, 'PF00917': 1, 'CREBBP': 1, 'modification,#,mRNA,Serine/arginine': 1, 'modification,#,#,serine/arginine-rich': 1, 'modification,#,mRNA,\"Serine/arginine-rich': 1, '13A)': 1, '13B)': 1, '19)\",SRS12_HUMAN,PF00076,SRS12_MOUSE,Alternative': 1, '6)\",SRSF6_HUMAN,PF00076,SRSF6_MOUSE,Alternative': 1, 'remodeling,CREST-BRG1,#,synovial': 1, 'PB022298': 1, 'PF05030': 1, 'remodeling,#,#,synovial': 1, 'remodeling,FACT,\"H3': 1, 'PB009245': 1, 'PF08512': 1, 'PF03531': 1, 'PB012195': 1, 'FACT\",\"H2A': 1, 'PF00557': 1, 'PB018241': 1, 'PF08644': 1, 'PF14635': 1, 'PF14641': 1, 'PB017741': 1, 'PB027919': 1, 'PF00575': 1, 'PF14633': 1, 'PF14632': 1, 'PF14639': 1, 'chaperone,\"TFTC-HAT': 1, 'PB018361': 1, 'PB018885': 1, 'H3K9me1': 1, 'PB024134': 1, 'PB032221': 1, 'PB049669': 1, 'PB059617': 1, 'PB006511': 1, 'motif,SUZ12': 1, 'modification,APOB_mRNA_editosome,mRNA,\"synaptotagmin': 1, 'RNA': 1, 'chaperone,STAGA,H2A,transcriptional': 1, 'TF\",\"PCAF': 1, 'adaptor': 1, 'TFTC-HAT': 1, 'TAF1,\"H3ac': 1, 'MLL4/WBP7\",\"H3': 1, 'box': 1, 'PF12157': 1, 'PB010720': 1, 'PF09247': 1, 'PF15288': 1, 'STAGA\",\"H3': 1, 'PF03540': 1, 'PB025018': 1, 'PF03847': 1, 'PF07524': 1, 'PB041790': 1, 'PB007817': 1, 'PB000906': 1, 'PF05236': 1, 'PF07531': 1, 'PB022190': 1, 'PB034937': 1, 'PF04494': 1, 'PF07571': 1, 'PB025920': 1, 'PF02969': 1, 'chaperone,\"PCAF': 1, 'PB032143': 1, 'PB032396': 1, 'PF04658': 1, 'chaperone,#,#,\"TAF8': 1, 'PF10406': 1, 'STAGA': 1, 'PF02291': 1, 'chaperone,TFTC-HAT,#,\"TAF9B': 1, 'TDG,\"C': 1, '5mC': 1, 'modification,#,\"G:U': 1, 'G:T': 1, 'glycosylase,TDG_HUMAN,UDG': 1, 'PB027056': 1, 'PF00567': 1, 'modification,#,piRNA,tudor': 1, 'PB019308': 1, 'modification,#,mC,tet': 1, 'PB035074': 1, 'PB006123': 1, 'PF12851': 1, 'PB034477': 1, 'PB019260': 1, 'PB018291': 1, 'modification,RING2-L3MBTL2,#,transcription': 1, 'PF02319': 1, 'modification\",Ino80,#,TCF3': 1, 'PB026015': 1, 'modification,#,mRNA,Thyroid': 1, 'PF03920': 1, 'TLE2,\"H3K4': 1, 'TF\",#,\"H3ac': 1, 'enhancer': 1, 'PB009699': 1, 'PB003701': 1, 'remodeling,#,#,transition': 1, 'remodeling,#,#,\"tonsoku-like': 1, 'PF13516': 1, 'PB027656': 1, 'PB051992': 1, 'remodeling,#,DNA,topoisomerase': 1, 'PF00521': 1, 'PF08070': 1, 'PF02518': 1, 'PF01751': 1, 'remodeling,WINAC,#,topoisomerase': 1, 'PF08563': 1, 'PF07710': 1, 'PB014288': 1, 'PB021507': 1, 'PB024656': 1, 'PB024892': 1, 'PB000044': 1, 'PB000689': 1, 'PB009132': 1, 'PB009249': 1, 'modification,#,mRNA,\"Transformer-2': 1, '10)': 1, 'PB010855': 1, 'PF00643': 1, 'motif': 1, 'PB014213': 1, 'PB004946': 1, 'PF14634': 1, 'PB010963': 1, 'PB011449': 1, 'PB011628': 1, 'PCAF': 1, 'domain-associated': 1, 'PB011387': 1, 'PB012811': 1, 'PB015427': 1, 'PB003253': 1, 'PB006491': 1, 'PB006680': 1, 'H2AX': 1, 'serine': 1, 'modification,#,#,tRNA-yW': 1, 'enzyme': 1, 'UBE2H,\"H2Aub': 1, 'PF14075': 1, 'PB010464': 1, 'PB014814': 1, 'PB015951': 1, 'PB002665': 1, 'PF02207': 1, 'UBR5,\"H2Aub': 1, 'PF00658': 1, 'PB010556': 1, 'PB014783': 1, 'PB018259': 1, 'PB005733': 1, 'PB000664': 1, 'PB008293': 1, 'PB008878': 1, 'PF02182': 1, 'PF00240': 1, 'interaction': 1, 'PB176820': 1, 'PF00443': 1, 'PF14836': 1, 'PF14533': 1, 'PB024444': 1, 'USP22,\"H2A': 1, 'USP3,\"H2A': 1, 'PB020267': 1, 'PB012182': 1, 'PB045828': 1, 'PF12436': 1, 'PB002066': 1, 'complex\",#,vacuolar': 1, 'PF08265': 1, 'VRK1,\"H3S10ph': 1, 'MLL3,4\",\"H3K4': 1, 'repeat': 1, 'WDR77,\"H2Ame': 1, 'PF07525': 1, 'chaperone,ATAC,#,YEATS': 1, 'PB013347': 1, 'PB019332': 1, 'PF03366': 1, 'PF00244': 1, 'TF\",#,\"CG': 1, 'mCG': 1, 'motif\",zinc': 1, 'ZBTB7A,#,RNA modification,#,mRNA,Zinc finger and BTB domain-containing protein 7A (Factor binding IST protein 1) (FBI-1) (Factor that binds to inducer of short transcripts protein 1) (HIV-1 1st-binding protein 1) (Leukemia/lymphoma-related factor) (POZ and Krueppel erythroid myeloid ontogenic factor) (POK erythroid myeloid ontogenic factor) (Pokemon) (Pokemon 1) (TTF-I-interacting peptide 21) (TIP21) (Zinc finger protein 857A),ZBT7A_HUMAN,PF00651': 1, 'PF00096,ZBT7A_MOUSE,Alternative splicing': 1, 'type': 1, 'PB036867': 1, 'PF11569': 1, 'PB022595': 1, 'TF\",\"BHC': 1, 'motif,\"zinc': 1, '2\",ZMYM2_HUMAN,\"DUF3504': 1, 'PB010252': 1, 'PB002297': 1, '3\",ZMYM3_HUMAN,\"DUF3504': 1, 'PB024258': 1, 'PB005192': 1, 'PB007883': 1, 'PB008532': 1, 'PF12064': 1, 'PB019362': 1, 'remodeling,#,#,zinc': 1, 'PB017081': 1, 'PB027795': 1, 'PB003685': 1, 'PB040457': 1, 'PB027046': 1, 'PB000379': 1, 'PF04704': 1, 'PF13909': 1, 'PB014328': 1, 'PB019065': 1, 'PB022505': 1}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "def findnth(haystack, needle, n):\n",
        "    parts= haystack.split(needle, n+1)\n",
        "    if len(parts)<=n+1:\n",
        "        return -1\n",
        "    return len(haystack)-len(parts[-1])-len(needle)\n"
      ],
      "metadata": {
        "id": "R3Us3Us0Emxk"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "filteredFile = open(\"filtered.out\", \"w\")\n",
        "with open(\"test.out\", \"r\") as f:\n",
        "  while s := f.readline():\n",
        "    if s.startswith(\"#\"):\n",
        "      filteredFile.write(s)\n",
        "    else:\n",
        "      currentFamily = s[s.find(\"PF\"):findnth(s, \".\", 1)]\n",
        "      if dataset.get(currentFamily):\n",
        "        filteredFile.write(s)\n",
        "\n",
        "filteredFile.close()"
      ],
      "metadata": {
        "id": "jEq47-2F72ql"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}